<div xmlns:bean="http://ts.thomson.com/ua/bean" style="background-color: #000000; padding: 10px; color: #FFFFFF; font-family:  arial, sans-serif;">
<div style="height: 25px; width: 100px;">
<mat-icon data-mat-icon-name="clarivate" data-mat-icon-type="svg" aria-hidden="true" class="mat-icon notranslate clarivate-logo mat-icon-no-color" svgicon="clarivate" role="img">
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" focusable="false" preserveAspectRatio="xMidYMid meet" version="1.1" viewBox="0 0 92 18" height="100%" width="100%">
<title>Clarivate</title>
<g fill-rule="evenodd" fill="none" stroke-width="1" stroke="none">
<g fill-rule="nonzero" fill="#FFFFFF" transform="translate(-19.000000, -8.000000)">
<g transform="translate(1.000000, 1.000000)">
<g transform="translate(18.000000, 7.000000)">
<path d="M26.6812194,13.2997252 C28.2949641,13.3845068 29.7591419,12.3596036 30.2318671,10.8143235 L33.0910469,10.8143235 C32.3628779,13.742285 29.6973379,15.7705498 26.6812194,15.6916944 C24.8651813,15.7485215 23.1057028,15.0559504 21.8156554,13.7764967 C20.5256081,12.4970431 19.8185526,10.7433351 19.8604035,8.9268901 C19.8185367,7.11043445 20.5255848,5.35671042 21.8156337,4.07724261 C23.1056826,2.7977748 24.8651702,2.10519376 26.6812194,2.16201887 C29.6973451,2.0831702 32.3628861,4.11144761 33.0910469,7.03941847 L30.2318671,7.03941847 C29.7591419,5.49413828 28.2949641,4.46923515 26.6812194,4.55401674 C25.5407067,4.54133389 24.4470406,5.00703241 23.6657805,5.83803342 C22.8845203,6.66903442 22.487126,7.78932829 22.5700912,8.9268901 C22.4871313,10.064447 22.8845287,11.184734 23.6657888,12.0157277 C24.447049,12.8467215 25.540712,13.312413 26.6812194,13.2997252 L26.6812194,13.2997252 Z"/>
<polygon points="36.8477342 15.4674945 34.4371264 15.4674945 34.4371264 2.38628571 36.8477342 2.38628571"/>
<path d="M43.5380316,9.76781183 C43.5380316,9.15114738 43.0522397,8.75870193 42.1178663,8.75870193 C41.2070757,8.82048057 40.3236107,9.0957548 39.5389268,9.5622602 L38.5858286,7.73092465 C39.6930919,7.06514423 40.9567617,6.70409618 42.2486241,6.68441302 C44.1921648,6.68441302 45.9674886,7.58137515 45.9674886,9.84255787 L45.9674886,15.4674945 L43.5380316,15.4674945 L43.5380316,14.6639362 C42.9494098,15.3673061 42.0613957,15.7488608 41.146005,15.6917231 C40.3900371,15.7421039 39.645831,15.484584 39.0826798,14.9777443 C38.5195286,14.4709046 38.1853154,13.7578403 38.1560675,13.0007601 C38.1560675,11.3749188 39.3519468,10.3658089 41.2581433,10.3658089 L42.9400537,10.3658089 C43.3511665,10.3658089 43.5379934,10.1789247 43.5379934,9.86128266 L43.5380316,9.76781183 Z M43.5380316,12.4027631 L43.5380316,11.786089 C43.321275,11.9402969 43.0543042,12.0070182 42.790485,11.9729159 L41.8748173,11.9729159 C41.1086607,11.9729159 40.6602036,12.3092922 40.6602036,12.9072893 C40.6602036,13.486619 41.1272707,13.9164566 41.8748173,13.9164566 C42.303867,13.9699481 42.7348632,13.8332333 43.0546312,13.5422121 C43.3743992,13.2511908 43.5509896,12.8349402 43.5380316,12.4027631 L43.5380316,12.4027631 Z"/>
<path d="M52.6579104,9.3006299 L51.6301236,9.3006299 C50.3966703,9.3006299 49.8361037,9.8799596 49.8361037,11.3749188 L49.8361037,15.4674945 L47.4066179,15.4674945 L47.4066179,6.90863203 L49.7612715,6.90863203 L49.7612715,8.27284306 C50.1001881,7.34759017 51.0007831,6.74968637 51.9850429,6.79648425 C52.2115562,6.79362484 52.4375557,6.81874947 52.6579104,6.87128771 L52.6579104,9.3006299 Z"/>
<path d="M57.4232866,15.4674945 L54.9939445,15.4674945 L54.9939445,8.96425355 L53.7979312,8.96425355 L53.7979312,6.90863203 L57.4232866,6.90863203 L57.4232866,15.4674945 Z M56.0964104,2.38628347 C56.5129556,2.38558064 56.9126431,2.55074046 57.2071855,2.84528284 C57.5017278,3.13982521 57.6668877,3.53951271 57.6661848,3.95605794 C57.6609778,4.814152 56.9638874,5.50702694 56.1057775,5.50702694 C55.2476677,5.50702694 54.5505773,4.814152 54.5453585,3.95605794 C54.5417726,3.09491157 55.2352828,2.39302469 56.0964104,2.38628347 Z"/>
<polygon points="62.8802456 12.8512776 64.8236715 6.90863203 67.3838767 6.90863203 64.33787 15.4674945 61.3292077 15.4674945 58.4139588 6.90863203 61.0488526 6.90863203"/>
<path d="M72.8783522,9.76781183 C72.8783522,9.15114738 72.3924359,8.75870193 71.4580625,8.75870193 C70.5472784,8.82049943 69.6638208,9.09577238 68.8791325,9.5622602 L67.9261491,7.73092465 C69.0333627,7.06513461 70.2969953,6.70408494 71.5888202,6.68441302 C73.5323609,6.68441302 75.3076848,7.58137515 75.3076848,9.84255787 L75.3076848,15.4674945 L72.8783522,15.4674945 L72.8783522,14.6639362 C72.2896971,15.3672586 71.4017035,15.7487939 70.4863256,15.6916944 C69.7303436,15.7420941 68.986117,15.4845835 68.4229464,14.9777426 C67.8597758,14.4709017 67.5255471,13.7578265 67.4962924,13.0007314 C67.4962924,11.3748901 68.6923056,10.3657802 70.5984925,10.3657802 L72.2802785,10.3657802 C72.6913914,10.3657802 72.8783426,10.1788959 72.8783426,9.86125395 L72.8783522,9.76781183 Z M72.8783522,12.4027631 L72.8783522,11.786089 C72.6615874,11.9402794 72.3946236,12.006999 72.1308056,11.9729159 L71.2151378,11.9729159 C70.4488664,11.9729159 70.0004093,12.3092922 70.0004093,12.9072893 C70.0004093,13.486619 70.4676008,13.9164566 71.2151378,13.9164566 C71.6441833,13.9699279 72.0751679,13.8332061 72.394931,13.5421893 C72.7146941,13.2511724 72.8912904,12.8349341 72.8783522,12.4027631 L72.8783522,12.4027631 Z"/>
<path d="M79.9985637,3.65701635 L79.9985637,6.90863203 L81.8485762,6.90863203 L81.8485762,8.96425355 L79.9985637,8.96425355 L79.9985637,12.8325433 C79.9985637,13.3745286 80.4282865,13.4492747 80.8020646,13.4492747 C81.418729,13.4492747 81.9232553,13.3932056 81.9232553,13.3932056 L81.9232553,15.4674945 C81.9232553,15.4674945 80.7459859,15.5796423 80.0357932,15.5796423 C78.8211795,15.5796423 77.5691354,15.3740811 77.5691354,13.6548263 L77.5691354,8.96425355 L76.22363,8.96425355 L76.22363,6.90863203 L77.5691163,6.90863203 L77.5691163,3.65701635 L79.9985637,3.65701635 Z"/>
<path d="M91.5475359,11.8981794 L85.1751771,11.8981794 C85.3031552,12.7204351 85.9194738,13.3815958 86.7307322,13.5669136 C87.5419905,13.7522314 88.3842592,13.4242593 88.856573,12.7391298 L91.3793191,13.0007601 C90.4933306,14.9536735 88.3725856,16.0321989 86.2725384,15.5978628 C84.1724912,15.1635268 82.6535603,13.3322361 82.6149337,11.188092 C82.567807,9.98960603 83.0193435,8.82515843 83.8621298,7.97175307 C84.7049161,7.1183477 85.8636189,6.65226701 87.0626009,6.68438432 C89.4171397,6.68438432 91.5475359,8.21674525 91.5475359,11.8047373 L91.5475359,11.8981794 Z M85.1189837,10.2723955 L88.9499864,10.2723955 C88.8059978,9.34848922 87.9973156,8.67588427 87.0626392,8.70263282 C86.1147409,8.67034807 85.2869107,9.33893202 85.1189837,10.2723955 Z"/>
<path d="M12.6820274,9.85132226 C10.6885049,11.8556057 8.11575776,13.1830474 5.32707189,13.6462054 C5.77780891,14.8758828 6.40559556,16.0331959 7.19058498,17.0815575 C10.3903106,16.2989026 13.3004311,14.619851 15.5794509,12.2414352 C15.0651988,11.0374085 14.3787129,9.91449973 13.5415112,8.90790698 C13.271174,9.23328 12.9846793,9.54772625 12.6820274,9.85124571"/>
<path d="M5.32722498,13.6460332 C4.21304566,10.5852611 4.22627578,7.22758116 5.3645406,4.17568425 C4.07298567,3.95802358 2.75701252,3.92542943 1.45626578,4.07888372 C0.536506305,7.22972054 0.525090565,10.5762272 1.42333236,13.7332656 C2.72217338,13.8896343 4.03666286,13.8602685 5.32722498,13.6460524"/>
<path d="M7.25919787,0.751096346 C6.46424104,1.79423541 5.8259601,2.94792723 5.36457887,4.17561728 C8.13918123,4.64457901 10.6977043,5.96938232 12.6820178,7.96458738 C12.9850078,8.2673988 13.2715025,8.58184824 13.5415017,8.90793568 C14.3788161,7.90142183 15.0653154,6.77849641 15.5794509,5.57440744 C13.3171586,3.21319806 10.4324457,1.54091896 7.25919787,0.751096346"/>
</g>
</g>
</g>
</g>
</svg>
</mat-icon>
</div>
<path d="M5.32722498,13.6460332 C4.21304566,10.5852611 4.22627578,7.22758116 5.3645406,4.17568425 C4.07298567,3.95802358 2.75701252,3.92542943 1.45626578,4.07888372 C0.536506305,7.22972054 0.525090565,10.5762272 1.42333236,13.7332656 C2.72217338,13.8896343 4.03666286,13.8602685 5.32722498,13.6460524"></path>
<path d="M7.25919787,0.751096346 C6.46424104,1.79423541 5.8259601,2.94792723 5.36457887,4.17561728 C8.13918123,4.64457901 10.6977043,5.96938232 12.6820178,7.96458738 C12.9850078,8.2673988 13.2715025,8.58184824 13.5415017,8.90793568 C14.3788161,7.90142183 15.0653154,6.77849641 15.5794509,5.57440744 C13.3171586,3.21319806 10.4324457,1.54091896 7.25919787,0.751096346"></path>
</div><div style="width: 180px; height: 40px; padding: 10px;">
<mat-icon data-mat-icon-name="wos" data-mat-icon-type="svg" aria-hidden="true" style="width: 180px;" class="mat-icon notranslate mat-icon-no-color" svgicon="wos" role="img">
<svg xmlns="http://www.w3.org/2000/svg" focusable="false" preserveAspectRatio="xMidYMid meet" viewBox="0 0 1868 332" height="100%" width="100%">
<title>Web of Science</title>
<path d="M 108.92,75.00            C 108.92,75.00 147.34,75.00 147.34,75.00              147.34,75.00 182.84,208.12 182.84,208.12              182.84,208.12 183.11,208.12 183.11,208.12              183.11,208.12 212.80,75.00 212.80,75.00              212.80,75.00 250.42,75.00 250.42,75.00              250.42,75.00 205.11,261.00 205.11,261.00              205.11,261.00 162.44,261.00 162.44,261.00              162.44,261.00 127.47,127.34 127.47,127.34              127.47,127.34 93.28,261.00 93.28,261.00              93.28,261.00 50.61,261.00 50.61,261.00              50.61,261.00 4.77,75.00 4.77,75.00              4.77,75.00 44.52,75.00 44.52,75.00              44.52,75.00 73.14,208.12 73.14,208.12              73.14,208.12 73.41,208.12 73.41,208.12              73.41,208.12 108.92,75.00 108.92,75.00 Z            M 379.00,210.00            C 379.00,210.00 288.41,210.00 288.41,210.00              289.29,216.78 292.30,222.48 297.44,227.09              302.57,231.70 308.86,234.00 316.30,234.00              327.80,234.00 335.95,230.00 340.73,222.00              340.73,222.00 376.61,225.58 376.61,225.58              371.30,238.39 363.32,248.00 352.69,254.41              342.06,260.80 329.75,264.00 315.77,264.00              297.52,264.00 282.32,257.98 270.19,245.94              258.06,233.90 252.00,218.59 252.00,200.00              252.00,181.42 257.98,166.11 269.94,154.06              281.89,142.02 296.99,136.00 315.23,136.00              333.47,136.00 348.66,142.21 360.80,154.64              372.93,167.06 379.00,185.51 379.00,210.00 Z            M 333.30,171.28            C 328.34,167.09 322.32,165.00 315.23,165.00              308.15,165.00 302.12,167.01 297.16,171.03              292.20,175.04 289.02,180.37 287.61,187.00              287.61,187.00 342.06,187.00 342.06,187.00              341.19,180.71 338.27,175.47 333.30,171.28 Z            M 433.72,245.31            C 433.72,245.31 433.72,261.00 433.72,261.00              433.72,261.00 399.00,261.00 399.00,261.00              399.00,261.00 399.00,75.00 399.00,75.00              399.00,75.00 433.45,75.00 433.45,75.00              433.45,75.00 433.45,134.94 433.45,134.94              433.45,137.95 433.45,141.85 433.45,146.64              433.45,151.42 433.45,154.17 433.45,154.88              436.43,149.39 441.34,144.87 448.17,141.33              455.02,137.78 462.82,136.00 471.59,136.00              488.26,136.00 502.16,142.07 513.30,154.20              524.43,166.33 530.00,181.59 530.00,200.00              530.00,218.41 524.37,233.68 513.11,245.81              501.86,257.94 487.88,264.00 471.17,264.00              462.91,264.00 455.39,262.22 448.62,258.66              441.85,255.09 436.88,250.65 433.72,245.31 Z            M 485.39,176.23            C 479.65,169.98 472.34,166.86 463.45,166.86              454.58,166.86 447.27,169.98 441.53,176.23              435.79,182.48 432.92,190.37 432.92,199.88              432.92,209.20 435.79,217.07 441.53,223.50              447.27,229.93 454.58,233.14 463.45,233.14              472.34,233.14 479.65,229.93 485.39,223.50              491.13,217.07 494.00,209.20 494.00,199.88              494.00,190.37 491.13,182.48 485.39,176.23 Z            M 707.69,245.81            C 695.19,257.94 679.86,264.00 661.72,264.00              643.58,264.00 628.26,257.94 615.75,245.81              603.25,233.68 597.00,218.41 597.00,200.00              597.00,181.59 603.25,166.33 615.75,154.20              628.26,142.07 643.58,136.00 661.72,136.00              679.86,136.00 695.19,142.07 707.69,154.20              720.20,166.33 726.45,181.59 726.45,200.00              726.45,218.41 720.20,233.68 707.69,245.81 Z            M 661.84,166.86            C 652.85,166.86 645.54,170.00 639.91,176.28              634.27,182.55 631.45,190.46 631.45,200.00              631.45,209.54 634.27,217.45 639.91,223.73              645.54,230.01 652.85,233.14 661.84,233.14              670.66,233.14 677.89,230.01 683.53,223.73              689.18,217.45 692.00,209.54 692.00,200.00              692.00,190.46 689.18,182.55 683.53,176.28              677.89,170.00 670.66,166.86 661.84,166.86 Z            M 757.00,139.00            C 757.00,139.00 757.00,104.34 757.00,104.34              757.00,93.15 760.17,84.98 766.52,79.83              772.86,74.67 781.14,72.09 791.36,72.09              791.36,72.09 816.50,73.69 816.50,73.69              816.50,73.69 816.50,105.33 816.50,105.33              809.47,104.45 804.72,104.00 802.27,104.00              798.58,104.00 795.85,104.71 794.09,106.14              792.33,107.57 791.45,110.24 791.45,114.16              791.45,114.16 791.45,139.00 791.45,139.00              791.45,139.00 815.00,139.00 815.00,139.00              815.00,139.00 815.00,168.00 815.00,168.00              815.00,168.00 791.45,168.00 791.45,168.00              791.45,168.00 791.45,260.50 791.45,260.50              791.45,260.50 757.00,260.50 757.00,260.50              757.00,260.50 757.00,168.00 757.00,168.00              757.00,168.00 738.00,168.00 738.00,168.00              738.00,168.00 738.00,139.00 738.00,139.00              738.00,139.00 757.00,139.00 757.00,139.00 Z            M 1004.30,97.27            C 1004.30,97.27 983.11,124.06 983.11,124.06              969.92,111.36 956.73,105.00 943.53,105.00              937.11,105.00 931.59,106.60 926.95,109.80              922.32,112.98 920.00,117.24 920.00,122.56              920.00,131.06 926.74,138.42 940.22,144.62              940.22,144.62 955.66,151.80 955.66,151.80              971.60,159.25 983.48,167.31 991.28,176.00              999.09,184.68 1003.00,195.67 1003.00,208.97              1003.00,223.87 997.13,236.76 985.39,247.66              973.66,258.55 958.09,264.00 938.67,264.00              913.45,264.00 891.31,254.56 872.25,235.67              872.25,235.67 892.52,208.02 892.52,208.02              899.27,214.90 906.74,220.29 914.92,224.17              923.10,228.06 930.83,230.00 938.11,230.00              945.94,230.00 952.21,228.09 956.92,224.28              961.64,220.47 964.00,215.64 964.00,209.80              964.00,203.95 961.65,198.93 956.95,194.77              952.25,190.60 944.85,186.21 934.73,181.61              934.73,181.61 921.70,175.48 921.70,175.48              894.57,162.73 881.00,145.36 881.00,123.38              881.00,107.59 887.05,94.92 899.17,85.36              911.29,75.79 925.91,71.00 943.06,71.00              963.93,71.00 984.34,79.76 1004.30,97.27 Z            M 1082.92,166.86            C 1073.91,166.86 1066.58,170.00 1060.92,176.28              1055.27,182.55 1052.45,190.46 1052.45,200.00              1052.45,209.54 1055.27,217.45 1060.92,223.73              1066.58,230.01 1073.91,233.14 1082.92,233.14              1095.82,233.14 1104.74,226.95 1109.69,214.58              1109.69,214.58 1142.02,224.66 1142.02,224.66              1138.13,236.72 1130.75,246.29 1119.89,253.38              1109.02,260.46 1096.61,264.00 1082.66,264.00              1064.46,264.00 1049.13,257.94 1036.67,245.81              1024.22,233.68 1018.00,218.41 1018.00,200.00              1018.00,181.59 1024.22,166.33 1036.67,154.20              1049.13,142.07 1064.46,136.00 1082.66,136.00              1096.61,136.00 1109.02,139.59 1119.89,146.77              1130.75,153.95 1138.13,163.56 1142.02,175.61              1142.02,175.61 1109.69,185.42 1109.69,185.42              1107.39,179.77 1103.91,175.26 1099.22,171.91              1094.54,168.54 1089.11,166.86 1082.92,166.86 Z            M 1170.31,112.67            C 1166.10,108.45 1164.00,103.27 1164.00,97.12              1164.00,90.98 1166.15,85.76 1170.45,81.45              1174.75,77.15 1179.89,75.00 1185.86,75.00              1192.02,75.00 1197.24,77.15 1201.55,81.45              1205.85,85.76 1208.00,90.98 1208.00,97.12              1208.00,103.27 1205.85,108.45 1201.55,112.67              1197.24,116.89 1192.02,119.00 1185.86,119.00              1179.71,119.00 1174.53,116.89 1170.31,112.67 Z            M 1205.00,139.00            C 1205.00,139.00 1205.00,261.00 1205.00,261.00              1205.00,261.00 1170.55,261.00 1170.55,261.00              1170.55,261.00 1170.55,168.00 1170.55,168.00              1170.55,168.00 1153.50,168.00 1153.50,168.00              1153.50,168.00 1153.50,139.00 1153.50,139.00              1153.50,139.00 1205.00,139.00 1205.00,139.00 Z            M 1353.00,210.00            C 1353.00,210.00 1262.41,210.00 1262.41,210.00              1263.29,216.78 1266.30,222.48 1271.44,227.09              1276.57,231.70 1282.86,234.00 1290.30,234.00              1301.80,234.00 1309.95,230.00 1314.73,222.00              1314.73,222.00 1350.61,225.58 1350.61,225.58              1345.30,238.39 1337.32,248.00 1326.69,254.41              1316.06,260.80 1303.75,264.00 1289.77,264.00              1271.52,264.00 1256.32,257.98 1244.19,245.94              1232.06,233.90 1226.00,218.59 1226.00,200.00              1226.00,181.42 1231.98,166.11 1243.94,154.06              1255.89,142.02 1270.99,136.00 1289.23,136.00              1307.47,136.00 1322.66,142.21 1334.80,154.64              1346.93,167.06 1353.00,185.51 1353.00,210.00 Z            M 1307.30,171.28            C 1302.34,167.09 1296.32,165.00 1289.23,165.00              1282.15,165.00 1276.12,167.01 1271.16,171.03              1266.20,175.04 1263.02,180.37 1261.61,187.00              1261.61,187.00 1316.06,187.00 1316.06,187.00              1315.19,180.71 1312.27,175.47 1307.30,171.28 Z            M 1406.39,153.36            C 1415.92,141.79 1428.27,136.00 1443.45,136.00              1456.87,136.00 1467.72,140.04 1476.02,148.11              1484.30,156.17 1488.45,167.20 1488.45,181.20              1488.45,181.20 1488.45,261.00 1488.45,261.00              1488.45,261.00 1454.00,261.00 1454.00,261.00              1454.00,261.00 1454.00,189.86 1454.00,189.86              1454.00,182.57 1452.00,177.11 1448.00,173.47              1444.01,169.82 1439.09,168.00 1433.25,168.00              1423.50,168.00 1414.90,172.62 1407.45,181.86              1407.45,181.86 1407.45,261.00 1407.45,261.00              1407.45,261.00 1373.00,261.00 1373.00,261.00              1373.00,261.00 1373.00,139.00 1373.00,139.00              1373.00,139.00 1406.39,139.00 1406.39,139.00              1406.39,139.00 1406.39,153.36 1406.39,153.36 Z            M 1573.92,166.86            C 1564.91,166.86 1557.58,170.00 1551.92,176.28              1546.27,182.55 1543.45,190.46 1543.45,200.00              1543.45,209.54 1546.27,217.45 1551.92,223.73              1557.58,230.01 1564.91,233.14 1573.92,233.14              1586.82,233.14 1595.74,226.95 1600.69,214.58              1600.69,214.58 1633.02,224.66 1633.02,224.66              1629.13,236.72 1621.75,246.29 1610.89,253.38              1600.02,260.46 1587.61,264.00 1573.66,264.00              1555.46,264.00 1540.13,257.94 1527.67,245.81              1515.22,233.68 1509.00,218.41 1509.00,200.00              1509.00,181.59 1515.22,166.33 1527.67,154.20              1540.13,142.07 1555.46,136.00 1573.66,136.00              1587.61,136.00 1600.02,139.59 1610.89,146.77              1621.75,153.95 1629.13,163.56 1633.02,175.61              1633.02,175.61 1600.69,185.42 1600.69,185.42              1598.39,179.77 1594.91,175.26 1590.22,171.91              1585.54,168.54 1580.11,166.86 1573.92,166.86 Z            M 1773.00,210.00            C 1773.00,210.00 1682.41,210.00 1682.41,210.00              1683.29,216.78 1686.30,222.48 1691.44,227.09              1696.57,231.70 1702.86,234.00 1710.30,234.00              1721.80,234.00 1729.95,230.00 1734.73,222.00              1734.73,222.00 1770.61,225.58 1770.61,225.58              1765.30,238.39 1757.32,248.00 1746.69,254.41              1736.06,260.80 1723.75,264.00 1709.77,264.00              1691.52,264.00 1676.32,257.98 1664.19,245.94              1652.06,233.90 1646.00,218.59 1646.00,200.00              1646.00,181.42 1651.98,166.11 1663.94,154.06              1675.89,142.02 1690.99,136.00 1709.23,136.00              1727.47,136.00 1742.66,142.21 1754.80,154.64              1766.93,167.06 1773.00,185.51 1773.00,210.00 Z            M 1727.30,171.28            C 1722.34,167.09 1716.32,165.00 1709.23,165.00              1702.15,165.00 1696.12,167.01 1691.16,171.03              1686.20,175.04 1683.02,180.37 1681.61,187.00              1681.61,187.00 1736.06,187.00 1736.06,187.00              1735.19,180.71 1732.27,175.47 1727.30,171.28 Z" stroke-width="1" stroke="black" fill="black"/>
<path d="M 1769.00,76.00            C 1769.00,76.00 1800.00,76.00 1800.00,76.00              1800.00,76.00 1800.00,82.42 1800.00,82.42              1800.00,82.42 1788.00,82.42 1788.00,82.42              1788.00,82.42 1788.00,115.00 1788.00,115.00              1788.00,115.00 1780.84,115.00 1780.84,115.00              1780.84,115.00 1780.84,82.42 1780.84,82.42              1780.84,82.42 1769.00,82.42 1769.00,82.42              1769.00,82.42 1769.00,76.00 1769.00,76.00 Z            M 1827.27,106.64            C 1827.27,106.64 1818.95,106.64 1818.95,106.64              1818.95,106.64 1811.16,87.42 1811.16,87.42              1811.16,87.42 1811.16,115.00 1811.16,115.00              1811.16,115.00 1804.00,115.00 1804.00,115.00              1804.00,115.00 1804.00,76.00 1804.00,76.00              1804.00,76.00 1814.61,76.00 1814.61,76.00              1814.61,76.00 1823.25,98.11 1823.25,98.11              1823.25,98.11 1832.08,76.00 1832.08,76.00              1832.08,76.00 1842.00,76.00 1842.00,76.00              1842.00,76.00 1842.00,115.00 1842.00,115.00              1842.00,115.00 1834.84,115.00 1834.84,115.00              1834.84,115.00 1834.84,87.42 1834.84,87.42              1834.84,87.42 1827.27,106.64 1827.27,106.64 Z" stroke-width="1" stroke="black" fill="black"/>
</svg>
</mat-icon>
</div><table cellspacing="0" cellpadding="2" border="0">
<tr>
<td>49 record(s) printed from Clarivate Web of Science</td>
</tr>
</table><hr>
<table>
<tr>
<td><b>Record 1 of 49</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Endothelial Brg1 fine-tunes Notch signaling during zebrafish heart regeneration</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Xiao, CL (Xiao, Chenglu); Hou, JJ (Hou, Junjie); Wang, F (Wang, Fang); Song, YB (Song, Yabing); Zheng, JY (Zheng, Jiyuan); Luo, LF (Luo, Lingfei); Wang, JB (Wang, Jianbin); Ding, WQ (Ding, Wanqiu); Zhu, XJ (Zhu, Xiaojun); Xiong, JW (Xiong, Jing-Wei)</td>
</tr>

<tr xmlns:bean="http://ts.thomson.com/ua/bean" xmlns:exsl="http://exslt.org/common" xmlns:set="http://exslt.org/sets">
<td><b>Source:</b> 
<value>NPJ REGENERATIVE MEDICINE</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>8</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>1</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>21</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1038/s41536-023-00293-4</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2023 APR 7</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>6</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>7</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
2</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
24</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>71</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Myocardial Brg1 is essential for heart regeneration in zebrafish, but it remains unknown whether and how endothelial Brg1 plays a role in heart regeneration. Here, we found that both brg1 mRNA and protein were induced in cardiac endothelial cells after ventricular resection and endothelium-specific overexpression of dominant-negative Xenopus Brg1 (dn-xbrg1) inhibited myocardial proliferation and heart regeneration and increased cardiac fibrosis. RNA-seq and ChIP-seq analysis revealed that endothelium-specific overexpression of dn-xbrg1 changed the levels of H3K4me3 modifications in the promoter regions of the zebrafish genome and induced abnormal activation of Notch family genes upon injury. Mechanistically, Brg1 interacted with lysine demethylase 7aa (Kdm7aa) to fine-tune the level of H3K4me3 within the promoter regions of Notch family genes and thus regulated notch gene transcription. Together, this work demonstrates that the Brg1-Kdm7aa-Notch axis in cardiac endothelial cells, including the endocardium, regulates myocardial proliferation and regeneration via modulating the H3K4me3 of the notch promoters in zebrafish.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000964203300001</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>37029137</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
PEG-PLA NANOPARTICLES; CARDIOMYOCYTE PROLIFERATION; CHROMATIN; EXPRESSION; COMPLEX; SIRNA; NEUROGENESIS; ENDOCARDIUM; DEMETHYLASE; ACTIVATION</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Xiao, Chenglu; Hou, Junjie; Zheng, Jiyuan; Ding, Wanqiu; Zhu, Xiaojun; Xiong, Jing-Wei] Peking Univ, Inst Mol Med, Coll Future Technol, Beijing Key Lab Cardiometab Mol Med, Beijing 100871, Peoples R China. <br>
[Xiao, Chenglu; Hou, Junjie; Zheng, Jiyuan; Ding, Wanqiu; Zhu, Xiaojun; Xiong, Jing-Wei] Peking Univ, State Key Lab Nat &amp; Biomimet Drugs, Beijing 100871, Peoples R China. <br>
[Xiao, Chenglu] China Agr Univ, Coll Vet Med, Natl Key Lab Vet Publ Hlth Secur, Beijing 100193, Peoples R China. <br>
[Wang, Fang] Zhejiang Univ Technol, Coll Pharmaceut Sci, Hangzhou 310014, Peoples R China. <br>
[Song, Yabing; Wang, Jianbin] Tsinghua Univ, Sch Life Sci, Beijing 100084, Peoples R China. <br>
[Luo, Lingfei] Southwest Univ, Inst Dev Biol &amp; Regenerat Med, Beibei 400715, Chongqing, Peoples R China. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Ding, WQ; Zhu, XJ; Xiong, JW (corresponding author), Peking Univ, Inst Mol Med, Coll Future Technol, Beijing Key Lab Cardiometab Mol Med, Beijing 100871, Peoples R China.<br>Ding, WQ; Zhu, XJ; Xiong, JW (corresponding author), Peking Univ, State Key Lab Nat &amp; Biomimet Drugs, Beijing 100871, Peoples R China.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
dingwq@pku.edu.cn; zhuxiaojun@pku.edu.cn; jingwei_xiong@pku.edu.cn</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Peking University; Peking University; China Agricultural University; Zhejiang University of Technology; Tsinghua University; Southwest University - China</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Ding, Wanqiu</display_name>&nbsp;</td><td>JCE-9921-2023&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Xiong, Jing-Wei</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-8438-4782&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Xiao, Chenglu</display_name>&nbsp;</td><td>JPX-4798-2023&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Xiong, Jing-Wei</display_name>&nbsp;</td><td>A-7113-2011&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Ding, Wanqiu</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-6650-8662&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>luo, ling</display_name>&nbsp;</td><td>MIO-2796-2025&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>, Jingying</display_name>&nbsp;</td><td>&nbsp;</td><td>0009-0009-4051-4739&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>NATURE PORTFOLIO</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>HEIDELBERGER PLATZ 3, BERLIN, 14197, GERMANY</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Cell &amp; Tissue Engineering; Engineering, Biomedical</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Cell Biology; Engineering</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>C8FE1</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>2057-3995</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>NPJ REGEN MED</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>npj Regen. Med.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>13</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Key R&amp;D Program of China</grant_agency>&nbsp;</td><td>
<div>2019YFA0801602&nbsp;</div>
<div>2018YFA0800501&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Natural Science Foundation of China</grant_agency>&nbsp;</td><td>
<div>32230032&nbsp;</div>
<div>31701272&nbsp;</div>
<div>31730061&nbsp;</div>
<div>31430059&nbsp;</div>
<div>81870198&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Peking University Boya Program</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Peking-Tsinghua Center for Life Sciences</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>The authors thank Dr. I.C. Bruce (Guest Professor at the Institute of Molecular Medicine, Peking University) for commenting and revising the manuscript, and members of Dr. Jing-Wei Xiong's and Dr. Yan-Yi Huang's laboratories for helpful discussions and technical assistance. The authors acknowledge Chenyang Geng, Yun Zhang, Jing Sun, and Yang Xu for technical assistance at the Beijing Advanced Innovation Center of Genomics and the High-Throughput Sequencing Center at Peking University; Fangjin Chen, Ting Fang, and Wenzhong Zhang for technical help at the Computing Platform of the Center for Life Sciences of Peking University; Li-Ying Du and Hong-Xia Lv for technical help at the Flow Cytometry Core at the National Center for Protein Sciences at Peking University; Liqin Fu for technical help at the Optical Imaging Group of Core Facility, School of Life Sciences and National Biomedical Imaging Center of Peking University. This work was supported by the National Key R&amp;D Program of China (2019YFA0801602 and 2018YFA0800501) and grants from the National Natural Science Foundation of China (32230032, 31701272, 31730061, 31430059, and 81870198). C.X. was supported in part by postdoctoral fellowships from Peking University Boya Program and Peking-Tsinghua Center for Life Sciences.</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>gold, Green Submitted, Green Published</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 2 of 49</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Automated detection of diabetic subject using pre-trained 2D-CNN models with frequency spectrum images extracted from heart rate signals</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Yildirim, O (Yildirim, Ozal); Talo, M (Talo, Muhammed); Ay, B (Ay, Betul); Baloglu, UB (Baloglu, Ulas Baran); Aydin, G (Aydin, Galip); Acharya, UR (Acharya, U. Rajendra)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>COMPUTERS IN BIOLOGY AND MEDICINE</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>113</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>103387</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1016/j.compbiomed.2019.103387</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2019 OCT</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>102</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>115</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
1</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
51</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>52</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
In this study, a deep-transfer learning approach is proposed for the automated diagnosis of diabetes mellitus (DM), using heart rate (HR) signals obtained from electrocardiogram (ECG) data. Recent progress in deep learning has contributed significantly to improvement in the quality of healthcare. In order for deep learning models to perform well, large datasets are required for training. However, a difficulty in the biomedical field is the lack of clinical data with expert annotation. A recent, commonly implemented technique to train deep learning models using small datasets is to transfer the weighting, developed from a large dataset, to the current model. This deep learning transfer strategy is generally employed for two-dimensional signals. Herein, the weighting of models pre-trained using two-dimensional large image data was applied to one-dimensional HR signals. The one-dimensional HR signals were then converted into frequency spectrum images, which were utilized for application to well-known pre-trained models, specifically: AlexNet, VggNet, ResNet, and DenseNet. The DenseNet pre-trained model yielded the highest classification average accuracy of 97.62%, and sensitivity of 100%, to detect DM subjects via HR signal recordings. In the future, we intend to further test this developed model by utilizing additional data along with cloud-based storage to diagnose DM via heart signal analysis.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000496898100001</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>31421276</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Diabetes mellitus; Heart rate signals; Deep learning; Transfer learning</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
DISCRETE WAVELET TRANSFORM; COMPUTER-AIDED DIAGNOSIS; CLASSIFICATION; MELLITUS; RETINOPATHY; FEATURES; INDEX</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Yildirim, Ozal; Talo, Muhammed] Munzur Univ, Dept Comp Engn, Tunceli, Turkey. <br>
[Ay, Betul; Aydin, Galip] Firat Univ, Dept Comp Engn, Elazig, Turkey. <br>
[Baloglu, Ulas Baran] Univ Bristol, Dept Comp Sci, Bristol, Avon, England. <br>
[Acharya, U. Rajendra] Ngee Ann Polytech, Dept Elect &amp; Comp Engn, Singapore, Singapore. <br>
[Acharya, U. Rajendra] Singapore Sch Social Sci, Sch Sci &amp; Technol, Dept Biomed Engn, Singapore, Singapore. <br>
[Acharya, U. Rajendra] Taylors Univ, Fac Hlth &amp; Med Sci, Sch Med, Subang Jaya 47500, Malaysia. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Yildirim, O (corresponding author), Munzur Univ, Dept Comp Engn, Tunceli, Turkey.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
oyildirim@munzur.edu.tr</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Munzur University; Firat University; University of Bristol; Taylor's University</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Baloglu, Ulas Baran</display_name>&nbsp;</td><td>P-7444-2015&nbsp;</td><td>0000-0002-2045-9922&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Acharya, U Rajendra</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-2689-8552&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Aydin, Galip</display_name>&nbsp;</td><td>W-1473-2018&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>YILDIRIM, &Ouml;zal</display_name>&nbsp;</td><td>AAV-5584-2020&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Acharya, Rajendra</display_name>&nbsp;</td><td>E-3791-2010&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>BALOGLU, Ulas</display_name>&nbsp;</td><td>P-7444-2015&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Talo, Muhammed</display_name>&nbsp;</td><td>Y-6047-2019&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>AY, BET&Uuml;L</display_name>&nbsp;</td><td>JAN-6412-2023&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>PERGAMON-ELSEVIER SCIENCE LTD</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>THE BOULEVARD, LANGFORD LANE, KIDLINGTON, OXFORD OX5 1GB, ENGLAND</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Biology; Computer Science, Interdisciplinary Applications; Engineering, Biomedical; Mathematical &amp; Computational Biology</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Life Sciences &amp; Biomedicine - Other Topics; Computer Science; Engineering; Mathematical &amp; Computational Biology</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>JN4VW</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0010-4825</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1879-0534</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>COMPUT BIOL MED</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Comput. Biol. Med.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>10</value>
</td>
</tr>

<tr>
<td>
<b>Open Access:</b>

<value>Green Submitted</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 3 of 49</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Multi-level semantic adaptation for few-shot segmentation on cardiac image sequences</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Guo, SD (Guo, Saidi); Xu, L (Xu, Lin); Feng, C (Feng, Cheng); Xiong, HH (Xiong, Huahua); Gao, ZF (Gao, Zhifan); Zhang, HY (Zhang, Heye)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>MEDICAL IMAGE ANALYSIS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>73</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>102170</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1016/j.media.2021.102170</value>&nbsp;&nbsp;<b>Early Access Date:</b> 
<value>AUG 2021</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2021 OCT</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>42</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>44</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
4</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
46</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>109</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Obtaining manual labels is time-consuming and labor-intensive on cardiac image sequences. Few-shot segmentation can utilize limited labels to learn new tasks. However, it suffers from two challenges: spatial-temporal distribution bias and long-term information bias. These challenges derive from the impact of the time dimension on cardiac image sequences, resulting in serious over-adaptation. In this paper, we propose the multi-level semantic adaptation (MSA) for few-shot segmentation on cardiac image sequences. The MSA addresses the two biases by exploring the domain adaptation and the weight adaptation on the semantic features in multiple levels, including sequence-level, frame-level, and pixel-level. First, the MSA proposes the dual-level feature adjustment for domain adaptation in spatial and temporal directions. This adjustment explicitly aligns the frame-level feature and the sequence-level feature to improve the model adaptation on diverse modalities. Second, the MSA explores the hierarchical attention metric for weight adaptation in the frame-level feature and the pixel-level feature. This metric focuses on the similar frame and the target region to promote the model discrimination on the border features. The extensive experiments demonstrate that our MSA is effective in f ew-shot segmentation on cardiac image sequences with three modalities, i.e. MR, CT, and Echo (e.g. the average Dice is 0.9243), as well as superior to the ten state-of-the-art methods. (c) 2021 Elsevier B.V. All rights reserved.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000697062100005</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>34380105</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Cardiac image sequences; Few-shot segmentation; Domain adaptation; Attention mechanism</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
CAROTID-ARTERY WALL; LEFT-VENTRICLE; MODALITY ADAPTATION; DOMAIN ADAPTATION; ATTENTION; NETWORK; MR; QUANTIFICATION; REGRESSION; DIAGNOSIS</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Guo, Saidi; Gao, Zhifan; Zhang, Heye] Sun Yat Sen Univ, Sch Biomed Engn, Guangzhou, Peoples R China. <br>
[Xu, Lin] Gen Hosp Southern Theatre Command, PLA, Guangzhou, Guangdong, Peoples R China. <br>
[Xu, Lin] Southern Med Univ, Sch Clin Med 1, Guangzhou, Guangdong, Peoples R China. <br>
[Feng, Cheng] Third Peoples Hosp Shenzhen, Dept Ultrasound, Shenzhen, Guangdong, Peoples R China. <br>
[Xiong, Huahua] Shenzhen Univ, Shenzhen Peoples Hosp 2, Dept Ultrasound, Affiliated Hosp 1, Shenzhen, Guangdong, Peoples R China. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Gao, ZF; Zhang, HY (corresponding author), Sun Yat Sen Univ, Sch Biomed Engn, Guangzhou, Peoples R China.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
gaozhifan@mail.sysu.edu.cn; zhangheye@mail.sysu.edu.cn</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Sun Yat Sen University; Southern Medical University - China; The Third People's Hospital of Shenzhen; Shenzhen University</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Zhang, Heye</display_name>&nbsp;</td><td>JPK-4651-2023&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Gao, Zhifan</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-1576-4439&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>XIONG, HUAHUA</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-7468-7130&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Xu, Lin</display_name>&nbsp;</td><td>JOK-8307-2023&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>guo, saidi</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-2945-2599&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Gao, Zhifan</display_name>&nbsp;</td><td>O-9082-2019&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Xu, Lin</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-7750-8011&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Xiong, Huhahua</display_name>&nbsp;</td><td>HGV-0570-2022&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>ELSEVIER</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>RADARWEG 29, 1043 NX AMSTERDAM, NETHERLANDS</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Artificial Intelligence; Computer Science, Interdisciplinary Applications; Engineering, Biomedical; Radiology, Nuclear Medicine &amp; Medical Imaging</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Engineering; Radiology, Nuclear Medicine &amp; Medical Imaging</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>UR9LU</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>1361-8415</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1361-8423</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>MED IMAGE ANAL</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Med. Image Anal.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>19</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Key-Area Research and Development Program of Guangdong Province</grant_agency>&nbsp;</td><td>
<div>2019B010110001&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Key Program for International Cooperation Projects of Guangdong Province</grant_agency>&nbsp;</td><td>
<div>2018A050506031&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Natural Science Foundation of Guangdong Province</grant_agency>&nbsp;</td><td>
<div>2020B1515120061&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Natural Science Foundation of China</grant_agency>&nbsp;</td><td>
<div>U1801265&nbsp;</div>
<div>U1908211&nbsp;</div>
<div>61976222&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Guang-dong Natural Science Funds for Distinguished Young Scholar</grant_agency>&nbsp;</td><td>
<div>2019B151502031&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This study was supported in part by the Key-Area Research and Development Program of Guangdong Province (2019B010110 0 01) , in part by the Key Program for International Cooperation Projects of Guangdong Province (2018A050506031) , in part by the Natural Science Foundation of Guangdong Province (2020B1515120061) , in part by the National Natural Science Foundation of China (U1801265 , U1908211 , 61976222) , and in part by the Guang-dong Natural Science Funds for Distinguished Young Scholar (2019B151502031) .</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 4 of 49</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Echo-SyncNet: Self-Supervised Cardiac View Synchronization in Echocardiography</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Dezaki, FT (Dezaki, Fatemeh Taheri); Luong, C (Luong, Christina); Ginsberg, T (Ginsberg, Tom); Rohling, R (Rohling, Robert); Gin, K (Gin, Ken); Abolmaesumi, P (Abolmaesumi, Purang); Tsang, T (Tsang, Teresa)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE TRANSACTIONS ON MEDICAL IMAGING</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>40</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>8</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>2092-2104</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/TMI.2021.3071951</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2021 AUG</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>13</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>13</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
0</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
16</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>24</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
In echocardiography (echo), an electrocardiogram (ECG) is conventionally used to temporally align different cardiac views for assessing critical measurements. However, in emergencies or point-of-care situations, acquiring an ECG is often not an option, hence motivating the need for alternative temporal synchronization methods. Here, we propose Echo-SyncNet, a self-supervised learning framework to synchronize various cross-sectional 2D echo series without any human supervision or external inputs. The proposed framework takes advantage of two types of supervisory signals derived from the input data: spatiotemporal patterns found between the frames of a single cine (intra-view self-supervision) and interdependencies between multiple cines (inter-view self-supervision). The combined supervisory signals are used to learn a feature-rich and low dimensional embedding space where multiple echo cines can be temporally synchronized. Two intra-view self-supervisions are used, the first is based on the information encoded by the temporal ordering of a cine (temporal intra-view) and the second on the spatial similarities between nearby frames (spatial intra-view). The inter-view self-supervision is used to promote the learning of similar embeddings for frames captured from the same cardiac phase in different echo views. We evaluate the framework with multiple experiments: 1) Using data from 998 patients, Echo-SyncNet shows promising results for synchronizing Apical 2 chamber and Apical 4 chamber cardiac views, which are acquired spatially perpendicular to each other; 2) Using data from 3070 patients, our experiments reveal that the learned representations of Echo-SyncNet outperform a supervised deep learning method that is optimized for automatic detection of fine-grained cardiac cycle phase; 3) We go one step further and show the usefulness of the learned representations in a one-shot learning scenario of cardiac key-frame detection. Without any fine-tuning, key frames in 1188 validation patient studies are identified by synchronizing them with only one labeled reference cine. We do not make any prior assumption about what specific cardiac views are used for training, and hence we show that Echo-SyncNet can accurately generalize to views not present in its training set. Project repository: github.com/fatemehtd/Echo-SyncNet.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000679532100013</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>33835916</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Synchronization; Training; Electrocardiography; Task analysis; Annotations; Echocardiography; Two dimensional displays; Echocardiography; fine-grained phase classification; self-supervised learning; synchronization</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Dezaki, Fatemeh Taheri; Abolmaesumi, Purang] Univ British Columbia, Dept Elect &amp; Comp Engn, Vancouver, BC V6T 1Z4, Canada. <br>
[Luong, Christina; Gin, Ken] Univ British Columbia, Vancouver Gen Hosp, Dept Med, Div Cardiol,Echocardiography Lab, Vancouver, BC V5Z 1M9, Canada. <br>
[Ginsberg, Tom] Univ British Columbia, Dept Engn Phys, Vancouver, BC V6T 1Z4, Canada. <br>
[Rohling, Robert] Univ British Columbia, Dept Elect &amp; Comp Engn, Vancouver, BC V6T 1Z4, Canada. <br>
[Rohling, Robert] Univ British Columbia, Dept Mech Engn, Vancouver, BC V6T 1Z4, Canada. <br>
[Tsang, Teresa] Vancouver Gen Hosp, Vancouver, BC V5Z 1M9, Canada. <br>
[Tsang, Teresa] Univ British Columbia, Dept Med, Div Cardiol, Echocardiog Lab, Vancouver, BC V5Z IM9, Canada. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Dezaki, FT (corresponding author), Univ British Columbia, Dept Elect &amp; Comp Engn, Vancouver, BC V6T 1Z4, Canada.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
fatemeht@ece.ubc.ca; christina.luong@ubc.ca; tom.ginsberg@alumni.ubc.ca; rohling@ece.ubc.ca; kenneth.gin@vch.ca; purang@ece.ubc.ca; t.tsang@ubc.ca</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
University of British Columbia; University of British Columbia; University of British Columbia; University of British Columbia; University of British Columbia; University of British Columbia; University of British Columbia</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Gin, Kenneth</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-6331-7003&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Tsang, Teresa S.M.</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-4865-7119&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Abolmaesumi, Purang</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-7259-8609&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Abolmaesumi, Purang</display_name>&nbsp;</td><td>AAE-6670-2019&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Ginsberg, Tom</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-3458-5188&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Luong, Christina</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-7514-6069&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Dezaki, Fatemeh</display_name>&nbsp;</td><td>AAQ-5296-2021&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Tsang, Teresa</display_name>&nbsp;</td><td>AAO-3518-2021&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Interdisciplinary Applications; Engineering, Biomedical; Engineering, Electrical &amp; Electronic; Imaging Science &amp; Photographic Technology; Radiology, Nuclear Medicine &amp; Medical Imaging</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Engineering; Imaging Science &amp; Photographic Technology; Radiology, Nuclear Medicine &amp; Medical Imaging</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>TS3CQ</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0278-0062</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1558-254X</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE T MED IMAGING</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE Trans. Med. Imaging</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>13</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Natural Sciences and Engineering Research Council of Canada (NSERC)</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Canadian Institutes of Health Research (CIHR)</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported in part by the Natural Sciences and Engineering Research Council of Canada (NSERC) and in part by the Canadian Institutes of Health Research (CIHR). (Fatemeh Taheri Dezaki and Christina Luong contributed equally to this work as first authors.) (Purang Abolmaesumi and Teresa Tsang con-tributed equally to this work as senior authors.)</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>Green Submitted, Bronze</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 5 of 49</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Few-shot transfer learning for personalized atrial fibrillation detection using patient-based siamese network with single-lead ECG records</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Ng, YW (Ng, Yiuwai); Liao, MT (Liao, Min-Tsun); Chen, TL (Chen, Ting-Li); Lee, CK (Lee, Chih-Kuo); Chou, CY (Chou, Cheng-Ying); Wang, WC (Wang, Weichung)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>ARTIFICIAL INTELLIGENCE IN MEDICINE</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>144</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>102644</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1016/j.artmed.2023.102644</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2023 OCT</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>16</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>17</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
7</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
47</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>55</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
The proliferation of wearable devices has allowed the collection of electrocardiogram (ECG) recordings daily to monitor heart rhythm and rate. For example, 24-hour Holter monitors, cardiac patches, and smartwatches are widely used for ECG gathering and application. An automatic atrial fibrillation (AF) detector is required for timely ECG interpretation. Deep learning models can accurately identify AFs if large amounts of annotated data are available for model training. However, it is impractical to request sufficient labels for ECG recordings for an individual patient to train a personalized model. We propose a Siamese-network-based approach for transfer learning to address this issue. A pre-trained Siamese convolutional neural network is created by comparing two labeled ECG segments from the same patient. We sampled 30-second ECG segments with a 50% overlapping window from the ECG recordings of patients in the MIT-BIH Atrial Fibrillation Database. Subsequently, we independently detected the occurrence of AF in each patient in the Long-Term AF Database. By fine-tuning the model with the 1, 3, 5, 7, 9, or 11 ECG segments ranging from 30 to 180 s, our method achieved macro-F1 scores of 96.84%, 96.91%, 96.97%, 97.02%, 97.05%, and 97.07%, respectively.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001123888800001</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>37783539</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Atrial fibrillation; Arrhythmia detection; Electrocardiogram; Deep learning</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
TIME-SERIES; HEART-RATE; PROJECTIONS</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Ng, Yiuwai; Wang, Weichung] Natl Taiwan Univ, Inst Appl Math Sci, Taipei, Taiwan. <br>
[Liao, Min-Tsun; Lee, Chih-Kuo] Natl Taiwan Univ, Dept Internal Med, Div Cardiol, Hosp Hsin Chu Branch, Hsinchu, Taiwan. <br>
[Liao, Min-Tsun] Natl Taiwan Univ, Coll Med, Dept Internal Med, Taipei, Taiwan. <br>
[Chen, Ting-Li] Acad Sinica, Inst Stat Sci, Taipei, Taiwan. <br>
[Chou, Cheng-Ying] Natl Taiwan Univ, Dept Biomechatron Engn, Taipei, Taiwan. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Wang, WC (corresponding author), Natl Taiwan Univ, Inst Appl Math Sci, Taipei, Taiwan.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
r09946020@ntu.edu.tw; liaomintsun@gmail.com; tlchen@stat.sinica.edu.tw; keitheva2009@gmail.com; chengying@ntu.edu.tw; wwang@ntu.edu.tw</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
National Taiwan University; National Taiwan University; National Taiwan University; Academia Sinica - Taiwan; National Taiwan University</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>CHOU, CHENG-YING</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-5737-6960&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Wang, Wen-Ching</display_name>&nbsp;</td><td>ABI-2253-2020&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Chen, Ting-Li</display_name>&nbsp;</td><td>AAF-5752-2020&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Chen, Ting-Li</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-1375-0164&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>LIAO, MIN-TSUN</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-6283-6554&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Wang, Weichung</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-6154-7750&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Liao, Min-Tsun</display_name>&nbsp;</td><td>GPG-6828-2022&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>ELSEVIER</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>RADARWEG 29, 1043 NX AMSTERDAM, NETHERLANDS</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Artificial Intelligence; Engineering, Biomedical; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Engineering; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>CF7A2</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0933-3657</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1873-2860</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>ARTIF INTELL MED</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Artif. Intell. Med.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>10</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Science and Technology Council, Taiwan</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Center for Theoretical Sciences Mathematics Division</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>All Vista Healthcare Sub-center</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This study was supported by the National Science and Technology Council, Taiwan, MOST All Vista Healthcare Sub-center, and National Center for Theoretical Sciences Mathematics Division. The funding source had no role in study design, data collection, analysis or in-terpretation, report writing, or the decision to submit this paper for publication.</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 6 of 49</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>ECG synthesis for cardiac arrhythmias: Integrating self-supervised learning and generative adversarial networks</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Simone, L (Simone, Lorenzo); Bacciu, D (Bacciu, Davide); Gervasi, V (Gervasi, Vincenzo)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>ARTIFICIAL INTELLIGENCE IN MEDICINE</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>167</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>103162</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1016/j.artmed.2025.103162</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2025 SEP</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>0</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>0</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
6</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
6</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>62</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Arrhythmia classifiers relying on supervised deep learning models usually require a substantial amount of labeled clinical data. The distribution of these labels is strictly related to the statistics of cardiovascular diseases among the population, which inherently narrows models' performance for classification tasks. Furthermore, during acquisition and data retrieval from electronic health records, concerns arise regarding patient anonymization due to stringent clinical policies. We introduce a conditional generative architecture for electrocardiography time series, which integrates self-supervision and generative adversarial principles. Empirical validation confirms the enhancement of morphological plausibility in synthetic data, showcasing its effectiveness in generating realistic signals. We propose a novel model (ECGAN), proving its capability of conditioning the probability distribution of ECG recordings. The proposed methodology is assessed upon various rhythm abnormalities including severe congestive heart failure, myocardial infarction, sinus rhythm, and premature ventricular contractions. Our proposed workflow for synthetic time series assessment demonstrates competitive performance compared to state-of-the-art models, achieving an average improvement of 2.4% in arrhythmia classification accuracy across MIT-BIH, BIDMC, and PTB datasets, while ensuring realistic synthetic data and improving training stability.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001504012300001</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>40460597</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Generative deep learning; Electrocardiography; Self-supervision; Generative-adversarial networks; Data augmentation</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
DYNAMICAL MODEL; COUPLED VAN; MECHANISMS; SIGNALS; IMPACT</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Simone, Lorenzo; Bacciu, Davide; Gervasi, Vincenzo] Univ Pisa, Dept Comp Sci, Largo Bruno Pontecorvo, I-56127 Pisa, Italy. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Simone, L (corresponding author), Univ Pisa, Dept Comp Sci, Largo Bruno Pontecorvo, I-56127 Pisa, Italy.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
lorenzo.simone@di.unipi.it</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
University of Pisa</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Bacciu, Davide</display_name>&nbsp;</td><td>A-3904-2013&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Simone, Lorenzo</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-5010-7733&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>ELSEVIER</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>RADARWEG 29, 1043 NX AMSTERDAM, NETHERLANDS</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Artificial Intelligence; Engineering, Biomedical; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Engineering; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>3MR9T</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0933-3657</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1873-2860</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>ARTIF INTELL MED</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Artif. Intell. Med.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>12</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 7 of 49</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Self-Supervised Feature Learning for Cardiac Cine MR Image Reconstruction</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Xu, SY (Xu, Siying); Fruh, M (Fruh, Marcel); Hammernik, K (Hammernik, Kerstin); Lingg, A (Lingg, Andreas); Kubler, J (Kubler, Jens); Krumm, P (Krumm, Patrick); Rueckert, D (Rueckert, Daniel); Gatidis, S (Gatidis, Sergios); K&uuml;stner, T (Kustner, Thomas)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE TRANSACTIONS ON MEDICAL IMAGING</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>44</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>9</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>3858-3869</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/TMI.2025.3570226</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2025 SEP</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>0</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>0</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
2</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
2</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>53</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
We propose a self-supervised feature learning assisted reconstruction (SSFL-Recon) framework for MRI reconstruction to address the limitation of existing supervised learning methods. Although recent deep learning-based methods have shown promising performance in MRI reconstruction, most require fully-sampled images for supervised learning, which is challenging in practice considering long acquisition times under respiratory or organ motion. Moreover, nearly all fully-sampled datasets are obtained from conventional reconstruction of mildly accelerated datasets, thus potentially biasing the achievable performance. The numerous undersampled datasets with different accelerations in clinical practice, hence, remain underutilized. To address these issues, we first train a self-supervised feature extractor on undersampled images to learn sampling-insensitive features. The pre-learned features are subsequently embedded in the self-supervised reconstruction network to assist in removing artifacts. Experiments were conducted retrospectively on an in-house 2D cardiac Cine dataset, including 91 cardiovascular patients and 38 healthy subjects. The results demonstrate that the proposed SSFL-Recon framework outperforms existing self-supervised MRI reconstruction methods and even exhibits comparable or better performance to supervised learning up to 16 x retrospective undersampling. The feature learning strategy can effectively extract global representations, which have proven beneficial in removing artifacts and increasing generalization ability during reconstruction.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001575893400005</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>40408221</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Image reconstruction; Feature extraction; Representation learning; Magnetic resonance imaging; Training; Imaging; Contrastive learning; Sensitivity; Reconstruction algorithms; Data mining; Self-supervised learning; feature learning; contrastive learning; cardiac Cine MRI; MRI reconstruction</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
PARALLEL MRI; SPARSITY; SENSE</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Xu, Siying; Fruh, Marcel; Lingg, Andreas; Kubler, Jens; Krumm, Patrick; Gatidis, Sergios; Kustner, Thomas] Univ Tubingen, Dept Diagnost &amp; Intervent Radiol, Med Image &amp; Data Anal MIDAS Lab, D-72074 Tubingen, Germany. <br>
[Rueckert, Daniel] Tech Univ Munich, Sch Computat Informat &amp; Technol, D-80333 Munich, Germany. <br>
[Rueckert, Daniel] Tech Univ Munich, Klinikum Rechts Isar, D-80333 Munich, Germany. <br>
[Rueckert, Daniel] Imperial Coll London, Dept Comp, London SW7 2AZ, England. <br>
[Gatidis, Sergios] Stanford Univ, Dept Radiol, Stanford, CA 94305 USA. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Xu, SY (corresponding author), Univ Tubingen, Dept Diagnost &amp; Intervent Radiol, Med Image &amp; Data Anal MIDAS Lab, D-72074 Tubingen, Germany.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
siying.xu@med.uni-tuebingen.de; marcel.frueh@med.uni-tuebingen.de; k.hammernik@tum.de; andreas.lingg@med.uni-tuebingen.de; jens.kuebler@med.uni-tuebingen.de; patrick.krumm@med.uni-tuebingen.de; daniel.rueckert@tum.de; sergios.gatidis@med.uni-tuebingen.de; thomas.kuestner@med.uni-tuebingen.de</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Eberhard Karls University of Tubingen; Eberhard Karls University Hospital; Technical University of Munich; Technical University of Munich; Imperial College London; Stanford University</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Krumm, Patrick</display_name>&nbsp;</td><td>AAQ-6293-2020&nbsp;</td><td>0000-0003-1705-8439&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Rueckert, Daniel</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-5683-5889&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Interdisciplinary Applications; Engineering, Biomedical; Engineering, Electrical &amp; Electronic; Imaging Science &amp; Photographic Technology; Radiology, Nuclear Medicine &amp; Medical Imaging</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Engineering; Imaging Science &amp; Photographic Technology; Radiology, Nuclear Medicine &amp; Medical Imaging</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>7OM6C</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0278-0062</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1558-254X</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE T MED IMAGING</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE Trans. Med. Imaging</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>12</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>DeutscheForschungsgemeinschaft (DFG, German Research Foundation)</grant_agency>&nbsp;</td><td>
<div>390727645&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported by the DeutscheForschungsgemeinschaft (DFG, German Research Foundation) through Germany's Excellence Strategy-Clusters of Excellence (EXC) 2064/1 under Project 390727645.</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>Green Submitted</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 8 of 49</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Few-Shot Learning by a Cascaded Framework With Shape-Constrained Pseudo Label Assessment for Whole Heart Segmentation</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Wang, WJ (Wang, Wenji); Xia, Q (Xia, Qing); Hu, ZQ (Hu, Zhiqiang); Yan, ZN (Yan, Zhennan); Li, ZW (Li, Zhuowei); Wu, Y (Wu, Yang); Huang, N (Huang, Ning); Gao, Y (Gao, Yue); Metaxas, D (Metaxas, Dimitris); Zhang, ST (Zhang, Shaoting)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE TRANSACTIONS ON MEDICAL IMAGING</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>40</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>10</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>2629-2641</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/TMI.2021.3053008</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2021 OCT</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>39</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>40</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
1</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
34</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>51</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Automatic and accurate 3D cardiac image segmentation plays a crucial role in cardiac disease diagnosis and treatment. Even though CNN based techniques have achieved great success in medical image segmentation, the expensive annotation, large memory consumption, and insufficient generalization ability still pose challenges to their application in clinical practice, especially in the case of 3D segmentation from high-resolution and large-dimension volumetric imaging. In this paper, we propose a few-shot learning framework by combining ideas of semi-supervised learning and self-training for whole heart segmentation and achieve promising accuracy with a Dice score of 0.890 and a Hausdorff distance of 18.539 mm with only four labeled data for training. When more labeled data provided, the model can generalize better across institutions. The key to success lies in the selection and evolution of high-quality pseudo labels in cascaded learning. A shape-constrained network is built to assess the quality of pseudo labels, and the self-training stages with alternative global-local perspectives are employed to improve the pseudo labels. We evaluate our method on the CTA dataset of the MM-WHS 2017 Challenge and a larger multi-center dataset. In the experiments, our method outperforms the state-of-the-art methods significantly and has great generalization ability on the unseen data. We also demonstrate, by a study of two 4D (3D+T) CTA data, the potential of our method to be applied in clinical practice.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000702638800009</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>33471751</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Whole heart segmentation; pseudo label; quality assessment; self-training; semi-supervised</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
SPARSE REPRESENTATION; MODELS</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Wang, Wenji; Xia, Qing; Hu, Zhiqiang; Li, Zhuowei; Huang, Ning; Zhang, Shaoting] SenseTime Res, Beijing 100080, Peoples R China. <br>
[Xia, Qing] Tsinghua Univ, Dept Software, Beijing 100084, Peoples R China. <br>
[Yan, Zhennan] SenseBrain Technol Ltd LLC, Princeton, NJ 08540 USA. <br>
[Wu, Yang] Chinese Peoples Liberat Army Gen Hosp, Beijing 100853, Peoples R China. <br>
[Gao, Yue] Tsinghua Univ, Dept Software, Beijing 100084, Peoples R China. <br>
[Metaxas, Dimitris] Rutgers State Univ, Dept Comp Sci, Piscataway, NJ 08854 USA. <br>
[Xia, Qing] Shanghai Jiao Tong Univ, Res Inst, Shanghai 200240, Peoples R China. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Xia, Q (corresponding author), SenseTime Res, Beijing 100080, Peoples R China.<br>Xia, Q (corresponding author), Tsinghua Univ, Dept Software, Beijing 100084, Peoples R China.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
wangwenji@sensetime.com; xiaqing@sensetime.com; huzhiqiang@sensetime.com; zhennanyan@sensebrain.site; lizhuowei@sensetime.com; 2225012218@qq.com; huangning@sensetime.com; gaoyue@tsinghua.edu.cn; dnm@cs.rutgers.edu; zhangshaoting@sensetime.com</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Tsinghua University; Chinese People's Liberation Army General Hospital; Tsinghua University; Rutgers University System; Rutgers University New Brunswick; Shanghai Jiao Tong University</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Gao, Yue</display_name>&nbsp;</td><td>B-3376-2012&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Xia, Qing</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-0328-7882&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>hu, zhiqiang</display_name>&nbsp;</td><td>HMD-5811-2023&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Zhang, Shaoting</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-8719-448X&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Wang, Wenji</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-5529-9052&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Interdisciplinary Applications; Engineering, Biomedical; Engineering, Electrical &amp; Electronic; Imaging Science &amp; Photographic Technology; Radiology, Nuclear Medicine &amp; Medical Imaging</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Engineering; Imaging Science &amp; Photographic Technology; Radiology, Nuclear Medicine &amp; Medical Imaging</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>WA1FC</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0278-0062</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1558-254X</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE T MED IMAGING</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE Trans. Med. Imaging</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>13</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Key Research and Development Project of China</grant_agency>&nbsp;</td><td>
<div>Z201100006820064&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>STCSM</grant_agency>&nbsp;</td><td>
<div>2020YFC2004800&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Beijing Postdoctoral Research Foundation</grant_agency>&nbsp;</td><td>
<div>19511121400&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported in part by the BeijingNova Program under Grant Z201100006820064, in part by the National Key Research and Development Project of China under Grant 2020YFC2004800, in part by the STCSM under Grant 19511121400, and in part by the Beijing Postdoctoral Research Foundation.</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 9 of 49</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Self-supervised representation learning from 12-lead ECG data</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Mehari, T (Mehari, Temesgen); Strodthoff, N (Strodthoff, Nils)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>COMPUTERS IN BIOLOGY AND MEDICINE</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>141</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>105114</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1016/j.compbiomed.2021.105114</value>&nbsp;&nbsp;<b>Early Access Date:</b> 
<value>DEC 2021</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2022 FEB</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>95</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>106</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
5</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
95</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>39</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Clinical 12-lead electrocardiography (ECG) is one of the most widely encountered kinds of biosignals. Despite the increased availability of public ECG datasets, label scarcity remains a central challenge in the field. Self-supervised learning represents a promising way to alleviate this issue. This would allow to train more powerful models given the same amount of labeled data and to incorporate or improve predictions about rare diseases, for which training datasets are inherently limited. In this work, we put forward the first comprehensive assessment of self-supervised representation learning from clinical 12-lead ECG data. To this end, we adapt state-of-the-art self-supervised methods based on instance discrimination and latent forecasting to the ECG domain. In a first step, we learn contrastive representations and evaluate their quality based on linear evaluation performance on a recently established, comprehensive, clinical ECG classification task. In a second step, we analyze the impact of self-supervised pretraining on finetuned ECG classifiers as compared to purely supervised performance. For the best-performing method, an adaptation of contrastive predictive coding, we find a linear evaluation performance only 0.5% below supervised performance. For the finetuned models, we find improvements in downstream performance of roughly 1% compared to supervised performance, label efficiency, as well as robustness against physiological noise. This work clearly establishes the feasibility of extracting discriminative representations from ECG data via self-supervised learning and the numerous advantages when finetuning such representations on downstream tasks as compared to purely supervised training. As first comprehensive assessment of its kind in the ECG domain carried out exclusively on publicly available datasets, we hope to establish a first step towards reproducible progress in the rapidly evolving field of representation learning for biosignals.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000747358300005</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>34973584</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Deep neural networks; Electrocardiography; Time series analysis; Unsupervised learning; Self-supervised learning</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
ELECTROCARDIOGRAMS</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Mehari, Temesgen] Phys Tech Bundesanstalt, Berlin, Germany. <br>
[Mehari, Temesgen; Strodthoff, Nils] Fraunhofer Heinrich Hertz Inst, Berlin, Germany. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Strodthoff, N (corresponding author), Carl von Ossietzky Univ Oldenburg, Oldenburg, Germany.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
temesgen.mehari@ptb.de; nils.strodthoff@uol.de</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Physikalisch-Technische Bundesanstalt (PTB); Fraunhofer Gesellschaft</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Mehari, Temesgen</display_name>&nbsp;</td><td>KCX-8728-2024&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Strodthoff, Nils</display_name>&nbsp;</td><td>AAK-8661-2021&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>PERGAMON-ELSEVIER SCIENCE LTD</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>THE BOULEVARD, LANGFORD LANE, KIDLINGTON, OXFORD OX5 1GB, ENGLAND</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Biology; Computer Science, Interdisciplinary Applications; Engineering, Biomedical; Mathematical &amp; Computational Biology</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Life Sciences &amp; Biomedicine - Other Topics; Computer Science; Engineering; Mathematical &amp; Computational Biology</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>YN6HS</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0010-4825</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1879-0534</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>COMPUT BIOL MED</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Comput. Biol. Med.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>8</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Bundesministerium fur Bildung und Forschung through the BIFOLD - Berlin Institute for the Foundations of Learning and Data</grant_agency>&nbsp;</td><td>
<div>01IS18025A&nbsp;</div>
<div>01IS18037A&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>MedalCare</grant_agency>&nbsp;</td><td>
<div>18HLT07&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>EMPIR programme</grant_agency>&nbsp;</td><td>
<div>18HLT07&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>European Union's Horizon 2020 research and innovation programme</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported by the Bundesministerium fur Bildung und Forschung through the BIFOLD -Berlin Institute for the Foundations of Learning and Data (ref. 01IS18025A and ref. 01IS18037A) and 18HLT07 MedalCare. The project 18HLT07 MedalCare has received funding from the EMPIR programme co-financed by the participating states and from the European Union's Horizon 2020 research and innovation programme.</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>hybrid, Green Submitted</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 10 of 49</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>AVP-AP: Self-Supervised Automatic View Positioning in 3D Cardiac CT via Atlas Prompting</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Fan, XL (Fan, Xiaolin); Wang, Y (Wang, Yan); Zhang, YY (Zhang, Yingying); Bao, MK (Bao, Mingkun); Jia, BS (Jia, Bosen); Lu, D (Lu, Dong); Gu, YF (Gu, Yifan); Cheng, J (Cheng, Jian); Zhu, HG (Zhu, Haogang)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE TRANSACTIONS ON MEDICAL IMAGING</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>44</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>7</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>2921-2932</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/TMI.2025.3554785</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2025 JUL</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>0</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>0</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
8</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
8</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>53</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Automatic view positioning is crucial for cardiac computed tomography (CT) examinations, including disease diagnosis and surgical planning. However, it is highly challenging due to individual variability and large 3D search space. Existing work needs labor-intensive and time-consuming manual annotations to train view-specific models, which are limited to predicting only a fixed set of planes. However, in real clinical scenarios, the challenge of positioning semantic 2D slices with any orientation into varying coordinate space in arbitrary 3D volume remains unsolved. We thus introduce a novel framework, AVP-AP, the first to use Atlas Prompting for self-supervised Automatic View Positioning in the 3D CT volume. Specifically, this paper first proposes an atlas prompting method, which generates a 3D canonical atlas and trains a network to map slices into their corresponding positions in the atlas space via a self-supervised manner. Then, guided by atlas prompts corresponding to the given query images in a reference CT, we identify the coarse positions of slices in the target CT volume using rigid transformation between the 3D atlas and target CT volume, effectively reducing the search space. Finally, we refine the coarse positions by maximizing the similarity between the predicted slices and the query images in the feature space of a given foundation model. Our framework is flexible and efficient compared to other methods, outperforming other methods by 19.8% average structural similarity (SSIM) in arbitrary view positioning and achieving 9% SSIM in two-chamber view compared to four radiologists. Meanwhile, experiments on a public dataset validate our framework's generalizability.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001523480800028</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>40138235</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Computed tomography; Three-dimensional displays; Planning; Standards; Solid modeling; Image segmentation; Anatomical structure; Predictive models; Foundation models; Computational modeling; Automatic view positioning; atlas prompting; cardiac computed tomography; foundation model; self-supervised learning</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
TO-VOLUME REGISTRATION; IMAGE REGISTRATION; FRAMEWORK</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Fan, Xiaolin; Wang, Yan; Gu, Yifan] Beihang Univ, Sch Instrumentat &amp; Optoelect Engn, Beijing 100191, Peoples R China. <br>
[Zhang, Yingying; Zhu, Haogang] Beihang Univ, Hangzhou Int Innovat Inst, Data Sci &amp; Intelligent Comp Lab, Hangzhou 311115, Peoples R China. <br>
[Bao, Mingkun; Lu, Dong; Cheng, Jian; Zhu, Haogang] Beihang Univ, Sch Comp Sci &amp; Engn, Beijing 100191, Peoples R China. <br>
[Jia, Bosen] Victoria Univ Wellington, Sch Biol Sci, Wellington 6012, New Zealand. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Zhang, YY; Zhu, HG (corresponding author), Beihang Univ, Hangzhou Int Innovat Inst, Data Sci &amp; Intelligent Comp Lab, Hangzhou 311115, Peoples R China.<br>Cheng, J; Zhu, HG (corresponding author), Beihang Univ, Sch Comp Sci &amp; Engn, Beijing 100191, Peoples R China.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
fanxiaolin.buaa@qq.com; wangyan9509@gmail.com; yingyingzhangbuaa@163.com; bravomikekilo@buaa.edu.cn; jiabose@myvuw.ac.nz; donglusx@gmail.com; guyifan@buaa.edu.cn; jian_cheng@buaa.edu.cn; haogangzhu@buaa.edu.cn</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Beihang University; Beihang University; Beihang University; Victoria University Wellington</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Jia, Bosen</display_name>&nbsp;</td><td>GSD-1813-2022&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>zhang, yingying</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-0964-1774&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Interdisciplinary Applications; Engineering, Biomedical; Engineering, Electrical &amp; Electronic; Imaging Science &amp; Photographic Technology; Radiology, Nuclear Medicine &amp; Medical Imaging</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Engineering; Imaging Science &amp; Photographic Technology; Radiology, Nuclear Medicine &amp; Medical Imaging</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>4PJ0F</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0278-0062</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1558-254X</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE T MED IMAGING</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE Trans. Med. Imaging</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>12</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Natural Science Foundation of China</grant_agency>&nbsp;</td><td>
<div>62406014&nbsp;</div>
<div>U21A20523&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Beijing Natural Science Foundation</grant_agency>&nbsp;</td><td>
<div>7244325&nbsp;</div>
<div>L222152&nbsp;</div>
<div>L242038&nbsp;</div>
<div>4252004&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Start-Up Funds of Hangzhou International Innovation Institute of Beihang University</grant_agency>&nbsp;</td><td>
<div>2024KQ045&nbsp;</div>
<div>2024KQ027&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Peking University Third Hospital Fund for Interdisciplinary Research</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported in part by the National Natural Science Foundation of China under 62406014 and Grant U21A20523; in part by Beijing Natural Science Foundation underGrant 7244325, Grant L222152, Grant L242038, and Grant 4252004; in part by the Start-Up Funds of Hangzhou International Innovation Institute of Beihang University under Grant 2024KQ045 and Grant 2024KQ027; and in part by Peking University Third Hospital Fund for Interdisciplinary Research.</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>Green Submitted</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 11 of 49</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>MBSS-T1: Model-based subject-specific self-supervised motion correction for robust cardiac T1 mapping</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Hanania, E (Hanania, Eyal); Zehavi-Lenz, A (Zehavi-Lenz, Adi); Volovik, I (Volovik, Ilya); Link-Sourani, D (Link-Sourani, Daphna); Cohen, I (Cohen, Israel); Freiman, M (Freiman, Moti)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>MEDICAL IMAGE ANALYSIS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>102</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>103495</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1016/j.media.2025.103495</value>&nbsp;&nbsp;<b>Early Access Date:</b> 
<value>FEB 2025</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2025 MAY</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>0</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>1</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
0</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
3</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>41</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Cardiac T1 mapping is a valuable quantitative MRI technique for diagnosing diffuse myocardial diseases. Traditional methods, relying on breath-hold sequences and cardiac triggering based on an ECG signal, face challenges with patient compliance, limiting their effectiveness. Image registration can enable motion-robust cardiac T1 mapping, but inherent intensity differences between time points pose a challenge. We present MBSST1, a subject-specific self-supervised model for motion correction in cardiac T1 mapping. Physical constraints, implemented through a loss function comparing synthesized and motion-corrected images, enforce signal decay behavior, while anatomical constraints, applied via a Dice loss, ensure realistic deformations. The unique combination of these constraints results in motion-robust cardiac T1 mapping along the longitudinal relaxation axis. Ina 5-fold experiment on a public dataset of 210 patients (STONE sequence) and an internal dataset of 19 patients (MOLLI sequence), MBSS-T1 outperformed baseline deep-learning registration methods. It achieved superior model fitting quality (R2: 0.975 vs. 0.941, 0.946 for STONE; 0.987 vs. 0.982, 0.965 for MOLLI free- breathing; 0.994 vs. 0.993, 0.991 for MOLLI breath-hold), anatomical alignment (Dice: 0.89 vs. 0.84, 0.88 for STONE; 0.963 vs. 0.919, 0.851 for MOLLI free-breathing; 0.954 vs. 0.924, 0.871 for MOLLI breath-hold), and visual quality (4.33 vs. 3.38, 3.66 for STONE; 4.1 vs. 3.5, 3.28 for MOLLI free-breathing; 3.79 vs. 3.15, 2.84 for MOLLI breath-hold). MBSS-T1 enables motion-robust T1 mapping for broader patient populations, overcoming challenges such as suboptimal compliance, and facilitates free-breathing cardiac T1 mapping without requiring large annotated datasets. Our code is available at https://github.com/TechnionComputationalMRILab/MBSST1.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001432441300001</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>39987819</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
T1 mapping; Motion correction; Myocardial imaging; Deep learning; Model-based deep learning; Free-breathing MRI</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
IMAGE REGISTRATION; MYOCARDIAL T-1; VOLUME; FRAMEWORK</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Hanania, Eyal; Cohen, Israel] Technion IIT, Fac Elect &amp; Comp Engn, Haifa, Israel. <br>
[Zehavi-Lenz, Adi; Link-Sourani, Daphna; Freiman, Moti] Technion IIT, Fac Biomed Engn, Haifa, Israel. <br>
[Zehavi-Lenz, Adi; Link-Sourani, Daphna; Freiman, Moti] Technion IIT, Fac Biomed Engn, May Blum Dahl MRI Res Ctr, Haifa, Israel. <br>
[Volovik, Ilya] Bnai Zion Med Ctr, Haifa, Israel. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Hanania, E (corresponding author), Technion IIT, Fac Elect &amp; Comp Engn, Haifa, Israel.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
EyalHan@campus.technion.ac.il</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Technion Israel Institute of Technology; Technion Israel Institute of Technology; Technion Israel Institute of Technology; Bnai Zion Medical Center</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Cohen, Israel</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-2556-3972&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Freiman, Moti</display_name>&nbsp;</td><td>B-2016-2013&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Hanania, Eyal</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-5268-8674&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>ELSEVIER</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>RADARWEG 29, 1043 NX AMSTERDAM, NETHERLANDS</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Artificial Intelligence; Computer Science, Interdisciplinary Applications; Engineering, Biomedical; Radiology, Nuclear Medicine &amp; Medical Imaging</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Engineering; Radiology, Nuclear Medicine &amp; Medical Imaging</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>Y5H8V</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>1361-8415</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1361-8423</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>MED IMAGE ANAL</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Med. Image Anal.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>13</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Israel-US Binational Science Foundation</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Israeli Ministry of Science and Technology</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Israel Innovation Authority</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Joint Microsoft Education</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Israel Inter-university Computation Center (IUCC) program</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported in part by research grants from the Israel-US Binational Science Foundation, the Israeli Ministry of Science and Technology, the Israel Innovation Authority, and the joint Microsoft Education and the Israel Inter-university Computation Center (IUCC) program.</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 12 of 49</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Explainable cardiac pathology classification on cine MRI with motion characterization by semi-supervised learning of apparent flow</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Zheng, Q (Zheng, Qiao); Delingette, H (Delingette, Herve); Ayache, N (Ayache, Nicholas)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>MEDICAL IMAGE ANALYSIS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>56</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>80-95</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1016/j.media.2019.06.001</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2019 AUG</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>67</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>77</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
0</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
24</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>45</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
We propose a method to classify cardiac pathology based on a novel approach to extract image derived features to characterize the shape and motion of the heart. An original semi-supervised learning procedure, which makes efficient use of a large amount of non-segmented images and a small amount of images segmented manually by experts, is developed to generate pixel-wise apparent flow between two time points of a 2D+t cine MRI image sequence. Combining the apparent flow maps and cardiac segmentation masks, we obtain a local apparent flow corresponding to the 2D motion of myocardium and ventricular cavities. This leads to the generation of time series of the radius and thickness of myocardial segments to represent cardiac motion. These time series of motion features are reliable and explainable characteristics of pathological cardiac motion. Furthermore, they are combined with shape-related features to classify cardiac pathologies. Using only nine feature values as input, we propose an explainable, simple and flexible model for pathology classification. On ACDC training set and testing set, the model achieves 95% and 94% respectively as classification accuracy. Its performance is hence comparable to that of the state-of-the-art. Comparison with various other models is performed to outline some advantages of our model. (C) 2019 Elsevier B.V. All rights reserved.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000480375900007</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>31200290</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Cardiac pathology; Classification; Cine MRI; Motion; Deep learning; Semi-supervised learning; Neural network; Apparent flow</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
REGISTRATION; DEMONS</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Zheng, Qiao; Delingette, Herve; Ayache, Nicholas] Univ Cote dAzur, INRIA, 2004 Route Lucioles, F-06902 Sophia Antipolis, France. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Zheng, Q (corresponding author), Univ Cote dAzur, INRIA, 2004 Route Lucioles, F-06902 Sophia Antipolis, France.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
qiao.zheng@inria.fr</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Inria; Universite Cote d'Azur</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Delingette, Herve</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-6050-5949&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Delingette, Herve</display_name>&nbsp;</td><td>NOF-5139-2025&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>ELSEVIER</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>RADARWEG 29, 1043 NX AMSTERDAM, NETHERLANDS</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Artificial Intelligence; Computer Science, Interdisciplinary Applications; Engineering, Biomedical; Radiology, Nuclear Medicine &amp; Medical Imaging</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Engineering; Radiology, Nuclear Medicine &amp; Medical Imaging</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>IP9NC</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>1361-8415</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1361-8423</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>MED IMAGE ANAL</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Med. Image Anal.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>16</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>European Research Council (MedYMA)</grant_agency>&nbsp;</td><td>
<div>ERC-AdG-2011-291080&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>The authors acknowledge the partial support from the European Research Council (MedYMA ERC-AdG-2011-291080).</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>Green Submitted</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 13 of 49</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Score-Based Diffusion Models With Self-Supervised Learning for Accelerated 3D Multi-Contrast Cardiac MR Imaging</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Liu, YY (Liu, Yuanyuan); Cui, ZX (Cui, Zhuo-Xu); Qin, SC (Qin, Shucong); Liu, CC (Liu, Congcong); Zheng, HR (Zheng, Hairong); Wang, HF (Wang, Haifeng); Zhou, YH (Zhou, Yihang); Liang, D (Liang, Dong); Zhu, YJ (Zhu, Yanjie)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE TRANSACTIONS ON MEDICAL IMAGING</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>44</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>6</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>2436-2448</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/TMI.2025.3534206</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2025 JUN</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>0</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>0</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
11</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
12</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>78</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
scan time significantly hinders the widespread applications of three-dimensional multi-contrast cardiac magnetic resonance (3D-MC-CMR) imaging. This study aims to accelerate 3D-MC-CMR acquisition by a novel method based on score-based diffusion models with self-supervised learning. Specifically, we first establish a mapping between the undersampled k-space measurements and the MR images, utilizing a self-supervised Bayesian reconstruction network. Secondly, we develop a joint score-based diffusion model on 3D-MC-CMR images to capture their inherent distribution. Senior Member, The 3D-MC-CMR images are finally reconstructed using the conditioned Langenvin Markov chain Monte Carlo sampling. This approach enables accurate reconstruction without fully sampled training data. Its performance was tested on the dataset acquired by a 3D joint myocardial T1 and T1 rho mapping sequence. The T1 and T1 rho maps were estimated via a dictionary matching method from the reconstructed images. Experimental results show that the proposed method outperforms traditional compressed sensing and existing self-supervised deep learning MRI reconstruction methods. It also achieves high quality T1 and T1 rho parametric maps close to the reference maps, even at a high acceleration rate of 14.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001502493100015</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>40031249</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Image reconstruction; Diffusion models; Magnetic resonance imaging; Training; Self-supervised learning; Three-dimensional displays; Myocardium; Data models; Neural networks; Interpolation; 3D cardiac magnetic resonance imaging; self-supervised; diffusion models; multi-contrast</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
PARALLEL MRI; RESONANCE; COMBINATION</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Liu, Yuanyuan; Qin, Shucong; Zheng, Hairong; Wang, Haifeng; Zhu, Yanjie] Chinese Acad Sci, Shenzhen Inst Adv Technol, Paul C Lauterbur Res Ctr Biomed Imaging, Shenzhen 518055, Guangdong, Peoples R China. <br>
[Liu, Yuanyuan] Natl Innovat Ctr Adv Med Devices, Shenzhen 518045, Guangdong, Peoples R China. <br>
[Cui, Zhuo-Xu; Zhou, Yihang; Liang, Dong] Chinese Acad Sci, Shenzhen Inst Adv Technol, Res Ctr Med AI, Shenzhen 518055, Guangdong, Peoples R China. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Zhu, YJ (corresponding author), Chinese Acad Sci, Shenzhen Inst Adv Technol, Paul C Lauterbur Res Ctr Biomed Imaging, Shenzhen 518055, Guangdong, Peoples R China.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
liuyy@siat.ac.cn; zx.cui@siat.ac.cn; idginshucong@163.com; cc.liu@siat.ac.cn; hr.zheng@siat.ac.cn; hf.wang1@siat.ac.cn; yh.zhou2@siat.ac.cn; dong.liang@siat.ac.cn; yj.zhu@siat.ac.cn</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Chinese Academy of Sciences; Shenzhen Institute of Advanced Technology, CAS; Chinese Academy of Sciences; Shenzhen Institute of Advanced Technology, CAS</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Liu, Yuanyuan</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-0131-2519&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Haifeng, Wang</display_name>&nbsp;</td><td>HQY-7721-2023&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Liang, Dong</display_name>&nbsp;</td><td>A-3335-2011&nbsp;</td><td>0000-0001-6257-0875&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>liu, xiao</display_name>&nbsp;</td><td>HMD-7454-2023&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Liu, Congcong</display_name>&nbsp;</td><td>HLG-9805-2023&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Interdisciplinary Applications; Engineering, Biomedical; Engineering, Electrical &amp; Electronic; Imaging Science &amp; Photographic Technology; Radiology, Nuclear Medicine &amp; Medical Imaging</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Engineering; Imaging Science &amp; Photographic Technology; Radiology, Nuclear Medicine &amp; Medical Imaging</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>3KL6Y</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0278-0062</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1558-254X</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE T MED IMAGING</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE Trans. Med. Imaging</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>13</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Key Research and Development Program of China</grant_agency>&nbsp;</td><td>
<div>2021YFF0501402&nbsp;</div>
<div>2023YFA1011403&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Natural Science Foundation of China</grant_agency>&nbsp;</td><td>
<div>62322119&nbsp;</div>
<div>62201561&nbsp;</div>
<div>62206273&nbsp;</div>
<div>62476268&nbsp;</div>
<div>12226008&nbsp;</div>
<div>62125111&nbsp;</div>
<div>62106252&nbsp;</div>
<div>U22A20344&nbsp;</div>
<div>52293425&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Guangdong Basic and Applied Basic Research Foundation</grant_agency>&nbsp;</td><td>
<div>2021A1515110540&nbsp;</div>
<div>2023A1515110476&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Key Laboratory for Magnetic Resonance and Multimodality Imaging of Guangdong Province</grant_agency>&nbsp;</td><td>
<div>2023B1212060052&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Shenzhen Science and Technology Program</grant_agency>&nbsp;</td><td>
<div>RCYX20210609104444089&nbsp;</div>
<div>JCYJ20240813155840052&nbsp;</div>
<div>JCYJ20220818101205012&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported in part by the National Key Research and Development Program of China under Grant 2021YFF0501402 and Grant 2023YFA1011403; in part by the National Natural Science Foundation of China under Grant 62322119, Grant 62201561, Grant 62206273, Grant 62476268, Grant 12226008, Grant 62125111, Grant 62106252, Grant U22A20344, and Grant 52293425; in part by Guangdong Basic and Applied Basic Research Foundation under Grant 2021A1515110540 and Grant 2023A1515110476; in part by Key Laboratory for Magnetic Resonance and Multimodality Imaging of Guangdong Province under Grant 2023B1212060052; and in part by Shenzhen Science and Technology Program under Grant RCYX20210609104444089, Grant JCYJ20240813155840052, and Grant JCYJ20220818101205012.</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 14 of 49</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Unsupervised Domain Adaptation With Variational Approximation for Cardiac Segmentation</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Wu, FP (Wu, Fuping); Zhuang, XH (Zhuang, Xiahai)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE TRANSACTIONS ON MEDICAL IMAGING</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>40</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>12</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>3555-3567</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/TMI.2021.3090412</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2021 DEC</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>60</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>65</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
2</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
57</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>50</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Unsupervised domain adaptation is useful in medical image segmentation. Particularly, when ground truths of the target images are not available, domain adaptation can train a target-specific model by utilizing the existing labeled images from other modalities. Most of the reported works mapped images of both the source and target domains into a common latent feature space, and then reduced their discrepancy either implicitly with adversarial training or explicitly by directly minimizing a discrepancy metric. In this work, we propose a new framework, where the latent features of both domains are driven towards a common and parameterized variational form, whose conditional distribution given the image is Gaussian. This is achieved by two networks based on variational auto-encoders (VAEs) and a regularization for this variational approximation. Both of the VAEs, each for one domain, contain a segmentation module, where the source segmentation is trained in a supervised manner, while the target one is trained unsupervisedly. We validated the proposed domain adaptation method using two cardiac segmentation tasks, i.e., the cross-modality (CT and MR) whole heart segmentation and the cross-sequence cardiac MR segmentation. Results show that the proposed method achieved better accuracies compared to two state-of-the-art approaches and demonstrated good potential for cardiac segmentation. Furthermore, the proposed explicit regularization was shown to be effective and efficient in narrowing down the distribution gap between domains, which is useful for unsupervised domain adaptation. The code and data have been released via https://zmiclab.github.io/projects.html.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000724511900027</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>34143733</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Image segmentation; Training; Task analysis; Data models; Adaptation models; Measurement; Feature extraction; Domain adaptation; variational approximation; explicit domain discrepancy; cardiac segmentation</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
IMAGE; ALIGNMENT; NETWORK</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Wu, Fuping] Fudan Univ, Dept Stat Sch Management, Shanghai 200433, Peoples R China. <br>
[Wu, Fuping; Zhuang, Xiahai] Fudan Univ, Sch Data Sci, Shanghai 200433, Peoples R China. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Zhuang, XH (corresponding author), Fudan Univ, Sch Data Sci, Shanghai 200433, Peoples R China.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
17110690006@fudan.edu.cn; zxh@fudan.edu.cn</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Fudan University; Fudan University</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Zhuang, Xiahai</display_name>&nbsp;</td><td>AAH-6334-2019&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Wu, Fuping</display_name>&nbsp;</td><td>ABG-4545-2021&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Wu, Fuping</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-7179-4766&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Interdisciplinary Applications; Engineering, Biomedical; Engineering, Electrical &amp; Electronic; Imaging Science &amp; Photographic Technology; Radiology, Nuclear Medicine &amp; Medical Imaging</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Engineering; Imaging Science &amp; Photographic Technology; Radiology, Nuclear Medicine &amp; Medical Imaging</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>XG1HV</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0278-0062</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1558-254X</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE T MED IMAGING</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE Trans. Med. Imaging</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>13</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Natural Science Foundation of China</grant_agency>&nbsp;</td><td>
<div>61971142&nbsp;</div>
<div>11871165&nbsp;</div>
<div>62111530195&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported by the National Natural Science Foundation of China under Grant 61971142, Grant 11871165, and Grant 62111530195.</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 15 of 49</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Structure-Driven Unsupervised Domain Adaptation for Cross-Modality Cardiac Segmentation</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Cui, ZM (Cui, Zhiming); Li, CJ (Li, Changjian); Du, ZX (Du, Zhixu); Chen, NL (Chen, Nenglun); Wei, GD (Wei, Guodong); Chen, RN (Chen, Runnan); Yang, L (Yang, Lei); Shen, DG (Shen, Dinggang); Wang, WP (Wang, Wenping)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE TRANSACTIONS ON MEDICAL IMAGING</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>40</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>12</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>3604-3616</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/TMI.2021.3090432</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2021 DEC</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>29</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>33</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
2</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
45</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>52</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Performance degradation due to domain shift remains a major challenge in medical image analysis. Unsupervised domain adaptation that transfers knowledge learned from the source domain with ground truth labels to the target domain without any annotation is the mainstream solution to resolve this issue. In this paper, we present a novel unsupervised domain adaptation framework for cross-modality cardiac segmentation, by explicitly capturing a common cardiac structure embedded across different modalities to guide cardiac segmentation. In particular, we first extract a set of 3D landmarks, in a self-supervised manner, to represent the cardiac structure of different modalities. The high-level structure information is then combined with another complementary feature, the Canny edges, to produce accurate cardiac segmentation results both in the source and target domains. We extensively evaluate our method on the MICCAI 2017 MM-WHS dataset for cardiac segmentation. The evaluation, comparison and comprehensive ablation studies demonstrate that our approach achieves satisfactory segmentation results and outperforms state-of-the-art unsupervised domain adaptation methods by a significant margin.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000724511900031</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>34161240</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Image segmentation; Feature extraction; Computed tomography; Magnetic resonance imaging; Three-dimensional displays; Task analysis; Image edge detection; Cross-modality learning; unsupervised domain adaptation; structure distillation; cardiac segmentation</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
SYNERGISTIC IMAGE; NETWORK; MRI</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Cui, Zhiming; Shen, Dinggang] ShanghaiTech Univ, Sch Biomed Engn, Shanghai 201210, Peoples R China. <br>
[Cui, Zhiming; Du, Zhixu; Chen, Nenglun; Wei, Guodong; Chen, Runnan; Yang, Lei; Wang, Wenping] Univ Hong Kong, Dept Comp Sci, Hong Kong, Peoples R China. <br>
[Li, Changjian] UCL, Dept Comp Sci, London WC1E 6EA, England. <br>
[Shen, Dinggang] Shanghai United Imaging Intelligence Co Ltd, Shanghai 200030, Peoples R China. <br>
[Shen, Dinggang] Korea Univ, Dept Artificial Intelligence, Seoul 02841, South Korea. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Shen, DG (corresponding author), ShanghaiTech Univ, Sch Biomed Engn, Shanghai 201210, Peoples R China.<br>Wang, WP (corresponding author), Univ Hong Kong, Dept Comp Sci, Hong Kong, Peoples R China.<br>Li, CJ (corresponding author), UCL, Dept Comp Sci, London WC1E 6EA, England.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
zmcui@cs.hku.hk; chjili2011@gmail.com; dzx3501@connect.hku.hk; chennenglun@gmail.com; g.d.wei.china@gmail.com; runnanchen@modontics.com; yanglei.dalian@gmail.com; dinggang.shen@gmail.com; wenping@cs.hku.hk</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
ShanghaiTech University; University of Hong Kong; University of London; University College London; Korea University</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Shen, Dinggang</display_name>&nbsp;</td><td>ABF-6812-2020&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Du, Zhixu</display_name>&nbsp;</td><td>LQK-5797-2024&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Wei, Guodong</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-6975-9865&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Li, Changjian</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-0448-4957&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>chen, runnan</display_name>&nbsp;</td><td>KFA-0707-2024&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Interdisciplinary Applications; Engineering, Biomedical; Engineering, Electrical &amp; Electronic; Imaging Science &amp; Photographic Technology; Radiology, Nuclear Medicine &amp; Medical Imaging</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Engineering; Imaging Science &amp; Photographic Technology; Radiology, Nuclear Medicine &amp; Medical Imaging</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>XG1HV</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0278-0062</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1558-254X</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE T MED IMAGING</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE Trans. Med. Imaging</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>13</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 16 of 49</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>A multi-module algorithm for heartbeat classification based on unsupervised learning and adaptive feature transfer</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Wang, YN (Wang, Yanan); Hu, SC (Hu, Shuaicong); Liu, J (Liu, Jian); Zhong, GY (Zhong, Gaoyan); Yang, CW (Yang, Cuiwei)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>COMPUTERS IN BIOLOGY AND MEDICINE</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>170</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>108072</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1016/j.compbiomed.2024.108072</value>&nbsp;&nbsp;<b>Early Access Date:</b> 
<value>JAN 2024</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2024 MAR</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>3</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>3</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
3</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
26</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>54</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
The scarcity of annotated data is a common issue in the realm of heartbeat classification based on deep learning. Transfer learning (TL) has emerged as an effective strategy for addressing this issue. However, current TL techniques in this realm overlook the probability distribution differences between the source domain (SD) and target domain (TD) databases. The motivation of this paper is to address the challenge of labeled data scarcity at the model level while exploring an effective method to eliminate domain discrepancy between SD and TD databases, especially when SD and TD are derived from inconsistent tasks. This study proposes a multi-module heartbeat classification algorithm. Initially, unsupervised feature extractors are designed to extract rich features from unlabeled SD and TD data. Subsequently, a novel adaptive transfer method is proposed to effectively eliminate domain discrepancy between features of SD for pre-training (PTF-SD) and features of TD for fine-tuning (FTF-TD). Finally, the adapted PTF-SD is employed to pre-train a designed classifier, and FTF-TD is used for classifier fine-tuning, with the objective of evaluating the algorithm's performance on the TD task. In our experiments, MNIST-DB serves as the SD database for handwritten digit image classification task, MIT-DB as the TD database for heartbeat classification task. The overall accuracy of classifying heartbeats into normal heartbeats, supraventricular ectopic beats (SVEBs), and ventricular ectopic beats (VEBs) reaches 96.7 %. Specifically, the sensitivity (Sen), positive predictive value (PPV), and F1 score for SVEBs are 0.802, 0.701, and 0.748, respectively. For VEBs, Sen, PPV, and F1 score are 0.976, 0.840, and 0.903, respectively. The results indicate that the proposed multi-module algorithm effectively addresses the challenge labeled data scarcity in heartbeat classification through unsupervised learning and adaptive feature transfer methods.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001171387100001</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>38301518</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Annotated data scarcity; Heartbeat classification; Adaptive feature transfer; Unsupervised learning; Domain discrepancy</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
CONVOLUTIONAL NEURAL-NETWORK</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Wang, Yanan; Hu, Shuaicong; Liu, Jian; Zhong, Gaoyan; Yang, Cuiwei] Fudan Univ, Ctr Biomed Engn, Sch Informat Sci &amp; Technol, Shanghai 200433, Peoples R China. <br>
[Yang, Cuiwei] Key Lab Med Imaging Comp &amp; Comp Assisted Intervent, Shanghai 200093, Peoples R China. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Yang, CW (corresponding author), Fudan Univ, Ctr Biomed Engn, Sch Informat Sci &amp; Technol, Shanghai 200433, Peoples R China.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
wangyn20@fudan.edu.cn; schu22@m.fudan.edu.cn; liuj22@m.fudan.edu.cn; zhonggaoyan@126.com; yangcw@fudan.edu.cn</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Fudan University</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Hu, Shuaicong</display_name>&nbsp;</td><td>NIU-9967-2025&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Yang, Cuiwei</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-3338-5835&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>PERGAMON-ELSEVIER SCIENCE LTD</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>THE BOULEVARD, LANGFORD LANE, KIDLINGTON, OXFORD OX5 1GB, ENGLAND</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Biology; Computer Science, Interdisciplinary Applications; Engineering, Biomedical; Mathematical &amp; Computational Biology</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Life Sciences &amp; Biomedicine - Other Topics; Computer Science; Engineering; Mathematical &amp; Computational Biology</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>JE1D8</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0010-4825</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1879-0534</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>COMPUT BIOL MED</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Comput. Biol. Med.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>15</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Shanghai Science and Technology Support Project</grant_agency>&nbsp;</td><td>
<div>18441900900&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported in part by Shanghai Science and Technology Support Project under Grant 18441900900.</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 17 of 49</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Unsupervised ECG Analysis: A Review</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Nezamabadi, K (Nezamabadi, Kasra); Sardaripour, N (Sardaripour, Neda); Haghi, B (Haghi, Benyamin); Forouzanfar, M (Forouzanfar, Mohamad)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE REVIEWS IN BIOMEDICAL ENGINEERING</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>16</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>208-224</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/RBME.2022.3154893</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2023 </value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>23</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>28</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
16</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
124</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>157</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Electrocardiography is the gold standard technique for detecting abnormal heart conditions. Automatic detection of electrocardiogram (ECG) abnormalities helps clinicians analyze the large amount of data produced daily by cardiac monitors. As thenumber of abnormal ECG samples with cardiologist-supplied labels required to train supervised machine learning models is limited, there is a growing need for unsupervised learning methods for ECG analysis. Unsupervised learning aims to partition ECG samples into distinct abnormality classes without cardiologist-supplied labels-a process referred to as ECG clustering. In addition to abnormality detection, ECG clustering has recently discovered inter and intra-individual patterns that reveal valuable information about the whole body and mind, such as emotions, mental disorders, and metabolic levels. ECG clustering can also resolve specific challenges facing supervised learning systems, such as the imbalanced data problem, and can enhance biometric systems. While several reviews exist on supervised ECG systems, a comprehensive review of unsupervised ECG analysis techniques is still lacking. This study reviews ECG clustering techniques developed mainly in the last decade. The focus will be on recent machine learning and deep learning algorithms and their practical applications. We critically review and compare these techniques, discuss their applications and limitations, and provide future research directions. This review provides further insights into ECG clustering and presents the necessary information required to adopt the appropriate algorithm for a specific application.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000967021400001</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>35226604</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Review</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Electrocardiography; Clustering algorithms; Unsupervised learning; Particle swarm optimization; Heart; Deep learning; Computational efficiency; Electrocardiogram (ECG); machine learning; unsupervised learning; clustering; deep learning</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
CLUSTERING NEURAL-NETWORK; ATRIAL-FIBRILLATION; WAVELET TRANSFORM; CLASSIFICATION; SYSTEM; IDENTIFICATION; OPTIMIZATION; ARCHITECTURE; ARRHYTHMIAS; COMPONENTS</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Nezamabadi, Kasra] Univ Delaware, Dept Comp &amp; Informat Sci, Newark, DE 19716 USA. <br>
[Sardaripour, Neda] Vanderbilt Univ, Dept Biomed Engn, Nashville, TN 37235 USA. <br>
[Haghi, Benyamin] CALTECH, Dept Elect Engn, Pasadena, CA 91125 USA. <br>
[Forouzanfar, Mohamad] Univ Quebec, Ecole Technol Super ETS, Montreal, PQ H3C 1K3, Canada. <br>
[Forouzanfar, Mohamad] Ctr Rech Inst Univ Geriatrie Montreal CRIUGM, Montreal, PQ H3W 1W5, Canada. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Forouzanfar, M (corresponding author), Univ Quebec, Ecole Technol Super ETS, Montreal, PQ H3C 1K3, Canada.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
kasra@udel.edu; neda.sardaripour@vanderbilt.edu; benyamin.a.haghi@caltech.edu; mohamad.forouzanfar@etsmtl.ca</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
University of Delaware; Vanderbilt University; California Institute of Technology; University of Quebec; Ecole de Technologie Superieure - Canada; University of Quebec Montreal; Universite de Montreal</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Nezemabadi, Kasra</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-9844-7729&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Forouzanfar, Mohamad</display_name>&nbsp;</td><td>L-3892-2014&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Sardaripour, Neda</display_name>&nbsp;</td><td>ABB-8392-2022&nbsp;</td><td>0000-0001-9136-5889&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Allahgholizadeh Haghi, Benyamin</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-4839-7647&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Forouzanfar, Mohamad</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-8849-0144&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Nezamabadi, Kasra</display_name>&nbsp;</td><td>HSH-5349-2023&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Engineering, Biomedical</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Engineering</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>D2IX4</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>1937-3333</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1941-1189</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE REV BIOMED ENG</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE Rev. Biomed. Eng.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>17</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Natural Sciences and Engineering Research Council of Canada (NSERC)</grant_agency>&nbsp;</td><td>
<div>RGPIN-2021-03924&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>The work of Mohamad Forouzanfarwas supported in part by the Natural Sciences and Engineering Research Council of Canada (NSERC) under Grant RGPIN-2021-03924</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 18 of 49</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Toward Accurate Cardiac MRI Segmentation With Variational Autoencoder-Based Unsupervised Domain Adaptation</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Cui, HF (Cui, Hengfei); Li, Y (Li, Yan); Wang, YF (Wang, Yifan); Xu, D (Xu, Di); Wu, LM (Wu, Lian-Ming); Xia, Y (Xia, Yong)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE TRANSACTIONS ON MEDICAL IMAGING</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>43</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>8</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>2924-2936</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/TMI.2024.3382624</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2024 AUG</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>5</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>7</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
11</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
32</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>36</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Accurate myocardial segmentation is crucial in the diagnosis and treatment of myocardial infarction (MI), especially in Late Gadolinium Enhancement (LGE) cardiac magnetic resonance (CMR) images, where the infarcted myocardium exhibits a greater brightness. However, segmentation annotations for LGE images are usually not available. Although knowledge gained from CMR images of other modalities with ample annotations, such as balanced-Steady State Free Precession (bSSFP), can be transferred to the LGE images, the difference in image distribution between the two modalities (i.e., domain shift) usually results in a significant degradation in model performance. To alleviate this, an end-to-end Variational autoencoder based feature Alignment Module Combining Explicit and Implicit features (VAMCEI) is proposed. We first re-derive the Kullback-Leibler (KL) divergence between the posterior distributions of the two domains as a measure of the global distribution distance. Second, we calculate the prototype contrastive loss between the two domains, bringing closer the prototypes of the same category across domains and pushing away the prototypes of different categories within or across domains. Finally, a domain discriminator is added to the output space, which indirectly aligns the feature distribution and forces the extracted features to be more favorable for segmentation. In addition, by combining CycleGAN and VAMCEI, we propose a more refined multi-stage unsupervised domain adaptation (UDA) framework for myocardial structure segmentation. We conduct extensive experiments on the MSCMRSeg 2019, MyoPS 2020 and MM-WHS 2017 datasets. The experimental results demonstrate that our framework achieves superior performances than state-of-the-art methods.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001285367200007</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>38546999</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Feature extraction; Image segmentation; Myocardium; Semantics; Annotations; Image reconstruction; Training; Cardiac segmentation; domain adaptation; multi-modal MRI; variational autoencoder</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Cui, Hengfei] Northwestern Polytech Univ, Sch Comp Sci, Natl Engn Lab Integrated Aerosp Ground Ocean Big D, Xian 710072, Peoples R China. <br>
[Cui, Hengfei] Northwestern Polytech Univ, Res &amp; Dev Inst, Shenzhen 518057, Peoples R China. <br>
[Cui, Hengfei] Northwestern Polytech Univ, Chongqing Innovat Ctr, Chongqing 401135, Peoples R China. <br>
[Li, Yan; Wang, Yifan; Xia, Yong] Northwestern Polytech Univ, Sch Comp Sci, Natl Engn Lab Integrated Aerosp Ground Ocean Big D, Xian 710072, Peoples R China. <br>
[Xu, Di] Media Innovat Lab, Huawei Cloud, Xian 710075, Peoples R China. <br>
[Wu, Lian-Ming] Shanghai Jiao Tong Univ, Renji Hosp, Sch Med, Dept Radiol, Shanghai 200127, Peoples R China. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Xia, Y (corresponding author), Northwestern Polytech Univ, Sch Comp Sci, Natl Engn Lab Integrated Aerosp Ground Ocean Big D, Xian 710072, Peoples R China.<br>Wu, LM (corresponding author), Shanghai Jiao Tong Univ, Renji Hosp, Sch Med, Dept Radiol, Shanghai 200127, Peoples R China.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
hfcui@nwpu.edu.cn; yanli.master1@gmail.com; yifanwang0229@163.com; xudi21@huawei.com; wlmssmu@126.com; yxia@nwpu.edu.cn</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Northwestern Polytechnical University; Northwestern Polytechnical University; Northwestern Polytechnical University; Northwestern Polytechnical University; Huawei Technologies; Shanghai Jiao Tong University</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Li, Yan</display_name>&nbsp;</td><td>&nbsp;</td><td>0009-0000-1331-9182&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Wu, Lian-Ming</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-7381-5436&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Wang, Yifan</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-4519-7535&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Cui, Hengfei</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-8625-2521&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Wang, Yifan</display_name>&nbsp;</td><td>JDM-1982-2023&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Interdisciplinary Applications; Engineering, Biomedical; Engineering, Electrical &amp; Electronic; Imaging Science &amp; Photographic Technology; Radiology, Nuclear Medicine &amp; Medical Imaging</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Engineering; Imaging Science &amp; Photographic Technology; Radiology, Nuclear Medicine &amp; Medical Imaging</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>A8Z6G</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0278-0062</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1558-254X</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE T MED IMAGING</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE Trans. Med. Imaging</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>13</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Natural Science Foundation of China</grant_agency>&nbsp;</td><td>
<div>62271405&nbsp;</div>
<div>62171377&nbsp;</div>
<div>82171884&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Guangdong Basic and Applied Basic Research Foundation</grant_agency>&nbsp;</td><td>
<div>2023A1515012847&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Natural Science Foundation of Chongqing, China</grant_agency>&nbsp;</td><td>
<div>CSTB2023NSCQ-MSX0286;&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Key Research and Development Program of Shaanxi Province, China</grant_agency>&nbsp;</td><td>
<div>2022GY-084&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Shenzhen Science and Technology Program</grant_agency>&nbsp;</td><td>
<div>JCYJ20220530161616036&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Shanghai Municipal Commission of Science and Technology Medical Innovation Research Special Project</grant_agency>&nbsp;</td><td>
<div>23Y11906900&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Shanghai "Yiyuan New Star" Outstanding Youth Talent (Excellent Program)</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported in part by the National Natural Science Foundation of China under Grant 62271405, Grant 62171377, and Grant 82171884; in part by Guangdong Basic and Applied Basic Research Foundation under Grant 2023A1515012847; in part by the Natural Science Foundation of Chongqing, China, under Grant CSTB2023NSCQ-MSX0286; in part by the Key Research and Development Program of Shaanxi Province, China, under Grant 2022GY-084; in part by Shenzhen Science and Technology Program under Grant JCYJ20220530161616036; in part by Shanghai Municipal Commission of Science and Technology Medical Innovation Research Special Project under Grant 23Y11906900; and in part by Shanghai "Yiyuan New Star" Outstanding Youth Talent (Excellent Program).</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 19 of 49</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Bidirectional cross-modality unsupervised domain adaptation using generative adversarial networks for cardiac image segmentation</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Cui, HF (Cui, Hengfei); Chang, YW (Chang Yuwen); Lei, J (Lei Jiang); Yong, X (Yong Xia); Zhang, YN (Zhang, Yanning)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>COMPUTERS IN BIOLOGY AND MEDICINE</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>136</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>104726</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1016/j.compbiomed.2021.104726</value>&nbsp;&nbsp;<b>Early Access Date:</b> 
<value>AUG 2021</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2021 SEP</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>27</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>30</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
4</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
53</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>50</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Background: A novel Generative Adversarial Networks (GAN) based bidirectional cross-modality unsupervised domain adaptation (GBCUDA) framework is developed for cardiac image segmentation, which can effectively tackle the problem of network's segmentation performance degradation when adapting to the target domain without ground truth labels. Method: GBCUDA uses GAN for image alignment, applies adversarial learning to extract image features, and gradually enhances the domain invariance of extracted features. The shared encoder performs an end-to-end learning task in which features that differ between the two domains complement each other. The selfattention mechanism is incorporated to the GAN network, which can generate details based on the prompts of all feature positions. Furthermore, spectrum normalization is implemented to stabilize the training of GAN, and knowledge distillation loss is introduced to process high-level feature-maps in order to better complete the crossmode segmentation task. Results: The effectiveness of our proposed unsupervised domain adaptation framework is tested over the MultiModality Whole Heart Segmentation (MM-WHS) Challenge 2017 dataset. The proposed method is able to improve the average Dice from 74.1% to 81.5% for the four cardiac substructures, and reduce the average symmetric surface distance (ASD) from 7.0 to 5.8 over CT images. For MRI images, our proposed framework trained on CT images gives the average Dice of 59.2% and reduces the average ASD from 5.7 to 4.9. Conclusions: The evaluation results demonstrate our method's effectiveness on domain adaptation and the superiority to the current state-of-the-art domain adaptation methods.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000707411400002</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>34371318</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Generative adversarial networks; Unsupervised domain adaptation; Cardiac segmentation; Self-attention mechanism; Knowledge distillation loss</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
SYNERGISTIC IMAGE; HEART</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Cui, Hengfei; Chang Yuwen; Lei Jiang; Yong Xia; Zhang, Yanning] Northwestern Polytech Univ, Sch Comp Sci, Natl Engn Lab Integrated Aerosp Ground Ocean Big, Xian 710072, Peoples R China. <br>
[Cui, Hengfei; Chang Yuwen; Lei Jiang; Yong Xia] Northwestern Polytech Univ, Sch Comp Sci &amp; Engn, Ctr Multidisciplinary Convergence Comp CMCC, Xian 710072, Peoples R China. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Cui, HF (corresponding author), Northwestern Polytech Univ, Sch Comp Sci, Natl Engn Lab Integrated Aerosp Ground Ocean Big, Xian 710072, Peoples R China.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
hfcui@nwpu.edu.cn</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Northwestern Polytechnical University; Northwestern Polytechnical University</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>lei, jiang</display_name>&nbsp;</td><td>GQA-9644-2022&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Zhang, Zhi-jun</display_name>&nbsp;</td><td>HGE-3958-2022&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Cui, Hengfei</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-8625-2521&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>PERGAMON-ELSEVIER SCIENCE LTD</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>THE BOULEVARD, LANGFORD LANE, KIDLINGTON, OXFORD OX5 1GB, ENGLAND</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Biology; Computer Science, Interdisciplinary Applications; Engineering, Biomedical; Mathematical &amp; Computational Biology</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Life Sciences &amp; Biomedicine - Other Topics; Computer Science; Engineering; Mathematical &amp; Computational Biology</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>WH0XE</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0010-4825</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1879-0534</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>COMPUT BIOL MED</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Comput. Biol. Med.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>10</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Natural Science Foundation of China</grant_agency>&nbsp;</td><td>
<div>61801393&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Fundamental Research Funds for the Central Universities</grant_agency>&nbsp;</td><td>
<div>3102020QD1001&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>The study was supported in part by the National Natural Science Foundation of China under Grant 61801393, and in part by the Fundamental Research Funds for the Central Universities under Grant 3102020QD1001.</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 20 of 49</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Unlocking the diagnostic potential of electrocardiograms through information transfer from cardiac magnetic resonance imaging</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Turgut,&Ouml; (Turgut, Ozgun); M&uuml;ller, P (Mueller, Philip); Hager, P (Hager, Paul); Shit, S (Shit, Suprosanna); Starck, S (Starck, Sophie); Menten, MJ (Menten, Martin J.); Martens, E (Martens, Eimo); Rueckert, D (Rueckert, Daniel)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>MEDICAL IMAGE ANALYSIS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>101</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>103451</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1016/j.media.2024.103451</value>&nbsp;&nbsp;<b>Early Access Date:</b> 
<value>JAN 2025</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2025 APR</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>0</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>0</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
3</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
8</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>73</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Cardiovascular diseases (CVD) can be diagnosed using various diagnostic modalities. The electrocardiogram (ECG) is a cost-effective and widely available diagnostic aid that provides functional information of the heart. However, its ability to classify and spatially localise CVD is limited. In contrast, cardiac magnetic resonance (CMR) imaging provides detailed structural information of the heart and thus enables evidence- based diagnosis of CVD, but long scan times and high costs limit its use in clinical routine. In this work, we present a deep learning strategy for cost-effective and comprehensive cardiac screening solely from ECG. Our approach combines multimodal contrastive learning with masked data modelling to transfer domain-specific information from CMR imaging to ECG representations. In extensive experiments using data from 40,044 UK Biobank subjects, we demonstrate the utility and generalisability of our method for subject-specific risk prediction of CVD and the prediction of cardiac phenotypes using only ECG data. Specifically, our novel multimodal pre-training paradigm improves performance by up to 12.19% for risk prediction and 27.59% for phenotype prediction. Ina qualitative analysis, we demonstrate that our learned ECG representations incorporate information from CMR image regions of interest. Our entire pipeline is publicly available at https://github.com/oetu/MMCL-ECG-CMR.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001409219400001</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>39793216</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Electrocardiogram; Cardiac magnetic resonance imaging; Multimodal contrastive learning; Masked data modelling; Self-supervised learning</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
DIABETES-MELLITUS; HEART-FAILURE; SEGMENTATION; DYSFUNCTION</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Turgut, Ozgun; Mueller, Philip; Hager, Paul; Starck, Sophie; Menten, Martin J.; Rueckert, Daniel] Tech Univ Munich, Sch Computat Informat &amp; Technol, Munich, Germany. <br>
[Turgut, Ozgun; Mueller, Philip; Hager, Paul; Starck, Sophie; Martens, Eimo; Rueckert, Daniel] Tech Univ Munich, Sch Med, Klinikum Rechts Isar, Munich, Germany. <br>
[Shit, Suprosanna] Univ Zurich, Dept Quant Biomed, Zurich, Switzerland. <br>
[Menten, Martin J.; Rueckert, Daniel] Munich Ctr Machine Learning, Munich, Germany. <br>
[Menten, Martin J.; Rueckert, Daniel] Imperial Coll London, Dept Comp, London, England. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Turgut,&Ouml; (corresponding author), Tech Univ Munich, Sch Computat Informat &amp; Technol, Munich, Germany.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
oezguen.turgut@tum.de</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Technical University of Munich; Technical University of Munich; University of Zurich; Imperial College London</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>M&uuml;ller, Philip</display_name>&nbsp;</td><td>HOH-3993-2023&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Hager, Paul</display_name>&nbsp;</td><td>KVZ-2285-2024&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Shit, Suprosanna</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-4435-7207&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Rueckert, Daniel</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-5683-5889&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Menten, Martin</display_name>&nbsp;</td><td>JAO-3681-2023&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Shit, Suprosanna</display_name>&nbsp;</td><td>AET-7423-2022&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Martens, Eimo</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-5801-0901&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Hager, Paul</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-2166-0262&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Martens, Eimo</display_name>&nbsp;</td><td>AAX-3975-2021&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Turgut, Ozgun</display_name>&nbsp;</td><td>&nbsp;</td><td>0009-0002-8704-0277&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Muller, Philip</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-8186-6479&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>ELSEVIER</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>RADARWEG 29, 1043 NX AMSTERDAM, NETHERLANDS</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Artificial Intelligence; Computer Science, Interdisciplinary Applications; Engineering, Biomedical; Radiology, Nuclear Medicine &amp; Medical Imaging</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Engineering; Radiology, Nuclear Medicine &amp; Medical Imaging</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>U1B0L</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>1361-8415</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1361-8423</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>MED IMAGE ANAL</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Med. Image Anal.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>11</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>UK Biobank Resource</grant_agency>&nbsp;</td><td>
<div>87802&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This research has been conducted using the UK Biobank Resource under Application Number 87802.</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>hybrid</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 21 of 49</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Adapt Everywhere: Unsupervised Adaptation of Point-Clouds and Entropy Minimization for Multi-Modal Cardiac Image Segmentation</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Vesal, S (Vesal, Sulaiman); Gu, MX (Gu, Mingxuan); Kosti, R (Kosti, Ronak); Maier, A (Maier, Andreas); Ravikumar, N (Ravikumar, Nishant)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE TRANSACTIONS ON MEDICAL IMAGING</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>40</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>7</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>1838-1851</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/TMI.2021.3066683</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2021 JUL</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>33</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>36</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
2</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
37</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>47</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Deep learning models are sensitive to domain shift phenomena. A model trained on images from one domain cannot generalise well when tested on images from a different domain, despite capturing similar anatomical structures. It is mainly because the data distribution between the two domains is different. Moreover, creating annotation for every new modality is a tedious and time-consuming task, which also suffers from high inter- and intra- observer variability. Unsupervised domain adaptation (UDA) methods intend to reduce the gap between source and target domains by leveraging source domain labelled data to generate labels for the target domain. However, current state-of-the-art (SOTA) UDA methods demonstrate degraded performance when there is insufficient data in source and target domains. In this paper, we present a novel UDA method for multi-modal cardiac image segmentation. The proposed method is based on adversarial learning and adapts network features between source and target domain in different spaces. The paper introduces an end-to-end framework that integrates: a) entropy minimization, b) output feature space alignment and c) a novel point-cloud shape adaptation based on the latent features learned by the segmentation model. We validated our method on two cardiac datasets by adapting from the annotated source domain, bSSFP-MRI (balanced Steady-State Free Procession-MRI), to the unannotated target domain, LGE-MRI (Late-gadolinium enhance-MRI), for the multi-sequence dataset; and from MRI (source) to CT (target) for the cross-modality dataset. The results highlighted that by enforcing adversarial learning in different parts of the network, the proposed method delivered promising performance, compared to other SOTA methods.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000668842500009</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>33729930</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Image segmentation; Shape; Entropy; Magnetic resonance imaging; Minimization; Adaptation models; Training; Unsupervised domain adaptation; cardiac segmentation; multi-modal segmentation; adversarial learning; point-clouds; entropy minimization</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
CARDIOVASCULAR MAGNETIC-RESONANCE; DOMAIN ADAPTATION; SYNERGISTIC IMAGE</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Vesal, Sulaiman; Gu, Mingxuan; Kosti, Ronak; Maier, Andreas] Friedrich Alexander Univ Erlangen Nuremberg, Pattern Recognit Lab, D-91054 Erlangen, Germany. <br>
[Ravikumar, Nishant] Univ Leeds, Ctr Computat Imaging &amp; Simulat Technol Biomed CIS, Sch Comp, Leeds Inst Cardiovasc &amp; Metab Med LICAMM,Sch Med, Leeds LS2 9JT, W Yorkshire, England. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Vesal, S (corresponding author), Friedrich Alexander Univ Erlangen Nuremberg, Pattern Recognit Lab, D-91054 Erlangen, Germany.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
sulaiman.vesal@fau.cie; mingxuan.gu@fau.de; ronak.kosti@fau.de; andreas.maier@fau.de; n.ravikumar@leeds.ac.uk</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
University of Erlangen Nuremberg; University of Leeds</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Maier, Andreas</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-9550-5284&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>KOSTI, RONAK</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-2453-7876&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Ravikumar, Nishant</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-0134-107X&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Vesal, Sulaiman</display_name>&nbsp;</td><td>ABF-2407-2021&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Vesal, Sulaiman</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-6156-9338&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Maier, Andreas</display_name>&nbsp;</td><td>AAV-6505-2021&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Gu, Mingxuan</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-5244-4397&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Gu, Mingxuan</display_name>&nbsp;</td><td>ACT-7421-2022&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Interdisciplinary Applications; Engineering, Biomedical; Engineering, Electrical &amp; Electronic; Imaging Science &amp; Photographic Technology; Radiology, Nuclear Medicine &amp; Medical Imaging</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Engineering; Imaging Science &amp; Photographic Technology; Radiology, Nuclear Medicine &amp; Medical Imaging</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>TC7TP</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0278-0062</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1558-254X</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE T MED IMAGING</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE Trans. Med. Imaging</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>14</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Project EFI-BIG-THERA: Integrative "BigData Modeling"</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>Manuscript received January 27, 2021; revised March 8, 2021; accepted March 14, 2021. Date of publication March 17, 2021; date of current version June 30, 2021. This work was supported by the Project EFI-BIG-THERA: Integrative "BigData Modeling" for the development of novel therapeutic approaches for breast cancer. (Corresponding author: Sulaiman Vesal.)</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 22 of 49</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Super-Resolution of Cardiac MR Cine Imaging using Conditional GANs and Unsupervised Transfer Learning</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Xia, Y (Xia, Yan); Ravikumar, N (Ravikumar, Nishant); Greenwood, JP (Greenwood, John P.); Neubauer, S (Neubauer, Stefan); Petersen, SE (Petersen, Steffen E.); Frangi, AF (Frangi, Alejandro F.)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>MEDICAL IMAGE ANALYSIS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>71</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>102037</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1016/j.media.2021.102037</value>&nbsp;&nbsp;<b>Early Access Date:</b> 
<value>APR 2021</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2021 JUL</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>46</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>50</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
5</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
39</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>60</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
High-resolution (HR), isotropic cardiac Magnetic Resonance (MR) cine imaging is challenging since it requires long acquisition and patient breath-hold times. Instead, 2D balanced steady-state free precession (SSFP) sequence is widely used in clinical routine. However, it produces highly-anisotropic image stacks, with large through-plane spacing that can hinder subsequent image analysis. To resolve this, we propose a novel, robust adversarial learning super-resolution (SR) algorithm based on conditional generative adversarial nets (GANs), that incorporates a state-of-the-art optical flow component to generate an auxiliary image to guide image synthesis. The approach is designed for real-world clinical scenarios and requires neither multiple low-resolution (LR) scans with multiple views, nor the corresponding HR scans, and is trained in an end-to-end unsupervised transfer learning fashion. The designed framework effectively incorporates visual properties and relevant structures of input images and can synthesise 3D isotropic, anatomically plausible cardiac MR images, consistent with the acquired slices. Experimental results show that the proposed SR method outperforms several state-of-the-art methods both qualitatively and quantitatively. We show that subsequent image analyses including ventricle segmentation, cardiac quantification, and non-rigid registration can benefit from the super-resolved, isotropic cardiac MR images, to produce more accurate quantitative results, without increasing the acquisition time. The average Dice similarity coefficient (DSC) for the left ventricular (LV) cavity and myocardium are 0.95 and 0.81, respectively, between real and synthesised slice segmentation. For non-rigid registration and motion tracking through the cardiac cycle, the proposed method improves the average DSC from 0.75 to 0.86, compared to the original resolution images.
<br>(c) 2021 The Authors. Published by Elsevier B.V. This is an open access article under the CC BY license ( http://creativecommons.org/licenses/by/4.0/ )</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000663615600008</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>33910110</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Cardiac MRI; Deep learning; Super-resolution; Conditional generative adversarial net; Optical flow; Conditional batch normalisation</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
CARDIOVASCULAR MAGNETIC-RESONANCE; UK BIOBANK; RESOLUTION; RATIONALE; DESIGN</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Xia, Yan; Ravikumar, Nishant; Frangi, Alejandro F.] Univ Leeds, Sch Comp, Ctr Computat Imaging &amp; Simulat Technol Biomed CIS, Leeds, W Yorkshire, England. <br>
[Xia, Yan; Ravikumar, Nishant; Greenwood, John P.; Frangi, Alejandro F.] Univ Leeds, Sch Med, Leeds Inst Cardiovasc &amp; Metab Med LICAMM, Leeds, W Yorkshire, England. <br>
[Neubauer, Stefan] Univ Oxford, John Radcliffe Hosp, Div Cardiovasc Med, Oxford Ctr Clin Magnet Resonance Res, Oxford, England. <br>
[Petersen, Steffen E.] Queen Mary Univ London, NIHR Barts Biomed Res Ctr, William Harvey Res Inst, London, England. <br>
[Petersen, Steffen E.] Barts Hlth NHS Trust, St Bartholomews Hosp, Barts Heart Ctr, London, England. <br>
[Frangi, Alejandro F.] Katholieke Univ Leuven, Cardiovasc Sci Dept, Med Imaging Res Ctr MIRC, Leuven, Belgium. <br>
[Frangi, Alejandro F.] Katholieke Univ Leuven, Dept Elect Engn, Med Imaging Res Ctr MIRC, Leuven, Belgium. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Xia, Y (corresponding author), Univ Leeds, Sch Comp, Ctr Computat Imaging &amp; Simulat Technol Biomed CIS, Leeds, W Yorkshire, England.<br>Xia, Y (corresponding author), Univ Leeds, Sch Med, Leeds Inst Cardiovasc &amp; Metab Med LICAMM, Leeds, W Yorkshire, England.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
y.xia@leeds.ac.uk</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
University of Leeds; University of Leeds; University of Oxford; University of London; Queen Mary University London; Barts Health NHS Trust; University of London; Queen Mary University London; KU Leuven; KU Leuven</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Neubauer, Stefan</display_name>&nbsp;</td><td>B-8448-2011&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Neubauer, Stefan</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-9017-5645&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Frangi, Alejandro</display_name>&nbsp;</td><td>C-6500-2008&nbsp;</td><td>0000-0002-2675-528X&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Greenwood, John</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-2861-0914&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Ravikumar, Nishant</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-0134-107X&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Petersen, Steffen</display_name>&nbsp;</td><td>A-8389-2011&nbsp;</td><td>0000-0003-4622-5160&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>ELSEVIER</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>RADARWEG 29, 1043 NX AMSTERDAM, NETHERLANDS</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Artificial Intelligence; Computer Science, Interdisciplinary Applications; Engineering, Biomedical; Radiology, Nuclear Medicine &amp; Medical Imaging</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Engineering; Radiology, Nuclear Medicine &amp; Medical Imaging</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>SV1WP</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>1361-8415</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1361-8423</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>MED IMAGE ANAL</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Med. Image Anal.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>17</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Royal Academy of Engineering Chair in Emerging Technologies Scheme</grant_agency>&nbsp;</td><td>
<div>CiET1819/19&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>EPSRC</grant_agency>&nbsp;</td><td>
<div>POC041&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>MedIAN Network</grant_agency>&nbsp;</td><td>
<div>EP/N026993/1&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Engineering and Physical Sciences Research Council (EPSRC)</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>EPSRC through TUSCA</grant_agency>&nbsp;</td><td>
<div>EP/V04799X/1&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>SmartHeart EPSRC programme</grant_agency>&nbsp;</td><td>
<div>EP/P001009/1&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>British Heart Foundation</grant_agency>&nbsp;</td><td>
<div>PG/14/89/31194&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Institute for Health Research (NIHR) Barts Biomedical Research Centre</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>European Union' s Horizon 2020 research and innovation programme</grant_agency>&nbsp;</td><td>
<div>825903&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>British Heart Foundation</grant_agency>&nbsp;</td><td>
<div>PG/14/89/31194&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Engineering and Physical Sciences Research Council</grant_agency>&nbsp;</td><td>
<div>EP/N026993/1&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This research has been conducted using the UK Biobank Resource under Application 11350. The cardiac MR images presented in Figs. 1, 2, 6, 7, 8, 9, 10, 11, 16 and 18 in the manuscript were reproduced with the permission of UK Biobank (c). The authors are grateful to all UK Biobank participants and staff. AFF acknowledges support from the Royal Academy of Engineering Chair in Emerging Technologies Scheme (CiET1819/19) , EPSRC-funded Grow MedTech CardioX (POC041) , and the MedIAN Network (EP/N026993/1) funded by the Engineering and Physical Sciences Research Council (EPSRC) . The work of AFF is also partially funded by EPSRC through TUSCA (EP/V04799X/1) . SN acknowledges the National Institute for Health Research (NIHR) Oxford Biomedical Research Centre based at The Oxford University Hospitals Trust at the University of Oxford, and the British Heart Foundation Centre of Research Excellence. SEP acknowledges support from the SmartHeart EPSRC programme grant ( www.nihr.ac.uk; EP/P001009/1) , the British Heart Foundation for funding the manual analysis to create a cardiovascular magnetic resonance imaging reference standard for the UK Biobank imaging resource in 5,0 0 0 CMR scans ( www.bhf.org.uk; PG/14/89/31194) , and the National Institute for Health Research (NIHR) Barts Biomedical Research Centre. SEP has received funding from the European Union' s Horizon 2020 research and innovation programme under grant agreement No 825903 (euCanSHare project) .</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>Green Accepted, hybrid</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 23 of 49</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Diagnostic quality assessment for low-dimensional ECG representations</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Kov&aacute;cs, P (Kovacs, Peter); B&ouml;ck, C (Boeck, Carl); Tschoellitsch, T (Tschoellitsch, Thomas); Huemer, M (Huemer, Mario); Meier, J (Meier, Jens)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>COMPUTERS IN BIOLOGY AND MEDICINE</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>150</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>106086</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1016/j.compbiomed.2022.106086</value>&nbsp;&nbsp;<b>Early Access Date:</b> 
<value>SEP 2022</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2022 NOV</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>3</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>3</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
0</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
6</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>30</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
There have been several attempts to quantify the diagnostic distortion caused by algorithms that perform low -dimensional electrocardiogram (ECG) representation. However, there is no universally accepted quantitative measure that allows the diagnostic distortion arising from denoising, compression, and ECG beat representation algorithms to be determined. Hence, the main objective of this work was to develop a framework to enable biomedical engineers to efficiently and reliably assess diagnostic distortion resulting from ECG processing algorithms. We propose a semiautomatic framework for quantifying the diagnostic resemblance between original and denoised/reconstructed ECGs. Evaluation of the ECG must be done manually, but is kept simple and does not require medical training. In a case study, we quantified the agreement between raw and reconstructed (denoised) ECG recordings by means of kappa-based statistical tests. The proposed methodology takes into account that the observers may agree by chance alone. Consequently, for the case study, our statistical analysis reports the "true", beyond-chance agreement in contrast to other, less robust measures, such as simple percent agreement calculations. Our framework allows efficient assessment of clinically important diagnostic distortion, a potential side effect of ECG (pre-)processing algorithms. Accurate quantification of a possible diagnostic loss is critical to any subsequent ECG signal analysis, for instance, the detection of ischemic ST episodes in long-term ECG recordings.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000878510400004</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>36191392</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Baseline removal; Clinical evaluation; Diagnostic distortion measures; ECG denoising; Kappa statistics</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
HIGH AGREEMENT; LOW KAPPA</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Kovacs, Peter] Eotvos Lorand Univ, Dept Numer Anal, Pazmany Peter Setany 1, H-1117 Budapest, Hungary. <br>
[Boeck, Carl] Johannes Kepler Univ Linz, Inst Signal Proc, JKU LIT SAL eSPML Lab, Altenberger Str 69, A-4040 Linz, Austria. <br>
[Tschoellitsch, Thomas; Meier, Jens] Johannes Kepler Univ Linz, Clin Anesthesiol &amp; Intens Care Med, Krankenhausstr 9, A-4020 Linz, Austria. <br>
[Huemer, Mario] Johannes Kepler Univ Linz, Inst Signal Proc, Altenberger Str 69, A-4040 Linz, Austria. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Kov&aacute;cs, P (corresponding author), Eotvos Lorand Univ, Dept Numer Anal, Pazmany Peter Setany 1, H-1117 Budapest, Hungary.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
kovika@inf.elte.hu; carl.boeck@jku.at; Thomas.Tschoellitsch@kepleruniklinkum.at; mario.huemer@jku.at; Jens.Meier@kepleruniklinkum.at</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Eotvos Lorand University; Johannes Kepler University Linz; Johannes Kepler University Linz; Johannes Kepler University Linz</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Bock, Carl</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-2207-2411&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Tschoellitsch, Thomas</display_name>&nbsp;</td><td>HJP-9501-2023&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Kov&aacute;cs, P&eacute;ter</display_name>&nbsp;</td><td>H-7307-2017&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>PERGAMON-ELSEVIER SCIENCE LTD</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>THE BOULEVARD, LANGFORD LANE, KIDLINGTON, OXFORD OX5 1GB, ENGLAND</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Biology; Computer Science, Interdisciplinary Applications; Engineering, Biomedical; Mathematical &amp; Computational Biology</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Life Sciences &amp; Biomedicine - Other Topics; Computer Science; Engineering; Mathematical &amp; Computational Biology</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>5X3NV</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0010-4825</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1879-0534</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>COMPUT BIOL MED</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Comput. Biol. Med.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>9</value>
</td>
</tr>

<tr>
<td>
<b>Open Access:</b>

<value>Green Published, hybrid, Green Submitted</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 24 of 49</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>An automated ECG signal quality assessment method for unsupervised diagnostic systems</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Satija, U (Satija, Udit); Ramkumar, B (Ramkumar, Barathram); Manikandan, MS (Manikandan, M. Sabarimalai)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>BIOCYBERNETICS AND BIOMEDICAL ENGINEERING</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>38</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>1</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>54-70</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1016/j.bbe.2017.10.002</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2018 </value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>25</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>30</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
1</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
28</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>46</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
In this paper, the authors present an automated method for quality assessment of electrocardiogram (ECG) signal. Our proposed method not only detects and classifies the ECG noises but also localizes the ECG noises which can play a crucial role in extracting reliable clinical features for ECG analysis systems. The proposed method is based on three stages: Wavelet decomposition of ECG signal into sub-bands; simultaneous ECG signal and noise reconstruction; extraction of temporal features such as maximum absolute amplitude, zerocrossings, kurtosis and autocorrelation function for detection, localization and classification of ECG noises including flat line (FL), time-varying noise or pause (TVN), baseline wander (BW), abrupt change (AB), power line interference (PLI), muscle artifacts (MA) and additive white Gaussian noise (AWGN). The proposed method is tested and validated against manually annotated ECG signals corrupted with aforementioned noises taken from MIT-BIH arrhythmia database, Physionet challenge database, and real-time recorded ECG signals. Comparative detection and classification results depict the superior performance of the proposed method over state of art methods. Detection results show that our method can achieve an average sensitivity (Se), average specificity (Sp) and accuracy (A) of 99.61%, 98.51%, 99.49% respectively. Also, the method achieves a Se of 98.18%, and Sp of 94.97% for real-time recorded ECG signals. The method has an average timing error of 0.14 s in localizing the noise segments. Further, classification results demonstrate that the proposed method achieves an average sensitivity (Se), average positive predictivity (PP) and classification accuracy (Ac) of 98.53%, 98.89%, 97.50% respectively. (C) 2017 Nalecz Institute of Biocybernetics and Biomedical Engineering of the Polish Academy of Sciences. Published by Elsevier B.V. All rights reserved.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000425533400004</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Signal quality assessment; Electrocardiogram; Baseline wander; Muscle artifacts</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
DETECTION ALGORITHM; ELECTROCARDIOGRAM</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Satija, Udit] Shiv Nadar Univ, Dept Elect Engn, Greater Noida 201314, Uttar Pradesh, India. <br>
[Ramkumar, Barathram; Manikandan, M. Sabarimalai] Indian Inst Technol Bhubaneswar, Jatni, Odisha, India. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Satija, U (corresponding author), Shiv Nadar Univ, Dept Elect Engn, Greater Noida 201314, Uttar Pradesh, India.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
us11@iitbbs.ac.in; barathram24@gmail.com; msm@iitbbs.ac.in</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Shiv Nadar University; Indian Institute of Technology System (IIT System); Indian Institute of Technology (IIT) - Bhubaneswar</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Manikandan, M Sabarimalai</display_name>&nbsp;</td><td>V-6236-2019&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>SATIJA, UDIT</display_name>&nbsp;</td><td>AAA-5332-2021&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Manikandan, M Sabarimalai</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-6878-4911&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>SATIJA, UDIT</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-3851-765X&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>ELSEVIER SCIENCE BV</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>PO BOX 211, 1000 AE AMSTERDAM, NETHERLANDS</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Engineering, Biomedical</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Engineering</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>FW7VA</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0208-5216</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>BIOCYBERN BIOMED ENG</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Biocybern. Biomed. Eng.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>17</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 25 of 49</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Automated cardiac segmentation of cross-modal medical images using unsupervised multi-domain adaptation and spatial neural attention structure</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Liu, JP (Liu, Jinping); Liu, H (Liu, Hui); Gong, SB (Gong, Subo); Tang, ZH (Tang, Zhaohui); Xie, YF (Xie, Yongfang); Yin, HZ (Yin, Huazhan); Niyoyita, JP (Niyoyita, Jean Paul)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>MEDICAL IMAGE ANALYSIS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>72</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>102135</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1016/j.media.2021.102135</value>&nbsp;&nbsp;<b>Early Access Date:</b> 
<value>JUN 2021</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2021 AUG</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>35</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>42</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
10</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
65</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>54</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Accurate cardiac segmentation of multimodal images, e.g., magnetic resonance (MR), computed tomography (CT) images, plays a pivot role in auxiliary diagnoses, treatments and postoperative assessments of cardiovascular diseases. However, training a well-behaved segmentation model for the cross-modal cardiac image analysis is challenging, due to their diverse appearances/distributions from different devices and acquisition conditions. For instance, a well-trained segmentation model based on the source domain of MR images is often failed in the segmentation of CT images. In this work, a cross-modal images oriented cardiac segmentation scheme is proposed using a symmetric full convolutional neural network (SFCNN) with the unsupervised multi-domain adaptation (UMDA) and a spatial neural attention (SNA) structure, termed UMDA-SNA-SFCNN, having the merits of without the requirement of any annotation on the test domain. Specifically, UMDA-SNA-SFCNN incorporates SNA to the classic adversarial domain adaptation network to highlight the relevant regions, while restraining the irrelevant areas in the cross modal images, so as to suppress the negative transfer in the process of unsupervised domain adaptation. In addition, the multi-layer feature discriminators and a predictive segmentation-mask discriminator are established to connect the multi-layer features and segmentation mask of the backbone network, SFCNN, to realize the fine-grained alignment of unsupervised cross-modal feature domains. Extensive confirmative and comparative experiments on the benchmark Multi-Modality Whole Heart Challenge dataset show that the proposed model is superior to the state-of-the-art cross-modal segmentation methods. (c) 2021 Elsevier B.V. All rights reserved.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000681314900010</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>34182202</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Cross-modal cardiac image segmentation; Unsupervised multi-domain adaptation; Spatial neural attention; Symmetric full convolutional neural; network</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
DOMAIN ADAPTATION</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Liu, Jinping; Liu, Hui; Yin, Huazhan] Hunan Normal Univ, Hunan Prov Key Lab Intelligent Comp &amp; Language In, Changsha 410081, Peoples R China. <br>
[Liu, Jinping; Liu, Hui] Hunan Normal Univ, Minist Educ, Key Lab Comp &amp; Stochast Math, Changsha 410081, Peoples R China. <br>
[Liu, Jinping] Hunan Normal Univ, Hunan Xiangjiang Artificial Intelligence Acad, Changsha 410081, Peoples R China. <br>
[Gong, Subo] Cent South Univ, Xiangya Hosp 2, Dept Geriatr Med, Changsha 410011, Peoples R China. <br>
[Tang, Zhaohui; Xie, Yongfang] Cent South Univ, Sch Automat, Changsha 410083, Peoples R China. <br>
[Yin, Huazhan] Hunan Normal Univ, Cognit &amp; Human Behav Key Lab Hunan Prov, Changsha 410081, Peoples R China. <br>
[Niyoyita, Jean Paul] Univ Rwanda, Coll Sci &amp; Technol, Kigali 3286, Rwanda. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Liu, JP (corresponding author), Hunan Normal Univ, Coll Informat Sci &amp; Engn, Hunan Prov Key Lab Intelligent Comp &amp; Language In, HN731, Changsha, Peoples R China.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
ljp202518@163.com</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Hunan Normal University; Hunan Normal University; Hunan Normal University; Central South University; Central South University; Hunan Normal University; University of Rwanda</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Liu, Jinping</display_name>&nbsp;</td><td>AAM-7723-2021&nbsp;</td><td>0000-0002-8669-882X&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Liu, Hui</display_name>&nbsp;</td><td>G-2051-2014&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Niyoyita, Jean Paul</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-5281-459X&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>ELSEVIER</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>RADARWEG 29, 1043 NX AMSTERDAM, NETHERLANDS</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Artificial Intelligence; Computer Science, Interdisciplinary Applications; Engineering, Biomedical; Radiology, Nuclear Medicine &amp; Medical Imaging</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Engineering; Radiology, Nuclear Medicine &amp; Medical Imaging</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>TU8XW</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>1361-8415</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1361-8423</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>MED IMAGE ANAL</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Med. Image Anal.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>12</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Natural Science Foundation of China (NSFC)</grant_agency>&nbsp;</td><td>
<div>61971188&nbsp;</div>
<div>61771492&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Science Fund for Distinguished Young Scholars</grant_agency>&nbsp;</td><td>
<div>61725306&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>NSFC</grant_agency>&nbsp;</td><td>
<div>U1701261&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Guangdong provincial government</grant_agency>&nbsp;</td><td>
<div>U1701261&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>project of Educational Commission of Hunan Province of China</grant_agency>&nbsp;</td><td>
<div>19B364&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported by the National Natural Science Foundation of China (NSFC) under Grant Nos. 61971188, 61771492, the National Science Fund for Distinguished Young Scholars under grant No. 61725306 and the joint fund of NSFC and Guangdong provincial government under grant No. U1701261, in part by the project of Educational Commission of Hunan Province of China un-der grant No. 19B364.</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 26 of 49</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Universal ventricular coordinates: A generic framework for describing position within the heart and transferring data</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Bayer, J (Bayer, Jason); Prassl, AJ (Prassl, Anton J.); Pashaei, A (Pashaei, Ali); Gomez, JF (Gomez, Juan F.); Frontera, A (Frontera, Antonio); Neic, A (Neic, Aurel); Plank, G (Plank, Gernot); Vigmond, EJ (Vigmond, Edward J.)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>MEDICAL IMAGE ANALYSIS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>45</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>83-93</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1016/j.media.2018.01.005</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2018 APR</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>79</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>87</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
0</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
0</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>26</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Being able to map a particular set of cardiac ventricles to a generic topologically equivalent representation has many applications, including facilitating comparison of different hearts, as well as mapping quantities and structures of interest between them. In this paper we describe Universal Ventricular Coordinates (UVC), which can be used to describe position within any biventricular heart. UVC comprise four unique coordinates that we have chosen to be intuitive, well defined, and relevant for physiological descriptions. We describe how to determine these coordinates for any volumetric mesh by illustrating how to properly assign boundary conditions and utilize solutions to Laplace's equation. Using UVC, we transferred scalar, vector, and tensor data between four unstructured ventricular meshes from three different species. Performing the mappings was very fast, on the order of a few minutes, since mesh nodes were searched in a KD tree. Distance errors in mapping mesh nodes back and forth between meshes were less than the size of an element. Analytically derived fiber directions were also mapped across meshes and compared, showing &lt;5 degrees difference over most of the ventricles. The ability to transfer gradients was also demonstrated. Topologically variable structures, like papillary muscles, required further definition outside of the UVC framework. In conclusion, UVC can aid in transferring many types of data between different biventricular geometries. (C) 2018 The Author(s). Published by Elsevier B.V.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000427664400007</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>29414438</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Mapping; Coordinates; Volumetric meshes; Deformation</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
REPOLARIZATION GRADIENTS; FIBER ORIENTATION; MODELS; ANATOMY; FAILURE</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Bayer, Jason; Pashaei, Ali; Gomez, Juan F.; Frontera, Antonio; Vigmond, Edward J.] Bordeaux Fdn, LIRYC Electrophysiol &amp; Heart Modeling Inst, Ave Haut Leveque, F-33600 Pessac, France. <br>
[Bayer, Jason; Pashaei, Ali; Gomez, Juan F.; Vigmond, Edward J.] Univ Bordeaux, IMB Bordeaux Inst Math, 351 Cours Liberat, F-33405 Talence, France. <br>
[Prassl, Anton J.; Neic, Aurel; Plank, Gernot] Med Univ Graz, Gottfried Schatz Res Ctr, Biophys, Neue Stiftingtalstr 6, A-8010 Graz, Austria. <br>
[Frontera, Antonio] Hop Haut Leveque, Dept Electrophysiol, 1 Ave Magellan, F-33100 Pessac, France. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Vigmond, EJ (corresponding author), Bordeaux Fdn, LIRYC Electrophysiol &amp; Heart Modeling Inst, Ave Haut Leveque, F-33600 Pessac, France.<br>Vigmond, EJ (corresponding author), Univ Bordeaux, IMB Bordeaux Inst Math, 351 Cours Liberat, F-33405 Talence, France.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
jason.bayer@ihu-liryc.fr; anton.prassl@medunigraz.at; ali.pashaei@u-bordeaux.fr; juan.gomez@ihu-liryc.fr; a.frontera@gmail.com; aurel.neic@medunigraz.at; gernot.plank@medunigraz.at; edward.vigmond@u-bordeaux.fr</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
CHU Bordeaux; Universite de Bordeaux; Medical University of Graz; Universite de Bordeaux; CHU Bordeaux</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Neic, Aurel</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-5192-1307&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>F Gomez, Juan</display_name>&nbsp;</td><td>B-5228-2017&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Frontera, Antonio</display_name>&nbsp;</td><td>AAG-6538-2019&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Plank, Gernot</display_name>&nbsp;</td><td>C-9498-2011&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Bayer, Jason</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-0269-3499&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Vigmond, Edward</display_name>&nbsp;</td><td>E-8988-2012&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Bayer, Jason</display_name>&nbsp;</td><td>HKV-2373-2023&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Gomez, Juan F</display_name>&nbsp;</td><td>B-5228-2017&nbsp;</td><td>0000-0003-0253-4842&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Frontera, Antonio</display_name>&nbsp;</td><td>HIR-6883-2022&nbsp;</td><td>0000-0002-0372-4480&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Vigmond, Edward</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-1388-3589&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Prassl, Anton J</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-1920-1377&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>ELSEVIER SCIENCE BV</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>PO BOX 211, 1000 AE AMSTERDAM, NETHERLANDS</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Artificial Intelligence; Computer Science, Interdisciplinary Applications; Engineering, Biomedical; Radiology, Nuclear Medicine &amp; Medical Imaging</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Engineering; Radiology, Nuclear Medicine &amp; Medical Imaging</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>FZ5VM</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>1361-8415</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1361-8423</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>MED IMAGE ANAL</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Med. Image Anal.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>11</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Agence Nationale de Recherche (ANR) as part of the Investissements d'Avenir program</grant_agency>&nbsp;</td><td>
<div>ANR-10-IAHU-04&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>ANR</grant_agency>&nbsp;</td><td>
<div>ANR-16-CE19-0009&nbsp;</div>
<div>ANR-13-MONU-0004-02&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Whitaker International Program</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Lefoulon-Delalande Foundation</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Austrian Science Fund (FWF)</grant_agency>&nbsp;</td><td>
<div>F3210-N18&nbsp;</div>
<div>12760-B30&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>EU</grant_agency>&nbsp;</td><td>
<div>CardioProof 611232&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>ERACoSysMed PUSHCART project</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Austrian Science Fund (FWF)</grant_agency>&nbsp;</td><td>
<div>I2760&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Agence Nationale de la Recherche (ANR)</grant_agency>&nbsp;</td><td>
<div>ANR-16-CE19-0009&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was funded in part by the Agence Nationale de Recherche (ANR) as part of the Investissements d'Avenir program, grant ANR-10-IAHU-04. J.Bayer was supported in part by the ANR grant ANR-16-CE19-0009, the Whitaker International Program administered by the Institute of International Education, and by the Lefoulon-Delalande Foundation administered by the Institute of France. A. Pashaei was supported by the ANR grant ANR-13-MONU-0004-02. Funding to G Plank was provided by the grants F3210-N18 and 12760-B30 from the Austrian Science Fund (FWF) and the EU grant CardioProof 611232. E. Vigmond and G. Plank were also supported by the ERACoSysMed PUSHCART project. Simulations for this study were performed on the high performance computing platform Avakas, which is maintained by the Mesocentre de Calcul Intensif Aquitain (MCIA). We would also like to acknowledge PRACE support for computations performed on Marconi@CINECA, Italy and ARCHER, UK.</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>hybrid, Green Submitted</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 27 of 49</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Unsupervised Domain Adaptation From Axial to Short-Axis Multi-Slice Cardiac MR Images by Incorporating Pretrained Task Networks</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Koehler, S (Koehler, Sven); Hussain, T (Hussain, Tarique); Blair, Z (Blair, Zach); Huffaker, T (Huffaker, Tyler); Ritzmann, F (Ritzmann, Florian); Tandon, A (Tandon, Animesh); Pickardt, T (Pickardt, Thomas); Sarikouch, S (Sarikouch, Samir); Latus, H (Latus, Heiner); Greil, G (Greil, Gerald); Wolf, I (Wolf, Ivo); Engelhardt, S (Engelhardt, Sandy)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE TRANSACTIONS ON MEDICAL IMAGING</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>40</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>10</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>2939-2953</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/TMI.2021.3052972</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2021 OCT</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>10</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>11</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
0</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
14</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>44</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Anisotropic multi-slice Cardiac Magnetic Resonance (CMR) Images are conventionally acquired in patient-specific short-axis (SAX) orientation. In specific cardiovascular diseases that affect right ventricular (RV) morphology, acquisitions in standard axial (AX) orientation are preferred by some investigators, due to potential superiority in RV volume measurement for treatment planning. Unfortunately, due to the rare occurrence of these diseases, data in this domain is scarce. Recent research in deep learning-based methods mainly focused on SAX CMR images and they had proven to be very successful. In this work, we show that there is a considerable domain shift between AX and SAX images, and therefore, direct application of existing models yield sub-optimal results on AX samples. We propose a novel unsupervised domain adaptation approach, which uses task-related probabilities in an attention mechanism. Beyond that, cycle consistency is imposed on the learned patient-individual 3D rigid transformation to improve stability when automatically re-sampling the AX images to SAX orientations. The network was trained on 122 registered 3D AX-SAX CMR volume pairs from a multi-centric patient cohort. A mean 3D Dice of 0.86 +/- 0.06 for the left ventricle, 0.65 +/- 0.08 for the myocardium, and 0.77 +/- 0.10 for the right ventricle could be achieved. This is an improvement of 25% in Dice for RV in comparison to direct application on axial slices. To conclude, our pre-trained task module has neither seen CMR images nor labels from the target domain, but is able to segment them after the domain gap is reduced. Code: https://github.com/Cardio-AI/3d-mri-domain-adaptation</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000702638800033</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>33471750</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Task analysis; Heart; Image segmentation; Three-dimensional displays; Deep learning; Training; Biomedical imaging; Cardiac magnetic resonance; short axis images; spatial transformer networks; unsupervised domain adaptation; competence network for congenital heart defects</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
SEGMENTATION; ORIENTATION</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Koehler, Sven; Ritzmann, Florian; Engelhardt, Sandy] Univ Heidelberg Hosp, Dept Internal Med 3, Artificial Intelligence Cardiovasc Med Grp, D-69120 Heidelberg, Germany. <br>
[Koehler, Sven; Ritzmann, Florian; Engelhardt, Sandy] German Ctr Cardiovasc Res DZHK, D-68167 Heidelberg, Germany. <br>
[Hussain, Tarique; Tandon, Animesh; Greil, Gerald] Univ Texas Southwestern Med Ctr Dallas, Dept Pediat, Dallas, TX 75390 USA. <br>
[Hussain, Tarique; Tandon, Animesh; Greil, Gerald] Univ Texas Southwestern Med Ctr Dallas, Div Cardiol, Dallas, TX 75390 USA. <br>
[Hussain, Tarique; Tandon, Animesh; Greil, Gerald] Univ Texas Southwestern Med Ctr Dallas, Dept Radiol, Dallas, TX 75390 USA. <br>
[Hussain, Tarique; Tandon, Animesh; Greil, Gerald] Univ Texas Southwestern Med Ctr Dallas, Adv Imaging Res Ctr, Dallas, TX 75390 USA. <br>
[Blair, Zach; Huffaker, Tyler] Univ Texas Southwestern Med Ctr Dallas, Dept Pediat, Dallas, TX 75390 USA. <br>
[Pickardt, Thomas] German Competence Network Congenital Heart, D-13353 Berlin, Germany. <br>
[Pickardt, Thomas] German Ctr Cardiovasc Res DZHK, D-68167 Berlin, Germany. <br>
[Sarikouch, Samir] Hannover Med Sch, Dept Cardiothorac, Transplantat &amp; Vasc Surg, D-30625 Hannover, Germany. <br>
[Sarikouch, Samir] German Competence Network Congenital Heart Defect, D-13353 Berlin, Germany. <br>
[Sarikouch, Samir] German Ctr Cardiovasc Res DZHK, D-68167 Berlin, Germany. <br>
[Latus, Heiner] Tech Univ Munich, Dept Paediat &amp; Congenital Heart Defects, German Heart Ctr Munich, D-80636 Munich, Germany. <br>
[Wolf, Ivo] Mannheim Univ Appl Sci, Dept Comp Sci, D-68163 Mannheim, Germany. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Engelhardt, S (corresponding author), Univ Heidelberg Hosp, Dept Internal Med 3, Artificial Intelligence Cardiovasc Med Grp, D-69120 Heidelberg, Germany.<br>Engelhardt, S (corresponding author), German Ctr Cardiovasc Res DZHK, D-68167 Heidelberg, Germany.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
sven.koehler@med.uni-heidelberg.de; mohammad.hussain@utsouthwestern.edu; zachary.blair@utsouthwestern.edu; tyler.huffaker@utsouthwestern.edu; florian.ritzmann2@hs-mannheim.de; animesh.tandon@utsouthwestern.edu; pickardt@kompetenznetz-ahf.de; sarikouch.samir@mh-hannover.de; latus@dhm.mhn.de; gerald.greil@utsouthwestern.edu; i.wolf@hs-mannheim.de; sandy.engelhardt@med.uni-heidelberg.de</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Ruprecht Karls University Heidelberg; German Centre for Cardiovascular Research; University of Texas System; University of Texas Southwestern Medical Center; University of Texas System; University of Texas Southwestern Medical Center; University of Texas System; University of Texas Southwestern Medical Center; University of Texas System; University of Texas Southwestern Medical Center; University of Texas System; University of Texas Southwestern Medical Center; German Centre for Cardiovascular Research; Hannover Medical School; German Centre for Cardiovascular Research; Technical University of Munich; German Heart Centre Munich</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Koehler, Sven</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-4989-8766&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Koehler, Sven</display_name>&nbsp;</td><td>HMP-6265-2023&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Greil, Gerald</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-7288-6566&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Hussain, Tariq</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-1332-2418&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Huffaker, Tyler</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-9442-8600&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Pickardt, Thomas</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-0019-6325&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Tandon, Animesh</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-9769-8801&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Wolf, Ivo</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-6519-6484&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Hussain, Tarique</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-4091-992X&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Interdisciplinary Applications; Engineering, Biomedical; Engineering, Electrical &amp; Electronic; Imaging Science &amp; Photographic Technology; Radiology, Nuclear Medicine &amp; Medical Imaging</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Engineering; Imaging Science &amp; Photographic Technology; Radiology, Nuclear Medicine &amp; Medical Imaging</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>WA1FC</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0278-0062</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1558-254X</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE T MED IMAGING</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE Trans. Med. Imaging</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>15</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Informatics for Life Project through the Klaus Tschira Foundation</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Competence Network for Congenital Heart Defects through the Federal Ministry of Education and Research</grant_agency>&nbsp;</td><td>
<div>01GI0601&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>German Centre for Cardiovascular Research (DZHK)</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported in part by the Informatics for Life Project through the Klaus Tschira Foundation, in part by the Competence Network for Congenital Heart Defects through the Federal Ministry of Education and Research under Grant 01GI0601, and in part by the German Centre for Cardiovascular Research (DZHK).</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>Green Submitted, Green Accepted</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 28 of 49</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Towards federated transfer learning in electrocardiogram signal analysis</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Chorney, W (Chorney, Wesley); Wang, HF (Wang, Haifeng)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>COMPUTERS IN BIOLOGY AND MEDICINE</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>170</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>107984</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1016/j.compbiomed.2024.107984</value>&nbsp;&nbsp;<b>Early Access Date:</b> 
<value>JAN 2024</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2024 MAR</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>7</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>10</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
6</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
20</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>79</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Modern methods in artificial intelligence perform very well on many healthcare datasets, at times outperforming trained doctors. However, many assumptions made in model training are not justifiable in clinical settings. In this work, we propose a method to train classifiers for electrocardiograms, able to deal with data of disparate input dimensions, distributed across different institutions, and able to protect patient privacy. In addition, we propose a simple method for creating federated datasets from any centralized dataset. We use autoencoders in conjunction with federated learning to model a highly heterogeneous modeling problem using the Massachusetts Institute of Technology Beth Israel Hospital Arrhythmia dataset, the Computing in Cardiology 2017 challenge dataset, and the PTB-XL dataset. For an encoding dimension of 1000, our federated classifier achieves an accuracy, precision, recall, and F1 score of 73.0%, 66.6%, 73.0%, and 69.7%, respectively. Our results suggest that dropping commonly made assumptions significantly complicate training and that as a result, estimates of performance of many machine learning models may overestimate performance when adopted for clinical settings.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001168093000001</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>38244469</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Federated transfer learning; Electrocardiograms; Autoencoders; Data privacy; Healthcare</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
ATRIAL-FIBRILLATION; ECG SIGNALS; NONINVASIVE ASSESSMENT; MYOCARDIAL-INFARCTION; HELICOBACTER-PYLORI; DIAGNOSIS; AI; ARRHYTHMIAS; CHALLENGES</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Chorney, Wesley] Mississippi State Univ, Computat Engn, Mississippi State, MS 39762 USA. <br>
[Wang, Haifeng] Mississippi State Univ, Ind &amp; Syst Engn, Mississippi State, MS 39762 USA. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Chorney, W (corresponding author), Mississippi State Univ, Computat Engn, Mississippi State, MS 39762 USA.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
wwc102@msstate.edu; wang@ise.msstate.edu</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Mississippi State University; Mississippi State University</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Chorney, Wesley</display_name>&nbsp;</td><td>&nbsp;</td><td>0009-0008-2598-6837&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Chorney, Wesley</display_name>&nbsp;</td><td>IWV-4143-2023&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>PERGAMON-ELSEVIER SCIENCE LTD</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>THE BOULEVARD, LANGFORD LANE, KIDLINGTON, OXFORD OX5 1GB, ENGLAND</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Biology; Computer Science, Interdisciplinary Applications; Engineering, Biomedical; Mathematical &amp; Computational Biology</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Life Sciences &amp; Biomedicine - Other Topics; Computer Science; Engineering; Mathematical &amp; Computational Biology</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>IR6J4</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0010-4825</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1879-0534</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>COMPUT BIOL MED</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Comput. Biol. Med.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>15</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 29 of 49</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Improving electrocardiogram-based detection of rare genetic heart disease using transfer learning: An application to phospholamban p.Arg14del mutation carriers</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Lopes, RR (Lopes, Ricardo R.); Bleijendaal, H (Bleijendaal, Hidde); Ramos, LA (Ramos, Lucas A.); Verstraelen, TE (Verstraelen, Tom E.); Amin, AS (Amin, Ahmad S.); Wilde, AAM (Wilde, Arthur A. M.); Pinto, YM (Pinto, Yigal M.); de Mol, BAJM (de Mol, Bas A. J. M.); Marquering, HA (Marquering, Henk A.)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>COMPUTERS IN BIOLOGY AND MEDICINE</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>131</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>104262</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1016/j.compbiomed.2021.104262</value>&nbsp;&nbsp;<b>Early Access Date:</b> 
<value>FEB 2021</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2021 APR</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>30</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>35</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
1</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
10</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>31</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
The pathogenic mutation p.Arg14del in the gene encoding Phospholamban (PLN) is known to cause cardiomyopathy and leads to increased risk of sudden cardiac death. Automatic tools might improve the detection of patients with this rare disease. Deep learning is currently the state-of-the-art in signal processing but requires large amounts of data to train the algorithms. In situations with relatively small amounts of data, like PLN, transfer learning may improve accuracy. We propose an ECG-based detection of the PLN mutation using transfer learning from a model originally trained for sex identification. The sex identification model was trained with 256,278 ECGs and subsequently finetuned for PLN detection (155 ECGs of patients with PLN) with two control groups: a balanced age/sex matched group and a randomly selected imbalanced population. The data was split in 10 folds and 20% of the training data was used for validation and early stopping. The models were evaluated with the area under the receiver operating characteristic curve (AUROC) of the testing data. We used gradient activation for explanation of the prediction models. The models trained with transfer learning outperformed the models trained from scratch for both the balanced (AUROC 0.87 vs AUROC 0.71) and imbalanced (AUROC 0.0.90 vs AUROC 0.65) population. The proposed approach was able to improve the accuracy of a rare disease detection model by transfer learning information from a non-manual annotated and abundant label with only limited data available.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000631651900003</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>33607378</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Phospholamban; Deep learning; Transfer learning; ECG analysis; Cardiomyopathy; Genetic heart disease</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Lopes, Ricardo R.; Ramos, Lucas A.; Marquering, Henk A.] Univ Amsterdam, Dept Biomed Engn &amp; Phys, Amsterdam UMC, Meibergdreef 9, NL-1105 AZ Amsterdam, Netherlands. <br>
[Lopes, Ricardo R.; Marquering, Henk A.] Univ Amsterdam, Dept Radiol &amp; Nucl Med, Amsterdam UMC, Amsterdam, Netherlands. <br>
[Bleijendaal, Hidde; Ramos, Lucas A.] Univ Amsterdam, Dept Clin Epidemiol Biostat &amp; Bioinformat, Amsterdam UMC, Amsterdam, Netherlands. <br>
[Bleijendaal, Hidde; Verstraelen, Tom E.; Amin, Ahmad S.; Wilde, Arthur A. M.; Pinto, Yigal M.; de Mol, Bas A. J. M.] Univ Amsterdam, Amsterdam UMC, Dept Clin &amp; Expt Cardiol, Amsterdam Cardiovasc Sci,Heart Ctr, Amsterdam, Netherlands. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Lopes, RR (corresponding author), Univ Amsterdam, Dept Biomed Engn &amp; Phys, Amsterdam UMC, Meibergdreef 9, NL-1105 AZ Amsterdam, Netherlands.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
r.riccilopes@amsterdamumc.nl</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
University of Amsterdam; University of Amsterdam; University of Amsterdam; University of Amsterdam</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Verstraelen, Tom</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-4340-6815&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Bleijendaal, Hidde</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-1506-8385&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Ramos, Lucas</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-7738-9200&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Ramos, Lucas</display_name>&nbsp;</td><td>W-7401-2019&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Verstraelen, Tom</display_name>&nbsp;</td><td>ABE-5949-2020&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Ricci Lopes, Ricardo</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-5515-1488&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>de Mol, Bastian</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-6888-9430&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Marquering, Henk</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-1414-6313&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Amin, Ahmad</display_name>&nbsp;</td><td>NFT-0652-2025&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Marquering, Henk</display_name>&nbsp;</td><td>B-9954-2013&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Lopes, Ricardo</display_name>&nbsp;</td><td>Y-9109-2019&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>PERGAMON-ELSEVIER SCIENCE LTD</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>THE BOULEVARD, LANGFORD LANE, KIDLINGTON, OXFORD OX5 1GB, ENGLAND</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Biology; Computer Science, Interdisciplinary Applications; Engineering, Biomedical; Mathematical &amp; Computational Biology</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Life Sciences &amp; Biomedicine - Other Topics; Computer Science; Engineering; Mathematical &amp; Computational Biology</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>RA8EI</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0010-4825</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1879-0534</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>COMPUT BIOL MED</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Comput. Biol. Med.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>8</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>ITEA3 PARTNER</grant_agency>&nbsp;</td><td>
<div>16017&nbsp;</div>
<div>14003&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>University of Amsterdam Research Priority Area Medical Integromics</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>Part of this work was funded by ITEA3 PARTNER (16017), ITEA3 Medolution (14003) and University of Amsterdam Research Priority Area Medical Integromics.</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>hybrid</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 30 of 49</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>MVKT-ECG: Efficient single-lead ECG classification for multi-label arrhythmia by multi-view knowledge transferring</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Qin, YZ (Qin, Yuzhen); Sun, L (Sun, Li); Chen, H (Chen, Hui); Yang, WM (Yang, Wenming); Zhang, WQ (Zhang, Wei-Qiang); Fei, JT (Fei, Jintao); Wang, GJ (Wang, Guijin)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>COMPUTERS IN BIOLOGY AND MEDICINE</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>166</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>107503</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1016/j.compbiomed.2023.107503</value>&nbsp;&nbsp;<b>Early Access Date:</b> 
<value>OCT 2023</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2023 NOV</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>11</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>11</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
14</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
91</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>48</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Electrocardiogram (ECG) is a widely used technique for diagnosing cardiovascular disease. The widespread emergence of smart ECG devices has sparked the demand for intelligent single-lead ECG-based diagnostic systems. However, it is challenging to develop a single-lead-based ECG interpretation model for multiple disease diagnosis due to the lack of some key disease information. We aim to improve the diagnostic capabilities of single-lead ECG for multi-label disease classification in a new teacher-student manner, where the teacher trained by multi-lead ECG educates a student who observes only single-lead ECG We present a new disease aware Contrastive Lead-information Transferring (CLT) to improve the mutual disease information between the single-lead-based ECG interpretation model and multi-lead-based ECG interpretation model. Moreover, We modify the traditional Knowledge Distillation into Multi-label disease Knowledge Distillation (MKD) to make it applicable for multi-label disease diagnosis. The whole knowledge transferring process is inter-lead Multi -View Knowledge Transferring of ECG (MVKT-ECG). By employing the training strategy, we can effectively transfer comprehensive disease knowledge from various views of ECG, such as the 12-lead ECG, to a single-lead-based ECG interpretation model. This enables the model to extract intricate details from single-lead ECG signals and enhances the model's capability of diagnosing and identifying single-lead signals. Extensive experiments on two commonly used public multi-label datasets, ICBEB2018 and PTB-XL demonstrate that our MVKT-ECG yields exceptional diagnostic performance improvements for single-lead ECG. The student outperforms its baseline observably on the PTB-XL dataset (1.3 % on PTB.super, and 1.4 % on PTB.sub), and on ICBEB2018 dataset (3.2 %).</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001088270400001</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>37806055</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Single-lead Electrocardiogram; Arrhythmia classification; Deep learning; Knowledge transfer; Neural network</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Qin, Yuzhen; Yang, Wenming] Tsinghua Univ, Shenzhen Int Grad Sch, Shenzhen 518071, Peoples R China. <br>
[Sun, Li; Chen, Hui; Zhang, Wei-Qiang; Wang, Guijin] Tsinghua Univ, Dept Elect Engn, Beijing 100084, Peoples R China. <br>
[Fei, Jintao] Beijing Tsinghua Changgung Hosp, Beijing 102218, Peoples R China. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Wang, GJ (corresponding author), Tsinghua Univ, Dept Elect Engn, Beijing 100084, Peoples R China.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
wangguijin@tsinghua.edu.cn</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Tsinghua University; Tsinghua University</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Yang, Wenming</display_name>&nbsp;</td><td>HJH-8634-2023&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>PERGAMON-ELSEVIER SCIENCE LTD</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>THE BOULEVARD, LANGFORD LANE, KIDLINGTON, OXFORD OX5 1GB, ENGLAND</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Biology; Computer Science, Interdisciplinary Applications; Engineering, Biomedical; Mathematical &amp; Computational Biology</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Life Sciences &amp; Biomedicine - Other Topics; Computer Science; Engineering; Mathematical &amp; Computational Biology</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>U9XR8</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0010-4825</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1879-0534</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>COMPUT BIOL MED</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Comput. Biol. Med.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>10</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Natural Science Foun-dation of China</grant_agency>&nbsp;</td><td>
<div>62276153&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This research was supported by the National Natural Science Foun-dation of China under Grant No. 62276153.</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 31 of 49</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Disentangled representation learning in cardiac image analysis</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Chartsias, A (Chartsias, Agisilaos); Joyce, T (Joyce, Thomas); Papanastasiou, G (Papanastasiou, Giorgos); Semple, S (Semple, Scott); Williams, M (Williams, Michelle); Newby, DE (Newby, David E.); Dharmakumar, R (Dharmakumar, Rohan); Tsaftaris, SA (Tsaftaris, Sotirios A.)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>MEDICAL IMAGE ANALYSIS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>58</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>101535</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1016/j.media.2019.101535</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2019 DEC</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>123</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>133</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
1</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
40</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>49</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Typically, a medical image offers spatial information on the anatomy (and pathology) modulated by imaging specific characteristics. Many imaging modalities including Magnetic Resonance Imaging (MRI) and Computed Tomography (CT) can be interpreted in this way. We can venture further and consider that a medical image naturally factors into some spatial factors depicting anatomy and factors that denote the imaging characteristics. Here, we explicitly learn this decomposed (disentangled) representation of imaging data, focusing in particular on cardiac images. We propose Spatial Decomposition Network (SDNet), which factorises 2D medical images into spatial anatomical factors and non-spatial modality factors. We demonstrate that this high-level representation is ideally suited for several medical image analysis tasks, such as semi-supervised segmentation, multi-task segmentation and regression, and image-to-image synthesis. Specifically, we show that our model can match the performance of fully supervised segmentation models, using only a fraction of the labelled images. Critically, we show that our factorised representation also benefits from supervision obtained either when we use auxiliary tasks to train the model in a multi-task setting (e.g. regressing to known cardiac indices), or when aggregating multimodal data from different sources (e.g. pooling together MRI and CT data). To explore the properties of the learned factorisation, we perform latent-space arithmetic and show that we can synthesise CT from MR and vice versa, by swapping the modality factors. We also demonstrate that the factor holding image specific information can be used to predict the input modality with high accuracy. Code will be made available at https://github.comiagis85/anatomy_modality_decomposition. (C) 2019 Elsevier B.V. All rights reserved.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000496605700025</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>31351230</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Disentangled representation learning; Cardiac magnetic resonance imaging; Semi-supervised segmentation; Multitask learning</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
WHOLE HEART SEGMENTATION</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Chartsias, Agisilaos; Joyce, Thomas; Tsaftaris, Sotirios A.] Univ Edinburgh, Inst Digital Commun, Sch Engn, West Mains Rd, Edinburgh EH9 3FB, Midlothian, Scotland. <br>
[Papanastasiou, Giorgos; Semple, Scott; Williams, Michelle; Newby, David E.] Edinburgh Imaging Facil QMRI, Edinburgh EH16 4TJ, Midlothian, Scotland. <br>
[Papanastasiou, Giorgos; Semple, Scott; Williams, Michelle; Newby, David E.] Ctr Cardiovasc Sci, Edinburgh EH16 4TJ, Midlothian, Scotland. <br>
[Dharmakumar, Rohan] Cedars Sinai Med Ctr, Los Angeles, CA 90048 USA. <br>
[Tsaftaris, Sotirios A.] Alan Turing Inst, London, England. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Chartsias, A (corresponding author), Univ Edinburgh, Inst Digital Commun, Sch Engn, West Mains Rd, Edinburgh EH9 3FB, Midlothian, Scotland.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
agis.chartsias@ed.ac.uk</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
University of Edinburgh; Cedars Sinai Medical Center; Alan Turing Institute</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Newby, David</display_name>&nbsp;</td><td>AAB-1364-2019&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Papanastasiou, Giorgos</display_name>&nbsp;</td><td>ABH-8943-2020&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Williams, Michelle</display_name>&nbsp;</td><td>K-7555-2012&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Newby, David</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-7971-4628&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Tsaftaris, Sotirios</display_name>&nbsp;</td><td>E-3725-2010&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Papanastasiou, Giorgos</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-1939-296X&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Joyce, Thomas</display_name>&nbsp;</td><td>IWU-5793-2023&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>ELSEVIER</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>RADARWEG 29, 1043 NX AMSTERDAM, NETHERLANDS</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Artificial Intelligence; Computer Science, Interdisciplinary Applications; Engineering, Biomedical; Radiology, Nuclear Medicine &amp; Medical Imaging</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Engineering; Radiology, Nuclear Medicine &amp; Medical Imaging</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>JN0PE</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>1361-8415</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1361-8423</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>MED IMAGE ANAL</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Med. Image Anal.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>13</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>US National Institutes of Health</grant_agency>&nbsp;</td><td>
<div>1R01HL136578-01&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>UK EPSRC</grant_agency>&nbsp;</td><td>
<div>EP/P022928/1&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Royal Academy of Engineering</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>EPSRC</grant_agency>&nbsp;</td><td>
<div>EP/P022928/1&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>MRC</grant_agency>&nbsp;</td><td>
<div>G0701127&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>British Heart Foundation</grant_agency>&nbsp;</td><td>
<div>CH/09/002/26360&nbsp;</div>
<div>RG/16/10/32375&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Engineering and Physical Sciences Research Council</grant_agency>&nbsp;</td><td>
<div>EP/P022928/1&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Medical Research Council</grant_agency>&nbsp;</td><td>
<div>G0701127&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported in part by the US National Institutes of Health (1R01HL136578-01) and UK EPSRC (EP/P022928/1), and used resources provided by the Edinburgh Compute and Data Facility (http://www.ecdf.ed.ac.uk/). S.A.Tsaftaris acknowledges the support of the Royal Academy of Engineering and the Research Chairs and Senior Research Fellowships scheme.</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>Green Submitted, Green Accepted</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 32 of 49</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Acoustic feature based unsupervised approach of heart sound event detection</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Das, S (Das, Sangita); Pal, S (Pal, Saurabh); Mitra, M (Mitra, Madhuchhanda)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>COMPUTERS IN BIOLOGY AND MEDICINE</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>126</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>103990</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1016/j.compbiomed.2020.103990</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2020 NOV</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>28</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>29</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
3</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
56</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>46</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
This paper represents an unsupervised approach to detect the positions of S1, S2 heart sound events in a Phonocardiogram (PCG) recording. Insufficiency of correctly annotated heart sound database drives us to investigate unsupervised techniques. Gammatone filter bank features are used to characterize the spectral pattern of fundamental heart sound events from noise contaminated PCG data. An unsupervised spectral clustering technique is employed for segmentation of S1/S2 and non-S1/S2 heart sound events. A Feature winning score is computed to identify the S1/52 and non-S1/S2 frames. Finally, time based threshold is applied to detect the accurate positions of S1 and S2 heart sounds. The performance of spectral clustering is compared with other clustering methods. The proposed method offers a maximum Fl-score of 98% and 92.5% for normal and abnormal PCG data respectively on 2016 PhysioNet/CinC challenge dataset. The heart sound annotation algorithm provided by PhysioNet has been used as the ground truth after hand correction.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000582723600001</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>32987200</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Heart sound; Phonocardiogram (PCG); Spectral clustering; Heart sound segmentation</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
NEURAL-NETWORK CLASSIFICATION; SEGMENTATION; RECOGNITION; NOISE; IDENTIFICATION; SEPARATION; IMAGE</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Das, Sangita; Pal, Saurabh; Mitra, Madhuchhanda] Univ Calcutta, Dept Appl Phys, 92APC Rd, Kolkata 700009, W Bengal, India. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Mitra, M (corresponding author), Univ Calcutta, Dept Appl Phys, 92APC Rd, Kolkata 700009, W Bengal, India.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
mmaphy@caluniv.ac.in</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
University of Calcutta</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Pal, Saurabh</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-5139-6573&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>PERGAMON-ELSEVIER SCIENCE LTD</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>THE BOULEVARD, LANGFORD LANE, KIDLINGTON, OXFORD OX5 1GB, ENGLAND</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Biology; Computer Science, Interdisciplinary Applications; Engineering, Biomedical; Mathematical &amp; Computational Biology</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Life Sciences &amp; Biomedicine - Other Topics; Computer Science; Engineering; Mathematical &amp; Computational Biology</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>OH6RT</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0010-4825</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1879-0534</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>COMPUT BIOL MED</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Comput. Biol. Med.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>11</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Council of Scientific and Industrial Research (CSIR)</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>The authors thank Dr. Santanu Guha, Professor and Head, Department of Cardiology, Medical College, Kolkata for his supports and valuable suggestions. We gratefully acknowledge Council of Scientific and Industrial Research (CSIR) for their financial support.</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 33 of 49</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>A probabilistic deep motion model for unsupervised cardiac shape anomaly assessment</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Zakeri, A (Zakeri, Arezoo); Hokmabadi, A (Hokmabadi, Alireza); Ravikumar, N (Ravikumar, Nishant); Frangi, AF (Frangi, Alejandro F.); Gooya, A (Gooya, Ali)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>MEDICAL IMAGE ANALYSIS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>75</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>102276</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1016/j.media.2021.102276</value>&nbsp;&nbsp;<b>Early Access Date:</b> 
<value>NOV 2021</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2022 JAN</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>8</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>8</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
2</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
10</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>54</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Automatic shape anomaly detection in large-scale imaging data can be useful for screening suboptimal segmentations and pathologies altering the cardiac morphology without intensive manual labour. We propose a deep probabilistic model for local anomaly detection in sequences of heart shapes, modelled as point sets, in a cardiac cycle. A deep recurrent encoder-decoder network captures the spatio-temporal dependencies to predict the next shape in the cycle and thus derive the outlier points that are attributed to excessive deviations from the network prediction. A predictive mixture distribution models the inlier and outlier classes via Gaussian and uniform distributions, respectively. A Gibbs sampling Expectation Maximisation (EM) algorithm computes soft anomaly scores of the points via the posterior probabilities of each class in the E-step and estimates the parameters of the network and the predictive distribution in the M-step. We demonstrate the versatility of the method using two shape datasets derived from: (i) one million biventricular CMR images from 20,0 0 0 participants in the UK Biobank (UKB), and (ii) routine diagnostic imaging from Multi-Centre, Multi-Vendor, and Multi-Disease Cardiac Image (M&amp;Ms). Experiments show that the detected shape anomalies in the UKB dataset are mostly associated with poor segmentation quality, and the predicted shape sequences show significant improvement over the input sequences. Furthermore, evaluations on U-Net based shapes from the M&amp;Ms dataset reveals that the anomalies are attributable to the underlying pathologies that affect the ventricles. The proposed model can therefore be used as an effective mechanism to sift shape anomalies in large-scale cardiac imaging pipelines for further analysis. (c) 2021 Elsevier B.V. All rights reserved.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000718407400014</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>34753021</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Cardiac shape quality control; Deep learning; Expectation-maximisation; Probabilistic model; Spatio-temporal anomaly detection; UK Biobank</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
MR-IMAGES; ATLAS; ABNORMALITIES</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Zakeri, Arezoo; Hokmabadi, Alireza; Ravikumar, Nishant; Frangi, Alejandro F.; Gooya, Ali] Univ Leeds, Ctr Computat Imaging &amp; Simulat Technol Biomed CIS, Sch Comp, Leeds, W Yorkshire, England. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Zakeri, A; Gooya, A (corresponding author), Univ Leeds, Ctr Computat Imaging &amp; Simulat Technol Biomed CIS, Sch Comp, Leeds, W Yorkshire, England.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
a.zakeri@leeds.ac.uk; a.gooya@leeds.ac.uk</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
University of Leeds</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Hokmabadi, Alireza</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-1407-4540&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Gooya, Ali</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-5135-4800&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Frangi, Alejandro</display_name>&nbsp;</td><td>C-6500-2008&nbsp;</td><td>0000-0002-2675-528X&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Zakeri, Arezoo</display_name>&nbsp;</td><td>HKV-6795-2023&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Ravikumar, Nishant</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-0134-107X&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Zakeri, Arezoo</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-6011-2333&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>ELSEVIER</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>RADARWEG 29, 1043 NX AMSTERDAM, NETHERLANDS</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Artificial Intelligence; Computer Science, Interdisciplinary Applications; Engineering, Biomedical; Radiology, Nuclear Medicine &amp; Medical Imaging</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Engineering; Radiology, Nuclear Medicine &amp; Medical Imaging</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>WX2BN</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>1361-8415</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1361-8423</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>MED IMAGE ANAL</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Med. Image Anal.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>15</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Engineering and Physical Sciences Research Council (EPSRC)</grant_agency>&nbsp;</td><td>
<div>EP/S012796/1&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Royal Academy of Engineering Chair in Emerging Technologies Scheme</grant_agency>&nbsp;</td><td>
<div>CiET1819/19&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>CardioX</grant_agency>&nbsp;</td><td>
<div>GrowMedTech POC041&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>EPSRC</grant_agency>&nbsp;</td><td>
<div>EP/S012796/1&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Medical Research Council</grant_agency>&nbsp;</td><td>
<div>MC_PC_17228&nbsp;</div>
<div>MC_qA137853&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported by the Engineering and Physical Sciences Research Council (EPSRC) (grant number EP/S012796/1). AFF is partially supported by a Royal Academy of Engineering Chair in Emerging Technologies Scheme (CiET1819/19) and CardioX (GrowMedTech POC041). The authors would like to thank Dr. Rahman Attar for making the UKB shape dataset available. The UKB CMR dataset and the ground-truth contours have been provided under UK Biobank Access Application #11350. The authors thank all UK Biobank participants and staff.</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 34 of 49</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Assessing the signal quality of electrocardiograms from varied acquisition sources: A generic machine learning pipeline for model generation</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Albaba, A (Albaba, Adnan); Simo, N (Simo, Neide); Wang, YY (Wang, Yuyang); Hendriks, RC (Hendriks, Richard C.); De Raedt, W (De Raedt, Walter); Van Hoof, C (Van Hoof, Chris)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>COMPUTERS IN BIOLOGY AND MEDICINE</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>130</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>104164</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1016/j.compbiomed.2020.104164</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2021 MAR</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>17</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>17</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
0</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
16</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>43</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Background and objective: Long-term electrocardiogram monitoring comes at the expense of signal quality. During unconstrained movements, the electrocardiogram is often corrupted by motion artefacts, which can lead to inaccurate physiological information. In this situation, automated quality assessment methods are useful to increase the reliability of the measurements. A generic machine learning pipeline that generates classification models for electrocardiogram quality assessment is presented in this article. The presented pipeline is tested on signals from varied acquisition sources, towards selecting segments that can be used for heart rate analysis in lifestyle applications.
<br>Methods: Electrocardiogram recordings from traditional, wearable and ubiquitous devices, are segmented in 10 s windows and manually labeled by experienced researchers into two quality classes. To capture the electrocardiogram dynamics, a comprehensive set of 43 features is extracted from each segment, based on the time-domain signal, its Fast Fourier Transform, the Autocorrelation function and the Stationary Wavelet Transform. To select the most relevant features for each acquisition source we employ both a customized hybrid approach and the state-of-the-art Neighborhood Component Analysis method and compare them. Support Vector Machines (SVM), Decision Trees, K-Nearest-Neighbors and supervised ensemble methods are tested as possible binary classifiers.
<br>Results: The results for the best performing models on traditional, wearable and ubiquitous electrocardiogram datasets are, respectively: balanced-accuracy: 89%, F1-score: 93% with the Fine Gaussian SVM model and 10 features; balanced-accuracy: 93%, F1-score: 93% with the Fine Gaussian SVM model and 11 features; balanced accuracy: 95%, F1-score: 86%, with the Fine Gaussian SVM model and 8 features.
<br>Conclusions: According to the results, our generic pipeline can generate classification models tailored to individual acquisition sources, provided that a standard Lead I or Lead II is available. Such models accurately establish whether the electrocardiogram quality is good or bad for heart rate analysis. Furthermore, removing bad quality segments decreases errors in heart rate calculation.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000623917000009</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>33360108</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Electrocardiogram; Wearables; Ubiquitous; Non-contact; Classification; Feature selection; Motion artefact; Signal quality</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
QRS DETECTION; ECG</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Albaba, Adnan; Simo, Neide; Van Hoof, Chris] Univ Leuven, Dept Elect Engn, Leuven, Belgium. <br>
[Albaba, Adnan; Simo, Neide; De Raedt, Walter; Van Hoof, Chris] IMEC, Connected Hlth Solut Grp, Leuven, Belgium. <br>
[Wang, Yuyang; Hendriks, Richard C.] Delft Univ Technol, Dept Elect Engn, Delft, Netherlands. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
De Raedt, W (corresponding author), IMEC, Connected Hlth Solut Grp, Leuven, Belgium.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
adnan.albaba@imec.be; neide.simoescapela@kuleuven.be; y.wang-54@student.tudelft.nl; r.c.hendriks@tudelft.nl; walter.deraedt@imec.be; chris.vanhoof@imec.be</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
KU Leuven; Interuniversity Microelectronics Centre; Delft University of Technology</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>De Raedt, Walter</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-7117-7976&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Van Hoof, Chris</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-4645-3326&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Albaba, Adnan</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-9945-2650&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Simoes-Capela, Neide</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-0550-4964&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Albaba, Adnan</display_name>&nbsp;</td><td>MAH-3139-2025&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>PERGAMON-ELSEVIER SCIENCE LTD</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>THE BOULEVARD, LANGFORD LANE, KIDLINGTON, OXFORD OX5 1GB, ENGLAND</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Biology; Computer Science, Interdisciplinary Applications; Engineering, Biomedical; Mathematical &amp; Computational Biology</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Life Sciences &amp; Biomedicine - Other Topics; Computer Science; Engineering; Mathematical &amp; Computational Biology</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>QP6AG</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0010-4825</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1879-0534</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>COMPUT BIOL MED</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Comput. Biol. Med.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>11</value>
</td>
</tr>

<tr>
<td>
<b>Open Access:</b>

<value>hybrid</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 35 of 49</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Learning-Based Regularization for Cardiac Strain Analysis via Domain Adaptation</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Lu, AE (Lu, Allen); Ahn, SS (Ahn, Shawn S.); Ta, K (Ta, Kevinminh); Parajuli, N (Parajuli, Nripesh); Stendahl, JC (Stendahl, John C.); Liu, Z (Liu, Zhao); Boutagy, NE (Boutagy, Nabil E.); Jeng, GS (Jeng, Geng-Shi); Staib, LH (Staib, Lawrence H.); O'Donnell, M (O'Donnell, Matthew); Sinusas, AJ (Sinusas, Albert J.); Duncan, JS (Duncan, James S.)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE TRANSACTIONS ON MEDICAL IMAGING</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>40</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>9</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>2233-2245</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/TMI.2021.3074033</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2021 SEP</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>14</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>15</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
0</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
15</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>48</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Reliable motion estimation and strain analysis using 3D+ time echocardiography (4DE) for localization and characterization of myocardial injury is valuable for early detection and targeted interventions. However, motion estimation is difficult due to the low-SNR that stems from the inherent image properties of 4DE, and intelligent regularization is critical for producing reliable motion estimates. In this work, we incorporated the notion of domain adaptation into a supervised neural network regularization framework. We first propose a semi-supervised Multi-Layered Perceptron (MLP) network with biomechanical constraints for learning a latent representation that is shown to have more physiologically plausible displacements. We extended this framework to include a supervised loss term on synthetic data and showed the effects of biomechanical constraints on the network's ability for domain adaptation. We validated the semi-supervised regularization method on in vivo data with implanted sonomicrometers. Finally, we showed the ability of our semi-supervised learning regularization approach to identify infarct regions using estimated regional strain maps with good agreement to manually traced infarct regions from postmortem excised hearts.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000692208500005</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>33872145</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Strain; Adaptation models; Myocardium; Finite element analysis; Feature extraction; Estimation; Data models; Cardiac function; echocardiography; motion analysis; machine learning</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
SPECKLE TRACKING; NONRIGID REGISTRATION; DEFORMATION; IMAGES; MOTION</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Lu, Allen; Ahn, Shawn S.; Ta, Kevinminh; Staib, Lawrence H.; Duncan, James S.] Yale Univ, Dept Biomed Engn, New Haven, CT 06511 USA. <br>
[Parajuli, Nripesh] Capt Hlth, San Francisco, CA 94132 USA. <br>
[Stendahl, John C.; Boutagy, Nabil E.; Sinusas, Albert J.] Yale Univ, Sect Cardiovasc Med, Dept Internal Med, New Haven, CT 06511 USA. <br>
[Liu, Zhao; Staib, Lawrence H.; Sinusas, Albert J.; Duncan, James S.] Yale Univ, Dept Radiol &amp; Biomed Imaging, New Haven, CT 06511 USA. <br>
[Jeng, Geng-Shi] Natl Yang Ming Chiao Tung Univ, Inst Elect, Hsinchu 30010, Taiwan. <br>
[Staib, Lawrence H.; Duncan, James S.] Yale Univ, Dept Elect Engn, New Haven, CT 06511 USA. <br>
[O'Donnell, Matthew] Univ Washington, Dept Bioengn, Seattle, WA 98015 USA. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Ahn, SS (corresponding author), Yale Univ, Dept Biomed Engn, New Haven, CT 06511 USA.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
haiyinglu.allen@gmail.com; shawn.ahn@yale.edu; kevinminh.ta@yale.edu; nripesh.parajuli.21@gmail.com; john.stendahl@yale.edu; zhao.liu@yale.edu; nabil.boutagy@yale.edu; jeng@nctu.edu.tw; lawrence.staib@yale.edu; odonnel@uw.edu; albert.sinusas@yale.edu; james.duncan@yale.edu</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Yale University; Yale University; Yale University; National Yang Ming Chiao Tung University; Yale University; University of Washington; University of Washington Seattle</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Ahn, Shawn</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-5961-3376&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Staib, Lawrence</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-9516-5136&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Sinusas, Albert</display_name>&nbsp;</td><td>A-7235-2009&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>, Geng-Shi</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-4105-573X&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Stendahl, John</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-1568-9280&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Interdisciplinary Applications; Engineering, Biomedical; Engineering, Electrical &amp; Electronic; Imaging Science &amp; Photographic Technology; Radiology, Nuclear Medicine &amp; Medical Imaging</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Engineering; Imaging Science &amp; Photographic Technology; Radiology, Nuclear Medicine &amp; Medical Imaging</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>UK8IN</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0278-0062</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1558-254X</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE T MED IMAGING</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE Trans. Med. Imaging</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>13</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Institute of Health</grant_agency>&nbsp;</td><td>
<div>R01HL121226&nbsp;</div>
<div>T32HL098069&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Institute of Health Medical Scientist Training Program</grant_agency>&nbsp;</td><td>
<div>T32GM007205&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Heart Lung and Blood Institute</grant_agency>&nbsp;</td><td>
<div>R01HL121226&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Institute of General Medical Sciences</grant_agency>&nbsp;</td><td>
<div>T32GM136651&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported in part by the National Institute of Health under Grant R01HL121226 and Grant T32HL098069 and in part by the National Institute of Health Medical Scientist Training Program under Grant T32GM007205.</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>hybrid, Green Accepted</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 36 of 49</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Mechanosensation mediates volume adaptation of cardiac cells and spheroids in 3D</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Chin, IL (Chin, Ian L.); Amos, SE (Amos, Sebastian E.); Jeong, JH (Jeong, Ji Hoon); Hool, L (Hool, Livia); Hwang, Y (Hwang, Yongsung); Choi, YS (Choi, Yu Suk)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>MATERIALS TODAY BIO</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>16</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>100391</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1016/j.mtbio.2022.100391</value>&nbsp;&nbsp;<b>Early Access Date:</b> 
<value>AUG 2022</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2022 DEC</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>10</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>11</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
1</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
16</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>40</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
With the adoption of 3-dimensional (3D) cell culture for in vitro modelling of cardiac function and regenerative medicine applications, there is an increased need to understand cardiomyocyte mechanosensation in 3D. With existing studies of cardiomyocyte mechanosensation primarily focussed on the behaviour of individual cells in a 2-Dimensional context, it is unclear whether mechanosensation is the same in a 3D, multicellular context. In this study, H9C2 cardiac-derived myoblasts were encapsulated as individual cells and as cell spheroids within stiffness gradient gelatin methacryloyl (GelMA) hydrogels to investigate individual and collective cardiac cell mechano-sensation in 3D. Over a 3.68-17.52 kPa stiffness range, it was found that H9C2 cells have a limited capacity to adapt their volume to increasing substrate stiffness, demonstrated by the lack of changes in cell volume and shape across the stiffness gradient. Morphological trends were reflected by the expression of the mechanomarkers YAP, MRTF-A and Lamin-A, which were better correlated with cell and nuclear volume than with substrate stiffness. The localisation of YAP and MRTF-A were dependent on the relative volumes of the cytoplasm and nucleus while Lamin-A expression was elevated with increasing cytoplasmic and nuclear volumes. When cultured as spheroids rather than as individual cells, H9C2 cells adopted a distinct morphology with comparably smaller nuclei than individually cultured cells, while retaining the same overall cell volume. As spheroids, H9C2 cells were sensitive to stiffness cues, shown by decreasing YAP and MRTF-A nuclear localisation, increasing Lamin-A expression, and increasing vinculin expression with increasing substrate stiffness. Like the individually cultured H9C2 cells, mechanomarker expression was correlated to volume adaptation. With increasing cytoplasmic volume, YAP and MRTF-A became less nuclear localised, vinculin expression was increased, and with increasing nuclear volume, the Lamin-A expression fincreased. Together, these data suggest that cardiac cell volume adaptation may be enhanced by cell-cell interactions.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000848024700007</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>36042852</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Mechanobiology; Heart disease; Myocardial infarction; Fibrosis; Cell biomechanics</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
MYOCARDIAL-INFARCTION; TRANSCRIPTION FACTORS; MECHANICS; MATURATION; YAP/TAZ; LEADS</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Chin, Ian L.; Amos, Sebastian E.; Hool, Livia; Choi, Yu Suk] Univ Western Australia, Sch Human Sci, Perth, WA, Australia. <br>
[Hool, Livia] Victor Chang Cardiac Res Inst, Sydney, NSW, Australia. <br>
[Jeong, Ji Hoon; Hwang, Yongsung] Soonchunhyang Univ, Soonchunhyang Inst Medi bio Sci SIMS, Cheonan Si 31151, Chungnam Do, South Korea. <br>
[Jeong, Ji Hoon; Hwang, Yongsung] Soonchunhyang Univ, Dept Integrated Biomed Sci, Asan 31538, Chungnam Do, South Korea. <br>
[Choi, Yu Suk] Univ Western Australia, Sch Human Sci, 35 Stirling Highway, Crawley, WA 6009, Australia. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Choi, YS (corresponding author), Univ Western Australia, Sch Human Sci, 35 Stirling Highway, Crawley, WA 6009, Australia.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
yusuk.choi@uwa.edu.au</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
University of Western Australia; Victor Chang Cardiac Research Institute; Soonchunhyang University; Soonchunhyang University; University of Western Australia</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Chin, Ian</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-5834-6168&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Choi, Yu Suk</display_name>&nbsp;</td><td>J-1532-2014&nbsp;</td><td>0000-0002-0241-0447&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Choi, Yu</display_name>&nbsp;</td><td>J-1532-2014&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Hool, Livia</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-7758-5252&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Amos, Sebastian Elliot</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-1324-4533&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Hool, Livia</display_name>&nbsp;</td><td>H-5059-2014&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>ELSEVIER</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>RADARWEG 29, 1043 NX AMSTERDAM, NETHERLANDS</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Engineering, Biomedical; Materials Science, Biomaterials</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Engineering; Materials Science</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>4E7TO</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2590-0064</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>MATER TODAY BIO</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Mater. Today Bio</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>12</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Health and Medical Research Council (NHMRC)</grant_agency>&nbsp;</td><td>
<div>1098449&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Heart Foundation Future Leader Fellowship</grant_agency>&nbsp;</td><td>
<div>101173&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Future Health Research and Innovation Fund WANMA 2021</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>NHMRC Senior Research Fellowship</grant_agency>&nbsp;</td><td>
<div>APP1117366&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Australian Government Research Training Program Scholarship</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Hackett Postgraduate Research Scholarship</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>UWA fellowship support</grant_agency>&nbsp;</td><td>
<div>2018/RA/1/1997/70&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported National Health and Medical Research Council (NHMRC) Project Grant 1098449 (to YS.C) , Heart Foundation Future Leader Fellowship 101173 (to YS.C) , UWA fellowship support 2018/RA/1/1997/70 (to YS.C) , Future Health Research and Innovation Fund WANMA 2021 (to YS.C) , NHMRC Senior Research Fellowship APP1117366 (to L.C.H) , Australian Government Research Training Program Scholarship (to I.L.C and to S.E.A) and Hackett Postgraduate Research Scholarship (to S.E.A) .</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>Green Published, gold</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 37 of 49</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Non-Rigid Respiratory Motion Estimation of Whole-Heart Coronary MR Images Using Unsupervised Deep Learning</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Qi, HK (Qi, Haikun); Fuin, N (Fuin, Niccolo); Cruz, G (Cruz, Gastao); Pan, JZ (Pan, Jiazhen); Kuestner, T (Kuestner, Thomas); Bustin, A (Bustin, Aurelien); Botnar, RM (Botnar, Rene M.); Prieto, C (Prieto, Claudia)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE TRANSACTIONS ON MEDICAL IMAGING</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>40</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>1</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>444-454</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/TMI.2020.3029205</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2021 JAN</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>36</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>38</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
5</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
31</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>52</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Non-rigid motion-corrected reconstruction has been proposed to account for the complex motion of the heart in free-breathing 3D coronary magnetic resonance angiography (CMRA). This reconstruction framework requires efficient and accurate estimation of non-rigid motion fields from undersampled images at different respiratory positions (or bins). However, state-of-the-art registration methods can be time-consuming. This article presents a novel unsupervised deep learning-based strategy for fast estimation of inter-bin 3D non-rigid respiratory motion fields for motion-corrected free-breathing CMRA. The proposed 3D respiratory motion estimation network (RespME-net) is trained as a deep encoder-decoder network, taking pairs of 3D image patches extracted from CMRA volumes as input and outputting the motion field between image patches. Using image warping by the estimated motion field, a loss function that imposes image similarity and motion smoothness is adopted to enable training without ground truth motion field. RespME-net is trained patch-wise to circumvent the challenges of training a 3D network volume-wise which requires large amounts of GPU memory and 3D datasets. We perform 5-fold cross-validation with 45 CMRA datasets and demonstrate that RespME-net can predict 3D non-rigid motion fields with subpixel accuracy (0.44 +/- 0.38 mm) within similar to 10 seconds, being similar to 20 times faster than a GPU-implemented state-of-the-art non-rigid registration method. Moreover, we perform non-rigid motion-compensated CMRA reconstruction for 9 additional patients. The proposed RespME-net has achieved similar motion-corrected CMRA image quality to the conventional registration method regarding coronary artery length and sharpness.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000604883800038</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>33021937</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Three-dimensional displays; Motion estimation; Image reconstruction; Two dimensional displays; Biomedical imaging; Image registration; Optimization; Motion estimation; deep learning; motion-compensated reconstruction; coronary magnetic resonance angiography</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
MAGNETIC-RESONANCE; OPTICAL-FLOW; QUANTITATIVE-ANALYSIS; REGISTRATION; DEFORMATION; ANGIOGRAPHY; ALIGNMENT; MODELS</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Qi, Haikun; Fuin, Niccolo; Cruz, Gastao; Pan, Jiazhen; Kuestner, Thomas; Bustin, Aurelien; Botnar, Rene M.; Prieto, Claudia] Kings Coll London, Sch Biomed Engn &amp; Imaging Sci, London SE1 7EH, England. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Qi, HK (corresponding author), Kings Coll London, Sch Biomed Engn &amp; Imaging Sci, London SE1 7EH, England.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
haikun.qi@kcl.ac.uk; niccolo.fuin@kcl.ac.uk; gastao.cruz@kcl.ac.uk; jiazhen.pan@gmx.de; thomas.kuestner@kcl.ac.uk; aurelien.bustin@kcl.ac.uk; rene.botnar@kcl.ac.uk; claudia.prieto@kcl.ac.uk</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
University of London; King's College London</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Qi, Haikun</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-4709-5185&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Bustin, Aurelien</display_name>&nbsp;</td><td>AAC-9237-2020&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Kuestner, Thomas</display_name>&nbsp;</td><td>ABE-7866-2020&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Botnar, Rene</display_name>&nbsp;</td><td>AAZ-2112-2020&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Cruz, Gastao</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-7397-9104&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Prieto, Claudia</display_name>&nbsp;</td><td>F-8308-2013&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Botnar, Rene</display_name>&nbsp;</td><td>E-6875-2012&nbsp;</td><td>0000-0003-2811-2509&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Cruz, Gastao</display_name>&nbsp;</td><td>KSM-1438-2024&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Interdisciplinary Applications; Engineering, Biomedical; Engineering, Electrical &amp; Electronic; Imaging Science &amp; Photographic Technology; Radiology, Nuclear Medicine &amp; Medical Imaging</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Engineering; Imaging Science &amp; Photographic Technology; Radiology, Nuclear Medicine &amp; Medical Imaging</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>PO0TG</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0278-0062</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1558-254X</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE T MED IMAGING</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE Trans. Med. Imaging</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>11</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Engineering and Physical Sciences Research Council (EPSRC)</grant_agency>&nbsp;</td><td>
<div>EP/P032311/1&nbsp;</div>
<div>EP/P001009/1&nbsp;</div>
<div>EP/P007619/1&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>British Heart Foundation (BHF)</grant_agency>&nbsp;</td><td>
<div>PG/18/59/33955&nbsp;</div>
<div>RG/20/1/34802&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>King's BHF Centre of Research Excellence</grant_agency>&nbsp;</td><td>
<div>RE/18/2/34213&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Wellcome EPSRC Centre for Medical Engineering</grant_agency>&nbsp;</td><td>
<div>NS/A000049/1&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Department of Health via the National Institute for Health Research (NIHR) Cardiovascular Health Technology Cooperative (HTC) and Comprehensive Biomedical Research Centre</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>King's College London</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>King's College Hospital NHS Foundation Trust</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Engineering and Physical Sciences Research Council</grant_agency>&nbsp;</td><td>
<div>EP/P001009/1&nbsp;</div>
<div>EP/P007619/1&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>EPSRC</grant_agency>&nbsp;</td><td>
<div>EP/P032311/1&nbsp;</div>
<div>EP/P001009/1&nbsp;</div>
<div>EP/P007619/1&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>Thiswork was supported in part by the Engineering and Physical Sciences Research Council (EPSRC) under Grant EP/P032311/1, Grant EP/P001009/1, and Grant EP/P007619/1; in part by the British Heart Foundation (BHF) under Grant PG/18/59/33955 and Grant RG/20/1/34802; in part by the King's BHF Centre of Research Excellence under Grant RE/18/2/34213; in part by theWellcome EPSRC Centre for Medical Engineering under Grant NS/A000049/1; and in part by the Department of Health via the National Institute for Health Research (NIHR) Cardiovascular Health Technology Cooperative (HTC) and Comprehensive Biomedical Research Centre awarded to the Guy's and St Thomas' NHS Foundation Trust in partnership with the King's College London and the King's College Hospital NHS Foundation Trust.</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 38 of 49</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Robust Optical Flow Estimation in Cardiac Ultrasound Images Using a Sparse Representation</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Ouzir, N (Ouzir, Nora); Basarab, A (Basarab, Adrian); Lairez, O (Lairez, Olivier); Tourneret, JY (Tourneret, Jean-Yves)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE TRANSACTIONS ON MEDICAL IMAGING</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>38</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>3</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>741-752</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/TMI.2018.2870947</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2019 MAR</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>25</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>26</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
1</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
24</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>45</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
This paper introduces a robust 2-D cardiac motion estimation method. The problem is formulated as an energy minimization with an optical flow-based data fidelity term and two regularization terms imposing spatial smoothness and the sparsity of the motion field in an appropriate cardiac motion dictionary. Robustness to outliers, such as imaging artefacts and anatomical motion boundaries, is introduced using robust weighting functions for the data fidelity term as well as for the spatial and sparse regularizations. The motion fields and the weights are computed jointly using an iteratively re-weighted minimization strategy. The proposed robust approach is evaluated on synthetic data and realistic simulation sequences with available ground-truth by comparing the performance with state-of-the-art algorithms. Finally, the proposed method is validated using two sequences of in vivo images. The obtained results show the interest of the proposed approach for 2-D cardiac ultrasound imaging.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000460662400009</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>30235121</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Cardiac ultrasound; robust motion estimation; optical flow; sparse regularization; dictionary learning</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
MOTION ESTIMATION; B-MODE; TISSUE DOPPLER; NONRIGID REGISTRATION; STRAIN ESTIMATION; ECHOCARDIOGRAPHY; OPTIMIZATION</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Ouzir, Nora; Tourneret, Jean-Yves] Univ Toulouse, Signal &amp; Image Dept, IRIT INP ENSEEIHT Tesa, F-31071 Toulouse, France. <br>
[Basarab, Adrian] Univ Toulouse, CNRS, Signal &amp; Image Dept, IRIT,UMR 5505, F-31062 Toulouse, France. <br>
[Lairez, Olivier] Univ Paul Sabatier, CHU Toulouse, INSERM, UMR 1048,Inst Malad Metab &amp; Cardiovasc, Toulouse, France. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Ouzir, N (corresponding author), Univ Toulouse, Signal &amp; Image Dept, IRIT INP ENSEEIHT Tesa, F-31071 Toulouse, France.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
nora.ouzir@enseeiht.fr; adrian.basarab@irit.fr; jean-yves.tourneret@enseeiht.fr</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Universite Federale Toulouse Midi-Pyrenees (ComUE); Universite de Toulouse; Institut National Polytechnique de Toulouse; Universite Toulouse III - Paul Sabatier; Universite de Toulouse; Universite Federale Toulouse Midi-Pyrenees (ComUE); Universite Toulouse III - Paul Sabatier; Institut National Polytechnique de Toulouse; Universite Toulouse 1 Capitole; Universite de Toulouse - Jean Jaures; Centre National de la Recherche Scientifique (CNRS); CNRS - Institute for Information Sciences &amp; Technologies (INS2I); Universite de Toulouse; Universite Toulouse III - Paul Sabatier; CHU de Toulouse; Institut National de la Sante et de la Recherche Medicale (Inserm)</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Basarab, Adrian</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-5642-7244&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>O, Lairez</display_name>&nbsp;</td><td>B-7152-2016&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Ouzir, Nora</display_name>&nbsp;</td><td>ACS-1544-2022&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Interdisciplinary Applications; Engineering, Biomedical; Engineering, Electrical &amp; Electronic; Imaging Science &amp; Photographic Technology; Radiology, Nuclear Medicine &amp; Medical Imaging</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Engineering; Imaging Science &amp; Photographic Technology; Radiology, Nuclear Medicine &amp; Medical Imaging</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>HO1JV</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0278-0062</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1558-254X</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE T MED IMAGING</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE Trans. Med. Imaging</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>12</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>thematic trimester on image processing of the CIMI Labex, Toulouse, France</grant_agency>&nbsp;</td><td>
<div>ANR-11-LABX-0040-CIMI&nbsp;</div>
<div>ANR-11-IDEX-0002-02&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported by the thematic trimester on image processing of the CIMI Labex, Toulouse, France, underGrant ANR-11-LABX-0040-CIMI within the Program ANR-11-IDEX-0002-02.</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>Green Submitted</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 39 of 49</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Co-attention spatial transformer network for unsupervised motion tracking and cardiac strain analysis in 3D echocardiography</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Ahn, SS (Ahn, Shawn S.); Ta, K (Ta, Kevinminh); Thorn, SL (Thorn, Stephanie L.); Onofrey, JA (Onofrey, John A.); Melvinsdottir, IH (Melvinsdottir, Inga H.); Lee, SP (Lee, Supum); Langdon, J (Langdon, Jonathan); Sinusas, AJ (Sinusas, Albert J.); Duncan, JS (Duncan, James S.)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>MEDICAL IMAGE ANALYSIS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>84</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>102711</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1016/j.media.2022.102711</value>&nbsp;&nbsp;<b>Early Access Date:</b> 
<value>DEC 2022</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2023 FEB</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>13</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>13</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
3</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
18</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>47</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Myocardial ischemia/infarction causes wall-motion abnormalities in the left ventricle. Therefore, reliable motion estimation and strain analysis using 3D+time echocardiography for localization and characterization of myocardial injury is valuable for early detection and targeted interventions. Previous unsupervised cardiac motion tracking methods rely on heavily-weighted regularization functions to smooth out the noisy displacement fields in echocardiography. In this work, we present a Co-Attention Spatial Transformer Network (STN) for improved motion tracking and strain analysis in 3D echocardiography. Co-Attention STN aims to extract inter-frame dependent features between frames to improve the motion tracking in otherwise noisy 3D echocardiography images. We also propose a novel temporal constraint to further regularize the motion field to produce smooth and realistic cardiac displacement paths over time without prior assumptions on cardiac motion. Our experimental results on both synthetic and in vivo 3D echocardiography datasets demonstrate that our Co-Attention STN provides superior performance compared to existing methods. Strain analysis from Co-Attention STNs also correspond well with the matched SPECT perfusion maps, demonstrating the clinical utility for using 3D echocardiography for infarct localization.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000911016400001</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>36525845</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Unsupervised motion tracking; Spatiotemporal attention; Echocardiography</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
GLOBAL LONGITUDINAL STRAIN; SPECKLE-TRACKING; DEFORMATION; HEART; MRI</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Ahn, Shawn S.; Ta, Kevinminh; Onofrey, John A.; Sinusas, Albert J.; Duncan, James S.] Yale Univ, Dept Biomed Engn, New Haven, CT 06520 USA. <br>
[Thorn, Stephanie L.; Melvinsdottir, Inga H.; Lee, Supum; Sinusas, Albert J.] Yale Univ, Dept Internal Med, Sect Cardiovasc Med, New Haven, CT USA. <br>
[Onofrey, John A.; Langdon, Jonathan; Sinusas, Albert J.; Duncan, James S.] Yale Univ, Dept Radiol &amp; Biomed Imaging, New Haven, CT USA. <br>
[Duncan, James S.] Yale Univ, Dept Elect Engn, New Haven, CT USA. <br>
[Duncan, James S.] 300 Cedar St, New Haven, CT 06519 USA. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Ahn, SS; Duncan, JS (corresponding author), Yale Univ, Dept Biomed Engn, New Haven, CT 06520 USA.<br>Duncan, JS (corresponding author), Yale Univ, Dept Radiol &amp; Biomed Imaging, New Haven, CT USA.<br>Duncan, JS (corresponding author), Yale Univ, Dept Elect Engn, New Haven, CT USA.<br>Duncan, JS (corresponding author), 300 Cedar St, New Haven, CT 06519 USA.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
shawn.ahn@yale.edu; james.duncan@yale.edu</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Yale University; Yale University; Yale University; Yale University</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Lee, Supum</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-5017-9032&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Onofrey, John</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-9432-0448&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Ahn, Shawn</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-5961-3376&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Duncan, James</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-5167-9856&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Langdon, Jonathan</display_name>&nbsp;</td><td>JGL-9505-2023&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Onofrey, John</display_name>&nbsp;</td><td>T-9841-2019&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>ELSEVIER</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>RADARWEG 29, 1043 NX AMSTERDAM, NETHERLANDS</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Artificial Intelligence; Computer Science, Interdisciplinary Applications; Engineering, Biomedical; Radiology, Nuclear Medicine &amp; Medical Imaging</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Engineering; Radiology, Nuclear Medicine &amp; Medical Imaging</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>7S8QJ</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>1361-8415</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1361-8423</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>MED IMAGE ANAL</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Med. Image Anal.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>10</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>NIH, USA</grant_agency>&nbsp;</td><td>
<div>R01HL121226&nbsp;</div>
<div>R01HL137365&nbsp;</div>
<div>T32HL098069&nbsp;</div>
<div>S10RR02555&nbsp;</div>
<div>1S10OD028738-01A1&nbsp;</div>
<div>F30HL158154&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>NIH Medical Scientist Training Program</grant_agency>&nbsp;</td><td>
<div>T32GM007205&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>The authors are immensely thankful for the past and present members of the Yale Translational Research Imaging Center (YTRIC) who were involved in the image acquisition process. This research was made possible through the utilization of the Yale Translational Research Imaging Center. This work was supported in part by the NIH, USA grants R01HL121226, R01HL137365, T32HL098069, S10RR02555, 1S10OD028738-01A1, F30HL158154 and NIH Medical Scientist Training Program Grant T32GM007205.</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>Green Submitted</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 40 of 49</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Learning Representations from Heart Sound: A Comparative Study on Shallow and Deep Models</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Qian, K (Qian, Kun); Bao, ZH (Bao, Zhihao); Zhao, ZH (Zhao, Zhonghao); Koike, T (Koike, Tomoya); Dong, FQ (Dong, Fengquan); Schmitt, M (Schmitt, Maximilian); Dong, QX (Dong, Qunxi); Shen, J (Shen, Jian); Jiang, WP (Jiang, Weipeng); Jiang, YJ (Jiang, Yajuan); Dong, B (Dong, Bo); Dai, ZY (Dai, Zhenyu); Hu, B (Hu, Bin); Schuller, BW (Schuller, Bjoern W.); Yamamoto, Y (Yamamoto, Yoshiharu)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>CYBORG AND BIONIC SYSTEMS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>5</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>0075</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.34133/cbsystems.0075</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2024 MAR 4</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>18</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>18</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
5</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
14</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>56</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Leveraging the power of artificial intelligence to facilitate an automatic analysis and monitoring of heart sounds has increasingly attracted tremendous efforts in the past decade. Nevertheless, lacking on standard open-access database made it difficult to maintain a sustainable and comparable research before the first release of the PhysioNet CinC Challenge Dataset. However, inconsistent standards on data collection, annotation, and partition are still restraining a fair and efficient comparison between different works. To this line, we introduced and benchmarked a first version of the Heart Sounds Shenzhen (HSS) corpus. Motivated and inspired by the previous works based on HSS, we redefined the tasks and make comprehensive investigation on shallow and deep models in this study. First, we segmented the heart sound recording into shorter recordings (10 s), which makes it more similar to the human auscultation case. Second, we redefined the classification tasks. Besides using the 3 class categories (normal, moderate, and mild/severe) adopted in HSS, we added a binary classification task in this study, i.e., normal and abnormal. In this work, we provided detailed benchmarks based on both the classic machine learning and the state -of -the -art deep learning technologies, which are reproducible by using open-source toolkits. Last but not least, we analyzed the feature contributions of best performance achieved by the benchmark to make the results more convincing and interpretable.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001235429500001</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>38440319</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
its cutting-edge subset; deep learning (DL); automatic analyz</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
CLASSIFICATION; RECOGNITION; GENERATION; DIAGNOSIS; EFFICIENT</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Qian, Kun; Bao, Zhihao; Zhao, Zhonghao; Dong, Qunxi; Shen, Jian; Hu, Bin] Beijing Inst Technol, Minist Educ, Key Lab Brain Hlth Intelligent Evaluat &amp; Intervent, Beijing 100081, Peoples R China. <br>
[Qian, Kun; Bao, Zhihao; Zhao, Zhonghao; Dong, Qunxi; Shen, Jian; Hu, Bin] Beijing Inst Technol, Sch Med Technol, Beijing 100081, Peoples R China. <br>
[Koike, Tomoya; Yamamoto, Yoshiharu] Univ Tokyo, Grad Sch Educ, Educ Physiol Lab, Tokyo 1130033, Japan. <br>
[Dong, Fengquan; Jiang, Weipeng; Jiang, Yajuan; Dong, Bo] Shenzhen Univ, Dept Cardiol, Gen Hosp, Shenzhen 518055, Peoples R China. <br>
[Schmitt, Maximilian; Schuller, Bjoern W.] Tech Univ Munich, CHI Chair Hlth Informat, D-81675 Munich, Germany. <br>
[Dai, Zhenyu] Wenzhou Med Univ, Dept Cardiovasc, Affiliated Hosp 1, Wenzhou 325000, Peoples R China. <br>
[Schuller, Bjoern W.] Imperial Coll London, GLAM Grp Language Audio &amp; Mus, London SW7 2AZ, England. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Hu, B (corresponding author), Beijing Inst Technol, Minist Educ, Key Lab Brain Hlth Intelligent Evaluat &amp; Intervent, Beijing 100081, Peoples R China.<br>Hu, B (corresponding author), Beijing Inst Technol, Sch Med Technol, Beijing 100081, Peoples R China.<br>Dong, FQ (corresponding author), Shenzhen Univ, Dept Cardiol, Gen Hosp, Shenzhen 518055, Peoples R China.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
fengquan.dong@foxmail.com; bh@bit.edu.cn</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Beijing Institute of Technology; Beijing Institute of Technology; University of Tokyo; Shenzhen University; Technical University of Munich; Wenzhou Medical University; Imperial College London</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>ZHAO, Zhonghao</display_name>&nbsp;</td><td>IRY-9316-2023&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Dai, Zhenyu</display_name>&nbsp;</td><td>LZG-5654-2025&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Hu, Bin</display_name>&nbsp;</td><td>ACD-0145-2022&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Schuller, Bj&ouml;rn</display_name>&nbsp;</td><td>D-3241-2011&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>AMER ASSOC ADVANCEMENT SCIENCE</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>1200 NEW YORK AVE, NW, WASHINGTON, DC 20005 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Engineering, Biomedical; Robotics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Engineering; Robotics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>SO6J5</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>2692-7632</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>CYBORG BIONIC SYST</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Cyborg Bionic Syst.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>12</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Ministry of Science and Technology of the People"s Republic of China</grant_agency>&nbsp;</td><td>
<div>2021ZD0201900&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Natural Science Foundation of China</grant_agency>&nbsp;</td><td>
<div>62227807&nbsp;</div>
<div>62272044&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Beijing Institute of Technology, China</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Natural Science Foundation of Shenzhen University General Hospital</grant_agency>&nbsp;</td><td>
<div>SUGH2018QD013&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Shenzhen Science and Technology Innovation Commission Project</grant_agency>&nbsp;</td><td>
<div>JCYJ20190808120613189&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Ministry of Education, Culture, Sports, Science and Technology (MEXT) , Japan</grant_agency>&nbsp;</td><td>
<div>20H00569&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was partially supported by the Ministry of Science and Technology of the People"s Republic of China with the STI2030-Major Projects (2021ZD0201900) ; the National Natural Science Foundation of China (No. 62227807 and 62272044) ; the Teli Young Fellow Program from the Beijing Institute of Technology, China; the Natural Science Foundation of Shenzhen University General Hospital (No. SUGH2018QD013) , China; the Shenzhen Science and Technology Innovation Commission Project (No. JCYJ20190808120613189) , China; and the Grants-in-Aid for Scientific Research (No. 20H00569) from the Ministry of Education, Culture, Sports, Science and Technology (MEXT) , Japan.</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>Green Submitted, gold</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 41 of 49</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>A Computational Cardiac Model for the Adaptation to Pulmonary Arterial Hypertension in the Rat</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Avazmohammadi, R (Avazmohammadi, Reza); Mendiola, EA (Mendiola, Emilio A.); Soares, JS (Soares, Joao S.); Li, DS (Li, David S.); Chen, ZQ (Chen, Zhiqiang); Merchant, S (Merchant, Samer); Hsu, EW (Hsu, Edward W.); Vanderslice, P (Vanderslice, Peter); Dixon, RAF (Dixon, Richard A. F.); Sacks, MS (Sacks, Michael S.)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>ANNALS OF BIOMEDICAL ENGINEERING</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>47</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>1</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>138-153</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1007/s10439-018-02130-y</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2019 JAN</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>33</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>41</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
0</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
4</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>34</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Pulmonary arterial hypertension (PAH) imposes pressure overload on the right ventricle (RV), leading to RV enlargement via the growth of cardiac myocytes and remodeling of the collagen fiber architecture. The effects of these alterations on the functional behavior of the right ventricular free wall (RVFW) and organ-level cardiac function remain largely unexplored. Computational heart models in the rat (RHMs) of the normal and hypertensive states can be quite valuable in simulating the effects of PAH on cardiac function to gain insights into the pathophysiology of underlying myocardium remodeling. We thus developed high-fidelity biventricular finite element RHMs for the normal and post-PAH hypertensive states using extensive experimental data collected from rat hearts. We then applied the RHM to investigate the transmural nature of RVFW remodeling and its connection to wall stress elevation under PAH. We found a strong correlation between the longitudinally-dominated fiber-level adaptation of the RVFW and the transmural alterations of relevant wall stress components. We further conducted several numerical experiments to gain new insights on how the RV responds both normally and in the post-PAH state. We found that the effect of pressure overload alone on the increased contractility of the RV is comparable to the effects of changes in the RV geometry and stiffness. Furthermore, our RHMs provided fresh perspectives on long-standing questions of the functional role of the interventricular septum in RV function. Specifically, we demonstrated that an inaccurate identification of the mechanical adaptation of the septum can lead to a significant underestimation of RVFW contractility in the post-PAH state. These findings show how integrated experimental-computational models can facilitate a more comprehensive understanding of the cardiac remodeling events during PAH.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000454772800010</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>30264263</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Pulmonary hypertension; In silico biventricular model; Fiber reorientation; Wall stress; Contractility</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
LEFT-VENTRICULAR FUNCTION; MYOCARDIAL FIBROSIS; PRESSURE; CONTRACTILITY</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Avazmohammadi, Reza; Mendiola, Emilio A.; Li, David S.; Sacks, Michael S.] Univ Texas Austin, Willerson Ctr Cardiovasc Modeling &amp; Simulat, Inst Computat Engn &amp; Sci, Austin, TX 78712 USA. <br>
[Avazmohammadi, Reza; Mendiola, Emilio A.; Li, David S.; Sacks, Michael S.] Univ Texas Austin, Dept Biomed Engn, Austin, TX 78712 USA. <br>
[Soares, Joao S.] Virginia Commonwealth Univ, Dept Mech &amp; Nucl Engn, Richmond, VA USA. <br>
[Chen, Zhiqiang; Vanderslice, Peter; Dixon, Richard A. F.] Texas Heart Inst, Dept Mol Cardiol, Houston, TX 77025 USA. <br>
[Merchant, Samer; Hsu, Edward W.] Univ Utah, Dept Biomed Engn, Salt Lake City, UT USA. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Sacks, MS (corresponding author), Univ Texas Austin, Willerson Ctr Cardiovasc Modeling &amp; Simulat, Inst Computat Engn &amp; Sci, Austin, TX 78712 USA.<br>Sacks, MS (corresponding author), Univ Texas Austin, Dept Biomed Engn, Austin, TX 78712 USA.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
msacks@ices.utexas.edu</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
University of Texas System; University of Texas Austin; University of Texas System; University of Texas Austin; Virginia Commonwealth University; Texas Heart Institute; Utah System of Higher Education; University of Utah</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Li, David</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-6489-0191&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Da Silva Soares, Joao Filipe</display_name>&nbsp;</td><td>AAI-1802-2020&nbsp;</td><td>0000-0002-1216-987X&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Dixon, Richard</display_name>&nbsp;</td><td>K-6863-2013&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Sacks, Michael</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-3199-2204&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Soares, Joao</display_name>&nbsp;</td><td>AAI-1802-2020&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Mendiola, Emilio Agustin</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-7774-0811&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>SPRINGER</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>233 SPRING ST, NEW YORK, NY 10013 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Engineering, Biomedical</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Engineering</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>HG2DL</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0090-6964</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1573-9686</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>ANN BIOMED ENG</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Ann. Biomed. Eng.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>16</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>US National Institutes of Health</grant_agency>&nbsp;</td><td>
<div>5F32 HL132543-02&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>U.S. National Institutive of Healthy</grant_agency>&nbsp;</td><td>
<div>T32EB007507&nbsp;</div>
<div>F31HL139113&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>W.A. Moncrief, Jr. SBES endowment</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>American Heart Association</grant_agency>&nbsp;</td><td>
<div>18CDA34110383&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>American Heart Association (AHA)</grant_agency>&nbsp;</td><td>
<div>18CDA34110383&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported by the US National Institutes of Health and American Heart Association awards (Nos. 5F32 HL132543-02 and 18CDA34110383, respectively) to R.A., the U.S. National Institutive of Healthy (Nos. T32EB007507 and F31HL139113) to D.S.L., and the W.A. Moncrief, Jr. SBES endowment to M.S.S.</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 42 of 49</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Heart failure disease prediction and stratification with temporal electronic health records data using patient representation</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Liang, Y (Liang, Ye); Guo, CH (Guo, Chonghui)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>BIOCYBERNETICS AND BIOMEDICAL ENGINEERING</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>43</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>1</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>124-141</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1016/j.bbe.2022.12.008</value>&nbsp;&nbsp;<b>Early Access Date:</b> 
<value>JAN 2023</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2023 JAN-MAR</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>13</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>15</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
4</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
46</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>62</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Accurate early prediction of heart failure and identification of heart failure sub-phenotypes can enable in-time interventions and treatments, assist with policy decisions, and lead to a better understanding of disease pathophysiology in groups of patients. However, decision making more challenging for clinicians since the available data is complex, heterogeneous, temporal, and different in granularity. Even with much data, it is difficult for a cardiologist to pre-judge a patient's heart condition at the next visit by relying on data from only one visit. Moreover, complicated and overloaded information bewilders clinicians, bringing obstacles to the stratification of patients and the mining of disease typical patterns in subgroups. To overcome these issues, this study proposes a novel Patient Representation model based on a temporal Bidirectional neural network with an Attention mechanism deep learning model called tBNA-PR. tBNA-PR effectively models heterogeneous and temporal Electronic Health Records (tEHRs) data from past and future directions to obtain informative patient representation to realize accurate heart failure prediction and reasonable patient stratification. Additionally, this study extracts typical diagnosis and prescriptions for disease patterns exploration and identifies significant features of sub-phenotypes for subgroup explanation in the context of complex clinical settings to provide better quality healthcare services and clinical decision support. This study leverages a real-world dataset MIMIC-III database. We carried out experiments on the prediction of heart failure to investigate tBNA-PR, which obtains prediction accuracy of 0.78, F1-Score of 0.7671, and AUC of 0.7198, showing a certain superiority compared with several state-of-the-art benchmarks. Moreover, we identified three distinct sub-phenotypes in all heart failure patients in the dataset with the clustering method and subgroup analysis. Sub-phenotype I has characteristics of more long-term anticoagulants. This sub-group has more patients who have the thrombotic disease. Sub-phenotype II has features of more patients having kidney disease, pneumonia, urinary tract infection, and coronary heart disease surgery history. Subphenotype III has characteristics of more patients having acidosis, depressive disorder, esophageal reflux, obstructive sleep apnea, and acquired hypothyroidism. Statistical tests show that the features, including age, creatinine, hemoglobin, urea nitrogen, and blood potassium, are significantly different among the three sub-phenotypes and have particular high importance. The resultant findings from this work have practical implications for clin-ical decision support.CO 2022 Nalecz Institute of Biocybernetics and Biomedical Engineering of the Polish Academy of Sciences. Published by Elsevier B.V. All rights reserved.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000919132200001</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Electronic health records; Heart failure; Disease prediction; Patient stratification; Deep learning</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
MACHINE LEARNING APPROACH; RISK PREDICTION; VALIDATION; DIAGNOSIS</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Liang, Ye; Guo, Chonghui] Dalian Univ Technol, Inst Syst Engn, Dalian 116024, Liaoning, Peoples R China. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Guo, CH (corresponding author), Dalian Univ Technol, Inst Syst Engn, Dalian 116024, Liaoning, Peoples R China.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
dlutguo@dlut.edu.cn</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Dalian University of Technology</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Liang, Ye</display_name>&nbsp;</td><td>&nbsp;</td><td>0009-0007-8689-9414&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Liang, Ye</display_name>&nbsp;</td><td>HJY-9540-2023&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>ELSEVIER</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>RADARWEG 29, 1043 NX AMSTERDAM, NETHERLANDS</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Engineering, Biomedical</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Engineering</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>8E7EP</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0208-5216</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>BIOCYBERN BIOMED ENG</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Biocybern. Biomed. Eng.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>18</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Natural Science Foundation of China</grant_agency>&nbsp;</td><td>
<div>71771034&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Fundamental Research Funds for the Central Universities</grant_agency>&nbsp;</td><td>
<div>DUT21YG108&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Dalian High level Talents Innovation Support Plan</grant_agency>&nbsp;</td><td>
<div>2021RD01&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported by the National Natural Science Foundation of China (Grant Nos.71771034) , the Fundamental Research Funds for the Central Universities (DUT21YG108) , and the Dalian High level Talents Innovation Support Plan (No. 2021RD01) .</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 43 of 49</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Reconstruction and completion of high-resolution 3D cardiac shapes using anisotropic CMRI segmentations and continuous implicit neural representations</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Sander, J (Sander, Jorg); de Vos, BD (de Vos, Bob D.); Bruns, S (Bruns, Steffen); Planken, N (Planken, Nils); Viergever, MA (Viergever, Max A.); Leiner, T (Leiner, Tim); Isgum, I (Isgum, Ivana)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>COMPUTERS IN BIOLOGY AND MEDICINE</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>164</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>107266</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1016/j.compbiomed.2023.107266</value>&nbsp;&nbsp;<b>Early Access Date:</b> 
<value>JUL 2023</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2023 SEP</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>7</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>8</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
2</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
16</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>47</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Since the onset of computer-aided diagnosis in medical imaging, voxel-based segmentation has emerged as the primary methodology for automatic analysis of left ventricle (LV) function and morphology in cardiac magnetic resonance images (CMRI). In standard clinical practice, simultaneous multi-slice 2D cine short-axis MR imaging is performed under multiple breath-holds resulting in highly anisotropic 3D images. Furthermore, sparse-view CMRI often lacks whole heart coverage caused by large slice thickness and often suffers from inter -slice misalignment induced by respiratory motion. Therefore, these volumes only provide limited information about the true 3D cardiac anatomy which may hamper highly accurate assessment of functional and anatomical abnormalities. To address this, we propose a method that learns a continuous implicit function representing 3D LV shapes by training an auto-decoder. For training, high-resolution segmentations from cardiac CT angiography are used. The ability of our approach to reconstruct and complete high-resolution shapes from manually or automatically obtained sparse-view cardiac shape information is evaluated by using paired high-and low-resolution CMRI LV segmentations. The results show that the reconstructed LV shapes have an unconstrained subvoxel resolution and appear smooth and plausible in through-plane direction. Furthermore, Bland-Altman analysis reveals that reconstructed high-resolution ventricle volumes are closer to the corresponding reference volumes than reference low-resolution volumes with bias of [limits of agreement] -3.51 [-18.87, 11.85] mL, and 12.96 [-10.01, 35.92] mL respectively. Finally, the results demonstrate that the proposed approach allows recovering missing shape information and can indirectly correct for limited motion-induced artifacts.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001048414600001</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>37494823</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Cardiac MRI; Neural implicit function; Super-resolution; Shape completion; Shape reconstruction</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
HEART</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Sander, Jorg; de Vos, Bob D.; Bruns, Steffen; Isgum, Ivana] Locat Univ Amsterdam, Dept Biomed Engn &amp; Phys, Amsterdam Univ Med Ctr, Amsterdam, Netherlands. <br>
[Sander, Jorg; Isgum, Ivana] Amsterdam Cardiovasc Sci, Amsterdam, Netherlands. <br>
[Planken, Nils; Isgum, Ivana] Locat Univ Amsterdam, Dept Radiol &amp; Nucl Med, Amsterdam Univ Med Ctr, Amsterdam, Netherlands. <br>
[Sander, Jorg; Isgum, Ivana] Univ Amsterdam, Informat Inst, Amsterdam, Netherlands. <br>
[Viergever, Max A.] Univ Med Ctr Utrecht, Image Sci Inst, Utrecht, Netherlands. <br>
[Leiner, Tim] Mayo Clin, Dept Radiol, Rochester, MN USA. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Sander, J (corresponding author), Locat Univ Amsterdam, Dept Biomed Engn &amp; Phys, Amsterdam Univ Med Ctr, Amsterdam, Netherlands.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
j.sander1@amsterdamumc.nl</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
University of Amsterdam; Utrecht University; Utrecht University Medical Center; Mayo Clinic</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Planken, R. Nils</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-3940-4670&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Viergever, Max</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-2582-042X&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Isgum, Ivana</display_name>&nbsp;</td><td>H-8659-2014&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Planken, RN</display_name>&nbsp;</td><td>JAC-4538-2023&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>PERGAMON-ELSEVIER SCIENCE LTD</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>THE BOULEVARD, LANGFORD LANE, KIDLINGTON, OXFORD OX5 1GB, ENGLAND</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Biology; Computer Science, Interdisciplinary Applications; Engineering, Biomedical; Mathematical &amp; Computational Biology</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Life Sciences &amp; Biomedicine - Other Topics; Computer Science; Engineering; Mathematical &amp; Computational Biology</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>P1OX0</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0010-4825</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1879-0534</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>COMPUT BIOL MED</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Comput. Biol. Med.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>15</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Dutch Technology Foundation</grant_agency>&nbsp;</td><td>
<div>P15-26&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>Acknowledgments This study was performed within the DLMedIA program (P15-26) funded by Dutch Technology Foundation with participation of Pie Medical Imaging.</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>hybrid</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 44 of 49</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>M2-like macrophages transplantation protects against the doxorubicin-induced heart failure via mitochondrial transfer</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Liu, YH (Liu, Yihai); Wu, MY (Wu, Mingyue); Zhong, CX (Zhong, Chongxia); Xu, B (Xu, Biao); Kang, LN (Kang, Lina)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>BIOMATERIALS RESEARCH</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>26</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>1</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>14</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1186/s40824-022-00260-y</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2022 APR 11</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>42</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>46</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
6</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
52</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>42</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Aims: The alternatively activated macrophages have shown a cardioprotective effect in heart failure. However, the effect of M2 adoptive transfer in non-ischemic heart failure is unknown. In this study, we evaluated the efficacy of M-CSF plus IL-4 induced M2-like macrophages transplantation in doxorubicin-induced cardiotoxicity.
<br>Methods: Bone marrow mononuclear cells were polarized as CCR2(+)CD206(+) M2-like macrophages by a combination of M-CSF plus IL-4 treatment. C57BL/6 mice received a single intraperitoneal injection of doxorubicin (15 mg/kg). The treatment group were treated with M2-like macrophages (1 x 10&lt;^&gt;6 cells per mouse; i.v.) once a week for 2 weeks. After 3 weeks, we examined the percentage of resident cells and cardiac function. Furthermore, we evaluated cardiac fibrosis, cardiomyocyte apoptosis and circulating inflammatory factors. Finally, we investigated the mitochondria transfer in vitro in a direct and indirect co-culture conditions.
<br>Results: Cardiac function was significantly improved in doxorubicin-induced heart failure by adoptive transfer of M2-like macrophages. Besides, M2-like macrophages treatment attenuated cardiac fibrosis and cardiomyocyte apoptosis, as well as increased the level of circulating IL-4 and Th2 response. In vitro, M2-like macrophages could transfer mitochondria to injured cardiomyocytes in a direct and indirect way.
<br>Conclusions: In our study, adoptive transfer of M2-like macrophages could protect against the doxorubicin-induced cardiotoxicity, which may be partly attributed to mitochondria transfer. And M2-like macrophages transplantation could become a treatment for non-ischemic heart failure in the clinical practice.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000782210400001</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>35410296</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
M2-like macrophages; Cardiotoxicity; Heart failure; Mitochondria; Cell transplantation</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
MESENCHYMAL STEM-CELLS; ADOPTIVE TRANSFER; INJURY; EXPRESSION; RECOVERY; NEURONS; RATS</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Liu, Yihai; Wu, Mingyue; Zhong, Chongxia; Xu, Biao; Kang, Lina] Nanjing Univ, Dept Cardiol, Affiliated Drum Tower Hosp, Med Sch, 321 Zhongshan Rd, Nanjing 210008, Jiangsu, Peoples R China. <br>
[Liu, Yihai] Nanjing Med Univ, Dept Cardiol, Nanjing Drum Tower Hosp, Clin Sch, Nanjing 210008, Jiangsu, Peoples R China. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Xu, B; Kang, LN (corresponding author), Nanjing Univ, Dept Cardiol, Affiliated Drum Tower Hosp, Med Sch, 321 Zhongshan Rd, Nanjing 210008, Jiangsu, Peoples R China.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
xubiao62@nju.edu.cn; kanglina@njglyy.com</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Nanjing University; Nanjing University; Nanjing Medical University</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>w, y</display_name>&nbsp;</td><td>JXN-7788-2024&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>AMER ASSOC ADVANCEMENT SCIENCE</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>1200 NEW YORK AVE, NW, WASHINGTON, DC 20005 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Engineering, Biomedical; Materials Science, Biomaterials</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Engineering; Materials Science</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>0M5RI</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>1226-4601</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>2055-7124</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>BIOMATER RES</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Biomater. Res.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>14</value>
</td>
</tr>

<tr>
<td>
<b>Open Access:</b>

<value>gold, Green Published</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 45 of 49</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Low-dimensional representation of cardiac motion using Barycentric Subspaces: A new group-wise paradigm for estimation, analysis, and reconstruction</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Roh&eacute;, MM (Rohe, Marc-Michel); Sermesant, M (Sermesant, Maxime); Pennec, X (Pennec, Xavier)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>MEDICAL IMAGE ANALYSIS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>45</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>1-12</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1016/j.media.2017.12.008</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2018 APR</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>8</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>9</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
0</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
8</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>26</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
One major challenge when trying to build low-dimensional representation of the cardiac motion is its natural circular pattern during a cycle, therefore making the mean image a poor descriptor of the whole sequence. Therefore, traditional approaches for the analysis of the cardiac deformation use one specific frame of the sequence - the end-diastolic (ED) frame - as a reference to study the whole motion. Consequently, this methodology is biased by this empirical choice. Moreover, the ED image might be a poor reference when looking at large deformation for example at the end-systolic (ES) frame. In this paper, we propose a novel approach to study cardiac motion in 4D image sequences using low-dimensional sub-space analysis. Instead of building subspaces relying on a mean value we use a novel type of subspaces called Barycentric Subspaces which are implicitly defined as the weighted Karcher means of k + 1 reference images instead of being defined with respect to one reference image. In the first part of this article, we introduce the methodological framework and the algorithms used to manipulate images within these new subspaces: how to compute the projection of a given image on the Barycentric Subspace with its coordinates, and the opposite operation of computing an image from a set of references and coordinates. Then we show how this framework can be applied to cardiac motion problems and lead to significant improvements over the single reference method. Firstly, by computing the low-dimensional representation of two populations we show that the parameters extracted correspond to relevant cardiac motion features leading to an efficient representation and discrimination of both groups. Secondly, in motion estimation, we use the projection on this low-dimensional subspace as an additional prior on the regularization in cardiac motion tracking, efficiently reducing the error of the registration between the ED and ES by almost 30%. We also derive a symmetric and transitive formulation of the registration that can be used both for frame-to-frame and frame-to-reference registration. Finally, we look at the reconstruction of the images using our proposed low-dimensional representation and show that this multi-references method using Barycentric Subspaces performs better than traditional approaches based on a single reference. (C) 2018 Elsevier B.V. All rights reserved.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000427664400001</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>29324241</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Low-dimensional analysis; Cardiac motion; Registration; Image synthesis</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
DIFFEOMORPHIC REGISTRATION; DEFORMATION ALGORITHMS; DEMONS</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Rohe, Marc-Michel; Sermesant, Maxime; Pennec, Xavier] Univ Cote Dazur, INRIA, Asclepios Res Grp, Nice, France. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Roh&eacute;, MM (corresponding author), Univ Cote Dazur, INRIA, Asclepios Res Grp, Nice, France.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
marc-michel.rohe@inria.fr</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Inria; Universite Cote d'Azur</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Pennec, Xavier</display_name>&nbsp;</td><td>L-2537-2013&nbsp;</td><td>0000-0002-6617-7664&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Sermesant, Maxime</display_name>&nbsp;</td><td>AAC-4870-2019&nbsp;</td><td>0000-0002-6256-8350&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>ELSEVIER SCIENCE BV</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>PO BOX 211, 1000 AE AMSTERDAM, NETHERLANDS</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Artificial Intelligence; Computer Science, Interdisciplinary Applications; Engineering, Biomedical; Radiology, Nuclear Medicine &amp; Medical Imaging</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Engineering; Radiology, Nuclear Medicine &amp; Medical Imaging</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>FZ5VM</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>1361-8415</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1361-8423</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>MED IMAGE ANAL</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Med. Image Anal.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>12</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>EU FP7</grant_agency>&nbsp;</td><td>
<div>600932&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>The authors acknowledge the partial funding by the EU FP7-funded project MD-Paedigree (Grant agreement 600932)</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>Green Submitted</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 46 of 49</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>CF Distance: A New Domain Discrepancy Metric and Application to Explicit Domain Adaptation for Cross-Modality Cardiac Image Segmentation</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Wu, FP (Wu, Fuping); Zhuang, XH (Zhuang, Xiahai)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE TRANSACTIONS ON MEDICAL IMAGING</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>39</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>12</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>4274-4285</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/TMI.2020.3016144</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2020 DEC</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>64</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>69</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
0</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
44</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>42</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Domain adaptation has great values in unpaired cross-modality image segmentation, where the training images with gold standard segmentation are not available from the target image domain. The aim is to reduce the distribution discrepancy between the source and target domains. Hence, an effective measurement for this discrepancy is critical. In this work, we propose a new metric based on characteristic functions of distributions. This metric, referred to as CF distance, enables explicit domain adaptation, in contrast to the implicit manners minimizing domain discrepancy via adversarial training. Based on this CF distance, we propose an unsupervised domain adaptation framework for cross-modality cardiac segmentation, which consists of image reconstruction and prior distribution matching. We validated the method on two tasks, i.e., the CT-MR cross-modality segmentation and the multi-sequence cardiac MR segmentation. Results showed that the proposed explicit metric was effective in domain adaptation, and the segmentation method delivered promising and superior performance, compared to other state-of-the-art techniques. The data and source code of this work has been released via https://zmiclab.github.io/projects.html.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000595547500045</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>32784131</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Measurement; Training; Image segmentation; Neural networks; Task analysis; Image reconstruction; Optimization; Cardiacsegmentation; omain adaptation; domain discrepancy; characteristic function</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
NETWORK</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Wu, Fuping] Fudan Univ, Dept Stat, Sch Management, Shanghai 200433, Peoples R China. <br>
[Wu, Fuping; Zhuang, Xiahai] Fudan Univ, Sch Data Sci, Shanghai 200433, Peoples R China. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Zhuang, XH (corresponding author), Fudan Univ, Sch Data Sci, Shanghai 200433, Peoples R China.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
17110690006@fudan.edu.cn; zxh@fudan.edu.cn</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Fudan University; Fudan University</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Zhuang, Xiahai</display_name>&nbsp;</td><td>AAH-6334-2019&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Wu, Fuping</display_name>&nbsp;</td><td>ABG-4545-2021&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Wu, Fuping</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-7179-4766&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Interdisciplinary Applications; Engineering, Biomedical; Engineering, Electrical &amp; Electronic; Imaging Science &amp; Photographic Technology; Radiology, Nuclear Medicine &amp; Medical Imaging</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Engineering; Imaging Science &amp; Photographic Technology; Radiology, Nuclear Medicine &amp; Medical Imaging</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>PA3OL</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0278-0062</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1558-254X</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE T MED IMAGING</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE Trans. Med. Imaging</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>12</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Natural Science Foundation of China</grant_agency>&nbsp;</td><td>
<div>61971142&nbsp;</div>
<div>11871165&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported by the National Natural Science Foundation of China under Grant 61971142 and Grant 11871165.</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 47 of 49</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Automatic sleep-stage classification of heart rate and actigraphy data using deep and transfer learning approaches</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Ma, YJX (Ma, Yaopeng J. X.); Zschocke, J (Zschocke, Johannes); Glos, M (Glos, Martin); Kluge, M (Kluge, Maria); Penzel, T (Penzel, Thomas); Kantelhardt, JW (Kantelhardt, Jan W.); Bartsch, RP (Bartsch, Ronny P.)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>COMPUTERS IN BIOLOGY AND MEDICINE</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>163</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>107193</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1016/j.compbiomed.2023.107193</value>&nbsp;&nbsp;<b>Early Access Date:</b> 
<value>JUL 2023</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2023 SEP</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>13</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>13</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
3</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
34</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>64</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Manual sleep-stage scoring based on full-night polysomnography data recorded in a sleep lab has been the gold standard of clinical sleep medicine. This costly and time-consuming approach is unfit for long-term studies as well as assessment of sleep on a population level. With the vast amount of physiological data becoming available from wrist-worn devices, deep learning techniques provide an opportunity for fast and reliable automatic sleep-stage classification tasks. However, training a deep neural network requires large annotated sleep databases, which are not available for long-term epidemiological studies. In this paper, we introduce an end-to-end temporal convolutional neural network able to automatically score sleep stages from raw heartbeat RR interval (RRI) and wrist actigraphy data. Moreover, a transfer learning approach enables the training of the network on a large public database (Sleep Heart Health Study, SHHS) and its subsequent application to a much smaller database recorded by a wristband device. The transfer learning significantly shortens training time and improves sleep-scoring accuracy from 68.9% to 73.8% and inter-rater reliability (Cohen's kappa) from 0.51 to 0.59. We also found that for the SHHS database, automatic sleep-scoring accuracy using deep learning shows a logarithmic relationship with the training size. Although deep learning approaches for automatic sleep scoring are not yet comparable to the inter-rater reliability among sleep technicians, performance is expected to significantly improve in the near future when more large public databases become available. We anticipate those deep learning techniques, when combined with our transfer learning approach, will leverage automatic sleep scoring of physiological data from wearable devices and enable the investigation of sleep in large cohort studies.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001039211500001</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>37421734</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Sleep-stage scoring; Heart-rate variability; Wrist actigraphy; Deep learning; Transfer learning; Convolutional neural network; Epidemiological studies</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
RECHTSCHAFFEN; VARIABILITY; DYNAMICS; DESIGN; KALES; AASM</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Ma, Yaopeng J. X.; Bartsch, Ronny P.] Bar Ilan Univ, Dept Phys, Ramat Gan, Israel. <br>
[Zschocke, Johannes] Martin Luther Univ Halle Wittenberg, Interdisciplinary Ctr Hlth Sci, Inst Med Epidemiol Biometr &amp; Informat IMEBI, Halle, Germany. <br>
[Zschocke, Johannes; Kantelhardt, Jan W.] Martin Luther Univ Halle Wittenberg, Inst Phys, Halle, Germany. <br>
[Glos, Martin; Kluge, Maria; Penzel, Thomas] Charite Univ Med Berlin, Interdisciplinary Sleep Med Ctr, Berlin, Germany. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Bartsch, RP (corresponding author), Bar Ilan Univ, Dept Phys, Ramat Gan, Israel.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
yaopeng.ma97@gmail.com; bartsch.ronny@gmail.com</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Bar Ilan University; Martin Luther University Halle Wittenberg; Martin Luther University Halle Wittenberg; Free University of Berlin; Humboldt University of Berlin; Charite Universitatsmedizin Berlin</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Bartsch, Ronny</display_name>&nbsp;</td><td>E-7117-2012&nbsp;</td><td>0000-0001-8336-5141&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Penzel, Thomas</display_name>&nbsp;</td><td>M-2344-2014&nbsp;</td><td>0000-0002-4304-0112&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Kluge, Maria</display_name>&nbsp;</td><td>&nbsp;</td><td>0009-0008-6714-4478&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Ma, Yaopeng J. X.</display_name>&nbsp;</td><td>KBQ-9248-2024&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Ma, Yaopeng J. X.</display_name>&nbsp;</td><td>&nbsp;</td><td>0009-0004-6484-0129&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Kantelhardt, Jan W.</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-9293-0293&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Kantelhardt, Jan</display_name>&nbsp;</td><td>AAF-7955-2019&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>PERGAMON-ELSEVIER SCIENCE LTD</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>THE BOULEVARD, LANGFORD LANE, KIDLINGTON, OXFORD OX5 1GB, ENGLAND</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Biology; Computer Science, Interdisciplinary Applications; Engineering, Biomedical; Mathematical &amp; Computational Biology</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Life Sciences &amp; Biomedicine - Other Topics; Computer Science; Engineering; Mathematical &amp; Computational Biology</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>N8BW3</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0010-4825</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1879-0534</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>COMPUT BIOL MED</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Comput. Biol. Med.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>11</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>German Israeli Foundation (GIF)</grant_agency>&nbsp;</td><td>
<div>I-1372-303.7/2016&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Heart, Lung, and Blood Institute</grant_agency>&nbsp;</td><td>
<div>U01HL53916&nbsp;</div>
<div>U01HL53931&nbsp;</div>
<div>U01HL53934&nbsp;</div>
<div>U01HL53937&nbsp;</div>
<div>U01HL64360&nbsp;</div>
<div>U01HL53938&nbsp;</div>
<div>U01HL53940&nbsp;</div>
<div>U01HL53941&nbsp;</div>
<div>U01HL63463&nbsp;</div>
<div>R24 HL114473&nbsp;</div>
<div>75N92019R002&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>&amp; nbsp;This work was supported by the German Israeli Foundation (GIF) Grant no I-1372-303.7/2016. The Sleep Heart Health Study (SHHS) was supported by National Heart, Lung, and Blood Institute cooperative agreements U01HL53916 (University of California, Davis) , U01HL53931 (New York University) , U01HL53934 (University of Minnesota) , U01HL53937 and U01HL64360 (Johns Hopkins University) , U01HL53938 (University of Arizona) , U01HL53940 (University of Washington) , U01HL53941 (Boston University) , and U01HL63463 (Case Western Reserve University) . The National Sleep Research Resource was supported by the National Heart, Lung, and Blood Institute (R24 HL114473, 75N92019R002) .</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 48 of 49</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Dual-Teacher plus plus : Exploiting Intra-Domain and Inter-Domain Knowledge With Reliable Transfer for Cardiac Segmentation</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Li, K (Li, Kang); Wang, SJ (Wang, Shujun); Yu, LQ (Yu, Lequan); Heng, PA (Heng, Pheng Ann)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE TRANSACTIONS ON MEDICAL IMAGING</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>40</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>10</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>2771-2782</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/TMI.2020.3038828</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2021 OCT</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>24</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>27</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
1</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
37</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>61</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Annotation scarcity is a long-standing problem in medical image analysis area. To efficiently leverage limited annotations, abundant unlabeled data are additionally exploited in semi-supervised learning, while well-established cross-modality data are investigated in domain adaptation. In this paper, we aim to explore the feasibility of concurrently leveraging both unlabeled data and cross-modality data for annotation-efficient cardiac segmentation. To this end, we propose a cutting-edge semi-supervised domain adaptation framework, namely Dual-Teacher++. Besides directly learning from limited labeled target domain data (e.g., CT) via a student model adopted by previous literature, we design novel dual teacher models, including an inter-domain teacher model to explore cross-modality priors from source domain (e.g., MR) and an intra-domain teacher model to investigate the knowledge beneath unlabeled target domain. In this way, the dual teacher models would transfer acquired inter- and intra-domain knowledge to the student model for further integration and exploitation. Moreover, to encourag reliable dual-domain knowledge transfer, we enhance the inter-domain knowledge transfer on the samples with higher similarity to target domain after appearance alignment, and also strengthen intra-domain knowledge transfer of unlabeled target data with higher prediction confidence. In this way, the student model can obtain reliable dual-domain knowledge and yield improved performance on target domain data. We extensively evaluated the feasibility of our method on the MM-WHS 2017 challenge dataset. The experiments have demonstrated the superiority of our framework over other semi-supervised learning and domain adaptation methods. Moreover, our performance gains could be yielded in bidirections, i.e., adapting from MR to CT, and from CT to MR. Our code will be available at https://github.com/kli-lalala/Dual-Teacher-.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000702638800020</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>33201808</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Adaptation models; Data models; Reliability; Predictive models; Computed tomography; Annotations; Semisupervised learning; Semi-supervised domain adaptation; cross-modality; cardiac segmentation</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
ADAPTATION; IMAGE</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Li, Kang; Wang, Shujun; Heng, Pheng Ann] Chinese Univ Hong Kong, Dept Comp Sci Engn, Hong Kong, Peoples R China. <br>
[Yu, Lequan] Stanford Univ, Dept Radiat Oncol, Palo Alto, CA 94306 USA. <br>
[Heng, Pheng Ann] Chinese Acad Sci, Shenzhen Inst Adv Technol, Guangdong Prov Key Lab Comp Vis &amp; Virtual Real, Shenzhen 518055, Peoples R China. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Yu, LQ (corresponding author), Stanford Univ, Dept Radiat Oncol, Palo Alto, CA 94306 USA.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
kli@cse.cuhk.edu.hk; sjwang@cse.cuhk.edu.hk; lequany@stanford.edu; pheng@cse.cuhk.edu.hk</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Chinese University of Hong Kong; Stanford University; Chinese Academy of Sciences; Shenzhen Institute of Advanced Technology, CAS</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Yu, Lequan</display_name>&nbsp;</td><td>U-5377-2019&nbsp;</td><td>0000-0002-9315-6527&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>WANG, Shujun</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-1495-3278&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Li, Kang</display_name>&nbsp;</td><td>LMN-3556-2024&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Wang, Shujun</display_name>&nbsp;</td><td>AAD-7001-2020&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>LI, Kang</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-0149-6912&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Heng, Pheng</display_name>&nbsp;</td><td>KJL-6056-2024&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Heng, Pheng Ann</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-3055-5034&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Interdisciplinary Applications; Engineering, Biomedical; Engineering, Electrical &amp; Electronic; Imaging Science &amp; Photographic Technology; Radiology, Nuclear Medicine &amp; Medical Imaging</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Engineering; Imaging Science &amp; Photographic Technology; Radiology, Nuclear Medicine &amp; Medical Imaging</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>WA1FC</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0278-0062</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1558-254X</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE T MED IMAGING</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE Trans. Med. Imaging</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>12</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Key-Area Research and Development Program of GuangdongProvince, China</grant_agency>&nbsp;</td><td>
<div>2020B010165004&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Hong Kong Innovation and Technology Fund</grant_agency>&nbsp;</td><td>
<div>ITS/311/18FP&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Natural Science Foundation of China</grant_agency>&nbsp;</td><td>
<div>U1813204&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported in part by the Key-Area Research and Development Program of GuangdongProvince, China, under Project 2020B010165004, in part by the Hong Kong Innovation and Technology Fund under Project ITS/311/18FP, and in part by the National Natural Science Foundation of China under Project U1813204.</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 49 of 49</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Fetal Congenital Heart Disease Echocardiogram Screening Based on DGACNN: Adversarial One-Class Classification Combined with Video Transfer Learning</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Gong, YX (Gong, Yuxin); Zhang, YY (Zhang, Yingying); Zhu, HG (Zhu, Haogang); Lv, J (Lv, Jing); Cheng, Q (Cheng, Qian); Zhang, HJ (Zhang, Hongjia); He, YH (He, Yihua); Wang, SL (Wang, Shuliang)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE TRANSACTIONS ON MEDICAL IMAGING</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>39</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>4</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>1206-1222</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/TMI.2019.2946059</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2020 APR</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>67</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>78</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
2</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
55</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>39</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Fetal congenital heart disease (FHD) is a common and serious congenital malformation in children. In Asia, FHD birth defect rates have reached as high as 9.3 0025. For the early detection of birth defects and mortality, echocardiography remains the most effective method for screening fetal heart malformations. However, standard echocardiograms of the fetal heart, especially four-chamber view images, are difficult to obtain. In addition, the pathophysiological changes in fetal hearts during different pregnancy periods lead to ever-changing two-dimensional fetal heart structures and hemodynamics, and it requires extensive professional knowledge to recognize and judge disease development. Thus, research on the automatic screening for FHD is necessary. In this paper, we proposed a new model named DGACNN that shows the best performance in recognizing FHD, achieving a rate of 85. The motivation for this network is to deal with the problem that there are insufficient training datasets to train a robust model. There are many unlabeled video slices, but they are tough and time-consuming to annotate. Thus, how to use these un-annotated video slices to improve the DGACNN capability for recognizing FHD, in terms of both recognition accuracy and robustness, is very meaningful for FHD screening. The architecture of DGACNN comprises two parts, that is, DANomaly and GACNN (Wgan-GP and CNN). DANomaly, similar to the ALOCC network, but incorporates cycle adversarial learning to train an end-to-end one-class classification (OCC) network that is more robust and has a higher accuracy than ALOCC in screening video slices. For the GACNN architecture, we use FCH (four chamber heart) video slices at around the end-systole, as screened by DANomaly, to train a WGAN-GP for the purpose of obtaining ideal low-level features that can robustly improve the FHD recognition accuracy. A few annotated video slices, as screened by DANomaly, can also be used for data augmentation so as to improve the FHD recognition further. The experiments show that the DGACNN outperforms other state-of-the-art networks by 10025; in recognizing FHD. A comparison experiment shows that the proposed network already outperforms the performance of expert cardiologists in recognizing FHD, reaching 84 025; in a test. Thus, the proposed architecture has high potential for helping cardiologists complete early FHD screenings.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000525265800036</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>31603775</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Fetal congenital heart disease; one-class classification; generative adversarial network; transfer learning; echocardiography; four-chamber heart</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
CONVOLUTIONAL NEURAL-NETWORKS; UNITED-STATES</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Gong, Yuxin; Zhang, Yingying; Zhu, Haogang; Cheng, Qian] Beihang Univ, Sch Comp Sci &amp; Engn, State Key Lab Software Dev Environm, Beijing 100083, Peoples R China. <br>
[Gong, Yuxin; Zhang, Yingying] Beihang Univ, Sch Biol Sci &amp; Med Engn, Beijing 100083, Peoples R China. <br>
[Zhu, Haogang] Beihang Univ, Beijing Adv Innovat Ctr Biomed Engn, Beijing 100083, Peoples R China. <br>
[Zhu, Haogang] Beihang Univiers, Hefei Innovat Res Inst, Beijing 100083, Peoples R China. <br>
[Lv, Jing; Zhang, Hongjia; He, Yihua] Capital Med Univ, Beijing Anzhen Hosp, Dept Ultrasound, Beijing 100069, Peoples R China. <br>
[Wang, Shuliang] Beijing Inst Technol, Sch Comp Sci &amp; Technol, Beijing 100811, Peoples R China. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Zhu, HG (corresponding author), Beihang Univ, Sch Comp Sci &amp; Engn, State Key Lab Software Dev Environm, Beijing 100083, Peoples R China.<br>He, YH (corresponding author), Capital Med Univ, Beijing Anzhen Hosp, Dept Ultrasound, Beijing 100069, Peoples R China.<br>Wang, SL (corresponding author), Beijing Inst Technol, Sch Comp Sci &amp; Technol, Beijing 100811, Peoples R China.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
gongyuxinbuaa@163.com; haogangzhu@buaa.edu.cn; heyihuaecho@hotmail.com; slwang2011@bit.edu.cn</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Beihang University; Beihang University; Beihang University; Capital Medical University; Beijing Institute of Technology</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>lv, jing</display_name>&nbsp;</td><td>JXM-6679-2024&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Cheng, Qian</display_name>&nbsp;</td><td>M-6702-2017&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>gong, yuxin</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-4692-8931&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>zhang, yingying</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-0964-1774&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>WANG, Shuliang</display_name>&nbsp;</td><td>A-2626-2012&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Interdisciplinary Applications; Engineering, Biomedical; Engineering, Electrical &amp; Electronic; Imaging Science &amp; Photographic Technology; Radiology, Nuclear Medicine &amp; Medical Imaging</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Engineering; Imaging Science &amp; Photographic Technology; Radiology, Nuclear Medicine &amp; Medical Imaging</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>LC4AC</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0278-0062</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1558-254X</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE T MED IMAGING</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE Trans. Med. Imaging</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>17</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>State Key Laboratory of Software Development Environment, Beihang University, Beijing, China</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Beijing Advanced Innovation Center for Biomedical Engineering, Beihang University, China</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Hefei Innovation Research Institute, Beihang University, China</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Natural Science Foundation</grant_agency>&nbsp;</td><td>
<div>61702027&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Beijing Science and Technology Plan Project</grant_agency>&nbsp;</td><td>
<div>Z171100000117022&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Beijing Municipal Science &amp; Technology Commission</grant_agency>&nbsp;</td><td>
<div>Z181100001918008&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Beijing Lab for Cardiovascular Precision Medicine</grant_agency>&nbsp;</td><td>
<div>PXM2018014226000013&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported in part by the State Key Laboratory of Software Development Environment, Beihang University, Beijing, China, in part by the Beijing Advanced Innovation Center for Biomedical Engineering, Beihang University, China, in part by the Hefei Innovation Research Institute, Beihang University, China, in part by the National Natural Science Foundation under Grant 61702027, in part by the Beijing Science and Technology Plan Project under Grant Z171100000117022, in part by the Beijing Municipal Science &amp; Technology Commission under Grant Z181100001918008, and in part by the Beijing Lab for Cardiovascular Precision Medicine under Grant PXM2018014226000013.</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr>
<table xmlns:bean="http://ts.thomson.com/ua/bean">
<tr>
<td>End of File</td>
</tr>
</table><div style="background-color: #000000; padding: 10px; color: #FFFFFF; font-family: arial, sans-serif;">
<div style="height: 25px; width: 100px;">
<mat-icon data-mat-icon-name="clarivate" data-mat-icon-type="svg" aria-hidden="true" class="mat-icon notranslate clarivate-logo mat-icon-no-color" svgicon="clarivate" role="img">
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" focusable="false" preserveAspectRatio="xMidYMid meet" version="1.1" viewBox="0 0 92 18" height="100%" width="100%">
<title>Clarivate</title>
<g fill-rule="evenodd" fill="none" stroke-width="1" stroke="none">
<g fill-rule="nonzero" fill="#FFFFFF" transform="translate(-19.000000, -8.000000)">
<g transform="translate(1.000000, 1.000000)">
<g transform="translate(18.000000, 7.000000)">
<path d="M26.6812194,13.2997252 C28.2949641,13.3845068 29.7591419,12.3596036 30.2318671,10.8143235 L33.0910469,10.8143235 C32.3628779,13.742285 29.6973379,15.7705498 26.6812194,15.6916944 C24.8651813,15.7485215 23.1057028,15.0559504 21.8156554,13.7764967 C20.5256081,12.4970431 19.8185526,10.7433351 19.8604035,8.9268901 C19.8185367,7.11043445 20.5255848,5.35671042 21.8156337,4.07724261 C23.1056826,2.7977748 24.8651702,2.10519376 26.6812194,2.16201887 C29.6973451,2.0831702 32.3628861,4.11144761 33.0910469,7.03941847 L30.2318671,7.03941847 C29.7591419,5.49413828 28.2949641,4.46923515 26.6812194,4.55401674 C25.5407067,4.54133389 24.4470406,5.00703241 23.6657805,5.83803342 C22.8845203,6.66903442 22.487126,7.78932829 22.5700912,8.9268901 C22.4871313,10.064447 22.8845287,11.184734 23.6657888,12.0157277 C24.447049,12.8467215 25.540712,13.312413 26.6812194,13.2997252 L26.6812194,13.2997252 Z"/>
<polygon points="36.8477342 15.4674945 34.4371264 15.4674945 34.4371264 2.38628571 36.8477342 2.38628571"/>
<path d="M43.5380316,9.76781183 C43.5380316,9.15114738 43.0522397,8.75870193 42.1178663,8.75870193 C41.2070757,8.82048057 40.3236107,9.0957548 39.5389268,9.5622602 L38.5858286,7.73092465 C39.6930919,7.06514423 40.9567617,6.70409618 42.2486241,6.68441302 C44.1921648,6.68441302 45.9674886,7.58137515 45.9674886,9.84255787 L45.9674886,15.4674945 L43.5380316,15.4674945 L43.5380316,14.6639362 C42.9494098,15.3673061 42.0613957,15.7488608 41.146005,15.6917231 C40.3900371,15.7421039 39.645831,15.484584 39.0826798,14.9777443 C38.5195286,14.4709046 38.1853154,13.7578403 38.1560675,13.0007601 C38.1560675,11.3749188 39.3519468,10.3658089 41.2581433,10.3658089 L42.9400537,10.3658089 C43.3511665,10.3658089 43.5379934,10.1789247 43.5379934,9.86128266 L43.5380316,9.76781183 Z M43.5380316,12.4027631 L43.5380316,11.786089 C43.321275,11.9402969 43.0543042,12.0070182 42.790485,11.9729159 L41.8748173,11.9729159 C41.1086607,11.9729159 40.6602036,12.3092922 40.6602036,12.9072893 C40.6602036,13.486619 41.1272707,13.9164566 41.8748173,13.9164566 C42.303867,13.9699481 42.7348632,13.8332333 43.0546312,13.5422121 C43.3743992,13.2511908 43.5509896,12.8349402 43.5380316,12.4027631 L43.5380316,12.4027631 Z"/>
<path d="M52.6579104,9.3006299 L51.6301236,9.3006299 C50.3966703,9.3006299 49.8361037,9.8799596 49.8361037,11.3749188 L49.8361037,15.4674945 L47.4066179,15.4674945 L47.4066179,6.90863203 L49.7612715,6.90863203 L49.7612715,8.27284306 C50.1001881,7.34759017 51.0007831,6.74968637 51.9850429,6.79648425 C52.2115562,6.79362484 52.4375557,6.81874947 52.6579104,6.87128771 L52.6579104,9.3006299 Z"/>
<path d="M57.4232866,15.4674945 L54.9939445,15.4674945 L54.9939445,8.96425355 L53.7979312,8.96425355 L53.7979312,6.90863203 L57.4232866,6.90863203 L57.4232866,15.4674945 Z M56.0964104,2.38628347 C56.5129556,2.38558064 56.9126431,2.55074046 57.2071855,2.84528284 C57.5017278,3.13982521 57.6668877,3.53951271 57.6661848,3.95605794 C57.6609778,4.814152 56.9638874,5.50702694 56.1057775,5.50702694 C55.2476677,5.50702694 54.5505773,4.814152 54.5453585,3.95605794 C54.5417726,3.09491157 55.2352828,2.39302469 56.0964104,2.38628347 Z"/>
<polygon points="62.8802456 12.8512776 64.8236715 6.90863203 67.3838767 6.90863203 64.33787 15.4674945 61.3292077 15.4674945 58.4139588 6.90863203 61.0488526 6.90863203"/>
<path d="M72.8783522,9.76781183 C72.8783522,9.15114738 72.3924359,8.75870193 71.4580625,8.75870193 C70.5472784,8.82049943 69.6638208,9.09577238 68.8791325,9.5622602 L67.9261491,7.73092465 C69.0333627,7.06513461 70.2969953,6.70408494 71.5888202,6.68441302 C73.5323609,6.68441302 75.3076848,7.58137515 75.3076848,9.84255787 L75.3076848,15.4674945 L72.8783522,15.4674945 L72.8783522,14.6639362 C72.2896971,15.3672586 71.4017035,15.7487939 70.4863256,15.6916944 C69.7303436,15.7420941 68.986117,15.4845835 68.4229464,14.9777426 C67.8597758,14.4709017 67.5255471,13.7578265 67.4962924,13.0007314 C67.4962924,11.3748901 68.6923056,10.3657802 70.5984925,10.3657802 L72.2802785,10.3657802 C72.6913914,10.3657802 72.8783426,10.1788959 72.8783426,9.86125395 L72.8783522,9.76781183 Z M72.8783522,12.4027631 L72.8783522,11.786089 C72.6615874,11.9402794 72.3946236,12.006999 72.1308056,11.9729159 L71.2151378,11.9729159 C70.4488664,11.9729159 70.0004093,12.3092922 70.0004093,12.9072893 C70.0004093,13.486619 70.4676008,13.9164566 71.2151378,13.9164566 C71.6441833,13.9699279 72.0751679,13.8332061 72.394931,13.5421893 C72.7146941,13.2511724 72.8912904,12.8349341 72.8783522,12.4027631 L72.8783522,12.4027631 Z"/>
<path d="M79.9985637,3.65701635 L79.9985637,6.90863203 L81.8485762,6.90863203 L81.8485762,8.96425355 L79.9985637,8.96425355 L79.9985637,12.8325433 C79.9985637,13.3745286 80.4282865,13.4492747 80.8020646,13.4492747 C81.418729,13.4492747 81.9232553,13.3932056 81.9232553,13.3932056 L81.9232553,15.4674945 C81.9232553,15.4674945 80.7459859,15.5796423 80.0357932,15.5796423 C78.8211795,15.5796423 77.5691354,15.3740811 77.5691354,13.6548263 L77.5691354,8.96425355 L76.22363,8.96425355 L76.22363,6.90863203 L77.5691163,6.90863203 L77.5691163,3.65701635 L79.9985637,3.65701635 Z"/>
<path d="M91.5475359,11.8981794 L85.1751771,11.8981794 C85.3031552,12.7204351 85.9194738,13.3815958 86.7307322,13.5669136 C87.5419905,13.7522314 88.3842592,13.4242593 88.856573,12.7391298 L91.3793191,13.0007601 C90.4933306,14.9536735 88.3725856,16.0321989 86.2725384,15.5978628 C84.1724912,15.1635268 82.6535603,13.3322361 82.6149337,11.188092 C82.567807,9.98960603 83.0193435,8.82515843 83.8621298,7.97175307 C84.7049161,7.1183477 85.8636189,6.65226701 87.0626009,6.68438432 C89.4171397,6.68438432 91.5475359,8.21674525 91.5475359,11.8047373 L91.5475359,11.8981794 Z M85.1189837,10.2723955 L88.9499864,10.2723955 C88.8059978,9.34848922 87.9973156,8.67588427 87.0626392,8.70263282 C86.1147409,8.67034807 85.2869107,9.33893202 85.1189837,10.2723955 Z"/>
<path d="M12.6820274,9.85132226 C10.6885049,11.8556057 8.11575776,13.1830474 5.32707189,13.6462054 C5.77780891,14.8758828 6.40559556,16.0331959 7.19058498,17.0815575 C10.3903106,16.2989026 13.3004311,14.619851 15.5794509,12.2414352 C15.0651988,11.0374085 14.3787129,9.91449973 13.5415112,8.90790698 C13.271174,9.23328 12.9846793,9.54772625 12.6820274,9.85124571"/>
<path d="M5.32722498,13.6460332 C4.21304566,10.5852611 4.22627578,7.22758116 5.3645406,4.17568425 C4.07298567,3.95802358 2.75701252,3.92542943 1.45626578,4.07888372 C0.536506305,7.22972054 0.525090565,10.5762272 1.42333236,13.7332656 C2.72217338,13.8896343 4.03666286,13.8602685 5.32722498,13.6460524"/>
<path d="M7.25919787,0.751096346 C6.46424104,1.79423541 5.8259601,2.94792723 5.36457887,4.17561728 C8.13918123,4.64457901 10.6977043,5.96938232 12.6820178,7.96458738 C12.9850078,8.2673988 13.2715025,8.58184824 13.5415017,8.90793568 C14.3788161,7.90142183 15.0653154,6.77849641 15.5794509,5.57440744 C13.3171586,3.21319806 10.4324457,1.54091896 7.25919787,0.751096346"/>
</g>
</g>
</g>
</g>
</svg>
</mat-icon>
</div>
<path d="M5.32722498,13.6460332 C4.21304566,10.5852611 4.22627578,7.22758116 5.3645406,4.17568425 C4.07298567,3.95802358 2.75701252,3.92542943 1.45626578,4.07888372 C0.536506305,7.22972054 0.525090565,10.5762272 1.42333236,13.7332656 C2.72217338,13.8896343 4.03666286,13.8602685 5.32722498,13.6460524"></path>
<path d="M7.25919787,0.751096346 C6.46424104,1.79423541 5.8259601,2.94792723 5.36457887,4.17561728 C8.13918123,4.64457901 10.6977043,5.96938232 12.6820178,7.96458738 C12.9850078,8.2673988 13.2715025,8.58184824 13.5415017,8.90793568 C14.3788161,7.90142183 15.0653154,6.77849641 15.5794509,5.57440744 C13.3171586,3.21319806 10.4324457,1.54091896 7.25919787,0.751096346"></path>
</div>
