<div xmlns:bean="http://ts.thomson.com/ua/bean" style="background-color: #000000; padding: 10px; color: #FFFFFF; font-family:  arial, sans-serif;">
<div style="height: 25px; width: 100px;">
<mat-icon data-mat-icon-name="clarivate" data-mat-icon-type="svg" aria-hidden="true" class="mat-icon notranslate clarivate-logo mat-icon-no-color" svgicon="clarivate" role="img">
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" focusable="false" preserveAspectRatio="xMidYMid meet" version="1.1" viewBox="0 0 92 18" height="100%" width="100%">
<title>Clarivate</title>
<g fill-rule="evenodd" fill="none" stroke-width="1" stroke="none">
<g fill-rule="nonzero" fill="#FFFFFF" transform="translate(-19.000000, -8.000000)">
<g transform="translate(1.000000, 1.000000)">
<g transform="translate(18.000000, 7.000000)">
<path d="M26.6812194,13.2997252 C28.2949641,13.3845068 29.7591419,12.3596036 30.2318671,10.8143235 L33.0910469,10.8143235 C32.3628779,13.742285 29.6973379,15.7705498 26.6812194,15.6916944 C24.8651813,15.7485215 23.1057028,15.0559504 21.8156554,13.7764967 C20.5256081,12.4970431 19.8185526,10.7433351 19.8604035,8.9268901 C19.8185367,7.11043445 20.5255848,5.35671042 21.8156337,4.07724261 C23.1056826,2.7977748 24.8651702,2.10519376 26.6812194,2.16201887 C29.6973451,2.0831702 32.3628861,4.11144761 33.0910469,7.03941847 L30.2318671,7.03941847 C29.7591419,5.49413828 28.2949641,4.46923515 26.6812194,4.55401674 C25.5407067,4.54133389 24.4470406,5.00703241 23.6657805,5.83803342 C22.8845203,6.66903442 22.487126,7.78932829 22.5700912,8.9268901 C22.4871313,10.064447 22.8845287,11.184734 23.6657888,12.0157277 C24.447049,12.8467215 25.540712,13.312413 26.6812194,13.2997252 L26.6812194,13.2997252 Z"/>
<polygon points="36.8477342 15.4674945 34.4371264 15.4674945 34.4371264 2.38628571 36.8477342 2.38628571"/>
<path d="M43.5380316,9.76781183 C43.5380316,9.15114738 43.0522397,8.75870193 42.1178663,8.75870193 C41.2070757,8.82048057 40.3236107,9.0957548 39.5389268,9.5622602 L38.5858286,7.73092465 C39.6930919,7.06514423 40.9567617,6.70409618 42.2486241,6.68441302 C44.1921648,6.68441302 45.9674886,7.58137515 45.9674886,9.84255787 L45.9674886,15.4674945 L43.5380316,15.4674945 L43.5380316,14.6639362 C42.9494098,15.3673061 42.0613957,15.7488608 41.146005,15.6917231 C40.3900371,15.7421039 39.645831,15.484584 39.0826798,14.9777443 C38.5195286,14.4709046 38.1853154,13.7578403 38.1560675,13.0007601 C38.1560675,11.3749188 39.3519468,10.3658089 41.2581433,10.3658089 L42.9400537,10.3658089 C43.3511665,10.3658089 43.5379934,10.1789247 43.5379934,9.86128266 L43.5380316,9.76781183 Z M43.5380316,12.4027631 L43.5380316,11.786089 C43.321275,11.9402969 43.0543042,12.0070182 42.790485,11.9729159 L41.8748173,11.9729159 C41.1086607,11.9729159 40.6602036,12.3092922 40.6602036,12.9072893 C40.6602036,13.486619 41.1272707,13.9164566 41.8748173,13.9164566 C42.303867,13.9699481 42.7348632,13.8332333 43.0546312,13.5422121 C43.3743992,13.2511908 43.5509896,12.8349402 43.5380316,12.4027631 L43.5380316,12.4027631 Z"/>
<path d="M52.6579104,9.3006299 L51.6301236,9.3006299 C50.3966703,9.3006299 49.8361037,9.8799596 49.8361037,11.3749188 L49.8361037,15.4674945 L47.4066179,15.4674945 L47.4066179,6.90863203 L49.7612715,6.90863203 L49.7612715,8.27284306 C50.1001881,7.34759017 51.0007831,6.74968637 51.9850429,6.79648425 C52.2115562,6.79362484 52.4375557,6.81874947 52.6579104,6.87128771 L52.6579104,9.3006299 Z"/>
<path d="M57.4232866,15.4674945 L54.9939445,15.4674945 L54.9939445,8.96425355 L53.7979312,8.96425355 L53.7979312,6.90863203 L57.4232866,6.90863203 L57.4232866,15.4674945 Z M56.0964104,2.38628347 C56.5129556,2.38558064 56.9126431,2.55074046 57.2071855,2.84528284 C57.5017278,3.13982521 57.6668877,3.53951271 57.6661848,3.95605794 C57.6609778,4.814152 56.9638874,5.50702694 56.1057775,5.50702694 C55.2476677,5.50702694 54.5505773,4.814152 54.5453585,3.95605794 C54.5417726,3.09491157 55.2352828,2.39302469 56.0964104,2.38628347 Z"/>
<polygon points="62.8802456 12.8512776 64.8236715 6.90863203 67.3838767 6.90863203 64.33787 15.4674945 61.3292077 15.4674945 58.4139588 6.90863203 61.0488526 6.90863203"/>
<path d="M72.8783522,9.76781183 C72.8783522,9.15114738 72.3924359,8.75870193 71.4580625,8.75870193 C70.5472784,8.82049943 69.6638208,9.09577238 68.8791325,9.5622602 L67.9261491,7.73092465 C69.0333627,7.06513461 70.2969953,6.70408494 71.5888202,6.68441302 C73.5323609,6.68441302 75.3076848,7.58137515 75.3076848,9.84255787 L75.3076848,15.4674945 L72.8783522,15.4674945 L72.8783522,14.6639362 C72.2896971,15.3672586 71.4017035,15.7487939 70.4863256,15.6916944 C69.7303436,15.7420941 68.986117,15.4845835 68.4229464,14.9777426 C67.8597758,14.4709017 67.5255471,13.7578265 67.4962924,13.0007314 C67.4962924,11.3748901 68.6923056,10.3657802 70.5984925,10.3657802 L72.2802785,10.3657802 C72.6913914,10.3657802 72.8783426,10.1788959 72.8783426,9.86125395 L72.8783522,9.76781183 Z M72.8783522,12.4027631 L72.8783522,11.786089 C72.6615874,11.9402794 72.3946236,12.006999 72.1308056,11.9729159 L71.2151378,11.9729159 C70.4488664,11.9729159 70.0004093,12.3092922 70.0004093,12.9072893 C70.0004093,13.486619 70.4676008,13.9164566 71.2151378,13.9164566 C71.6441833,13.9699279 72.0751679,13.8332061 72.394931,13.5421893 C72.7146941,13.2511724 72.8912904,12.8349341 72.8783522,12.4027631 L72.8783522,12.4027631 Z"/>
<path d="M79.9985637,3.65701635 L79.9985637,6.90863203 L81.8485762,6.90863203 L81.8485762,8.96425355 L79.9985637,8.96425355 L79.9985637,12.8325433 C79.9985637,13.3745286 80.4282865,13.4492747 80.8020646,13.4492747 C81.418729,13.4492747 81.9232553,13.3932056 81.9232553,13.3932056 L81.9232553,15.4674945 C81.9232553,15.4674945 80.7459859,15.5796423 80.0357932,15.5796423 C78.8211795,15.5796423 77.5691354,15.3740811 77.5691354,13.6548263 L77.5691354,8.96425355 L76.22363,8.96425355 L76.22363,6.90863203 L77.5691163,6.90863203 L77.5691163,3.65701635 L79.9985637,3.65701635 Z"/>
<path d="M91.5475359,11.8981794 L85.1751771,11.8981794 C85.3031552,12.7204351 85.9194738,13.3815958 86.7307322,13.5669136 C87.5419905,13.7522314 88.3842592,13.4242593 88.856573,12.7391298 L91.3793191,13.0007601 C90.4933306,14.9536735 88.3725856,16.0321989 86.2725384,15.5978628 C84.1724912,15.1635268 82.6535603,13.3322361 82.6149337,11.188092 C82.567807,9.98960603 83.0193435,8.82515843 83.8621298,7.97175307 C84.7049161,7.1183477 85.8636189,6.65226701 87.0626009,6.68438432 C89.4171397,6.68438432 91.5475359,8.21674525 91.5475359,11.8047373 L91.5475359,11.8981794 Z M85.1189837,10.2723955 L88.9499864,10.2723955 C88.8059978,9.34848922 87.9973156,8.67588427 87.0626392,8.70263282 C86.1147409,8.67034807 85.2869107,9.33893202 85.1189837,10.2723955 Z"/>
<path d="M12.6820274,9.85132226 C10.6885049,11.8556057 8.11575776,13.1830474 5.32707189,13.6462054 C5.77780891,14.8758828 6.40559556,16.0331959 7.19058498,17.0815575 C10.3903106,16.2989026 13.3004311,14.619851 15.5794509,12.2414352 C15.0651988,11.0374085 14.3787129,9.91449973 13.5415112,8.90790698 C13.271174,9.23328 12.9846793,9.54772625 12.6820274,9.85124571"/>
<path d="M5.32722498,13.6460332 C4.21304566,10.5852611 4.22627578,7.22758116 5.3645406,4.17568425 C4.07298567,3.95802358 2.75701252,3.92542943 1.45626578,4.07888372 C0.536506305,7.22972054 0.525090565,10.5762272 1.42333236,13.7332656 C2.72217338,13.8896343 4.03666286,13.8602685 5.32722498,13.6460524"/>
<path d="M7.25919787,0.751096346 C6.46424104,1.79423541 5.8259601,2.94792723 5.36457887,4.17561728 C8.13918123,4.64457901 10.6977043,5.96938232 12.6820178,7.96458738 C12.9850078,8.2673988 13.2715025,8.58184824 13.5415017,8.90793568 C14.3788161,7.90142183 15.0653154,6.77849641 15.5794509,5.57440744 C13.3171586,3.21319806 10.4324457,1.54091896 7.25919787,0.751096346"/>
</g>
</g>
</g>
</g>
</svg>
</mat-icon>
</div>
<path d="M5.32722498,13.6460332 C4.21304566,10.5852611 4.22627578,7.22758116 5.3645406,4.17568425 C4.07298567,3.95802358 2.75701252,3.92542943 1.45626578,4.07888372 C0.536506305,7.22972054 0.525090565,10.5762272 1.42333236,13.7332656 C2.72217338,13.8896343 4.03666286,13.8602685 5.32722498,13.6460524"></path>
<path d="M7.25919787,0.751096346 C6.46424104,1.79423541 5.8259601,2.94792723 5.36457887,4.17561728 C8.13918123,4.64457901 10.6977043,5.96938232 12.6820178,7.96458738 C12.9850078,8.2673988 13.2715025,8.58184824 13.5415017,8.90793568 C14.3788161,7.90142183 15.0653154,6.77849641 15.5794509,5.57440744 C13.3171586,3.21319806 10.4324457,1.54091896 7.25919787,0.751096346"></path>
</div><div style="width: 180px; height: 40px; padding: 10px;">
<mat-icon data-mat-icon-name="wos" data-mat-icon-type="svg" aria-hidden="true" style="width: 180px;" class="mat-icon notranslate mat-icon-no-color" svgicon="wos" role="img">
<svg xmlns="http://www.w3.org/2000/svg" focusable="false" preserveAspectRatio="xMidYMid meet" viewBox="0 0 1868 332" height="100%" width="100%">
<title>Web of Science</title>
<path d="M 108.92,75.00            C 108.92,75.00 147.34,75.00 147.34,75.00              147.34,75.00 182.84,208.12 182.84,208.12              182.84,208.12 183.11,208.12 183.11,208.12              183.11,208.12 212.80,75.00 212.80,75.00              212.80,75.00 250.42,75.00 250.42,75.00              250.42,75.00 205.11,261.00 205.11,261.00              205.11,261.00 162.44,261.00 162.44,261.00              162.44,261.00 127.47,127.34 127.47,127.34              127.47,127.34 93.28,261.00 93.28,261.00              93.28,261.00 50.61,261.00 50.61,261.00              50.61,261.00 4.77,75.00 4.77,75.00              4.77,75.00 44.52,75.00 44.52,75.00              44.52,75.00 73.14,208.12 73.14,208.12              73.14,208.12 73.41,208.12 73.41,208.12              73.41,208.12 108.92,75.00 108.92,75.00 Z            M 379.00,210.00            C 379.00,210.00 288.41,210.00 288.41,210.00              289.29,216.78 292.30,222.48 297.44,227.09              302.57,231.70 308.86,234.00 316.30,234.00              327.80,234.00 335.95,230.00 340.73,222.00              340.73,222.00 376.61,225.58 376.61,225.58              371.30,238.39 363.32,248.00 352.69,254.41              342.06,260.80 329.75,264.00 315.77,264.00              297.52,264.00 282.32,257.98 270.19,245.94              258.06,233.90 252.00,218.59 252.00,200.00              252.00,181.42 257.98,166.11 269.94,154.06              281.89,142.02 296.99,136.00 315.23,136.00              333.47,136.00 348.66,142.21 360.80,154.64              372.93,167.06 379.00,185.51 379.00,210.00 Z            M 333.30,171.28            C 328.34,167.09 322.32,165.00 315.23,165.00              308.15,165.00 302.12,167.01 297.16,171.03              292.20,175.04 289.02,180.37 287.61,187.00              287.61,187.00 342.06,187.00 342.06,187.00              341.19,180.71 338.27,175.47 333.30,171.28 Z            M 433.72,245.31            C 433.72,245.31 433.72,261.00 433.72,261.00              433.72,261.00 399.00,261.00 399.00,261.00              399.00,261.00 399.00,75.00 399.00,75.00              399.00,75.00 433.45,75.00 433.45,75.00              433.45,75.00 433.45,134.94 433.45,134.94              433.45,137.95 433.45,141.85 433.45,146.64              433.45,151.42 433.45,154.17 433.45,154.88              436.43,149.39 441.34,144.87 448.17,141.33              455.02,137.78 462.82,136.00 471.59,136.00              488.26,136.00 502.16,142.07 513.30,154.20              524.43,166.33 530.00,181.59 530.00,200.00              530.00,218.41 524.37,233.68 513.11,245.81              501.86,257.94 487.88,264.00 471.17,264.00              462.91,264.00 455.39,262.22 448.62,258.66              441.85,255.09 436.88,250.65 433.72,245.31 Z            M 485.39,176.23            C 479.65,169.98 472.34,166.86 463.45,166.86              454.58,166.86 447.27,169.98 441.53,176.23              435.79,182.48 432.92,190.37 432.92,199.88              432.92,209.20 435.79,217.07 441.53,223.50              447.27,229.93 454.58,233.14 463.45,233.14              472.34,233.14 479.65,229.93 485.39,223.50              491.13,217.07 494.00,209.20 494.00,199.88              494.00,190.37 491.13,182.48 485.39,176.23 Z            M 707.69,245.81            C 695.19,257.94 679.86,264.00 661.72,264.00              643.58,264.00 628.26,257.94 615.75,245.81              603.25,233.68 597.00,218.41 597.00,200.00              597.00,181.59 603.25,166.33 615.75,154.20              628.26,142.07 643.58,136.00 661.72,136.00              679.86,136.00 695.19,142.07 707.69,154.20              720.20,166.33 726.45,181.59 726.45,200.00              726.45,218.41 720.20,233.68 707.69,245.81 Z            M 661.84,166.86            C 652.85,166.86 645.54,170.00 639.91,176.28              634.27,182.55 631.45,190.46 631.45,200.00              631.45,209.54 634.27,217.45 639.91,223.73              645.54,230.01 652.85,233.14 661.84,233.14              670.66,233.14 677.89,230.01 683.53,223.73              689.18,217.45 692.00,209.54 692.00,200.00              692.00,190.46 689.18,182.55 683.53,176.28              677.89,170.00 670.66,166.86 661.84,166.86 Z            M 757.00,139.00            C 757.00,139.00 757.00,104.34 757.00,104.34              757.00,93.15 760.17,84.98 766.52,79.83              772.86,74.67 781.14,72.09 791.36,72.09              791.36,72.09 816.50,73.69 816.50,73.69              816.50,73.69 816.50,105.33 816.50,105.33              809.47,104.45 804.72,104.00 802.27,104.00              798.58,104.00 795.85,104.71 794.09,106.14              792.33,107.57 791.45,110.24 791.45,114.16              791.45,114.16 791.45,139.00 791.45,139.00              791.45,139.00 815.00,139.00 815.00,139.00              815.00,139.00 815.00,168.00 815.00,168.00              815.00,168.00 791.45,168.00 791.45,168.00              791.45,168.00 791.45,260.50 791.45,260.50              791.45,260.50 757.00,260.50 757.00,260.50              757.00,260.50 757.00,168.00 757.00,168.00              757.00,168.00 738.00,168.00 738.00,168.00              738.00,168.00 738.00,139.00 738.00,139.00              738.00,139.00 757.00,139.00 757.00,139.00 Z            M 1004.30,97.27            C 1004.30,97.27 983.11,124.06 983.11,124.06              969.92,111.36 956.73,105.00 943.53,105.00              937.11,105.00 931.59,106.60 926.95,109.80              922.32,112.98 920.00,117.24 920.00,122.56              920.00,131.06 926.74,138.42 940.22,144.62              940.22,144.62 955.66,151.80 955.66,151.80              971.60,159.25 983.48,167.31 991.28,176.00              999.09,184.68 1003.00,195.67 1003.00,208.97              1003.00,223.87 997.13,236.76 985.39,247.66              973.66,258.55 958.09,264.00 938.67,264.00              913.45,264.00 891.31,254.56 872.25,235.67              872.25,235.67 892.52,208.02 892.52,208.02              899.27,214.90 906.74,220.29 914.92,224.17              923.10,228.06 930.83,230.00 938.11,230.00              945.94,230.00 952.21,228.09 956.92,224.28              961.64,220.47 964.00,215.64 964.00,209.80              964.00,203.95 961.65,198.93 956.95,194.77              952.25,190.60 944.85,186.21 934.73,181.61              934.73,181.61 921.70,175.48 921.70,175.48              894.57,162.73 881.00,145.36 881.00,123.38              881.00,107.59 887.05,94.92 899.17,85.36              911.29,75.79 925.91,71.00 943.06,71.00              963.93,71.00 984.34,79.76 1004.30,97.27 Z            M 1082.92,166.86            C 1073.91,166.86 1066.58,170.00 1060.92,176.28              1055.27,182.55 1052.45,190.46 1052.45,200.00              1052.45,209.54 1055.27,217.45 1060.92,223.73              1066.58,230.01 1073.91,233.14 1082.92,233.14              1095.82,233.14 1104.74,226.95 1109.69,214.58              1109.69,214.58 1142.02,224.66 1142.02,224.66              1138.13,236.72 1130.75,246.29 1119.89,253.38              1109.02,260.46 1096.61,264.00 1082.66,264.00              1064.46,264.00 1049.13,257.94 1036.67,245.81              1024.22,233.68 1018.00,218.41 1018.00,200.00              1018.00,181.59 1024.22,166.33 1036.67,154.20              1049.13,142.07 1064.46,136.00 1082.66,136.00              1096.61,136.00 1109.02,139.59 1119.89,146.77              1130.75,153.95 1138.13,163.56 1142.02,175.61              1142.02,175.61 1109.69,185.42 1109.69,185.42              1107.39,179.77 1103.91,175.26 1099.22,171.91              1094.54,168.54 1089.11,166.86 1082.92,166.86 Z            M 1170.31,112.67            C 1166.10,108.45 1164.00,103.27 1164.00,97.12              1164.00,90.98 1166.15,85.76 1170.45,81.45              1174.75,77.15 1179.89,75.00 1185.86,75.00              1192.02,75.00 1197.24,77.15 1201.55,81.45              1205.85,85.76 1208.00,90.98 1208.00,97.12              1208.00,103.27 1205.85,108.45 1201.55,112.67              1197.24,116.89 1192.02,119.00 1185.86,119.00              1179.71,119.00 1174.53,116.89 1170.31,112.67 Z            M 1205.00,139.00            C 1205.00,139.00 1205.00,261.00 1205.00,261.00              1205.00,261.00 1170.55,261.00 1170.55,261.00              1170.55,261.00 1170.55,168.00 1170.55,168.00              1170.55,168.00 1153.50,168.00 1153.50,168.00              1153.50,168.00 1153.50,139.00 1153.50,139.00              1153.50,139.00 1205.00,139.00 1205.00,139.00 Z            M 1353.00,210.00            C 1353.00,210.00 1262.41,210.00 1262.41,210.00              1263.29,216.78 1266.30,222.48 1271.44,227.09              1276.57,231.70 1282.86,234.00 1290.30,234.00              1301.80,234.00 1309.95,230.00 1314.73,222.00              1314.73,222.00 1350.61,225.58 1350.61,225.58              1345.30,238.39 1337.32,248.00 1326.69,254.41              1316.06,260.80 1303.75,264.00 1289.77,264.00              1271.52,264.00 1256.32,257.98 1244.19,245.94              1232.06,233.90 1226.00,218.59 1226.00,200.00              1226.00,181.42 1231.98,166.11 1243.94,154.06              1255.89,142.02 1270.99,136.00 1289.23,136.00              1307.47,136.00 1322.66,142.21 1334.80,154.64              1346.93,167.06 1353.00,185.51 1353.00,210.00 Z            M 1307.30,171.28            C 1302.34,167.09 1296.32,165.00 1289.23,165.00              1282.15,165.00 1276.12,167.01 1271.16,171.03              1266.20,175.04 1263.02,180.37 1261.61,187.00              1261.61,187.00 1316.06,187.00 1316.06,187.00              1315.19,180.71 1312.27,175.47 1307.30,171.28 Z            M 1406.39,153.36            C 1415.92,141.79 1428.27,136.00 1443.45,136.00              1456.87,136.00 1467.72,140.04 1476.02,148.11              1484.30,156.17 1488.45,167.20 1488.45,181.20              1488.45,181.20 1488.45,261.00 1488.45,261.00              1488.45,261.00 1454.00,261.00 1454.00,261.00              1454.00,261.00 1454.00,189.86 1454.00,189.86              1454.00,182.57 1452.00,177.11 1448.00,173.47              1444.01,169.82 1439.09,168.00 1433.25,168.00              1423.50,168.00 1414.90,172.62 1407.45,181.86              1407.45,181.86 1407.45,261.00 1407.45,261.00              1407.45,261.00 1373.00,261.00 1373.00,261.00              1373.00,261.00 1373.00,139.00 1373.00,139.00              1373.00,139.00 1406.39,139.00 1406.39,139.00              1406.39,139.00 1406.39,153.36 1406.39,153.36 Z            M 1573.92,166.86            C 1564.91,166.86 1557.58,170.00 1551.92,176.28              1546.27,182.55 1543.45,190.46 1543.45,200.00              1543.45,209.54 1546.27,217.45 1551.92,223.73              1557.58,230.01 1564.91,233.14 1573.92,233.14              1586.82,233.14 1595.74,226.95 1600.69,214.58              1600.69,214.58 1633.02,224.66 1633.02,224.66              1629.13,236.72 1621.75,246.29 1610.89,253.38              1600.02,260.46 1587.61,264.00 1573.66,264.00              1555.46,264.00 1540.13,257.94 1527.67,245.81              1515.22,233.68 1509.00,218.41 1509.00,200.00              1509.00,181.59 1515.22,166.33 1527.67,154.20              1540.13,142.07 1555.46,136.00 1573.66,136.00              1587.61,136.00 1600.02,139.59 1610.89,146.77              1621.75,153.95 1629.13,163.56 1633.02,175.61              1633.02,175.61 1600.69,185.42 1600.69,185.42              1598.39,179.77 1594.91,175.26 1590.22,171.91              1585.54,168.54 1580.11,166.86 1573.92,166.86 Z            M 1773.00,210.00            C 1773.00,210.00 1682.41,210.00 1682.41,210.00              1683.29,216.78 1686.30,222.48 1691.44,227.09              1696.57,231.70 1702.86,234.00 1710.30,234.00              1721.80,234.00 1729.95,230.00 1734.73,222.00              1734.73,222.00 1770.61,225.58 1770.61,225.58              1765.30,238.39 1757.32,248.00 1746.69,254.41              1736.06,260.80 1723.75,264.00 1709.77,264.00              1691.52,264.00 1676.32,257.98 1664.19,245.94              1652.06,233.90 1646.00,218.59 1646.00,200.00              1646.00,181.42 1651.98,166.11 1663.94,154.06              1675.89,142.02 1690.99,136.00 1709.23,136.00              1727.47,136.00 1742.66,142.21 1754.80,154.64              1766.93,167.06 1773.00,185.51 1773.00,210.00 Z            M 1727.30,171.28            C 1722.34,167.09 1716.32,165.00 1709.23,165.00              1702.15,165.00 1696.12,167.01 1691.16,171.03              1686.20,175.04 1683.02,180.37 1681.61,187.00              1681.61,187.00 1736.06,187.00 1736.06,187.00              1735.19,180.71 1732.27,175.47 1727.30,171.28 Z" stroke-width="1" stroke="black" fill="black"/>
<path d="M 1769.00,76.00            C 1769.00,76.00 1800.00,76.00 1800.00,76.00              1800.00,76.00 1800.00,82.42 1800.00,82.42              1800.00,82.42 1788.00,82.42 1788.00,82.42              1788.00,82.42 1788.00,115.00 1788.00,115.00              1788.00,115.00 1780.84,115.00 1780.84,115.00              1780.84,115.00 1780.84,82.42 1780.84,82.42              1780.84,82.42 1769.00,82.42 1769.00,82.42              1769.00,82.42 1769.00,76.00 1769.00,76.00 Z            M 1827.27,106.64            C 1827.27,106.64 1818.95,106.64 1818.95,106.64              1818.95,106.64 1811.16,87.42 1811.16,87.42              1811.16,87.42 1811.16,115.00 1811.16,115.00              1811.16,115.00 1804.00,115.00 1804.00,115.00              1804.00,115.00 1804.00,76.00 1804.00,76.00              1804.00,76.00 1814.61,76.00 1814.61,76.00              1814.61,76.00 1823.25,98.11 1823.25,98.11              1823.25,98.11 1832.08,76.00 1832.08,76.00              1832.08,76.00 1842.00,76.00 1842.00,76.00              1842.00,76.00 1842.00,115.00 1842.00,115.00              1842.00,115.00 1834.84,115.00 1834.84,115.00              1834.84,115.00 1834.84,87.42 1834.84,87.42              1834.84,87.42 1827.27,106.64 1827.27,106.64 Z" stroke-width="1" stroke="black" fill="black"/>
</svg>
</mat-icon>
</div><table cellspacing="0" cellpadding="2" border="0">
<tr>
<td>37 record(s) printed from Clarivate Web of Science</td>
</tr>
</table><hr>
<table>
<tr>
<td><b>Record 1 of 37</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Cardiac Severity Classification Using Pre Trained Neural Networks</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Malleswari, PN (Malleswari, Pinjala N.); Bindu, CH (Bindu, Ch. Hima); Prasad, KS (Prasad, K. Satya)</td>
</tr>

<tr xmlns:bean="http://ts.thomson.com/ua/bean" xmlns:exsl="http://exslt.org/common" xmlns:set="http://exslt.org/sets">
<td><b>Source:</b> 
<value>INTERDISCIPLINARY SCIENCES-COMPUTATIONAL LIFE SCIENCES</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>13</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>3</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>443-450</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1007/s12539-021-00416-9</value>&nbsp;&nbsp;<b>Early Access Date:</b> 
<value>JAN 2021</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2021 SEP</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>5</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>7</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
0</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
10</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>28</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Electrocardiogram (ECG) is the most effective instrument for making decisions about various forms of heart disease. As a result, several researchers have focused on the ECG signal to extract the features of heartbeats with high precision and efficiency. This article offers a hybrid approach to classifying different cardiac conditions using the Feed Forward Back Propagation Neural Network (FFBPNN), by providing a pre-processed ECG signal as an excitation. The modified ECG signal is obtained through the combination of EMD (Empirical Mode Decomposition) and DWT (Discrete Wavelet Transform). In this proposed method, the input signal is first decomposed into the Intrinsic Mode Functions (IMF's) and the first three IMF's are combined to obtain a modified partially denoted ECG sample and then DWT is used to obtain an improved denoised signal. This pre-processed signal is classified using the Neural Network architecture. For the EMD approach, the ECG-based EMD-DWT signal provides improved classification accuracy of 67, 0762 percent, 90, 4305 percent for the DWT approach, and 95,0797 percent for the proposed technique. The methodology is applied to the MIT-BIH database and, in terms of classification accuracy, is found to be higher than the different methodologies.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000609955900001</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>33481208</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
ECG; EMD-DWT; ECG classification; FFBPNN; MIT-BIH database; Accuracy</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
DEEP LEARNING APPROACH; ECG SIGNAL; FEATURES</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Malleswari, Pinjala N.] JNTUK, Univ Coll Engn, Kakinada, AP, India. <br>
[Bindu, Ch. Hima] Vasavi Coll Engn, ECE Dept, Hyderabad, Telangana, India. <br>
[Prasad, K. Satya] Vignans Fdn Sci Technol &amp; Res Univ, Guntur Tenali Rd, Vadlamudi, AP, India. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Malleswari, PN (corresponding author), JNTUK, Univ Coll Engn, Kakinada, AP, India.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
pinjalamalleswari@gmail.com; hb.muvvala@gmail.com; prasad_kodati@yahoo.co.in</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Jawaharlal Nehru Technological University - Kakinada; Vignan's Foundation for Science, Technology &amp; Research (VFSTR)</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>N MALLESWARI, PINJALA</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-8324-0581&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>BINDU, HIMA</display_name>&nbsp;</td><td>F-6086-2016&nbsp;</td><td>0000-0002-1268-7802&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>CHINNI, HIMABINDU</display_name>&nbsp;</td><td>F-6086-2016&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Malleswari, P N</display_name>&nbsp;</td><td>IXW-8974-2023&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>SPRINGER HEIDELBERG</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>TIERGARTENSTRASSE 17, D-69121 HEIDELBERG, GERMANY</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Mathematical &amp; Computational Biology</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Mathematical &amp; Computational Biology</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>UC5QS</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>1913-2751</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1867-1462</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>INTERDISCIP SCI</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Interdiscip. Sci.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>8</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 2 of 37</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Automated detection of diabetic subject using pre-trained 2D-CNN models with frequency spectrum images extracted from heart rate signals</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Yildirim, O (Yildirim, Ozal); Talo, M (Talo, Muhammed); Ay, B (Ay, Betul); Baloglu, UB (Baloglu, Ulas Baran); Aydin, G (Aydin, Galip); Acharya, UR (Acharya, U. Rajendra)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>COMPUTERS IN BIOLOGY AND MEDICINE</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>113</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>103387</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1016/j.compbiomed.2019.103387</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2019 OCT</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>102</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>115</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
1</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
51</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>52</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
In this study, a deep-transfer learning approach is proposed for the automated diagnosis of diabetes mellitus (DM), using heart rate (HR) signals obtained from electrocardiogram (ECG) data. Recent progress in deep learning has contributed significantly to improvement in the quality of healthcare. In order for deep learning models to perform well, large datasets are required for training. However, a difficulty in the biomedical field is the lack of clinical data with expert annotation. A recent, commonly implemented technique to train deep learning models using small datasets is to transfer the weighting, developed from a large dataset, to the current model. This deep learning transfer strategy is generally employed for two-dimensional signals. Herein, the weighting of models pre-trained using two-dimensional large image data was applied to one-dimensional HR signals. The one-dimensional HR signals were then converted into frequency spectrum images, which were utilized for application to well-known pre-trained models, specifically: AlexNet, VggNet, ResNet, and DenseNet. The DenseNet pre-trained model yielded the highest classification average accuracy of 97.62%, and sensitivity of 100%, to detect DM subjects via HR signal recordings. In the future, we intend to further test this developed model by utilizing additional data along with cloud-based storage to diagnose DM via heart signal analysis.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000496898100001</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>31421276</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Diabetes mellitus; Heart rate signals; Deep learning; Transfer learning</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
DISCRETE WAVELET TRANSFORM; COMPUTER-AIDED DIAGNOSIS; CLASSIFICATION; MELLITUS; RETINOPATHY; FEATURES; INDEX</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Yildirim, Ozal; Talo, Muhammed] Munzur Univ, Dept Comp Engn, Tunceli, Turkey. <br>
[Ay, Betul; Aydin, Galip] Firat Univ, Dept Comp Engn, Elazig, Turkey. <br>
[Baloglu, Ulas Baran] Univ Bristol, Dept Comp Sci, Bristol, Avon, England. <br>
[Acharya, U. Rajendra] Ngee Ann Polytech, Dept Elect &amp; Comp Engn, Singapore, Singapore. <br>
[Acharya, U. Rajendra] Singapore Sch Social Sci, Sch Sci &amp; Technol, Dept Biomed Engn, Singapore, Singapore. <br>
[Acharya, U. Rajendra] Taylors Univ, Fac Hlth &amp; Med Sci, Sch Med, Subang Jaya 47500, Malaysia. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Yildirim, O (corresponding author), Munzur Univ, Dept Comp Engn, Tunceli, Turkey.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
oyildirim@munzur.edu.tr</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Munzur University; Firat University; University of Bristol; Taylor's University</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Baloglu, Ulas Baran</display_name>&nbsp;</td><td>P-7444-2015&nbsp;</td><td>0000-0002-2045-9922&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Acharya, U Rajendra</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-2689-8552&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Aydin, Galip</display_name>&nbsp;</td><td>W-1473-2018&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>YILDIRIM, &Ouml;zal</display_name>&nbsp;</td><td>AAV-5584-2020&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Acharya, Rajendra</display_name>&nbsp;</td><td>E-3791-2010&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>BALOGLU, Ulas</display_name>&nbsp;</td><td>P-7444-2015&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Talo, Muhammed</display_name>&nbsp;</td><td>Y-6047-2019&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>AY, BET&Uuml;L</display_name>&nbsp;</td><td>JAN-6412-2023&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>PERGAMON-ELSEVIER SCIENCE LTD</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>THE BOULEVARD, LANGFORD LANE, KIDLINGTON, OXFORD OX5 1GB, ENGLAND</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Biology; Computer Science, Interdisciplinary Applications; Engineering, Biomedical; Mathematical &amp; Computational Biology</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Life Sciences &amp; Biomedicine - Other Topics; Computer Science; Engineering; Mathematical &amp; Computational Biology</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>JN4VW</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0010-4825</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1879-0534</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>COMPUT BIOL MED</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Comput. Biol. Med.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>10</value>
</td>
</tr>

<tr>
<td>
<b>Open Access:</b>

<value>Green Submitted</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 3 of 37</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>A Generalisable Heartbeat Classifier Leveraging Self-Supervised Learning for ECG Analysis During Magnetic Resonance Imaging</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Aublin, PG (Aublin, Pierre Gabriel); Felblinger, J (Felblinger, Jacques); Oster, J (Oster, Julien)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE JOURNAL OF BIOMEDICAL AND HEALTH INFORMATICS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>28</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>9</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>5147-5155</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/JBHI.2024.3411792</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2024 SEP</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>1</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>1</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
0</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
24</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>44</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Electrocardiogram (ECG) is acquired during Magnetic Resonance Imaging (MRI) to monitor patients and synchronize image acquisition with the heart motion. ECG signals are highly distorted during MRI due to the complex electromagnetic environment. Automated ECG analysis is therefore complicated in this context and there is no reference technique in MRI to classify pathological heartbeats. Imaging arrhythmic patients is hence difficult in MRI. Deep Learning based heartbeat classifier have been suggested but require large databases whereas existing annotated sets of ECG in MRI are very small. We proposed a Siamese network to leverage a large database of unannotated ECGs outside MRI. This was used to develop an efficient representation of ECG signals, further used to develop a heartbeat classifier. We extensively tested several data augmentations and self-supervised learning (SSL) techniques and assessed the generalization of the obtained classifier to ECG signals acquired in MRI. These augmentations included random noises and a model simulating MRI specific artefacts. SSL pretraining improved the generalizability of heartbeat classifiers in MRI (F1=0.75) compared to Deep Learning not relying on SSL (F1=0.46) and another classical machine learning approach (F1=0.40). These promising results seem to indicate that the use of SSL techniques can learn efficient ECG signal representation, and are useful for the development of Deep Learning models even when only scarce annotated medical data are available.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001309075700034</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>38857140</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Electrocardiogram; magnethohydrodynamic effect; deep learning; representation learning; Siamese network; Electrocardiogram; magnethohydrodynamic effect; deep learning; representation learning; Siamese network</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
ELECTROCARDIOGRAM; SIGNALS</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Aublin, Pierre Gabriel; Felblinger, Jacques; Oster, Julien] Univ Lorraine, IADI, U1254, Inserm, F-54000 Nancy, France. <br>
[Aublin, Pierre Gabriel] Univ Augsburg, Chair Diagnost Sensing, D-86159 Augsburg, Germany. <br>
[Felblinger, Jacques; Oster, Julien] Univ Lorraine, CHRU Nancy, Inserm, CIC IT1433, F-54035 Nancy, France. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Oster, J (corresponding author), Univ Lorraine, IADI, U1254, Inserm, F-54000 Nancy, France.<br>Oster, J (corresponding author), Univ Lorraine, CHRU Nancy, Inserm, CIC IT1433, F-54035 Nancy, France.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
julien.oster@inserm.fr</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Universite de Lorraine; Institut National de la Sante et de la Recherche Medicale (Inserm); University of Augsburg; CHU de Nancy; Universite de Lorraine; Institut National de la Sante et de la Recherche Medicale (Inserm)</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Felblinger, Jacques</display_name>&nbsp;</td><td>ABG-2033-2021&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Felblinger, jacques</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-9245-6595&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>oster, Julien</display_name>&nbsp;</td><td>F-3597-2017&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Aublin, Pierre G.</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-8082-7811&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Information Systems; Computer Science, Interdisciplinary Applications; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>F3T3E</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2168-2194</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>2168-2208</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE J BIOMED HEALTH</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE J. Biomed. Health Inform.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>9</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>ERA-CVD Joint Transnational Call 2019, MEIDIC-VTACH</grant_agency>&nbsp;</td><td>
<div>ANR-19-ECVD-0004&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Agence Nationale de la Recherche (ANR)</grant_agency>&nbsp;</td><td>
<div>ANR-19-ECVD-0004&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported by ERA-CVD Joint Transnational Call 2019, MEIDIC-VTACH under Grant ANR-19-ECVD-0004.</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 4 of 37</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>ECG Biometric Authentication Using Self-Supervised Learning for IoT Edge Sensors</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Wang, GX (Wang, Guoxin); Shanker, S (Shanker, Shreejith); Nag, A (Nag, Avishek); Lian, Y (Lian, Yong); John, D (John, Deepu)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE JOURNAL OF BIOMEDICAL AND HEALTH INFORMATICS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>28</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>11</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>6606-6618</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/JBHI.2024.3455803</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2024 NOV</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>10</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>11</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
6</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
31</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>43</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Wearable Internet of Things (IoT) devices are gaining ground for continuous physiological data acquisition and health monitoring. These physiological signals can be used for security applications to achieve continuous authentication and user convenience due to passive data acquisition. This paper investigates an electrocardiogram (ECG) based biometric user authentication system using features derived from the Convolutional Neural Network (CNN) and self-supervised contrastive learning. Contrastive learning enables us to use large unlabeled datasets to train the model and establish its generalizability. We propose approaches enabling the CNN encoder to extract appropriate features that distinguish the user from other subjects. When evaluated using the PTB ECG database with 290 subjects, the proposed technique achieved an authentication accuracy of 99.15%. To test its generalizability, we applied the model to two new datasets, the MIT-BIH Arrhythmia Database and the ECG-ID Database, achieving over 98.5% accuracy without any modifications. Furthermore, we show that repeating the authentication step three times can increase accuracy to nearly 100% for both PTBDB and ECGIDDB. This paper also presents model optimizations for embedded device deployment, which makes the system more relevant to real-world scenarios. To deploy our model in IoT edge sensors, we optimized the model complexity by applying quantization and pruning. The optimized model achieves 98.67% accuracy on PTBDB, with 0.48% accuracy loss and 62.6% CPU cycles compared to the unoptimized model. An accuracy-vs-time-complexity tradeoff analysis is performed, and results are presented for different optimization levels.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001359232100010</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>39250357</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Feature extraction; Electrocardiography; Authentication; Accuracy; Training; Contrastive learning; Biological system modeling; electrocardiogram authentication; IoT devices</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
FRAMEWORK</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Wang, Guoxin; John, Deepu] Univ Coll Dublin, Sch Elect &amp; Elect Engn, Dublin, Ireland. <br>
[Shanker, Shreejith] Trinity Coll Dublin, Dept Elect &amp; Elect Engn, Dublin, Ireland. <br>
[Nag, Avishek] Univ Coll Dublin, Sch Comp Sci, Dublin, Ireland. <br>
[Lian, Yong] York Univ, Dept Elect Engn &amp; Comp Sci, Toronto, ON M3J 1P3, Canada. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
John, D (corresponding author), Univ Coll Dublin, Sch Elect &amp; Elect Engn, Dublin, Ireland.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
guoxin.wang@ucdconnect.ie; shankers@tcd.ie; avishek.nag@ucd.ie; plian@yorku.ca; deepu.john@ucd.ie</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
University College Dublin; Trinity College Dublin; University College Dublin; York University - Canada</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Wang, Guoxin</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-3619-3399&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Lian, Yong</display_name>&nbsp;</td><td>MSY-3562-2025&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Shanker, Shreejith</display_name>&nbsp;</td><td>HGT-9976-2022&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Nag, Avishek</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-1702-1492&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Lian, Yong</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-5289-5219&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>John, Deepu</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-6139-1100&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>John, Deepu</display_name>&nbsp;</td><td>AAL-7045-2020&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Nag, Avishek</display_name>&nbsp;</td><td>T-9268-2019&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Wang, Guoxin</display_name>&nbsp;</td><td>LKM-8856-2024&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Information Systems; Computer Science, Interdisciplinary Applications; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>M7I4Z</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2168-2194</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>2168-2208</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE J BIOMED HEALTH</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE J. Biomed. Health Inform.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>13</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>China Scholarship Council</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>JEDAI Project through the Horizon 2020 FET Chist-Era Program</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Microelectronic Circuit Centre Ireland</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported in part by the China Scholarship Council, in part by the JEDAI Project through the Horizon 2020 FET Chist-Era Program and, in part by Microelectronic Circuit Centre Ireland.</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>Green Submitted</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 5 of 37</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>In-Distribution and Out-of-Distribution Self-Supervised ECG Representation Learning for Arrhythmia Detection</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Soltanieh, S (Soltanieh, Sahar); Hashemi, J (Hashemi, Javad); Etemad, A (Etemad, Ali)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE JOURNAL OF BIOMEDICAL AND HEALTH INFORMATICS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>28</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>2</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>789-800</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/JBHI.2023.3331626</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2024 FEB</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>4</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>5</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
7</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
41</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>44</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
This paper presents a systematic investigation into the effectiveness of Self-Supervised Learning (SSL) methods for Electrocardiogram (ECG) arrhythmia detection. We begin by conducting a novel analysis of the data distributions on three popular ECG-based arrhythmia datasets: PTB-XL, Chapman, and Ribeiro. To the best of our knowledge, our study is the first to quantitatively explore and characterize these distributions in the area. We then perform a comprehensive set of experiments using different augmentations and parameters to evaluate the effectiveness of various SSL methods, namely SimCRL, BYOL, and SwAV, for ECG representation learning, where we observe the best performance achieved by SwAV. Furthermore, our analysis shows that SSL methods achieve highly competitive results to those achieved by supervised state-of-the-art methods. To further assess the performance of these methods on both In-Distribution (ID) and Out-of-Distribution (OOD) ECG data, we conduct cross-dataset training and testing experiments. Our comprehensive experiments show almost identical results when comparing ID and OOD schemes, indicating that SSL techniques can learn highly effective representations that generalize well across different OOD datasets. This finding can have major implications for ECG-based arrhythmia detection. Lastly, to further analyze our results, we perform detailed per-disease studies on the performance of the SSL methods on the three datasets.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001203362500009</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>37948139</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Arrhythmia detection; contrastive learning; electrocardiogram; self-supervised learning; in-distribution; out-of-distribution</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Soltanieh, Sahar; Etemad, Ali] Queens Univ, Dept Elect &amp; Comp Engn, Kingston, ON K7L 3N6, Canada. <br>
[Hashemi, Javad] Queens Univ, Sch Comp, Kingston, ON K7L 3N6, Canada. <br>
[Etemad, Ali] Queens Univ, Ingenu Labs Res Inst, Kingston, ON K7L 3N6, Canada. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Soltanieh, S (corresponding author), Queens Univ, Dept Elect &amp; Comp Engn, Kingston, ON K7L 3N6, Canada.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
sahar.soltanieh@queensu.ca; javad.hashemi@queensu.ca; ali.etemad@queensu.ca</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Queens University - Canada; Queens University - Canada; Ingenuity Labs Research Institute; Queens University - Canada</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Hashemi, Javad</display_name>&nbsp;</td><td>D-5827-2019&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Soltanieh, Sahar</display_name>&nbsp;</td><td>LCE-1157-2024&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Soltanieh, Sahar</display_name>&nbsp;</td><td>&nbsp;</td><td>0009-0004-0724-1816&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Etemad, Ali</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-7128-0220&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Information Systems; Computer Science, Interdisciplinary Applications; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>NW0A0</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2168-2194</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>2168-2208</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE J BIOMED HEALTH</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE J. Biomed. Health Inform.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>12</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Natural Sciences and Engineering Research Council of Canada</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>No Statement Available</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 6 of 37</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Self-Supervised, Non-Contact Heartbeat Detection Based on Ballistocardiograms Utilizing Physiological Information Guidance</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Jiao, CZ (Jiao, Changzhe); Yang, AY (Yang, Aoyu); Zhao, HT (Zhao, Hantao); Yi, RH (Yi, Ruhan); Gou, SP (Gou, Shuiping); Sha, Y (Sha, Yu); Wen, WS (Wen, Wanshun); Jiao, LC (Jiao, Licheng); Skubic, M (Skubic, Marjorie)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE JOURNAL OF BIOMEDICAL AND HEALTH INFORMATICS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>29</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>4</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>2589-2602</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/JBHI.2024.3509875</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2025 APR</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>0</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>0</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
5</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
5</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>33</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Ballistocardiograms (BCG) is a passive, non-contact heart rate detection technology that requires no action on the part of the individual. However, during the BCG signal acquisition process, the surface pressure generated by cardiac contraction is easily disturbed by external factors, and as people's health deteriorates, the j-peak (the main peak of the BCG signal) is no longer prominent. Our aim is to establish a non-contact, self-supervised heart rate detection method based on physiological information, to improve the accuracy and robustness of BCG heart rate detection under wider and more adverse conditions. The algorithm is guided by the heart rate estimation based on BCG itself, thereby reconstructing a signal with physiological significance. We also propose a heartbeat mapping algorithm based on Bidirectional Long Short-Term Memory Network (BiLSTM) for extracting global deep features, achieving real-time heartbeat prediction, and eliminating local deviations brought about by reconstruction. To verify the effectiveness of the proposed method, this paper evaluated 40 young subjects and 4 elderly subjects. Compared with the existing state-of-the-art methods, beat-to-beat heart rate estimation and heartbeat detection both performed excellently, surpassing most methods using precise labels. The experimental results show that the proposed method achieves effective heartbeat detection, demonstrating robustness and effectiveness in the face of unavoidable noise and variations.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001459663700041</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>40030671</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Ballistocardiograms; heartbeat detection; non-contact; non-contact; physiological significance; physiological significance; self-supervised; self-supervised; self-supervised</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
BIDIRECTIONAL LSTM; SYSTEM</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Jiao, Changzhe; Yang, Aoyu; Zhao, Hantao; Gou, Shuiping; Sha, Yu; Jiao, Licheng] Xidian Univ, Sch Artificial Intelligence, Key Lab Intelligent Percept &amp; Image Understanding, Educ Minist China, Xian 710071, Peoples R China. <br>
[Yi, Ruhan; Skubic, Marjorie] Univ Missouri, Dept Elect Engn &amp; Comp Sci, Columbia, MO 65211 USA. <br>
[Wen, Wanshun] Zhejiang Prov Peoples Hosp, Dept Rehabil Med, Hangzhou 310024, Peoples R China. <br>
[Wen, Wanshun] Hangzhou Med Coll, Affiliated Peoples Hosp, Hangzhou 310053, Peoples R China. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Gou, SP (corresponding author), Xidian Univ, Sch Artificial Intelligence, Key Lab Intelligent Percept &amp; Image Understanding, Educ Minist China, Xian 710071, Peoples R China.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
shpgou@mail.xidian.edu.cn</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Xidian University; University of Missouri System; University of Missouri Columbia; Hangzhou Medical College; Zhejiang Provincial People's Hospital; Hangzhou Medical College</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Jiao, Changzhe</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-1392-8348&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Information Systems; Computer Science, Interdisciplinary Applications; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>0ZK6T</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2168-2194</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>2168-2208</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE J BIOMED HEALTH</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE J. Biomed. Health Inform.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>14</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Natural Science Foundation of China</grant_agency>&nbsp;</td><td>
<div>62372358&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Inter-discipline Guidance Project of Xidian University</grant_agency>&nbsp;</td><td>
<div>21103240003&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Natural Science Foundation of Zhejiang Province</grant_agency>&nbsp;</td><td>
<div>LGF20H020010&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported in part by the National Natural Science Foundation of China under Grant 62372358 in part by the Inter-discipline Guidance Project of Xidian University under Grant 21103240003, and in part by the Natural Science Foundation of Zhejiang Province under Grant LGF20H020010.</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 7 of 37</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Self-supervised representation learning from 12-lead ECG data</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Mehari, T (Mehari, Temesgen); Strodthoff, N (Strodthoff, Nils)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>COMPUTERS IN BIOLOGY AND MEDICINE</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>141</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>105114</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1016/j.compbiomed.2021.105114</value>&nbsp;&nbsp;<b>Early Access Date:</b> 
<value>DEC 2021</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2022 FEB</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>95</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>106</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
5</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
95</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>39</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Clinical 12-lead electrocardiography (ECG) is one of the most widely encountered kinds of biosignals. Despite the increased availability of public ECG datasets, label scarcity remains a central challenge in the field. Self-supervised learning represents a promising way to alleviate this issue. This would allow to train more powerful models given the same amount of labeled data and to incorporate or improve predictions about rare diseases, for which training datasets are inherently limited. In this work, we put forward the first comprehensive assessment of self-supervised representation learning from clinical 12-lead ECG data. To this end, we adapt state-of-the-art self-supervised methods based on instance discrimination and latent forecasting to the ECG domain. In a first step, we learn contrastive representations and evaluate their quality based on linear evaluation performance on a recently established, comprehensive, clinical ECG classification task. In a second step, we analyze the impact of self-supervised pretraining on finetuned ECG classifiers as compared to purely supervised performance. For the best-performing method, an adaptation of contrastive predictive coding, we find a linear evaluation performance only 0.5% below supervised performance. For the finetuned models, we find improvements in downstream performance of roughly 1% compared to supervised performance, label efficiency, as well as robustness against physiological noise. This work clearly establishes the feasibility of extracting discriminative representations from ECG data via self-supervised learning and the numerous advantages when finetuning such representations on downstream tasks as compared to purely supervised training. As first comprehensive assessment of its kind in the ECG domain carried out exclusively on publicly available datasets, we hope to establish a first step towards reproducible progress in the rapidly evolving field of representation learning for biosignals.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000747358300005</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>34973584</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Deep neural networks; Electrocardiography; Time series analysis; Unsupervised learning; Self-supervised learning</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
ELECTROCARDIOGRAMS</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Mehari, Temesgen] Phys Tech Bundesanstalt, Berlin, Germany. <br>
[Mehari, Temesgen; Strodthoff, Nils] Fraunhofer Heinrich Hertz Inst, Berlin, Germany. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Strodthoff, N (corresponding author), Carl von Ossietzky Univ Oldenburg, Oldenburg, Germany.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
temesgen.mehari@ptb.de; nils.strodthoff@uol.de</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Physikalisch-Technische Bundesanstalt (PTB); Fraunhofer Gesellschaft</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Mehari, Temesgen</display_name>&nbsp;</td><td>KCX-8728-2024&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Strodthoff, Nils</display_name>&nbsp;</td><td>AAK-8661-2021&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>PERGAMON-ELSEVIER SCIENCE LTD</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>THE BOULEVARD, LANGFORD LANE, KIDLINGTON, OXFORD OX5 1GB, ENGLAND</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Biology; Computer Science, Interdisciplinary Applications; Engineering, Biomedical; Mathematical &amp; Computational Biology</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Life Sciences &amp; Biomedicine - Other Topics; Computer Science; Engineering; Mathematical &amp; Computational Biology</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>YN6HS</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0010-4825</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1879-0534</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>COMPUT BIOL MED</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Comput. Biol. Med.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>8</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Bundesministerium fur Bildung und Forschung through the BIFOLD - Berlin Institute for the Foundations of Learning and Data</grant_agency>&nbsp;</td><td>
<div>01IS18025A&nbsp;</div>
<div>01IS18037A&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>MedalCare</grant_agency>&nbsp;</td><td>
<div>18HLT07&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>EMPIR programme</grant_agency>&nbsp;</td><td>
<div>18HLT07&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>European Union's Horizon 2020 research and innovation programme</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported by the Bundesministerium fur Bildung und Forschung through the BIFOLD -Berlin Institute for the Foundations of Learning and Data (ref. 01IS18025A and ref. 01IS18037A) and 18HLT07 MedalCare. The project 18HLT07 MedalCare has received funding from the EMPIR programme co-financed by the participating states and from the European Union's Horizon 2020 research and innovation programme.</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>hybrid, Green Submitted</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 8 of 37</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Variational infinite heterogeneous mixture model for semi-supervised clustering of heart enhancers</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Mehdi, TF (Mehdi, Tahmid F.); Singh, G (Singh, Gurdeep); Mitchell, JA (Mitchell, Jennifer A.); Moses, AM (Moses, Alan M.)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>BIOINFORMATICS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>35</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>18</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>3232-3239</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1093/bioinformatics/btz064</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2019 SEP 15</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>1</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>1</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
0</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
4</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>37</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Motivation: Mammalian genomes can contain thousands of enhancers but only a subset are actively driving gene expression in a given cellular context. Integrated genomic datasets can be harnessed to predict active enhancers. One challenge in integration of large genomic datasets is the increasing heterogeneity: continuous, binary and discrete features may all be relevant. Coupled with the typically small numbers of training examples, semi-supervised approaches for heterogeneous data are needed; however, current enhancer prediction methods are not designed to handle heterogeneous data in the semi-supervised paradigm.
<br>Results: We implemented a Dirichlet Process Heterogeneous Mixture model that infers Gaussian, Bernoulli and Poisson distributions over features. We derived a novel variational inference algorithm to handle semi-supervised learning tasks where certain observations are forced to cluster together. We applied this model to enhancer candidates in mouse heart tissues based on heterogeneous features. We constrained a small number of known active enhancers to appear in the same cluster, and 47 additional regions clustered with them. Many of these are located near heart-specific genes. The model also predicted 1176 active promoters, suggesting that it can discover new enhancers and promoters.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000487327500004</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>30753279</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
SAMPLING METHODS; GENE; SIGNATURES; EXPRESSION; DISCOVERY; DATABASE; BROWSER</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Mehdi, Tahmid F.; Moses, Alan M.] Univ Toronto, Dept Comp Sci, Toronto, ON M5S 3G5, Canada. <br>
[Singh, Gurdeep; Mitchell, Jennifer A.; Moses, Alan M.] Univ Toronto, Dept Cell &amp; Syst Biol, Toronto, ON M5S 3G5, Canada. <br>
[Mitchell, Jennifer A.; Moses, Alan M.] Univ Toronto, Ctr Anal Genome Evolut &amp; Funct, Toronto, ON M5S 3G5, Canada. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Moses, AM (corresponding author), Univ Toronto, Dept Comp Sci, Toronto, ON M5S 3G5, Canada.<br>Moses, AM (corresponding author), Univ Toronto, Dept Cell &amp; Syst Biol, Toronto, ON M5S 3G5, Canada.<br>Moses, AM (corresponding author), Univ Toronto, Ctr Anal Genome Evolut &amp; Funct, Toronto, ON M5S 3G5, Canada.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
alan.moses@utoronto.ca</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
University of Toronto; University of Toronto; University of Toronto</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Singh, Gurdeep</display_name>&nbsp;</td><td>AAU-7832-2020&nbsp;</td><td>0000-0003-2519-0456&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>OXFORD UNIV PRESS</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>GREAT CLARENDON ST, OXFORD OX2 6DP, ENGLAND</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Biochemical Research Methods; Biotechnology &amp; Applied Microbiology; Computer Science, Interdisciplinary Applications; Mathematical &amp; Computational Biology; Statistics &amp; Probability</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Biochemistry &amp; Molecular Biology; Biotechnology &amp; Applied Microbiology; Computer Science; Mathematical &amp; Computational Biology; Mathematics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>IZ8BR</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>1367-4803</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1367-4811</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>BIOINFORMATICS</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Bioinformatics</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>8</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Natural Sciences and Engineering Research Council of Canada (NSERC)</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Canada Foundation for Innovation</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Ontario Ministry of Research and Innovation</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Connaught International Scholarships</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported by the Natural Sciences and Engineering Research Council of Canada (NSERC) discovery grants (to AMM and JAM), and infrastructure grants from the Canada Foundation for Innovation and the Ontario Ministry of Research and Innovation (to AMM). Studentship funding to support this work was provided by Connaught International Scholarships (GS).</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>Green Submitted, Bronze</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 9 of 37</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>CalibrationPhys: Self-Supervised Video-Based Heart and Respiratory Rate Measurements by Calibrating Between Multiple Cameras</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Akamatsu, Y (Akamatsu, Yusuke); Umematsu, T (Umematsu, Terumi); Imaoka, H (Imaoka, Hitoshi)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE JOURNAL OF BIOMEDICAL AND HEALTH INFORMATICS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>28</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>3</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>1460-1471</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/JBHI.2023.3345486</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2024 MAR</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>5</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>6</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
3</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
8</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>45</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Video-based heart and respiratory rate measurements using facial videos are more useful and user-friendly than traditional contact-based sensors. However, most of the current deep learning approaches require ground-truth pulse and respiratory waves for model training, which are expensive to collect. In this paper, we propose CalibrationPhys, a self-supervised video-based heart and respiratory rate measurement method that calibrates between multiple cameras. CalibrationPhys trains deep learning models without supervised labels by using facial videos captured simultaneously by multiple cameras. Contrastive learning is performed so that the pulse and respiratory waves predicted from the synchronized videos using multiple cameras are positive and those from different videos are negative. CalibrationPhys also improves the robustness of the models by means of a data augmentation technique and successfully leverages a pre-trained model for a particular camera. Experimental results utilizing two datasets demonstrate that CalibrationPhys outperforms state-of-the-art heart and respiratory rate measurement methods. Since we optimize camera-specific models using only videos from multiple cameras, our approach makes it easy to use arbitrary cameras for heart and respiratory rate measurements.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001180907300045</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>38127597</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Remote photoplethysmography; heart rate; respiratory rate; facial video; convolutional neural network; contrastive learning; pre-training</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
NONCONTACT</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Akamatsu, Yusuke; Umematsu, Terumi; Imaoka, Hitoshi] NEC Corp Ltd, Biometr Res Labs, Kawasaki 2118666, Japan. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Akamatsu, Y (corresponding author), NEC Corp Ltd, Biometr Res Labs, Kawasaki 2118666, Japan.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
yusuke-akamatsu@nec.com; terumi@nec.com; h-imaoka_cb@nec.com</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
NEC Corporation</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Akamatsu, Yusuke</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-9123-3955&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Akamatsu, Yusuke</display_name>&nbsp;</td><td>HNO-9146-2023&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Umematsu, Terumi</display_name>&nbsp;</td><td>AAY-6635-2021&nbsp;</td><td>0000-0002-6328-0674&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Information Systems; Computer Science, Interdisciplinary Applications; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>KO4Y9</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2168-2194</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>2168-2208</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE J BIOMED HEALTH</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE J. Biomed. Health Inform.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>12</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 10 of 37</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>An Efficient and Private ECG Classification System Using Split and Semi-Supervised Learning</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Ayad, A (Ayad, Ahmad); Barhoush, M (Barhoush, Mahdi); Frei, M (Frei, Marian); Volker, B (Volker, Benedikt); Schmeink, A (Schmeink, Anke)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE JOURNAL OF BIOMEDICAL AND HEALTH INFORMATICS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>27</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>9</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>4261-4272</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/JBHI.2023.3281977</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2023 SEP</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>6</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>6</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
1</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
17</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>68</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Electrocardiography (ECG) is a standard diagnostic tool for evaluating the overall heart's electrical activity and is vital for detecting many cardiovascular diseases. Classifying ECG recordings using deep neural networks has been investigated in literature and has shown very good performance. However, this performance assumes that the training data is centralized, which is often not the case in real-life scenarios, where data resides in multiple places and only a small portion of it is labeled. Therefore, in this work, we propose an ECG classification system that focuses on preserving data privacy and enhancing overall system efficiency. We analyzed the complexity of previously proposed deep learning-based models and showed that the temporal convolutional network-based models (TCN) were the most efficient. Then, we built on the TCN models a modified split-learning (SL) system that achieves the same classification performance as the basic SL but reduces the communication overhead between the server and the client by 71.7% as well as reducing the computations at the client by 46.5% compared to the original SL system based on the TCN network. Finally, we implement semi-supervised learning in our system to enhance its classification performance by 9.1%-15.7%, when the training data consists only of 10% labeled data. We have tested our proposed system on a test IoT setup and it achieved satisfactory classification accuracy while being private and energy efficient for green-AI applications.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001064467900006</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>37262112</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
ECG; privacy-aware machine learning; distributed machine learning; edge computing; energy-efficient machine learning; split-learning; autoencoder; semi-supervised learning; mean teacher; FixMatch</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Ayad, Ahmad; Barhoush, Mahdi; Frei, Marian; Volker, Benedikt; Schmeink, Anke] RWTH Univ, Chair Informat Theory &amp; Data Analyt INDA, D-52074 Aachen, Germany. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Ayad, A (corresponding author), RWTH Univ, Chair Informat Theory &amp; Data Analyt INDA, D-52074 Aachen, Germany.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
ahmad.ayad@inda.rwth-aachen.de; mahdi.barhoush@inda.rwth-aachen.de; marian.frei@rwth-aachen.de; benedikt.voelker@rwth-aachen.de; anke.schmeink@inda.rwth-aachen.de</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
RWTH Aachen University</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Ayad, Ahmad</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-9081-1274&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Schmeink, Anke</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-9929-2925&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Frei, Marian</display_name>&nbsp;</td><td>&nbsp;</td><td>0009-0002-6899-3013&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Barhoush, Mahdi</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-3787-0662&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Volker, Benedikt</display_name>&nbsp;</td><td>&nbsp;</td><td>0009-0009-7266-7676&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Information Systems; Computer Science, Interdisciplinary Applications; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>R5AE7</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2168-2194</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>2168-2208</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE J BIOMED HEALTH</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE J. Biomed. Health Inform.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>12</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Federal Ministry of Education and Research (BMBF, Germany) through NeuroSys: Impulse durch Anwendungen (Projekt D)</grant_agency>&nbsp;</td><td>
<div>03ZU1106DA&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported by the Federal Ministry of Education and Research (BMBF, Germany) through NeuroSys: Impulse durch Anwendungen (Projekt D) under Grant 03ZU1106DA.&amp; nbsp;</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 11 of 37</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Semi-Supervised Learning for Low-Cost Personalized Obstructive Sleep Apnea Detection Using Unsupervised Deep Learning and Single-Lead Electrocardiogram</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Hu, SC (Hu, Shuaicong); Wang, YN (Wang, Ya'nan); Liu, J (Liu, Jian); Yang, CW (Yang, Cuiwei); Wang, AG (Wang, Aiguo); Li, KZ (Li, Kuanzheng); Liu, WX (Liu, Wenxin)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE JOURNAL OF BIOMEDICAL AND HEALTH INFORMATICS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>27</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>11</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>5281-5292</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/JBHI.2023.3304299</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2023 NOV</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>25</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>27</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
5</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
32</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>38</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Objective: Obstructive sleep apnea (OSA) is a common sleep-related breathing disorder that can lead to a wide range of health issues if left untreated. This study aims to address the lack of research on personalized models for single-lead electrocardiogram (ECG)-based OSA detection, by proposing an automatic semi-supervised algorithm for automated low-cost personalization fine-tuning. Methods: We utilize a convolutional neural network (CNN)-based auto-encoder (AE) with a modified training objective to detect anomalous region of OSA. An indicator based on model outputs is utilized as a benchmark measure to assign pseudo-labels with confidence to each sample. Finally, we perform validation of the semi-supervised algorithm on the same database and cross-database scenarios. Results: By introducing semi-supervised personalization, the accuracy, AUC, and mean absolute error (MAE) of the general model (GM) of 35 subjects from the same database are improved from 86.3%, 0.915, and 5.178 to 90.3%, 0.948, and 2.593. Simultaneously, in the validation of 25 subjects from a cross-database, the accuracy, AUC, and MAE of the GM are enhanced from 75.6%, 0.800, and 9.149 to 84.3%, 0.881, and 3.509. Conclusion: The improved version of AE demonstrates excellent adaptability in identifying abnormal features in OSA, employing a data-driven approach to assign pseudo-labels for unknown data automatically. Additionally, leveraging the pseudo-labels through a semi-supervised fine-tuning strategy provides a solution to overcome the limitation of clinical annotations, facilitating low-cost implementation of personalized models. Significance: The semisupervised approach proposed in this article provides a high-performance and annotation-free solution for personalized adjustment of automatic OSA detection.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001129955100008</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>37566509</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Obstructive sleep apnea (OSA); electrocardiogram (ECG); auto-encoder (AE); semi-supervised</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
SYSTEM</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Hu, Shuaicong; Wang, Ya'nan; Liu, Jian] Fudan Univ, Sch Informat Sci &amp; Technol, Ctr Biomed Engn, Shanghai 200433, Peoples R China. <br>
[Yang, Cuiwei] Fudan Univ, Sch Informat Sci &amp; Technol, Ctr Biomed Engn, Shanghai 200433, Peoples R China. <br>
[Yang, Cuiwei] Key Lab Med Imaging Comp &amp; Comp Assisted Interven, Shanghai 200093, Peoples R China. <br>
[Wang, Aiguo; Li, Kuanzheng; Liu, Wenxin] Xinghua City Peoples Hosp, Taizhou 225700, Jiangsu, Peoples R China. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Yang, CW (corresponding author), Fudan Univ, Sch Informat Sci &amp; Technol, Ctr Biomed Engn, Shanghai 200433, Peoples R China.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
22110720104@m.fudan.edu.cn; wangyn20@fudan.edu.cn; 22110720117@m.fudan.edu.cn; yangcw@fudan.edu.cn; xhwag@163.com; stronger1831@126.com; 450930173@qq.com</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Fudan University; Fudan University</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Liu, Wenxin</display_name>&nbsp;</td><td>LXW-5765-2024&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Yang, Cuiwei</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-3338-5835&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Hu, shuaicong</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-5458-0416&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Wang, Ya'nan</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-5599-3423&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Wang, Aiguo</display_name>&nbsp;</td><td>HNP-7198-2023&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Hu, Shuaicong</display_name>&nbsp;</td><td>NIU-9967-2025&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Liu, Jian</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-5791-7221&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Information Systems; Computer Science, Interdisciplinary Applications; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>DC9R2</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2168-2194</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>2168-2208</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE J BIOMED HEALTH</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE J. Biomed. Health Inform.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>12</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Shanghai Science and Technology Support Project</grant_agency>&nbsp;</td><td>
<div>18441900900&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Medical Scientific Research Key Project of Jiangsu Commission of Health</grant_agency>&nbsp;</td><td>
<div>ZDB2020025&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported in part by Shanghai Science and Technology Support Project under Grant 18441900900 and in part by the Medical Scientific Research Key Project of Jiangsu Commission of Health under Grant ZDB2020025.</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 12 of 37</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>A multi-module algorithm for heartbeat classification based on unsupervised learning and adaptive feature transfer</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Wang, YN (Wang, Yanan); Hu, SC (Hu, Shuaicong); Liu, J (Liu, Jian); Zhong, GY (Zhong, Gaoyan); Yang, CW (Yang, Cuiwei)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>COMPUTERS IN BIOLOGY AND MEDICINE</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>170</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>108072</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1016/j.compbiomed.2024.108072</value>&nbsp;&nbsp;<b>Early Access Date:</b> 
<value>JAN 2024</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2024 MAR</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>3</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>3</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
3</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
26</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>54</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
The scarcity of annotated data is a common issue in the realm of heartbeat classification based on deep learning. Transfer learning (TL) has emerged as an effective strategy for addressing this issue. However, current TL techniques in this realm overlook the probability distribution differences between the source domain (SD) and target domain (TD) databases. The motivation of this paper is to address the challenge of labeled data scarcity at the model level while exploring an effective method to eliminate domain discrepancy between SD and TD databases, especially when SD and TD are derived from inconsistent tasks. This study proposes a multi-module heartbeat classification algorithm. Initially, unsupervised feature extractors are designed to extract rich features from unlabeled SD and TD data. Subsequently, a novel adaptive transfer method is proposed to effectively eliminate domain discrepancy between features of SD for pre-training (PTF-SD) and features of TD for fine-tuning (FTF-TD). Finally, the adapted PTF-SD is employed to pre-train a designed classifier, and FTF-TD is used for classifier fine-tuning, with the objective of evaluating the algorithm's performance on the TD task. In our experiments, MNIST-DB serves as the SD database for handwritten digit image classification task, MIT-DB as the TD database for heartbeat classification task. The overall accuracy of classifying heartbeats into normal heartbeats, supraventricular ectopic beats (SVEBs), and ventricular ectopic beats (VEBs) reaches 96.7 %. Specifically, the sensitivity (Sen), positive predictive value (PPV), and F1 score for SVEBs are 0.802, 0.701, and 0.748, respectively. For VEBs, Sen, PPV, and F1 score are 0.976, 0.840, and 0.903, respectively. The results indicate that the proposed multi-module algorithm effectively addresses the challenge labeled data scarcity in heartbeat classification through unsupervised learning and adaptive feature transfer methods.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001171387100001</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>38301518</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Annotated data scarcity; Heartbeat classification; Adaptive feature transfer; Unsupervised learning; Domain discrepancy</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
CONVOLUTIONAL NEURAL-NETWORK</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Wang, Yanan; Hu, Shuaicong; Liu, Jian; Zhong, Gaoyan; Yang, Cuiwei] Fudan Univ, Ctr Biomed Engn, Sch Informat Sci &amp; Technol, Shanghai 200433, Peoples R China. <br>
[Yang, Cuiwei] Key Lab Med Imaging Comp &amp; Comp Assisted Intervent, Shanghai 200093, Peoples R China. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Yang, CW (corresponding author), Fudan Univ, Ctr Biomed Engn, Sch Informat Sci &amp; Technol, Shanghai 200433, Peoples R China.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
wangyn20@fudan.edu.cn; schu22@m.fudan.edu.cn; liuj22@m.fudan.edu.cn; zhonggaoyan@126.com; yangcw@fudan.edu.cn</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Fudan University</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Hu, Shuaicong</display_name>&nbsp;</td><td>NIU-9967-2025&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Yang, Cuiwei</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-3338-5835&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>PERGAMON-ELSEVIER SCIENCE LTD</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>THE BOULEVARD, LANGFORD LANE, KIDLINGTON, OXFORD OX5 1GB, ENGLAND</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Biology; Computer Science, Interdisciplinary Applications; Engineering, Biomedical; Mathematical &amp; Computational Biology</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Life Sciences &amp; Biomedicine - Other Topics; Computer Science; Engineering; Mathematical &amp; Computational Biology</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>JE1D8</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0010-4825</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1879-0534</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>COMPUT BIOL MED</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Comput. Biol. Med.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>15</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Shanghai Science and Technology Support Project</grant_agency>&nbsp;</td><td>
<div>18441900900&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported in part by Shanghai Science and Technology Support Project under Grant 18441900900.</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 13 of 37</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Constraint-Based Unsupervised Domain Adaptation Network for Multi-Modality Cardiac Image Segmentation</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Du, XQ (Du, Xiuquan); Liu, YG (Liu, Yueguo)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE JOURNAL OF BIOMEDICAL AND HEALTH INFORMATICS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>26</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>1</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>67-78</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/JBHI.2021.3126874</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2022 JAN</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>14</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>17</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
3</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
34</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>44</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
The cardiac CT and MRI images depict the various structures of the heart, which are very valuable for analyzing heart function. However, due to the difference in the shape of the cardiac images and imaging techniques, automatic segmentation is challenging. To solve this challenge, in this paper, we propose a new constraint-based unsupervised domain adaptation network. This network first performs mutual translation of images between different domains, it can provide training data for the segmentation model, and ensure domain invariance at the image level. Then, we input the target domain into the source domain segmentation model to obtain pseudo-labels and introduce cross-domain self-supervised learning between the two segmentation models. Here, a new loss function is designed to ensure the accuracy of the pseudo-labels. In addition, a cross-domain consistency loss is also introduced. Finally, we construct a multi-level aggregation segmentation network to obtain more refined target domain information. We validate our method on the public whole heart image segmentation challenge dataset and obtain experimental results of 82.9% and 5.5 on dice and average symmetric surface distance (ASSD), respectively. These experimental results prove that our method can provide important assistance in the clinical evaluation of unannotated cardiac datasets.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000745829300012</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>34757915</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Image segmentation; Computed tomography; Heart; Training; Biomedical imaging; Magnetic resonance imaging; Adversarial machine learning; Unsupervised domain adaptation; multi-modality cardiac image; self-supervised learning</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
SYNERGISTIC IMAGE</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Du, Xiuquan] Anhui Univ, Key Lab Intelligent Comp &amp; Signal Proc, Minist Educ, Hefei 240601, Peoples R China. <br>
[Du, Xiuquan; Liu, Yueguo] Anhui Univ, Sch Comp Sci &amp; Technol, Hefei 240601, Peoples R China. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Du, XQ (corresponding author), Anhui Univ, Key Lab Intelligent Comp &amp; Signal Proc, Minist Educ, Hefei 240601, Peoples R China.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
dxqllp@163.com; 2567470422@qq.com</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Anhui University; Anhui University</td>
</tr>

<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Information Systems; Computer Science, Interdisciplinary Applications; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>YL3YF</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2168-2194</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>2168-2208</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE J BIOMED HEALTH</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE J. Biomed. Health Inform.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>12</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Natural Science Research project of Anhui province</grant_agency>&nbsp;</td><td>
<div>KJ2020A0035&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported in part by Natural Science Research project of Anhui province under Grant KJ2020A0035.</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 14 of 37</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Bidirectional cross-modality unsupervised domain adaptation using generative adversarial networks for cardiac image segmentation</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Cui, HF (Cui, Hengfei); Chang, YW (Chang Yuwen); Lei, J (Lei Jiang); Yong, X (Yong Xia); Zhang, YN (Zhang, Yanning)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>COMPUTERS IN BIOLOGY AND MEDICINE</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>136</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>104726</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1016/j.compbiomed.2021.104726</value>&nbsp;&nbsp;<b>Early Access Date:</b> 
<value>AUG 2021</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2021 SEP</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>27</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>30</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
4</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
53</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>50</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Background: A novel Generative Adversarial Networks (GAN) based bidirectional cross-modality unsupervised domain adaptation (GBCUDA) framework is developed for cardiac image segmentation, which can effectively tackle the problem of network's segmentation performance degradation when adapting to the target domain without ground truth labels. Method: GBCUDA uses GAN for image alignment, applies adversarial learning to extract image features, and gradually enhances the domain invariance of extracted features. The shared encoder performs an end-to-end learning task in which features that differ between the two domains complement each other. The selfattention mechanism is incorporated to the GAN network, which can generate details based on the prompts of all feature positions. Furthermore, spectrum normalization is implemented to stabilize the training of GAN, and knowledge distillation loss is introduced to process high-level feature-maps in order to better complete the crossmode segmentation task. Results: The effectiveness of our proposed unsupervised domain adaptation framework is tested over the MultiModality Whole Heart Segmentation (MM-WHS) Challenge 2017 dataset. The proposed method is able to improve the average Dice from 74.1% to 81.5% for the four cardiac substructures, and reduce the average symmetric surface distance (ASD) from 7.0 to 5.8 over CT images. For MRI images, our proposed framework trained on CT images gives the average Dice of 59.2% and reduces the average ASD from 5.7 to 4.9. Conclusions: The evaluation results demonstrate our method's effectiveness on domain adaptation and the superiority to the current state-of-the-art domain adaptation methods.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000707411400002</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>34371318</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Generative adversarial networks; Unsupervised domain adaptation; Cardiac segmentation; Self-attention mechanism; Knowledge distillation loss</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
SYNERGISTIC IMAGE; HEART</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Cui, Hengfei; Chang Yuwen; Lei Jiang; Yong Xia; Zhang, Yanning] Northwestern Polytech Univ, Sch Comp Sci, Natl Engn Lab Integrated Aerosp Ground Ocean Big, Xian 710072, Peoples R China. <br>
[Cui, Hengfei; Chang Yuwen; Lei Jiang; Yong Xia] Northwestern Polytech Univ, Sch Comp Sci &amp; Engn, Ctr Multidisciplinary Convergence Comp CMCC, Xian 710072, Peoples R China. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Cui, HF (corresponding author), Northwestern Polytech Univ, Sch Comp Sci, Natl Engn Lab Integrated Aerosp Ground Ocean Big, Xian 710072, Peoples R China.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
hfcui@nwpu.edu.cn</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Northwestern Polytechnical University; Northwestern Polytechnical University</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>lei, jiang</display_name>&nbsp;</td><td>GQA-9644-2022&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Zhang, Zhi-jun</display_name>&nbsp;</td><td>HGE-3958-2022&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Cui, Hengfei</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-8625-2521&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>PERGAMON-ELSEVIER SCIENCE LTD</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>THE BOULEVARD, LANGFORD LANE, KIDLINGTON, OXFORD OX5 1GB, ENGLAND</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Biology; Computer Science, Interdisciplinary Applications; Engineering, Biomedical; Mathematical &amp; Computational Biology</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Life Sciences &amp; Biomedicine - Other Topics; Computer Science; Engineering; Mathematical &amp; Computational Biology</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>WH0XE</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0010-4825</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1879-0534</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>COMPUT BIOL MED</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Comput. Biol. Med.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>10</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Natural Science Foundation of China</grant_agency>&nbsp;</td><td>
<div>61801393&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Fundamental Research Funds for the Central Universities</grant_agency>&nbsp;</td><td>
<div>3102020QD1001&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>The study was supported in part by the National Natural Science Foundation of China under Grant 61801393, and in part by the Fundamental Research Funds for the Central Universities under Grant 3102020QD1001.</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 15 of 37</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Patient contrastive learning: A performant, expressive, and practical approach to electrocardiogram modeling</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Diamant, N (Diamant, Nathaniel); Reinertsen, E (Reinertsen, Erik); Song, SV (Song, Steven); Aguirre, AD (Aguirre, Aaron D.); Stultz, CM (Stultz, Collin M.); Batra, P (Batra, Puneet)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>PLOS COMPUTATIONAL BIOLOGY</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>18</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>2</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>e1009862</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1371/journal.pcbi.1009862</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2022 FEB</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>26</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>32</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
0</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
15</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>34</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Supervised machine learning applications in health care are often limited due to a scarcity of labeled training data. To mitigate the effect of small sample size, we introduce a pre-training approach, Patient Contrastive Learning of Representations (PCLR), which creates latent representations of electrocardiograms (ECGs) from a large number of unlabeled examples using contrastive learning. The resulting representations are expressive, performant, and practical across a wide spectrum of clinical tasks. We develop PCLR using a large health care system with over 3.2 million 12-lead ECGs and demonstrate that training linear models on PCLR representations achieves a 51% performance increase, on average, over six training set sizes and four tasks (sex classification, age regression, and the detection of left ventricular hypertrophy and atrial fibrillation), relative to training neural network models from scratch. We also compared PCLR to three other ECG pre-training approaches (supervised pre-training, unsupervised pre-training with an autoencoder, and pre-training using a contrastive multi ECG-segment approach), and show significant performance benefits in three out of four tasks. We found an average performance benefit of 47% over the other models and an average of a 9% performance benefit compared to best model for each task. We release PCLR to enable others to extract ECG representations at https://github.com/ broadinstitute/ml4h/tree/master/model_zoo/PCLR.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000757563200006</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>35157695</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
ATRIAL-FIBRILLATION; RISK-FACTOR; CLASSIFICATION</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Diamant, Nathaniel; Reinertsen, Erik; Stultz, Collin M.] MIT, Res Lab Elect, 77 Massachusetts Ave, Cambridge, MA 02139 USA. <br>
[Diamant, Nathaniel; Batra, Puneet] Broad Inst MIT &amp; Harvard, Data Sci Platform, Cambridge, MA USA. <br>
[Reinertsen, Erik; Song, Steven; Aguirre, Aaron D.; Stultz, Collin M.] Massachusetts Gen Hosp, Div Cardiol, Boston, MA USA. <br>
[Aguirre, Aaron D.] Massachusetts Gen Hosp, Ctr Syst Biol, Res Inst, Boston, MA 02114 USA. <br>
[Aguirre, Aaron D.] Harvard Med Sch, Boston, MA 02115 USA. <br>
[Aguirre, Aaron D.] Massachusetts Gen Hosp, Wellman Ctr Photomed, Res Inst, Boston, MA 02114 USA. <br>
[Stultz, Collin M.] MIT, Dept Elect Engn &amp; Comp Sci, 77 Massachusetts Ave, Cambridge, MA 02139 USA. <br>
[Stultz, Collin M.] Harvard Div Hlth Sci &amp; Technol, Cambridge, MA USA. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Batra, P (corresponding author), Broad Inst MIT &amp; Harvard, Data Sci Platform, Cambridge, MA USA.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
pbatra@broadinstitute.org</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Massachusetts Institute of Technology (MIT); Harvard University; Massachusetts Institute of Technology (MIT); Broad Institute; Harvard University; Harvard University Medical Affiliates; Massachusetts General Hospital; Harvard University; Harvard University Medical Affiliates; Massachusetts General Hospital; Harvard University; Harvard Medical School; Harvard University; Harvard University Medical Affiliates; Massachusetts General Hospital; Massachusetts Institute of Technology (MIT)</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Stultz, Collin</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-3415-242X&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Song, Steven</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-5868-3877&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Diamant, Nathaniel</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-1738-304X&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Batra, Puneet</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-6822-0593&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>PUBLIC LIBRARY SCIENCE</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>1160 BATTERY STREET, STE 100, SAN FRANCISCO, CA 94111 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Biochemical Research Methods; Mathematical &amp; Computational Biology</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Biochemistry &amp; Molecular Biology; Mathematical &amp; Computational Biology</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>ZC5MD</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>1553-734X</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1553-7358</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>PLOS COMPUT BIOL</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>PLoS Comput. Biol.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>16</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>IBM Research</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Bayer AG</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Quanta Computer</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Controlled Risk Insurance Company/Risk Management Foundation (CRICO)</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>P.B. and N.D. were supported by a grant from IBM Research to investigate the application of machine learning into cardiovascular disease, and from Bayer AG for new methods investigating Heart Failure and Stroke. C.M.S. was supported by a grant from Quanta Computer. A.D.A. acknowledges funding from the Controlled Risk Insurance Company/Risk Management Foundation (CRICO). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>gold, Green Submitted</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 16 of 37</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>DeepMiceTL: a deep transfer learning based prediction of mice cardiac conduction diseases using early electrocardiograms</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Liao, Y (Liao, Ying); Xiang, YS (Xiang, Yisha); Zheng, MJ (Zheng, Mingjie); Wang, J (Wang, Jun)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>BRIEFINGS IN BIOINFORMATICS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>24</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>3</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1093/bib/bbad109</value>&nbsp;&nbsp;<b>Early Access Date:</b> 
<value>MAR 2023</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2023 MAY 19</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>0</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>0</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
0</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
13</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>37</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Cardiac conduction disease is a major cause of morbidity and mortality worldwide. There is considerable clinical significance and an emerging need of early detection of these diseases for preventive treatment success before more severe arrhythmias occur. However, developing such early screening tools is challenging due to the lack of early electrocardiograms (ECGs) before symptoms occur in patients. Mouse models are widely used in cardiac arrhythmia research. The goal of this paper is to develop deep learning models to predict cardiac conduction diseases in mice using their early ECGs. We hypothesize that mutant mice present subtle abnormalities in their early ECGs before severe arrhythmias present. These subtle patterns can be detected by deep learning though they are hard to be identified by human eyes. We propose a deep transfer learning model, DeepMiceTL, which leverages knowledge from human ECGs to learn mouse ECG patterns. We further apply the Bayesian optimization and k-fold cross validation methods to tune the hyperparameters of the DeepMiceTL. Our results show that DeepMiceTL achieves a promising performance (F1-score: 83.8%, accuracy: 84.8%) in predicting the occurrence of cardiac conduction diseases using early mouse ECGs. This study is among the first efforts that use state-of-the-art deep transfer learning to identify ECG patterns during the early course of cardiac conduction disease in mice. Our approach not only could help in cardiac conduction disease research in mice, but also suggest a feasibility for early clinical diagnosis of human cardiac conduction diseases and other types of cardiac arrythmias using deep transfer learning in the future.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000954577800001</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>36935112</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
cardiac conduction disease; electrocardiogram (ECG); mouse model; deep transfer learning; Bayesian optimization</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
SICK SINUS SYNDROME; DIPHTHERIA-TOXIN FRAGMENT; NODE</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Liao, Ying] Texas Tech Univ, Dept Ind Mfg &amp; Syst Engn, Lubbock, TX USA. <br>
[Xiang, Yisha] Univ Houston, Dept Ind Engn, Houston, TX USA. <br>
[Zheng, Mingjie; Wang, Jun] Univ Texas Hlth Sci Ctr Houston, McGovern Med Sch, Dept Pediat, Houston, TX USA. <br>
[Xiang, Yisha] Univ Houston, Dept Ind Engn, Houston, TX 77004 USA. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Xiang, YS (corresponding author), Univ Houston, Dept Ind Engn, Houston, TX 77004 USA.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
yxiang4@Central.UH.EDU</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Texas Tech University System; Texas Tech University; University of Houston System; University of Houston; University of Texas System; University of Texas Health Science Center Houston; University of Houston System; University of Houston</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Xiang, Yisha</display_name>&nbsp;</td><td>L-9847-2016&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>OXFORD UNIV PRESS</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>GREAT CLARENDON ST, OXFORD OX2 6DP, ENGLAND</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Biochemical Research Methods; Mathematical &amp; Computational Biology</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Biochemistry &amp; Molecular Biology; Mathematical &amp; Computational Biology</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>I8HN6</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>1467-5463</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1477-4054</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>BRIEF BIOINFORM</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Brief. Bioinform.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>10</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Institutes of Health</grant_agency>&nbsp;</td><td>
<div>R01HL142704&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>American Heart Association</grant_agency>&nbsp;</td><td>
<div>970606&nbsp;</div>
<div>902940&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>American Heart Association (AHA)</grant_agency>&nbsp;</td><td>
<div>970606&nbsp;</div>
<div>902940&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>The results are based on M.Z. and J.W.'s unpublished mice ECG data. Data collection is supported by funds from the National Institutes of Health (R01HL142704 to J.W.) and American Heart Association (970606 to J.W., 902940 to M.Z.).</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>Green Published, Bronze</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 17 of 37</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Diagnostic quality assessment for low-dimensional ECG representations</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Kov&aacute;cs, P (Kovacs, Peter); B&ouml;ck, C (Boeck, Carl); Tschoellitsch, T (Tschoellitsch, Thomas); Huemer, M (Huemer, Mario); Meier, J (Meier, Jens)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>COMPUTERS IN BIOLOGY AND MEDICINE</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>150</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>106086</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1016/j.compbiomed.2022.106086</value>&nbsp;&nbsp;<b>Early Access Date:</b> 
<value>SEP 2022</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2022 NOV</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>3</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>3</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
0</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
6</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>30</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
There have been several attempts to quantify the diagnostic distortion caused by algorithms that perform low -dimensional electrocardiogram (ECG) representation. However, there is no universally accepted quantitative measure that allows the diagnostic distortion arising from denoising, compression, and ECG beat representation algorithms to be determined. Hence, the main objective of this work was to develop a framework to enable biomedical engineers to efficiently and reliably assess diagnostic distortion resulting from ECG processing algorithms. We propose a semiautomatic framework for quantifying the diagnostic resemblance between original and denoised/reconstructed ECGs. Evaluation of the ECG must be done manually, but is kept simple and does not require medical training. In a case study, we quantified the agreement between raw and reconstructed (denoised) ECG recordings by means of kappa-based statistical tests. The proposed methodology takes into account that the observers may agree by chance alone. Consequently, for the case study, our statistical analysis reports the "true", beyond-chance agreement in contrast to other, less robust measures, such as simple percent agreement calculations. Our framework allows efficient assessment of clinically important diagnostic distortion, a potential side effect of ECG (pre-)processing algorithms. Accurate quantification of a possible diagnostic loss is critical to any subsequent ECG signal analysis, for instance, the detection of ischemic ST episodes in long-term ECG recordings.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000878510400004</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>36191392</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Baseline removal; Clinical evaluation; Diagnostic distortion measures; ECG denoising; Kappa statistics</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
HIGH AGREEMENT; LOW KAPPA</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Kovacs, Peter] Eotvos Lorand Univ, Dept Numer Anal, Pazmany Peter Setany 1, H-1117 Budapest, Hungary. <br>
[Boeck, Carl] Johannes Kepler Univ Linz, Inst Signal Proc, JKU LIT SAL eSPML Lab, Altenberger Str 69, A-4040 Linz, Austria. <br>
[Tschoellitsch, Thomas; Meier, Jens] Johannes Kepler Univ Linz, Clin Anesthesiol &amp; Intens Care Med, Krankenhausstr 9, A-4020 Linz, Austria. <br>
[Huemer, Mario] Johannes Kepler Univ Linz, Inst Signal Proc, Altenberger Str 69, A-4040 Linz, Austria. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Kov&aacute;cs, P (corresponding author), Eotvos Lorand Univ, Dept Numer Anal, Pazmany Peter Setany 1, H-1117 Budapest, Hungary.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
kovika@inf.elte.hu; carl.boeck@jku.at; Thomas.Tschoellitsch@kepleruniklinkum.at; mario.huemer@jku.at; Jens.Meier@kepleruniklinkum.at</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Eotvos Lorand University; Johannes Kepler University Linz; Johannes Kepler University Linz; Johannes Kepler University Linz</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Bock, Carl</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-2207-2411&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Tschoellitsch, Thomas</display_name>&nbsp;</td><td>HJP-9501-2023&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Kov&aacute;cs, P&eacute;ter</display_name>&nbsp;</td><td>H-7307-2017&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>PERGAMON-ELSEVIER SCIENCE LTD</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>THE BOULEVARD, LANGFORD LANE, KIDLINGTON, OXFORD OX5 1GB, ENGLAND</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Biology; Computer Science, Interdisciplinary Applications; Engineering, Biomedical; Mathematical &amp; Computational Biology</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Life Sciences &amp; Biomedicine - Other Topics; Computer Science; Engineering; Mathematical &amp; Computational Biology</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>5X3NV</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0010-4825</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1879-0534</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>COMPUT BIOL MED</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Comput. Biol. Med.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>9</value>
</td>
</tr>

<tr>
<td>
<b>Open Access:</b>

<value>Green Published, hybrid, Green Submitted</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 18 of 37</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Automated ECG Noise Detection and Classification System for Unsupervised Healthcare Monitoring</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Satija, U (Satija, Udit); Ramkumar, B (Ramkumar, Barathram); Manikandan, MS (Manikandan, M. Sabarimalai)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE JOURNAL OF BIOMEDICAL AND HEALTH INFORMATICS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>22</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>3</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>722-732</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/JBHI.2017.2686436</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2018 MAY</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>113</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>130</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
4</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
62</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>39</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Objective: Automatic detection and classification of noises can play a vital role in the development of robust unsupervised electrocardiogram (ECG) analysis systems. This paper proposes a novel unified framework for automatic detection, localization, and classification of single and combined ECG noises. Methods: The proposed framework consists of the modified ensemble empirical mode decomposition (CEEMD), the short-term temporal feature extraction, and the decision-rule-based noise detection and classification. In the proposed framework, ECG signals are first decomposed using the modified CEEMD algorithm for discriminating the ECG components from the noises and artifacts. Then, the short-term temporal features such as maximum absolute amplitude, number of zerocrossings, and local maximum peak amplitude of the autocorelation function are computed from the extracted high-frequency and low-frequency signals. Finally, a decision rule-based algorithm is presented for detecting the presence of noises and classifying the processed ECG signals into six signal groups: noise-free ECG, ECG+BW, ECG+MA, ECG+PLI, ECG+BW+PLI, and ECG+BW+MA. Results: The proposed framework is rigorously evaluated on five benchmark ECG databases and the real-time ECG signals. The proposed framework achieves an average sensitivity of 99.12%, specificity of 98.56%, and overall accuracy of 98.90% in detecting the presence of noises. Classification results show that the framework achieves an average sensitivity, positive predictivity, and classification accuracy of 98.93%, 98.39%, and 97.38%, respectively. Conclusion: The proposed framework not only achieves better noise detection and classification rates than the current state-of-the-art methods but also accurately localizes short bursts of noises with low end-point delineation errors. Significance: Extensive studies on benchmark databases demonstrate that the proposed framework is more suitable for reducing false alarm rates and selecting appropriate noise-specific denoising techniques in automated ECG analysis applications.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000431374500010</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>28333651</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Baseline wander; ECG denoising; ECG noise detection and classification; ECG signal quality assessment; muscle artifacts</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
SIGNAL-QUALITY INDEXES; REAL-TIME; ELECTROCARDIOGRAM SIGNAL; DATA FUSION; UNIT</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Satija, Udit; Ramkumar, Barathram; Manikandan, M. Sabarimalai] Indian Inst Technol Bhubaneswar, Sch Elect Sci, Bhubaneswar 751013, India. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Satija, U (corresponding author), Indian Inst Technol Bhubaneswar, Sch Elect Sci, Bhubaneswar 751013, India.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
us11@iitbbs.ac.in; barathram@iitbbs.ac.in; msm@iitbbs.ac.in</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Indian Institute of Technology System (IIT System); Indian Institute of Technology (IIT) - Bhubaneswar</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Manikandan, M Sabarimalai</display_name>&nbsp;</td><td>V-6236-2019&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Manikandan, M Sabarimalai</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-6878-4911&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>SATIJA, UDIT</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-3851-765X&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>SATIJA, UDIT</display_name>&nbsp;</td><td>AAA-5332-2021&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Information Systems; Computer Science, Interdisciplinary Applications; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>GE6XN</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2168-2194</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE J BIOMED HEALTH</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE J. Biomed. Health Inform.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>11</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 19 of 37</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>ST-GAN: A Swin Transformer-Based Generative Adversarial Network for Unsupervised Domain Adaptation of Cross-Modality Cardiac Segmentation</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Zhang, YF (Zhang, Yifan); Wang, YH (Wang, Yonghui); Xu, LS (Xu, Lisheng); Yao, YD (Yao, Yudong); Qian, W (Qian, Wei); Qi, L (Qi, Lin)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE JOURNAL OF BIOMEDICAL AND HEALTH INFORMATICS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>28</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>2</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>893-904</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/JBHI.2023.3336965</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2024 FEB</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>3</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>3</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
4</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
47</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>50</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Unsupervised domain adaptation (UDA) methods have shown great potential in cross-modality medical image segmentation tasks, where target domain labels are unavailable. However, the domain shift among different image modalities remains challenging, because the conventional UDA methods are based on convolutional neural networks (CNNs), which tend to focus on the texture of images and cannot establish the global semantic relevance of features due to the locality of CNNs. This paper proposes a novel end-to-end Swin Transformer-based generative adversarial network (ST-GAN) for cross-modality cardiac segmentation. In the generator of ST-GAN, we utilize the local receptive fields of CNNs to capture spatial information and introduce the Swin Transformer to extract global semantic information, which enables the generator to better extract the domain-invariant features in UDA tasks. In addition, we design a multi-scale feature fuser to sufficiently fuse the features acquired at different stages and improve the robustness of the UDA network. We extensively evaluated our method with two cross-modality cardiac segmentation tasks on the MS-CMR 2019 dataset and the M&amp;Ms dataset. The results of two different tasks show the validity of ST-GAN compared with the state-of-the-art cross-modality cardiac image segmentation methods.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001203362500022</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>38019618</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Adversarial learning; cross-modality cardiac segmentation; multi-scale fusion; swin transformer; unsupervised domain adaptation</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
IMAGE</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Zhang, Yifan; Wang, Yonghui; Xu, Lisheng; Qian, Wei; Qi, Lin] Northeastern Univ, Coll Med &amp; Biol Informat Engn, Shenyang 110004, Peoples R China. <br>
[Xu, Lisheng; Qi, Lin] Minist Educ, Engn Res Ctr Med Imaging &amp; Intelligent Anal, Shenyang 110169, Peoples R China. <br>
[Xu, Lisheng; Qi, Lin] Northeastern Univ, Key Lab Med Image Comp, Minist Educ, Shenyang 110169, Peoples R China. <br>
[Yao, Yudong] Stevens Inst Technol, Elect &amp; Comp Engn Dept, Hoboken, NJ 07030 USA. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Qi, L (corresponding author), Northeastern Univ, Coll Med &amp; Biol Informat Engn, Shenyang 110004, Peoples R China.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
zyf0lzh@gmail.com; wang18852865993@gmail.com; xuls@bmie.neu.edu.cn; Yu-Dong.Yao@stevens.edu; wqian@bmie.neu.edu.cn; qilin@bmie.neu.edu.cn</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Northeastern University - China; Northeastern University - China; Stevens Institute of Technology</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>XU, Lisheng</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-8360-3605&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Zhang, Yifan</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-0141-1536&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>XU, Lisheng</display_name>&nbsp;</td><td>C-2974-2008&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Information Systems; Computer Science, Interdisciplinary Applications; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>NW0A0</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2168-2194</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>2168-2208</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE J BIOMED HEALTH</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE J. Biomed. Health Inform.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>12</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Key Research and Development Program of China</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>No Statement Available</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 20 of 37</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>I-Vector-Based Patient Adaptation of Deep Neural Networks for Automatic Heartbeat Classification</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Xu, SS (Xu, Sean Shensheng); Mak, MW (Mak, Man-Wai); Cheung, CC (Cheung, Chi-Chung)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE JOURNAL OF BIOMEDICAL AND HEALTH INFORMATICS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>24</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>3</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>717-727</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/JBHI.2019.2919732</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2020 MAR</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>22</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>24</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
1</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
33</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>36</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Automatic classification of electrocardiogram (ECG) signals is important for diagnosing heart arrhythmias. A big challenge in automatic ECG classification is the variation in the waveforms and characteristics of ECG signals among different patients. To address this issue, this paper proposes adapting a patient-independent deep neural network (DNN) using the information in the patient-dependent identity vectors (i-vectors). The adapted networks, namely i-vector adapted patient-specific DNNs (iAP-DNNs), are tuned toward the ECG characteristics of individual patients. For each patient, his/her ECG waveforms are compressed into an i-vector using a factor analysis model. Then, this i-vector is injected into the middle hidden layer of the patient-independent DNN. Stochastic gradient descent is then applied to fine-tune the whole network to form a patient-specific classifier. As a result, the adaptation makes use of not only the raw ECG waveforms from the specific patient but also the compact representation of his/her ECG characteristics through the i-vector. Analysis on the hidden-layer activations shows that by leveraging the information in the i-vectors, the iAP-DNNs are more capable of discriminating normal heartbeats against arrhythmic heartbeats than the networks that use the patient-specific ECG only for the adaptation. Experimental results based on the MIT-BIH database suggest that the iAP-DNNs perform better than existing patient-specific classifiers in terms of various performance measures. In particular, the sensitivity and specificity of the existing methods are all under the receiver operating characteristic curves of the iAP-DNNs.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000519730300008</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>31150349</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Electrocardiography; Statistics; Training; Sociology; Heart rate variability; Feature extraction; Heart beat; ECG classification; Arrhythmias; deep neural networks; i-vectors; DNN adaptation</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
ECG CLASSIFICATION</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Xu, Sean Shensheng; Mak, Man-Wai; Cheung, Chi-Chung] Hong Kong Polytech Univ, Dept Elect &amp; Informat Engn, Hong Kong, Peoples R China. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Mak, MW (corresponding author), Hong Kong Polytech Univ, Dept Elect &amp; Informat Engn, Hong Kong, Peoples R China.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
seanssx@yahoo.com; enmwmak@polyu.edu.hk; lawrence.chi-chung.cheung@polyu.edu.hk</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Hong Kong Polytechnic University</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Mak, Man-Wai</display_name>&nbsp;</td><td>C-3750-2014&nbsp;</td><td>0000-0001-8854-3760&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Mak, Manwai</display_name>&nbsp;</td><td>C-3750-2014&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>XU, SHENSHENG</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-8846-5061&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Information Systems; Computer Science, Interdisciplinary Applications; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>KU5DO</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2168-2194</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>2168-2208</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE J BIOMED HEALTH</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE J. Biomed. Health Inform.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>11</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>RGC of Hong Kong</grant_agency>&nbsp;</td><td>
<div>PolyU152137/17E&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported by the RGC of Hong Kong, under Grant PolyU152137/17E.</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>Green Accepted</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 21 of 37</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Towards federated transfer learning in electrocardiogram signal analysis</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Chorney, W (Chorney, Wesley); Wang, HF (Wang, Haifeng)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>COMPUTERS IN BIOLOGY AND MEDICINE</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>170</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>107984</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1016/j.compbiomed.2024.107984</value>&nbsp;&nbsp;<b>Early Access Date:</b> 
<value>JAN 2024</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2024 MAR</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>7</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>10</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
6</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
20</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>79</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Modern methods in artificial intelligence perform very well on many healthcare datasets, at times outperforming trained doctors. However, many assumptions made in model training are not justifiable in clinical settings. In this work, we propose a method to train classifiers for electrocardiograms, able to deal with data of disparate input dimensions, distributed across different institutions, and able to protect patient privacy. In addition, we propose a simple method for creating federated datasets from any centralized dataset. We use autoencoders in conjunction with federated learning to model a highly heterogeneous modeling problem using the Massachusetts Institute of Technology Beth Israel Hospital Arrhythmia dataset, the Computing in Cardiology 2017 challenge dataset, and the PTB-XL dataset. For an encoding dimension of 1000, our federated classifier achieves an accuracy, precision, recall, and F1 score of 73.0%, 66.6%, 73.0%, and 69.7%, respectively. Our results suggest that dropping commonly made assumptions significantly complicate training and that as a result, estimates of performance of many machine learning models may overestimate performance when adopted for clinical settings.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001168093000001</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>38244469</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Federated transfer learning; Electrocardiograms; Autoencoders; Data privacy; Healthcare</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
ATRIAL-FIBRILLATION; ECG SIGNALS; NONINVASIVE ASSESSMENT; MYOCARDIAL-INFARCTION; HELICOBACTER-PYLORI; DIAGNOSIS; AI; ARRHYTHMIAS; CHALLENGES</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Chorney, Wesley] Mississippi State Univ, Computat Engn, Mississippi State, MS 39762 USA. <br>
[Wang, Haifeng] Mississippi State Univ, Ind &amp; Syst Engn, Mississippi State, MS 39762 USA. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Chorney, W (corresponding author), Mississippi State Univ, Computat Engn, Mississippi State, MS 39762 USA.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
wwc102@msstate.edu; wang@ise.msstate.edu</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Mississippi State University; Mississippi State University</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Chorney, Wesley</display_name>&nbsp;</td><td>&nbsp;</td><td>0009-0008-2598-6837&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Chorney, Wesley</display_name>&nbsp;</td><td>IWV-4143-2023&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>PERGAMON-ELSEVIER SCIENCE LTD</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>THE BOULEVARD, LANGFORD LANE, KIDLINGTON, OXFORD OX5 1GB, ENGLAND</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Biology; Computer Science, Interdisciplinary Applications; Engineering, Biomedical; Mathematical &amp; Computational Biology</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Life Sciences &amp; Biomedicine - Other Topics; Computer Science; Engineering; Mathematical &amp; Computational Biology</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>IR6J4</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0010-4825</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1879-0534</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>COMPUT BIOL MED</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Comput. Biol. Med.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>15</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 22 of 37</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>sCL-ST: Supervised Contrastive Learning With Semantic Transformations for Multiple Lead ECG Arrhythmia Classification</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Le, DC (Le, Duc); Truong, S (Truong, Sang); Brijesh, P (Brijesh, Patel); Adjeroh, DA (Adjeroh, Donald A.); Le, N (Le, Ngan)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE JOURNAL OF BIOMEDICAL AND HEALTH INFORMATICS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>27</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>6</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>2818-2828</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/JBHI.2023.3246241</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2023 JUN</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>16</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>20</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
1</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
41</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>53</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
The automatic classification of electrocardiogram (ECG) signals has played an important role in cardiovascular diseases diagnosis and prediction. With recent advancements in deep neural networks (DNNs), particularly Convolutional Neural Networks (CNNs), learning deep features automatically from the original data is becoming an effective and widespread approach in a variety of intelligent tasks including biomedical and health informatics. However, most of the existing approaches are trained on either 1D CNNs or 2D CNNs, and they suffer from the limitations of random phenomena (i.e. random initial weights). Furthermore, the ability to train such DNNs in a supervised manner in healthcare is often limited due to the scarcity of labeled training data. To address the problems of weight initialization and limited annotated data, in this work, we leverage recent self-supervised learning technique, namely, contrastive learning, and present supervised contrastive learning (sCL). Different from existing self-supervised contrastive learning approaches, which often generate false negatives because of random selection of negative anchors, our contrastive learning makes use of labeled data to pull the same class closer together and push different classes far apart to avoid potential false negatives. Furthermore, unlike other kinds of signals (e.g. speech, image, video), ECG signal is sensitive to changes, and inappropriate transformation could directly affect diagnosis results. To deal with this issue, we present two semantic transformations, i.e. semantic split-join and semantic weighted peaks noise smoothing. The proposed deep neural network sCL-ST with supervised contrastive learning and semantic transformations is trained as an end-to-end framework for the multi-label classification of 12-lead ECGs. Our sCL-ST network contains two sub-networks i.e. pre-text task and down-stream task. Our experimental results have been evaluated on 12-lead PhysioNet 2020 dataset and shown that our proposed network outperforms the state-of-the-art existing approaches.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001004541400023</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>37028019</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
ECG; arrhythmia classification; multiple lead; contrastive learning; self-supervised learning</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
DEEP; ALGORITHM</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Le, Duc; Truong, Sang; Le, Ngan] Univ Arkansas, Dept Comp Sci &amp; Comp Engn, Fayetteville, AR 72703 USA. <br>
[Brijesh, Patel] West Virginia Univ, Sch Med, Div Cardiol, Morgantown, WV 26506 USA. <br>
[Adjeroh, Donald A.] West Virginia Univ, Dept Comp Sci &amp; Elect Engn LCSEE, Morgantown, WV 26506 USA. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Le, N (corresponding author), Univ Arkansas, Dept Comp Sci &amp; Comp Engn, Fayetteville, AR 72703 USA.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
minhducl@uark.edu; sangt@uark.edu; brijesh.patel@wvumedicine.org; Donald.Adjeroh@mail.wvu.edu; thile@uark.edu</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
University of Arkansas System; University of Arkansas Fayetteville; West Virginia University; West Virginia University</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Le, Duc</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-7771-7729&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Le, Ngan</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-2571-0511&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Information Systems; Computer Science, Interdisciplinary Applications; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>I7KP3</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2168-2194</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>2168-2208</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE J BIOMED HEALTH</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE J. Biomed. Health Inform.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>11</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>U.S. National Science Foundation</grant_agency>&nbsp;</td><td>
<div>OIA-1946391&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>NSF</grant_agency>&nbsp;</td><td>
<div>1920920&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported in part by U.S. National Science Foundation under Award OIA-1946391 and in part by NSF under Grant 1920920.</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 23 of 37</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Scaling Representation Learning From Ubiquitous ECG With State-Space Models</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Avramidis, K (Avramidis, Kleanthis); Kunc, D (Kunc, Dominika); Perz, B (Perz, Bartosz); Adsul, K (Adsul, Kranti); Feng, TT (Feng, Tiantian); Kazienko, P (Kazienko, Przemyslaw); Saganowski, S (Saganowski, Stanislaw); Narayanan, S (Narayanan, Shrikanth)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE JOURNAL OF BIOMEDICAL AND HEALTH INFORMATICS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>28</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>10</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>5877-5889</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/JBHI.2024.3416897</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2024 OCT</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>4</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>4</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
8</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
18</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>82</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Ubiquitous sensing from wearable devices in the wild holds promise for enhancing human well-being, from diagnosing clinical conditions and measuring stress to building adaptive health promoting scaffolds. But the large volumes of data therein across heterogeneous contexts pose challenges for conventional supervised learning approaches. Representation Learning from biological signals is an emerging realm catalyzed by the recent advances in computational modeling and the abundance of publicly shared databases. The electrocardiogram (ECG) is the primary researched modality in this context, with applications in health monitoring, stress and affect estimation. Yet, most studies are limited by small-scale controlled data collection and over-parameterized architecture choices. We introduce &lt;bold&gt;WildECG&lt;/bold&gt;, a pre-trained state-space model for representation learning from ECG signals. We train this model in a self-supervised manner with 275 000 10 s ECG recordings collected in the wild and evaluate it on a range of downstream tasks. The proposed model is a robust backbone for ECG analysis, providing competitive performance on most of the tasks considered, while demonstrating efficacy in low-resource regimes.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001329782300043</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>38935470</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Electrocardiography; Task analysis; Biological system modeling; Data models; Bioinformatics; Training; State-space methods; ubiquitous computing; self-supervised learning; state-space models</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
ARTIFICIAL-INTELLIGENCE; ELECTROCARDIOGRAM; CLASSIFICATION; EMOTION</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Avramidis, Kleanthis; Adsul, Kranti; Feng, Tiantian; Narayanan, Shrikanth] Univ Southern Calif, Viterbi Sch Engn, Los Angeles, CA 90089 USA. <br>
[Kunc, Dominika; Perz, Bartosz; Kazienko, Przemyslaw; Saganowski, Stanislaw] Wroclaw Univ Sci &amp; Technol, Dept Artificial Intelligence, PL-50370 Wroclaw, Poland. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Avramidis, K (corresponding author), Univ Southern Calif, Viterbi Sch Engn, Los Angeles, CA 90089 USA.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
avramidi@usc.edu; dominika.kunc@pwr.edu.pl; bartosz.perz@pwr.edu.pl; kadsul@usc.edu; tiantiaf@usc.edu; kazienko@pwr.edu.pl; stanislaw.saganowski@pwr.edu.pl; shri@ee.usc.edu</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
University of Southern California; Wroclaw University of Science &amp; Technology</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Perz, Bartosz</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-9940-5117&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Kazienko, Przemysław</display_name>&nbsp;</td><td>F-1849-2014&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Kazienko, Przemyslaw</display_name>&nbsp;</td><td>F-1849-2014&nbsp;</td><td>0000-0001-5868-356X&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Kunc, Dominika</display_name>&nbsp;</td><td>LIH-0162-2024&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Kunc, Dominika</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-2918-0423&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Perz, Bartosz</display_name>&nbsp;</td><td>AHE-2470-2022&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Feng, Tiantian</display_name>&nbsp;</td><td>X-4695-2019&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Saganowski, Stanisław</display_name>&nbsp;</td><td>AAF-1696-2019&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Information Systems; Computer Science, Interdisciplinary Applications; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>I4D7I</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2168-2194</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>2168-2208</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE J BIOMED HEALTH</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE J. Biomed. Health Inform.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>13</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>NSF SCH</grant_agency>&nbsp;</td><td>
<div>2204942&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>DARPA</grant_agency>&nbsp;</td><td>
<div>N660012324006&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Toyota</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Science Centre, Poland</grant_agency>&nbsp;</td><td>
<div>2020/37/B/ST6/03806&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>NAWA STER Programme Internationalisation of Wroclaw University of Science and Technology Doctoral School</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Polish National Agency for Academic Exchange (NAWA) - the Bekker Programme</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was conducted at USC SAIL and supported in part by NSF SCH under Grant 2204942, in part by DARPA under Cooperative Agreement number N660012324006, in part by Toyota, in part by National Science Centre, Poland under Project 2020/37/B/ST6/03806,in part by the NAWA STER Programme Internationalisation of Wroclaw University of Science and Technology Doctoral School, and in part by the Polish National Agency for Academic Exchange (NAWA) - the BekkerProgramme.</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 24 of 37</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Improving electrocardiogram-based detection of rare genetic heart disease using transfer learning: An application to phospholamban p.Arg14del mutation carriers</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Lopes, RR (Lopes, Ricardo R.); Bleijendaal, H (Bleijendaal, Hidde); Ramos, LA (Ramos, Lucas A.); Verstraelen, TE (Verstraelen, Tom E.); Amin, AS (Amin, Ahmad S.); Wilde, AAM (Wilde, Arthur A. M.); Pinto, YM (Pinto, Yigal M.); de Mol, BAJM (de Mol, Bas A. J. M.); Marquering, HA (Marquering, Henk A.)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>COMPUTERS IN BIOLOGY AND MEDICINE</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>131</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>104262</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1016/j.compbiomed.2021.104262</value>&nbsp;&nbsp;<b>Early Access Date:</b> 
<value>FEB 2021</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2021 APR</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>30</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>35</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
1</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
10</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>31</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
The pathogenic mutation p.Arg14del in the gene encoding Phospholamban (PLN) is known to cause cardiomyopathy and leads to increased risk of sudden cardiac death. Automatic tools might improve the detection of patients with this rare disease. Deep learning is currently the state-of-the-art in signal processing but requires large amounts of data to train the algorithms. In situations with relatively small amounts of data, like PLN, transfer learning may improve accuracy. We propose an ECG-based detection of the PLN mutation using transfer learning from a model originally trained for sex identification. The sex identification model was trained with 256,278 ECGs and subsequently finetuned for PLN detection (155 ECGs of patients with PLN) with two control groups: a balanced age/sex matched group and a randomly selected imbalanced population. The data was split in 10 folds and 20% of the training data was used for validation and early stopping. The models were evaluated with the area under the receiver operating characteristic curve (AUROC) of the testing data. We used gradient activation for explanation of the prediction models. The models trained with transfer learning outperformed the models trained from scratch for both the balanced (AUROC 0.87 vs AUROC 0.71) and imbalanced (AUROC 0.0.90 vs AUROC 0.65) population. The proposed approach was able to improve the accuracy of a rare disease detection model by transfer learning information from a non-manual annotated and abundant label with only limited data available.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000631651900003</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>33607378</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Phospholamban; Deep learning; Transfer learning; ECG analysis; Cardiomyopathy; Genetic heart disease</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Lopes, Ricardo R.; Ramos, Lucas A.; Marquering, Henk A.] Univ Amsterdam, Dept Biomed Engn &amp; Phys, Amsterdam UMC, Meibergdreef 9, NL-1105 AZ Amsterdam, Netherlands. <br>
[Lopes, Ricardo R.; Marquering, Henk A.] Univ Amsterdam, Dept Radiol &amp; Nucl Med, Amsterdam UMC, Amsterdam, Netherlands. <br>
[Bleijendaal, Hidde; Ramos, Lucas A.] Univ Amsterdam, Dept Clin Epidemiol Biostat &amp; Bioinformat, Amsterdam UMC, Amsterdam, Netherlands. <br>
[Bleijendaal, Hidde; Verstraelen, Tom E.; Amin, Ahmad S.; Wilde, Arthur A. M.; Pinto, Yigal M.; de Mol, Bas A. J. M.] Univ Amsterdam, Amsterdam UMC, Dept Clin &amp; Expt Cardiol, Amsterdam Cardiovasc Sci,Heart Ctr, Amsterdam, Netherlands. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Lopes, RR (corresponding author), Univ Amsterdam, Dept Biomed Engn &amp; Phys, Amsterdam UMC, Meibergdreef 9, NL-1105 AZ Amsterdam, Netherlands.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
r.riccilopes@amsterdamumc.nl</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
University of Amsterdam; University of Amsterdam; University of Amsterdam; University of Amsterdam</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Verstraelen, Tom</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-4340-6815&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Bleijendaal, Hidde</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-1506-8385&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Ramos, Lucas</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-7738-9200&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Ramos, Lucas</display_name>&nbsp;</td><td>W-7401-2019&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Verstraelen, Tom</display_name>&nbsp;</td><td>ABE-5949-2020&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Ricci Lopes, Ricardo</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-5515-1488&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>de Mol, Bastian</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-6888-9430&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Marquering, Henk</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-1414-6313&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Amin, Ahmad</display_name>&nbsp;</td><td>NFT-0652-2025&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Marquering, Henk</display_name>&nbsp;</td><td>B-9954-2013&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Lopes, Ricardo</display_name>&nbsp;</td><td>Y-9109-2019&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>PERGAMON-ELSEVIER SCIENCE LTD</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>THE BOULEVARD, LANGFORD LANE, KIDLINGTON, OXFORD OX5 1GB, ENGLAND</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Biology; Computer Science, Interdisciplinary Applications; Engineering, Biomedical; Mathematical &amp; Computational Biology</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Life Sciences &amp; Biomedicine - Other Topics; Computer Science; Engineering; Mathematical &amp; Computational Biology</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>RA8EI</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0010-4825</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1879-0534</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>COMPUT BIOL MED</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Comput. Biol. Med.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>8</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>ITEA3 PARTNER</grant_agency>&nbsp;</td><td>
<div>16017&nbsp;</div>
<div>14003&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>University of Amsterdam Research Priority Area Medical Integromics</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>Part of this work was funded by ITEA3 PARTNER (16017), ITEA3 Medolution (14003) and University of Amsterdam Research Priority Area Medical Integromics.</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>hybrid</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 25 of 37</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Heartbeats Classification Using Hybrid Time-Frequency Analysis and Transfer Learning Based on ResNet</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Zhang, YT (Zhang, Yatao); Li, JY (Li, Junyan); Wei, SS (Wei, Shoushui); Zhou, FY (Zhou, Fengyu); Li, D (Li, Dong)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE JOURNAL OF BIOMEDICAL AND HEALTH INFORMATICS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>25</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>11</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>4175-4184</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/JBHI.2021.3085318</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2021 NOV</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>56</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>66</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
1</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
90</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>33</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
The classification of heartbeats is an important method for cardiac arrhythmia analysis. This study proposes a novel heartbeat classification method using hybrid time-frequency analysis and transfer learning based on ResNet-101. The proposed method has the following major advantages over the afore-mentioned methods: it avoids the need for manual features extraction in the traditional machine learning method, and it utilizes 2-D time-frequency diagrams which provide not only frequency and energy information but also preserve the morphological characteristic within the ECG recordings, and it owns enough deep to make better use of performance of CNN. The method deploys a hybrid time-frequency analysis of the Hilbert transform (HT) and the Wigner-Ville distribution (WVD) to transform 1-D ECG recordings into 2-D time-frequency diagrams which were then fed into a transfer learning classifier based on ResNet-101 for two classification tasks (i.e., 5 heartbeat categories assigned by the ANSI/AAMI standard (i.e., N, V, S, Q and F) and 14 original beat kinds of the MIT/BIH arrhythmia database). For 5 heartbeat categories classification, the results show the F1-score of N, V, S, Q and F categories are F-N 0.9899, F-V 0.9845, F-S 0.9376, F-Q 0.9968, F-F 0.8889, respectively, and the overall F1-score is 0.9595 using the combination data balancing. The results show the average values for accuracy, sensitivity, specificity, predictive value and F1-score on test set for 14 beat kinds the MIT-BIH arrhythmia database are 99.75%, 91.36%, 99.85%, 90.81% and 0.9016, respectively. Compared with other methods, the proposed method can yield more accurate results.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000714714200016</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>34077377</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Heartbeat classification; arrhythmias; time-frequency analysis; Transfer learning; ResNet-101</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
ATRIAL-FIBRILLATION; NEURAL-NETWORK; ECG</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Zhang, Yatao; Li, Dong] Shandong Univ, Sch Mech Elect &amp; Informat Engn, Weihai 264209, Peoples R China. <br>
[Li, Junyan] Weihaiwei Peoples Hosp, Dept Neurol, Weihai 264209, Peoples R China. <br>
[Wei, Shoushui; Zhou, Fengyu] Shandong Univ, Sch Control Sci &amp; Engn, Jinan 17923, Peoples R China. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Zhang, YT (corresponding author), Shandong Univ, Sch Mech Elect &amp; Informat Engn, Weihai 264209, Peoples R China.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
zytboy@sdu.edu.cn; whkunyushan@163.com; sswei@sdu.edu.cn; zhoufengyu@sdu.edu.cn; dongli@sdu.edu.cn</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Shandong University; Shandong University</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Jiang, Bin</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-6621-3979&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>zhang, yatao</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-6152-0806&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Zhou, Fengyu</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-5140-7036&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Information Systems; Computer Science, Interdisciplinary Applications; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>WR7ZD</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2168-2194</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>2168-2208</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE J BIOMED HEALTH</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE J. Biomed. Health Inform.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>10</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Natural Science Foundation of China</grant_agency>&nbsp;</td><td>
<div>82072014&nbsp;</div>
<div>62076149&nbsp;</div>
<div>61702138&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>China Postdoctoral Science Foundation</grant_agency>&nbsp;</td><td>
<div>2017M612280&nbsp;</div>
<div>2020T130368&nbsp;</div>
<div>2019M662360&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Intergovernmental Project of National Key Research, and Development Program/Hong Kong, Macao, and Taiwan key projects</grant_agency>&nbsp;</td><td>
<div>SQ2019YFE010670&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Key R&amp;D project of Shandong Province</grant_agency>&nbsp;</td><td>
<div>2018GSF118133&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Young Scholars Program of Shandong University, Weihai</grant_agency>&nbsp;</td><td>
<div>1050501318006&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Science and Technology Development Plan of Weihai City</grant_agency>&nbsp;</td><td>
<div>1050413421912&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported in part by the National Natural Science Foundation of China under Grants 82072014, 62076149, and 61702138, in part by the China Postdoctoral Science Foundation under Grants 2017M612280, 2020T130368, and 2019M662360, in part by the Intergovernmental Project of National Key Research, and Development Program/Hong Kong, Macao, and Taiwan key projects under Grant SQ2019YFE010670, in part by the Key R&amp;D project of Shandong Province under Grant 2018GSF118133, in part by the Young Scholars Program of Shandong University, Weihai under Grant 1050501318006, and in part by the Science and Technology Development Plan of Weihai City under Grant 1050413421912.</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 26 of 37</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>MVKT-ECG: Efficient single-lead ECG classification for multi-label arrhythmia by multi-view knowledge transferring</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Qin, YZ (Qin, Yuzhen); Sun, L (Sun, Li); Chen, H (Chen, Hui); Yang, WM (Yang, Wenming); Zhang, WQ (Zhang, Wei-Qiang); Fei, JT (Fei, Jintao); Wang, GJ (Wang, Guijin)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>COMPUTERS IN BIOLOGY AND MEDICINE</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>166</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>107503</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1016/j.compbiomed.2023.107503</value>&nbsp;&nbsp;<b>Early Access Date:</b> 
<value>OCT 2023</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2023 NOV</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>11</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>11</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
14</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
91</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>48</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Electrocardiogram (ECG) is a widely used technique for diagnosing cardiovascular disease. The widespread emergence of smart ECG devices has sparked the demand for intelligent single-lead ECG-based diagnostic systems. However, it is challenging to develop a single-lead-based ECG interpretation model for multiple disease diagnosis due to the lack of some key disease information. We aim to improve the diagnostic capabilities of single-lead ECG for multi-label disease classification in a new teacher-student manner, where the teacher trained by multi-lead ECG educates a student who observes only single-lead ECG We present a new disease aware Contrastive Lead-information Transferring (CLT) to improve the mutual disease information between the single-lead-based ECG interpretation model and multi-lead-based ECG interpretation model. Moreover, We modify the traditional Knowledge Distillation into Multi-label disease Knowledge Distillation (MKD) to make it applicable for multi-label disease diagnosis. The whole knowledge transferring process is inter-lead Multi -View Knowledge Transferring of ECG (MVKT-ECG). By employing the training strategy, we can effectively transfer comprehensive disease knowledge from various views of ECG, such as the 12-lead ECG, to a single-lead-based ECG interpretation model. This enables the model to extract intricate details from single-lead ECG signals and enhances the model's capability of diagnosing and identifying single-lead signals. Extensive experiments on two commonly used public multi-label datasets, ICBEB2018 and PTB-XL demonstrate that our MVKT-ECG yields exceptional diagnostic performance improvements for single-lead ECG. The student outperforms its baseline observably on the PTB-XL dataset (1.3 % on PTB.super, and 1.4 % on PTB.sub), and on ICBEB2018 dataset (3.2 %).</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001088270400001</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>37806055</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Single-lead Electrocardiogram; Arrhythmia classification; Deep learning; Knowledge transfer; Neural network</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Qin, Yuzhen; Yang, Wenming] Tsinghua Univ, Shenzhen Int Grad Sch, Shenzhen 518071, Peoples R China. <br>
[Sun, Li; Chen, Hui; Zhang, Wei-Qiang; Wang, Guijin] Tsinghua Univ, Dept Elect Engn, Beijing 100084, Peoples R China. <br>
[Fei, Jintao] Beijing Tsinghua Changgung Hosp, Beijing 102218, Peoples R China. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Wang, GJ (corresponding author), Tsinghua Univ, Dept Elect Engn, Beijing 100084, Peoples R China.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
wangguijin@tsinghua.edu.cn</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Tsinghua University; Tsinghua University</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Yang, Wenming</display_name>&nbsp;</td><td>HJH-8634-2023&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>PERGAMON-ELSEVIER SCIENCE LTD</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>THE BOULEVARD, LANGFORD LANE, KIDLINGTON, OXFORD OX5 1GB, ENGLAND</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Biology; Computer Science, Interdisciplinary Applications; Engineering, Biomedical; Mathematical &amp; Computational Biology</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Life Sciences &amp; Biomedicine - Other Topics; Computer Science; Engineering; Mathematical &amp; Computational Biology</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>U9XR8</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0010-4825</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1879-0534</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>COMPUT BIOL MED</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Comput. Biol. Med.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>10</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Natural Science Foun-dation of China</grant_agency>&nbsp;</td><td>
<div>62276153&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This research was supported by the National Natural Science Foun-dation of China under Grant No. 62276153.</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 27 of 37</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Acoustic feature based unsupervised approach of heart sound event detection</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Das, S (Das, Sangita); Pal, S (Pal, Saurabh); Mitra, M (Mitra, Madhuchhanda)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>COMPUTERS IN BIOLOGY AND MEDICINE</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>126</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>103990</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1016/j.compbiomed.2020.103990</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2020 NOV</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>28</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>29</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
3</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
56</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>46</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
This paper represents an unsupervised approach to detect the positions of S1, S2 heart sound events in a Phonocardiogram (PCG) recording. Insufficiency of correctly annotated heart sound database drives us to investigate unsupervised techniques. Gammatone filter bank features are used to characterize the spectral pattern of fundamental heart sound events from noise contaminated PCG data. An unsupervised spectral clustering technique is employed for segmentation of S1/S2 and non-S1/S2 heart sound events. A Feature winning score is computed to identify the S1/52 and non-S1/S2 frames. Finally, time based threshold is applied to detect the accurate positions of S1 and S2 heart sounds. The performance of spectral clustering is compared with other clustering methods. The proposed method offers a maximum Fl-score of 98% and 92.5% for normal and abnormal PCG data respectively on 2016 PhysioNet/CinC challenge dataset. The heart sound annotation algorithm provided by PhysioNet has been used as the ground truth after hand correction.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000582723600001</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>32987200</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Heart sound; Phonocardiogram (PCG); Spectral clustering; Heart sound segmentation</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
NEURAL-NETWORK CLASSIFICATION; SEGMENTATION; RECOGNITION; NOISE; IDENTIFICATION; SEPARATION; IMAGE</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Das, Sangita; Pal, Saurabh; Mitra, Madhuchhanda] Univ Calcutta, Dept Appl Phys, 92APC Rd, Kolkata 700009, W Bengal, India. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Mitra, M (corresponding author), Univ Calcutta, Dept Appl Phys, 92APC Rd, Kolkata 700009, W Bengal, India.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
mmaphy@caluniv.ac.in</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
University of Calcutta</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Pal, Saurabh</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-5139-6573&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>PERGAMON-ELSEVIER SCIENCE LTD</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>THE BOULEVARD, LANGFORD LANE, KIDLINGTON, OXFORD OX5 1GB, ENGLAND</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Biology; Computer Science, Interdisciplinary Applications; Engineering, Biomedical; Mathematical &amp; Computational Biology</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Life Sciences &amp; Biomedicine - Other Topics; Computer Science; Engineering; Mathematical &amp; Computational Biology</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>OH6RT</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0010-4825</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1879-0534</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>COMPUT BIOL MED</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Comput. Biol. Med.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>11</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Council of Scientific and Industrial Research (CSIR)</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>The authors thank Dr. Santanu Guha, Professor and Head, Department of Cardiology, Medical College, Kolkata for his supports and valuable suggestions. We gratefully acknowledge Council of Scientific and Industrial Research (CSIR) for their financial support.</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 28 of 37</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Inter-Patient ECG Classification With Symbolic Representations and Multi-Perspective Convolutional Neural Networks</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Niu, JH (Niu, Jinghao); Tang, YQ (Tang, Yongqiang); Sun, ZY (Sun, Zhengya); Zhang, WS (Zhang, Wensheng)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE JOURNAL OF BIOMEDICAL AND HEALTH INFORMATICS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>24</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>5</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>1321-1332</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/JBHI.2019.2942938</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2020 MAY</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>104</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>112</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
2</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
85</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>47</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
This paper presents a novel deep learning framework for the inter-patient electrocardiogram (ECG) heartbeat classification. A symbolization approach especially designed for ECG is introduced, which can jointly represent the morphology and rhythm of the heartbeat and alleviate the influence of inter-patient variation through baseline correction. The symbolic representation of the heartbeat is used by a multi-perspective convolutional neural network (MPCNN) to learn features automatically and classify the heartbeat. We evaluate our method for the detection of the supraventricular ectopic beat (SVEB) and ventricular ectopic beat (VEB) on MIT-BIH arrhythmia dataset. Compared with the state-of-the-art methods based on manual features or deep learning models, our method shows superior performance: the overall accuracy of 96.4%, F1 scores for SVEB and VEB of 76.6% and 89.7%, respectively. The ablation study on our method validates the effectiveness of the proposed symbolization approach and joint representation architecture, which can help the deep learning model to learn more general features and improve the ability of generalization for unseen patients. Because our method achieves a competitive inter-patient heartbeat classification performance without complex handcrafted features or the intervention of the human expert, it can also be adjusted to handle various other tasks relative to ECG classification.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000535614100009</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>31545750</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Heart beat; Electrocardiography; Heart rate variability; Deep learning; Feature extraction; Task analysis; Informatics; ECG classification; biomedical monitoring; convolutional neural network; deep learning</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
HEARTBEAT CLASSIFICATION; ARRHYTHMIA DETECTION; FEATURES</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Niu, Jinghao; Tang, Yongqiang; Sun, Zhengya; Zhang, Wensheng] Chinese Acad Sci, Inst Automat, Beijing 100190, Peoples R China. <br>
[Niu, Jinghao; Tang, Yongqiang; Sun, Zhengya; Zhang, Wensheng] Univ Chinese Acad Sci, Beijing 100049, Peoples R China. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Zhang, WS (corresponding author), Chinese Acad Sci, Inst Automat, Beijing 100190, Peoples R China.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
niujinghao2015@ia.ac.cn; tangyongqiang2014@ia.ac.cn; zhengya.sun@ia.ac.cn; zhangwenshengia@hotmail.com</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Chinese Academy of Sciences; Institute of Automation, CAS; Chinese Academy of Sciences; University of Chinese Academy of Sciences, CAS</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Niu, Jinghao</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-9598-297X&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>tang, yong qiang</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-9333-8200&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Information Systems; Computer Science, Interdisciplinary Applications; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>LR3SW</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2168-2194</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>2168-2208</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE J BIOMED HEALTH</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE J. Biomed. Health Inform.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>12</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Key R&amp;D Program of China</grant_agency>&nbsp;</td><td>
<div>2017YFC0803700&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Natural Science Foundation of China</grant_agency>&nbsp;</td><td>
<div>61432008&nbsp;</div>
<div>61876183&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Beijing Municipal Natural Science Foundation</grant_agency>&nbsp;</td><td>
<div>4172063&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported in part by the National Key R&amp;D Program of China (2017YFC0803700), in part by the National Natural Science Foundation of China under Grant 61432008 and Grant 61876183, and in part by Beijing Municipal Natural Science Foundation under Grant 4172063.</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 29 of 37</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Assessing the signal quality of electrocardiograms from varied acquisition sources: A generic machine learning pipeline for model generation</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Albaba, A (Albaba, Adnan); Simo, N (Simo, Neide); Wang, YY (Wang, Yuyang); Hendriks, RC (Hendriks, Richard C.); De Raedt, W (De Raedt, Walter); Van Hoof, C (Van Hoof, Chris)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>COMPUTERS IN BIOLOGY AND MEDICINE</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>130</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>104164</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1016/j.compbiomed.2020.104164</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2021 MAR</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>17</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>17</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
0</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
16</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>43</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Background and objective: Long-term electrocardiogram monitoring comes at the expense of signal quality. During unconstrained movements, the electrocardiogram is often corrupted by motion artefacts, which can lead to inaccurate physiological information. In this situation, automated quality assessment methods are useful to increase the reliability of the measurements. A generic machine learning pipeline that generates classification models for electrocardiogram quality assessment is presented in this article. The presented pipeline is tested on signals from varied acquisition sources, towards selecting segments that can be used for heart rate analysis in lifestyle applications.
<br>Methods: Electrocardiogram recordings from traditional, wearable and ubiquitous devices, are segmented in 10 s windows and manually labeled by experienced researchers into two quality classes. To capture the electrocardiogram dynamics, a comprehensive set of 43 features is extracted from each segment, based on the time-domain signal, its Fast Fourier Transform, the Autocorrelation function and the Stationary Wavelet Transform. To select the most relevant features for each acquisition source we employ both a customized hybrid approach and the state-of-the-art Neighborhood Component Analysis method and compare them. Support Vector Machines (SVM), Decision Trees, K-Nearest-Neighbors and supervised ensemble methods are tested as possible binary classifiers.
<br>Results: The results for the best performing models on traditional, wearable and ubiquitous electrocardiogram datasets are, respectively: balanced-accuracy: 89%, F1-score: 93% with the Fine Gaussian SVM model and 10 features; balanced-accuracy: 93%, F1-score: 93% with the Fine Gaussian SVM model and 11 features; balanced accuracy: 95%, F1-score: 86%, with the Fine Gaussian SVM model and 8 features.
<br>Conclusions: According to the results, our generic pipeline can generate classification models tailored to individual acquisition sources, provided that a standard Lead I or Lead II is available. Such models accurately establish whether the electrocardiogram quality is good or bad for heart rate analysis. Furthermore, removing bad quality segments decreases errors in heart rate calculation.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000623917000009</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>33360108</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Electrocardiogram; Wearables; Ubiquitous; Non-contact; Classification; Feature selection; Motion artefact; Signal quality</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
QRS DETECTION; ECG</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Albaba, Adnan; Simo, Neide; Van Hoof, Chris] Univ Leuven, Dept Elect Engn, Leuven, Belgium. <br>
[Albaba, Adnan; Simo, Neide; De Raedt, Walter; Van Hoof, Chris] IMEC, Connected Hlth Solut Grp, Leuven, Belgium. <br>
[Wang, Yuyang; Hendriks, Richard C.] Delft Univ Technol, Dept Elect Engn, Delft, Netherlands. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
De Raedt, W (corresponding author), IMEC, Connected Hlth Solut Grp, Leuven, Belgium.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
adnan.albaba@imec.be; neide.simoescapela@kuleuven.be; y.wang-54@student.tudelft.nl; r.c.hendriks@tudelft.nl; walter.deraedt@imec.be; chris.vanhoof@imec.be</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
KU Leuven; Interuniversity Microelectronics Centre; Delft University of Technology</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>De Raedt, Walter</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-7117-7976&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Van Hoof, Chris</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-4645-3326&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Albaba, Adnan</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-9945-2650&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Simoes-Capela, Neide</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-0550-4964&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Albaba, Adnan</display_name>&nbsp;</td><td>MAH-3139-2025&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>PERGAMON-ELSEVIER SCIENCE LTD</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>THE BOULEVARD, LANGFORD LANE, KIDLINGTON, OXFORD OX5 1GB, ENGLAND</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Biology; Computer Science, Interdisciplinary Applications; Engineering, Biomedical; Mathematical &amp; Computational Biology</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Life Sciences &amp; Biomedicine - Other Topics; Computer Science; Engineering; Mathematical &amp; Computational Biology</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>QP6AG</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0010-4825</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1879-0534</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>COMPUT BIOL MED</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Comput. Biol. Med.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>11</value>
</td>
</tr>

<tr>
<td>
<b>Open Access:</b>

<value>hybrid</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 30 of 37</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Deep Representation Learning With Sample Generation and Augmented Attention Module for Imbalanced ECG Classification</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Zubair, M (Zubair, Muhammad); Woo, S (Woo, Sungpil); Lim, S (Lim, Sunhwan); Kim, D (Kim, Daeyoung)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE JOURNAL OF BIOMEDICAL AND HEALTH INFORMATICS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>28</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>5</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>2461-2472</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/JBHI.2023.3325540</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2024 MAY</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>10</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>10</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
5</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
42</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>61</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Developing an efficient heartbeat monitoring system has become a focal point in numerous healthcare applications. Specifically, in the last few years, heartbeat classification for arrhythmia detection has gained considerable interest from researchers. This paper presents a novel deep representation learning method for the efficient detection of arrhythmic beats. To mitigate the issues associated with the imbalanced data distribution, a novel re-sampling strategy is introduced. Unlike the existing oversampling methods, the proposed technique transforms majority-class samples into minority-class samples with a novel translation loss function. This approach assists the model in learning a more generalized representation of crucially important minority class samples. Moreover, by exploiting an auxiliary feature, an augmented attention module is designed that focuses on the most relevant and target-specific information. We adopted an inter-patient classification paradigm to evaluate the proposed method. The experimental results of this study on the MIT-BIH arrhythmia database clearly indicate that the proposed model with augmented attention mechanism and over-sampling strategy significantly learns a balanced deep representation and improves the classification performance of vital heartbeats.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001221547700062</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>37851553</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Arrhythmia detection; beat classification; imbalanced learning; remote health monitoring</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
CONVOLUTIONAL NEURAL-NETWORK; HEARTBEAT CLASSIFICATION; MORPHOLOGY; FEATURES; MODEL</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Zubair, Muhammad; Woo, Sungpil; Lim, Sunhwan] Elect &amp; Telecommun Res Inst ETRI, Autonomous IoT Res Sect, Daejeon 34129, South Korea. <br>
[Woo, Sungpil; Kim, Daeyoung] Korea Adv Inst Sci &amp; Technol KAIST, Dept Sch Comp, Daejeon 34141, South Korea. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Woo, S (corresponding author), Elect &amp; Telecommun Res Inst ETRI, Autonomous IoT Res Sect, Daejeon 34129, South Korea.<br>Woo, S (corresponding author), Korea Adv Inst Sci &amp; Technol KAIST, Dept Sch Comp, Daejeon 34141, South Korea.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
zubair5608@etri.re.kr; woosungpil@etri.re.kr; shlim@etri.re.kr; kimd@kaist.ac.kr</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Electronics &amp; Telecommunications Research Institute - Korea (ETRI); Korea Advanced Institute of Science &amp; Technology (KAIST)</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Zubair, Muhammad</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-3595-4969&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>ZUBAIR, MUHAMMAD</display_name>&nbsp;</td><td>JVN-7799-2024&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Information Systems; Computer Science, Interdisciplinary Applications; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>QN4V6</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2168-2194</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>2168-2208</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE J BIOMED HEALTH</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE J. Biomed. Health Inform.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>12</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Institute of Information &amp; Communications Technology Planning Evaluation</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>No Statement Available</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>Bronze</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 31 of 37</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>A Generic Quality Control Framework for Fetal Ultrasound Cardiac Four-Chamber Planes</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Dong, JB (Dong, Jinbao); Liu, SF (Liu, Shengfeng); Liao, YM (Liao, Yimei); Wen, HX (Wen, Huaxuan); Lei, BY (Lei, Baiying); Li, SL (Li, Shengli); Wang, TF (Wang, Tianfu)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE JOURNAL OF BIOMEDICAL AND HEALTH INFORMATICS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>24</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>4</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>931-942</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/JBHI.2019.2948316</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2020 APR</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>72</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>95</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
1</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
55</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>52</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Quality control/assessment of ultrasound (US) images is an essential step in clinical diagnosis. This process is usually done manually, suffering from some drawbacks, such as dependence on operator's experience and extensive labors, as well as high inter- and intra-observer variation. Automatic quality assessment of US images is therefore highly desirable. Fetal US cardiac four-chamber plane (CFP) is one of the most commonly used cardiac views, which was used in the diagnosis of heart anomalies in the early 1980s. In this paper, we propose a generic deep learning framework for automatic quality control of fetal US CFPs. The proposed framework consists of three networks: (1) a basic CNN (B-CNN), roughly classifying four-chamber views from the raw data; (2) a deeper CNN (D-CNN), determining the gain and zoom of the target images in a multi-task learning manner; and (3) the aggregated residual visual block net (ARVBNet), detecting the key anatomical structures on a plane. Based on the output of the three networks, overall quantitative score of each CFP is obtained, so as to achieve fully automatic quality control. Experiments on a fetal US dataset demonstrated our proposed method achieved a highest mean average precision (mAP) of 93.52% at a fast speed of 101 frames per second (FPS). In order to demonstrate the adaptability and generalization capacity, the proposed detection network (i.e., ARVBNet) has also been validated on the PASCAL VOC dataset, obtaining a highest mAP of 81.2% when input size is approximately 300 x 300.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000525354900001</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>31634851</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Quality control; Anatomical structure; Quality assessment; Ultrasonic imaging; Real-time systems; Quality control; cardiac four-chamber planes (CFPs); convolutional neural network (CNN); real-time object detection; aggregated residual visual block (ARVB)</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
HEART</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Dong, Jinbao; Liu, Shengfeng; Lei, Baiying; Wang, Tianfu] Shenzhen Univ, Natl Reg Key Technol Engn Lab Med Ultrasound, Guangdong Prov Key Lab Biomed Measurements &amp; Ultr, Sch Biomed Engn,Hlth Sci Ctr, Shenzhen 518060, Peoples R China. <br>
[Liao, Yimei; Wen, Huaxuan; Li, Shengli] Southern Med Univ, Shenzhen Matern &amp; Child Healthcare Hosp, Dept Ultrasound, Shenzhen 518028, Peoples R China. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Wang, TF (corresponding author), Shenzhen Univ, Natl Reg Key Technol Engn Lab Med Ultrasound, Guangdong Prov Key Lab Biomed Measurements &amp; Ultr, Sch Biomed Engn,Hlth Sci Ctr, Shenzhen 518060, Peoples R China.<br>Li, SL (corresponding author), Southern Med Univ, Shenzhen Matern &amp; Child Healthcare Hosp, Dept Ultrasound, Shenzhen 518028, Peoples R China.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
995238847@qq.com; liusf2009@163.com; liaoyimei1987@qq.com; whxwell@126.com; leiby@szu.edu.cn; lishengli63@126.com; tfwang@szu.edu.cn</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Shenzhen University; Southern Medical University - China; Shenzhen Maternity &amp; Child Healthcare Hospital</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>liu, sheng feng</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-6902-3329&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>liu, shengfeng</display_name>&nbsp;</td><td>N-2303-2018&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Lei, Baiying</display_name>&nbsp;</td><td>GQO-8422-2022&nbsp;</td><td>0000-0002-3087-2550&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Liao, Yimei</display_name>&nbsp;</td><td>ABG-8261-2021&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Li, Shengli</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-0570-4165&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Information Systems; Computer Science, Interdisciplinary Applications; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>LC5GX</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2168-2194</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>2168-2208</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE J BIOMED HEALTH</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE J. Biomed. Health Inform.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>12</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Natural Science Foundation of China</grant_agency>&nbsp;</td><td>
<div>81571758&nbsp;</div>
<div>81771922&nbsp;</div>
<div>61871274&nbsp;</div>
<div>61801305&nbsp;</div>
<div>61501305&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Key Research and Develop Program</grant_agency>&nbsp;</td><td>
<div>2016YFC0104700&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported in part by the National Natural Science Foundation of China under Grant 81571758, Grant 81771922, Grant 61871274, Grant 61801305, and Grant 61501305; and in part by the National Key Research and Develop Program (2016YFC0104700).</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 32 of 37</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Improving the Quality of Fetal Heart Ultrasound Imaging With Multihead Enhanced Self-Attention and Contrastive Learning</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Zhang, YY (Zhang, Yingying); Zhu, HG (Zhu, Haogang); Cheng, J (Cheng, Jian); Wang, JY (Wang, Jingyi); Gu, XY (Gu, Xiaoyan); Han, JC (Han, Jiancheng); Zhang, Y (Zhang, Ye); Zhao, Y (Zhao, Ying); He, YH (He, Yihua); Zhang, HJ (Zhang, Hongjia)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE JOURNAL OF BIOMEDICAL AND HEALTH INFORMATICS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>27</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>11</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>5518-5529</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/JBHI.2023.3303573</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2023 NOV</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>9</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>10</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
4</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
22</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>51</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Fetal congenital heart disease (FCHD) is a common, serious birth defect affecting similar to 1% of newborns annually. Fetal echocardiography is the most effective and important technique for prenatal FCHD diagnosis. The pre-requisites for accurate ultrasound FCHD diagnosis are accurate view recognition and high-quality diagnostic view extraction. However, these manual clinical procedures have drawbacks such as, varying technical capabilities and in-efficiency. Therefore, the automatic identification of high-quality multiview fetal heart scan images is highly desirable to improve prenatal diagnosis efficiency and accuracy of FCHD. Here, we present a framework for multiview fetal heart ultrasound image recognition and quality assessment that comprises two parts: a multiview classification and localization network (MCLN) and an improved contrastive learning network (ICLN). In the MCLN, a multihead enhanced self-attention mechanism is applied to construct the classification network and identify six accurate and interpretable views of the fetal heart. In the ICLN, anatomical structure standardization and image clarity are considered. With contrastive learning, the absolute loss, feature relative loss and predicted value relative loss are combined to achieve favorable quality assessment results. Experiments show that the MCLN outperforms other state-of-the-art networks by 1.52-13.61% when determining the F1 score in six standard view recognition tasks, and the ICLN is comparable to the performance of expert cardiologists in the quality assessment of fetal heart ultrasound images, reaching 97% on a test set within 2 points for the four-chamber view task. Thus, our architecture offers great potential in helping cardiologists improve quality control for fetal echocardiographic images in clinical practice.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001129955100029</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>37556337</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Contrastive learning; fetal congenital heart disease; fetal echocardiography; view recognition; quality assessment; multihead enhanced self-attention</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
NEURAL-NETWORKS; ECHOCARDIOGRAPHY; DISEASE; MANAGEMENT</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Zhang, Yingying] Beihang Univ, Sch Biol Sci &amp; Med Engn, Beijing 100191, Peoples R China. <br>
[Zhu, Haogang; Cheng, Jian] Beihang Univ, Sch Comp Sci &amp; Engn, State Key Lab Software Dev Environm, Beijing 100191, Peoples R China. <br>
[Zhu, Haogang; Cheng, Jian] Zhongguancun Lab, Beijing 102206, Peoples R China. <br>
[Wang, Jingyi; Gu, Xiaoyan; Han, Jiancheng; Zhang, Ye; Zhao, Ying; He, Yihua] Capital Med Univ, Beijing Anzhen Hosp, Echocardiog Med Ctr, Beijing 100054, Peoples R China. <br>
[Zhang, Hongjia] Beijing Lab Cardiovasc Precis Med, Beijing 100029, Peoples R China. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Zhu, HG (corresponding author), Beihang Univ, Sch Comp Sci &amp; Engn, State Key Lab Software Dev Environm, Beijing 100191, Peoples R China.<br>Zhu, HG (corresponding author), Zhongguancun Lab, Beijing 102206, Peoples R China.<br>He, YH (corresponding author), Capital Med Univ, Beijing Anzhen Hosp, Echocardiog Med Ctr, Beijing 100054, Peoples R China.<br>Zhang, HJ (corresponding author), Beijing Lab Cardiovasc Precis Med, Beijing 100029, Peoples R China.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
zhangyingying@buaa.edu.cn; haogangzhu@buaa.edu.cn; jian_cheng@buaa.edu.cn; wangjingyianzhen@163.com; xiaoyan_gu@yahoo.com; han_jc1977@hotmail.com; zhang3389@qq.com; yingzhaoecho@163.com; heyihuae-cho@hotmail.com; zhanghongjia722@hotmail.com</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Beihang University; Beihang University; Zhongguancun Laboratory; Capital Medical University</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>贺, 艺华</display_name>&nbsp;</td><td>AFV-8567-2022&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Cheng, Jian</display_name>&nbsp;</td><td>J-9124-2017&nbsp;</td><td>0000-0003-1435-673X&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Wang, Jingyi</display_name>&nbsp;</td><td>AAS-7653-2020&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>zhang, yingying</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-0964-1774&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Zhang, Jian</display_name>&nbsp;</td><td>F-7865-2015&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Information Systems; Computer Science, Interdisciplinary Applications; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>DC9R2</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2168-2194</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>2168-2208</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE J BIOMED HEALTH</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE J. Biomed. Health Inform.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>12</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Key R&amp;D Program of China</grant_agency>&nbsp;</td><td>
<div>2021ZD0140407&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Natural Science Foundation of China</grant_agency>&nbsp;</td><td>
<div>U21A20523&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Beijing Natural Science Foundation</grant_agency>&nbsp;</td><td>
<div>L222152&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Beijing Municipal Administration of Hospitals Incubating Program</grant_agency>&nbsp;</td><td>
<div>PX2022026&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Fundamental Research Funds for the Central Universities, Beijing Advanced Innovation Center for Big Data-Based Precision Medicine, Beihang University, Beijing, China</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>State Key Laboratory of Software Development Environment, Beihang University, Beijing, China, Beijing Lab for Cardiovascular Precision Medicine</grant_agency>&nbsp;</td><td>
<div>PXM2018_014226_000013&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Beijing Key Laboratory of Maternal-Fetal Medicine in Fetal Heart Disease</grant_agency>&nbsp;</td><td>
<div>BZ0308&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported in part by the National Key R&amp;D Program of China under Grant 2021ZD0140407, in part by the National Natural Science Foundation of China under Grant U21A20523, in part by the Beijing Natural Science Foundation under Grant L222152, in part by the Beijing Municipal Administration of Hospitals Incubating Program under Grant PX2022026, in part by the Fundamental Research Funds for the Central Universities, Beijing Advanced Innovation Center for Big Data-Based Precision Medicine, Beihang University, Beijing, China, in part by the State Key Laboratory of Software Development Environment, Beihang University, Beijing, China, Beijing Lab for Cardiovascular Precision Medicine under Grant PXM2018_014226_000013, and in part by the Beijing Key Laboratory of Maternal-Fetal Medicine in Fetal Heart Disease under Grant BZ0308.</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>Bronze</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 33 of 37</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>rU-Net, Multi-Scale Feature Fusion and Transfer Learning: Unlocking the Potential of Cuffless Blood Pressure Monitoring With PPG and ECG</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Chen, JM (Chen, Jiaming); Zhou, XL (Zhou, Xueling); Feng, L (Feng, Lei); Ling, BWK (Ling, Bingo Wing-Kuen); Han, LY (Han, Lianyi); Zhang, HT (Zhang, Hongtao)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE JOURNAL OF BIOMEDICAL AND HEALTH INFORMATICS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>29</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>1</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>166-176</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/JBHI.2024.3483301</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2025 JAN</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>0</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>0</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
13</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
28</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>52</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
This study introduces an innovative deep-learning model for cuffless blood pressure estimation using PPG and ECG signals, demonstrating state-of-the-art performance on the largest clean dataset, PulseDB. The rU-Net architecture, a fusion of U-Net and ResNet, enhances both generalization and feature extraction accuracy. Accurate multi-scale feature capture is facilitated by short-time Fourier transform (STFT) time-frequency distributions and multi-head attention mechanisms, allowing data-driven feature selection. The inclusion of demographic parameters as supervisory information further elevates performance. On the calibration-based dataset, our model excels, achieving outstanding accuracy (SBP MAE +/- std: 4.49 +/- 4.86 mmHg, DBP MAE +/- std: 2.69 +/- 3.10 mmHg), surpassing AAMI standards and earning a BHS Grade A rating. Addressing the challenge of calibration-free data, we propose a fine-tuning-based transfer learning approach. Remarkably, with only 10% data transfer, our model attains exceptional accuracy (SBP MAE +/- std: 4.14 +/- 5.01 mmHg, DBP MAE +/- std: 2.48 +/- 2.93 mmHg). This study sets the stage for the development of highly accurate and reliable wearable cuffless blood pressure monitoring devices.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001394517600007</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>39423074</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Feature extraction; Accuracy; Estimation; Electrocardiography; Blood pressure; Time series analysis; Data models; Time-frequency analysis; Monitoring; electrocardiogram; photoplethysmography; STFT; transfer learning; transfer learning</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
WAVE-FORM; PHOTOPLETHYSMOGRAPHY</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Chen, Jiaming; Zhou, Xueling; Zhang, Hongtao] Fudan Univ, Sch Informat Sci &amp; Technol, Shanghai 200437, Peoples R China. <br>
[Chen, Jiaming; Zhou, Xueling; Feng, Lei; Han, Lianyi; Zhang, Hongtao] Greater Bay Area Inst Precis Med, Guangzhou 511462, Peoples R China. <br>
[Ling, Bingo Wing-Kuen] Guangdong Univ Technol, Sch Informat Engn, Guangzhou 510006, Peoples R China. <br>
[Han, Lianyi] Fudan Univ, Huashan Hosp, Sch Life Sci, Dept Dermatol, Shanghai 200437, Peoples R China. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Zhang, HT (corresponding author), Fudan Univ, Sch Informat Sci &amp; Technol, Shanghai 200437, Peoples R China.<br>Han, LY (corresponding author), Fudan Univ, Huashan Hosp, Sch Life Sci, Dept Dermatol, Shanghai 200437, Peoples R China.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
jmchen22@m.fudan.edu.cn; xueling_zhou@fudan.edu.cn; fenglei@ipm-gba.org.cn; yongquanling@gdut.edu.cn; hanlianyi@fudan.edu.cn; zhanghongtao@ipm-gba.org.cn</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Fudan University; Guangdong University of Technology; Fudan University</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Feng, Lei</display_name>&nbsp;</td><td>AAV-7856-2020&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>zhang, hongtao</display_name>&nbsp;</td><td>NXB-9922-2025&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Chen, Jiaming</display_name>&nbsp;</td><td>GXM-7165-2022&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Han, Lianyi</display_name>&nbsp;</td><td>D-1499-2009&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Information Systems; Computer Science, Interdisciplinary Applications; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>R9J4Q</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2168-2194</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>2168-2208</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE J BIOMED HEALTH</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE J. Biomed. Health Inform.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>11</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 34 of 37</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Learning Spatio-Temporal Pulse Representation With Global-Local Interaction and Supervision for Remote Prediction of Heart Rate</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Zhao, CC (Zhao, Changchen); Zhou, MH (Zhou, Menghao); Zhao, Z (Zhao, Zheng); Huang, B (Huang, Bin); Rao, B (Rao, Bing)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE JOURNAL OF BIOMEDICAL AND HEALTH INFORMATICS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>28</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>2</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>609-620</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/JBHI.2023.3252091</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2024 FEB</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>12</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>12</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
6</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
18</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>52</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Recent studies have demonstrated the benefit of extracting and fusing pulse signals from multi-scale region-of-interests (ROIs). However, these methods suffer from heavy computational load. This paper aims to effectively utilize multi-scale rPPG features with a more compact architecture. Inspired by recent research works exploring two-path architecture that leverages global and local information with bidirectional bridge in between. This paper designs a novel architecture Global-Local Interaction and Supervision Network (GLISNet), which uses a local path to learn representations in the original scale and a global path to learn representations in the other scale capturing multi-scale information. A light-weight rPPG signal generation block is attached to the output of each path that maps the pulse representation to the pulse output. A hybrid loss function is utilized enabling the local and global representations to learn directly from the training data. Extensive experiments are conducted on two publicly available datasets, and results demonstrate that GLISNet achieves superior performance in terms of signal-to-noise ratio (SNR), mean absolute error (MAE), and root mean squared error (RMSE). In terms of SNR, GLISNet has an increase of 4.41% compared with the second best algorithm PhysNet on PURE dataset. The MAE has a decrease of 13.16% compared with the second best algorithm DeeprPPG on UBFC-rPPG dataset. The RMSE has a decrease of 26.29% compared with the second best algorithm PhysNet on UBFC-rPPG dataset. Experiments on MIHR dataset demonstrates the robustness of GLISNet under low-light environment.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001203362500041</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>37028087</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Remote photoplethysmography; heart rate estimation; pulse representation; CNN; long-range dependency</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Zhao, Changchen; Zhao, Zheng; Huang, Bin] Beihang Univ, Hangzhou Innovat Inst, Hangzhou 310051, Peoples R China. <br>
[Zhou, Menghao] Zhejiang Univ Technol, Coll Informat Engn, Hangzhou 310023, Peoples R China. <br>
[Rao, Bing] Hangzhou City Univ, Hangzhou 310015, Peoples R China. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Rao, B (corresponding author), Hangzhou City Univ, Hangzhou 310015, Peoples R China.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
cczhao@zjut.edu.cn; tracker.z@qq.com; zhaozhengbuaa@qq.com; marshuangbin@buaa.edu.cn; raob@zucc.edu.cn</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Beihang University; Zhejiang University of Technology; Hangzhou City University</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Huang, Bin</display_name>&nbsp;</td><td>ISA-9354-2023&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Zhao, Changchen</display_name>&nbsp;</td><td>AEV-3957-2022&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Huang, Bin</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-0894-5335&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Zhao, Changchen</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-0546-6016&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Rao, Bing</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-3760-8979&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Information Systems; Computer Science, Interdisciplinary Applications; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>NW0A0</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2168-2194</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>2168-2208</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE J BIOMED HEALTH</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE J. Biomed. Health Inform.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>12</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Natural Science Foundation of China</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>No Statement Available</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 35 of 37</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Reconstruction and completion of high-resolution 3D cardiac shapes using anisotropic CMRI segmentations and continuous implicit neural representations</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Sander, J (Sander, Jorg); de Vos, BD (de Vos, Bob D.); Bruns, S (Bruns, Steffen); Planken, N (Planken, Nils); Viergever, MA (Viergever, Max A.); Leiner, T (Leiner, Tim); Isgum, I (Isgum, Ivana)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>COMPUTERS IN BIOLOGY AND MEDICINE</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>164</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>107266</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1016/j.compbiomed.2023.107266</value>&nbsp;&nbsp;<b>Early Access Date:</b> 
<value>JUL 2023</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2023 SEP</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>7</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>8</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
2</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
16</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>47</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Since the onset of computer-aided diagnosis in medical imaging, voxel-based segmentation has emerged as the primary methodology for automatic analysis of left ventricle (LV) function and morphology in cardiac magnetic resonance images (CMRI). In standard clinical practice, simultaneous multi-slice 2D cine short-axis MR imaging is performed under multiple breath-holds resulting in highly anisotropic 3D images. Furthermore, sparse-view CMRI often lacks whole heart coverage caused by large slice thickness and often suffers from inter -slice misalignment induced by respiratory motion. Therefore, these volumes only provide limited information about the true 3D cardiac anatomy which may hamper highly accurate assessment of functional and anatomical abnormalities. To address this, we propose a method that learns a continuous implicit function representing 3D LV shapes by training an auto-decoder. For training, high-resolution segmentations from cardiac CT angiography are used. The ability of our approach to reconstruct and complete high-resolution shapes from manually or automatically obtained sparse-view cardiac shape information is evaluated by using paired high-and low-resolution CMRI LV segmentations. The results show that the reconstructed LV shapes have an unconstrained subvoxel resolution and appear smooth and plausible in through-plane direction. Furthermore, Bland-Altman analysis reveals that reconstructed high-resolution ventricle volumes are closer to the corresponding reference volumes than reference low-resolution volumes with bias of [limits of agreement] -3.51 [-18.87, 11.85] mL, and 12.96 [-10.01, 35.92] mL respectively. Finally, the results demonstrate that the proposed approach allows recovering missing shape information and can indirectly correct for limited motion-induced artifacts.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001048414600001</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>37494823</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Cardiac MRI; Neural implicit function; Super-resolution; Shape completion; Shape reconstruction</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
HEART</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Sander, Jorg; de Vos, Bob D.; Bruns, Steffen; Isgum, Ivana] Locat Univ Amsterdam, Dept Biomed Engn &amp; Phys, Amsterdam Univ Med Ctr, Amsterdam, Netherlands. <br>
[Sander, Jorg; Isgum, Ivana] Amsterdam Cardiovasc Sci, Amsterdam, Netherlands. <br>
[Planken, Nils; Isgum, Ivana] Locat Univ Amsterdam, Dept Radiol &amp; Nucl Med, Amsterdam Univ Med Ctr, Amsterdam, Netherlands. <br>
[Sander, Jorg; Isgum, Ivana] Univ Amsterdam, Informat Inst, Amsterdam, Netherlands. <br>
[Viergever, Max A.] Univ Med Ctr Utrecht, Image Sci Inst, Utrecht, Netherlands. <br>
[Leiner, Tim] Mayo Clin, Dept Radiol, Rochester, MN USA. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Sander, J (corresponding author), Locat Univ Amsterdam, Dept Biomed Engn &amp; Phys, Amsterdam Univ Med Ctr, Amsterdam, Netherlands.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
j.sander1@amsterdamumc.nl</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
University of Amsterdam; Utrecht University; Utrecht University Medical Center; Mayo Clinic</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Planken, R. Nils</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-3940-4670&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Viergever, Max</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-2582-042X&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Isgum, Ivana</display_name>&nbsp;</td><td>H-8659-2014&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Planken, RN</display_name>&nbsp;</td><td>JAC-4538-2023&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>PERGAMON-ELSEVIER SCIENCE LTD</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>THE BOULEVARD, LANGFORD LANE, KIDLINGTON, OXFORD OX5 1GB, ENGLAND</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Biology; Computer Science, Interdisciplinary Applications; Engineering, Biomedical; Mathematical &amp; Computational Biology</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Life Sciences &amp; Biomedicine - Other Topics; Computer Science; Engineering; Mathematical &amp; Computational Biology</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>P1OX0</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0010-4825</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1879-0534</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>COMPUT BIOL MED</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Comput. Biol. Med.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>15</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Dutch Technology Foundation</grant_agency>&nbsp;</td><td>
<div>P15-26&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>Acknowledgments This study was performed within the DLMedIA program (P15-26) funded by Dutch Technology Foundation with participation of Pie Medical Imaging.</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>hybrid</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 36 of 37</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Identifying metabolic adaptations characteristic of cardiotoxicity using paired transcriptomics and metabolomics data integrated with a computational model of heart metabolism</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Dougherty, BV (Dougherty, Bonnie V.); Moore, CJ (Moore, Connor J.); Rawls, KD (Rawls, Kristopher D.); Jenior, ML (Jenior, Matthew L.); Chun, B (Chun, Bryan); Nagdas, S (Nagdas, Sarbajeet); Saucerman, JJ (Saucerman, Jeffrey J.); Kolling, GL (Kolling, Glynis L.); Wallqvist, A (Wallqvist, Anders); Papin, JA (Papin, Jason A.)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>PLOS COMPUTATIONAL BIOLOGY</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>20</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>2</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>e1011919</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1371/journal.pcbi.1011919</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2024 FEB</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>1</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>1</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
0</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
2</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>54</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Improvements in the diagnosis and treatment of cancer have revealed long-term side effects of chemotherapeutics, particularly cardiotoxicity. Here, we present paired transcriptomics and metabolomics data characterizing in vitro cardiotoxicity to three compounds: 5-fluorouracil, acetaminophen, and doxorubicin. Standard gene enrichment and metabolomics approaches identify some commonly affected pathways and metabolites but are not able to readily identify metabolic adaptations in response to cardiotoxicity. The paired data was integrated with a genome-scale metabolic network reconstruction of the heart to identify shifted metabolic functions, unique metabolic reactions, and changes in flux in metabolic reactions in response to these compounds. Using this approach, we confirm previously seen changes in the p53 pathway by doxorubicin and RNA synthesis by 5-fluorouracil, we find evidence for an increase in phospholipid metabolism in response to acetaminophen, and we see a shift in central carbon metabolism suggesting an increase in metabolic demand after treatment with doxorubicin and 5-fluorouracil.
<br>Improvements in the diagnosis and treatment of cancer have revealed long-term side effects of chemotherapeutics, particularly cardiotoxicity. Here, we compare the cardiotoxic effects of 3 drugs, 5-fluorouracil, acetaminophen, and doxorubicin, using transcriptomic and metabolomic data. By integrating these data into a genome-scale metabolic network reconstruction of the heart, we were able to identify metabolic adaptations in response to cardiotoxicity that avoided detection in traditional gene enrichment and metabolomic techniques. Using this approach, we confirm known mechanisms of doxorubicin-induced cardiotoxicity and provide hypotheses and potential mechanisms for metabolic adaptations in cardiotoxicity for 5-fluorouracil, doxorubicin, and acetaminophen. We hope future work using our novel paired transcriptomic and metabolic characterization of in vitro cardiotoxicity can help improve the current network heart model and further characterize the role of chemotherapeutics in cardiotoxicity.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001181567100004</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>38422168</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
NITRIC-OXIDE; 5-FLUOROURACIL; EXPRESSION; PATHWAY</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Dougherty, Bonnie V.; Moore, Connor J.; Rawls, Kristopher D.; Jenior, Matthew L.; Chun, Bryan; Saucerman, Jeffrey J.; Kolling, Glynis L.; Papin, Jason A.] Univ Virginia, Dept Biomed Engn, Charlottesville, VA 22904 USA. <br>
[Nagdas, Sarbajeet] Univ Virginia Hlth Syst, Dept Microbiol Immunol &amp; Canc Biol, Charlottesville, VA USA. <br>
[Kolling, Glynis L.; Papin, Jason A.] Univ Virginia, Dept Med, Div Infect Dis &amp; Int Hlth, Charlottesville, VA 22908 USA. <br>
[Wallqvist, Anders] US Army Med Res &amp; Dev Command, Dept Def Biotechnol, High Performance Comp Software Applicat Inst, Telemed &amp; Adv Technol Res Ctr, Ft Detrick, MD USA. <br>
[Papin, Jason A.] Univ Virginia, Dept Biochem &amp; Mol Genet, Charlottesville, VA 22904 USA. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Papin, JA (corresponding author), Univ Virginia, Dept Biomed Engn, Charlottesville, VA 22904 USA.<br>Papin, JA (corresponding author), Univ Virginia, Dept Med, Div Infect Dis &amp; Int Hlth, Charlottesville, VA 22908 USA.<br>Papin, JA (corresponding author), Univ Virginia, Dept Biochem &amp; Mol Genet, Charlottesville, VA 22904 USA.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
papin@virginia.edu</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
University of Virginia; University of Virginia; University of Virginia (UVA) Health System; University of Virginia; United States Department of Defense; United States Army; United States Army Medical Research &amp; Development Command (USAMRDC); University of Virginia</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Papin, Jason</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-2769-5805&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Wallqvist, Anders</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-9775-7469&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Moore, Connor</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-7788-8958&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Rawls, Kristopher</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-5873-0127&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>PUBLIC LIBRARY SCIENCE</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>1160 BATTERY STREET, STE 100, SAN FRANCISCO, CA 94111 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Biochemical Research Methods; Mathematical &amp; Computational Biology</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Biochemistry &amp; Molecular Biology; Mathematical &amp; Computational Biology</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>KR0D2</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>1553-734X</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1553-7358</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>PLOS COMPUT BIOL</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>PLoS Comput. Biol.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>21</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>United States Department of Defense</grant_agency>&nbsp;</td><td>
<div>W81XWH-14-C-0054&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Institutes of Health</grant_agency>&nbsp;</td><td>
<div>HL137755&nbsp;</div>
<div>DK132369&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Science Foundation Graduate Research Fellowship Program</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>Support for this project was provided by the United States Department of Defense (W81XWH-14-C-0054 to JAP), the National Institutes of Health (NIH grant HL137755 to JJS, DK132369 to JAP), and the National Science Foundation Graduate Research Fellowship Program (awarded to BVD). The opinions and assertions contained herein are the private views of the authors and are not to be construed as the official or as reflecting the views of the U.S. Army or the U.S. Department of Defense. This manuscript has been approved for public release with unlimited distribution. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>gold, Green Submitted</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 37 of 37</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Automatic sleep-stage classification of heart rate and actigraphy data using deep and transfer learning approaches</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Ma, YJX (Ma, Yaopeng J. X.); Zschocke, J (Zschocke, Johannes); Glos, M (Glos, Martin); Kluge, M (Kluge, Maria); Penzel, T (Penzel, Thomas); Kantelhardt, JW (Kantelhardt, Jan W.); Bartsch, RP (Bartsch, Ronny P.)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>COMPUTERS IN BIOLOGY AND MEDICINE</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>163</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>107193</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1016/j.compbiomed.2023.107193</value>&nbsp;&nbsp;<b>Early Access Date:</b> 
<value>JUL 2023</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2023 SEP</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>13</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>13</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
3</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
34</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>64</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Manual sleep-stage scoring based on full-night polysomnography data recorded in a sleep lab has been the gold standard of clinical sleep medicine. This costly and time-consuming approach is unfit for long-term studies as well as assessment of sleep on a population level. With the vast amount of physiological data becoming available from wrist-worn devices, deep learning techniques provide an opportunity for fast and reliable automatic sleep-stage classification tasks. However, training a deep neural network requires large annotated sleep databases, which are not available for long-term epidemiological studies. In this paper, we introduce an end-to-end temporal convolutional neural network able to automatically score sleep stages from raw heartbeat RR interval (RRI) and wrist actigraphy data. Moreover, a transfer learning approach enables the training of the network on a large public database (Sleep Heart Health Study, SHHS) and its subsequent application to a much smaller database recorded by a wristband device. The transfer learning significantly shortens training time and improves sleep-scoring accuracy from 68.9% to 73.8% and inter-rater reliability (Cohen's kappa) from 0.51 to 0.59. We also found that for the SHHS database, automatic sleep-scoring accuracy using deep learning shows a logarithmic relationship with the training size. Although deep learning approaches for automatic sleep scoring are not yet comparable to the inter-rater reliability among sleep technicians, performance is expected to significantly improve in the near future when more large public databases become available. We anticipate those deep learning techniques, when combined with our transfer learning approach, will leverage automatic sleep scoring of physiological data from wearable devices and enable the investigation of sleep in large cohort studies.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001039211500001</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>37421734</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Sleep-stage scoring; Heart-rate variability; Wrist actigraphy; Deep learning; Transfer learning; Convolutional neural network; Epidemiological studies</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
RECHTSCHAFFEN; VARIABILITY; DYNAMICS; DESIGN; KALES; AASM</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Ma, Yaopeng J. X.; Bartsch, Ronny P.] Bar Ilan Univ, Dept Phys, Ramat Gan, Israel. <br>
[Zschocke, Johannes] Martin Luther Univ Halle Wittenberg, Interdisciplinary Ctr Hlth Sci, Inst Med Epidemiol Biometr &amp; Informat IMEBI, Halle, Germany. <br>
[Zschocke, Johannes; Kantelhardt, Jan W.] Martin Luther Univ Halle Wittenberg, Inst Phys, Halle, Germany. <br>
[Glos, Martin; Kluge, Maria; Penzel, Thomas] Charite Univ Med Berlin, Interdisciplinary Sleep Med Ctr, Berlin, Germany. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Bartsch, RP (corresponding author), Bar Ilan Univ, Dept Phys, Ramat Gan, Israel.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
yaopeng.ma97@gmail.com; bartsch.ronny@gmail.com</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Bar Ilan University; Martin Luther University Halle Wittenberg; Martin Luther University Halle Wittenberg; Free University of Berlin; Humboldt University of Berlin; Charite Universitatsmedizin Berlin</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Bartsch, Ronny</display_name>&nbsp;</td><td>E-7117-2012&nbsp;</td><td>0000-0001-8336-5141&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Penzel, Thomas</display_name>&nbsp;</td><td>M-2344-2014&nbsp;</td><td>0000-0002-4304-0112&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Kluge, Maria</display_name>&nbsp;</td><td>&nbsp;</td><td>0009-0008-6714-4478&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Ma, Yaopeng J. X.</display_name>&nbsp;</td><td>KBQ-9248-2024&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Ma, Yaopeng J. X.</display_name>&nbsp;</td><td>&nbsp;</td><td>0009-0004-6484-0129&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Kantelhardt, Jan W.</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-9293-0293&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Kantelhardt, Jan</display_name>&nbsp;</td><td>AAF-7955-2019&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>PERGAMON-ELSEVIER SCIENCE LTD</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>THE BOULEVARD, LANGFORD LANE, KIDLINGTON, OXFORD OX5 1GB, ENGLAND</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Biology; Computer Science, Interdisciplinary Applications; Engineering, Biomedical; Mathematical &amp; Computational Biology</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Life Sciences &amp; Biomedicine - Other Topics; Computer Science; Engineering; Mathematical &amp; Computational Biology</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>N8BW3</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0010-4825</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1879-0534</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>COMPUT BIOL MED</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Comput. Biol. Med.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>11</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>German Israeli Foundation (GIF)</grant_agency>&nbsp;</td><td>
<div>I-1372-303.7/2016&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Heart, Lung, and Blood Institute</grant_agency>&nbsp;</td><td>
<div>U01HL53916&nbsp;</div>
<div>U01HL53931&nbsp;</div>
<div>U01HL53934&nbsp;</div>
<div>U01HL53937&nbsp;</div>
<div>U01HL64360&nbsp;</div>
<div>U01HL53938&nbsp;</div>
<div>U01HL53940&nbsp;</div>
<div>U01HL53941&nbsp;</div>
<div>U01HL63463&nbsp;</div>
<div>R24 HL114473&nbsp;</div>
<div>75N92019R002&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>&amp; nbsp;This work was supported by the German Israeli Foundation (GIF) Grant no I-1372-303.7/2016. The Sleep Heart Health Study (SHHS) was supported by National Heart, Lung, and Blood Institute cooperative agreements U01HL53916 (University of California, Davis) , U01HL53931 (New York University) , U01HL53934 (University of Minnesota) , U01HL53937 and U01HL64360 (Johns Hopkins University) , U01HL53938 (University of Arizona) , U01HL53940 (University of Washington) , U01HL53941 (Boston University) , and U01HL63463 (Case Western Reserve University) . The National Sleep Research Resource was supported by the National Heart, Lung, and Blood Institute (R24 HL114473, 75N92019R002) .</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr>
<table xmlns:bean="http://ts.thomson.com/ua/bean">
<tr>
<td>End of File</td>
</tr>
</table><div style="background-color: #000000; padding: 10px; color: #FFFFFF; font-family: arial, sans-serif;">
<div style="height: 25px; width: 100px;">
<mat-icon data-mat-icon-name="clarivate" data-mat-icon-type="svg" aria-hidden="true" class="mat-icon notranslate clarivate-logo mat-icon-no-color" svgicon="clarivate" role="img">
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" focusable="false" preserveAspectRatio="xMidYMid meet" version="1.1" viewBox="0 0 92 18" height="100%" width="100%">
<title>Clarivate</title>
<g fill-rule="evenodd" fill="none" stroke-width="1" stroke="none">
<g fill-rule="nonzero" fill="#FFFFFF" transform="translate(-19.000000, -8.000000)">
<g transform="translate(1.000000, 1.000000)">
<g transform="translate(18.000000, 7.000000)">
<path d="M26.6812194,13.2997252 C28.2949641,13.3845068 29.7591419,12.3596036 30.2318671,10.8143235 L33.0910469,10.8143235 C32.3628779,13.742285 29.6973379,15.7705498 26.6812194,15.6916944 C24.8651813,15.7485215 23.1057028,15.0559504 21.8156554,13.7764967 C20.5256081,12.4970431 19.8185526,10.7433351 19.8604035,8.9268901 C19.8185367,7.11043445 20.5255848,5.35671042 21.8156337,4.07724261 C23.1056826,2.7977748 24.8651702,2.10519376 26.6812194,2.16201887 C29.6973451,2.0831702 32.3628861,4.11144761 33.0910469,7.03941847 L30.2318671,7.03941847 C29.7591419,5.49413828 28.2949641,4.46923515 26.6812194,4.55401674 C25.5407067,4.54133389 24.4470406,5.00703241 23.6657805,5.83803342 C22.8845203,6.66903442 22.487126,7.78932829 22.5700912,8.9268901 C22.4871313,10.064447 22.8845287,11.184734 23.6657888,12.0157277 C24.447049,12.8467215 25.540712,13.312413 26.6812194,13.2997252 L26.6812194,13.2997252 Z"/>
<polygon points="36.8477342 15.4674945 34.4371264 15.4674945 34.4371264 2.38628571 36.8477342 2.38628571"/>
<path d="M43.5380316,9.76781183 C43.5380316,9.15114738 43.0522397,8.75870193 42.1178663,8.75870193 C41.2070757,8.82048057 40.3236107,9.0957548 39.5389268,9.5622602 L38.5858286,7.73092465 C39.6930919,7.06514423 40.9567617,6.70409618 42.2486241,6.68441302 C44.1921648,6.68441302 45.9674886,7.58137515 45.9674886,9.84255787 L45.9674886,15.4674945 L43.5380316,15.4674945 L43.5380316,14.6639362 C42.9494098,15.3673061 42.0613957,15.7488608 41.146005,15.6917231 C40.3900371,15.7421039 39.645831,15.484584 39.0826798,14.9777443 C38.5195286,14.4709046 38.1853154,13.7578403 38.1560675,13.0007601 C38.1560675,11.3749188 39.3519468,10.3658089 41.2581433,10.3658089 L42.9400537,10.3658089 C43.3511665,10.3658089 43.5379934,10.1789247 43.5379934,9.86128266 L43.5380316,9.76781183 Z M43.5380316,12.4027631 L43.5380316,11.786089 C43.321275,11.9402969 43.0543042,12.0070182 42.790485,11.9729159 L41.8748173,11.9729159 C41.1086607,11.9729159 40.6602036,12.3092922 40.6602036,12.9072893 C40.6602036,13.486619 41.1272707,13.9164566 41.8748173,13.9164566 C42.303867,13.9699481 42.7348632,13.8332333 43.0546312,13.5422121 C43.3743992,13.2511908 43.5509896,12.8349402 43.5380316,12.4027631 L43.5380316,12.4027631 Z"/>
<path d="M52.6579104,9.3006299 L51.6301236,9.3006299 C50.3966703,9.3006299 49.8361037,9.8799596 49.8361037,11.3749188 L49.8361037,15.4674945 L47.4066179,15.4674945 L47.4066179,6.90863203 L49.7612715,6.90863203 L49.7612715,8.27284306 C50.1001881,7.34759017 51.0007831,6.74968637 51.9850429,6.79648425 C52.2115562,6.79362484 52.4375557,6.81874947 52.6579104,6.87128771 L52.6579104,9.3006299 Z"/>
<path d="M57.4232866,15.4674945 L54.9939445,15.4674945 L54.9939445,8.96425355 L53.7979312,8.96425355 L53.7979312,6.90863203 L57.4232866,6.90863203 L57.4232866,15.4674945 Z M56.0964104,2.38628347 C56.5129556,2.38558064 56.9126431,2.55074046 57.2071855,2.84528284 C57.5017278,3.13982521 57.6668877,3.53951271 57.6661848,3.95605794 C57.6609778,4.814152 56.9638874,5.50702694 56.1057775,5.50702694 C55.2476677,5.50702694 54.5505773,4.814152 54.5453585,3.95605794 C54.5417726,3.09491157 55.2352828,2.39302469 56.0964104,2.38628347 Z"/>
<polygon points="62.8802456 12.8512776 64.8236715 6.90863203 67.3838767 6.90863203 64.33787 15.4674945 61.3292077 15.4674945 58.4139588 6.90863203 61.0488526 6.90863203"/>
<path d="M72.8783522,9.76781183 C72.8783522,9.15114738 72.3924359,8.75870193 71.4580625,8.75870193 C70.5472784,8.82049943 69.6638208,9.09577238 68.8791325,9.5622602 L67.9261491,7.73092465 C69.0333627,7.06513461 70.2969953,6.70408494 71.5888202,6.68441302 C73.5323609,6.68441302 75.3076848,7.58137515 75.3076848,9.84255787 L75.3076848,15.4674945 L72.8783522,15.4674945 L72.8783522,14.6639362 C72.2896971,15.3672586 71.4017035,15.7487939 70.4863256,15.6916944 C69.7303436,15.7420941 68.986117,15.4845835 68.4229464,14.9777426 C67.8597758,14.4709017 67.5255471,13.7578265 67.4962924,13.0007314 C67.4962924,11.3748901 68.6923056,10.3657802 70.5984925,10.3657802 L72.2802785,10.3657802 C72.6913914,10.3657802 72.8783426,10.1788959 72.8783426,9.86125395 L72.8783522,9.76781183 Z M72.8783522,12.4027631 L72.8783522,11.786089 C72.6615874,11.9402794 72.3946236,12.006999 72.1308056,11.9729159 L71.2151378,11.9729159 C70.4488664,11.9729159 70.0004093,12.3092922 70.0004093,12.9072893 C70.0004093,13.486619 70.4676008,13.9164566 71.2151378,13.9164566 C71.6441833,13.9699279 72.0751679,13.8332061 72.394931,13.5421893 C72.7146941,13.2511724 72.8912904,12.8349341 72.8783522,12.4027631 L72.8783522,12.4027631 Z"/>
<path d="M79.9985637,3.65701635 L79.9985637,6.90863203 L81.8485762,6.90863203 L81.8485762,8.96425355 L79.9985637,8.96425355 L79.9985637,12.8325433 C79.9985637,13.3745286 80.4282865,13.4492747 80.8020646,13.4492747 C81.418729,13.4492747 81.9232553,13.3932056 81.9232553,13.3932056 L81.9232553,15.4674945 C81.9232553,15.4674945 80.7459859,15.5796423 80.0357932,15.5796423 C78.8211795,15.5796423 77.5691354,15.3740811 77.5691354,13.6548263 L77.5691354,8.96425355 L76.22363,8.96425355 L76.22363,6.90863203 L77.5691163,6.90863203 L77.5691163,3.65701635 L79.9985637,3.65701635 Z"/>
<path d="M91.5475359,11.8981794 L85.1751771,11.8981794 C85.3031552,12.7204351 85.9194738,13.3815958 86.7307322,13.5669136 C87.5419905,13.7522314 88.3842592,13.4242593 88.856573,12.7391298 L91.3793191,13.0007601 C90.4933306,14.9536735 88.3725856,16.0321989 86.2725384,15.5978628 C84.1724912,15.1635268 82.6535603,13.3322361 82.6149337,11.188092 C82.567807,9.98960603 83.0193435,8.82515843 83.8621298,7.97175307 C84.7049161,7.1183477 85.8636189,6.65226701 87.0626009,6.68438432 C89.4171397,6.68438432 91.5475359,8.21674525 91.5475359,11.8047373 L91.5475359,11.8981794 Z M85.1189837,10.2723955 L88.9499864,10.2723955 C88.8059978,9.34848922 87.9973156,8.67588427 87.0626392,8.70263282 C86.1147409,8.67034807 85.2869107,9.33893202 85.1189837,10.2723955 Z"/>
<path d="M12.6820274,9.85132226 C10.6885049,11.8556057 8.11575776,13.1830474 5.32707189,13.6462054 C5.77780891,14.8758828 6.40559556,16.0331959 7.19058498,17.0815575 C10.3903106,16.2989026 13.3004311,14.619851 15.5794509,12.2414352 C15.0651988,11.0374085 14.3787129,9.91449973 13.5415112,8.90790698 C13.271174,9.23328 12.9846793,9.54772625 12.6820274,9.85124571"/>
<path d="M5.32722498,13.6460332 C4.21304566,10.5852611 4.22627578,7.22758116 5.3645406,4.17568425 C4.07298567,3.95802358 2.75701252,3.92542943 1.45626578,4.07888372 C0.536506305,7.22972054 0.525090565,10.5762272 1.42333236,13.7332656 C2.72217338,13.8896343 4.03666286,13.8602685 5.32722498,13.6460524"/>
<path d="M7.25919787,0.751096346 C6.46424104,1.79423541 5.8259601,2.94792723 5.36457887,4.17561728 C8.13918123,4.64457901 10.6977043,5.96938232 12.6820178,7.96458738 C12.9850078,8.2673988 13.2715025,8.58184824 13.5415017,8.90793568 C14.3788161,7.90142183 15.0653154,6.77849641 15.5794509,5.57440744 C13.3171586,3.21319806 10.4324457,1.54091896 7.25919787,0.751096346"/>
</g>
</g>
</g>
</g>
</svg>
</mat-icon>
</div>
<path d="M5.32722498,13.6460332 C4.21304566,10.5852611 4.22627578,7.22758116 5.3645406,4.17568425 C4.07298567,3.95802358 2.75701252,3.92542943 1.45626578,4.07888372 C0.536506305,7.22972054 0.525090565,10.5762272 1.42333236,13.7332656 C2.72217338,13.8896343 4.03666286,13.8602685 5.32722498,13.6460524"></path>
<path d="M7.25919787,0.751096346 C6.46424104,1.79423541 5.8259601,2.94792723 5.36457887,4.17561728 C8.13918123,4.64457901 10.6977043,5.96938232 12.6820178,7.96458738 C12.9850078,8.2673988 13.2715025,8.58184824 13.5415017,8.90793568 C14.3788161,7.90142183 15.0653154,6.77849641 15.5794509,5.57440744 C13.3171586,3.21319806 10.4324457,1.54091896 7.25919787,0.751096346"></path>
</div>
