<div xmlns:bean="http://ts.thomson.com/ua/bean" style="background-color: #000000; padding: 10px; color: #FFFFFF; font-family:  arial, sans-serif;">
<div style="height: 25px; width: 100px;">
<mat-icon data-mat-icon-name="clarivate" data-mat-icon-type="svg" aria-hidden="true" class="mat-icon notranslate clarivate-logo mat-icon-no-color" svgicon="clarivate" role="img">
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" focusable="false" preserveAspectRatio="xMidYMid meet" version="1.1" viewBox="0 0 92 18" height="100%" width="100%">
<title>Clarivate</title>
<g fill-rule="evenodd" fill="none" stroke-width="1" stroke="none">
<g fill-rule="nonzero" fill="#FFFFFF" transform="translate(-19.000000, -8.000000)">
<g transform="translate(1.000000, 1.000000)">
<g transform="translate(18.000000, 7.000000)">
<path d="M26.6812194,13.2997252 C28.2949641,13.3845068 29.7591419,12.3596036 30.2318671,10.8143235 L33.0910469,10.8143235 C32.3628779,13.742285 29.6973379,15.7705498 26.6812194,15.6916944 C24.8651813,15.7485215 23.1057028,15.0559504 21.8156554,13.7764967 C20.5256081,12.4970431 19.8185526,10.7433351 19.8604035,8.9268901 C19.8185367,7.11043445 20.5255848,5.35671042 21.8156337,4.07724261 C23.1056826,2.7977748 24.8651702,2.10519376 26.6812194,2.16201887 C29.6973451,2.0831702 32.3628861,4.11144761 33.0910469,7.03941847 L30.2318671,7.03941847 C29.7591419,5.49413828 28.2949641,4.46923515 26.6812194,4.55401674 C25.5407067,4.54133389 24.4470406,5.00703241 23.6657805,5.83803342 C22.8845203,6.66903442 22.487126,7.78932829 22.5700912,8.9268901 C22.4871313,10.064447 22.8845287,11.184734 23.6657888,12.0157277 C24.447049,12.8467215 25.540712,13.312413 26.6812194,13.2997252 L26.6812194,13.2997252 Z"/>
<polygon points="36.8477342 15.4674945 34.4371264 15.4674945 34.4371264 2.38628571 36.8477342 2.38628571"/>
<path d="M43.5380316,9.76781183 C43.5380316,9.15114738 43.0522397,8.75870193 42.1178663,8.75870193 C41.2070757,8.82048057 40.3236107,9.0957548 39.5389268,9.5622602 L38.5858286,7.73092465 C39.6930919,7.06514423 40.9567617,6.70409618 42.2486241,6.68441302 C44.1921648,6.68441302 45.9674886,7.58137515 45.9674886,9.84255787 L45.9674886,15.4674945 L43.5380316,15.4674945 L43.5380316,14.6639362 C42.9494098,15.3673061 42.0613957,15.7488608 41.146005,15.6917231 C40.3900371,15.7421039 39.645831,15.484584 39.0826798,14.9777443 C38.5195286,14.4709046 38.1853154,13.7578403 38.1560675,13.0007601 C38.1560675,11.3749188 39.3519468,10.3658089 41.2581433,10.3658089 L42.9400537,10.3658089 C43.3511665,10.3658089 43.5379934,10.1789247 43.5379934,9.86128266 L43.5380316,9.76781183 Z M43.5380316,12.4027631 L43.5380316,11.786089 C43.321275,11.9402969 43.0543042,12.0070182 42.790485,11.9729159 L41.8748173,11.9729159 C41.1086607,11.9729159 40.6602036,12.3092922 40.6602036,12.9072893 C40.6602036,13.486619 41.1272707,13.9164566 41.8748173,13.9164566 C42.303867,13.9699481 42.7348632,13.8332333 43.0546312,13.5422121 C43.3743992,13.2511908 43.5509896,12.8349402 43.5380316,12.4027631 L43.5380316,12.4027631 Z"/>
<path d="M52.6579104,9.3006299 L51.6301236,9.3006299 C50.3966703,9.3006299 49.8361037,9.8799596 49.8361037,11.3749188 L49.8361037,15.4674945 L47.4066179,15.4674945 L47.4066179,6.90863203 L49.7612715,6.90863203 L49.7612715,8.27284306 C50.1001881,7.34759017 51.0007831,6.74968637 51.9850429,6.79648425 C52.2115562,6.79362484 52.4375557,6.81874947 52.6579104,6.87128771 L52.6579104,9.3006299 Z"/>
<path d="M57.4232866,15.4674945 L54.9939445,15.4674945 L54.9939445,8.96425355 L53.7979312,8.96425355 L53.7979312,6.90863203 L57.4232866,6.90863203 L57.4232866,15.4674945 Z M56.0964104,2.38628347 C56.5129556,2.38558064 56.9126431,2.55074046 57.2071855,2.84528284 C57.5017278,3.13982521 57.6668877,3.53951271 57.6661848,3.95605794 C57.6609778,4.814152 56.9638874,5.50702694 56.1057775,5.50702694 C55.2476677,5.50702694 54.5505773,4.814152 54.5453585,3.95605794 C54.5417726,3.09491157 55.2352828,2.39302469 56.0964104,2.38628347 Z"/>
<polygon points="62.8802456 12.8512776 64.8236715 6.90863203 67.3838767 6.90863203 64.33787 15.4674945 61.3292077 15.4674945 58.4139588 6.90863203 61.0488526 6.90863203"/>
<path d="M72.8783522,9.76781183 C72.8783522,9.15114738 72.3924359,8.75870193 71.4580625,8.75870193 C70.5472784,8.82049943 69.6638208,9.09577238 68.8791325,9.5622602 L67.9261491,7.73092465 C69.0333627,7.06513461 70.2969953,6.70408494 71.5888202,6.68441302 C73.5323609,6.68441302 75.3076848,7.58137515 75.3076848,9.84255787 L75.3076848,15.4674945 L72.8783522,15.4674945 L72.8783522,14.6639362 C72.2896971,15.3672586 71.4017035,15.7487939 70.4863256,15.6916944 C69.7303436,15.7420941 68.986117,15.4845835 68.4229464,14.9777426 C67.8597758,14.4709017 67.5255471,13.7578265 67.4962924,13.0007314 C67.4962924,11.3748901 68.6923056,10.3657802 70.5984925,10.3657802 L72.2802785,10.3657802 C72.6913914,10.3657802 72.8783426,10.1788959 72.8783426,9.86125395 L72.8783522,9.76781183 Z M72.8783522,12.4027631 L72.8783522,11.786089 C72.6615874,11.9402794 72.3946236,12.006999 72.1308056,11.9729159 L71.2151378,11.9729159 C70.4488664,11.9729159 70.0004093,12.3092922 70.0004093,12.9072893 C70.0004093,13.486619 70.4676008,13.9164566 71.2151378,13.9164566 C71.6441833,13.9699279 72.0751679,13.8332061 72.394931,13.5421893 C72.7146941,13.2511724 72.8912904,12.8349341 72.8783522,12.4027631 L72.8783522,12.4027631 Z"/>
<path d="M79.9985637,3.65701635 L79.9985637,6.90863203 L81.8485762,6.90863203 L81.8485762,8.96425355 L79.9985637,8.96425355 L79.9985637,12.8325433 C79.9985637,13.3745286 80.4282865,13.4492747 80.8020646,13.4492747 C81.418729,13.4492747 81.9232553,13.3932056 81.9232553,13.3932056 L81.9232553,15.4674945 C81.9232553,15.4674945 80.7459859,15.5796423 80.0357932,15.5796423 C78.8211795,15.5796423 77.5691354,15.3740811 77.5691354,13.6548263 L77.5691354,8.96425355 L76.22363,8.96425355 L76.22363,6.90863203 L77.5691163,6.90863203 L77.5691163,3.65701635 L79.9985637,3.65701635 Z"/>
<path d="M91.5475359,11.8981794 L85.1751771,11.8981794 C85.3031552,12.7204351 85.9194738,13.3815958 86.7307322,13.5669136 C87.5419905,13.7522314 88.3842592,13.4242593 88.856573,12.7391298 L91.3793191,13.0007601 C90.4933306,14.9536735 88.3725856,16.0321989 86.2725384,15.5978628 C84.1724912,15.1635268 82.6535603,13.3322361 82.6149337,11.188092 C82.567807,9.98960603 83.0193435,8.82515843 83.8621298,7.97175307 C84.7049161,7.1183477 85.8636189,6.65226701 87.0626009,6.68438432 C89.4171397,6.68438432 91.5475359,8.21674525 91.5475359,11.8047373 L91.5475359,11.8981794 Z M85.1189837,10.2723955 L88.9499864,10.2723955 C88.8059978,9.34848922 87.9973156,8.67588427 87.0626392,8.70263282 C86.1147409,8.67034807 85.2869107,9.33893202 85.1189837,10.2723955 Z"/>
<path d="M12.6820274,9.85132226 C10.6885049,11.8556057 8.11575776,13.1830474 5.32707189,13.6462054 C5.77780891,14.8758828 6.40559556,16.0331959 7.19058498,17.0815575 C10.3903106,16.2989026 13.3004311,14.619851 15.5794509,12.2414352 C15.0651988,11.0374085 14.3787129,9.91449973 13.5415112,8.90790698 C13.271174,9.23328 12.9846793,9.54772625 12.6820274,9.85124571"/>
<path d="M5.32722498,13.6460332 C4.21304566,10.5852611 4.22627578,7.22758116 5.3645406,4.17568425 C4.07298567,3.95802358 2.75701252,3.92542943 1.45626578,4.07888372 C0.536506305,7.22972054 0.525090565,10.5762272 1.42333236,13.7332656 C2.72217338,13.8896343 4.03666286,13.8602685 5.32722498,13.6460524"/>
<path d="M7.25919787,0.751096346 C6.46424104,1.79423541 5.8259601,2.94792723 5.36457887,4.17561728 C8.13918123,4.64457901 10.6977043,5.96938232 12.6820178,7.96458738 C12.9850078,8.2673988 13.2715025,8.58184824 13.5415017,8.90793568 C14.3788161,7.90142183 15.0653154,6.77849641 15.5794509,5.57440744 C13.3171586,3.21319806 10.4324457,1.54091896 7.25919787,0.751096346"/>
</g>
</g>
</g>
</g>
</svg>
</mat-icon>
</div>
<path d="M5.32722498,13.6460332 C4.21304566,10.5852611 4.22627578,7.22758116 5.3645406,4.17568425 C4.07298567,3.95802358 2.75701252,3.92542943 1.45626578,4.07888372 C0.536506305,7.22972054 0.525090565,10.5762272 1.42333236,13.7332656 C2.72217338,13.8896343 4.03666286,13.8602685 5.32722498,13.6460524"></path>
<path d="M7.25919787,0.751096346 C6.46424104,1.79423541 5.8259601,2.94792723 5.36457887,4.17561728 C8.13918123,4.64457901 10.6977043,5.96938232 12.6820178,7.96458738 C12.9850078,8.2673988 13.2715025,8.58184824 13.5415017,8.90793568 C14.3788161,7.90142183 15.0653154,6.77849641 15.5794509,5.57440744 C13.3171586,3.21319806 10.4324457,1.54091896 7.25919787,0.751096346"></path>
</div><div style="width: 180px; height: 40px; padding: 10px;">
<mat-icon data-mat-icon-name="wos" data-mat-icon-type="svg" aria-hidden="true" style="width: 180px;" class="mat-icon notranslate mat-icon-no-color" svgicon="wos" role="img">
<svg xmlns="http://www.w3.org/2000/svg" focusable="false" preserveAspectRatio="xMidYMid meet" viewBox="0 0 1868 332" height="100%" width="100%">
<title>Web of Science</title>
<path d="M 108.92,75.00            C 108.92,75.00 147.34,75.00 147.34,75.00              147.34,75.00 182.84,208.12 182.84,208.12              182.84,208.12 183.11,208.12 183.11,208.12              183.11,208.12 212.80,75.00 212.80,75.00              212.80,75.00 250.42,75.00 250.42,75.00              250.42,75.00 205.11,261.00 205.11,261.00              205.11,261.00 162.44,261.00 162.44,261.00              162.44,261.00 127.47,127.34 127.47,127.34              127.47,127.34 93.28,261.00 93.28,261.00              93.28,261.00 50.61,261.00 50.61,261.00              50.61,261.00 4.77,75.00 4.77,75.00              4.77,75.00 44.52,75.00 44.52,75.00              44.52,75.00 73.14,208.12 73.14,208.12              73.14,208.12 73.41,208.12 73.41,208.12              73.41,208.12 108.92,75.00 108.92,75.00 Z            M 379.00,210.00            C 379.00,210.00 288.41,210.00 288.41,210.00              289.29,216.78 292.30,222.48 297.44,227.09              302.57,231.70 308.86,234.00 316.30,234.00              327.80,234.00 335.95,230.00 340.73,222.00              340.73,222.00 376.61,225.58 376.61,225.58              371.30,238.39 363.32,248.00 352.69,254.41              342.06,260.80 329.75,264.00 315.77,264.00              297.52,264.00 282.32,257.98 270.19,245.94              258.06,233.90 252.00,218.59 252.00,200.00              252.00,181.42 257.98,166.11 269.94,154.06              281.89,142.02 296.99,136.00 315.23,136.00              333.47,136.00 348.66,142.21 360.80,154.64              372.93,167.06 379.00,185.51 379.00,210.00 Z            M 333.30,171.28            C 328.34,167.09 322.32,165.00 315.23,165.00              308.15,165.00 302.12,167.01 297.16,171.03              292.20,175.04 289.02,180.37 287.61,187.00              287.61,187.00 342.06,187.00 342.06,187.00              341.19,180.71 338.27,175.47 333.30,171.28 Z            M 433.72,245.31            C 433.72,245.31 433.72,261.00 433.72,261.00              433.72,261.00 399.00,261.00 399.00,261.00              399.00,261.00 399.00,75.00 399.00,75.00              399.00,75.00 433.45,75.00 433.45,75.00              433.45,75.00 433.45,134.94 433.45,134.94              433.45,137.95 433.45,141.85 433.45,146.64              433.45,151.42 433.45,154.17 433.45,154.88              436.43,149.39 441.34,144.87 448.17,141.33              455.02,137.78 462.82,136.00 471.59,136.00              488.26,136.00 502.16,142.07 513.30,154.20              524.43,166.33 530.00,181.59 530.00,200.00              530.00,218.41 524.37,233.68 513.11,245.81              501.86,257.94 487.88,264.00 471.17,264.00              462.91,264.00 455.39,262.22 448.62,258.66              441.85,255.09 436.88,250.65 433.72,245.31 Z            M 485.39,176.23            C 479.65,169.98 472.34,166.86 463.45,166.86              454.58,166.86 447.27,169.98 441.53,176.23              435.79,182.48 432.92,190.37 432.92,199.88              432.92,209.20 435.79,217.07 441.53,223.50              447.27,229.93 454.58,233.14 463.45,233.14              472.34,233.14 479.65,229.93 485.39,223.50              491.13,217.07 494.00,209.20 494.00,199.88              494.00,190.37 491.13,182.48 485.39,176.23 Z            M 707.69,245.81            C 695.19,257.94 679.86,264.00 661.72,264.00              643.58,264.00 628.26,257.94 615.75,245.81              603.25,233.68 597.00,218.41 597.00,200.00              597.00,181.59 603.25,166.33 615.75,154.20              628.26,142.07 643.58,136.00 661.72,136.00              679.86,136.00 695.19,142.07 707.69,154.20              720.20,166.33 726.45,181.59 726.45,200.00              726.45,218.41 720.20,233.68 707.69,245.81 Z            M 661.84,166.86            C 652.85,166.86 645.54,170.00 639.91,176.28              634.27,182.55 631.45,190.46 631.45,200.00              631.45,209.54 634.27,217.45 639.91,223.73              645.54,230.01 652.85,233.14 661.84,233.14              670.66,233.14 677.89,230.01 683.53,223.73              689.18,217.45 692.00,209.54 692.00,200.00              692.00,190.46 689.18,182.55 683.53,176.28              677.89,170.00 670.66,166.86 661.84,166.86 Z            M 757.00,139.00            C 757.00,139.00 757.00,104.34 757.00,104.34              757.00,93.15 760.17,84.98 766.52,79.83              772.86,74.67 781.14,72.09 791.36,72.09              791.36,72.09 816.50,73.69 816.50,73.69              816.50,73.69 816.50,105.33 816.50,105.33              809.47,104.45 804.72,104.00 802.27,104.00              798.58,104.00 795.85,104.71 794.09,106.14              792.33,107.57 791.45,110.24 791.45,114.16              791.45,114.16 791.45,139.00 791.45,139.00              791.45,139.00 815.00,139.00 815.00,139.00              815.00,139.00 815.00,168.00 815.00,168.00              815.00,168.00 791.45,168.00 791.45,168.00              791.45,168.00 791.45,260.50 791.45,260.50              791.45,260.50 757.00,260.50 757.00,260.50              757.00,260.50 757.00,168.00 757.00,168.00              757.00,168.00 738.00,168.00 738.00,168.00              738.00,168.00 738.00,139.00 738.00,139.00              738.00,139.00 757.00,139.00 757.00,139.00 Z            M 1004.30,97.27            C 1004.30,97.27 983.11,124.06 983.11,124.06              969.92,111.36 956.73,105.00 943.53,105.00              937.11,105.00 931.59,106.60 926.95,109.80              922.32,112.98 920.00,117.24 920.00,122.56              920.00,131.06 926.74,138.42 940.22,144.62              940.22,144.62 955.66,151.80 955.66,151.80              971.60,159.25 983.48,167.31 991.28,176.00              999.09,184.68 1003.00,195.67 1003.00,208.97              1003.00,223.87 997.13,236.76 985.39,247.66              973.66,258.55 958.09,264.00 938.67,264.00              913.45,264.00 891.31,254.56 872.25,235.67              872.25,235.67 892.52,208.02 892.52,208.02              899.27,214.90 906.74,220.29 914.92,224.17              923.10,228.06 930.83,230.00 938.11,230.00              945.94,230.00 952.21,228.09 956.92,224.28              961.64,220.47 964.00,215.64 964.00,209.80              964.00,203.95 961.65,198.93 956.95,194.77              952.25,190.60 944.85,186.21 934.73,181.61              934.73,181.61 921.70,175.48 921.70,175.48              894.57,162.73 881.00,145.36 881.00,123.38              881.00,107.59 887.05,94.92 899.17,85.36              911.29,75.79 925.91,71.00 943.06,71.00              963.93,71.00 984.34,79.76 1004.30,97.27 Z            M 1082.92,166.86            C 1073.91,166.86 1066.58,170.00 1060.92,176.28              1055.27,182.55 1052.45,190.46 1052.45,200.00              1052.45,209.54 1055.27,217.45 1060.92,223.73              1066.58,230.01 1073.91,233.14 1082.92,233.14              1095.82,233.14 1104.74,226.95 1109.69,214.58              1109.69,214.58 1142.02,224.66 1142.02,224.66              1138.13,236.72 1130.75,246.29 1119.89,253.38              1109.02,260.46 1096.61,264.00 1082.66,264.00              1064.46,264.00 1049.13,257.94 1036.67,245.81              1024.22,233.68 1018.00,218.41 1018.00,200.00              1018.00,181.59 1024.22,166.33 1036.67,154.20              1049.13,142.07 1064.46,136.00 1082.66,136.00              1096.61,136.00 1109.02,139.59 1119.89,146.77              1130.75,153.95 1138.13,163.56 1142.02,175.61              1142.02,175.61 1109.69,185.42 1109.69,185.42              1107.39,179.77 1103.91,175.26 1099.22,171.91              1094.54,168.54 1089.11,166.86 1082.92,166.86 Z            M 1170.31,112.67            C 1166.10,108.45 1164.00,103.27 1164.00,97.12              1164.00,90.98 1166.15,85.76 1170.45,81.45              1174.75,77.15 1179.89,75.00 1185.86,75.00              1192.02,75.00 1197.24,77.15 1201.55,81.45              1205.85,85.76 1208.00,90.98 1208.00,97.12              1208.00,103.27 1205.85,108.45 1201.55,112.67              1197.24,116.89 1192.02,119.00 1185.86,119.00              1179.71,119.00 1174.53,116.89 1170.31,112.67 Z            M 1205.00,139.00            C 1205.00,139.00 1205.00,261.00 1205.00,261.00              1205.00,261.00 1170.55,261.00 1170.55,261.00              1170.55,261.00 1170.55,168.00 1170.55,168.00              1170.55,168.00 1153.50,168.00 1153.50,168.00              1153.50,168.00 1153.50,139.00 1153.50,139.00              1153.50,139.00 1205.00,139.00 1205.00,139.00 Z            M 1353.00,210.00            C 1353.00,210.00 1262.41,210.00 1262.41,210.00              1263.29,216.78 1266.30,222.48 1271.44,227.09              1276.57,231.70 1282.86,234.00 1290.30,234.00              1301.80,234.00 1309.95,230.00 1314.73,222.00              1314.73,222.00 1350.61,225.58 1350.61,225.58              1345.30,238.39 1337.32,248.00 1326.69,254.41              1316.06,260.80 1303.75,264.00 1289.77,264.00              1271.52,264.00 1256.32,257.98 1244.19,245.94              1232.06,233.90 1226.00,218.59 1226.00,200.00              1226.00,181.42 1231.98,166.11 1243.94,154.06              1255.89,142.02 1270.99,136.00 1289.23,136.00              1307.47,136.00 1322.66,142.21 1334.80,154.64              1346.93,167.06 1353.00,185.51 1353.00,210.00 Z            M 1307.30,171.28            C 1302.34,167.09 1296.32,165.00 1289.23,165.00              1282.15,165.00 1276.12,167.01 1271.16,171.03              1266.20,175.04 1263.02,180.37 1261.61,187.00              1261.61,187.00 1316.06,187.00 1316.06,187.00              1315.19,180.71 1312.27,175.47 1307.30,171.28 Z            M 1406.39,153.36            C 1415.92,141.79 1428.27,136.00 1443.45,136.00              1456.87,136.00 1467.72,140.04 1476.02,148.11              1484.30,156.17 1488.45,167.20 1488.45,181.20              1488.45,181.20 1488.45,261.00 1488.45,261.00              1488.45,261.00 1454.00,261.00 1454.00,261.00              1454.00,261.00 1454.00,189.86 1454.00,189.86              1454.00,182.57 1452.00,177.11 1448.00,173.47              1444.01,169.82 1439.09,168.00 1433.25,168.00              1423.50,168.00 1414.90,172.62 1407.45,181.86              1407.45,181.86 1407.45,261.00 1407.45,261.00              1407.45,261.00 1373.00,261.00 1373.00,261.00              1373.00,261.00 1373.00,139.00 1373.00,139.00              1373.00,139.00 1406.39,139.00 1406.39,139.00              1406.39,139.00 1406.39,153.36 1406.39,153.36 Z            M 1573.92,166.86            C 1564.91,166.86 1557.58,170.00 1551.92,176.28              1546.27,182.55 1543.45,190.46 1543.45,200.00              1543.45,209.54 1546.27,217.45 1551.92,223.73              1557.58,230.01 1564.91,233.14 1573.92,233.14              1586.82,233.14 1595.74,226.95 1600.69,214.58              1600.69,214.58 1633.02,224.66 1633.02,224.66              1629.13,236.72 1621.75,246.29 1610.89,253.38              1600.02,260.46 1587.61,264.00 1573.66,264.00              1555.46,264.00 1540.13,257.94 1527.67,245.81              1515.22,233.68 1509.00,218.41 1509.00,200.00              1509.00,181.59 1515.22,166.33 1527.67,154.20              1540.13,142.07 1555.46,136.00 1573.66,136.00              1587.61,136.00 1600.02,139.59 1610.89,146.77              1621.75,153.95 1629.13,163.56 1633.02,175.61              1633.02,175.61 1600.69,185.42 1600.69,185.42              1598.39,179.77 1594.91,175.26 1590.22,171.91              1585.54,168.54 1580.11,166.86 1573.92,166.86 Z            M 1773.00,210.00            C 1773.00,210.00 1682.41,210.00 1682.41,210.00              1683.29,216.78 1686.30,222.48 1691.44,227.09              1696.57,231.70 1702.86,234.00 1710.30,234.00              1721.80,234.00 1729.95,230.00 1734.73,222.00              1734.73,222.00 1770.61,225.58 1770.61,225.58              1765.30,238.39 1757.32,248.00 1746.69,254.41              1736.06,260.80 1723.75,264.00 1709.77,264.00              1691.52,264.00 1676.32,257.98 1664.19,245.94              1652.06,233.90 1646.00,218.59 1646.00,200.00              1646.00,181.42 1651.98,166.11 1663.94,154.06              1675.89,142.02 1690.99,136.00 1709.23,136.00              1727.47,136.00 1742.66,142.21 1754.80,154.64              1766.93,167.06 1773.00,185.51 1773.00,210.00 Z            M 1727.30,171.28            C 1722.34,167.09 1716.32,165.00 1709.23,165.00              1702.15,165.00 1696.12,167.01 1691.16,171.03              1686.20,175.04 1683.02,180.37 1681.61,187.00              1681.61,187.00 1736.06,187.00 1736.06,187.00              1735.19,180.71 1732.27,175.47 1727.30,171.28 Z" stroke-width="1" stroke="black" fill="black"/>
<path d="M 1769.00,76.00            C 1769.00,76.00 1800.00,76.00 1800.00,76.00              1800.00,76.00 1800.00,82.42 1800.00,82.42              1800.00,82.42 1788.00,82.42 1788.00,82.42              1788.00,82.42 1788.00,115.00 1788.00,115.00              1788.00,115.00 1780.84,115.00 1780.84,115.00              1780.84,115.00 1780.84,82.42 1780.84,82.42              1780.84,82.42 1769.00,82.42 1769.00,82.42              1769.00,82.42 1769.00,76.00 1769.00,76.00 Z            M 1827.27,106.64            C 1827.27,106.64 1818.95,106.64 1818.95,106.64              1818.95,106.64 1811.16,87.42 1811.16,87.42              1811.16,87.42 1811.16,115.00 1811.16,115.00              1811.16,115.00 1804.00,115.00 1804.00,115.00              1804.00,115.00 1804.00,76.00 1804.00,76.00              1804.00,76.00 1814.61,76.00 1814.61,76.00              1814.61,76.00 1823.25,98.11 1823.25,98.11              1823.25,98.11 1832.08,76.00 1832.08,76.00              1832.08,76.00 1842.00,76.00 1842.00,76.00              1842.00,76.00 1842.00,115.00 1842.00,115.00              1842.00,115.00 1834.84,115.00 1834.84,115.00              1834.84,115.00 1834.84,87.42 1834.84,87.42              1834.84,87.42 1827.27,106.64 1827.27,106.64 Z" stroke-width="1" stroke="black" fill="black"/>
</svg>
</mat-icon>
</div><table cellspacing="0" cellpadding="2" border="0">
<tr>
<td>38 record(s) printed from Clarivate Web of Science</td>
</tr>
</table><hr>
<table>
<tr>
<td><b>Record 1 of 38</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>xECGNet: Fine-tuning attention map within convolutional neural network to improve detection and explainability of concurrent cardiac arrhythmias</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Yoo, J (Yoo, Jungsun); Jun, TJ (Jun, Tae Joon); Kim, YH (Kim, Young-Hak)</td>
</tr>

<tr xmlns:bean="http://ts.thomson.com/ua/bean" xmlns:exsl="http://exslt.org/common" xmlns:set="http://exslt.org/sets">
<td><b>Source:</b> 
<value>COMPUTER METHODS AND PROGRAMS IN BIOMEDICINE</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>208</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>106281</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1016/j.cmpb.2021.106281</value>&nbsp;&nbsp;<b>Early Access Date:</b> 
<value>JUL 2021</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2021 SEP</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>23</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>26</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
1</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
27</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>44</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Background and objective Detecting abnormal patterns within an electrocardiogram (ECG) is crucial for diagnosing cardiovascular diseases. We start from two unresolved problems in applying deep-learning based ECG classification models to clinical practice: first, although multiple cardiac arrhythmia (CA) types may co-occur in real life, the majority of previous detection methods have focused on one-to-one relationships between ECG and CA type, and second, it has been difficult to explain how neural-network based CA classifiers make decisions. We hypothesize that fine-tuning attention maps with regard to all possible combinations of ground-truth (GT) labels will improve both the detection and interpretability of co-occurring CAs. Methods To test our hypothesis, we propose an end-to-end convolutional neural network (CNN), xECGNet, that fine-tunes the attention map to resemble the averaged response maps of GT labels. Fine-tuning is achieved by adding to the objective function a regularization loss between the attention map and the reference (averaged) map. Performance is assessed by F1 score and subset accuracy. Results The main experiment demonstrates that fine-tuning alone significantly improves a model's multilabel subset accuracy from 75.8% to 84.5% when compared with the baseline model. Also, xECGNet shows the highest F1 score of 0.812 and yields a more explainable map that encompasses multiple CA types, when compared to other baseline methods. Conclusions xECGNet has implications in that it tackles the two obstacles for the clinical application of CNN-based CA detection models with a simple solution of adding one additional term to the objective function. (c) 2021 Elsevier B.V. All rights reserved.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000685504200012</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>34333207</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Cardiac arrhythmia; ECG; Medical AI; Multilabel classification; Explainable AI; Convolutional neural network</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
DATA AUGMENTATION; CLASSIFICATION</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Yoo, Jungsun] Asan Med Ctr, Div Cardiol, Seoul, South Korea. <br>
[Jun, Tae Joon] Asan Inst Life Sci, Asan Med Ctr, Hlth Innovat Big Data Ctr, Seoul, South Korea. <br>
[Kim, Young-Hak] Univ Ulsan, Asan Med Ctr, Div Cardiol, Coll Med, Seoul, South Korea. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Jun, TJ (corresponding author), Asan Inst Life Sci, Asan Med Ctr, Hlth Innovat Big Data Ctr, Seoul, South Korea.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
taejoon@amc.seoul.kr; mdyhkimk@amc.seoul.kr</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
University of Ulsan; Asan Medical Center; University of Ulsan; Asan Medical Center; University of Ulsan; Asan Medical Center</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Yoo, Jungsun</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-8341-9226&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Jun, Tae Joon</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-6808-5149&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Kim, Young-Hoon</display_name>&nbsp;</td><td>B-5332-2011&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>ELSEVIER IRELAND LTD</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>ELSEVIER HOUSE, BROOKVALE PLAZA, EAST PARK SHANNON, CO, CLARE, 00000, IRELAND</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Interdisciplinary Applications; Computer Science, Theory &amp; Methods; Engineering, Biomedical; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Engineering; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>UA9WK</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0169-2607</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1872-7565</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>COMPUT METH PROG BIO</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Comput. Meth. Programs Biomed.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>12</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Korea Medical Device Development Fund grant - Korean government (the Ministry of Science and ICT)</grant_agency>&nbsp;</td><td>
<div>202012B06&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Korea Medical Device Development Fund grant - Korean government (Ministry of Trade, Industry and Energy)</grant_agency>&nbsp;</td><td>
<div>202012B06&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Korea Medical Device Development Fund grant - Korean government (Ministry of Health &amp; Welfare, Republic of Korea)</grant_agency>&nbsp;</td><td>
<div>202012B06&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Korea Medical Device Development Fund grant - Korean government (Ministry of Food and Drug Safety)</grant_agency>&nbsp;</td><td>
<div>202012B06&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Institute for Information &amp; Communications Technology Promotion (IITP), Republic of Korea - Korea government(MSIT)</grant_agency>&nbsp;</td><td>
<div>2017-0-00053&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported by the Korea Medical Device Development Fund grant funded by the Korean government (the Ministry of Science and ICT, the Ministry of Trade, Industry and Energy, the Ministry of Health &amp; Welfare, Republic of Korea, the Ministry of Food and Drug Safety) (Project Number: 202012B06) and by the Institute for Information &amp; Communications Technology Promotion (IITP), Republic of Korea grant funded by the Korea government(MSIT) (2017-0-00053, A Technology Development of Artificial Intelligence Doctors for Cardiovascular Disease).</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 2 of 38</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Semi-supervised Strong-Teacher Consistency Learning for few-shot cardiac MRI image segmentation</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Qiu, YT (Qiu, Yuting); Meng, JM (Meng, James); Li, BH (Li, Baihua)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>COMPUTER METHODS AND PROGRAMS IN BIOMEDICINE</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>261</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>108613</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1016/j.cmpb.2025.108613</value>&nbsp;&nbsp;<b>Early Access Date:</b> 
<value>FEB 2025</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2025 APR</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>3</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>3</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
9</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
11</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>47</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Background and Objective: Cardiovascular disease is a leading cause of mortality worldwide. Automated analysis of heart structures in MRI is crucial for effective diagnostics. While supervised learning has advanced the field of medical image segmentation, it however requires extensive labelled data, which is often limited for cardiac MRI. Methods: Drawing on the principle of consistency learning, we introduce a novel semi-supervised Strong- Teacher Consistency Network for few-shot multi-class cardiac MRI image segmentation, leveraging largely available unlabelled data. This model incorporates a student-teacher architecture. A multi-teacher structure is introduced to learn diverse perspectives and avoid local optimals when dealing with largely varying cardiac structures and anatomical features. It employs a hybrid loss that emphasizes consistency between student and teacher representations, alongside supervised losses (e.g., Dice and Cross-entropy), tailored to the challenge of unlabelled data. Additionally, we introduced feature-space virtual adversarial training to enhance robust feature learning and model stability. Results: Evaluation and ablation studies on the MM-WHS and ACDC benchmark datasets show that the proposed model outperforms nine state-of-the-art semi-supervised methods, particularly with limited annotated data. It achieves 90.14% accuracy on MM-WHS and 78.45% accuracy on ACDC at labelling rates of 25% and 1%, respectively. It also highlights its unique advantages over fully-supervised and single-teacher approaches.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001417945700001</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>39893807</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Cardiac image; Semi-supervised learning; Mean teacher; Medical image segmentation</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Qiu, Yuting; Li, Baihua] Loughborough Univ, Dept Comp Sci, Loughborough LE11 3TU, Leics, England. <br>
[Meng, James] Univ East Anglia, Norwich Med Sch, Norwich NR4 7TJ, Norfolk, England. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Li, BH (corresponding author), Loughborough Univ, Dept Comp Sci, Loughborough LE11 3TU, Leics, England.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
Y.Qiu@lboro.ac.uk; james.meng@doctors.org.uk; B.Li@lboro.ac.uk</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Loughborough University; University of East Anglia</td>
</tr>

<tr>
<td>
<b>Publisher:</b>

<value>ELSEVIER IRELAND LTD</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>ELSEVIER HOUSE, BROOKVALE PLAZA, EAST PARK SHANNON, CO, CLARE, 00000, IRELAND</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Interdisciplinary Applications; Computer Science, Theory &amp; Methods; Engineering, Biomedical; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Engineering; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>W3X4Q</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0169-2607</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1872-7565</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>COMPUT METH PROG BIO</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Comput. Meth. Programs Biomed.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>12</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 3 of 38</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Few-shot transfer learning for personalized atrial fibrillation detection using patient-based siamese network with single-lead ECG records</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Ng, YW (Ng, Yiuwai); Liao, MT (Liao, Min-Tsun); Chen, TL (Chen, Ting-Li); Lee, CK (Lee, Chih-Kuo); Chou, CY (Chou, Cheng-Ying); Wang, WC (Wang, Weichung)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>ARTIFICIAL INTELLIGENCE IN MEDICINE</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>144</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>102644</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1016/j.artmed.2023.102644</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2023 OCT</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>16</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>17</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
7</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
47</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>55</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
The proliferation of wearable devices has allowed the collection of electrocardiogram (ECG) recordings daily to monitor heart rhythm and rate. For example, 24-hour Holter monitors, cardiac patches, and smartwatches are widely used for ECG gathering and application. An automatic atrial fibrillation (AF) detector is required for timely ECG interpretation. Deep learning models can accurately identify AFs if large amounts of annotated data are available for model training. However, it is impractical to request sufficient labels for ECG recordings for an individual patient to train a personalized model. We propose a Siamese-network-based approach for transfer learning to address this issue. A pre-trained Siamese convolutional neural network is created by comparing two labeled ECG segments from the same patient. We sampled 30-second ECG segments with a 50% overlapping window from the ECG recordings of patients in the MIT-BIH Atrial Fibrillation Database. Subsequently, we independently detected the occurrence of AF in each patient in the Long-Term AF Database. By fine-tuning the model with the 1, 3, 5, 7, 9, or 11 ECG segments ranging from 30 to 180 s, our method achieved macro-F1 scores of 96.84%, 96.91%, 96.97%, 97.02%, 97.05%, and 97.07%, respectively.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001123888800001</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>37783539</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Atrial fibrillation; Arrhythmia detection; Electrocardiogram; Deep learning</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
TIME-SERIES; HEART-RATE; PROJECTIONS</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Ng, Yiuwai; Wang, Weichung] Natl Taiwan Univ, Inst Appl Math Sci, Taipei, Taiwan. <br>
[Liao, Min-Tsun; Lee, Chih-Kuo] Natl Taiwan Univ, Dept Internal Med, Div Cardiol, Hosp Hsin Chu Branch, Hsinchu, Taiwan. <br>
[Liao, Min-Tsun] Natl Taiwan Univ, Coll Med, Dept Internal Med, Taipei, Taiwan. <br>
[Chen, Ting-Li] Acad Sinica, Inst Stat Sci, Taipei, Taiwan. <br>
[Chou, Cheng-Ying] Natl Taiwan Univ, Dept Biomechatron Engn, Taipei, Taiwan. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Wang, WC (corresponding author), Natl Taiwan Univ, Inst Appl Math Sci, Taipei, Taiwan.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
r09946020@ntu.edu.tw; liaomintsun@gmail.com; tlchen@stat.sinica.edu.tw; keitheva2009@gmail.com; chengying@ntu.edu.tw; wwang@ntu.edu.tw</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
National Taiwan University; National Taiwan University; National Taiwan University; Academia Sinica - Taiwan; National Taiwan University</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>CHOU, CHENG-YING</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-5737-6960&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Wang, Wen-Ching</display_name>&nbsp;</td><td>ABI-2253-2020&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Chen, Ting-Li</display_name>&nbsp;</td><td>AAF-5752-2020&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Chen, Ting-Li</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-1375-0164&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>LIAO, MIN-TSUN</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-6283-6554&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Wang, Weichung</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-6154-7750&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Liao, Min-Tsun</display_name>&nbsp;</td><td>GPG-6828-2022&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>ELSEVIER</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>RADARWEG 29, 1043 NX AMSTERDAM, NETHERLANDS</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Artificial Intelligence; Engineering, Biomedical; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Engineering; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>CF7A2</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0933-3657</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1873-2860</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>ARTIF INTELL MED</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Artif. Intell. Med.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>10</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Science and Technology Council, Taiwan</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Center for Theoretical Sciences Mathematics Division</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>All Vista Healthcare Sub-center</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This study was supported by the National Science and Technology Council, Taiwan, MOST All Vista Healthcare Sub-center, and National Center for Theoretical Sciences Mathematics Division. The funding source had no role in study design, data collection, analysis or in-terpretation, report writing, or the decision to submit this paper for publication.</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 4 of 38</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>ECG synthesis for cardiac arrhythmias: Integrating self-supervised learning and generative adversarial networks</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Simone, L (Simone, Lorenzo); Bacciu, D (Bacciu, Davide); Gervasi, V (Gervasi, Vincenzo)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>ARTIFICIAL INTELLIGENCE IN MEDICINE</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>167</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>103162</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1016/j.artmed.2025.103162</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2025 SEP</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>0</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>0</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
6</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
6</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>62</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Arrhythmia classifiers relying on supervised deep learning models usually require a substantial amount of labeled clinical data. The distribution of these labels is strictly related to the statistics of cardiovascular diseases among the population, which inherently narrows models' performance for classification tasks. Furthermore, during acquisition and data retrieval from electronic health records, concerns arise regarding patient anonymization due to stringent clinical policies. We introduce a conditional generative architecture for electrocardiography time series, which integrates self-supervision and generative adversarial principles. Empirical validation confirms the enhancement of morphological plausibility in synthetic data, showcasing its effectiveness in generating realistic signals. We propose a novel model (ECGAN), proving its capability of conditioning the probability distribution of ECG recordings. The proposed methodology is assessed upon various rhythm abnormalities including severe congestive heart failure, myocardial infarction, sinus rhythm, and premature ventricular contractions. Our proposed workflow for synthetic time series assessment demonstrates competitive performance compared to state-of-the-art models, achieving an average improvement of 2.4% in arrhythmia classification accuracy across MIT-BIH, BIDMC, and PTB datasets, while ensuring realistic synthetic data and improving training stability.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001504012300001</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>40460597</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Generative deep learning; Electrocardiography; Self-supervision; Generative-adversarial networks; Data augmentation</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
DYNAMICAL MODEL; COUPLED VAN; MECHANISMS; SIGNALS; IMPACT</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Simone, Lorenzo; Bacciu, Davide; Gervasi, Vincenzo] Univ Pisa, Dept Comp Sci, Largo Bruno Pontecorvo, I-56127 Pisa, Italy. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Simone, L (corresponding author), Univ Pisa, Dept Comp Sci, Largo Bruno Pontecorvo, I-56127 Pisa, Italy.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
lorenzo.simone@di.unipi.it</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
University of Pisa</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Bacciu, Davide</display_name>&nbsp;</td><td>A-3904-2013&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Simone, Lorenzo</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-5010-7733&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>ELSEVIER</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>RADARWEG 29, 1043 NX AMSTERDAM, NETHERLANDS</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Artificial Intelligence; Engineering, Biomedical; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Engineering; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>3MR9T</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0933-3657</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1873-2860</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>ARTIF INTELL MED</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Artif. Intell. Med.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>12</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 5 of 38</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>A Generalisable Heartbeat Classifier Leveraging Self-Supervised Learning for ECG Analysis During Magnetic Resonance Imaging</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Aublin, PG (Aublin, Pierre Gabriel); Felblinger, J (Felblinger, Jacques); Oster, J (Oster, Julien)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE JOURNAL OF BIOMEDICAL AND HEALTH INFORMATICS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>28</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>9</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>5147-5155</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/JBHI.2024.3411792</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2024 SEP</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>1</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>1</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
0</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
24</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>44</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Electrocardiogram (ECG) is acquired during Magnetic Resonance Imaging (MRI) to monitor patients and synchronize image acquisition with the heart motion. ECG signals are highly distorted during MRI due to the complex electromagnetic environment. Automated ECG analysis is therefore complicated in this context and there is no reference technique in MRI to classify pathological heartbeats. Imaging arrhythmic patients is hence difficult in MRI. Deep Learning based heartbeat classifier have been suggested but require large databases whereas existing annotated sets of ECG in MRI are very small. We proposed a Siamese network to leverage a large database of unannotated ECGs outside MRI. This was used to develop an efficient representation of ECG signals, further used to develop a heartbeat classifier. We extensively tested several data augmentations and self-supervised learning (SSL) techniques and assessed the generalization of the obtained classifier to ECG signals acquired in MRI. These augmentations included random noises and a model simulating MRI specific artefacts. SSL pretraining improved the generalizability of heartbeat classifiers in MRI (F1=0.75) compared to Deep Learning not relying on SSL (F1=0.46) and another classical machine learning approach (F1=0.40). These promising results seem to indicate that the use of SSL techniques can learn efficient ECG signal representation, and are useful for the development of Deep Learning models even when only scarce annotated medical data are available.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001309075700034</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>38857140</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Electrocardiogram; magnethohydrodynamic effect; deep learning; representation learning; Siamese network; Electrocardiogram; magnethohydrodynamic effect; deep learning; representation learning; Siamese network</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
ELECTROCARDIOGRAM; SIGNALS</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Aublin, Pierre Gabriel; Felblinger, Jacques; Oster, Julien] Univ Lorraine, IADI, U1254, Inserm, F-54000 Nancy, France. <br>
[Aublin, Pierre Gabriel] Univ Augsburg, Chair Diagnost Sensing, D-86159 Augsburg, Germany. <br>
[Felblinger, Jacques; Oster, Julien] Univ Lorraine, CHRU Nancy, Inserm, CIC IT1433, F-54035 Nancy, France. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Oster, J (corresponding author), Univ Lorraine, IADI, U1254, Inserm, F-54000 Nancy, France.<br>Oster, J (corresponding author), Univ Lorraine, CHRU Nancy, Inserm, CIC IT1433, F-54035 Nancy, France.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
julien.oster@inserm.fr</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Universite de Lorraine; Institut National de la Sante et de la Recherche Medicale (Inserm); University of Augsburg; CHU de Nancy; Universite de Lorraine; Institut National de la Sante et de la Recherche Medicale (Inserm)</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Felblinger, Jacques</display_name>&nbsp;</td><td>ABG-2033-2021&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Felblinger, jacques</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-9245-6595&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>oster, Julien</display_name>&nbsp;</td><td>F-3597-2017&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Aublin, Pierre G.</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-8082-7811&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Information Systems; Computer Science, Interdisciplinary Applications; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>F3T3E</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2168-2194</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>2168-2208</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE J BIOMED HEALTH</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE J. Biomed. Health Inform.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>9</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>ERA-CVD Joint Transnational Call 2019, MEIDIC-VTACH</grant_agency>&nbsp;</td><td>
<div>ANR-19-ECVD-0004&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Agence Nationale de la Recherche (ANR)</grant_agency>&nbsp;</td><td>
<div>ANR-19-ECVD-0004&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported by ERA-CVD Joint Transnational Call 2019, MEIDIC-VTACH under Grant ANR-19-ECVD-0004.</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 6 of 38</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>ECG Biometric Authentication Using Self-Supervised Learning for IoT Edge Sensors</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Wang, GX (Wang, Guoxin); Shanker, S (Shanker, Shreejith); Nag, A (Nag, Avishek); Lian, Y (Lian, Yong); John, D (John, Deepu)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE JOURNAL OF BIOMEDICAL AND HEALTH INFORMATICS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>28</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>11</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>6606-6618</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/JBHI.2024.3455803</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2024 NOV</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>10</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>11</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
6</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
31</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>43</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Wearable Internet of Things (IoT) devices are gaining ground for continuous physiological data acquisition and health monitoring. These physiological signals can be used for security applications to achieve continuous authentication and user convenience due to passive data acquisition. This paper investigates an electrocardiogram (ECG) based biometric user authentication system using features derived from the Convolutional Neural Network (CNN) and self-supervised contrastive learning. Contrastive learning enables us to use large unlabeled datasets to train the model and establish its generalizability. We propose approaches enabling the CNN encoder to extract appropriate features that distinguish the user from other subjects. When evaluated using the PTB ECG database with 290 subjects, the proposed technique achieved an authentication accuracy of 99.15%. To test its generalizability, we applied the model to two new datasets, the MIT-BIH Arrhythmia Database and the ECG-ID Database, achieving over 98.5% accuracy without any modifications. Furthermore, we show that repeating the authentication step three times can increase accuracy to nearly 100% for both PTBDB and ECGIDDB. This paper also presents model optimizations for embedded device deployment, which makes the system more relevant to real-world scenarios. To deploy our model in IoT edge sensors, we optimized the model complexity by applying quantization and pruning. The optimized model achieves 98.67% accuracy on PTBDB, with 0.48% accuracy loss and 62.6% CPU cycles compared to the unoptimized model. An accuracy-vs-time-complexity tradeoff analysis is performed, and results are presented for different optimization levels.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001359232100010</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>39250357</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Feature extraction; Electrocardiography; Authentication; Accuracy; Training; Contrastive learning; Biological system modeling; electrocardiogram authentication; IoT devices</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
FRAMEWORK</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Wang, Guoxin; John, Deepu] Univ Coll Dublin, Sch Elect &amp; Elect Engn, Dublin, Ireland. <br>
[Shanker, Shreejith] Trinity Coll Dublin, Dept Elect &amp; Elect Engn, Dublin, Ireland. <br>
[Nag, Avishek] Univ Coll Dublin, Sch Comp Sci, Dublin, Ireland. <br>
[Lian, Yong] York Univ, Dept Elect Engn &amp; Comp Sci, Toronto, ON M3J 1P3, Canada. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
John, D (corresponding author), Univ Coll Dublin, Sch Elect &amp; Elect Engn, Dublin, Ireland.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
guoxin.wang@ucdconnect.ie; shankers@tcd.ie; avishek.nag@ucd.ie; plian@yorku.ca; deepu.john@ucd.ie</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
University College Dublin; Trinity College Dublin; University College Dublin; York University - Canada</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Wang, Guoxin</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-3619-3399&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Lian, Yong</display_name>&nbsp;</td><td>MSY-3562-2025&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Shanker, Shreejith</display_name>&nbsp;</td><td>HGT-9976-2022&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Nag, Avishek</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-1702-1492&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Lian, Yong</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-5289-5219&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>John, Deepu</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-6139-1100&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>John, Deepu</display_name>&nbsp;</td><td>AAL-7045-2020&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Nag, Avishek</display_name>&nbsp;</td><td>T-9268-2019&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Wang, Guoxin</display_name>&nbsp;</td><td>LKM-8856-2024&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Information Systems; Computer Science, Interdisciplinary Applications; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>M7I4Z</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2168-2194</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>2168-2208</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE J BIOMED HEALTH</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE J. Biomed. Health Inform.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>13</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>China Scholarship Council</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>JEDAI Project through the Horizon 2020 FET Chist-Era Program</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Microelectronic Circuit Centre Ireland</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported in part by the China Scholarship Council, in part by the JEDAI Project through the Horizon 2020 FET Chist-Era Program and, in part by Microelectronic Circuit Centre Ireland.</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>Green Submitted</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 7 of 38</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>In-Distribution and Out-of-Distribution Self-Supervised ECG Representation Learning for Arrhythmia Detection</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Soltanieh, S (Soltanieh, Sahar); Hashemi, J (Hashemi, Javad); Etemad, A (Etemad, Ali)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE JOURNAL OF BIOMEDICAL AND HEALTH INFORMATICS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>28</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>2</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>789-800</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/JBHI.2023.3331626</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2024 FEB</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>4</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>5</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
7</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
41</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>44</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
This paper presents a systematic investigation into the effectiveness of Self-Supervised Learning (SSL) methods for Electrocardiogram (ECG) arrhythmia detection. We begin by conducting a novel analysis of the data distributions on three popular ECG-based arrhythmia datasets: PTB-XL, Chapman, and Ribeiro. To the best of our knowledge, our study is the first to quantitatively explore and characterize these distributions in the area. We then perform a comprehensive set of experiments using different augmentations and parameters to evaluate the effectiveness of various SSL methods, namely SimCRL, BYOL, and SwAV, for ECG representation learning, where we observe the best performance achieved by SwAV. Furthermore, our analysis shows that SSL methods achieve highly competitive results to those achieved by supervised state-of-the-art methods. To further assess the performance of these methods on both In-Distribution (ID) and Out-of-Distribution (OOD) ECG data, we conduct cross-dataset training and testing experiments. Our comprehensive experiments show almost identical results when comparing ID and OOD schemes, indicating that SSL techniques can learn highly effective representations that generalize well across different OOD datasets. This finding can have major implications for ECG-based arrhythmia detection. Lastly, to further analyze our results, we perform detailed per-disease studies on the performance of the SSL methods on the three datasets.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001203362500009</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>37948139</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Arrhythmia detection; contrastive learning; electrocardiogram; self-supervised learning; in-distribution; out-of-distribution</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Soltanieh, Sahar; Etemad, Ali] Queens Univ, Dept Elect &amp; Comp Engn, Kingston, ON K7L 3N6, Canada. <br>
[Hashemi, Javad] Queens Univ, Sch Comp, Kingston, ON K7L 3N6, Canada. <br>
[Etemad, Ali] Queens Univ, Ingenu Labs Res Inst, Kingston, ON K7L 3N6, Canada. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Soltanieh, S (corresponding author), Queens Univ, Dept Elect &amp; Comp Engn, Kingston, ON K7L 3N6, Canada.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
sahar.soltanieh@queensu.ca; javad.hashemi@queensu.ca; ali.etemad@queensu.ca</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Queens University - Canada; Queens University - Canada; Ingenuity Labs Research Institute; Queens University - Canada</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Hashemi, Javad</display_name>&nbsp;</td><td>D-5827-2019&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Soltanieh, Sahar</display_name>&nbsp;</td><td>LCE-1157-2024&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Soltanieh, Sahar</display_name>&nbsp;</td><td>&nbsp;</td><td>0009-0004-0724-1816&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Etemad, Ali</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-7128-0220&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Information Systems; Computer Science, Interdisciplinary Applications; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>NW0A0</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2168-2194</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>2168-2208</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE J BIOMED HEALTH</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE J. Biomed. Health Inform.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>12</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Natural Sciences and Engineering Research Council of Canada</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>No Statement Available</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 8 of 38</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Self-Supervised, Non-Contact Heartbeat Detection Based on Ballistocardiograms Utilizing Physiological Information Guidance</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Jiao, CZ (Jiao, Changzhe); Yang, AY (Yang, Aoyu); Zhao, HT (Zhao, Hantao); Yi, RH (Yi, Ruhan); Gou, SP (Gou, Shuiping); Sha, Y (Sha, Yu); Wen, WS (Wen, Wanshun); Jiao, LC (Jiao, Licheng); Skubic, M (Skubic, Marjorie)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE JOURNAL OF BIOMEDICAL AND HEALTH INFORMATICS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>29</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>4</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>2589-2602</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/JBHI.2024.3509875</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2025 APR</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>0</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>0</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
5</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
5</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>33</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Ballistocardiograms (BCG) is a passive, non-contact heart rate detection technology that requires no action on the part of the individual. However, during the BCG signal acquisition process, the surface pressure generated by cardiac contraction is easily disturbed by external factors, and as people's health deteriorates, the j-peak (the main peak of the BCG signal) is no longer prominent. Our aim is to establish a non-contact, self-supervised heart rate detection method based on physiological information, to improve the accuracy and robustness of BCG heart rate detection under wider and more adverse conditions. The algorithm is guided by the heart rate estimation based on BCG itself, thereby reconstructing a signal with physiological significance. We also propose a heartbeat mapping algorithm based on Bidirectional Long Short-Term Memory Network (BiLSTM) for extracting global deep features, achieving real-time heartbeat prediction, and eliminating local deviations brought about by reconstruction. To verify the effectiveness of the proposed method, this paper evaluated 40 young subjects and 4 elderly subjects. Compared with the existing state-of-the-art methods, beat-to-beat heart rate estimation and heartbeat detection both performed excellently, surpassing most methods using precise labels. The experimental results show that the proposed method achieves effective heartbeat detection, demonstrating robustness and effectiveness in the face of unavoidable noise and variations.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001459663700041</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>40030671</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Ballistocardiograms; heartbeat detection; non-contact; non-contact; physiological significance; physiological significance; self-supervised; self-supervised; self-supervised</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
BIDIRECTIONAL LSTM; SYSTEM</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Jiao, Changzhe; Yang, Aoyu; Zhao, Hantao; Gou, Shuiping; Sha, Yu; Jiao, Licheng] Xidian Univ, Sch Artificial Intelligence, Key Lab Intelligent Percept &amp; Image Understanding, Educ Minist China, Xian 710071, Peoples R China. <br>
[Yi, Ruhan; Skubic, Marjorie] Univ Missouri, Dept Elect Engn &amp; Comp Sci, Columbia, MO 65211 USA. <br>
[Wen, Wanshun] Zhejiang Prov Peoples Hosp, Dept Rehabil Med, Hangzhou 310024, Peoples R China. <br>
[Wen, Wanshun] Hangzhou Med Coll, Affiliated Peoples Hosp, Hangzhou 310053, Peoples R China. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Gou, SP (corresponding author), Xidian Univ, Sch Artificial Intelligence, Key Lab Intelligent Percept &amp; Image Understanding, Educ Minist China, Xian 710071, Peoples R China.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
shpgou@mail.xidian.edu.cn</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Xidian University; University of Missouri System; University of Missouri Columbia; Hangzhou Medical College; Zhejiang Provincial People's Hospital; Hangzhou Medical College</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Jiao, Changzhe</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-1392-8348&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Information Systems; Computer Science, Interdisciplinary Applications; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>0ZK6T</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2168-2194</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>2168-2208</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE J BIOMED HEALTH</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE J. Biomed. Health Inform.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>14</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Natural Science Foundation of China</grant_agency>&nbsp;</td><td>
<div>62372358&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Inter-discipline Guidance Project of Xidian University</grant_agency>&nbsp;</td><td>
<div>21103240003&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Natural Science Foundation of Zhejiang Province</grant_agency>&nbsp;</td><td>
<div>LGF20H020010&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported in part by the National Natural Science Foundation of China under Grant 62372358 in part by the Inter-discipline Guidance Project of Xidian University under Grant 21103240003, and in part by the Natural Science Foundation of Zhejiang Province under Grant LGF20H020010.</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 9 of 38</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Bootstrap each lead's latent: A novel method for self-supervised learning of multilead electrocardiograms</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Liu, WH (Liu, Wenhan); Pan, SR (Pan, Shurong); Li, ZT (Li, Zhoutong); Chang, S (Chang, Sheng); Huang, QJ (Huang, Qijun); Jiang, N (Jiang, Nan)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>COMPUTER METHODS AND PROGRAMS IN BIOMEDICINE</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>257</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>108452</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1016/j.cmpb.2024.108452</value>&nbsp;&nbsp;<b>Early Access Date:</b> 
<value>OCT 2024</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2024 DEC</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>1</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>1</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
2</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
5</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>39</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Background and Objective: Electrocardiogram (ECG) is one of the most important diagnostic tools for cardiovascular diseases (CVDs). Recent studies show that deep learning models can be trained using labeled ECGs to achieve automatic detection of CVDs, assisting cardiologists in diagnosis. However, the deep learning models heavily rely on labels in training, while manual labeling is costly and time-consuming. This paper proposes a new self-supervised learning (SSL) method for multilead ECGs: bootstrap each lead's latent (BELL) to reduce the reliance and boost model performance in various tasks, especially when training data are insufficient. Method: BELL is a variant of the well-known bootstrap your own latent (BYOL). The BELL aims to learn prior knowledge from unlabeled ECGs by pretraining, benefitting downstream tasks. It leverages the characteristics of multilead ECGs. First, BELL uses the multiple-branch skeleton, which is more effective in processing multilead ECGs. Moreover, it proposes intra-lead and inter-lead mean square error (MSE) to guide pretraining, and their fusion can result in better performances. Additionally, BELL inherits the main advantage of the BYOL: No negative pair is used in pretraining, making it more efficient. Results: In most cases, BELL surpasses previous works in the experiments. More importantly, the pretraining improves model performances by 0.69% similar to 8.89% in downstream tasks when only 10% of training data are available. Furthermore, BELL shows excellent adaptability to uncurated ECG data from a real-world hospital. Only slight performance degradation occurs (&lt;1% in most cases) when using these data. Conclusion: The results suggest that the BELL can alleviate the reliance on manual ECG labels from cardiologists, a critical bottleneck of the current deep learning-based models. In this way, the BELL can also help deep learning extend its application on automatic ECG analysis, reducing the cardiologists' burden in real-world diagnosis.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001337102200001</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>39393284</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Electrocardiogram (ECG); Self-supervised learning (SSL); Deep learning; Transfer learning</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
CONVOLUTIONAL NEURAL-NETWORK</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Liu, Wenhan; Jiang, Nan] East China Jiaotong Univ, Sch Informat &amp; Software Engn, Nanchang 330013, Peoples R China. <br>
[Pan, Shurong; Chang, Sheng; Huang, Qijun] Wuhan Univ, Sch Phys &amp; Technol, Wuhan 430072, Peoples R China. <br>
[Li, Zhoutong] Shanghai Jiao Tong Univ, Shanghai Peoples Hosp 9, Sch Med, Huangpu Branch, Shanghai 200011, Peoples R China. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Liu, WH (corresponding author), East China Jiaotong Univ, Sch Informat &amp; Software Engn, Nanchang 330013, Peoples R China.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
WHliu@ecjtu.edu.cn</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
East China Jiaotong University; Wuhan University; Shanghai Jiao Tong University</td>
</tr>

<tr>
<td>
<b>Publisher:</b>

<value>ELSEVIER IRELAND LTD</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>ELSEVIER HOUSE, BROOKVALE PLAZA, EAST PARK SHANNON, CO, CLARE, 00000, IRELAND</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Interdisciplinary Applications; Computer Science, Theory &amp; Methods; Engineering, Biomedical; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Engineering; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>J4X1X</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0169-2607</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1872-7565</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>COMPUT METH PROG BIO</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Comput. Meth. Programs Biomed.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>12</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 10 of 38</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Deep echocardiography: data-efficient supervised and semi-supervised deep learning towards automated diagnosis of cardiac disease</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Madani, A (Madani, Ali); Ong, JR (Ong, Jia Rui); Tibrewal, A (Tibrewal, Anshul); Mofrad, MRK (Mofrad, Mohammad R. K.)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>NPJ DIGITAL MEDICINE</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>1</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>59</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1038/s41746-018-0065-x</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2018 OCT 18</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>106</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>137</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
4</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
21</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>33</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Deep learning and computer vision algorithms can deliver highly accurate and automated interpretation of medical imaging to augment and assist clinicians. However, medical imaging presents uniquely pertinent obstacles such as a lack of accessible data or a high-cost of annotation. To address this, we developed data-efficient deep learning classifiers for prediction tasks in cardiology. Using pipeline supervised models to focus relevant structures, we achieve an accuracy of 94.4% for 15-view still-image echocardiographic view classification and 91.2% accuracy for binary left ventricular hypertrophy classification. We then develop semi-supervised generative adversarial network models that can learn from both labeled and unlabeled data in a generalizable fashion. We achieve greater than 80% accuracy in view classification with only 4% of labeled data used in solely supervised techniques and achieve 92.3% accuracy for left ventricular hypertrophy classification. In exploring trade-offs between model type, resolution, data resources, and performance, we present a comprehensive analysis and improvements of efficient deep learning solutions for medical imaging assessment especially in cardiology.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000449685000001</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>31304338</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
COMPUTER-AIDED DIAGNOSIS; VIEW CLASSIFICATION; NEURAL-NETWORKS</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Madani, Ali; Mofrad, Mohammad R. K.] Univ Calif Berkeley, Dept Bioengn &amp; Mech Engn, Mol Cell Biomech Lab, Berkeley, CA 94720 USA. <br>
[Ong, Jia Rui; Tibrewal, Anshul] Univ Calif Berkeley, Dept Elect Engn &amp; Comp Sci, Berkeley, CA 94720 USA. <br>
[Mofrad, Mohammad R. K.] UCSF, Berkeley Grad Program Bioengn, Berkeley, CA 94143 USA. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Mofrad, MRK (corresponding author), Univ Calif Berkeley, Dept Bioengn &amp; Mech Engn, Mol Cell Biomech Lab, Berkeley, CA 94720 USA.<br>Mofrad, MRK (corresponding author), UCSF, Berkeley Grad Program Bioengn, Berkeley, CA 94143 USA.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
mofrad@berkeley.edu</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
University of California System; University of California Berkeley; University of California System; University of California Berkeley; University of California System; University of California San Francisco</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Kaazempur Mofrad, Mohammad</display_name>&nbsp;</td><td>R-6010-2019&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Mofrad, Mohammad</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-7004-4859&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>NATURE PORTFOLIO</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>HEIDELBERGER PLATZ 3, BERLIN, 14197, GERMANY</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Health Care Sciences &amp; Services; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Health Care Sciences &amp; Services; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>GZ7TB</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2398-6352</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>NPJ DIGIT MED</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>npj Digit. Med.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>11</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Dr. Leopold und Carmen Ellinger Stiftung grant through the UCT Three-Way PhD Global Partnership Programme</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>We would like to thank Xuanyu Zhou for assistance with data engineering and initial tests. All datasets were obtained and de-identified, with waived consent in compliance with the Institutional Review Board (IRB) at the University of California, San Francisco (UCSF). Methods were performed in accordance with relevant regulations and guidelines. Lastly, we would like to thank members of UCSF (clinicians, programmers, and staff) that assisted in support tasks. A.M. was partially supported by the Dr. Leopold und Carmen Ellinger Stiftung grant through the UCT Three-Way PhD Global Partnership Programme (to M.M.).</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>Green Published, gold</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 11 of 38</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>An edge-device-compatible algorithm for valvular heart diseases screening using phonocardiogram signals with a lightweight convolutional neural network and self-supervised learning</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Ma, SC (Ma, Shichao); Chen, JY (Chen, Junyi); Ho, JWK (Ho, Joshua W. K.)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>COMPUTER METHODS AND PROGRAMS IN BIOMEDICINE</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>243</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>107906</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1016/j.cmpb.2023.107906</value>&nbsp;&nbsp;<b>Early Access Date:</b> 
<value>NOV 2023</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2024 JAN</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>10</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>11</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
3</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
24</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>67</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Background and objectives: Detection and classification of heart murmur using mobile-phone-collected sound is an emerging approach to the scale-up screening of valvular heart disease at a population level. Nonetheless, the widespread adoption of artificial intelligence (AI) methods for this type of mobile health (mHealth) application requires highly accurate and lightweight AI models that can be deployed in consumer-grade mobile devices. This study presents a lightweight deep learning model and a self-supervised learning (SSL) method to utilise unla-belled data to improve the accuracy of valvular heart disease classification using phonocardiogram data.Methods: This study proposes a lightweight convolutional neural network (CNN) that consists of ten times fewer parameters than other deep learning models to classify phonocardiogram data. SSL is applied to harness a large collection of unlabelled data as pre-training to enhance the accuracy and robustness of the model and reduce the number of epochs required to converge. A mobile application prototype that encapsulates the model is developed to perform in-device inference and fine-turning.Results: The proposed lightweight model achieves an average accuracy of 98.65% in 10-fold cross-validation. When coupled with SSL using unlabelled data, the pre-trained model can reach an average accuracy higher than 99.4% in 10-fold cross-validation. Furthermore, SSL-trained models have a 4-20% improvement in classification accuracy over non-SSL-trained models when tested with perturbed or noisy data, suggesting that SSL improves robustness of the model. When deployed on common smartphones, in-device fine-tuning and inference of the model can be completed within 0.03-0.37 s, which is considerably faster than 0.22-5.7 s by a standard CNN model that have ten times the number of parameters. Our lightweight model also consumes only a third of the power compared to the larger standard model.Conclusion: This work presents a lightweight and accurate phonocardiogram classifier that supports near real-time performance on standard mobile devices.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001111884500001</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>37950925</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Valvular heart diseases screening; Self -supervised learning; Deep learning; Convolutional neural network; Mobile health; Digital health</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
AUTOENCODER; SYSTEM</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Ma, Shichao; Chen, Junyi; Ho, Joshua W. K.] Univ Hong Kong, Li Ka Shing Fac Med, Sch Biomed Sci, Pokfulam, Hong Kong, Peoples R China. <br>
[Ma, Shichao; Chen, Junyi; Ho, Joshua W. K.] Lab Data Discovery Hlth Ltd D24H, Hong Kong Sci Pk, Hong Kong, Peoples R China. <br>
[Ho, Joshua W. K.] Univ Hong Kong, Sch Biomed Sci, Pokfulam, L4 Lab Block 44 21,Sassoon Rd, Hong Kong, Peoples R China. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Ho, JWK (corresponding author), Univ Hong Kong, Sch Biomed Sci, Pokfulam, L4 Lab Block 44 21,Sassoon Rd, Hong Kong, Peoples R China.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
jwkho@hku.hk</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
University of Hong Kong; University of Hong Kong</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Ma, Shichao</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-4792-9055&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Chen, Junyi</display_name>&nbsp;</td><td>JZD-3402-2024&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Ho, Joshua W. K.</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-2331-7011&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>junyi, CHEN</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-9992-7273&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Ma, Shichao</display_name>&nbsp;</td><td>JVN-1985-2024&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Ho, Joshua</display_name>&nbsp;</td><td>AAK-7759-2020&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>ELSEVIER IRELAND LTD</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>ELSEVIER HOUSE, BROOKVALE PLAZA, EAST PARK SHANNON, CO, CLARE, 00000, IRELAND</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Interdisciplinary Applications; Computer Science, Theory &amp; Methods; Engineering, Biomedical; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Engineering; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>Z4OK1</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0169-2607</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1872-7565</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>COMPUT METH PROG BIO</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Comput. Meth. Programs Biomed.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>12</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>AIR@InnoHK</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported in part by AIR@InnoHK administered by the Innovation and Technology Commission. Many thanks to Dr Chi Yan Ooi and Emily Nicola Estrada Stables for proofreading the manuscript.</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 12 of 38</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>CalibrationPhys: Self-Supervised Video-Based Heart and Respiratory Rate Measurements by Calibrating Between Multiple Cameras</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Akamatsu, Y (Akamatsu, Yusuke); Umematsu, T (Umematsu, Terumi); Imaoka, H (Imaoka, Hitoshi)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE JOURNAL OF BIOMEDICAL AND HEALTH INFORMATICS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>28</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>3</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>1460-1471</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/JBHI.2023.3345486</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2024 MAR</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>5</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>6</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
3</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
8</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>45</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Video-based heart and respiratory rate measurements using facial videos are more useful and user-friendly than traditional contact-based sensors. However, most of the current deep learning approaches require ground-truth pulse and respiratory waves for model training, which are expensive to collect. In this paper, we propose CalibrationPhys, a self-supervised video-based heart and respiratory rate measurement method that calibrates between multiple cameras. CalibrationPhys trains deep learning models without supervised labels by using facial videos captured simultaneously by multiple cameras. Contrastive learning is performed so that the pulse and respiratory waves predicted from the synchronized videos using multiple cameras are positive and those from different videos are negative. CalibrationPhys also improves the robustness of the models by means of a data augmentation technique and successfully leverages a pre-trained model for a particular camera. Experimental results utilizing two datasets demonstrate that CalibrationPhys outperforms state-of-the-art heart and respiratory rate measurement methods. Since we optimize camera-specific models using only videos from multiple cameras, our approach makes it easy to use arbitrary cameras for heart and respiratory rate measurements.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001180907300045</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>38127597</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Remote photoplethysmography; heart rate; respiratory rate; facial video; convolutional neural network; contrastive learning; pre-training</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
NONCONTACT</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Akamatsu, Yusuke; Umematsu, Terumi; Imaoka, Hitoshi] NEC Corp Ltd, Biometr Res Labs, Kawasaki 2118666, Japan. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Akamatsu, Y (corresponding author), NEC Corp Ltd, Biometr Res Labs, Kawasaki 2118666, Japan.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
yusuke-akamatsu@nec.com; terumi@nec.com; h-imaoka_cb@nec.com</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
NEC Corporation</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Akamatsu, Yusuke</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-9123-3955&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Akamatsu, Yusuke</display_name>&nbsp;</td><td>HNO-9146-2023&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Umematsu, Terumi</display_name>&nbsp;</td><td>AAY-6635-2021&nbsp;</td><td>0000-0002-6328-0674&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Information Systems; Computer Science, Interdisciplinary Applications; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>KO4Y9</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2168-2194</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>2168-2208</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE J BIOMED HEALTH</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE J. Biomed. Health Inform.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>12</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 13 of 38</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>An Efficient and Private ECG Classification System Using Split and Semi-Supervised Learning</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Ayad, A (Ayad, Ahmad); Barhoush, M (Barhoush, Mahdi); Frei, M (Frei, Marian); Volker, B (Volker, Benedikt); Schmeink, A (Schmeink, Anke)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE JOURNAL OF BIOMEDICAL AND HEALTH INFORMATICS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>27</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>9</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>4261-4272</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/JBHI.2023.3281977</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2023 SEP</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>6</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>6</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
1</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
17</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>68</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Electrocardiography (ECG) is a standard diagnostic tool for evaluating the overall heart's electrical activity and is vital for detecting many cardiovascular diseases. Classifying ECG recordings using deep neural networks has been investigated in literature and has shown very good performance. However, this performance assumes that the training data is centralized, which is often not the case in real-life scenarios, where data resides in multiple places and only a small portion of it is labeled. Therefore, in this work, we propose an ECG classification system that focuses on preserving data privacy and enhancing overall system efficiency. We analyzed the complexity of previously proposed deep learning-based models and showed that the temporal convolutional network-based models (TCN) were the most efficient. Then, we built on the TCN models a modified split-learning (SL) system that achieves the same classification performance as the basic SL but reduces the communication overhead between the server and the client by 71.7% as well as reducing the computations at the client by 46.5% compared to the original SL system based on the TCN network. Finally, we implement semi-supervised learning in our system to enhance its classification performance by 9.1%-15.7%, when the training data consists only of 10% labeled data. We have tested our proposed system on a test IoT setup and it achieved satisfactory classification accuracy while being private and energy efficient for green-AI applications.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001064467900006</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>37262112</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
ECG; privacy-aware machine learning; distributed machine learning; edge computing; energy-efficient machine learning; split-learning; autoencoder; semi-supervised learning; mean teacher; FixMatch</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Ayad, Ahmad; Barhoush, Mahdi; Frei, Marian; Volker, Benedikt; Schmeink, Anke] RWTH Univ, Chair Informat Theory &amp; Data Analyt INDA, D-52074 Aachen, Germany. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Ayad, A (corresponding author), RWTH Univ, Chair Informat Theory &amp; Data Analyt INDA, D-52074 Aachen, Germany.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
ahmad.ayad@inda.rwth-aachen.de; mahdi.barhoush@inda.rwth-aachen.de; marian.frei@rwth-aachen.de; benedikt.voelker@rwth-aachen.de; anke.schmeink@inda.rwth-aachen.de</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
RWTH Aachen University</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Ayad, Ahmad</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-9081-1274&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Schmeink, Anke</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-9929-2925&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Frei, Marian</display_name>&nbsp;</td><td>&nbsp;</td><td>0009-0002-6899-3013&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Barhoush, Mahdi</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-3787-0662&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Volker, Benedikt</display_name>&nbsp;</td><td>&nbsp;</td><td>0009-0009-7266-7676&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Information Systems; Computer Science, Interdisciplinary Applications; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>R5AE7</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2168-2194</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>2168-2208</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE J BIOMED HEALTH</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE J. Biomed. Health Inform.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>12</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Federal Ministry of Education and Research (BMBF, Germany) through NeuroSys: Impulse durch Anwendungen (Projekt D)</grant_agency>&nbsp;</td><td>
<div>03ZU1106DA&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported by the Federal Ministry of Education and Research (BMBF, Germany) through NeuroSys: Impulse durch Anwendungen (Projekt D) under Grant 03ZU1106DA.&amp; nbsp;</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 14 of 38</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Semi-Supervised Learning for Low-Cost Personalized Obstructive Sleep Apnea Detection Using Unsupervised Deep Learning and Single-Lead Electrocardiogram</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Hu, SC (Hu, Shuaicong); Wang, YN (Wang, Ya'nan); Liu, J (Liu, Jian); Yang, CW (Yang, Cuiwei); Wang, AG (Wang, Aiguo); Li, KZ (Li, Kuanzheng); Liu, WX (Liu, Wenxin)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE JOURNAL OF BIOMEDICAL AND HEALTH INFORMATICS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>27</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>11</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>5281-5292</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/JBHI.2023.3304299</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2023 NOV</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>25</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>27</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
5</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
32</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>38</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Objective: Obstructive sleep apnea (OSA) is a common sleep-related breathing disorder that can lead to a wide range of health issues if left untreated. This study aims to address the lack of research on personalized models for single-lead electrocardiogram (ECG)-based OSA detection, by proposing an automatic semi-supervised algorithm for automated low-cost personalization fine-tuning. Methods: We utilize a convolutional neural network (CNN)-based auto-encoder (AE) with a modified training objective to detect anomalous region of OSA. An indicator based on model outputs is utilized as a benchmark measure to assign pseudo-labels with confidence to each sample. Finally, we perform validation of the semi-supervised algorithm on the same database and cross-database scenarios. Results: By introducing semi-supervised personalization, the accuracy, AUC, and mean absolute error (MAE) of the general model (GM) of 35 subjects from the same database are improved from 86.3%, 0.915, and 5.178 to 90.3%, 0.948, and 2.593. Simultaneously, in the validation of 25 subjects from a cross-database, the accuracy, AUC, and MAE of the GM are enhanced from 75.6%, 0.800, and 9.149 to 84.3%, 0.881, and 3.509. Conclusion: The improved version of AE demonstrates excellent adaptability in identifying abnormal features in OSA, employing a data-driven approach to assign pseudo-labels for unknown data automatically. Additionally, leveraging the pseudo-labels through a semi-supervised fine-tuning strategy provides a solution to overcome the limitation of clinical annotations, facilitating low-cost implementation of personalized models. Significance: The semisupervised approach proposed in this article provides a high-performance and annotation-free solution for personalized adjustment of automatic OSA detection.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001129955100008</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>37566509</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Obstructive sleep apnea (OSA); electrocardiogram (ECG); auto-encoder (AE); semi-supervised</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
SYSTEM</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Hu, Shuaicong; Wang, Ya'nan; Liu, Jian] Fudan Univ, Sch Informat Sci &amp; Technol, Ctr Biomed Engn, Shanghai 200433, Peoples R China. <br>
[Yang, Cuiwei] Fudan Univ, Sch Informat Sci &amp; Technol, Ctr Biomed Engn, Shanghai 200433, Peoples R China. <br>
[Yang, Cuiwei] Key Lab Med Imaging Comp &amp; Comp Assisted Interven, Shanghai 200093, Peoples R China. <br>
[Wang, Aiguo; Li, Kuanzheng; Liu, Wenxin] Xinghua City Peoples Hosp, Taizhou 225700, Jiangsu, Peoples R China. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Yang, CW (corresponding author), Fudan Univ, Sch Informat Sci &amp; Technol, Ctr Biomed Engn, Shanghai 200433, Peoples R China.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
22110720104@m.fudan.edu.cn; wangyn20@fudan.edu.cn; 22110720117@m.fudan.edu.cn; yangcw@fudan.edu.cn; xhwag@163.com; stronger1831@126.com; 450930173@qq.com</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Fudan University; Fudan University</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Liu, Wenxin</display_name>&nbsp;</td><td>LXW-5765-2024&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Yang, Cuiwei</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-3338-5835&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Hu, shuaicong</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-5458-0416&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Wang, Ya'nan</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-5599-3423&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Wang, Aiguo</display_name>&nbsp;</td><td>HNP-7198-2023&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Hu, Shuaicong</display_name>&nbsp;</td><td>NIU-9967-2025&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Liu, Jian</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-5791-7221&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Information Systems; Computer Science, Interdisciplinary Applications; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>DC9R2</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2168-2194</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>2168-2208</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE J BIOMED HEALTH</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE J. Biomed. Health Inform.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>12</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Shanghai Science and Technology Support Project</grant_agency>&nbsp;</td><td>
<div>18441900900&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Medical Scientific Research Key Project of Jiangsu Commission of Health</grant_agency>&nbsp;</td><td>
<div>ZDB2020025&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported in part by Shanghai Science and Technology Support Project under Grant 18441900900 and in part by the Medical Scientific Research Key Project of Jiangsu Commission of Health under Grant ZDB2020025.</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 15 of 38</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>CPSS: Fusing consistency regularization and pseudo-labeling techniques for semi-supervised deep cardiovascular disease detection using all unlabeled electrocardiograms</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Shi, JG (Shi, Jiguang); Liu, WH (Liu, Wenhan); Zhang, HC (Zhang, Huaicheng); Chang, S (Chang, Sheng); Wang, H (Wang, Hao); He, J (He, Jin); Huang, QJ (Huang, Qijun)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>COMPUTER METHODS AND PROGRAMS IN BIOMEDICINE</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>254</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>108315</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1016/j.cmpb.2024.108315</value>&nbsp;&nbsp;<b>Early Access Date:</b> 
<value>JUL 2024</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2024 SEP</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>1</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>1</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
4</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
18</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>45</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Background and objective: Deep learning usually achieves good performance in the supervised way, which requires a large amount of labeled data. However, manual labeling of electrocardiograms (ECGs) is laborious that requires much medical knowledge. Semi-supervised learning (SSL) provides an effective way of leveraging unlabeled data to improve model performance, providing insight for solving this problem. The objective of this study is to improve the performance of cardiovascular disease (CVD) detection by fully utilizing unlabeled ECG. Methods: A novel SSL algorithm fusing consistency regularization and pseudo-labeling techniques (CPSS) is proposed. CPSS consists of supervised learning and unsupervised learning. For supervised learning, the labeled ECGs are mapped into prediction vectors by the classifier. The cross-entropy loss function is used to optimize the classifier. For unsupervised learning, the unlabeled ECGs are weakly and strongly augmented, and a consistency loss is used to minimize the difference between the classifier's predictions for the two augmentations. Pseudo-labeling techniques include positive pseudo-labeling (PL) and ranking-based negative pseudo-labeling (RNL). PL introduces pseudo-labels for data with high prediction confidence. RNL assigns negative pseudo-labels to the lower-ranked categories in the prediction vectors to leverage data with low prediction confidence. In this study, VGGNet and ResNet are used as classifiers, which are jointly optimized by labeled and unlabeled ECGs. Results: CPSS has been validated on several databases. With the same number of labeled ECGs (10%), it improves the accuracies over pure supervised learning by 13.59%, 4.60%, and 5.38% in the CPSC2018, PTB-XL, and Chapman databases, respectively. CPSS achieves comparable results to the fully supervised method with only 10% of labeled ECGs, which reduces the labeling workload by 90%. In addition, to verify the practicality of CPSS, a cardiovascular disease monitoring system is designed by heterogeneously deploying the trained classifiers on an SoC (system-on-a-chip), which can detect CVD in real time. Conclusion: The results of this study indicate that the proposed CPSS can significantly improve the performance of CVD detection using unlabeled ECG, which reduces the burden of ECG labeling in deep learning. In addition, the designed monitoring system makes the proposed CPSS promising for real-world applications.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001352992800001</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>38991373</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Electrocardiogram (ECG); Semi-supervised learning (SSL); Consistency regularization; Pseudo-labeling; System-on-a-chip (SoC)</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Shi, Jiguang; Liu, Wenhan; Zhang, Huaicheng; Chang, Sheng; Wang, Hao; He, Jin; Huang, Qijun] Wuhan Univ, Sch Phys &amp; Technol, Wuhan 430072, Peoples R China. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Huang, QJ (corresponding author), Wuhan Univ, Sch Phys &amp; Technol, Wuhan 430072, Peoples R China.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
huangqj@whu.edu.cn</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Wuhan University</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Shi, Jiguang</display_name>&nbsp;</td><td>LDN-4491-2024&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Zhang, Huaicheng</display_name>&nbsp;</td><td>KFA-9029-2024&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>ELSEVIER IRELAND LTD</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>ELSEVIER HOUSE, BROOKVALE PLAZA, EAST PARK SHANNON, CO, CLARE, 00000, IRELAND</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Interdisciplinary Applications; Computer Science, Theory &amp; Methods; Engineering, Biomedical; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Engineering; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>L8E2Q</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0169-2607</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1872-7565</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>COMPUT METH PROG BIO</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Comput. Meth. Programs Biomed.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>13</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 16 of 38</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>A foundational vision transformer improves diagnostic performance for electrocardiograms</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Vaid, A (Vaid, Akhil); Jiang, J (Jiang, Joy); Sawant, A (Sawant, Ashwin); Lerakis, S (Lerakis, Stamatios); Argulian, E (Argulian, Edgar); Ahuja, Y (Ahuja, Yuri); Lampert, J (Lampert, Joshua); Charney, A (Charney, Alexander); Greenspan, H (Greenspan, Hayit); Narula, J (Narula, Jagat); Glicksberg, B (Glicksberg, Benjamin); Nadkarni, GN (Nadkarni, Girish N.)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>NPJ DIGITAL MEDICINE</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>6</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>1</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>108</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1038/s41746-023-00840-9</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2023 JUN 6</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>45</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>51</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
4</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
45</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>30</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
The electrocardiogram (ECG) is a ubiquitous diagnostic modality. Convolutional neural networks (CNNs) applied towards ECG analysis require large sample sizes, and transfer learning approaches for biomedical problems may result in suboptimal performance when pre-training is done on natural images. We leveraged masked image modeling to create a vision-based transformer model, HeartBEiT, for electrocardiogram waveform analysis. We pre-trained this model on 8.5 million ECGs and then compared performance vs. standard CNN architectures for diagnosis of hypertrophic cardiomyopathy, low left ventricular ejection fraction and ST elevation myocardial infarction using differing training sample sizes and independent validation datasets. We find that HeartBEiT has significantly higher performance at lower sample sizes compared to other models. We also find that HeartBEiT improves explainability of diagnosis by highlighting biologically relevant regions of the EKG vs. standard CNNs. Domain specific pre-trained transformer models may exceed the classification performance of models trained on natural images especially in very low data regimes. The combination of the architecture and such pre-training allows for more accurate, granular explainability of model predictions.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001003671300002</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>37280346</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
NEURAL-NETWORKS</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Vaid, Akhil; Jiang, Joy; Charney, Alexander; Nadkarni, Girish N.] Icahn Sch Med Mt Sinai, Charles Bronfman Inst Personalized Med, New York, NY 10029 USA. <br>
[Vaid, Akhil; Jiang, Joy; Nadkarni, Girish N.] Icahn Sch Med Mt Sinai, Mt Sinai Clin Intelligence Ctr, New York, NY 10029 USA. <br>
[Vaid, Akhil; Charney, Alexander; Glicksberg, Benjamin; Nadkarni, Girish N.] Icahn Sch Med Mt Sinai, Dept Genet &amp; Genom Sci, New York, NY 10029 USA. <br>
[Vaid, Akhil; Glicksberg, Benjamin; Nadkarni, Girish N.] Hasso Plattner Inst Digital Hlth Mt Sinai, New York, NY 10029 USA. <br>
[Sawant, Ashwin] Icahn Sch Med Mt Sinai, Dept Med, New York, NY USA. <br>
[Lerakis, Stamatios; Argulian, Edgar; Lampert, Joshua; Narula, Jagat] Icahn Sch Med Mt Sinai, Mt Sinai Heart, New York, NY USA. <br>
[Lerakis, Stamatios; Argulian, Edgar; Lampert, Joshua; Narula, Jagat] Icahn Sch Med Mt Sinai, Dept Cardiol, New York, NY USA. <br>
[Ahuja, Yuri] NYU Langone Hlth, Dept Med, New York, NY USA. <br>
[Charney, Alexander] Icahn Sch Med Mt Sinai, Pamela Sklar Div Psychiat Genom, New York, NY USA. <br>
[Charney, Alexander] Icahn Sch Med Mt Sinai, Dept Psychiat, New York, NY USA. <br>
[Greenspan, Hayit] Tel Aviv Univ, Dept Biomed Engn, IL-6997801 Tel Aviv, Israel. <br>
[Nadkarni, Girish N.] Icahn Sch Med Mt Sinai, Dept Med, Div Nephrol, New York, NY USA. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Vaid, A (corresponding author), Icahn Sch Med Mt Sinai, Charles Bronfman Inst Personalized Med, New York, NY 10029 USA.<br>Vaid, A (corresponding author), Icahn Sch Med Mt Sinai, Mt Sinai Clin Intelligence Ctr, New York, NY 10029 USA.<br>Vaid, A (corresponding author), Icahn Sch Med Mt Sinai, Dept Genet &amp; Genom Sci, New York, NY 10029 USA.<br>Vaid, A (corresponding author), Hasso Plattner Inst Digital Hlth Mt Sinai, New York, NY 10029 USA.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
akhil.vaid@mssm.edu</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Icahn School of Medicine at Mount Sinai; Icahn School of Medicine at Mount Sinai; Icahn School of Medicine at Mount Sinai; Icahn School of Medicine at Mount Sinai; Icahn School of Medicine at Mount Sinai; Icahn School of Medicine at Mount Sinai; NYU Langone Medical Center; Icahn School of Medicine at Mount Sinai; Icahn School of Medicine at Mount Sinai; Tel Aviv University; Icahn School of Medicine at Mount Sinai</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Vaid, Akhil</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-3343-744X&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Lerakis, Stamatios</display_name>&nbsp;</td><td>AAR-7597-2021&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>mirzaei, hamed</display_name>&nbsp;</td><td>X-2374-2018&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Glicksberg, Benjamin</display_name>&nbsp;</td><td>I-9500-2019&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Lampert, Joshua</display_name>&nbsp;</td><td>CAG-2250-2022&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>NATURE PORTFOLIO</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>HEIDELBERGER PLATZ 3, BERLIN, 14197, GERMANY</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Health Care Sciences &amp; Services; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Health Care Sciences &amp; Services; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>I6DO1</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2398-6352</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>NPJ DIGIT MED</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>npj Digit. Med.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>8</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>&nbsp;</td><td>
<div>R01HL155915&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>&nbsp;</td><td>
<div>UL1TR004419&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>AcknowledgementsThis study was funded by R01HL155915 and Clinical and translational award for infrastructure UL1TR004419. The authors would like to thank Wei Guo, Lili Gai, and Eugene Fluder of the High Performance Computing group at Mount Sinai for making the infrastructure underlying this study possible.</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>gold, Green Published</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 17 of 38</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>TAC-ECG: A task-adaptive classification method for electrocardiogram based on cross-modal contrastive learning and low-rank convolutional adapter</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Wang, RJ (Wang, Rongjia); Dong, XD (Dong, Xunde); Liu, XL (Liu, Xiuling); Fang, YH (Fang, Yihai); Dou, JH (Dou, Jianhong); Qiang, YP (Qiang, Yupeng); Yang, Y (Yang, Yang); Hu, F (Hu, Fei)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>COMPUTER METHODS AND PROGRAMS IN BIOMEDICINE</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>270</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>108918</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1016/j.cmpb.2025.108918</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2025 OCT</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>0</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>0</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
11</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
11</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>70</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Background and Objective: Cardiovascular diseases are one of the major health threats to humans. Researchers have proposed numerous deep learning-based methods for the automatic analysis of electrocardiogram (ECG), achieving encouraging results. However, many existing methods are limited to task-specific model training and require retraining or full fine-tuning when confronted with a new ECG classification task, thus lacking flexibility in clinical applications. Methods: In this study, we propose a Task-Adaptive Classification method for ECG (TAC-ECG) based on cross-modal contrastive learning and low-rank convolutional adapters. TAC-ECG comprises two main phases. In the first phase, inspired by the Contrastive Language-Image Pre-training, we design the Contrastive ECG-Text Pre-training (CETP) to pre-train a robust ECG encoder. In the second phase, the pre-trained ECG encoder is frozen and integrated with a lightweight plug-in, the Low-Rank Convolutional Adapter (LRC-Adapter), forming an extensible ECG classification model. The frozen encoder extracts more discriminative features from the ECG signal, while the LRC-Adapter enables task-specific adaptation. For diverse ECG classification tasks, TAC-ECG only requires training the LRC-Adapter. This mechanism enables TAC-ECG to efficiently perform different ECG classification tasks, significantly reducing resource consumption and deployment costs in multi-tasking scenarios compared to traditional fully fine-tuned methods. Results: We conducted extensive experiments using six different network architectures as ECG encoders. Specifically, we performed ECG classification experiments on four datasets: CPSC2018, Cinc2017, PTB-XL, and Chapman, targeting 9-category, 3-category, 5-category, and 4-category classifications respectively. The TAC-ECG achieved highly competitive results using only approximately 3% of the trainable parameters and approximately 25% of the total parameters compared to the fully fine-tuned method. These results demonstrates the effectiveness and practicality of the TAC-ECG method. Conclusion: The TAC-ECG offers a flexible and efficient method for ECG classification, enabling rapid adaptation to diverse tasks and enhancing clinical diagnostic practicality.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001534968200001</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>40644854</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Electrocardiogram; Contrastive learning; Low-rank convolutional adapter; Task-adaptive</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
MYOCARDIAL-INFARCTION; NEURAL-NETWORK</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Wang, Rongjia; Dong, Xunde; Qiang, Yupeng; Yang, Yang; Hu, Fei] South China Univ Technol, Sch Automat Sci &amp; Engn, Guangzhou 510641, Peoples R China. <br>
[Liu, Xiuling] Hebei Univ, Coll Elect &amp; Informat Engn, Key Lab Digital Med Engn Hebei Prov, Baoding 071002, Peoples R China. <br>
[Fang, Yihai] Southern Med Univ, Zhujiang Hosp, Guangzhou 510280, Peoples R China. <br>
[Dou, Jianhong] Gen Hosp Southern Theater Command, Chinese Peoples Liberat Army, Guangzhou 510030, Peoples R China. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Dong, XD (corresponding author), South China Univ Technol, Sch Automat Sci &amp; Engn, Guangzhou 510641, Peoples R China.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
auwangrongjia@mail.scut.edu.cn; audxd@scut.edu.cn; liuxiuling121@hotmail.com; fang-yihai@163.com; doujianhong@hotmail.com; auqyp@mail.scut.edu.cn; 202221018052@mail.scut.edu.cn; 202320116052@mail.scut.edu.cn</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
South China University of Technology; Hebei University; Southern Medical University - China; Southern Theater Command General Hospital</td>
</tr>

<tr>
<td>
<b>Publisher:</b>

<value>ELSEVIER IRELAND LTD</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>ELSEVIER HOUSE, BROOKVALE PLAZA, EAST PARK SHANNON, CO, CLARE, 00000, IRELAND</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Interdisciplinary Applications; Computer Science, Theory &amp; Methods; Engineering, Biomedical; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Engineering; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>5GG4B</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0169-2607</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1872-7565</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>COMPUT METH PROG BIO</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Comput. Meth. Programs Biomed.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>12</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Natural Science Foundation of China</grant_agency>&nbsp;</td><td>
<div>82472110&nbsp;</div>
<div>U20A20224&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Natural Science Foundation of Guangdong Province, China</grant_agency>&nbsp;</td><td>
<div>2024A1515220144&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported by the National Natural Science Foundation of China (No. 62450100) ; the Natural Science Foundation of Guangdong Province, China (No. 2024A1515220144) ; the National Natural Science Foundation of China (Nos. 82472110 and U20A20224) .</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 18 of 38</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Constraint-Based Unsupervised Domain Adaptation Network for Multi-Modality Cardiac Image Segmentation</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Du, XQ (Du, Xiuquan); Liu, YG (Liu, Yueguo)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE JOURNAL OF BIOMEDICAL AND HEALTH INFORMATICS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>26</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>1</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>67-78</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/JBHI.2021.3126874</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2022 JAN</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>14</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>17</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
3</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
34</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>44</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
The cardiac CT and MRI images depict the various structures of the heart, which are very valuable for analyzing heart function. However, due to the difference in the shape of the cardiac images and imaging techniques, automatic segmentation is challenging. To solve this challenge, in this paper, we propose a new constraint-based unsupervised domain adaptation network. This network first performs mutual translation of images between different domains, it can provide training data for the segmentation model, and ensure domain invariance at the image level. Then, we input the target domain into the source domain segmentation model to obtain pseudo-labels and introduce cross-domain self-supervised learning between the two segmentation models. Here, a new loss function is designed to ensure the accuracy of the pseudo-labels. In addition, a cross-domain consistency loss is also introduced. Finally, we construct a multi-level aggregation segmentation network to obtain more refined target domain information. We validate our method on the public whole heart image segmentation challenge dataset and obtain experimental results of 82.9% and 5.5 on dice and average symmetric surface distance (ASSD), respectively. These experimental results prove that our method can provide important assistance in the clinical evaluation of unannotated cardiac datasets.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000745829300012</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>34757915</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Image segmentation; Computed tomography; Heart; Training; Biomedical imaging; Magnetic resonance imaging; Adversarial machine learning; Unsupervised domain adaptation; multi-modality cardiac image; self-supervised learning</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
SYNERGISTIC IMAGE</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Du, Xiuquan] Anhui Univ, Key Lab Intelligent Comp &amp; Signal Proc, Minist Educ, Hefei 240601, Peoples R China. <br>
[Du, Xiuquan; Liu, Yueguo] Anhui Univ, Sch Comp Sci &amp; Technol, Hefei 240601, Peoples R China. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Du, XQ (corresponding author), Anhui Univ, Key Lab Intelligent Comp &amp; Signal Proc, Minist Educ, Hefei 240601, Peoples R China.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
dxqllp@163.com; 2567470422@qq.com</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Anhui University; Anhui University</td>
</tr>

<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Information Systems; Computer Science, Interdisciplinary Applications; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>YL3YF</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2168-2194</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>2168-2208</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE J BIOMED HEALTH</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE J. Biomed. Health Inform.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>12</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Natural Science Research project of Anhui province</grant_agency>&nbsp;</td><td>
<div>KJ2020A0035&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported in part by Natural Science Research project of Anhui province under Grant KJ2020A0035.</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 19 of 38</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Unsupervised deep learning of electrocardiograms enables scalable human disease profiling</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Friedman, SF (Friedman, Sam F.); Khurshid, S (Khurshid, Shaan); Venn, RA (Venn, Rachael A.); Wang, X (Wang, Xin); Diamant, N (Diamant, Nate); Di Achille, P (Di Achille, Paolo); Weng, LC (Weng, Lu-Chen); Choi, SH (Choi, Seung Hoan); Reeder, C (Reeder, Christopher); Pirruccello, JP (Pirruccello, James P.); Singh, P (Singh, Pulkit); Lau, ES (Lau, Emily S.); Philippakis, A (Philippakis, Anthony); Anderson, CD (Anderson, Christopher D.); Maddah, M (Maddah, Mahnaz); Batra, P (Batra, Puneet); Ellinor, PT (Ellinor, Patrick T.); Ho, JE (Ho, Jennifer E.); Lubitz, SA (Lubitz, Steven A.)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>NPJ DIGITAL MEDICINE</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>8</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>1</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>23</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1038/s41746-024-01418-9</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2025 JAN 12</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>0</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>0</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
9</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
10</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>61</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
The 12-lead electrocardiogram (ECG) is inexpensive and widely available. Whether conditions across the human disease landscape can be detected using the ECG is unclear. We developed a deep learning denoising autoencoder and systematically evaluated associations between ECG encodings and similar to 1,600 Phecode-based diseases in three datasets separate from model development, and meta-analyzed the results. The latent space ECG model identified associations with 645 prevalent and 606 incident Phecodes. Associations were most enriched in the circulatory (n = 140, 82% of category-specific Phecodes), respiratory (n = 53, 62%) and endocrine/metabolic (n = 73, 45%) categories, with additional associations across the phenome. The strongest ECG association was with hypertension (p &lt; 2.2x10(-308)). The ECG latent space model demonstrated more associations than models using standard ECG intervals, and offered favorable discrimination of prevalent disease compared to models comprising age, sex, and race. We further demonstrate how latent space models can be used to generate disease-specific ECG waveforms and facilitate individual disease profiling.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001396247800001</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>39799251</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
ATRIAL-FIBRILLATION; HEART-FAILURE; DURATION; ECG</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Friedman, Sam F.; Diamant, Nate; Di Achille, Paolo; Reeder, Christopher; Singh, Pulkit; Maddah, Mahnaz; Batra, Puneet] Broad Inst MIT &amp; Harvard, Data Sci Platform, Cambridge, MA USA. <br>
[Khurshid, Shaan; Venn, Rachael A.; Wang, Xin; Weng, Lu-Chen; Lau, Emily S.; Ellinor, Patrick T.; Lubitz, Steven A.] Massachusetts Gen Hosp, Cardiovasc Res Ctr, Boston, MA 02114 USA. <br>
[Khurshid, Shaan; Venn, Rachael A.; Wang, Xin; Weng, Lu-Chen; Choi, Seung Hoan; Pirruccello, James P.; Lau, Emily S.; Ellinor, Patrick T.; Ho, Jennifer E.; Lubitz, Steven A.] Broad Inst MIT &amp; Harvard, Cardiovasc Dis Initiat, Cambridge, MA 02142 USA. <br>
[Khurshid, Shaan; Venn, Rachael A.; Ellinor, Patrick T.; Lubitz, Steven A.] Massachusetts Gen Hosp, Canc Ctr, Boston, MA 02114 USA. <br>
[Pirruccello, James P.] Univ Calif San Francisco, Inst Human Genet, San Francisco, CA USA. <br>
[Pirruccello, James P.] Univ Calif San Francisco, Div Cardiol, San Francisco, CA USA. <br>
[Lau, Emily S.] Massachusetts Gen Hosp, Div Cardiol, Boston, MA USA. <br>
[Philippakis, Anthony] Google Ventures, Cambridge, MA USA. <br>
[Anderson, Christopher D.] Brigham &amp; Womens Hosp, Dept Neurol, Boston, MA USA. <br>
[Anderson, Christopher D.] Massachusetts Gen Hosp, Ctr Genom Med, Boston, MA USA. <br>
[Anderson, Christopher D.] Massachusetts Gen Hosp, Henry &amp; Allison McCance Ctr Brain Hlth, Boston, MA USA. <br>
[Ho, Jennifer E.] Beth Israel Deaconess Med Ctr, Cardiovasc Inst, Boston, MA USA. <br>
[Ho, Jennifer E.] Beth Israel Deaconess Med Ctr, Dept Med, Div Cardiol, Boston, MA USA. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Lubitz, SA (corresponding author), Massachusetts Gen Hosp, Cardiovasc Res Ctr, Boston, MA 02114 USA.<br>Lubitz, SA (corresponding author), Broad Inst MIT &amp; Harvard, Cardiovasc Dis Initiat, Cambridge, MA 02142 USA.<br>Lubitz, SA (corresponding author), Massachusetts Gen Hosp, Canc Ctr, Boston, MA 02114 USA.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
slubitz@mgh.harvard.edu</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Harvard University; Massachusetts Institute of Technology (MIT); Broad Institute; Harvard University; Harvard University Medical Affiliates; Massachusetts General Hospital; Harvard University; Massachusetts Institute of Technology (MIT); Broad Institute; Harvard University; Harvard University Medical Affiliates; Massachusetts General Hospital; University of California System; University of California San Francisco; University of California System; University of California San Francisco; Harvard University; Harvard University Medical Affiliates; Massachusetts General Hospital; Alphabet Inc.; Google Incorporated; Harvard University; Harvard University Medical Affiliates; Brigham &amp; Women's Hospital; Harvard University; Harvard University Medical Affiliates; Massachusetts General Hospital; Harvard University; Harvard University Medical Affiliates; Massachusetts General Hospital; Harvard University; Harvard University Medical Affiliates; Beth Israel Deaconess Medical Center; Harvard University; Harvard University Medical Affiliates; Beth Israel Deaconess Medical Center</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Di Achille, Paolo</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-9256-0678&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Friedman, Sam Freesun</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-0688-2169&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Pirruccello, James</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-6088-4037&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Ellinor, Patrick T.</display_name>&nbsp;</td><td>OHR-7571-2025&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Reeder, Christopher</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-3893-2423&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Khurshid, Shaan</display_name>&nbsp;</td><td>R-7984-2017&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>NATURE PORTFOLIO</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>HEIDELBERGER PLATZ 3, BERLIN, 14197, GERMANY</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Health Care Sciences &amp; Services; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Health Care Sciences &amp; Services; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>S1X8H</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2398-6352</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>NPJ DIGIT MED</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>npj Digit. Med.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>13</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>U.S. Department of Health &amp; Human Services | NIH | National Heart, Lung, and Blood Institute (NHLBI)</grant_agency>&nbsp;</td><td>
<div>K23HL169839&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>NIH</grant_agency>&nbsp;</td><td>
<div>23CDA1050571&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>American Heart Association</grant_agency>&nbsp;</td><td>
<div>1R01HL139731&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Institutes of Health (NIH)</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>NHLBI BioData Catalyst Fellows program</grant_agency>&nbsp;</td><td>
<div>18SFRN34110082&nbsp;</div>
<div>965286&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>AHA</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Massachusetts Life Sciences Center</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>American Heart Association (AHA)</grant_agency>&nbsp;</td><td>
<div>23CDA1050571&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>Dr. Lubitz previously received support from NIH grants R01HL139731 and R01HL157635, and American Heart Association 18SFRN34250007. Dr. Anderson is supported by NIH grants R01NS103924 and U01NS069763 and American Heart Association grants 18SFRN34250007 and 21SFRN812095. Dr. Weng is supported by National Institutes of Health (NIH) grant 1R01HL139731. Dr. Choi is supported by the NHLBI BioData Catalyst Fellows program. Dr. Ellinor is supported by the NIH (1R01HL092577, K24HL105780), AHA (18SFRN34110082) and by MAESTRIA (965286). Dr. Lau is supported by the NIH (K23HL159243), the American Heart Association (853922), and the Massachusetts Life Sciences Center. Dr. Pirruccello is supported by the NIH (K08HL159346). Dr. Ho is supported by NIH grants R01HL134893, R01HL140224, R01HL160003, and K24HL153669. Dr. Khurshid is supported by NIH grant K23HL169839 and American Heart Association (23CDA1050571).</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>gold, Green Submitted</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 20 of 38</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Unsupervised detection and classification of heartbeats using the dissimilarity matrix in PCG signals</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Torre-Cruz, J (Torre-Cruz, J.); Martinez-Munoz, D (Martinez-Munoz, D.); Ruiz-Reyes, N (Ruiz-Reyes, N.); Munoz-Montoro, AJ (Munoz-Montoro, A. J.); Puentes-Chiachio, M (Puentes-Chiachio, M.); Canadas-Quesada, FJ (Canadas-Quesada, F. J.)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>COMPUTER METHODS AND PROGRAMS IN BIOMEDICINE</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>221</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>106909</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1016/j.cmpb.2022.106909</value>&nbsp;&nbsp;<b>Early Access Date:</b> 
<value>MAY 2022</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2022 JUN</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>11</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>12</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
2</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
26</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>60</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Background and objective : Auscultation is the first technique applied to the early diagnose of any cardiovascular disease (CVD) in rural areas and poor-resources countries because of its low cost and noninvasiveness. However, it highly depends on the physician's expertise to recognize specific heart sounds heard through the stethoscope. The analysis of phonocardiogram (PCG) signals attempts to segment each cardiac cycle into the four cardiac states (S1, systole, S2 and diastole) in order to develop automatic systems applied to an efficient and reliable detection and classification of heartbeats. In this work, we propose an unsupervised approach, based on time-frequency characteristics shown by cardiac sounds, to detect and classify heartbeats S1 and S2.Methods : The proposed system consists of a two-stage cascade. The first stage performs a rough heartbeat detection while the second stage refines the previous one, improving the temporal localization and also classifying the heartbeats into types S1 and S2. The first contribution is a novel approach that combines the dissimilarity matrix with the frame-level spectral divergence to locate heartbeats using the repetitiveness shown by the heart sounds and the temporal relationships between the intervals defined by the events S1/S2 and non-S1/S2 (systole and diastole). The second contribution is a verification-correction classification process based on a sliding window that allows the preservation of the temporal structure of the cardiac cycle in order to be applied in the heart sound classification. The proposed method has been assessed using the open access databases PASCAL, CirCor DigiScope Phonocardiogram and an additional sound mixing procedure considering both Additive White Gaussian Noise (AWGN) and different kinds of clinical ambient noises from a commercial database.Results : The proposed method outperforms the detection and classification performance of other recent state-of-the-art methods. Although our proposal achieves the best average accuracy for PCG signals without cardiac abnormalities, 99.4% in heartbeat detection and 97.2% in heartbeat classification, its worst average accuracy is always above 92% for PCG signals with cardiac abnormalities, signifying an improvement in heartbeat detection/classification above 10% compared to the other state-of-the-art methods evaluated.Conclusions : The proposed method provides the best detection/classification performance in realistic scenarios where the presence of cardiac anomalies as well as different types of clinical environmental noises are active in the PCG signal. Of note, the promising modelling of the temporal structures of the heart provided by the dissimilarity matrix together with the frame-level spectral divergence, as well as the removal of a significant number of spurious heart events and recovery of missing heart events, both corrected by the proposed verification-correction-classification algorithm, suggest that our proposal is a successful tool to be applied in heart segmentation. (c) 2022 Elsevier B.V. All rights reserved.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000807771600008</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>35649297</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Phonocardiography; Unsupervised; Detection; classification; Heartbeat S1; S2; Dissimilarity; Divergence</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
SOUND SEGMENTATION; IDENTIFICATION; EXTRACTION; TRANSFORM; SYSTEM</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Torre-Cruz, J.; Martinez-Munoz, D.; Ruiz-Reyes, N.; Canadas-Quesada, F. J.] Univ Jaen, Dept Telecommun Engn, Campus Cientif Tecnol Linares,Avda Univ, s-n, Jaen 23700, Spain. <br>
[Munoz-Montoro, A. J.] Univ Oviedo, Dept Comp Sci, Campus Gijon,s-n, Gijon 33203, Spain. <br>
[Puentes-Chiachio, M.] Univ Hosp Jaen, Cardiol, Ave Ejercito Espanol 10, Jaen 23007, Spain. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Torre-Cruz, J (corresponding author), Univ Jaen, Dept Telecommun Engn, Campus Cientif Tecnol Linares,Avda Univ, s-n, Jaen 23700, Spain.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
jtorre@ujaen.es</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Universidad de Jaen; University of Oviedo</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Mu&ntilde;oz-Montoro, Antonio Jes&uacute;s</display_name>&nbsp;</td><td>U-1635-2019&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Reyes, Nicol&aacute;s</display_name>&nbsp;</td><td>R-5878-2018&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Ca&ntilde;adas-Quesada, Francisco</display_name>&nbsp;</td><td>H-8839-2015&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Martinez-Mu&ntilde;oz, Damian</display_name>&nbsp;</td><td>H-7649-2015&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>DE LA TORRE CRUZ, JUAN</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-6291-4698&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Canadas Quesada, Francisco Jesus</display_name>&nbsp;</td><td>H-8839-2015&nbsp;</td><td>0000-0002-3873-6078&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Martinez-Munoz, Damian</display_name>&nbsp;</td><td>H-7649-2015&nbsp;</td><td>0000-0002-0953-5947&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Munoz-Montoro, Antonio Jesus</display_name>&nbsp;</td><td>U-1635-2019&nbsp;</td><td>0000-0001-9518-8955&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>ELSEVIER IRELAND LTD</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>ELSEVIER HOUSE, BROOKVALE PLAZA, EAST PARK SHANNON, CO, CLARE, 00000, IRELAND</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Interdisciplinary Applications; Computer Science, Theory &amp; Methods; Engineering, Biomedical; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Engineering; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>1X9MA</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0169-2607</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1872-7565</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>COMPUT METH PROG BIO</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Comput. Meth. Programs Biomed.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>20</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Programa Operativo FEDER Andalucia 2014-2020</grant_agency>&nbsp;</td><td>
<div>1257914&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Ministry of Economy, Knowledge and University, Junta de Andalucia</grant_agency>&nbsp;</td><td>
<div>P18-RT-1994&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Spanish Ministry of Science and Innovation</grant_agency>&nbsp;</td><td>
<div>PID2020-119082RB-C21&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported by the Programa Operativo FEDER Andalucia 2014-2020 under project with reference 1257914, the Ministry of Economy, Knowledge and University, Junta de Andalucia under Project P18-RT-1994 and the Spanish Ministry of Science and Innovation under Project PID2020-119082RB-C21.</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>Green Submitted</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 21 of 38</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Automated ECG Noise Detection and Classification System for Unsupervised Healthcare Monitoring</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Satija, U (Satija, Udit); Ramkumar, B (Ramkumar, Barathram); Manikandan, MS (Manikandan, M. Sabarimalai)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE JOURNAL OF BIOMEDICAL AND HEALTH INFORMATICS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>22</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>3</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>722-732</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/JBHI.2017.2686436</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2018 MAY</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>113</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>130</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
4</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
62</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>39</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Objective: Automatic detection and classification of noises can play a vital role in the development of robust unsupervised electrocardiogram (ECG) analysis systems. This paper proposes a novel unified framework for automatic detection, localization, and classification of single and combined ECG noises. Methods: The proposed framework consists of the modified ensemble empirical mode decomposition (CEEMD), the short-term temporal feature extraction, and the decision-rule-based noise detection and classification. In the proposed framework, ECG signals are first decomposed using the modified CEEMD algorithm for discriminating the ECG components from the noises and artifacts. Then, the short-term temporal features such as maximum absolute amplitude, number of zerocrossings, and local maximum peak amplitude of the autocorelation function are computed from the extracted high-frequency and low-frequency signals. Finally, a decision rule-based algorithm is presented for detecting the presence of noises and classifying the processed ECG signals into six signal groups: noise-free ECG, ECG+BW, ECG+MA, ECG+PLI, ECG+BW+PLI, and ECG+BW+MA. Results: The proposed framework is rigorously evaluated on five benchmark ECG databases and the real-time ECG signals. The proposed framework achieves an average sensitivity of 99.12%, specificity of 98.56%, and overall accuracy of 98.90% in detecting the presence of noises. Classification results show that the framework achieves an average sensitivity, positive predictivity, and classification accuracy of 98.93%, 98.39%, and 97.38%, respectively. Conclusion: The proposed framework not only achieves better noise detection and classification rates than the current state-of-the-art methods but also accurately localizes short bursts of noises with low end-point delineation errors. Significance: Extensive studies on benchmark databases demonstrate that the proposed framework is more suitable for reducing false alarm rates and selecting appropriate noise-specific denoising techniques in automated ECG analysis applications.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000431374500010</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>28333651</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Baseline wander; ECG denoising; ECG noise detection and classification; ECG signal quality assessment; muscle artifacts</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
SIGNAL-QUALITY INDEXES; REAL-TIME; ELECTROCARDIOGRAM SIGNAL; DATA FUSION; UNIT</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Satija, Udit; Ramkumar, Barathram; Manikandan, M. Sabarimalai] Indian Inst Technol Bhubaneswar, Sch Elect Sci, Bhubaneswar 751013, India. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Satija, U (corresponding author), Indian Inst Technol Bhubaneswar, Sch Elect Sci, Bhubaneswar 751013, India.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
us11@iitbbs.ac.in; barathram@iitbbs.ac.in; msm@iitbbs.ac.in</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Indian Institute of Technology System (IIT System); Indian Institute of Technology (IIT) - Bhubaneswar</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Manikandan, M Sabarimalai</display_name>&nbsp;</td><td>V-6236-2019&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Manikandan, M Sabarimalai</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-6878-4911&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>SATIJA, UDIT</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-3851-765X&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>SATIJA, UDIT</display_name>&nbsp;</td><td>AAA-5332-2021&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Information Systems; Computer Science, Interdisciplinary Applications; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>GE6XN</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2168-2194</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE J BIOMED HEALTH</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE J. Biomed. Health Inform.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>11</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 22 of 38</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>ST-GAN: A Swin Transformer-Based Generative Adversarial Network for Unsupervised Domain Adaptation of Cross-Modality Cardiac Segmentation</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Zhang, YF (Zhang, Yifan); Wang, YH (Wang, Yonghui); Xu, LS (Xu, Lisheng); Yao, YD (Yao, Yudong); Qian, W (Qian, Wei); Qi, L (Qi, Lin)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE JOURNAL OF BIOMEDICAL AND HEALTH INFORMATICS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>28</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>2</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>893-904</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/JBHI.2023.3336965</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2024 FEB</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>3</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>3</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
4</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
47</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>50</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Unsupervised domain adaptation (UDA) methods have shown great potential in cross-modality medical image segmentation tasks, where target domain labels are unavailable. However, the domain shift among different image modalities remains challenging, because the conventional UDA methods are based on convolutional neural networks (CNNs), which tend to focus on the texture of images and cannot establish the global semantic relevance of features due to the locality of CNNs. This paper proposes a novel end-to-end Swin Transformer-based generative adversarial network (ST-GAN) for cross-modality cardiac segmentation. In the generator of ST-GAN, we utilize the local receptive fields of CNNs to capture spatial information and introduce the Swin Transformer to extract global semantic information, which enables the generator to better extract the domain-invariant features in UDA tasks. In addition, we design a multi-scale feature fuser to sufficiently fuse the features acquired at different stages and improve the robustness of the UDA network. We extensively evaluated our method with two cross-modality cardiac segmentation tasks on the MS-CMR 2019 dataset and the M&amp;Ms dataset. The results of two different tasks show the validity of ST-GAN compared with the state-of-the-art cross-modality cardiac image segmentation methods.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001203362500022</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>38019618</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Adversarial learning; cross-modality cardiac segmentation; multi-scale fusion; swin transformer; unsupervised domain adaptation</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
IMAGE</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Zhang, Yifan; Wang, Yonghui; Xu, Lisheng; Qian, Wei; Qi, Lin] Northeastern Univ, Coll Med &amp; Biol Informat Engn, Shenyang 110004, Peoples R China. <br>
[Xu, Lisheng; Qi, Lin] Minist Educ, Engn Res Ctr Med Imaging &amp; Intelligent Anal, Shenyang 110169, Peoples R China. <br>
[Xu, Lisheng; Qi, Lin] Northeastern Univ, Key Lab Med Image Comp, Minist Educ, Shenyang 110169, Peoples R China. <br>
[Yao, Yudong] Stevens Inst Technol, Elect &amp; Comp Engn Dept, Hoboken, NJ 07030 USA. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Qi, L (corresponding author), Northeastern Univ, Coll Med &amp; Biol Informat Engn, Shenyang 110004, Peoples R China.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
zyf0lzh@gmail.com; wang18852865993@gmail.com; xuls@bmie.neu.edu.cn; Yu-Dong.Yao@stevens.edu; wqian@bmie.neu.edu.cn; qilin@bmie.neu.edu.cn</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Northeastern University - China; Northeastern University - China; Stevens Institute of Technology</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>XU, Lisheng</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-8360-3605&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Zhang, Yifan</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-0141-1536&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>XU, Lisheng</display_name>&nbsp;</td><td>C-2974-2008&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Information Systems; Computer Science, Interdisciplinary Applications; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>NW0A0</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2168-2194</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>2168-2208</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE J BIOMED HEALTH</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE J. Biomed. Health Inform.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>12</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Key Research and Development Program of China</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>No Statement Available</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 23 of 38</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>I-Vector-Based Patient Adaptation of Deep Neural Networks for Automatic Heartbeat Classification</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Xu, SS (Xu, Sean Shensheng); Mak, MW (Mak, Man-Wai); Cheung, CC (Cheung, Chi-Chung)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE JOURNAL OF BIOMEDICAL AND HEALTH INFORMATICS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>24</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>3</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>717-727</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/JBHI.2019.2919732</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2020 MAR</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>22</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>24</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
1</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
33</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>36</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Automatic classification of electrocardiogram (ECG) signals is important for diagnosing heart arrhythmias. A big challenge in automatic ECG classification is the variation in the waveforms and characteristics of ECG signals among different patients. To address this issue, this paper proposes adapting a patient-independent deep neural network (DNN) using the information in the patient-dependent identity vectors (i-vectors). The adapted networks, namely i-vector adapted patient-specific DNNs (iAP-DNNs), are tuned toward the ECG characteristics of individual patients. For each patient, his/her ECG waveforms are compressed into an i-vector using a factor analysis model. Then, this i-vector is injected into the middle hidden layer of the patient-independent DNN. Stochastic gradient descent is then applied to fine-tune the whole network to form a patient-specific classifier. As a result, the adaptation makes use of not only the raw ECG waveforms from the specific patient but also the compact representation of his/her ECG characteristics through the i-vector. Analysis on the hidden-layer activations shows that by leveraging the information in the i-vectors, the iAP-DNNs are more capable of discriminating normal heartbeats against arrhythmic heartbeats than the networks that use the patient-specific ECG only for the adaptation. Experimental results based on the MIT-BIH database suggest that the iAP-DNNs perform better than existing patient-specific classifiers in terms of various performance measures. In particular, the sensitivity and specificity of the existing methods are all under the receiver operating characteristic curves of the iAP-DNNs.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000519730300008</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>31150349</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Electrocardiography; Statistics; Training; Sociology; Heart rate variability; Feature extraction; Heart beat; ECG classification; Arrhythmias; deep neural networks; i-vectors; DNN adaptation</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
ECG CLASSIFICATION</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Xu, Sean Shensheng; Mak, Man-Wai; Cheung, Chi-Chung] Hong Kong Polytech Univ, Dept Elect &amp; Informat Engn, Hong Kong, Peoples R China. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Mak, MW (corresponding author), Hong Kong Polytech Univ, Dept Elect &amp; Informat Engn, Hong Kong, Peoples R China.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
seanssx@yahoo.com; enmwmak@polyu.edu.hk; lawrence.chi-chung.cheung@polyu.edu.hk</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Hong Kong Polytechnic University</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Mak, Man-Wai</display_name>&nbsp;</td><td>C-3750-2014&nbsp;</td><td>0000-0001-8854-3760&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Mak, Manwai</display_name>&nbsp;</td><td>C-3750-2014&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>XU, SHENSHENG</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-8846-5061&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Information Systems; Computer Science, Interdisciplinary Applications; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>KU5DO</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2168-2194</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>2168-2208</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE J BIOMED HEALTH</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE J. Biomed. Health Inform.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>11</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>RGC of Hong Kong</grant_agency>&nbsp;</td><td>
<div>PolyU152137/17E&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported by the RGC of Hong Kong, under Grant PolyU152137/17E.</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>Green Accepted</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 24 of 38</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>sCL-ST: Supervised Contrastive Learning With Semantic Transformations for Multiple Lead ECG Arrhythmia Classification</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Le, DC (Le, Duc); Truong, S (Truong, Sang); Brijesh, P (Brijesh, Patel); Adjeroh, DA (Adjeroh, Donald A.); Le, N (Le, Ngan)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE JOURNAL OF BIOMEDICAL AND HEALTH INFORMATICS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>27</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>6</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>2818-2828</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/JBHI.2023.3246241</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2023 JUN</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>16</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>20</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
1</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
41</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>53</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
The automatic classification of electrocardiogram (ECG) signals has played an important role in cardiovascular diseases diagnosis and prediction. With recent advancements in deep neural networks (DNNs), particularly Convolutional Neural Networks (CNNs), learning deep features automatically from the original data is becoming an effective and widespread approach in a variety of intelligent tasks including biomedical and health informatics. However, most of the existing approaches are trained on either 1D CNNs or 2D CNNs, and they suffer from the limitations of random phenomena (i.e. random initial weights). Furthermore, the ability to train such DNNs in a supervised manner in healthcare is often limited due to the scarcity of labeled training data. To address the problems of weight initialization and limited annotated data, in this work, we leverage recent self-supervised learning technique, namely, contrastive learning, and present supervised contrastive learning (sCL). Different from existing self-supervised contrastive learning approaches, which often generate false negatives because of random selection of negative anchors, our contrastive learning makes use of labeled data to pull the same class closer together and push different classes far apart to avoid potential false negatives. Furthermore, unlike other kinds of signals (e.g. speech, image, video), ECG signal is sensitive to changes, and inappropriate transformation could directly affect diagnosis results. To deal with this issue, we present two semantic transformations, i.e. semantic split-join and semantic weighted peaks noise smoothing. The proposed deep neural network sCL-ST with supervised contrastive learning and semantic transformations is trained as an end-to-end framework for the multi-label classification of 12-lead ECGs. Our sCL-ST network contains two sub-networks i.e. pre-text task and down-stream task. Our experimental results have been evaluated on 12-lead PhysioNet 2020 dataset and shown that our proposed network outperforms the state-of-the-art existing approaches.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001004541400023</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>37028019</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
ECG; arrhythmia classification; multiple lead; contrastive learning; self-supervised learning</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
DEEP; ALGORITHM</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Le, Duc; Truong, Sang; Le, Ngan] Univ Arkansas, Dept Comp Sci &amp; Comp Engn, Fayetteville, AR 72703 USA. <br>
[Brijesh, Patel] West Virginia Univ, Sch Med, Div Cardiol, Morgantown, WV 26506 USA. <br>
[Adjeroh, Donald A.] West Virginia Univ, Dept Comp Sci &amp; Elect Engn LCSEE, Morgantown, WV 26506 USA. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Le, N (corresponding author), Univ Arkansas, Dept Comp Sci &amp; Comp Engn, Fayetteville, AR 72703 USA.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
minhducl@uark.edu; sangt@uark.edu; brijesh.patel@wvumedicine.org; Donald.Adjeroh@mail.wvu.edu; thile@uark.edu</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
University of Arkansas System; University of Arkansas Fayetteville; West Virginia University; West Virginia University</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Le, Duc</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-7771-7729&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Le, Ngan</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-2571-0511&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Information Systems; Computer Science, Interdisciplinary Applications; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>I7KP3</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2168-2194</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>2168-2208</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE J BIOMED HEALTH</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE J. Biomed. Health Inform.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>11</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>U.S. National Science Foundation</grant_agency>&nbsp;</td><td>
<div>OIA-1946391&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>NSF</grant_agency>&nbsp;</td><td>
<div>1920920&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported in part by U.S. National Science Foundation under Award OIA-1946391 and in part by NSF under Grant 1920920.</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 25 of 38</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Scaling Representation Learning From Ubiquitous ECG With State-Space Models</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Avramidis, K (Avramidis, Kleanthis); Kunc, D (Kunc, Dominika); Perz, B (Perz, Bartosz); Adsul, K (Adsul, Kranti); Feng, TT (Feng, Tiantian); Kazienko, P (Kazienko, Przemyslaw); Saganowski, S (Saganowski, Stanislaw); Narayanan, S (Narayanan, Shrikanth)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE JOURNAL OF BIOMEDICAL AND HEALTH INFORMATICS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>28</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>10</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>5877-5889</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/JBHI.2024.3416897</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2024 OCT</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>4</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>4</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
8</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
18</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>82</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Ubiquitous sensing from wearable devices in the wild holds promise for enhancing human well-being, from diagnosing clinical conditions and measuring stress to building adaptive health promoting scaffolds. But the large volumes of data therein across heterogeneous contexts pose challenges for conventional supervised learning approaches. Representation Learning from biological signals is an emerging realm catalyzed by the recent advances in computational modeling and the abundance of publicly shared databases. The electrocardiogram (ECG) is the primary researched modality in this context, with applications in health monitoring, stress and affect estimation. Yet, most studies are limited by small-scale controlled data collection and over-parameterized architecture choices. We introduce &lt;bold&gt;WildECG&lt;/bold&gt;, a pre-trained state-space model for representation learning from ECG signals. We train this model in a self-supervised manner with 275 000 10 s ECG recordings collected in the wild and evaluate it on a range of downstream tasks. The proposed model is a robust backbone for ECG analysis, providing competitive performance on most of the tasks considered, while demonstrating efficacy in low-resource regimes.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001329782300043</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>38935470</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Electrocardiography; Task analysis; Biological system modeling; Data models; Bioinformatics; Training; State-space methods; ubiquitous computing; self-supervised learning; state-space models</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
ARTIFICIAL-INTELLIGENCE; ELECTROCARDIOGRAM; CLASSIFICATION; EMOTION</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Avramidis, Kleanthis; Adsul, Kranti; Feng, Tiantian; Narayanan, Shrikanth] Univ Southern Calif, Viterbi Sch Engn, Los Angeles, CA 90089 USA. <br>
[Kunc, Dominika; Perz, Bartosz; Kazienko, Przemyslaw; Saganowski, Stanislaw] Wroclaw Univ Sci &amp; Technol, Dept Artificial Intelligence, PL-50370 Wroclaw, Poland. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Avramidis, K (corresponding author), Univ Southern Calif, Viterbi Sch Engn, Los Angeles, CA 90089 USA.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
avramidi@usc.edu; dominika.kunc@pwr.edu.pl; bartosz.perz@pwr.edu.pl; kadsul@usc.edu; tiantiaf@usc.edu; kazienko@pwr.edu.pl; stanislaw.saganowski@pwr.edu.pl; shri@ee.usc.edu</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
University of Southern California; Wroclaw University of Science &amp; Technology</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Perz, Bartosz</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-9940-5117&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Kazienko, Przemysław</display_name>&nbsp;</td><td>F-1849-2014&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Kazienko, Przemyslaw</display_name>&nbsp;</td><td>F-1849-2014&nbsp;</td><td>0000-0001-5868-356X&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Kunc, Dominika</display_name>&nbsp;</td><td>LIH-0162-2024&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Kunc, Dominika</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-2918-0423&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Perz, Bartosz</display_name>&nbsp;</td><td>AHE-2470-2022&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Feng, Tiantian</display_name>&nbsp;</td><td>X-4695-2019&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Saganowski, Stanisław</display_name>&nbsp;</td><td>AAF-1696-2019&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Information Systems; Computer Science, Interdisciplinary Applications; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>I4D7I</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2168-2194</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>2168-2208</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE J BIOMED HEALTH</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE J. Biomed. Health Inform.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>13</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>NSF SCH</grant_agency>&nbsp;</td><td>
<div>2204942&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>DARPA</grant_agency>&nbsp;</td><td>
<div>N660012324006&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Toyota</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Science Centre, Poland</grant_agency>&nbsp;</td><td>
<div>2020/37/B/ST6/03806&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>NAWA STER Programme Internationalisation of Wroclaw University of Science and Technology Doctoral School</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Polish National Agency for Academic Exchange (NAWA) - the Bekker Programme</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was conducted at USC SAIL and supported in part by NSF SCH under Grant 2204942, in part by DARPA under Cooperative Agreement number N660012324006, in part by Toyota, in part by National Science Centre, Poland under Project 2020/37/B/ST6/03806,in part by the NAWA STER Programme Internationalisation of Wroclaw University of Science and Technology Doctoral School, and in part by the Polish National Agency for Academic Exchange (NAWA) - the BekkerProgramme.</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 26 of 38</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Heartbeats Classification Using Hybrid Time-Frequency Analysis and Transfer Learning Based on ResNet</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Zhang, YT (Zhang, Yatao); Li, JY (Li, Junyan); Wei, SS (Wei, Shoushui); Zhou, FY (Zhou, Fengyu); Li, D (Li, Dong)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE JOURNAL OF BIOMEDICAL AND HEALTH INFORMATICS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>25</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>11</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>4175-4184</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/JBHI.2021.3085318</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2021 NOV</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>56</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>66</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
1</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
90</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>33</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
The classification of heartbeats is an important method for cardiac arrhythmia analysis. This study proposes a novel heartbeat classification method using hybrid time-frequency analysis and transfer learning based on ResNet-101. The proposed method has the following major advantages over the afore-mentioned methods: it avoids the need for manual features extraction in the traditional machine learning method, and it utilizes 2-D time-frequency diagrams which provide not only frequency and energy information but also preserve the morphological characteristic within the ECG recordings, and it owns enough deep to make better use of performance of CNN. The method deploys a hybrid time-frequency analysis of the Hilbert transform (HT) and the Wigner-Ville distribution (WVD) to transform 1-D ECG recordings into 2-D time-frequency diagrams which were then fed into a transfer learning classifier based on ResNet-101 for two classification tasks (i.e., 5 heartbeat categories assigned by the ANSI/AAMI standard (i.e., N, V, S, Q and F) and 14 original beat kinds of the MIT/BIH arrhythmia database). For 5 heartbeat categories classification, the results show the F1-score of N, V, S, Q and F categories are F-N 0.9899, F-V 0.9845, F-S 0.9376, F-Q 0.9968, F-F 0.8889, respectively, and the overall F1-score is 0.9595 using the combination data balancing. The results show the average values for accuracy, sensitivity, specificity, predictive value and F1-score on test set for 14 beat kinds the MIT-BIH arrhythmia database are 99.75%, 91.36%, 99.85%, 90.81% and 0.9016, respectively. Compared with other methods, the proposed method can yield more accurate results.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000714714200016</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>34077377</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Heartbeat classification; arrhythmias; time-frequency analysis; Transfer learning; ResNet-101</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
ATRIAL-FIBRILLATION; NEURAL-NETWORK; ECG</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Zhang, Yatao; Li, Dong] Shandong Univ, Sch Mech Elect &amp; Informat Engn, Weihai 264209, Peoples R China. <br>
[Li, Junyan] Weihaiwei Peoples Hosp, Dept Neurol, Weihai 264209, Peoples R China. <br>
[Wei, Shoushui; Zhou, Fengyu] Shandong Univ, Sch Control Sci &amp; Engn, Jinan 17923, Peoples R China. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Zhang, YT (corresponding author), Shandong Univ, Sch Mech Elect &amp; Informat Engn, Weihai 264209, Peoples R China.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
zytboy@sdu.edu.cn; whkunyushan@163.com; sswei@sdu.edu.cn; zhoufengyu@sdu.edu.cn; dongli@sdu.edu.cn</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Shandong University; Shandong University</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Jiang, Bin</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-6621-3979&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>zhang, yatao</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-6152-0806&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Zhou, Fengyu</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-5140-7036&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Information Systems; Computer Science, Interdisciplinary Applications; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>WR7ZD</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2168-2194</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>2168-2208</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE J BIOMED HEALTH</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE J. Biomed. Health Inform.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>10</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Natural Science Foundation of China</grant_agency>&nbsp;</td><td>
<div>82072014&nbsp;</div>
<div>62076149&nbsp;</div>
<div>61702138&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>China Postdoctoral Science Foundation</grant_agency>&nbsp;</td><td>
<div>2017M612280&nbsp;</div>
<div>2020T130368&nbsp;</div>
<div>2019M662360&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Intergovernmental Project of National Key Research, and Development Program/Hong Kong, Macao, and Taiwan key projects</grant_agency>&nbsp;</td><td>
<div>SQ2019YFE010670&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Key R&amp;D project of Shandong Province</grant_agency>&nbsp;</td><td>
<div>2018GSF118133&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Young Scholars Program of Shandong University, Weihai</grant_agency>&nbsp;</td><td>
<div>1050501318006&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Science and Technology Development Plan of Weihai City</grant_agency>&nbsp;</td><td>
<div>1050413421912&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported in part by the National Natural Science Foundation of China under Grants 82072014, 62076149, and 61702138, in part by the China Postdoctoral Science Foundation under Grants 2017M612280, 2020T130368, and 2019M662360, in part by the Intergovernmental Project of National Key Research, and Development Program/Hong Kong, Macao, and Taiwan key projects under Grant SQ2019YFE010670, in part by the Key R&amp;D project of Shandong Province under Grant 2018GSF118133, in part by the Young Scholars Program of Shandong University, Weihai under Grant 1050501318006, and in part by the Science and Technology Development Plan of Weihai City under Grant 1050413421912.</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 27 of 38</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Inter-Patient ECG Classification With Symbolic Representations and Multi-Perspective Convolutional Neural Networks</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Niu, JH (Niu, Jinghao); Tang, YQ (Tang, Yongqiang); Sun, ZY (Sun, Zhengya); Zhang, WS (Zhang, Wensheng)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE JOURNAL OF BIOMEDICAL AND HEALTH INFORMATICS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>24</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>5</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>1321-1332</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/JBHI.2019.2942938</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2020 MAY</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>104</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>112</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
2</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
85</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>47</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
This paper presents a novel deep learning framework for the inter-patient electrocardiogram (ECG) heartbeat classification. A symbolization approach especially designed for ECG is introduced, which can jointly represent the morphology and rhythm of the heartbeat and alleviate the influence of inter-patient variation through baseline correction. The symbolic representation of the heartbeat is used by a multi-perspective convolutional neural network (MPCNN) to learn features automatically and classify the heartbeat. We evaluate our method for the detection of the supraventricular ectopic beat (SVEB) and ventricular ectopic beat (VEB) on MIT-BIH arrhythmia dataset. Compared with the state-of-the-art methods based on manual features or deep learning models, our method shows superior performance: the overall accuracy of 96.4%, F1 scores for SVEB and VEB of 76.6% and 89.7%, respectively. The ablation study on our method validates the effectiveness of the proposed symbolization approach and joint representation architecture, which can help the deep learning model to learn more general features and improve the ability of generalization for unseen patients. Because our method achieves a competitive inter-patient heartbeat classification performance without complex handcrafted features or the intervention of the human expert, it can also be adjusted to handle various other tasks relative to ECG classification.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000535614100009</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>31545750</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Heart beat; Electrocardiography; Heart rate variability; Deep learning; Feature extraction; Task analysis; Informatics; ECG classification; biomedical monitoring; convolutional neural network; deep learning</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
HEARTBEAT CLASSIFICATION; ARRHYTHMIA DETECTION; FEATURES</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Niu, Jinghao; Tang, Yongqiang; Sun, Zhengya; Zhang, Wensheng] Chinese Acad Sci, Inst Automat, Beijing 100190, Peoples R China. <br>
[Niu, Jinghao; Tang, Yongqiang; Sun, Zhengya; Zhang, Wensheng] Univ Chinese Acad Sci, Beijing 100049, Peoples R China. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Zhang, WS (corresponding author), Chinese Acad Sci, Inst Automat, Beijing 100190, Peoples R China.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
niujinghao2015@ia.ac.cn; tangyongqiang2014@ia.ac.cn; zhengya.sun@ia.ac.cn; zhangwenshengia@hotmail.com</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Chinese Academy of Sciences; Institute of Automation, CAS; Chinese Academy of Sciences; University of Chinese Academy of Sciences, CAS</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Niu, Jinghao</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-9598-297X&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>tang, yong qiang</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-9333-8200&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Information Systems; Computer Science, Interdisciplinary Applications; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>LR3SW</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2168-2194</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>2168-2208</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE J BIOMED HEALTH</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE J. Biomed. Health Inform.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>12</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Key R&amp;D Program of China</grant_agency>&nbsp;</td><td>
<div>2017YFC0803700&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Natural Science Foundation of China</grant_agency>&nbsp;</td><td>
<div>61432008&nbsp;</div>
<div>61876183&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Beijing Municipal Natural Science Foundation</grant_agency>&nbsp;</td><td>
<div>4172063&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported in part by the National Key R&amp;D Program of China (2017YFC0803700), in part by the National Natural Science Foundation of China under Grant 61432008 and Grant 61876183, and in part by Beijing Municipal Natural Science Foundation under Grant 4172063.</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 28 of 38</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Deep Representation Learning With Sample Generation and Augmented Attention Module for Imbalanced ECG Classification</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Zubair, M (Zubair, Muhammad); Woo, S (Woo, Sungpil); Lim, S (Lim, Sunhwan); Kim, D (Kim, Daeyoung)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE JOURNAL OF BIOMEDICAL AND HEALTH INFORMATICS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>28</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>5</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>2461-2472</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/JBHI.2023.3325540</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2024 MAY</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>10</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>10</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
5</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
42</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>61</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Developing an efficient heartbeat monitoring system has become a focal point in numerous healthcare applications. Specifically, in the last few years, heartbeat classification for arrhythmia detection has gained considerable interest from researchers. This paper presents a novel deep representation learning method for the efficient detection of arrhythmic beats. To mitigate the issues associated with the imbalanced data distribution, a novel re-sampling strategy is introduced. Unlike the existing oversampling methods, the proposed technique transforms majority-class samples into minority-class samples with a novel translation loss function. This approach assists the model in learning a more generalized representation of crucially important minority class samples. Moreover, by exploiting an auxiliary feature, an augmented attention module is designed that focuses on the most relevant and target-specific information. We adopted an inter-patient classification paradigm to evaluate the proposed method. The experimental results of this study on the MIT-BIH arrhythmia database clearly indicate that the proposed model with augmented attention mechanism and over-sampling strategy significantly learns a balanced deep representation and improves the classification performance of vital heartbeats.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001221547700062</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>37851553</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Arrhythmia detection; beat classification; imbalanced learning; remote health monitoring</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
CONVOLUTIONAL NEURAL-NETWORK; HEARTBEAT CLASSIFICATION; MORPHOLOGY; FEATURES; MODEL</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Zubair, Muhammad; Woo, Sungpil; Lim, Sunhwan] Elect &amp; Telecommun Res Inst ETRI, Autonomous IoT Res Sect, Daejeon 34129, South Korea. <br>
[Woo, Sungpil; Kim, Daeyoung] Korea Adv Inst Sci &amp; Technol KAIST, Dept Sch Comp, Daejeon 34141, South Korea. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Woo, S (corresponding author), Elect &amp; Telecommun Res Inst ETRI, Autonomous IoT Res Sect, Daejeon 34129, South Korea.<br>Woo, S (corresponding author), Korea Adv Inst Sci &amp; Technol KAIST, Dept Sch Comp, Daejeon 34141, South Korea.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
zubair5608@etri.re.kr; woosungpil@etri.re.kr; shlim@etri.re.kr; kimd@kaist.ac.kr</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Electronics &amp; Telecommunications Research Institute - Korea (ETRI); Korea Advanced Institute of Science &amp; Technology (KAIST)</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Zubair, Muhammad</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-3595-4969&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>ZUBAIR, MUHAMMAD</display_name>&nbsp;</td><td>JVN-7799-2024&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Information Systems; Computer Science, Interdisciplinary Applications; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>QN4V6</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2168-2194</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>2168-2208</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE J BIOMED HEALTH</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE J. Biomed. Health Inform.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>12</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Institute of Information &amp; Communications Technology Planning Evaluation</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>No Statement Available</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>Bronze</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 29 of 38</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>A Generic Quality Control Framework for Fetal Ultrasound Cardiac Four-Chamber Planes</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Dong, JB (Dong, Jinbao); Liu, SF (Liu, Shengfeng); Liao, YM (Liao, Yimei); Wen, HX (Wen, Huaxuan); Lei, BY (Lei, Baiying); Li, SL (Li, Shengli); Wang, TF (Wang, Tianfu)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE JOURNAL OF BIOMEDICAL AND HEALTH INFORMATICS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>24</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>4</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>931-942</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/JBHI.2019.2948316</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2020 APR</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>72</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>95</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
1</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
55</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>52</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Quality control/assessment of ultrasound (US) images is an essential step in clinical diagnosis. This process is usually done manually, suffering from some drawbacks, such as dependence on operator's experience and extensive labors, as well as high inter- and intra-observer variation. Automatic quality assessment of US images is therefore highly desirable. Fetal US cardiac four-chamber plane (CFP) is one of the most commonly used cardiac views, which was used in the diagnosis of heart anomalies in the early 1980s. In this paper, we propose a generic deep learning framework for automatic quality control of fetal US CFPs. The proposed framework consists of three networks: (1) a basic CNN (B-CNN), roughly classifying four-chamber views from the raw data; (2) a deeper CNN (D-CNN), determining the gain and zoom of the target images in a multi-task learning manner; and (3) the aggregated residual visual block net (ARVBNet), detecting the key anatomical structures on a plane. Based on the output of the three networks, overall quantitative score of each CFP is obtained, so as to achieve fully automatic quality control. Experiments on a fetal US dataset demonstrated our proposed method achieved a highest mean average precision (mAP) of 93.52% at a fast speed of 101 frames per second (FPS). In order to demonstrate the adaptability and generalization capacity, the proposed detection network (i.e., ARVBNet) has also been validated on the PASCAL VOC dataset, obtaining a highest mAP of 81.2% when input size is approximately 300 x 300.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000525354900001</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>31634851</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Quality control; Anatomical structure; Quality assessment; Ultrasonic imaging; Real-time systems; Quality control; cardiac four-chamber planes (CFPs); convolutional neural network (CNN); real-time object detection; aggregated residual visual block (ARVB)</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
HEART</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Dong, Jinbao; Liu, Shengfeng; Lei, Baiying; Wang, Tianfu] Shenzhen Univ, Natl Reg Key Technol Engn Lab Med Ultrasound, Guangdong Prov Key Lab Biomed Measurements &amp; Ultr, Sch Biomed Engn,Hlth Sci Ctr, Shenzhen 518060, Peoples R China. <br>
[Liao, Yimei; Wen, Huaxuan; Li, Shengli] Southern Med Univ, Shenzhen Matern &amp; Child Healthcare Hosp, Dept Ultrasound, Shenzhen 518028, Peoples R China. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Wang, TF (corresponding author), Shenzhen Univ, Natl Reg Key Technol Engn Lab Med Ultrasound, Guangdong Prov Key Lab Biomed Measurements &amp; Ultr, Sch Biomed Engn,Hlth Sci Ctr, Shenzhen 518060, Peoples R China.<br>Li, SL (corresponding author), Southern Med Univ, Shenzhen Matern &amp; Child Healthcare Hosp, Dept Ultrasound, Shenzhen 518028, Peoples R China.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
995238847@qq.com; liusf2009@163.com; liaoyimei1987@qq.com; whxwell@126.com; leiby@szu.edu.cn; lishengli63@126.com; tfwang@szu.edu.cn</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Shenzhen University; Southern Medical University - China; Shenzhen Maternity &amp; Child Healthcare Hospital</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>liu, sheng feng</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-6902-3329&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>liu, shengfeng</display_name>&nbsp;</td><td>N-2303-2018&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Lei, Baiying</display_name>&nbsp;</td><td>GQO-8422-2022&nbsp;</td><td>0000-0002-3087-2550&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Liao, Yimei</display_name>&nbsp;</td><td>ABG-8261-2021&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Li, Shengli</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-0570-4165&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Information Systems; Computer Science, Interdisciplinary Applications; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>LC5GX</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2168-2194</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>2168-2208</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE J BIOMED HEALTH</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE J. Biomed. Health Inform.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>12</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Natural Science Foundation of China</grant_agency>&nbsp;</td><td>
<div>81571758&nbsp;</div>
<div>81771922&nbsp;</div>
<div>61871274&nbsp;</div>
<div>61801305&nbsp;</div>
<div>61501305&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Key Research and Develop Program</grant_agency>&nbsp;</td><td>
<div>2016YFC0104700&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported in part by the National Natural Science Foundation of China under Grant 81571758, Grant 81771922, Grant 61871274, Grant 61801305, and Grant 61501305; and in part by the National Key Research and Develop Program (2016YFC0104700).</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 30 of 38</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Foundation versus domain-specific models for left ventricular segmentation on cardiac ultrasound</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Chao, CJ (Chao, Chieh-Ju); Gu, YR (Gu, Yunqi Richard); Kumar, W (Kumar, Wasan); Xiang, TG (Xiang, Tiange); Appari, L (Appari, Lalith); Wu, J (Wu, Justin); Farina, JM (Farina, Juan M.); Wraith, R (Wraith, Rachael); Jeong, J (Jeong, Jiwoon); Arsanjani, R (Arsanjani, Reza); Kane, GC (Kane, Garvan C.); Oh, JK (Oh, Jae K.); Langlotz, CP (Langlotz, Curtis P.); Banerjee, I (Banerjee, Imon); Fei-Fei, L (Fei-Fei, Li); Adeli, E (Adeli, Ehsan)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>NPJ DIGITAL MEDICINE</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>8</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>1</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>341</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1038/s41746-025-01730-y</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2025 JUN 6</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>0</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>0</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
8</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
8</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>44</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
The Segment Anything Model (SAM) was fine-tuned on the EchoNet-Dynamic dataset and evaluated on external transthoracic echocardiography (TTE) and Point-of-Care Ultrasound (POCUS) datasets from CAMUS (University Hospital of St Etienne) and Mayo Clinic (99 patients: 58 TTE, 41 POCUS). Fine-tuned SAM was superior or comparable to MedSAM. The fine-tuned SAM also outperformed EchoNet and U-Net models, demonstrating strong generalization, especially on apical 2-chamber (A2C) images (fine-tuned SAM vs. EchoNet: CAMUS-A2C: DSC 0.891 +/- 0.040 vs. 0.752 +/- 0.196, p &lt; 0.0001) and POCUS (DSC 0.857 +/- 0.047 vs. 0.667 +/- 0.279, p &lt; 0.0001). Additionally, SAM-enhanced workflow reduced annotation time by 50% (11.6 +/- 4.5 sec vs. 5.7 +/- 1.7 sec, p &lt; 0.0001) while maintaining segmentation quality. We demonstrated an effective strategy for fine-tuning a vision foundation model for enhancing clinical workflow efficiency and supporting human-AI collaboration.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001504261000002</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>40481190</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
EJECTION FRACTION; DECISION-MAKING; ECHOCARDIOGRAPHY; LIMITATIONS; IMPACT</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Chao, Chieh-Ju; Kane, Garvan C.; Oh, Jae K.] Mayo Clin, Dept Cardiovasc Med, Rochester, MN 55905 USA. <br>
[Chao, Chieh-Ju; Gu, Yunqi Richard; Kumar, Wasan; Xiang, Tiange; Wu, Justin; Fei-Fei, Li; Adeli, Ehsan] Stanford Univ, Stanford Inst Human Ctr Artificial Intelligence, Stanford, CA 94305 USA. <br>
[Appari, Lalith; Jeong, Jiwoon; Banerjee, Imon] Mayo Clin Arizona, Dept Radiol, Scottsdale, AZ USA. <br>
[Appari, Lalith; Jeong, Jiwoon; Banerjee, Imon] Arizona State Univ, Sch Comp &amp; Augmented Intelligence, Phoenix, AZ USA. <br>
[Farina, Juan M.; Wraith, Rachael; Arsanjani, Reza] Mayo Clin Arizona, Dept Cardiovasc Med, Scottsdale, AZ USA. <br>
[Langlotz, Curtis P.] Stanford Univ, Ctr Artificial Intelligence Med Imaging, Stanford, CA USA. <br>
[Langlotz, Curtis P.] Stanford Univ, Dept Radiol, Stanford, CA USA. <br>
[Fei-Fei, Li; Adeli, Ehsan] Stanford Univ, Dept Comp Sci, Stanford, CA 94305 USA. <br>
[Adeli, Ehsan] Stanford Univ, Dept Psychiat &amp; Behav Sci, Stanford, CA 94305 USA. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Fei-Fei, L; Adeli, E (corresponding author), Stanford Univ, Stanford Inst Human Ctr Artificial Intelligence, Stanford, CA 94305 USA.<br>Fei-Fei, L; Adeli, E (corresponding author), Stanford Univ, Dept Comp Sci, Stanford, CA 94305 USA.<br>Adeli, E (corresponding author), Stanford Univ, Dept Psychiat &amp; Behav Sci, Stanford, CA 94305 USA.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
feifeili@cs.stanford.edu; eadeli@stanford.edu</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Mayo Clinic; Stanford University; Mayo Clinic; Mayo Clinic Phoenix; Arizona State University; Arizona State University-Downtown Phoenix; Mayo Clinic; Mayo Clinic Phoenix; Stanford University; Stanford University; Stanford University; Stanford University</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>FARINA, JUAN MARIA</display_name>&nbsp;</td><td>JFS-8159-2023&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>NATURE PORTFOLIO</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>HEIDELBERGER PLATZ 3, BERLIN, 14197, GERMANY</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Health Care Sciences &amp; Services; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Health Care Sciences &amp; Services; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>3NB4U</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2398-6352</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>NPJ DIGIT MED</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>npj Digit. Med.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>9</value>
</td>
</tr>

<tr>
<td>
<b>Open Access:</b>

<value>gold, Green Submitted</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 31 of 38</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Improving the Quality of Fetal Heart Ultrasound Imaging With Multihead Enhanced Self-Attention and Contrastive Learning</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Zhang, YY (Zhang, Yingying); Zhu, HG (Zhu, Haogang); Cheng, J (Cheng, Jian); Wang, JY (Wang, Jingyi); Gu, XY (Gu, Xiaoyan); Han, JC (Han, Jiancheng); Zhang, Y (Zhang, Ye); Zhao, Y (Zhao, Ying); He, YH (He, Yihua); Zhang, HJ (Zhang, Hongjia)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE JOURNAL OF BIOMEDICAL AND HEALTH INFORMATICS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>27</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>11</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>5518-5529</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/JBHI.2023.3303573</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2023 NOV</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>9</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>10</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
4</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
22</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>51</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Fetal congenital heart disease (FCHD) is a common, serious birth defect affecting similar to 1% of newborns annually. Fetal echocardiography is the most effective and important technique for prenatal FCHD diagnosis. The pre-requisites for accurate ultrasound FCHD diagnosis are accurate view recognition and high-quality diagnostic view extraction. However, these manual clinical procedures have drawbacks such as, varying technical capabilities and in-efficiency. Therefore, the automatic identification of high-quality multiview fetal heart scan images is highly desirable to improve prenatal diagnosis efficiency and accuracy of FCHD. Here, we present a framework for multiview fetal heart ultrasound image recognition and quality assessment that comprises two parts: a multiview classification and localization network (MCLN) and an improved contrastive learning network (ICLN). In the MCLN, a multihead enhanced self-attention mechanism is applied to construct the classification network and identify six accurate and interpretable views of the fetal heart. In the ICLN, anatomical structure standardization and image clarity are considered. With contrastive learning, the absolute loss, feature relative loss and predicted value relative loss are combined to achieve favorable quality assessment results. Experiments show that the MCLN outperforms other state-of-the-art networks by 1.52-13.61% when determining the F1 score in six standard view recognition tasks, and the ICLN is comparable to the performance of expert cardiologists in the quality assessment of fetal heart ultrasound images, reaching 97% on a test set within 2 points for the four-chamber view task. Thus, our architecture offers great potential in helping cardiologists improve quality control for fetal echocardiographic images in clinical practice.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001129955100029</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>37556337</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Contrastive learning; fetal congenital heart disease; fetal echocardiography; view recognition; quality assessment; multihead enhanced self-attention</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
NEURAL-NETWORKS; ECHOCARDIOGRAPHY; DISEASE; MANAGEMENT</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Zhang, Yingying] Beihang Univ, Sch Biol Sci &amp; Med Engn, Beijing 100191, Peoples R China. <br>
[Zhu, Haogang; Cheng, Jian] Beihang Univ, Sch Comp Sci &amp; Engn, State Key Lab Software Dev Environm, Beijing 100191, Peoples R China. <br>
[Zhu, Haogang; Cheng, Jian] Zhongguancun Lab, Beijing 102206, Peoples R China. <br>
[Wang, Jingyi; Gu, Xiaoyan; Han, Jiancheng; Zhang, Ye; Zhao, Ying; He, Yihua] Capital Med Univ, Beijing Anzhen Hosp, Echocardiog Med Ctr, Beijing 100054, Peoples R China. <br>
[Zhang, Hongjia] Beijing Lab Cardiovasc Precis Med, Beijing 100029, Peoples R China. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Zhu, HG (corresponding author), Beihang Univ, Sch Comp Sci &amp; Engn, State Key Lab Software Dev Environm, Beijing 100191, Peoples R China.<br>Zhu, HG (corresponding author), Zhongguancun Lab, Beijing 102206, Peoples R China.<br>He, YH (corresponding author), Capital Med Univ, Beijing Anzhen Hosp, Echocardiog Med Ctr, Beijing 100054, Peoples R China.<br>Zhang, HJ (corresponding author), Beijing Lab Cardiovasc Precis Med, Beijing 100029, Peoples R China.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
zhangyingying@buaa.edu.cn; haogangzhu@buaa.edu.cn; jian_cheng@buaa.edu.cn; wangjingyianzhen@163.com; xiaoyan_gu@yahoo.com; han_jc1977@hotmail.com; zhang3389@qq.com; yingzhaoecho@163.com; heyihuae-cho@hotmail.com; zhanghongjia722@hotmail.com</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Beihang University; Beihang University; Zhongguancun Laboratory; Capital Medical University</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>贺, 艺华</display_name>&nbsp;</td><td>AFV-8567-2022&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Cheng, Jian</display_name>&nbsp;</td><td>J-9124-2017&nbsp;</td><td>0000-0003-1435-673X&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Wang, Jingyi</display_name>&nbsp;</td><td>AAS-7653-2020&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>zhang, yingying</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-0964-1774&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Zhang, Jian</display_name>&nbsp;</td><td>F-7865-2015&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Information Systems; Computer Science, Interdisciplinary Applications; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>DC9R2</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2168-2194</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>2168-2208</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE J BIOMED HEALTH</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE J. Biomed. Health Inform.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>12</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Key R&amp;D Program of China</grant_agency>&nbsp;</td><td>
<div>2021ZD0140407&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Natural Science Foundation of China</grant_agency>&nbsp;</td><td>
<div>U21A20523&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Beijing Natural Science Foundation</grant_agency>&nbsp;</td><td>
<div>L222152&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Beijing Municipal Administration of Hospitals Incubating Program</grant_agency>&nbsp;</td><td>
<div>PX2022026&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Fundamental Research Funds for the Central Universities, Beijing Advanced Innovation Center for Big Data-Based Precision Medicine, Beihang University, Beijing, China</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>State Key Laboratory of Software Development Environment, Beihang University, Beijing, China, Beijing Lab for Cardiovascular Precision Medicine</grant_agency>&nbsp;</td><td>
<div>PXM2018_014226_000013&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Beijing Key Laboratory of Maternal-Fetal Medicine in Fetal Heart Disease</grant_agency>&nbsp;</td><td>
<div>BZ0308&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported in part by the National Key R&amp;D Program of China under Grant 2021ZD0140407, in part by the National Natural Science Foundation of China under Grant U21A20523, in part by the Beijing Natural Science Foundation under Grant L222152, in part by the Beijing Municipal Administration of Hospitals Incubating Program under Grant PX2022026, in part by the Fundamental Research Funds for the Central Universities, Beijing Advanced Innovation Center for Big Data-Based Precision Medicine, Beihang University, Beijing, China, in part by the State Key Laboratory of Software Development Environment, Beihang University, Beijing, China, Beijing Lab for Cardiovascular Precision Medicine under Grant PXM2018_014226_000013, and in part by the Beijing Key Laboratory of Maternal-Fetal Medicine in Fetal Heart Disease under Grant BZ0308.</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>Bronze</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 32 of 38</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Unsupervised stochastic learning and reduced order modeling for global sensitivity analysis in cardiac electrophysiology models</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
El Mo&ccedil;ayd, N (El Mocayd, Nabil); Belhamadia, Y (Belhamadia, Youssef); Seaid, M (Seaid, Mohammed)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>COMPUTER METHODS AND PROGRAMS IN BIOMEDICINE</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>255</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>108311</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1016/j.cmpb.2024.108311</value>&nbsp;&nbsp;<b>Early Access Date:</b> 
<value>JUL 2024</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2024 OCT</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>0</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>0</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
1</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
1</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>63</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Background and Objective: Numerical simulations in electrocardiology are often affected by various uncertainties inherited from the lack of precise knowledge regarding input values including those related to the cardiac cell model, domain geometry, and boundary or initial conditions used in the mathematical modeling. Conventional techniques for uncertainty quantification in modeling electrical activities of the heart encounter significant challenges, primarily due to the high computational costs associated with fine temporal and spatial scales. Additionally, the need for numerous model evaluations to quantify ubiquitous uncertainties increases the computational challenges even further. Methods: In the present study, we propose a non-intrusive surrogate model to perform uncertainty quantification and global sensitivity analysis in cardiac electrophysiology models. The proposed method combines an unsupervised machine learning technique with the polynomial chaos expansion to reconstruct a surrogate model for the propagation and quantification of uncertainties in the electrical activity of the heart. The proposed methodology not only accurately quantifies uncertainties at a very low computational cost but more importantly, it captures the targeted quantity of interest as either the whole spatial field or the whole temporal period. In order to perform sensitivity analysis, aggregated Sobol indices are estimated directly from the spectral mode of the polynomial chaos expansion. Results: We conduct Uncertainty Quantification (UQ) and global Sensitivity Analysis (SA) considering both spatial and temporal variations, rather than limiting the analysis to specific Quantities of Interest (QoIs). To assess the comprehensive performance of our methodology in simulating cardiac electrical activity, we utilize the monodomain model. Additionally, sensitivity analysis is performed on the parameters of the Mitchell-Schaeffer cell model. Conclusions: Unlike conventional techniques for uncertainty quantification in modeling electrical activities, the proposed methodology performs at a low computational cost the sensitivity analysis on the cardiac electrical activity parameters. The results are fully reproducible and easily accessible, while the proposed reduced-order model represents a significant contribution to enhancing global sensitivity analysis in cardiac electrophysiology.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001275548400001</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>39032242</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Uncertainty quantification; Polynomial Chaos expansion; Unsupervised stochastic learning; Cardiac simulation; Cardiac electrophysiology</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
POLYNOMIAL CHAOS EXPANSION; UNCERTAINTY QUANTIFICATION; SIMULATION; SURROGATES; DYNAMICS; WAVES</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[El Mocayd, Nabil] Univ Mohammed VI Polytech, Coll Agr &amp; Environm Sci, Ben Guerir, Morocco. <br>
[Belhamadia, Youssef] Amer Univ Sharjah, Dept Math &amp; Stat, Sharjah, U Arab Emirates. <br>
[Seaid, Mohammed] Univ Durham, Dept Engn, South Rd, Durham DH1 3LE, England. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Belhamadia, Y (corresponding author), Amer Univ Sharjah, Dept Math &amp; Stat, Sharjah, U Arab Emirates.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
nabil.elmocayd@um6p.ma; ybelhamadia@aus.edu; m.seaid@durham.ac.uk</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Mohammed VI Polytechnic University; American University of Sharjah; Durham University</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>El Mocayd, Nabil</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-3258-8537&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>El Mocayd, Nabil</display_name>&nbsp;</td><td>ABC-5803-2021&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Seaid, Mohammed</display_name>&nbsp;</td><td>C-9619-2009&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>ELSEVIER IRELAND LTD</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>ELSEVIER HOUSE, BROOKVALE PLAZA, EAST PARK SHANNON, CO, CLARE, 00000, IRELAND</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Interdisciplinary Applications; Computer Science, Theory &amp; Methods; Engineering, Biomedical; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Engineering; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>ZL8O7</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0169-2607</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1872-7565</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>COMPUT METH PROG BIO</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Comput. Meth. Programs Biomed.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>14</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>American University of Sharjah, United Arab Emirates</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Calcul Quebec</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Compute Canada</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>The authors wish to acknowledge the financial support provided by the American University of Sharjah, United Arab Emirates. Computations were performed on the Colosse supercomputer at Universite Laval under the support of Calcul Quebec and Compute Canada</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>Green Submitted</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 33 of 38</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Using Deep Transfer Learning to Detect Hyperkalemia From Ambulatory Electrocardiogram Monitors in Intensive Care Units: Personalized Medicine Approach</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Chiu, IM (Chiu, I. -Min); Cheng, JY (Cheng, Jhu-Yin); Chen, TY (Chen, Tien -Yu); Wang, YM (Wang, Yi-Min); Cheng, CY (Cheng, Chi -Yung); Kung, CT (Kung, Chia-Te); Cheng, FJ (Cheng, Fu-Jen); Yau, FFF (Yau, Fei-Fei Flora); Lin, CHR (Lin, Chun-Hung Richard)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>JOURNAL OF MEDICAL INTERNET RESEARCH</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>24</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>12</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>e41163</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.2196/41163</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2022 DEC 6</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>3</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>3</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
2</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
14</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>28</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Background: Hyperkalemia is a critical condition, especially in intensive care units. So far, there have been no accurate and noninvasive methods for recognizing hyperkalemia events on ambulatory electrocardiogram monitors.
<br>Objective: This study aimed to improve the accuracy of hyperkalemia predictions from ambulatory electrocardiogram (ECG) monitors using a personalized transfer learning method; this would be done by training a generic model and refining it with personal data.
<br>Methods: This retrospective cohort study used open source data from the Waveform Database Matched Subset of the Medical Information Mart From Intensive Care III (MIMIC-III). We included patients with multiple serum potassium test results and matched ECG data from the MIMIC-III database. A 1D convolutional neural network-based deep learning model was first developed to predict hyperkalemia in a generic population. Once the model achieved a state-of-the-art performance, it was used in an active transfer learning process to perform patient-adaptive heartbeat classification tasks.
<br>Results: The results show that by acquiring data from each new patient, the personalized model can improve the accuracy of hyperkalemia detection significantly, from an average of 0.604 (SD 0.211) to 0.980 (SD 0.078), when compared with the generic model. Moreover, the area under the receiver operating characteristic curve level improved from 0.729 (SD 0.240) to 0.945 (SD 0.094).
<br>Conclusions: By using the deep transfer learning method, we were able to build a clinical standard model for hyperkalemia detection using ambulatory ECG monitors. These findings could potentially be extended to applications that continuously monitor one's ECGs for early alerts of hyperkalemia and help avoid unnecessary blood tests.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000915325800012</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>36469396</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
deep learning; transfer learning; hyperkalemia; electrocardiogram; ECG monitor; ICU; personalized medicine</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
POTASSIUM</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Chiu, I. -Min; Cheng, Chi -Yung; Lin, Chun-Hung Richard] Natl Sun Yat Sen Univ, Dept Comp Sci &amp; Engn, Kaohsiung, Taiwan. <br>
[Chiu, I. -Min; Cheng, Jhu-Yin; Wang, Yi-Min; Cheng, Chi -Yung; Kung, Chia-Te; Cheng, Fu-Jen; Yau, Fei-Fei Flora] Kaohsiung Chang Gung Mem Hosp, Dept Emergency Med, Kaohsiung Cty, Taiwan. <br>
[Chen, Tien -Yu] Kaohsiung Chang Gung Mem Hosp, Dept Internal Med, Div Cardiol, Kaohsiung, Taiwan. <br>
[Lin, Chun-Hung Richard] Natl Sun Yat Sen Univ, Dept Comp Sci &amp; Engn, 70 Lienhai Rd, Kaohsiung 804, Taiwan. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Lin, CHR (corresponding author), Natl Sun Yat Sen Univ, Dept Comp Sci &amp; Engn, 70 Lienhai Rd, Kaohsiung 804, Taiwan.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
lin@cse.nsysu.edu.tw</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
National Sun Yat Sen University; Chang Gung Memorial Hospital; Chang Gung Memorial Hospital; National Sun Yat Sen University</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Cheng, Abbie</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-5945-3993&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>CHEN, YU-WEN</display_name>&nbsp;</td><td>E-8238-2011&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Cheng, Yeung</display_name>&nbsp;</td><td>AFN-3087-2022&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Chen, Tien Yu</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-7805-9278&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Chaou, Chung-Hsien</display_name>&nbsp;</td><td>AAZ-7344-2020&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Wang, Yi-Min</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-7467-1964&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>CHEN, YUFEI</display_name>&nbsp;</td><td>KRO-7149-2024&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Cheng, Chi Yung</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-1109-9339&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Cheng, Fu-Jen</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-1960-2274&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>JMIR PUBLICATIONS, INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>130 QUEENS QUAY East, Unit 1100, TORONTO, ON M5A 0P6, CANADA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Health Care Sciences &amp; Services; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Health Care Sciences &amp; Services; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>7Z1LG</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>1438-8871</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>J MED INTERNET RES</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>J. Med. Internet Res.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>10</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Ministry of Science and Technology of Taiwan</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>&nbsp;</td><td>
<div>MOST 109-2314-B-182A-174&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>Acknowledgments This work was supported by the Ministry of Science and Technology of Taiwan (grant MOST 109-2314-B-182A-174) .</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>gold, Green Submitted</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 34 of 38</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Wearable Device Heart Rate and Activity Data in an Unsupervised Approach to Personalized Sleep Monitoring: Algorithm Validation</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Liu, JX (Liu, Jiaxing); Zhao, Y (Zhao, Yang); Lai, BY (Lai, Boya); Wang, HL (Wang, Hailiang); Tsui, KL (Tsui, Kwok Leung)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>JMIR MHEALTH AND UHEALTH</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>8</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>8</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>e18370</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.2196/18370</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2020 AUG 5</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>14</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>14</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
0</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
13</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>46</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Background: The proliferation of wearable devices that collect activity and heart rate data has facilitated new ways to measure sleeping and waking durations unobtrusively and longitudinally. Most existing sleep/wake identification algorithms are based on activity only and are trained on expensive and laboriously annotated polysomnography (PSG). Heart rate can also be reflective of sleep/wake transitions, which has motivated its investigation herein in an unsupervised algorithm. Moreover, it is necessary to develop a personalized approach to deal with interindividual variance in sleep/wake patterns.
<br>Objective: We aimed to develop an unsupervised personalized sleep/wake identification algorithm using multifaceted data to explore the benefits of incorporating both heart rate and activity level in these types of algorithms and to compare this approach's output with that of an existing commercial wearable device's algorithms.
<br>Methods: In this study, a total of 14 community-dwelling older adults wore wearable devices (Fitbit Alta; Fitbit Inc) 24 hours a day and 7 days a week over period of 3 months during which their heart rate and activity data were collected. After preprocessing the data, a model was developed to distinguish sleep/wake states based on each individual's data. We proposed the use of hidden Markov models and compared different modeling schemes. With the best model selected, sleep/wake patterns were characterized by estimated parameters in hidden Markov models, and sleep/wake states were identified.
<br>Results: When applying our proposed algorithm on a daily basis, we found there were significant differences in estimated parameters between weekday models and weekend models for some participants.
<br>Conclusions: Our unsupervised approach can be effectively implemented based on an individual's multifaceted sleep-related data from a commercial wearable device. A personalized model is shown to be necessary given the interindividual variability in estimated parameters.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000617734400001</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>32755887</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
sleep/wake identification; hidden Markov model; personalized health; unsupervised learning; sleep; physical activity; wearables; heart rate</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
WAKE IDENTIFICATION; DURATION; ACTIGRAPHY; HEALTH; POLYSOMNOGRAPHY; ASSOCIATIONS; METAANALYSIS; PARAMETERS; STATEMENT; MOVEMENT</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Liu, Jiaxing; Zhao, Yang; Lai, Boya; Wang, Hailiang; Tsui, Kwok Leung] City Univ Hong Kong, Sch Data Sci, Kowloon, Hong Kong, Peoples R China. <br>
[Zhao, Yang; Wang, Hailiang; Tsui, Kwok Leung] City Univ Hong Kong, Ctr Syst Informat Engn, Kowloon, Tat Chee Ave, Hong Kong 999077, Peoples R China. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Zhao, Y (corresponding author), City Univ Hong Kong, Ctr Syst Informat Engn, Kowloon, Tat Chee Ave, Hong Kong 999077, Peoples R China.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
yang.zhao@my.cityu.edu.hk</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
City University of Hong Kong; City University of Hong Kong</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>wang, hailiang</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-6668-7947&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>LIU, Jiaxing</display_name>&nbsp;</td><td>AAA-2762-2021&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>wang, hailiang</display_name>&nbsp;</td><td>LPP-4633-2024&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Zhao, Yang</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-1571-5394&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Tsui, Kwok-Leung</display_name>&nbsp;</td><td>JTU-2163-2023&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>LIU, Jiaxing</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-5184-4313&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>JMIR PUBLICATIONS, INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>130 QUEENS QUAY East, Unit 1100, TORONTO, ON M5A 0P6, CANADA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Health Care Sciences &amp; Services; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Health Care Sciences &amp; Services; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>QG7AB</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2291-5222</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>JMIR MHEALTH UHEALTH</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>JMIR mHealth uHealth</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>15</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Research Grants Council Theme-based Research Scheme</grant_agency>&nbsp;</td><td>
<div>T32-102/14N&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>City University of Hong Kong</grant_agency>&nbsp;</td><td>
<div>9610406&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This study was supported by the Research Grants Council Theme-based Research Scheme (T32-102/14N) and City University of Hong Kong (Grant No. 9610406).</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>Green Submitted, gold</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 35 of 38</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>rU-Net, Multi-Scale Feature Fusion and Transfer Learning: Unlocking the Potential of Cuffless Blood Pressure Monitoring With PPG and ECG</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Chen, JM (Chen, Jiaming); Zhou, XL (Zhou, Xueling); Feng, L (Feng, Lei); Ling, BWK (Ling, Bingo Wing-Kuen); Han, LY (Han, Lianyi); Zhang, HT (Zhang, Hongtao)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE JOURNAL OF BIOMEDICAL AND HEALTH INFORMATICS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>29</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>1</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>166-176</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/JBHI.2024.3483301</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2025 JAN</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>0</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>0</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
13</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
28</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>52</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
This study introduces an innovative deep-learning model for cuffless blood pressure estimation using PPG and ECG signals, demonstrating state-of-the-art performance on the largest clean dataset, PulseDB. The rU-Net architecture, a fusion of U-Net and ResNet, enhances both generalization and feature extraction accuracy. Accurate multi-scale feature capture is facilitated by short-time Fourier transform (STFT) time-frequency distributions and multi-head attention mechanisms, allowing data-driven feature selection. The inclusion of demographic parameters as supervisory information further elevates performance. On the calibration-based dataset, our model excels, achieving outstanding accuracy (SBP MAE +/- std: 4.49 +/- 4.86 mmHg, DBP MAE +/- std: 2.69 +/- 3.10 mmHg), surpassing AAMI standards and earning a BHS Grade A rating. Addressing the challenge of calibration-free data, we propose a fine-tuning-based transfer learning approach. Remarkably, with only 10% data transfer, our model attains exceptional accuracy (SBP MAE +/- std: 4.14 +/- 5.01 mmHg, DBP MAE +/- std: 2.48 +/- 2.93 mmHg). This study sets the stage for the development of highly accurate and reliable wearable cuffless blood pressure monitoring devices.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001394517600007</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>39423074</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Feature extraction; Accuracy; Estimation; Electrocardiography; Blood pressure; Time series analysis; Data models; Time-frequency analysis; Monitoring; electrocardiogram; photoplethysmography; STFT; transfer learning; transfer learning</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
WAVE-FORM; PHOTOPLETHYSMOGRAPHY</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Chen, Jiaming; Zhou, Xueling; Zhang, Hongtao] Fudan Univ, Sch Informat Sci &amp; Technol, Shanghai 200437, Peoples R China. <br>
[Chen, Jiaming; Zhou, Xueling; Feng, Lei; Han, Lianyi; Zhang, Hongtao] Greater Bay Area Inst Precis Med, Guangzhou 511462, Peoples R China. <br>
[Ling, Bingo Wing-Kuen] Guangdong Univ Technol, Sch Informat Engn, Guangzhou 510006, Peoples R China. <br>
[Han, Lianyi] Fudan Univ, Huashan Hosp, Sch Life Sci, Dept Dermatol, Shanghai 200437, Peoples R China. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Zhang, HT (corresponding author), Fudan Univ, Sch Informat Sci &amp; Technol, Shanghai 200437, Peoples R China.<br>Han, LY (corresponding author), Fudan Univ, Huashan Hosp, Sch Life Sci, Dept Dermatol, Shanghai 200437, Peoples R China.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
jmchen22@m.fudan.edu.cn; xueling_zhou@fudan.edu.cn; fenglei@ipm-gba.org.cn; yongquanling@gdut.edu.cn; hanlianyi@fudan.edu.cn; zhanghongtao@ipm-gba.org.cn</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Fudan University; Guangdong University of Technology; Fudan University</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Feng, Lei</display_name>&nbsp;</td><td>AAV-7856-2020&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>zhang, hongtao</display_name>&nbsp;</td><td>NXB-9922-2025&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Chen, Jiaming</display_name>&nbsp;</td><td>GXM-7165-2022&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Han, Lianyi</display_name>&nbsp;</td><td>D-1499-2009&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Information Systems; Computer Science, Interdisciplinary Applications; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>R9J4Q</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2168-2194</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>2168-2208</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE J BIOMED HEALTH</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE J. Biomed. Health Inform.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>11</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 36 of 38</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Learning Spatio-Temporal Pulse Representation With Global-Local Interaction and Supervision for Remote Prediction of Heart Rate</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Zhao, CC (Zhao, Changchen); Zhou, MH (Zhou, Menghao); Zhao, Z (Zhao, Zheng); Huang, B (Huang, Bin); Rao, B (Rao, Bing)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>IEEE JOURNAL OF BIOMEDICAL AND HEALTH INFORMATICS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>28</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>2</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>609-620</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1109/JBHI.2023.3252091</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2024 FEB</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>12</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>12</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
6</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
18</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>52</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Recent studies have demonstrated the benefit of extracting and fusing pulse signals from multi-scale region-of-interests (ROIs). However, these methods suffer from heavy computational load. This paper aims to effectively utilize multi-scale rPPG features with a more compact architecture. Inspired by recent research works exploring two-path architecture that leverages global and local information with bidirectional bridge in between. This paper designs a novel architecture Global-Local Interaction and Supervision Network (GLISNet), which uses a local path to learn representations in the original scale and a global path to learn representations in the other scale capturing multi-scale information. A light-weight rPPG signal generation block is attached to the output of each path that maps the pulse representation to the pulse output. A hybrid loss function is utilized enabling the local and global representations to learn directly from the training data. Extensive experiments are conducted on two publicly available datasets, and results demonstrate that GLISNet achieves superior performance in terms of signal-to-noise ratio (SNR), mean absolute error (MAE), and root mean squared error (RMSE). In terms of SNR, GLISNet has an increase of 4.41% compared with the second best algorithm PhysNet on PURE dataset. The MAE has a decrease of 13.16% compared with the second best algorithm DeeprPPG on UBFC-rPPG dataset. The RMSE has a decrease of 26.29% compared with the second best algorithm PhysNet on UBFC-rPPG dataset. Experiments on MIHR dataset demonstrates the robustness of GLISNet under low-light environment.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001203362500041</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>37028087</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Remote photoplethysmography; heart rate estimation; pulse representation; CNN; long-range dependency</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Zhao, Changchen; Zhao, Zheng; Huang, Bin] Beihang Univ, Hangzhou Innovat Inst, Hangzhou 310051, Peoples R China. <br>
[Zhou, Menghao] Zhejiang Univ Technol, Coll Informat Engn, Hangzhou 310023, Peoples R China. <br>
[Rao, Bing] Hangzhou City Univ, Hangzhou 310015, Peoples R China. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Rao, B (corresponding author), Hangzhou City Univ, Hangzhou 310015, Peoples R China.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
cczhao@zjut.edu.cn; tracker.z@qq.com; zhaozhengbuaa@qq.com; marshuangbin@buaa.edu.cn; raob@zucc.edu.cn</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Beihang University; Zhejiang University of Technology; Hangzhou City University</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Huang, Bin</display_name>&nbsp;</td><td>ISA-9354-2023&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Zhao, Changchen</display_name>&nbsp;</td><td>AEV-3957-2022&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Huang, Bin</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-0894-5335&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Zhao, Changchen</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-0546-6016&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Rao, Bing</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-3760-8979&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>IEEE-INST ELECTRICAL ELECTRONICS ENGINEERS INC</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>445 HOES LANE, PISCATAWAY, NJ 08855-4141 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Information Systems; Computer Science, Interdisciplinary Applications; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Mathematical &amp; Computational Biology; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>NW0A0</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2168-2194</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>2168-2208</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>IEEE J BIOMED HEALTH</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>IEEE J. Biomed. Health Inform.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>12</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Natural Science Foundation of China</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>No Statement Available</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 37 of 38</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Cross-Gram matrices and their use in transfer learning: Application to automatic REM detection using heart rate</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Muller, B (Muller, Bruno); Lengell&eacute;, R (Lengelle, Regis)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>COMPUTER METHODS AND PROGRAMS IN BIOMEDICINE</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>208</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>106280</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1016/j.cmpb.2021.106280</value>&nbsp;&nbsp;<b>Early Access Date:</b> 
<value>JUL 2021</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2021 SEP</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>1</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>1</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
0</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
4</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>34</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Background and objectives: while traditional sleep staging is achieved through the visual -expert -based -annotation of a polysomnography, it has the disadvantages of being unpractical and expensive. Alternatives have been developed over the years to relieve sleep staging from its heavy requirements, through the collection of more easily assessable signals and its automation using machine learning. How -ever, these alternatives have their limitations, some due to variabilities among and between subjects, other inherent to their use of sub-discriminative signals. Many new solutions rely on the evaluation of the Autonomic Nervous System (ANS) activation through the assessment of the heart-rate (HR); the latter is modulated by the aforementioned variabilities, which may result in data and concept shifts between what was learned and what we want to classify. Such adversary effects are usually tackled by Transfer Learning, dealing with problems where there are dif-ferences between what is known (source) and what we want to classify (target). In this paper, we pro -pose two new kernel-based methods of transfer learning and assess their performances in Rapid-Eye-Movement (REM) sleep stage detection, using solely the heart rate. Methods: our first contribution is the introduction of Kernel-Cross Alignment (KCA), a measure of sim-ilarity between a source and a target, which is a direct extension of Kernel-Target Alignment (KTA). To our knowledge, KCA has currently never been studied in the literature. Our second contribution is two alignment-based methods of transfer learning: Kernel-Target Alignment Transfer Learning (KTATL) and Kernel-Cross Alignment Transfer Learning (KCATL). Both methods differ from KTA, whose traditional use is kernel-tuning: in our methods, the kernel has been fixed beforehand, and our objective is the improvement of the estimation of unknown target labels by taking into account how observations relate to each other, which, as it will be explained, allows to transfer knowledge (trans-fer learning). Results: we compare performances with transfer learning (KCATL, KTATL) to performances without trans-fer using a fixed classifier (a Support Vector Classifier -SVC). In most cases, both transfer learning meth-ods result in an improvement of performances (higher detection rates for a fixed false-alarm rate). Our methods do not require iterative computations. Conclusion: we observe improved performances using our transfer methods, which are computationally efficient, as they only require the computation of a kernel matrix and are non-iterative. However, some optimisation aspects are still under investigation. (c) 2021 Elsevier B.V. All rights reserved.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000685504200001</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>34333204</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Data &amp; concept shift; Kernel methods; Transfer learning; Sleep staging; REM Detection</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
RATE-VARIABILITY; SLEEP; FEATURES; TIME; CLASSIFICATION; DOMAIN</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Muller, Bruno; Lengelle, Regis] Univ Technol Troyes, Inst Charles Delaunay, 12 Rue Marie Curie,CS 42060, F-10004 Troyes, France. <br>
[Muller, Bruno] PPRS, 4E Ave Gen Gaulle, F-68000 Colmar, France. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Muller, B (corresponding author), Univ Technol Troyes, Inst Charles Delaunay, 12 Rue Marie Curie,CS 42060, F-10004 Troyes, France.<br>Muller, B (corresponding author), PPRS, 4E Ave Gen Gaulle, F-68000 Colmar, France.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
bmu@ppr.research.com; regis.lengelle@utt.fr</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Universite de Technologie de Troyes</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Lengell&eacute;, R&eacute;gis</display_name>&nbsp;</td><td>AAJ-8128-2021&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>MULLER, Bruno</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-9163-4807&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Lengelle, Regis</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-3538-3799&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>ELSEVIER IRELAND LTD</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>ELSEVIER HOUSE, BROOKVALE PLAZA, EAST PARK SHANNON, CO, CLARE, 00000, IRELAND</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Interdisciplinary Applications; Computer Science, Theory &amp; Methods; Engineering, Biomedical; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Engineering; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>UA9WK</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0169-2607</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1872-7565</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>COMPUT METH PROG BIO</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Comput. Meth. Programs Biomed.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>8</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 38 of 38</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Machine learning algorithms to predict early pregnancy loss after in vitro fertilization-embryo transfer with fetal heart rate as a strong predictor</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Liu, LJ (Liu, Lijue); Jiao, YX (Jiao, Yongxia); Li, XH (Li, Xihong); Ouyang, Y (Ouyang, Yan); Shi, DN (Shi, Danni)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>COMPUTER METHODS AND PROGRAMS IN BIOMEDICINE</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>196</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>105624</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1016/j.cmpb.2020.105624</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2020 NOV</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>41</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>52</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
0</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
24</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>23</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Background and objective: According to previous studies, after in vitro fertilization-embryo transfer (IVF-ET) there exist a high early pregnancy loss (EPL) rate. The objectives of this study were to construct a prediction model of embryonic development by using machine learning algorithms based on historical case data, in this way doctors can make more accurate suggestions on the number of patient follow-ups, and provide decision support for doctors who are relatively inexperienced in clinical practice.
<br>Methods: We analyzed the significance of the same type of features between ongoing pregnancy samples and EPL samples. At the same time, by analyzing the correlation between days after embryo transfer (ETD) and fetal heart rate (FHR) of those normal embryo samples, a regression model between the two was established to obtain FHR model of normal development, and the residual analysis was used to further clarify the importance of FHR in predicting pregnancy outcome. Finally we applied six representative machine learning algorithms including Logistic Regression (LR), Support Vector Machine (SVM), Decision Tree (DT), Back Propagation Neural Network (BNN), XGBoost and Random Forest (RF) to build prediction models. Sensitivity was selected to evaluate prediction results, and accuracy of what each algorithm above predicted under both the conditions with and without FHR was compared as well.
<br>Results: There were statically significant differences in the same type of features between ongoing pregnancy samples and EPL samples, which could serve as predictors. FHR, of which the normal development showed a strong correlation with ETD, had great predictive value for embryonic development. Among the six predictive models the one predicted with the highest accuracy was Random Forest, of which recall ratio and Fl could reach 97%, and AUC could reach 0.97, FHR taken into account as a feature. In addition, Random Forest had a higher prediction accuracy rate for samples with longer ETD its accuracy rate could reach 99% when predicting those at 10 weeks after embryo transfer.
<br>Conclusion: In this study, we established and compared six classification models to accurately predict EPL after the appearance of embryonic cardiac activity undergoing IVF-ET. Finally, Random Forest model outperformed the others. The implementation of Random Forest model in clinical environment can assist doctors to make clinical decisions. (C) 2020 Elsevier B.V. All rights reserved.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000580609200047</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>32623348</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
In vitro fertilization-embryo transfer; Machine learning; Fetal heart rate; Random forest</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
YOLK-SAC DIAMETER; ULTRASOUND; MODEL</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Liu, Lijue; Jiao, Yongxia; Shi, Danni] Cent South Univ, Sch Automat, Changsha 410083, Hunan, Peoples R China. <br>
[Liu, Lijue] Hunan Zixing Intelligent Med Technol Co Ltd, Changsha 410000, Hunan, Peoples R China. <br>
[Li, Xihong; Ouyang, Yan] Reprod &amp; Genet Hosp CITIC Xiangya, 84 Xiangya Rd, Changsha 410078, Hunan, Peoples R China. <br>
[Ouyang, Yan] Cent South Univ, Inst Reprod &amp; Stem Cell Engn, 84 Xiangya Rd, Changsha 410078, Peoples R China. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Li, XH (corresponding author), Reprod &amp; Genet Hosp CITIC Xiangya, 84 Xiangya Rd, Changsha 410078, Hunan, Peoples R China.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
2575031980@qq.com</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Central South University; Central South University; Central South University</td>
</tr>

<tr>
<td>
<b>Publisher:</b>

<value>ELSEVIER IRELAND LTD</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>ELSEVIER HOUSE, BROOKVALE PLAZA, EAST PARK SHANNON, CO, CLARE, 00000, IRELAND</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Computer Science, Interdisciplinary Applications; Computer Science, Theory &amp; Methods; Engineering, Biomedical; Medical Informatics</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Computer Science; Engineering; Medical Informatics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>OE5ZU</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0169-2607</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1872-7565</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>COMPUT METH PROG BIO</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Comput. Meth. Programs Biomed.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>8</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>China Postdoctoral Science Foundation</grant_agency>&nbsp;</td><td>
<div>2019M652778&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Science and Technology Project of Health and Family Planning Commission of Hunan Province</grant_agency>&nbsp;</td><td>
<div>C20180289&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Science and Technology Major Project of the Ministry of the Science and Technology of Hunan Province</grant_agency>&nbsp;</td><td>
<div>2017SK1030&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Reproductive Center and the Imaging Department of the Reproductive and Genetic Hospital of CITIC-Xiangya</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported in part by the China Postdoctoral Science Foundation under grant NO. 2019M652778, the Science and Technology Project of Health and Family Planning Commission of Hunan Province under grant NO. C20180289 and the Science and Technology Major Project of the Ministry of the Science and Technology of Hunan Province under grant NO. 2017SK1030. The first four authors contributed equally to the paper. We gratefully acknowledge the help and support provided by the Reproductive Center and the Imaging Department of the Reproductive and Genetic Hospital of CITIC-Xiangya.</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr>
<table xmlns:bean="http://ts.thomson.com/ua/bean">
<tr>
<td>End of File</td>
</tr>
</table><div style="background-color: #000000; padding: 10px; color: #FFFFFF; font-family: arial, sans-serif;">
<div style="height: 25px; width: 100px;">
<mat-icon data-mat-icon-name="clarivate" data-mat-icon-type="svg" aria-hidden="true" class="mat-icon notranslate clarivate-logo mat-icon-no-color" svgicon="clarivate" role="img">
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" focusable="false" preserveAspectRatio="xMidYMid meet" version="1.1" viewBox="0 0 92 18" height="100%" width="100%">
<title>Clarivate</title>
<g fill-rule="evenodd" fill="none" stroke-width="1" stroke="none">
<g fill-rule="nonzero" fill="#FFFFFF" transform="translate(-19.000000, -8.000000)">
<g transform="translate(1.000000, 1.000000)">
<g transform="translate(18.000000, 7.000000)">
<path d="M26.6812194,13.2997252 C28.2949641,13.3845068 29.7591419,12.3596036 30.2318671,10.8143235 L33.0910469,10.8143235 C32.3628779,13.742285 29.6973379,15.7705498 26.6812194,15.6916944 C24.8651813,15.7485215 23.1057028,15.0559504 21.8156554,13.7764967 C20.5256081,12.4970431 19.8185526,10.7433351 19.8604035,8.9268901 C19.8185367,7.11043445 20.5255848,5.35671042 21.8156337,4.07724261 C23.1056826,2.7977748 24.8651702,2.10519376 26.6812194,2.16201887 C29.6973451,2.0831702 32.3628861,4.11144761 33.0910469,7.03941847 L30.2318671,7.03941847 C29.7591419,5.49413828 28.2949641,4.46923515 26.6812194,4.55401674 C25.5407067,4.54133389 24.4470406,5.00703241 23.6657805,5.83803342 C22.8845203,6.66903442 22.487126,7.78932829 22.5700912,8.9268901 C22.4871313,10.064447 22.8845287,11.184734 23.6657888,12.0157277 C24.447049,12.8467215 25.540712,13.312413 26.6812194,13.2997252 L26.6812194,13.2997252 Z"/>
<polygon points="36.8477342 15.4674945 34.4371264 15.4674945 34.4371264 2.38628571 36.8477342 2.38628571"/>
<path d="M43.5380316,9.76781183 C43.5380316,9.15114738 43.0522397,8.75870193 42.1178663,8.75870193 C41.2070757,8.82048057 40.3236107,9.0957548 39.5389268,9.5622602 L38.5858286,7.73092465 C39.6930919,7.06514423 40.9567617,6.70409618 42.2486241,6.68441302 C44.1921648,6.68441302 45.9674886,7.58137515 45.9674886,9.84255787 L45.9674886,15.4674945 L43.5380316,15.4674945 L43.5380316,14.6639362 C42.9494098,15.3673061 42.0613957,15.7488608 41.146005,15.6917231 C40.3900371,15.7421039 39.645831,15.484584 39.0826798,14.9777443 C38.5195286,14.4709046 38.1853154,13.7578403 38.1560675,13.0007601 C38.1560675,11.3749188 39.3519468,10.3658089 41.2581433,10.3658089 L42.9400537,10.3658089 C43.3511665,10.3658089 43.5379934,10.1789247 43.5379934,9.86128266 L43.5380316,9.76781183 Z M43.5380316,12.4027631 L43.5380316,11.786089 C43.321275,11.9402969 43.0543042,12.0070182 42.790485,11.9729159 L41.8748173,11.9729159 C41.1086607,11.9729159 40.6602036,12.3092922 40.6602036,12.9072893 C40.6602036,13.486619 41.1272707,13.9164566 41.8748173,13.9164566 C42.303867,13.9699481 42.7348632,13.8332333 43.0546312,13.5422121 C43.3743992,13.2511908 43.5509896,12.8349402 43.5380316,12.4027631 L43.5380316,12.4027631 Z"/>
<path d="M52.6579104,9.3006299 L51.6301236,9.3006299 C50.3966703,9.3006299 49.8361037,9.8799596 49.8361037,11.3749188 L49.8361037,15.4674945 L47.4066179,15.4674945 L47.4066179,6.90863203 L49.7612715,6.90863203 L49.7612715,8.27284306 C50.1001881,7.34759017 51.0007831,6.74968637 51.9850429,6.79648425 C52.2115562,6.79362484 52.4375557,6.81874947 52.6579104,6.87128771 L52.6579104,9.3006299 Z"/>
<path d="M57.4232866,15.4674945 L54.9939445,15.4674945 L54.9939445,8.96425355 L53.7979312,8.96425355 L53.7979312,6.90863203 L57.4232866,6.90863203 L57.4232866,15.4674945 Z M56.0964104,2.38628347 C56.5129556,2.38558064 56.9126431,2.55074046 57.2071855,2.84528284 C57.5017278,3.13982521 57.6668877,3.53951271 57.6661848,3.95605794 C57.6609778,4.814152 56.9638874,5.50702694 56.1057775,5.50702694 C55.2476677,5.50702694 54.5505773,4.814152 54.5453585,3.95605794 C54.5417726,3.09491157 55.2352828,2.39302469 56.0964104,2.38628347 Z"/>
<polygon points="62.8802456 12.8512776 64.8236715 6.90863203 67.3838767 6.90863203 64.33787 15.4674945 61.3292077 15.4674945 58.4139588 6.90863203 61.0488526 6.90863203"/>
<path d="M72.8783522,9.76781183 C72.8783522,9.15114738 72.3924359,8.75870193 71.4580625,8.75870193 C70.5472784,8.82049943 69.6638208,9.09577238 68.8791325,9.5622602 L67.9261491,7.73092465 C69.0333627,7.06513461 70.2969953,6.70408494 71.5888202,6.68441302 C73.5323609,6.68441302 75.3076848,7.58137515 75.3076848,9.84255787 L75.3076848,15.4674945 L72.8783522,15.4674945 L72.8783522,14.6639362 C72.2896971,15.3672586 71.4017035,15.7487939 70.4863256,15.6916944 C69.7303436,15.7420941 68.986117,15.4845835 68.4229464,14.9777426 C67.8597758,14.4709017 67.5255471,13.7578265 67.4962924,13.0007314 C67.4962924,11.3748901 68.6923056,10.3657802 70.5984925,10.3657802 L72.2802785,10.3657802 C72.6913914,10.3657802 72.8783426,10.1788959 72.8783426,9.86125395 L72.8783522,9.76781183 Z M72.8783522,12.4027631 L72.8783522,11.786089 C72.6615874,11.9402794 72.3946236,12.006999 72.1308056,11.9729159 L71.2151378,11.9729159 C70.4488664,11.9729159 70.0004093,12.3092922 70.0004093,12.9072893 C70.0004093,13.486619 70.4676008,13.9164566 71.2151378,13.9164566 C71.6441833,13.9699279 72.0751679,13.8332061 72.394931,13.5421893 C72.7146941,13.2511724 72.8912904,12.8349341 72.8783522,12.4027631 L72.8783522,12.4027631 Z"/>
<path d="M79.9985637,3.65701635 L79.9985637,6.90863203 L81.8485762,6.90863203 L81.8485762,8.96425355 L79.9985637,8.96425355 L79.9985637,12.8325433 C79.9985637,13.3745286 80.4282865,13.4492747 80.8020646,13.4492747 C81.418729,13.4492747 81.9232553,13.3932056 81.9232553,13.3932056 L81.9232553,15.4674945 C81.9232553,15.4674945 80.7459859,15.5796423 80.0357932,15.5796423 C78.8211795,15.5796423 77.5691354,15.3740811 77.5691354,13.6548263 L77.5691354,8.96425355 L76.22363,8.96425355 L76.22363,6.90863203 L77.5691163,6.90863203 L77.5691163,3.65701635 L79.9985637,3.65701635 Z"/>
<path d="M91.5475359,11.8981794 L85.1751771,11.8981794 C85.3031552,12.7204351 85.9194738,13.3815958 86.7307322,13.5669136 C87.5419905,13.7522314 88.3842592,13.4242593 88.856573,12.7391298 L91.3793191,13.0007601 C90.4933306,14.9536735 88.3725856,16.0321989 86.2725384,15.5978628 C84.1724912,15.1635268 82.6535603,13.3322361 82.6149337,11.188092 C82.567807,9.98960603 83.0193435,8.82515843 83.8621298,7.97175307 C84.7049161,7.1183477 85.8636189,6.65226701 87.0626009,6.68438432 C89.4171397,6.68438432 91.5475359,8.21674525 91.5475359,11.8047373 L91.5475359,11.8981794 Z M85.1189837,10.2723955 L88.9499864,10.2723955 C88.8059978,9.34848922 87.9973156,8.67588427 87.0626392,8.70263282 C86.1147409,8.67034807 85.2869107,9.33893202 85.1189837,10.2723955 Z"/>
<path d="M12.6820274,9.85132226 C10.6885049,11.8556057 8.11575776,13.1830474 5.32707189,13.6462054 C5.77780891,14.8758828 6.40559556,16.0331959 7.19058498,17.0815575 C10.3903106,16.2989026 13.3004311,14.619851 15.5794509,12.2414352 C15.0651988,11.0374085 14.3787129,9.91449973 13.5415112,8.90790698 C13.271174,9.23328 12.9846793,9.54772625 12.6820274,9.85124571"/>
<path d="M5.32722498,13.6460332 C4.21304566,10.5852611 4.22627578,7.22758116 5.3645406,4.17568425 C4.07298567,3.95802358 2.75701252,3.92542943 1.45626578,4.07888372 C0.536506305,7.22972054 0.525090565,10.5762272 1.42333236,13.7332656 C2.72217338,13.8896343 4.03666286,13.8602685 5.32722498,13.6460524"/>
<path d="M7.25919787,0.751096346 C6.46424104,1.79423541 5.8259601,2.94792723 5.36457887,4.17561728 C8.13918123,4.64457901 10.6977043,5.96938232 12.6820178,7.96458738 C12.9850078,8.2673988 13.2715025,8.58184824 13.5415017,8.90793568 C14.3788161,7.90142183 15.0653154,6.77849641 15.5794509,5.57440744 C13.3171586,3.21319806 10.4324457,1.54091896 7.25919787,0.751096346"/>
</g>
</g>
</g>
</g>
</svg>
</mat-icon>
</div>
<path d="M5.32722498,13.6460332 C4.21304566,10.5852611 4.22627578,7.22758116 5.3645406,4.17568425 C4.07298567,3.95802358 2.75701252,3.92542943 1.45626578,4.07888372 C0.536506305,7.22972054 0.525090565,10.5762272 1.42333236,13.7332656 C2.72217338,13.8896343 4.03666286,13.8602685 5.32722498,13.6460524"></path>
<path d="M7.25919787,0.751096346 C6.46424104,1.79423541 5.8259601,2.94792723 5.36457887,4.17561728 C8.13918123,4.64457901 10.6977043,5.96938232 12.6820178,7.96458738 C12.9850078,8.2673988 13.2715025,8.58184824 13.5415017,8.90793568 C14.3788161,7.90142183 15.0653154,6.77849641 15.5794509,5.57440744 C13.3171586,3.21319806 10.4324457,1.54091896 7.25919787,0.751096346"></path>
</div>
