<div xmlns:bean="http://ts.thomson.com/ua/bean" style="background-color: #000000; padding: 10px; color: #FFFFFF; font-family:  arial, sans-serif;">
<div style="height: 25px; width: 100px;">
<mat-icon data-mat-icon-name="clarivate" data-mat-icon-type="svg" aria-hidden="true" class="mat-icon notranslate clarivate-logo mat-icon-no-color" svgicon="clarivate" role="img">
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" focusable="false" preserveAspectRatio="xMidYMid meet" version="1.1" viewBox="0 0 92 18" height="100%" width="100%">
<title>Clarivate</title>
<g fill-rule="evenodd" fill="none" stroke-width="1" stroke="none">
<g fill-rule="nonzero" fill="#FFFFFF" transform="translate(-19.000000, -8.000000)">
<g transform="translate(1.000000, 1.000000)">
<g transform="translate(18.000000, 7.000000)">
<path d="M26.6812194,13.2997252 C28.2949641,13.3845068 29.7591419,12.3596036 30.2318671,10.8143235 L33.0910469,10.8143235 C32.3628779,13.742285 29.6973379,15.7705498 26.6812194,15.6916944 C24.8651813,15.7485215 23.1057028,15.0559504 21.8156554,13.7764967 C20.5256081,12.4970431 19.8185526,10.7433351 19.8604035,8.9268901 C19.8185367,7.11043445 20.5255848,5.35671042 21.8156337,4.07724261 C23.1056826,2.7977748 24.8651702,2.10519376 26.6812194,2.16201887 C29.6973451,2.0831702 32.3628861,4.11144761 33.0910469,7.03941847 L30.2318671,7.03941847 C29.7591419,5.49413828 28.2949641,4.46923515 26.6812194,4.55401674 C25.5407067,4.54133389 24.4470406,5.00703241 23.6657805,5.83803342 C22.8845203,6.66903442 22.487126,7.78932829 22.5700912,8.9268901 C22.4871313,10.064447 22.8845287,11.184734 23.6657888,12.0157277 C24.447049,12.8467215 25.540712,13.312413 26.6812194,13.2997252 L26.6812194,13.2997252 Z"/>
<polygon points="36.8477342 15.4674945 34.4371264 15.4674945 34.4371264 2.38628571 36.8477342 2.38628571"/>
<path d="M43.5380316,9.76781183 C43.5380316,9.15114738 43.0522397,8.75870193 42.1178663,8.75870193 C41.2070757,8.82048057 40.3236107,9.0957548 39.5389268,9.5622602 L38.5858286,7.73092465 C39.6930919,7.06514423 40.9567617,6.70409618 42.2486241,6.68441302 C44.1921648,6.68441302 45.9674886,7.58137515 45.9674886,9.84255787 L45.9674886,15.4674945 L43.5380316,15.4674945 L43.5380316,14.6639362 C42.9494098,15.3673061 42.0613957,15.7488608 41.146005,15.6917231 C40.3900371,15.7421039 39.645831,15.484584 39.0826798,14.9777443 C38.5195286,14.4709046 38.1853154,13.7578403 38.1560675,13.0007601 C38.1560675,11.3749188 39.3519468,10.3658089 41.2581433,10.3658089 L42.9400537,10.3658089 C43.3511665,10.3658089 43.5379934,10.1789247 43.5379934,9.86128266 L43.5380316,9.76781183 Z M43.5380316,12.4027631 L43.5380316,11.786089 C43.321275,11.9402969 43.0543042,12.0070182 42.790485,11.9729159 L41.8748173,11.9729159 C41.1086607,11.9729159 40.6602036,12.3092922 40.6602036,12.9072893 C40.6602036,13.486619 41.1272707,13.9164566 41.8748173,13.9164566 C42.303867,13.9699481 42.7348632,13.8332333 43.0546312,13.5422121 C43.3743992,13.2511908 43.5509896,12.8349402 43.5380316,12.4027631 L43.5380316,12.4027631 Z"/>
<path d="M52.6579104,9.3006299 L51.6301236,9.3006299 C50.3966703,9.3006299 49.8361037,9.8799596 49.8361037,11.3749188 L49.8361037,15.4674945 L47.4066179,15.4674945 L47.4066179,6.90863203 L49.7612715,6.90863203 L49.7612715,8.27284306 C50.1001881,7.34759017 51.0007831,6.74968637 51.9850429,6.79648425 C52.2115562,6.79362484 52.4375557,6.81874947 52.6579104,6.87128771 L52.6579104,9.3006299 Z"/>
<path d="M57.4232866,15.4674945 L54.9939445,15.4674945 L54.9939445,8.96425355 L53.7979312,8.96425355 L53.7979312,6.90863203 L57.4232866,6.90863203 L57.4232866,15.4674945 Z M56.0964104,2.38628347 C56.5129556,2.38558064 56.9126431,2.55074046 57.2071855,2.84528284 C57.5017278,3.13982521 57.6668877,3.53951271 57.6661848,3.95605794 C57.6609778,4.814152 56.9638874,5.50702694 56.1057775,5.50702694 C55.2476677,5.50702694 54.5505773,4.814152 54.5453585,3.95605794 C54.5417726,3.09491157 55.2352828,2.39302469 56.0964104,2.38628347 Z"/>
<polygon points="62.8802456 12.8512776 64.8236715 6.90863203 67.3838767 6.90863203 64.33787 15.4674945 61.3292077 15.4674945 58.4139588 6.90863203 61.0488526 6.90863203"/>
<path d="M72.8783522,9.76781183 C72.8783522,9.15114738 72.3924359,8.75870193 71.4580625,8.75870193 C70.5472784,8.82049943 69.6638208,9.09577238 68.8791325,9.5622602 L67.9261491,7.73092465 C69.0333627,7.06513461 70.2969953,6.70408494 71.5888202,6.68441302 C73.5323609,6.68441302 75.3076848,7.58137515 75.3076848,9.84255787 L75.3076848,15.4674945 L72.8783522,15.4674945 L72.8783522,14.6639362 C72.2896971,15.3672586 71.4017035,15.7487939 70.4863256,15.6916944 C69.7303436,15.7420941 68.986117,15.4845835 68.4229464,14.9777426 C67.8597758,14.4709017 67.5255471,13.7578265 67.4962924,13.0007314 C67.4962924,11.3748901 68.6923056,10.3657802 70.5984925,10.3657802 L72.2802785,10.3657802 C72.6913914,10.3657802 72.8783426,10.1788959 72.8783426,9.86125395 L72.8783522,9.76781183 Z M72.8783522,12.4027631 L72.8783522,11.786089 C72.6615874,11.9402794 72.3946236,12.006999 72.1308056,11.9729159 L71.2151378,11.9729159 C70.4488664,11.9729159 70.0004093,12.3092922 70.0004093,12.9072893 C70.0004093,13.486619 70.4676008,13.9164566 71.2151378,13.9164566 C71.6441833,13.9699279 72.0751679,13.8332061 72.394931,13.5421893 C72.7146941,13.2511724 72.8912904,12.8349341 72.8783522,12.4027631 L72.8783522,12.4027631 Z"/>
<path d="M79.9985637,3.65701635 L79.9985637,6.90863203 L81.8485762,6.90863203 L81.8485762,8.96425355 L79.9985637,8.96425355 L79.9985637,12.8325433 C79.9985637,13.3745286 80.4282865,13.4492747 80.8020646,13.4492747 C81.418729,13.4492747 81.9232553,13.3932056 81.9232553,13.3932056 L81.9232553,15.4674945 C81.9232553,15.4674945 80.7459859,15.5796423 80.0357932,15.5796423 C78.8211795,15.5796423 77.5691354,15.3740811 77.5691354,13.6548263 L77.5691354,8.96425355 L76.22363,8.96425355 L76.22363,6.90863203 L77.5691163,6.90863203 L77.5691163,3.65701635 L79.9985637,3.65701635 Z"/>
<path d="M91.5475359,11.8981794 L85.1751771,11.8981794 C85.3031552,12.7204351 85.9194738,13.3815958 86.7307322,13.5669136 C87.5419905,13.7522314 88.3842592,13.4242593 88.856573,12.7391298 L91.3793191,13.0007601 C90.4933306,14.9536735 88.3725856,16.0321989 86.2725384,15.5978628 C84.1724912,15.1635268 82.6535603,13.3322361 82.6149337,11.188092 C82.567807,9.98960603 83.0193435,8.82515843 83.8621298,7.97175307 C84.7049161,7.1183477 85.8636189,6.65226701 87.0626009,6.68438432 C89.4171397,6.68438432 91.5475359,8.21674525 91.5475359,11.8047373 L91.5475359,11.8981794 Z M85.1189837,10.2723955 L88.9499864,10.2723955 C88.8059978,9.34848922 87.9973156,8.67588427 87.0626392,8.70263282 C86.1147409,8.67034807 85.2869107,9.33893202 85.1189837,10.2723955 Z"/>
<path d="M12.6820274,9.85132226 C10.6885049,11.8556057 8.11575776,13.1830474 5.32707189,13.6462054 C5.77780891,14.8758828 6.40559556,16.0331959 7.19058498,17.0815575 C10.3903106,16.2989026 13.3004311,14.619851 15.5794509,12.2414352 C15.0651988,11.0374085 14.3787129,9.91449973 13.5415112,8.90790698 C13.271174,9.23328 12.9846793,9.54772625 12.6820274,9.85124571"/>
<path d="M5.32722498,13.6460332 C4.21304566,10.5852611 4.22627578,7.22758116 5.3645406,4.17568425 C4.07298567,3.95802358 2.75701252,3.92542943 1.45626578,4.07888372 C0.536506305,7.22972054 0.525090565,10.5762272 1.42333236,13.7332656 C2.72217338,13.8896343 4.03666286,13.8602685 5.32722498,13.6460524"/>
<path d="M7.25919787,0.751096346 C6.46424104,1.79423541 5.8259601,2.94792723 5.36457887,4.17561728 C8.13918123,4.64457901 10.6977043,5.96938232 12.6820178,7.96458738 C12.9850078,8.2673988 13.2715025,8.58184824 13.5415017,8.90793568 C14.3788161,7.90142183 15.0653154,6.77849641 15.5794509,5.57440744 C13.3171586,3.21319806 10.4324457,1.54091896 7.25919787,0.751096346"/>
</g>
</g>
</g>
</g>
</svg>
</mat-icon>
</div>
<path d="M5.32722498,13.6460332 C4.21304566,10.5852611 4.22627578,7.22758116 5.3645406,4.17568425 C4.07298567,3.95802358 2.75701252,3.92542943 1.45626578,4.07888372 C0.536506305,7.22972054 0.525090565,10.5762272 1.42333236,13.7332656 C2.72217338,13.8896343 4.03666286,13.8602685 5.32722498,13.6460524"></path>
<path d="M7.25919787,0.751096346 C6.46424104,1.79423541 5.8259601,2.94792723 5.36457887,4.17561728 C8.13918123,4.64457901 10.6977043,5.96938232 12.6820178,7.96458738 C12.9850078,8.2673988 13.2715025,8.58184824 13.5415017,8.90793568 C14.3788161,7.90142183 15.0653154,6.77849641 15.5794509,5.57440744 C13.3171586,3.21319806 10.4324457,1.54091896 7.25919787,0.751096346"></path>
</div><div style="width: 180px; height: 40px; padding: 10px;">
<mat-icon data-mat-icon-name="wos" data-mat-icon-type="svg" aria-hidden="true" style="width: 180px;" class="mat-icon notranslate mat-icon-no-color" svgicon="wos" role="img">
<svg xmlns="http://www.w3.org/2000/svg" focusable="false" preserveAspectRatio="xMidYMid meet" viewBox="0 0 1868 332" height="100%" width="100%">
<title>Web of Science</title>
<path d="M 108.92,75.00            C 108.92,75.00 147.34,75.00 147.34,75.00              147.34,75.00 182.84,208.12 182.84,208.12              182.84,208.12 183.11,208.12 183.11,208.12              183.11,208.12 212.80,75.00 212.80,75.00              212.80,75.00 250.42,75.00 250.42,75.00              250.42,75.00 205.11,261.00 205.11,261.00              205.11,261.00 162.44,261.00 162.44,261.00              162.44,261.00 127.47,127.34 127.47,127.34              127.47,127.34 93.28,261.00 93.28,261.00              93.28,261.00 50.61,261.00 50.61,261.00              50.61,261.00 4.77,75.00 4.77,75.00              4.77,75.00 44.52,75.00 44.52,75.00              44.52,75.00 73.14,208.12 73.14,208.12              73.14,208.12 73.41,208.12 73.41,208.12              73.41,208.12 108.92,75.00 108.92,75.00 Z            M 379.00,210.00            C 379.00,210.00 288.41,210.00 288.41,210.00              289.29,216.78 292.30,222.48 297.44,227.09              302.57,231.70 308.86,234.00 316.30,234.00              327.80,234.00 335.95,230.00 340.73,222.00              340.73,222.00 376.61,225.58 376.61,225.58              371.30,238.39 363.32,248.00 352.69,254.41              342.06,260.80 329.75,264.00 315.77,264.00              297.52,264.00 282.32,257.98 270.19,245.94              258.06,233.90 252.00,218.59 252.00,200.00              252.00,181.42 257.98,166.11 269.94,154.06              281.89,142.02 296.99,136.00 315.23,136.00              333.47,136.00 348.66,142.21 360.80,154.64              372.93,167.06 379.00,185.51 379.00,210.00 Z            M 333.30,171.28            C 328.34,167.09 322.32,165.00 315.23,165.00              308.15,165.00 302.12,167.01 297.16,171.03              292.20,175.04 289.02,180.37 287.61,187.00              287.61,187.00 342.06,187.00 342.06,187.00              341.19,180.71 338.27,175.47 333.30,171.28 Z            M 433.72,245.31            C 433.72,245.31 433.72,261.00 433.72,261.00              433.72,261.00 399.00,261.00 399.00,261.00              399.00,261.00 399.00,75.00 399.00,75.00              399.00,75.00 433.45,75.00 433.45,75.00              433.45,75.00 433.45,134.94 433.45,134.94              433.45,137.95 433.45,141.85 433.45,146.64              433.45,151.42 433.45,154.17 433.45,154.88              436.43,149.39 441.34,144.87 448.17,141.33              455.02,137.78 462.82,136.00 471.59,136.00              488.26,136.00 502.16,142.07 513.30,154.20              524.43,166.33 530.00,181.59 530.00,200.00              530.00,218.41 524.37,233.68 513.11,245.81              501.86,257.94 487.88,264.00 471.17,264.00              462.91,264.00 455.39,262.22 448.62,258.66              441.85,255.09 436.88,250.65 433.72,245.31 Z            M 485.39,176.23            C 479.65,169.98 472.34,166.86 463.45,166.86              454.58,166.86 447.27,169.98 441.53,176.23              435.79,182.48 432.92,190.37 432.92,199.88              432.92,209.20 435.79,217.07 441.53,223.50              447.27,229.93 454.58,233.14 463.45,233.14              472.34,233.14 479.65,229.93 485.39,223.50              491.13,217.07 494.00,209.20 494.00,199.88              494.00,190.37 491.13,182.48 485.39,176.23 Z            M 707.69,245.81            C 695.19,257.94 679.86,264.00 661.72,264.00              643.58,264.00 628.26,257.94 615.75,245.81              603.25,233.68 597.00,218.41 597.00,200.00              597.00,181.59 603.25,166.33 615.75,154.20              628.26,142.07 643.58,136.00 661.72,136.00              679.86,136.00 695.19,142.07 707.69,154.20              720.20,166.33 726.45,181.59 726.45,200.00              726.45,218.41 720.20,233.68 707.69,245.81 Z            M 661.84,166.86            C 652.85,166.86 645.54,170.00 639.91,176.28              634.27,182.55 631.45,190.46 631.45,200.00              631.45,209.54 634.27,217.45 639.91,223.73              645.54,230.01 652.85,233.14 661.84,233.14              670.66,233.14 677.89,230.01 683.53,223.73              689.18,217.45 692.00,209.54 692.00,200.00              692.00,190.46 689.18,182.55 683.53,176.28              677.89,170.00 670.66,166.86 661.84,166.86 Z            M 757.00,139.00            C 757.00,139.00 757.00,104.34 757.00,104.34              757.00,93.15 760.17,84.98 766.52,79.83              772.86,74.67 781.14,72.09 791.36,72.09              791.36,72.09 816.50,73.69 816.50,73.69              816.50,73.69 816.50,105.33 816.50,105.33              809.47,104.45 804.72,104.00 802.27,104.00              798.58,104.00 795.85,104.71 794.09,106.14              792.33,107.57 791.45,110.24 791.45,114.16              791.45,114.16 791.45,139.00 791.45,139.00              791.45,139.00 815.00,139.00 815.00,139.00              815.00,139.00 815.00,168.00 815.00,168.00              815.00,168.00 791.45,168.00 791.45,168.00              791.45,168.00 791.45,260.50 791.45,260.50              791.45,260.50 757.00,260.50 757.00,260.50              757.00,260.50 757.00,168.00 757.00,168.00              757.00,168.00 738.00,168.00 738.00,168.00              738.00,168.00 738.00,139.00 738.00,139.00              738.00,139.00 757.00,139.00 757.00,139.00 Z            M 1004.30,97.27            C 1004.30,97.27 983.11,124.06 983.11,124.06              969.92,111.36 956.73,105.00 943.53,105.00              937.11,105.00 931.59,106.60 926.95,109.80              922.32,112.98 920.00,117.24 920.00,122.56              920.00,131.06 926.74,138.42 940.22,144.62              940.22,144.62 955.66,151.80 955.66,151.80              971.60,159.25 983.48,167.31 991.28,176.00              999.09,184.68 1003.00,195.67 1003.00,208.97              1003.00,223.87 997.13,236.76 985.39,247.66              973.66,258.55 958.09,264.00 938.67,264.00              913.45,264.00 891.31,254.56 872.25,235.67              872.25,235.67 892.52,208.02 892.52,208.02              899.27,214.90 906.74,220.29 914.92,224.17              923.10,228.06 930.83,230.00 938.11,230.00              945.94,230.00 952.21,228.09 956.92,224.28              961.64,220.47 964.00,215.64 964.00,209.80              964.00,203.95 961.65,198.93 956.95,194.77              952.25,190.60 944.85,186.21 934.73,181.61              934.73,181.61 921.70,175.48 921.70,175.48              894.57,162.73 881.00,145.36 881.00,123.38              881.00,107.59 887.05,94.92 899.17,85.36              911.29,75.79 925.91,71.00 943.06,71.00              963.93,71.00 984.34,79.76 1004.30,97.27 Z            M 1082.92,166.86            C 1073.91,166.86 1066.58,170.00 1060.92,176.28              1055.27,182.55 1052.45,190.46 1052.45,200.00              1052.45,209.54 1055.27,217.45 1060.92,223.73              1066.58,230.01 1073.91,233.14 1082.92,233.14              1095.82,233.14 1104.74,226.95 1109.69,214.58              1109.69,214.58 1142.02,224.66 1142.02,224.66              1138.13,236.72 1130.75,246.29 1119.89,253.38              1109.02,260.46 1096.61,264.00 1082.66,264.00              1064.46,264.00 1049.13,257.94 1036.67,245.81              1024.22,233.68 1018.00,218.41 1018.00,200.00              1018.00,181.59 1024.22,166.33 1036.67,154.20              1049.13,142.07 1064.46,136.00 1082.66,136.00              1096.61,136.00 1109.02,139.59 1119.89,146.77              1130.75,153.95 1138.13,163.56 1142.02,175.61              1142.02,175.61 1109.69,185.42 1109.69,185.42              1107.39,179.77 1103.91,175.26 1099.22,171.91              1094.54,168.54 1089.11,166.86 1082.92,166.86 Z            M 1170.31,112.67            C 1166.10,108.45 1164.00,103.27 1164.00,97.12              1164.00,90.98 1166.15,85.76 1170.45,81.45              1174.75,77.15 1179.89,75.00 1185.86,75.00              1192.02,75.00 1197.24,77.15 1201.55,81.45              1205.85,85.76 1208.00,90.98 1208.00,97.12              1208.00,103.27 1205.85,108.45 1201.55,112.67              1197.24,116.89 1192.02,119.00 1185.86,119.00              1179.71,119.00 1174.53,116.89 1170.31,112.67 Z            M 1205.00,139.00            C 1205.00,139.00 1205.00,261.00 1205.00,261.00              1205.00,261.00 1170.55,261.00 1170.55,261.00              1170.55,261.00 1170.55,168.00 1170.55,168.00              1170.55,168.00 1153.50,168.00 1153.50,168.00              1153.50,168.00 1153.50,139.00 1153.50,139.00              1153.50,139.00 1205.00,139.00 1205.00,139.00 Z            M 1353.00,210.00            C 1353.00,210.00 1262.41,210.00 1262.41,210.00              1263.29,216.78 1266.30,222.48 1271.44,227.09              1276.57,231.70 1282.86,234.00 1290.30,234.00              1301.80,234.00 1309.95,230.00 1314.73,222.00              1314.73,222.00 1350.61,225.58 1350.61,225.58              1345.30,238.39 1337.32,248.00 1326.69,254.41              1316.06,260.80 1303.75,264.00 1289.77,264.00              1271.52,264.00 1256.32,257.98 1244.19,245.94              1232.06,233.90 1226.00,218.59 1226.00,200.00              1226.00,181.42 1231.98,166.11 1243.94,154.06              1255.89,142.02 1270.99,136.00 1289.23,136.00              1307.47,136.00 1322.66,142.21 1334.80,154.64              1346.93,167.06 1353.00,185.51 1353.00,210.00 Z            M 1307.30,171.28            C 1302.34,167.09 1296.32,165.00 1289.23,165.00              1282.15,165.00 1276.12,167.01 1271.16,171.03              1266.20,175.04 1263.02,180.37 1261.61,187.00              1261.61,187.00 1316.06,187.00 1316.06,187.00              1315.19,180.71 1312.27,175.47 1307.30,171.28 Z            M 1406.39,153.36            C 1415.92,141.79 1428.27,136.00 1443.45,136.00              1456.87,136.00 1467.72,140.04 1476.02,148.11              1484.30,156.17 1488.45,167.20 1488.45,181.20              1488.45,181.20 1488.45,261.00 1488.45,261.00              1488.45,261.00 1454.00,261.00 1454.00,261.00              1454.00,261.00 1454.00,189.86 1454.00,189.86              1454.00,182.57 1452.00,177.11 1448.00,173.47              1444.01,169.82 1439.09,168.00 1433.25,168.00              1423.50,168.00 1414.90,172.62 1407.45,181.86              1407.45,181.86 1407.45,261.00 1407.45,261.00              1407.45,261.00 1373.00,261.00 1373.00,261.00              1373.00,261.00 1373.00,139.00 1373.00,139.00              1373.00,139.00 1406.39,139.00 1406.39,139.00              1406.39,139.00 1406.39,153.36 1406.39,153.36 Z            M 1573.92,166.86            C 1564.91,166.86 1557.58,170.00 1551.92,176.28              1546.27,182.55 1543.45,190.46 1543.45,200.00              1543.45,209.54 1546.27,217.45 1551.92,223.73              1557.58,230.01 1564.91,233.14 1573.92,233.14              1586.82,233.14 1595.74,226.95 1600.69,214.58              1600.69,214.58 1633.02,224.66 1633.02,224.66              1629.13,236.72 1621.75,246.29 1610.89,253.38              1600.02,260.46 1587.61,264.00 1573.66,264.00              1555.46,264.00 1540.13,257.94 1527.67,245.81              1515.22,233.68 1509.00,218.41 1509.00,200.00              1509.00,181.59 1515.22,166.33 1527.67,154.20              1540.13,142.07 1555.46,136.00 1573.66,136.00              1587.61,136.00 1600.02,139.59 1610.89,146.77              1621.75,153.95 1629.13,163.56 1633.02,175.61              1633.02,175.61 1600.69,185.42 1600.69,185.42              1598.39,179.77 1594.91,175.26 1590.22,171.91              1585.54,168.54 1580.11,166.86 1573.92,166.86 Z            M 1773.00,210.00            C 1773.00,210.00 1682.41,210.00 1682.41,210.00              1683.29,216.78 1686.30,222.48 1691.44,227.09              1696.57,231.70 1702.86,234.00 1710.30,234.00              1721.80,234.00 1729.95,230.00 1734.73,222.00              1734.73,222.00 1770.61,225.58 1770.61,225.58              1765.30,238.39 1757.32,248.00 1746.69,254.41              1736.06,260.80 1723.75,264.00 1709.77,264.00              1691.52,264.00 1676.32,257.98 1664.19,245.94              1652.06,233.90 1646.00,218.59 1646.00,200.00              1646.00,181.42 1651.98,166.11 1663.94,154.06              1675.89,142.02 1690.99,136.00 1709.23,136.00              1727.47,136.00 1742.66,142.21 1754.80,154.64              1766.93,167.06 1773.00,185.51 1773.00,210.00 Z            M 1727.30,171.28            C 1722.34,167.09 1716.32,165.00 1709.23,165.00              1702.15,165.00 1696.12,167.01 1691.16,171.03              1686.20,175.04 1683.02,180.37 1681.61,187.00              1681.61,187.00 1736.06,187.00 1736.06,187.00              1735.19,180.71 1732.27,175.47 1727.30,171.28 Z" stroke-width="1" stroke="black" fill="black"/>
<path d="M 1769.00,76.00            C 1769.00,76.00 1800.00,76.00 1800.00,76.00              1800.00,76.00 1800.00,82.42 1800.00,82.42              1800.00,82.42 1788.00,82.42 1788.00,82.42              1788.00,82.42 1788.00,115.00 1788.00,115.00              1788.00,115.00 1780.84,115.00 1780.84,115.00              1780.84,115.00 1780.84,82.42 1780.84,82.42              1780.84,82.42 1769.00,82.42 1769.00,82.42              1769.00,82.42 1769.00,76.00 1769.00,76.00 Z            M 1827.27,106.64            C 1827.27,106.64 1818.95,106.64 1818.95,106.64              1818.95,106.64 1811.16,87.42 1811.16,87.42              1811.16,87.42 1811.16,115.00 1811.16,115.00              1811.16,115.00 1804.00,115.00 1804.00,115.00              1804.00,115.00 1804.00,76.00 1804.00,76.00              1804.00,76.00 1814.61,76.00 1814.61,76.00              1814.61,76.00 1823.25,98.11 1823.25,98.11              1823.25,98.11 1832.08,76.00 1832.08,76.00              1832.08,76.00 1842.00,76.00 1842.00,76.00              1842.00,76.00 1842.00,115.00 1842.00,115.00              1842.00,115.00 1834.84,115.00 1834.84,115.00              1834.84,115.00 1834.84,87.42 1834.84,87.42              1834.84,87.42 1827.27,106.64 1827.27,106.64 Z" stroke-width="1" stroke="black" fill="black"/>
</svg>
</mat-icon>
</div><table cellspacing="0" cellpadding="2" border="0">
<tr>
<td>32 record(s) printed from Clarivate Web of Science</td>
</tr>
</table><hr>
<table>
<tr>
<td><b>Record 1 of 32</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Self-supervised VICReg pre-training for Brugada ECG detection</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Ronan, R (Ronan, Robert); Tarabanis, C (Tarabanis, Constantine); Chinitz, L (Chinitz, Larry); Jankelson, L (Jankelson, Lior)</td>
</tr>

<tr xmlns:bean="http://ts.thomson.com/ua/bean" xmlns:exsl="http://exslt.org/common" xmlns:set="http://exslt.org/sets">
<td><b>Source:</b> 
<value>SCIENTIFIC REPORTS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>15</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>1</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>9396</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1038/s41598-025-94130-x</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2025 MAR 19</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>1</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>1</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
1</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
1</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>18</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Existing deep learning algorithms for electrocardiogram (ECG) classification rely on supervised training approaches requiring large volumes of reliably labeled data. This limits their applicability to rare cardiac diseases like Brugada syndrome (BrS), often lacking accurately labeled ECG examples. To address labeled data constraints and the resulting limitations of supervised training approaches, we developed a novel deep learning model for BrS ECG classification using the Variance-Invariance-Covariance Regularization (VICReg) architecture for self-supervised pre-training. The VICReg model outperformed a state-of-the-art neural network in all calculated metrics, achieving an area under the receiver operating and precision-recall curves of 0.88 and 0.82, respectively. We used the VICReg model to identify missed BrS cases and hence refine the previously underestimated institutional BrS prevalence and patient outcomes. Our results provide a novel approach to rare cardiac disease identification and challenge existing BrS prevalence estimates offering a framework for other rare cardiac conditions.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001447240300001</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>40102504</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Brugada syndrome; Deep learning; Artificial intelligence-enabled electrocardiogram; VICReg pre-training</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
ELECTROCARDIOGRAM; PREVALENCE</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Ronan, Robert; Tarabanis, Constantine; Chinitz, Larry; Jankelson, Lior] NYU, Leon H Charney Div Cardiol, Cardiac Electrophysiol, NYU Langone Hlth,Sch Med, New York, NY 10016 USA. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Jankelson, L (corresponding author), NYU, Leon H Charney Div Cardiol, Cardiac Electrophysiol, NYU Langone Hlth,Sch Med, New York, NY 10016 USA.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
Lior.jankelson@nyulangone.org</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
NYU Langone Medical Center; New York University</td>
</tr>

<tr>
<td>
<b>Publisher:</b>

<value>NATURE PORTFOLIO</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>HEIDELBERGER PLATZ 3, BERLIN, 14197, GERMANY</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Multidisciplinary Sciences</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Science &amp; Technology - Other Topics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>0HC9Y</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2045-2322</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>SCI REP-UK</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Sci Rep</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>7</value>
</td>
</tr>

<tr>
<td>
<b>Open Access:</b>

<value>gold, Green Submitted</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 2 of 32</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Temporal and spatial self supervised learning methods for electrocardiograms</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Chen, WP (Chen, Wenping); Wang, HB (Wang, Huibin); Zhang, LL (Zhang, Lili); Zhang, M (Zhang, Min)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>SCIENTIFIC REPORTS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>15</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>1</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>6029</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1038/s41598-025-90084-2</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2025 FEB 19</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>0</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>0</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
11</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
18</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>57</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
The limited availability of labeled ECG data restricts the application of supervised deep learning methods in ECG detection. Although existing self-supervised learning approaches have been applied to ECG analysis, they are predominantly image-based, which limits their effectiveness. To address these limitations and provide novel insights, we propose a Temporal-Spatial Self-Supervised Learning (TSSL) method specifically designed for ECG detection. TSSL leverages the intrinsic temporal and spatial characteristics of ECG signals to enhance feature representation. Temporally, ECG signals retain consistent identity information over time, enabling the model to generate stable representations for the same individual across different time points while isolating representations of different leads to preserve their unique features. Spatially, ECG signals from various leads capture the heart's activity from different perspectives, revealing both commonalities and distinct patterns. TSSL captures these correlations by maintaining consistency in the relationships between signals and their representations across different leads. Experimental results on the CPSC2018, Chapman, and PTB-XL databases demonstrate that TSSL introduces new capabilities by effectively utilizing temporal and spatial information, achieving superior performance compared to existing methods and approaching the performance of full-label training with only 10% of the labeled data. This highlights TSSL's ability to provide deeper insights and enhanced feature extraction beyond mere performance improvements. We make our code publicly available on https://github.com/cwp9731/temporal-spatial-self-supervised-learning.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001426697000032</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>39972080</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Deep learning; Self-supervised learning; Electrocardiogram; Representation extraction</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
ARRHYTHMIA DETECTION; RHYTHM</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Chen, Wenping; Zhang, Min] Hohai Univ, Coll Informat Sci &amp; Engn, Nanjing 211100, Peoples R China. <br>
[Wang, Huibin; Zhang, Lili] Hohai Univ, Coll Comp Sci &amp; Software Engn, Nanjing 211100, Peoples R China. <br>
[Zhang, Min] Gannan Univ Sci &amp; Technol, Coll Informat Engn, Ganzhou 341000, Peoples R China. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Wang, HB (corresponding author), Hohai Univ, Coll Comp Sci &amp; Software Engn, Nanjing 211100, Peoples R China.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
hbwang@hhu.edu.cn</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Hohai University; Hohai University; Gannan University of Science &amp; Technology</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Chen, Wenping</display_name>&nbsp;</td><td>KYR-4086-2024&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>NATURE PORTFOLIO</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>HEIDELBERGER PLATZ 3, BERLIN, 14197, GERMANY</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Multidisciplinary Sciences</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Science &amp; Technology - Other Topics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>X6V3S</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2045-2322</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>SCI REP-UK</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Sci Rep</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>19</value>
</td>
</tr>

<tr>
<td>
<b>Open Access:</b>

<value>gold, Green Submitted</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 3 of 32</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Detection of maternal and fetal stress from the electrocardiogram with self-supervised representation learning</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Sarkar, P (Sarkar, Pritam); Lobmaier, S (Lobmaier, Silvia); Fabre, B (Fabre, Bibiana); Gonz&aacute;lez, D (Gonzalez, Diego); Mueller, A (Mueller, Alexander); Frasch, MG (Frasch, Martin G.); Antonelli, MC (Antonelli, Marta C.); Etemad, A (Etemad, Ali)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>SCIENTIFIC REPORTS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>11</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>1</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>24146</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1038/s41598-021-03376-8</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2021 DEC 17</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>20</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>27</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
1</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
8</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>29</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
In the pregnant mother and her fetus, chronic prenatal stress results in entrainment of the fetal heartbeat by the maternal heartbeat, quantified by the fetal stress index (FSI). Deep learning (DL) is capable of pattern detection in complex medical data with high accuracy in noisy real-life environments, but little is known about DL's utility in non-invasive biometric monitoring during pregnancy. A recently established self-supervised learning (SSL) approach to DL provides emotional recognition from electrocardiogram (ECG). We hypothesized that SSL will identify chronically stressed mother-fetus dyads from the raw maternal abdominal electrocardiograms (aECG), containing fetal and maternal ECG. Chronically stressed mothers and controls matched at enrolment at 32 weeks of gestation were studied. We validated the chronic stress exposure by psychological inventory, maternal hair cortisol and FSI. We tested two variants of SSL architecture, one trained on the generic ECG features for emotional recognition obtained from public datasets and another transfer-learned on a subset of our data. Our DL models accurately detect the chronic stress exposure group (AUROC = 0.982 +/- 0.002), the individual psychological stress score (R2 = 0.943 +/- 0.009) and FSI at 34 weeks of gestation (R2 = 0.946 +/- 0.013), as well as the maternal hair cortisol at birth reflecting chronic stress exposure (0.931 +/- 0.006). The best performance was achieved with the DL model trained on the public dataset and using maternal ECG alone. The present DL approach provides a novel source of physiological insights into complex multi-modal relationships between different regulatory systems exposed to chronic stress. The final DL model can be deployed in low-cost regular ECG biosensors as a simple, ubiquitous early stress detection and monitoring tool during pregnancy. This discovery should enable early behavioral interventions.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000731322900001</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>34921162</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
HAIR</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Sarkar, Pritam; Etemad, Ali] Queens Univ, Dept ECE, Kingston, ON, Canada. <br>
[Sarkar, Pritam; Etemad, Ali] Queens Univ, Ingenu Labs Res Inst, Kingston, ON, Canada. <br>
[Lobmaier, Silvia; Antonelli, Marta C.] Tech Univ Munich, Dept Obstet &amp; Gynecol, Munich, Germany. <br>
[Frasch, Martin G.] Univ Washington, Dept Obstet &amp; Gynaecol, Seattle, WA 98195 USA. <br>
[Frasch, Martin G.] Univ Washington, Ctr Human Dev &amp; Disabil, Seattle, WA 98195 USA. <br>
[Fabre, Bibiana; Gonzalez, Diego] Univ Buenos Aires, Fac Farm &amp; Bioquim, Buenos Aires, DF, Argentina. <br>
[Mueller, Alexander] Tech Univ Munich, Dept Cardiol, Munich, Germany. <br>
[Antonelli, Marta C.] Univ Buenos Aires, Fac Med, IBCN, Buenos Aires, DF, Argentina. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Etemad, A (corresponding author), Queens Univ, Dept ECE, Kingston, ON, Canada.<br>Etemad, A (corresponding author), Queens Univ, Ingenu Labs Res Inst, Kingston, ON, Canada.<br>Lobmaier, S; Antonelli, MC (corresponding author), Tech Univ Munich, Dept Obstet &amp; Gynecol, Munich, Germany.<br>Frasch, MG (corresponding author), Univ Washington, Dept Obstet &amp; Gynaecol, Seattle, WA 98195 USA.<br>Frasch, MG (corresponding author), Univ Washington, Ctr Human Dev &amp; Disabil, Seattle, WA 98195 USA.<br>Antonelli, MC (corresponding author), Univ Buenos Aires, Fac Med, IBCN, Buenos Aires, DF, Argentina.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
silvia.lobmaier@tum.de; mfrasch@uw.edu; martantonelli@gmail.com; ali.etemad@queensu.ca</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Queens University - Canada; Ingenuity Labs Research Institute; Queens University - Canada; Technical University of Munich; University of Washington; University of Washington Seattle; University of Washington; University of Washington Seattle; University of Buenos Aires; Technical University of Munich; University of Buenos Aires</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Lobmaier, Silvia</display_name>&nbsp;</td><td>AAY-7515-2020&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Mueller, Alexander</display_name>&nbsp;</td><td>C-5995-2017&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Gonzalez, Diego</display_name>&nbsp;</td><td>M-6746-2014&nbsp;</td><td>0000-0001-9670-4827&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Frasch, Martin</display_name>&nbsp;</td><td>AAN-8507-2020&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Antonelli, Marta Cristina</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-7785-8748&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Sarkar, Pritam</display_name>&nbsp;</td><td>JEF-1544-2023&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>NATURE PORTFOLIO</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>HEIDELBERGER PLATZ 3, BERLIN, 14197, GERMANY</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Multidisciplinary Sciences</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Science &amp; Technology - Other Topics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>XQ1OS</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2045-2322</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>SCI REP-UK</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Sci Rep</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>10</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Hans Fischer Senior Fellowship</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>We gratefully acknowledge the contribution of Dr. Hau-Tieng Wu lab with SAVER code-driven mECG/fECG extraction. The project was developed and performed by own resources of Frauenklinik/Klinikum rechts der Isar and funding from Hans Fischer Senior Fellowship to MCA.</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>Green Submitted, gold</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 4 of 32</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Ontology-guided machine learning outperforms zero-shot foundation models for cardiac ultrasound text reports</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Subramaniam, S (Subramaniam, Suganya); Rizvi, S (Rizvi, Sara); Ramesh, R (Ramesh, Ramya); Sehgal, V (Sehgal, Vibhor); Gurusamy, B (Gurusamy, Brinda); Arif, H (Arif, Hikmatullah); Tran, J (Tran, Jeffrey); Thamman, R (Thamman, Ritu); Anyanwu, EC (Anyanwu, Emeka C.); Mastouri, R (Mastouri, Ronald); Mackensen, GB (Mackensen, G. Burkhard); Arnaout, R (Arnaout, Rima)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>SCIENTIFIC REPORTS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>15</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>1</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>5456</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1038/s41598-024-83540-y</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2025 FEB 14</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>2</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>2</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
5</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
5</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>24</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Big data can revolutionize research and quality improvement for cardiac ultrasound. Text reports are a critical part of such analyses. Cardiac ultrasound reports include structured and free text and vary across institutions, hampering attempts to mine text for useful insights. Natural language processing (NLP) can help and includes both statistical- and large language model based techniques. We tested whether we could use NLP to map cardiac ultrasound text to a three-level hierarchical ontology. We used statistical machine learning (EchoMap) and zero-shot inference using GPT. We tested eight datasets from 24 different institutions and compared both methods against clinician-scored ground truth. Despite all adhering to clinical guidelines, institutions differed in their structured reporting. EchoMap performed best with validation accuracy of 98% for the first ontology level, 93% for first and second levels, and 79% for all three. EchoMap retained performance across external test datasets and could extrapolate to examples not included in training. EchoMap's accuracy was comparable to zero-shot GPT at the first level of the ontology and outperformed GPT at second and third levels. We show that statistical machine learning can map text to structured ontology and may be especially useful for small, specialized text datasets.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001443188600037</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>39953053</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Natural language processing; Machine learning; Large language models; Echocardiography report; Ontology</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
ECHOCARDIOGRAPHY; HEART</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Subramaniam, Suganya; Rizvi, Sara; Arnaout, Rima] Univ Calif San Francisco, 521 Parnassus Ave,Rm 6222, San Francisco, CA 94143 USA. <br>
[Ramesh, Ramya; Sehgal, Vibhor; Gurusamy, Brinda] Univ Calif Berkeley, Berkeley, CA USA. <br>
[Arif, Hikmatullah; Mackensen, G. Burkhard] Univ Washington, Seattle, WA USA. <br>
[Tran, Jeffrey] Univ Arizona, Tucson, AZ USA. <br>
[Thamman, Ritu] Univ Pittsburgh, Pittsburgh, PA USA. <br>
[Anyanwu, Emeka C.] Univ Penn, Philadelphia, PA USA. <br>
[Mastouri, Ronald] Indiana Univ, Indianapolis, IN USA. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Arnaout, R (corresponding author), Univ Calif San Francisco, 521 Parnassus Ave,Rm 6222, San Francisco, CA 94143 USA.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
rima.arnaout@ucsf.edu</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
University of California System; University of California San Francisco; University of California System; University of California Berkeley; University of Washington; University of Washington Seattle; University of Arizona; Pennsylvania Commonwealth System of Higher Education (PCSHE); University of Pittsburgh; University of Pennsylvania; Indiana University System; Indiana University Indianapolis</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Mackensen, G.</display_name>&nbsp;</td><td>AAJ-3951-2021&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Gurusamy, Brinda</display_name>&nbsp;</td><td>LRS-8317-2024&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Thamman, Ritu</display_name>&nbsp;</td><td>AAE-3690-2020&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Sehgal, Vibhor</display_name>&nbsp;</td><td>HKV-7575-2023&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>NATURE PORTFOLIO</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>HEIDELBERGER PLATZ 3, BERLIN, 14197, GERMANY</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Multidisciplinary Sciences</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Science &amp; Technology - Other Topics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>0BD6Z</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2045-2322</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>SCI REP-UK</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Sci Rep</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>9</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Heart, Lung, and Blood Institute</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>We thank Marc Kohli and Ross Filice for background information about Radlex.</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>gold, Green Submitted</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 5 of 32</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Practical intelligent diagnostic algorithm for wearable 12-lead ECG via self-supervised learning on large-scale dataset</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Lai, JW (Lai, Jiewei); Tan, HX (Tan, Huixin); Wang, JL (Wang, Jinliang); Ji, L (Ji, Lei); Guo, J (Guo, Jun); Han, BS (Han, Baoshi); Shi, YJ (Shi, Yajun); Feng, QJ (Feng, Qianjin); Yang, W (Yang, Wei)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>NATURE COMMUNICATIONS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>14</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>1</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>3741</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1038/s41467-023-39472-8</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2023 JUN 23</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>48</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>55</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
13</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
86</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>46</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Cardiovascular disease is a major global public health problem, and intelligent diagnostic approaches play an increasingly important role in the analysis of electrocardiograms (ECGs). Convenient wearable ECG devices enable the detection of transient arrhythmias and improve patient health by making it possible to seek intervention during continuous monitoring. We collected 658,486 wearable 12-lead ECGs, among which 164,538 were annotated, and the remaining 493,948 were without diagnostic. We present four data augmentation operations and a self-supervised learning classification framework that can recognize 60 ECG diagnostic terms. Our model achieves an average area under the receiver-operating characteristic curve (AUROC) and average F1 score on the offline test of 0.975 and 0.575. The average sensitivity, specificity and F1-score during the 2-month online test are 0.736, 0.954 and 0.468, respectively. This approach offers real-time intelligent diagnosis, and detects abnormal segments in long-term ECG monitoring in the clinical setting for further diagnosis by cardiologists.
<br>Intelligent diagnostic algorithms for ECG are becoming increasingly important to reduce the workload of cardiologists, enable telemedicine and real-time monitoring. Here, the authors show a model based on self-supervised learning that can classify 60 diagnostic terms for ECG.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001018752400005</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>37353501</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
CLASSIFICATION; ELECTROCARDIOGRAM; RHYTHM</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Lai, Jiewei; Tan, Huixin; Feng, Qianjin; Yang, Wei] Southern Med Univ, Sch Biomed Engn, Guangzhou, Peoples R China. <br>
[Lai, Jiewei; Tan, Huixin; Feng, Qianjin; Yang, Wei] Guangdong Prov Key Lab Med Image Proc, Guangzhou, Peoples R China. <br>
[Lai, Jiewei; Tan, Huixin; Feng, Qianjin; Yang, Wei] Guangdong Prov Engn Lab Med Imaging &amp; Diagnost Tec, Guangzhou, Peoples R China. <br>
[Wang, Jinliang] CardioCloud Med Technol Beijing Co Ltd, Beijing, Peoples R China. <br>
[Ji, Lei] Chinese Peoples Liberat Army Gen Hosp, IT Dept, Beijing, Peoples R China. <br>
[Guo, Jun; Han, Baoshi; Shi, Yajun] Chinese Peoples Liberat Army Gen Hosp, Dept Cardiol, Beijing, Peoples R China. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Feng, QJ; Yang, W (corresponding author), Southern Med Univ, Sch Biomed Engn, Guangzhou, Peoples R China.<br>Feng, QJ; Yang, W (corresponding author), Guangdong Prov Key Lab Med Image Proc, Guangzhou, Peoples R China.<br>Feng, QJ; Yang, W (corresponding author), Guangdong Prov Engn Lab Med Imaging &amp; Diagnost Tec, Guangzhou, Peoples R China.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
fengqj99@smu.edu.cn; weiyanggm@gmail.com</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Southern Medical University - China; Chinese People's Liberation Army General Hospital; Chinese People's Liberation Army General Hospital</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Lai, Jiewei</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-9872-9956&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Feng, Qianjin</display_name>&nbsp;</td><td>Y-5438-2019&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Shi, Yajun</display_name>&nbsp;</td><td>GPS-8969-2022&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Guo, Jun</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-7225-3264&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Han, Bao-shi</display_name>&nbsp;</td><td>JPY-3511-2023&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>NATURE PORTFOLIO</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>HEIDELBERGER PLATZ 3, BERLIN, 14197, GERMANY</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Multidisciplinary Sciences</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Science &amp; Technology - Other Topics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>K8FX5</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>2041-1723</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>NAT COMMUN</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Nat. Commun.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>13</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Key Ramp;D Program of China</grant_agency>&nbsp;</td><td>
<div>2018YFC2001203&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Key Laboratory of Medical Image Processing of Guangdong Provincial</grant_agency>&nbsp;</td><td>
<div>2020B1212060039&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>AcknowledgementsWe are very grateful to the staffs and clinicians at Cardiocloud Medical Technology (Beijing) Co., Ltd. for supporting and annotating the large scale ECG dataset and deploying our model to the server for online testing: team of Yong Yan. This study was funded by the National Key R&amp;D Program of China (no.2018YFC2001203) and the Key Laboratory of Medical Image Processing of Guangdong Provincial (no. 2020B1212060039).</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>gold, Green Published</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 6 of 32</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Self-supervised learning for label-free segmentation in cardiac ultrasound</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Ferreira, DL (Ferreira, Danielle L.); Lau, C (Lau, Connor); Salaymang, Z (Salaymang, Zaynaf); Arnaout, R (Arnaout, Rima)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>NATURE COMMUNICATIONS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>16</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>1</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>4070</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1038/s41467-025-59451-5</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2025 APR 30</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>1</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>1</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
16</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
16</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>62</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Segmentation and measurement of cardiac chambers from ultrasound is critical, but laborious and poorly reproducible. Neural networks can assist, but supervised approaches require the same problematic manual annotations. We build a pipeline for self-supervised segmentation combining computer vision, clinical knowledge, and deep learning. We train on 450 echocardiograms and test on 18,423 echocardiograms (including external data), using the resulting segmentations to calculate measurements. Coefficient of determination (r2) between clinically measured and pipeline-predicted measurements (0.55-0.84) are comparable to inter-clinician variation and to supervised learning. Average accuracy for detecting abnormal chambers is 0.85 (0.71-0.97). A subset of test echocardiograms (n = 553) have corresponding cardiac MRIs (the gold standard). Correlation between pipeline and MRI measurements is similar to that of clinical echocardiogram. Finally, the pipeline segments the left ventricle with an average Dice score of 0.89 (95% CI [0.89]). Our results demonstrate a manual-label free, clinically valid, and scalable method for segmentation from ultrasound.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001479703200017</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>40307208</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
VENTRICULAR EJECTION FRACTION; 3-DIMENSIONAL ECHOCARDIOGRAPHY; VOLUMES; QUANTIFICATION</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Ferreira, Danielle L.; Lau, Connor; Salaymang, Zaynaf; Arnaout, Rima] Univ Calif San Francisco, Dept Med, Div Cardiol, 521 Parnassus Ave, San Francisco, CA 94143 USA. <br>
[Ferreira, Danielle L.; Lau, Connor; Arnaout, Rima] Univ Calif San Francisco, Bakar Computat Hlth Sci Inst, 490 Illinois St, San Francisco, CA 94143 USA. <br>
[Arnaout, Rima] Ctr Intelligent Imaging, Dept Radiol, 505 Parnassus Ave, San Francisco, CA 94143 USA. <br>
[Arnaout, Rima] UCSF UC Berkeley Joint Program Computat Precis Hlt, 505 Parnassus Ave, San Francisco, CA 94118 USA. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Arnaout, R (corresponding author), Univ Calif San Francisco, Dept Med, Div Cardiol, 521 Parnassus Ave, San Francisco, CA 94143 USA.<br>Arnaout, R (corresponding author), Univ Calif San Francisco, Bakar Computat Hlth Sci Inst, 490 Illinois St, San Francisco, CA 94143 USA.<br>Arnaout, R (corresponding author), Ctr Intelligent Imaging, Dept Radiol, 505 Parnassus Ave, San Francisco, CA 94143 USA.<br>Arnaout, R (corresponding author), UCSF UC Berkeley Joint Program Computat Precis Hlt, 505 Parnassus Ave, San Francisco, CA 94118 USA.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
rima.arnaout@ucsf.edu</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
University of California System; University of California San Francisco; University of California System; University of California San Francisco</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>L Ferreira, Danielle</display_name>&nbsp;</td><td>ACM-2557-2022&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Ferreira, Danielle</display_name>&nbsp;</td><td>ACM-2557-2022&nbsp;</td><td>0000-0001-6088-9655&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>NATURE PORTFOLIO</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>HEIDELBERGER PLATZ 3, BERLIN, 14197, GERMANY</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Multidisciplinary Sciences</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Science &amp; Technology - Other Topics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>2CZ0M</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>2041-1723</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>NAT COMMUN</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Nat. Commun.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>13</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>U.S. Department of Health &amp; Human Services | National Institutes of Health (NIH)</grant_agency>&nbsp;</td><td>
<div>R01HL150394&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Institutes of Health</grant_agency>&nbsp;</td><td>
<div>PR181763&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Department of Defense</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>We thank the patients, sonographers and clinicians whose work created the datasets used to develop and test this pipeline. We thank Yumiko Abe-Jones and Nader Najafi for help with data pulls. D.F. and R.A. were supported by the National Institutes of Health (R01HL150394) to R.A. R.A. is additionally supported by the Department of Defense (PR181763) and the Chan Zuckerberg Biohub.</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>gold, Green Submitted</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 7 of 32</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Transfer learning for ECG classification</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Weimann, K (Weimann, Kuba); Conrad, TOF (Conrad, Tim O. F.)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>SCIENTIFIC REPORTS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>11</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>1</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>5251</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1038/s41598-021-84374-8</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2021 MAR 4</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>132</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>152</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
4</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
73</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>33</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Remote monitoring devices, which can be worn or implanted, have enabled a more effective healthcare for patients with periodic heart arrhythmia due to their ability to constantly monitor heart activity. However, these devices record considerable amounts of electrocardiogram (ECG) data that needs to be interpreted by physicians. Therefore, there is a growing need to develop reliable methods for automatic ECG interpretation to assist the physicians. Here, we use deep convolutional neural networks (CNN) to classify raw ECG recordings. However, training CNNs for ECG classification often requires a large number of annotated samples, which are expensive to acquire. In this work, we tackle this problem by using transfer learning. First, we pretrain CNNs on the largest public data set of continuous raw ECG signals. Next, we finetune the networks on a small data set for classification of Atrial Fibrillation, which is the most common heart arrhythmia. We show that pretraining improves the performance of CNNs on the target task by up to 6.57%, effectively reducing the number of annotations required to achieve the same performance as CNNs that are not pretrained. We investigate both supervised as well as unsupervised pretraining approaches, which we believe will increase in relevance, since they do not rely on the expensive ECG annotations. The code is available on GitHub at https://github.com/kweimann/ecg-transfer-learning.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000626140000079</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>33664343</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Weimann, Kuba; Conrad, Tim O. F.] Zuse Inst Berlin, Dept Visual &amp; Data Centr Comp, Takustr 7, Berlin, Germany. <br>
[Conrad, Tim O. F.] Free Univ Berlin, Dept Math, Arnimallee 6, D-14195 Berlin, Germany. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Weimann, K (corresponding author), Zuse Inst Berlin, Dept Visual &amp; Data Centr Comp, Takustr 7, Berlin, Germany.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
kuba.weimann@zib.de</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Zuse Institute Berlin; Free University of Berlin</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Conrad, Tim</display_name>&nbsp;</td><td>N-2742-2014&nbsp;</td><td>0000-0002-5590-5726&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Weimann, Kuba</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-1525-8798&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>NATURE PORTFOLIO</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>HEIDELBERGER PLATZ 3, BERLIN, 14197, GERMANY</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Multidisciplinary Sciences</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Science &amp; Technology - Other Topics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>QS8JO</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2045-2322</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>SCI REP-UK</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Sci Rep</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>12</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>German Ministry for Education and Research (BMBF) as Berlin Big Data Center</grant_agency>&nbsp;</td><td>
<div>01IS14013A&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Berlin Center for Machine Learning</grant_agency>&nbsp;</td><td>
<div>01IS18037I&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Forschungscampus MODAL</grant_agency>&nbsp;</td><td>
<div>3FO18501&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported by the German Ministry for Education and Research (BMBF) as Berlin Big Data Center (01IS14013A) and the Berlin Center for Machine Learning (01IS18037I) and within the Forschungscampus MODAL (project grant 3FO18501).</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>Green Submitted, gold</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 8 of 32</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Association between class of foundational medication for heart failure and prognosis in heart failure with reduced/mildly reduced ejection fraction</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Ito, M (Ito, Miyuki); Maeda, D (Maeda, Daichi); Matsue, Y (Matsue, Yuya); Shiraishi, Y (Shiraishi, Yasuyuki); Dotare, T (Dotare, Taishi); Sunayama, T (Sunayama, Tsutomu); Nogi, K (Nogi, Kazutaka); Takei, M (Takei, Makoto); Ueda, T (Ueda, Tomoya); Nogi, M (Nogi, Maki); Ishihara, S (Ishihara, Satomi); Nakada, Y (Nakada, Yasuki); Kawakami, R (Kawakami, Rika); Kagiyama, N (Kagiyama, Nobuyuki); Kitai, T (Kitai, Takeshi); Oishi, S (Oishi, Shogo); Akiyama, E (Akiyama, Eiichi); Suzuki, S (Suzuki, Satoshi); Yamamoto, M (Yamamoto, Masayoshi); Kida, K (Kida, Keisuke); Okumura, T (Okumura, Takahiro); Nagatomo, Y (Nagatomo, Yuji); Kohno, T (Kohno, Takashi); Nakano, S (Nakano, Shintaro); Kohsaka, S (Kohsaka, Shun); Yoshikawa, T (Yoshikawa, Tsutomu); Saito, Y (Saito, Yoshihiko); Minamino, T (Minamino, Tohru)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>SCIENTIFIC REPORTS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>12</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>1</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>16611</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1038/s41598-022-20892-3</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2022 OCT 5</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>3</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>3</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
0</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
0</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>44</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
We clarified the association between changes in the number of foundational medications for heart failure (FMHF) during hospitalization for worsening heart failure (HF) and post-discharge prognosis. We retrospectively analyzed a combined dataset from three large-scale registries of hospitalized patients with HF in Japan (NARA-HF, WET-HF, and REALITY-AHF) and patients diagnosed with HF with reduced or mildly reduced left ventricular ejection fraction (HFr/mrEF) before admission. Patients were stratified by changes in the number of prescribed FMHF classes from admission to discharge: angiotensin-converting enzyme inhibitors or angiotensin receptor blockers, beta-blockers, and mineralocorticoid receptor blockers. Primary endpoint was the combined endpoint of HF rehospitalization and all-cause death within 1 year of discharge. The cohort comprised 1113 patients, and 482 combined endpoints were observed. Overall, FMHF prescriptions increased in 413 (37.1%) patients (increased group), remained unchanged in 607 (54.5%) (unchanged group), and decreased in 93 (8.4%) (decreased group) at discharge compared with that during admission. In the multivariable analysis, the increased group had a significantly lower incidence of the primary endpoint than the unchanged group (hazard ratio 0.56, 95% confidence interval 0.45-0.60; P &lt; 0.001). In conclusion, increase in FMHF classes during HF hospitalization is associated with a better prognosis in patients with HFr/mrEF.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000864277100011</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>36198895</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
NATRIURETIC PEPTIDE; BETA-BLOCKERS; GUIDELINE; MORTALITY; OUTCOMES; THERAPY; HOSPITALIZATION; POPULATION; MANAGEMENT; WITHDRAWAL</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Ito, Miyuki; Maeda, Daichi; Matsue, Yuya; Dotare, Taishi; Sunayama, Tsutomu; Kagiyama, Nobuyuki; Minamino, Tohru] Juntendo Univ, Dept Cardiovasc Biol &amp; Med, Grad Sch Med, Bunkyo Ku, 3-1-3 Hongo, Tokyo, Japan. <br>
[Shiraishi, Yasuyuki; Kohsaka, Shun] Keio Univ, Dept Med, Div Cardiol, Sch Med, Tokyo, Japan. <br>
[Nogi, Kazutaka; Ueda, Tomoya; Nogi, Maki; Ishihara, Satomi; Nakada, Yasuki; Saito, Yoshihiko] Nara Med Univ, Dept Cardiovasc Med, Kashihara, Nara, Japan. <br>
[Takei, Makoto] Tokyo Saiseikai Cent Hosp, Dept Cardiol, Tokyo, Japan. <br>
[Kawakami, Rika] Saiseikai Suita Hosp, Dept Cardiol, Suita, Osaka, Japan. <br>
[Kitai, Takeshi] Natl Cerebral &amp; Cardiovasc Ctr, Dept Cardiovasc Med, Osaka, Japan. <br>
[Oishi, Shogo] Himeji Cardiovasc Ctr, Dept Cardiol, Himeji, Hyogo, Japan. <br>
[Akiyama, Eiichi] Yokohama City Univ, Div Cardiol, Med Ctr, Yokohama, Kanagawa, Japan. <br>
[Suzuki, Satoshi] Fukushima Med Univ, Dept Cardiol &amp; Hematol, Fukushima, Japan. <br>
[Yamamoto, Masayoshi] Univ Tsukuba, Fac Med, Cardiovasc Div, Tsukuba, Ibaraki, Japan. <br>
[Kida, Keisuke] St Marianna Univ, Dept Pharmacol, Sch Med, Kawasaki, Kanagawa, Japan. <br>
[Okumura, Takahiro] Nagoya Univ, Dept Cardiol, Grad Sch Med, Nagoya, Aichi, Japan. <br>
[Nagatomo, Yuji] Natl Def Med Coll, Dept Cardiol, Tokorozawa, Saitama, Japan. <br>
[Kohno, Takashi] Kyorin Univ, Dept Cardiovasc Med, Sch Med, Tokyo, Japan. <br>
[Nakano, Shintaro] Saitama Med Univ, Int Med Ctr, Dept Cardiol, Saitama, Japan. <br>
[Yoshikawa, Tsutomu] Sakakibara Heart Inst, Dept Cardiol, Tokyo, Japan. <br>
[Minamino, Tohru] Japan Agcy Med Res &amp; Dev, Japan Agcy Med Res &amp; Dev Core Res Evolutionary Me, Tokyo, Japan. <br>
[Matsue, Yuya] Juntendo Univ, Grad Sch Med, Cardiovasc Resp Sleep Med, Bunkyo Ku, 3-1-3 Hongo, Tokyo, Japan. <br>
[Saito, Yoshihiko] Nara Prefectural Hosp Org, Nara Prefecture Seiwa Med Ctr, Nara, Japan. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Matsue, Y (corresponding author), Juntendo Univ, Dept Cardiovasc Biol &amp; Med, Grad Sch Med, Bunkyo Ku, 3-1-3 Hongo, Tokyo, Japan.<br>Matsue, Y (corresponding author), Juntendo Univ, Grad Sch Med, Cardiovasc Resp Sleep Med, Bunkyo Ku, 3-1-3 Hongo, Tokyo, Japan.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
yuya8950@gmail.com</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Juntendo University; Keio University; Nara Medical University; Tokyo Saiseikai Central Hospital; National Cerebral &amp; Cardiovascular Center - Japan; Yokohama City University; Fukushima Medical University; University of Tsukuba; Saint Marianna University; Nagoya University; National Defense Medical College - Japan; Kyorin University; Saitama Medical University; Juntendo University</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Minamino, Tohru</display_name>&nbsp;</td><td>P-8145-2018&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Kohsaka, Shun</display_name>&nbsp;</td><td>L-8695-2013&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Nagatomo, Yuji</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-3430-4614&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Kohno, Takashi</display_name>&nbsp;</td><td>M-8823-2013&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Kagiyama, Nobuyuki</display_name>&nbsp;</td><td>K-4450-2019&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Nagatomo, Yuji</display_name>&nbsp;</td><td>AAQ-1590-2020&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Kagiyama, Nobuyuki</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-2305-0049&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Kawakami, Rika</display_name>&nbsp;</td><td>HOA-8185-2023&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>NATURE PORTFOLIO</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>HEIDELBERGER PLATZ 3, BERLIN, 14197, GERMANY</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Multidisciplinary Sciences</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Science &amp; Technology - Other Topics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>5C5CE</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2045-2322</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>SCI REP-UK</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Sci Rep</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>9</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>JSPS KAKENHI</grant_agency>&nbsp;</td><td>
<div>21H03309&nbsp;</div>
<div>22K16152&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Cardiovascular Research Fund of Japan</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Japan Agency for Medical Research and Development</grant_agency>&nbsp;</td><td>
<div>201439013C&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Sakakibara Clinical Research Grants for the Promotion of Science</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>&nbsp;</td><td>
<div>18K15860&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>&nbsp;</td><td>
<div>23591062&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>&nbsp;</td><td>
<div>26461088&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>&nbsp;</td><td>
<div>17K09526&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was partially supported by JSPS KAKENHI (Y.M., 21H03309; D.M., 22K16152). REALITY registry was funded by the Cardiovascular Research Fund of Japan. WET-HF registry was supported by a Grant-in-Aid for Young Scientists (Y.Sh. JSPS KAKENHI, 18K15860), Grants-in-Aid for Scientific Research (T.Y. JSPS KAKENHI, 23591062 and 26461088; T.Ko. 17K09526), a grant from the Japan Agency for Medical Research and Development (S.K. 201439013C), and Sakakibara Clinical Research Grants for the Promotion of Science (T.Y. 2012-2018).</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>Green Published, gold</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 9 of 32</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Multifractal foundations of biomarker discovery for heart disease and stroke</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Mangalam, M (Mangalam, Madhur); Sadri, A (Sadri, Arash); Hayano, J (Hayano, Junichiro); Watanabe, E (Watanabe, Eiichi); Kiyono, K (Kiyono, Ken); Kelty-Stephen, DG (Kelty-Stephen, Damian G.)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>SCIENTIFIC REPORTS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>13</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>1</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>18316</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1038/s41598-023-45184-2</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2023 OCT 25</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>11</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>12</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
1</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
10</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>190</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Any reliable biomarker has to be specific, generalizable, and reproducible across individuals and contexts. The exact values of such a biomarker must represent similar health states in different individuals and at different times within the same individual to result in the minimum possible false-positive and false-negative rates. The application of standard cut-off points and risk scores across populations hinges upon the assumption of such generalizability. Such generalizability, in turn, hinges upon this condition that the phenomenon investigated by current statistical methods is ergodic, i.e., its statistical measures converge over individuals and time within the finite limit of observations. However, emerging evidence indicates that biological processes abound with nonergodicity, threatening this generalizability. Here, we present a solution for how to make generalizable inferences by deriving ergodic descriptions of nonergodic phenomena. For this aim, we proposed capturing the origin of ergodicity-breaking in many biological processes: cascade dynamics. To assess our hypotheses, we embraced the challenge of identifying reliable biomarkers for heart disease and stroke, which, despite being the leading cause of death worldwide and decades of research, lacks reliable biomarkers and risk stratification tools. We showed that raw R-R interval data and its common descriptors based on mean and variance are nonergodic and non-specific. On the other hand, the cascade-dynamical descriptors, the Hurst exponent encoding linear temporal correlations, and multifractal nonlinearity encoding nonlinear interactions across scales described the nonergodic heart rate variability more ergodically and were specific. This study inaugurates applying the critical concept of ergodicity in discovering and applying digital biomarkers of health and disease.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001089332200085</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>37880302</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
LEFT-VENTRICULAR FUNCTION; R INTERVAL DYNAMICS; RATE-VARIABILITY; DEVELOPMENTAL PROCESSES; TERM REPRODUCIBILITY; LOW-FREQUENCY; TIME; MODEL; NONERGODICITY; MORTALITY</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Mangalam, Madhur] Univ Nebraska, Dept Biomech, Div Biomech &amp; Res Dev, Omaha, NE 68182 USA. <br>
[Mangalam, Madhur] Univ Nebraska, Ctr Res Human Movement Variabil, Omaha, NE 68182 USA. <br>
[Sadri, Arash] Lyceum Sci Char, Tehran, Iran. <br>
[Sadri, Arash] Univ Tehran Med Sci, StudentsScientif Res Ctr, Interdisciplinary Neurosci Res Program, P94V 8MF, Tehran, Iran. <br>
[Hayano, Junichiro] Nagoya City Univ, Grad Sch Med, Nagoya, Aichi 4678601, Japan. <br>
[Watanabe, Eiichi] Fujita Hlth Univ, Bantane Hosp, Div Cardiol, Dept Internal Med, 3-6-10 Otobashi, Nagoya, Aichi 4540012, Japan. <br>
[Kiyono, Ken] Osaka Univ, Grad Sch Engn Sci, Osaka 5608531, Japan. <br>
[Kelty-Stephen, Damian G.] SUNY Coll New Paltz, Dept Psychol, New Paltz, NY 12561 USA. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Mangalam, M (corresponding author), Univ Nebraska, Dept Biomech, Div Biomech &amp; Res Dev, Omaha, NE 68182 USA.<br>Mangalam, M (corresponding author), Univ Nebraska, Ctr Res Human Movement Variabil, Omaha, NE 68182 USA.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
mmangalam@unomaha.edu</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
University of Nebraska System; University of Nebraska System; Tehran University of Medical Sciences; Nagoya City University; Fujita Health University; University of Osaka; State University of New York (SUNY) System; SUNY New Paltz</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Kelty-Stephen, Damian</display_name>&nbsp;</td><td>HJI-2370-2023&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Mangalam, Madhur</display_name>&nbsp;</td><td>AGS-6468-2022&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Sadri, Arash</display_name>&nbsp;</td><td>T-1740-2019&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Sadri, Arash</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-3726-243X&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Kiyono, Ken</display_name>&nbsp;</td><td>E-6325-2017&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Hayano, Junichiro</display_name>&nbsp;</td><td>NBY-4354-2025&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>NATURE PORTFOLIO</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>HEIDELBERGER PLATZ 3, BERLIN, 14197, GERMANY</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Multidisciplinary Sciences</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Science &amp; Technology - Other Topics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>W1MG4</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2045-2322</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>SCI REP-UK</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Sci Rep</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>20</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>This work was supported by the Center for Research in Human Movement Variability at the University of Nebraska at Omaha, funded by the NIH award P20GM109090.</grant_agency>&nbsp;</td><td>
<div>P20GM109090&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Center for Research in Human Movement Variability at the University of Nebraska at Omaha - NIH</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported by the Center for Research in Human Movement Variability at the University of Nebraska at Omaha, funded by the NIH award P20GM109090.</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>Green Published, gold</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 10 of 32</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Unsupervised analysis of whole transcriptome data from human pluripotent stem cells cardiac differentiation</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Agostinho, SP (Agostinho, Sofia P.); Branco, MA (Branco, Mariana A.); Nogueira, DES (Nogueira, Diogo E. S.); Diogo, MM (Diogo, Maria Margarida); Cabral, JMS (Cabral, Joaquim M. S.); Fred, ALN (Fred, Ana L. N.); Rodrigues, CAV (Rodrigues, Carlos A. V.)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>SCIENTIFIC REPORTS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>14</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>1</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>3110</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1038/s41598-024-52970-z</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2024 FEB 7</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>2</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>2</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
0</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
2</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>37</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
The main objective of the present work was to highlight differences and similarities in gene expression patterns between different pluripotent stem cell cardiac differentiation protocols, using a workflow based on unsupervised machine learning algorithms to analyse the transcriptome of cells cultured as a 2D monolayer or as 3D aggregates. This unsupervised approach effectively allowed to portray the transcriptomic changes that occurred throughout the differentiation processes, with a visual representation of the entire transcriptome. The results allowed to corroborate previously reported data and also to unveil new gene expression patterns. In particular, it was possible to identify a correlation between low cardiomyocyte differentiation efficiencies and the early expression of a set of non-mesodermal genes, which can be further explored as predictive markers of differentiation efficiency. The workflow here developed can also be applied to analyse other stem cell differentiation transcriptomic datasets, envisaging future clinical implementation of cellular therapies.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001158921600039</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>38326387</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
CARDIOMYOCYTE DIFFERENTIATION; PACKAGE</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Agostinho, Sofia P.; Branco, Mariana A.; Nogueira, Diogo E. S.; Diogo, Maria Margarida; Cabral, Joaquim M. S.; Fred, Ana L. N.; Rodrigues, Carlos A. V.] Univ Lisbon, Dept Bioengn, Inst Super Tecn, Av Rovisco Pais, P-1049001 Lisbon, Portugal. <br>
[Agostinho, Sofia P.; Branco, Mariana A.; Nogueira, Diogo E. S.; Diogo, Maria Margarida; Cabral, Joaquim M. S.; Rodrigues, Carlos A. V.] Univ Lisbon, iBB Inst Bioengn &amp; Biosci, Inst Super Tecn, Av Rovisco Pais, P-1049001 Lisbon, Portugal. <br>
[Agostinho, Sofia P.; Branco, Mariana A.; Nogueira, Diogo E. S.; Diogo, Maria Margarida; Cabral, Joaquim M. S.; Rodrigues, Carlos A. V.] Univ Lisbon, Inst Hlth &amp; Bioecon, Associate Lab i4HB, Inst Super Tecn, Av Rovisco Pais, P-1049001 Lisbon, Portugal. <br>
[Agostinho, Sofia P.; Fred, Ana L. N.] Inst Telecomunicacoes IT, Av Rovisco Pais 1,Torre Norte Piso 10, P-1049001 Lisbon, Portugal. <br>
[Branco, Mariana A.] Collaborat Lab Foster Translat &amp; Drug Discovery, P-3030197 Accelbio, Cantanhede, Portugal. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Agostinho, SP (corresponding author), Univ Lisbon, Dept Bioengn, Inst Super Tecn, Av Rovisco Pais, P-1049001 Lisbon, Portugal.<br>Agostinho, SP (corresponding author), Univ Lisbon, iBB Inst Bioengn &amp; Biosci, Inst Super Tecn, Av Rovisco Pais, P-1049001 Lisbon, Portugal.<br>Agostinho, SP (corresponding author), Univ Lisbon, Inst Hlth &amp; Bioecon, Associate Lab i4HB, Inst Super Tecn, Av Rovisco Pais, P-1049001 Lisbon, Portugal.<br>Agostinho, SP (corresponding author), Inst Telecomunicacoes IT, Av Rovisco Pais 1,Torre Norte Piso 10, P-1049001 Lisbon, Portugal.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
sofia.p.agostinho@tecnico.ulisboa.pt</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Universidade de Lisboa; Universidade de Lisboa; Universidade de Lisboa</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Branco, Mariana</display_name>&nbsp;</td><td>AAX-1077-2021&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Rodrigues, Carlos</display_name>&nbsp;</td><td>H-7714-2013&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Fred, Ana</display_name>&nbsp;</td><td>A-7464-2016&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Diogo, Maria Margarida</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-6270-4455&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>de Pinto e Lobo da Silva Agostinho, Sofia</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-7254-7916&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Branco, Mariana</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-9795-3323&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Diogo, Margarida</display_name>&nbsp;</td><td>ABF-4341-2020&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Nogueira, Diogo</display_name>&nbsp;</td><td>KBQ-2503-2024&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>NATURE PORTFOLIO</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>HEIDELBERGER PLATZ 3, BERLIN, 14197, GERMANY</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Multidisciplinary Sciences</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Science &amp; Technology - Other Topics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>HI7B1</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2045-2322</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>SCI REP-UK</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Sci Rep</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>13</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Fundaco para a Ciencia e Tecnologia (FCT) through iBB - Institute for Bioengineering and Biosciences</grant_agency>&nbsp;</td><td>
<div>UIDB/04565/2020&nbsp;</div>
<div>UIDP/04565/2020&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Associate Laboratory i4HB</grant_agency>&nbsp;</td><td>
<div>LA/P/0140/2020&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Project "SMART"</grant_agency>&nbsp;</td><td>
<div>PTDC/EQU-EQU/3853/2020&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>IT - Instituto de Telecomunicaces</grant_agency>&nbsp;</td><td>
<div>BIM/No16/2022 - B-B01049&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>GFI - The Good Food Institute</grant_agency>&nbsp;</td><td>
<div>22-CM-PT-DG-1-317&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>The authors acknowledge funding received from Fundac &amp; atilde;o para a Ciencia e Tecnologia (FCT) through iBB - Institute for Bioengineering and Biosciences (UIDB/04565/2020 and UIDP/04565/2020), Associate Laboratory i4HB (LA/P/0140/2020), project "SMART" (PTDC/EQU-EQU/3853/2020); from IT - Instituto de Telecomunicac &amp; otilde;es - research Grant BIM/No16/2022 - B-B01049; from GFI - The Good Food Institute granted to the project VitaminSea (22-CM-PT-DG-1-317).</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>gold, Green Published</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 11 of 32</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Right heart exercise-training-adaptation and remodelling in endurance athletes</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Conti, V (Conti, Valeria); Migliorini, F (Migliorini, Filippo); Pilone, M (Pilone, Marco); Barriopedro, M (Barriopedro, Maria, I); Ramos-Alvarez, JJ (Jose Ramos-Alvarez, Juan); Montero, FJC (Calderon Montero, Francisco Javer); Maffulli, N (Maffulli, Nicola)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>SCIENTIFIC REPORTS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>11</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>1</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>22532</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1038/s41598-021-02028-1</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2021 NOV 18</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>18</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>18</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
0</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
2</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>74</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Long-term sports training leads to myocardial adaptations, with remodelling of the heart chambers. However, while myocardial adaptations of the left heart are well described, remodelling of the right heart and its impact on the development of arrhythmias is still debated. To conduct a systematic review on right ventricle (RV) and right atrium (RA) structural and functional changes in athletes who participate in long-term endurance training. Systematic review. A systematic literature search was conducted. All the articles reporting right heart echocardiographic (ECHO) and cardiac magnetic resonance (CMR) parameters evaluated in endurance athletes and sedentary subjects were considered eligible. A multivariate analysis was conducted to investigate whether age, sex, body surface area (BSA), intensity of training are associated with RV ECHO, CMR parameters and RA ECHO parameters. A positive association between age and right atrium area (RAA) (P = 0.01) was found. This is a negative association to RV E/A (P = 0.004), and RV end diastolic diameter (RVED) longitudinal (P = 0.01). A positive association between BSA and RVED middle (P = 0.001), as well between BSA and RAA (P = 0.05) was found, along with a negative association with RV E/A (P = 0.002). A positive association between intensity of training and RV end systolic area (RVESA) (P = 0.03), RV end diastolic volume indexed (RVEDVI) (P = 0.01), RV end systolic volume indexed (RVESVI) (P = 0.01) was found, along with a negative association with ejection fraction (EF %) (P = 0.01). Endurance athletes demonstrated an association between RV remodelling and age, BSA and intensity of training.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000720520100049</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>34795399</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
RIGHT-VENTRICULAR ADAPTATION; SPECKLE-TRACKING; REFERENCE VALUES; ECHOCARDIOGRAPHY; ARRHYTHMIAS; DISPLACEMENT; METAANALYSIS; DETERMINANT; MORPHOLOGY; SOCIETY</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Conti, Valeria; Pilone, Marco; Maffulli, Nicola] Univ Salerno, Dept Med Surg &amp; Dent, Baronissi, Italy. <br>
[Migliorini, Filippo] RWTH Aachen Univ Clin, Dept Orthopaed Trauma &amp; Recontruct Surg, Pauwelsstr 30, D-52074 Aachen, Germany. <br>
[Barriopedro, Maria, I] Univ Politecn Madrid, Fac Phys Act &amp; Sport Sci, Madrid, Spain. <br>
[Jose Ramos-Alvarez, Juan; Calderon Montero, Francisco Javer] Univ Complutense Madrid, Escuela Med Deport, Dept Radiol Rehabil &amp; Fisioterapia, Madrid, Spain. <br>
[Maffulli, Nicola] Queen Mary Univ London, Ctr Sports &amp; Exercise Med, Barts &amp; London Sch Med &amp; Dent, Mile End Hosp, 275 Bancroft Rd, London E1 4DG, England. <br>
[Maffulli, Nicola] Univ Salerno, Dept Musculoskeletal Disorders, Sch Med Surg &amp; Dent, Salerno, Italy. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Migliorini, F (corresponding author), RWTH Aachen Univ Clin, Dept Orthopaed Trauma &amp; Recontruct Surg, Pauwelsstr 30, D-52074 Aachen, Germany.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
Migliorini.md@gmail.com</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
University of Salerno; RWTH Aachen University; RWTH Aachen University Hospital; Universidad Politecnica de Madrid; Facultad de Ciencias de la Actividad Fisica del Deporte (INEF); Complutense University of Madrid; University of London; Queen Mary University London; Barts Health NHS Trust; Mile End Hospital; University of Salerno</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Ramos-Alvarez, Juan Jose</display_name>&nbsp;</td><td>F-7672-2017&nbsp;</td><td>0000-0002-7575-0100&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Migliorini, Filippo</display_name>&nbsp;</td><td>AAZ-9529-2021&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Ramos-&Aacute;lvarez, Juan</display_name>&nbsp;</td><td>F-7672-2017&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Conti, Valeria</display_name>&nbsp;</td><td>AAV-8824-2021&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>NATURE PORTFOLIO</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>HEIDELBERGER PLATZ 3, BERLIN, 14197, GERMANY</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Multidisciplinary Sciences</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Science &amp; Technology - Other Topics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>XA2YW</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2045-2322</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>SCI REP-UK</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Sci Rep</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>12</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Projekt DEAL</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>Open Access funding enabled and organized by Projekt DEAL.</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>Green Published, gold</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 12 of 32</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Using domain adaptation for classification of healthy and disease conditions from mobile-captured images of standard 12-lead electrocardiograms</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Gliner, V (Gliner, Vadim); Makarov, V (Makarov, Vladimir); Avetisyan, AI (Avetisyan, Arutyun I.); Schuster, A (Schuster, Assaf); Yaniv, Y (Yaniv, Yael)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>SCIENTIFIC REPORTS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>13</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>1</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>14023</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1038/s41598-023-40693-6</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2023 AUG 28</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>3</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>3</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
1</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
7</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>33</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
12-lead electrocardiogram (ECG) recordings can be collected in any clinic and the interpretation is performed by a clinician. Modern machine learning tools may make them automatable. However, a large fraction of 12-lead ECG data is still available in printed paper or image only and comes in various formats. To digitize the data, smartphone cameras can be used. Nevertheless, this approach may introduce various artifacts and occlusions into the obtained images. Here we overcome the challenges of automating 12-lead ECG analysis using mobile-captured images and a deep neural network that is trained using a domain adversarial approach. The net achieved an average 0.91 receiver operating characteristic curve on tested images captured by a mobile device. Assessment on image from unseen 12-lead ECG formats that the network was not trained on achieved high accuracy. We further show that the network accuracy can be improved by including a small number of unlabeled samples from unknown formats in the training data. Finally, our models also achieve high accuracy using signals as input rather than images. Using a domain adaptation approach, we successfully classified cardiac conditions on images acquired by a mobile device and showed the generalizability of the classification using various unseen image formats.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001057740800048</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>37640921</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Gliner, Vadim; Schuster, Assaf] Technion IIT, Comp Sci Dept, Haifa, Israel. <br>
[Makarov, Vladimir] Novgorod State Univ, Syst Programming Lab, Veliky Novgorod, Russia. <br>
[Avetisyan, Arutyun I.] Russian Acad Sci, Ivannikov Inst Syst Programming, Moscow, Russia. <br>
[Yaniv, Yael] Technion IIT, Biomed Engn Fac, Lab Bioenerget &amp; Bioelect Syst, Haifa, Israel. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Yaniv, Y (corresponding author), Technion IIT, Biomed Engn Fac, Lab Bioenerget &amp; Bioelect Syst, Haifa, Israel.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
yaely@bm.technion.ac.il</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Technion Israel Institute of Technology; Yaroslav the Wise Novgorod State University; Russian Academy of Sciences; Ivannikov Institute for System Programming of the RAS; Technion Israel Institute of Technology</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Makarov, Vladimir</display_name>&nbsp;</td><td>B-4045-2018&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>NATURE PORTFOLIO</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>HEIDELBERGER PLATZ 3, BERLIN, 14197, GERMANY</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Multidisciplinary Sciences</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Science &amp; Technology - Other Topics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>Q5EE1</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2045-2322</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>SCI REP-UK</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Sci Rep</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>14</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Israel Ministry of Science and Technology</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Russian Foundation (RFBR)</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported by the Israel Ministry of Science and Technology and Russian Foundation (RFBR). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>Green Published, gold</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 13 of 32</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Cardiac adaptation in hibernating, free-ranging Scandinavian Brown Bears (&lt;i&gt;Ursus arctos&lt;/i&gt;)</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Jorgensen, PG (Jorgensen, Peter Godsk); Evans, A (Evans, Alina); Kindberg, J (Kindberg, Jonas); Olsen, LH (Olsen, Lisbeth Hoier); Galatius, S (Galatius, Soren); Fr&ouml;bert, O (Frobert, Ole)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>SCIENTIFIC REPORTS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>10</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>1</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>247</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1038/s41598-019-57126-y</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2020 JAN 14</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>12</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>13</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
1</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
13</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>38</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
During six months of annual hibernation, the brown bear undergoes unique physiological changes to adapt to decreased metabolic rate. We compared cardiac structural and functional measures of hibernating and active bears using comprehensive echocardiography. We performed echocardiography on 13 subadult free-ranging, anaesthetised Scandinavian brown bears (Ursus arctos) during late hibernation and in early summer. Mean heart rate was 26 beats per minute (standard deviation (SD): 8) during hibernation vs 71 (SD: 14) during active state. All left ventricular (LV) systolic and diastolic measures were decreased during hibernation: mean ejection fraction: 44.2% (SD: 6.0) active state vs 34.0 (SD: 8.1) hibernation, P = 0.001; global longitudinal strain: -11.2% (SD: 2.0) vs -8.8 (SD: 3.3), P = 0.03; global longitudinal strain rate: -0.82 (SD: 0.15) vs -0.41 (SD: 0.18), P &lt; 0.001; septal e': 9.8 cm/s (SD: 1.8) vs 5.2 (SD: 2.7), P &lt; 0.001. In general, measures of total myocardial motion (ejection fraction and global longitudinal strain) were decreased to a lesser extent than measures of myocardial velocities. In the hibernating brown bear, cardiac adaptation included decreased functional measures, primarily measures of myocardial velocities, but was not associated with cardiac atrophy. Understanding the mechanisms of these adaptations could provide pathophysiological insight of human pathological conditions such as heart failure.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000551360800003</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>31937799</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
HEART-FAILURE; ISOFLURANE; METABOLISM; HALOTHANE; VELOCITY; EXERCISE; CHANNELS</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Jorgensen, Peter Godsk] Univ Copenhagen, Gentofte Hosp, Dept Cardiol, Copenhagen, Denmark. <br>
[Evans, Alina] Inland Norway Univ Appl Sci, Dept Forestry &amp; Wildlife Management, Campus Evenstad, Elverum, Norway. <br>
[Kindberg, Jonas] Swedish Univ Agr Sci, Fac Forest Sci, Dept Wildlife Fish &amp; Environm Studies, Umea, Sweden. <br>
[Kindberg, Jonas] Norwegian Inst Nat Res, Trondheim, Norway. <br>
[Olsen, Lisbeth Hoier] Univ Copenhagen, Dept Vet &amp; Anim Sci, Frederiksberg, Denmark. <br>
[Galatius, Soren] Univ Copenhagen, Bispebjerg Hosp, Dept Cardiol, Copenhagen, Denmark. <br>
[Frobert, Ole] Orebro Univ Hosp, Fac Hlth, Dept Cardiol, Orebro, Sweden. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Jorgensen, PG (corresponding author), Univ Copenhagen, Gentofte Hosp, Dept Cardiol, Copenhagen, Denmark.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
petergodsk@gmail.com</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
University of Copenhagen; Herlev &amp; Gentofte Hospital; Inland Norway University of Applied Sciences; Swedish University of Agricultural Sciences; Norwegian Institute Nature Research; University of Copenhagen; University of Copenhagen; Copenhagen University Hospital; Bispebjerg Hospital; Orebro University</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Kindberg, Jonas</display_name>&nbsp;</td><td>AAA-5058-2020&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Olsen, Lisbeth Hoier</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-0292-3111&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Frobert, Ole</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-5846-345X&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Frobert, Ole</display_name>&nbsp;</td><td>AFP-8888-2022&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Kindberg, Jonas</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-1445-4524&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>NATURE PORTFOLIO</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>HEIDELBERGER PLATZ 3, BERLIN, 14197, GERMANY</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Multidisciplinary Sciences</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Science &amp; Technology - Other Topics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>MO2KE</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2045-2322</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>SCI REP-UK</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Sci Rep</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>9</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Swedish Environmental Protection Agency</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Norwegian Directorate for Nature Management</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Swedish Association for Hunting and Wildlife Management</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Research Council of Norway</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Austrian Science Fund</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Nordforsk researcher network grant</grant_agency>&nbsp;</td><td>
<div>44042&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Lundbeck Foundation</grant_agency>&nbsp;</td><td>
<div>R126-2012-12408&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Lundbeck Foundation</grant_agency>&nbsp;</td><td>
<div>R126-2012-12408&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>The Scandinavian Brown Bear Research Project was funded by the Swedish Environmental Protection Agency, the Norwegian Directorate for Nature Management, the Swedish Association for Hunting and Wildlife Management, the Research Council of Norway, the Austrian Science Fund, a Nordforsk researcher network grant (project no. 44042), and the Lundbeck Foundation (R126-2012-12408).</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>Green Submitted, gold</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 14 of 32</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Intercellular mitochondrial component transfer triggers ischemic cardiac fibrosis</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Zhang, C (Zhang, Chan); Hao, H (Hao, Hao); Wang, YS (Wang, Yishi); Mu, N (Mu, Nan); Jiang, WH (Jiang, Wenhua); Zhang, ZH (Zhang, Zihui); Yin, Y (Yin, Yue); Yu, L (Yu, Lu); Chang, ACY (Chang, Alex Chia Yu); Ma, H (Ma, Heng)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>SCIENCE BULLETIN</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>68</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>16</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>1784-1799</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1016/j.scib.2023.07.030</value>&nbsp;&nbsp;<b>Early Access Date:</b> 
<value>AUG 2023</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2023 AUG 30</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>22</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>25</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
4</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
35</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>54</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Myocardial fibrosis is the villain of sudden cardiac death. Myocardial ischemia/reperfusion (MI/R) injury induces cardiomyocyte damage or even death, which in turn stimulates fibroblast activation and fibrosis, but the intercellular communication mechanism remains unknown. Recent studies have shown that small extracellular vesicles (sEVs) significantly contribute to intercellular communication. Whether and how sEV might mediate post-MI/R cardiomyocyte/fibroblasts communication remain unknown. Here, in vivo and in vitro MI/R models were established. We demonstrate that sEVs derived from cardiomyocyte (Myo-sEVs) carry mitochondrial components, which enter fibroblasts to initiate myocardial fibrosis. Based on bioinformatics screening and experimental verification, the activating molecule in Beclin1-regulated autophagy protein 1 (autophagy/beclin-1 regulator 1, Ambra1) was found to be a critical component of these sEV and might be a new marker for Myo-sEVs. Interestingly, release of Ambra1'- Myo-sEVs was caused by secretory rather than canonical autophagy after MI/R injury and thereby escaped degradation. In ischemic and peripheral areas, Ambra1'-Myo-sEVs were internalized by fibrob-lasts, and the delivered mtDNA components to activate the fibroblast cyclic GMP-AMP synthase (cGAS)stimulator of interferon genes (STING) pathway to promote fibroblast activation and proliferation. In addition, our data show that Ambra1 is expressed on the EV surface and cardiac-specific Ambra1 down regulation inhibits the Ambra1'-Myo-sEVs release and fibroblast uptake, effectively inhibiting ischemic myocardial fibrosis. This finding newly provides the evidence that myocardial secretory autophagy plays a role in intercellular communication during cardiac fibrosis. Ambra1 is a newly characterized molecule with bioactivity and might be a marker for Myo-sEVs, providing new therapeutic targets for cardiac remodeling.(c) 2023 Science China Press. Published by Elsevier B.V. and Science China Press. All rights reserved.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001082338700001</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>37517989</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Intercellular mitochondrial component; transfer; Intercellular communication; Extracellular vesicles; Secretory autophagy; Myocardial fibrosis</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
AUTOPHAGY; EXOSOMES; INJURY; HEART</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Zhang, Chan; Jiang, Wenhua; Zhang, Zihui] Northwestern Polytech Univ, Inst Med Res, Xian Key Lab Stem Cell &amp; Regenerat Med, Xian 710072, Peoples R China. <br>
[Hao, Hao; Wang, Yishi; Mu, Nan; Yin, Yue; Ma, Heng] Fourth Mil Med Univ, Sch Basic Med, Dept Physiol &amp; Pathophysiol, Xian 710032, Peoples R China. <br>
[Chang, Alex Chia Yu] Shanghai Jiao Tong Univ, Peoples Hosp 9, Dept Cardiol, Sch Med, Shanghai 211125, Peoples R China. <br>
[Chang, Alex Chia Yu] Shanghai Jiao Tong Univ, Peoples Hosp 9, Shanghai Inst Precis Med, Sch Med, Shanghai 211125, Peoples R China. <br>
[Yu, Lu] Fourth Mil Med Univ, Xijing Hosp, Dept Pathol, Xian 710032, Peoples R China. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Ma, H (corresponding author), Fourth Mil Med Univ, Sch Basic Med, Dept Physiol &amp; Pathophysiol, Xian 710032, Peoples R China.<br>Chang, ACY (corresponding author), Shanghai Jiao Tong Univ, Peoples Hosp 9, Dept Cardiol, Sch Med, Shanghai 211125, Peoples R China.<br>Chang, ACY (corresponding author), Shanghai Jiao Tong Univ, Peoples Hosp 9, Shanghai Inst Precis Med, Sch Med, Shanghai 211125, Peoples R China.<br>Yu, L (corresponding author), Fourth Mil Med Univ, Xijing Hosp, Dept Pathol, Xian 710032, Peoples R China.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
yulu@fmmu.edu.cn; alexchang@shsmu.edu.cn; hengma@fmmu.edu.cn</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Northwestern Polytechnical University; Air Force Medical University; Shanghai Jiao Tong University; Shanghai Jiao Tong University; Air Force Medical University</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Yu, Lu</display_name>&nbsp;</td><td>&nbsp;</td><td>0009-0007-9449-8274&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Wang, Yishi</display_name>&nbsp;</td><td>IQU-0806-2023&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Chang, Alex Chia Yu</display_name>&nbsp;</td><td>KGM-6777-2024&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Wang, Yishi</display_name>&nbsp;</td><td>&nbsp;</td><td>0009-0004-6021-0357&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>ELSEVIER</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>RADARWEG 29, 1043 NX AMSTERDAM, NETHERLANDS</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Multidisciplinary Sciences</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Science &amp; Technology - Other Topics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>U1FT8</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2095-9273</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>2095-9281</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>SCI BULL</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Sci. Bull.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>16</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Natural Science Foundation of China</grant_agency>&nbsp;</td><td>
<div>82070261&nbsp;</div>
<div>82170251&nbsp;</div>
<div>82200386&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Key Research and Development Program of China</grant_agency>&nbsp;</td><td>
<div>2021YFA1100501&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Chinese Postdoctoral Science Foundation of China</grant_agency>&nbsp;</td><td>
<div>2022M712590&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Science and Technology Research and Development Program of Shaanxi Province, China</grant_agency>&nbsp;</td><td>
<div>2022JQ- 881&nbsp;</div>
<div>2021SF-324&nbsp;</div>
<div>2022SF-091&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Youth Innovation Team of Shaanxi Universities</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Innovation Foundation for Doctor Dissertation of Northwestern Polytechnical University</grant_agency>&nbsp;</td><td>
<div>CX2022068&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Seed Foundation of Innovation and Creation for Graduate Students in Northwestern Polytechnical University</grant_agency>&nbsp;</td><td>
<div>CX202065&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported by the National Natural Science Foundation of China (82070261, 82170251, and 82200386) , the National Key Research and Development Program of China (2021YFA1100501) , the Chinese Postdoctoral Science Foundation of China (2022M712590) , the Science and Technology Research and Development Program of Shaanxi Province, China (2022JQ- 881, 2021SF-324, and 2022SF-091) , the Youth Innovation Team of Shaanxi Universities, the Innovation Foundation for Doctor Dissertation of Northwestern Polytechnical University (CX2022068) , and the Seed Foundation of Innovation and Creation for Graduate Students in Northwestern Polytechnical University (CX202065) . The authors thank Professor Guo-Dong Yang from School of Basic Medicine and Professor Shi-Yu Liu and Bing-Dong Sui from School of Stomatology, Fourth Military Medical University for comments on the manuscript.</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 15 of 32</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Multiscale classification of heart failure phenotypes by unsupervised clustering of unstructured electronic medical record data</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Nagamine, T (Nagamine, Tasha); Gillette, B (Gillette, Brian); Pakhomov, A (Pakhomov, Alexey); Kahoun, J (Kahoun, John); Mayer, H (Mayer, Hannah); Burghaus, R (Burghaus, Rolf); Lippert, J (Lippert, Jorg); Saxena, M (Saxena, Mayur)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>SCIENTIFIC REPORTS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>10</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>1</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>21340</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1038/s41598-020-77286-6</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2020 DEC 7</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>25</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>28</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
0</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
9</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>53</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
As a leading cause of death and morbidity, heart failure (HF) is responsible for a large portion of healthcare and disability costs worldwide. Current approaches to define specific HF subpopulations may fail to account for the diversity of etiologies, comorbidities, and factors driving disease progression, and therefore have limited value for clinical decision making and development of novel therapies. Here we present a novel and data-driven approach to understand and characterize the real-world manifestation of HF by clustering disease and symptom-related clinical concepts (complaints) captured from unstructured electronic health record clinical notes. We used natural language processing to construct vectorized representations of patient complaints followed by clustering to group HF patients by similarity of complaint vectors. We then identified complaints that were significantly enriched within each cluster using statistical testing. Breaking the HF population into groups of similar patients revealed a clinically interpretable hierarchy of subgroups characterized by similar HF manifestation. Importantly, our methodology revealed well-known etiologies, risk factors, and comorbid conditions of HF (including ischemic heart disease, aortic valve disease, atrial fibrillation, congenital heart disease, various cardiomyopathies, obesity, hypertension, diabetes, and chronic kidney disease) and yielded additional insights into the details of each HF subgroup's clinical manifestation of HF. Our approach is entirely hypothesis free and can therefore be readily applied for discovery of novel insights in alternative diseases or patient populations.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000609197600014</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>33288774</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
PRESERVED EJECTION FRACTION; MEDIASTINAL LYMPHADENOPATHY; DISEASE; GUIDELINES; DIAGNOSIS; HYPERTENSION; MANAGEMENT; SUBTYPES; BURDEN; HFPEF</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Nagamine, Tasha; Pakhomov, Alexey; Kahoun, John; Saxena, Mayur] Droice Res, New York, NY 11201 USA. <br>
[Gillette, Brian] NYU Langone Hosp Long Isl, Dept Surg, Mineola, NY USA. <br>
[Gillette, Brian] NYU Long Isl Sch Med, Dept Fdn Med, Mineola, NY USA. <br>
[Kahoun, John] CityMD, Clin Informat, New York, NY USA. <br>
[Mayer, Hannah; Burghaus, Rolf; Lippert, Jorg] Bayer AG, Clin Pharmacometr, Wuppertal, Germany. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Saxena, M (corresponding author), Droice Res, New York, NY 11201 USA.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
mayur@droicelabs.com</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Bayer AG</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Pakhomov, Alexey</display_name>&nbsp;</td><td>B-3594-2014&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Burghaus, Rolf</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-7843-427X&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Middleton, Hannah</display_name>&nbsp;</td><td>HLW-9015-2023&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Lippert, J&ouml;rg</display_name>&nbsp;</td><td>I-3179-2012&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>NATURE PORTFOLIO</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>HEIDELBERGER PLATZ 3, BERLIN, 14197, GERMANY</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Multidisciplinary Sciences</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Science &amp; Technology - Other Topics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>PU3II</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2045-2322</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>SCI REP-UK</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Sci Rep</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>13</value>
</td>
</tr>

<tr>
<td>
<b>Open Access:</b>

<value>gold, Green Published</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 16 of 32</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Transfer Learning in ECG Classification from Human to Horse Using a Novel Parallel Neural Network Architecture</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Van Steenkiste, G (Van Steenkiste, Glenn); van Loon, G (van Loon, Gunther); Crevecoeur, G (Crevecoeur, Guillaume)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>SCIENTIFIC REPORTS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>10</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>1</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>186</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1038/s41598-019-57025-2</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2020 JAN 13</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>41</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>44</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
0</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
17</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>48</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Automatic or semi-automatic analysis of the equine electrocardiogram (eECG) is currently not possible because human or small animal ECG analysis software is unreliable due to a different ECG morphology in horses resulting from a different cardiac innervation. Both filtering, beat detection to classification for eECGs are currently poorly or not described in the literature. There are also no public databases available for eECGs as is the case for human ECGs. In this paper we propose the use of wavelet transforms for both filtering and QRS detection in eECGs. In addition, we propose a novel robust deep neural network using a parallel convolutional neural network architecture for ECG beat classification. The network was trained and tested using both the MIT-BIH arrhythmia and an own made eECG dataset with 26.440 beats on 4 classes: normal, premature ventricular contraction, premature atrial contraction and noise. The network was optimized using a genetic algorithm and an accuracy of 97.7% and 92.6% was achieved for the MIT-BIH and eECG database respectively. Afterwards, transfer learning from the MIT-BIH dataset to the eECG database was applied after which the average accuracy, recall, positive predictive value and F1 score of the network increased with an accuracy of 97.1%.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000511157600001</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>31932667</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
CARDIAC-ARRHYTHMIAS; QRS DETECTION; EXERCISE; ELECTROCARDIOGRAM</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Van Steenkiste, Glenn; van Loon, Gunther] Univ Ghent, Dept Large Anim Internal Med, B-9000 Ghent, Belgium. <br>
[Crevecoeur, Guillaume] Univ Ghent, Dept Electromech Syst &amp; Met Engn, B-9000 Ghent, Belgium. <br>
[Crevecoeur, Guillaume] Core Lab EEDT DC, Flanders Make, Belgium. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Van Steenkiste, G (corresponding author), Univ Ghent, Dept Large Anim Internal Med, B-9000 Ghent, Belgium.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
Glenn.VanSteenkiste@ugent.be</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Ghent University; Ghent University</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Van Steenkiste, Glenn</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-0165-5215&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>van Loon, Gunther</display_name>&nbsp;</td><td>B-8526-2009&nbsp;</td><td>0000-0001-5191-5241&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Crevecoeur, Guillaume</display_name>&nbsp;</td><td>AAK-6330-2021&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Crevecoeur, Guillaume</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-7630-8579&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Van Steenkiste, Glenn</display_name>&nbsp;</td><td>AAU-9705-2020&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>NATURE PORTFOLIO</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>HEIDELBERGER PLATZ 3, BERLIN, 14197, GERMANY</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Multidisciplinary Sciences</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Science &amp; Technology - Other Topics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>KI2DP</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2045-2322</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>SCI REP-UK</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Sci Rep</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>12</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Research Foundation Flanders (FWO-Vlaanderen)</grant_agency>&nbsp;</td><td>
<div>1S56217N&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>Glenn Van Steenkiste is a PhD fellow of the Research Foundation Flanders (FWO-Vlaanderen) with grant 1S56217N.</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>Green Submitted, gold</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 17 of 32</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Quantitative proteomics and single-nucleus transcriptomics of the sinus node elucidates the foundation of cardiac pacemaking</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Linscheid, N (Linscheid, Nora); Logantha, SJRJ (Logantha, Sunil Jit R. J.); Poulsen, C (Poulsen, Camilla); Zhang, S (Zhang, Shanzhuo); Schr&ouml;lkamp, M (Schrolkamp, Maren); Egerod, KL (Egerod, Kristoffer Lihme); Thompson, JJ (Thompson, Jonatan James); Kitmitto, A (Kitmitto, Ashraf); Galli, G (Galli, Gina); Humphries, MJ (Humphries, Martin J.); Zhang, HG (Zhang, Henggui); Pers, TH (Pers, Tune H.); Olsen, JV (Olsen, Jesper Velgaard); Boyett, M (Boyett, Mark); Lundby, A (Lundby, Alicia)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>NATURE COMMUNICATIONS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>10</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>2889</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1038/s41467-019-10709-9</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2019 JUN 28</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>81</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>89</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
1</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
18</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>89</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
The sinus node is a collection of highly specialised cells constituting the heart's pacemaker. The molecular underpinnings of its pacemaking abilities are debated. Using high-resolution mass spectrometry, we here quantify &gt;7,000 proteins from sinus node and neighbouring atrial muscle. Abundances of 575 proteins differ between the two tissues. By performing single-nucleus RNA sequencing of sinus node biopsies, we attribute measured protein abundances to specific cell types. The data reveal significant differences in ion channels responsible for the membrane clock, but not in Ca2+ clock proteins, suggesting that the membrane clock underpins pacemaking. Consistently, incorporation of ion channel expression differences into a biophysically-detailed atrial action potential model result in pacemaking and a sinus node-like action potential. Combining our quantitative proteomics data with computational modeling, we estimate ion channel copy numbers for sinus node myocytes. Our findings provide detailed insights into the unique molecular make-up of the cardiac pacemaker.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000473132200053</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>31253831</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
MOUSE SINOATRIAL NODE; LEUCINE-RICH PROTEOGLYCANS; CELL-MATRIX ADHESIONS; EXTRACELLULAR-MATRIX; GENE-EXPRESSION; MESSENGER-RNA; COMPUTATIONAL PLATFORM; CALCIUM-CHANNELS; BETHLEM MYOPATHY; COLLAGEN</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Linscheid, Nora; Poulsen, Camilla; Schrolkamp, Maren; Lundby, Alicia] Univ Copenhagen, Fac Hlth &amp; Med Sci, Dept Biomed Sci, DK-2200 Copenhagen, Denmark. <br>
[Logantha, Sunil Jit R. J.; Kitmitto, Ashraf; Galli, Gina; Boyett, Mark] Univ Manchester, Div Cardiovasc Sci, Manchester M13 9NT, Lancs, England. <br>
[Zhang, Shanzhuo] Harbin Inst Technol, Sch Comp Sci &amp; Technol, Harbin 150006, Heilongjiang, Peoples R China. <br>
[Egerod, Kristoffer Lihme; Thompson, Jonatan James; Pers, Tune H.] Univ Copenhagen, Fac Hlth &amp; Med Sci, Novo Nordisk Fdn Ctr Basic Metab Res, DK-2200 Copenhagen, Denmark. <br>
[Humphries, Martin J.] Univ Manchester, Wellcome Trust Ctr Cell Matrix Res, Fac Biol Med &amp; Hlth, Manchester M13 9PT, Lancs, England. <br>
[Zhang, Henggui] Univ Manchester, Biol Phys Grp, Sch Phys &amp; Astron, Manchester M13 9PL, Lancs, England. <br>
[Olsen, Jesper Velgaard; Lundby, Alicia] Univ Copenhagen, Fac Hlth &amp; Med Sci, Novo Nordisk Fdn Ctr Prot Res, DK-2200 Copenhagen, Denmark. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Lundby, A (corresponding author), Univ Copenhagen, Fac Hlth &amp; Med Sci, Dept Biomed Sci, DK-2200 Copenhagen, Denmark.<br>Boyett, M (corresponding author), Univ Manchester, Div Cardiovasc Sci, Manchester M13 9NT, Lancs, England.<br>Lundby, A (corresponding author), Univ Copenhagen, Fac Hlth &amp; Med Sci, Novo Nordisk Fdn Ctr Prot Res, DK-2200 Copenhagen, Denmark.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
mark.boyett@manchester.ac.uk; alicia.lundby@sund.ku.dk</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
University of Copenhagen; University of Manchester; Harbin Institute of Technology; Novo Nordisk Foundation; University of Copenhagen; University of Manchester; University of Manchester; University of Copenhagen</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Lundby, Alicia</display_name>&nbsp;</td><td>H-3322-2013&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Egerod, Kristoffer</display_name>&nbsp;</td><td>D-9481-2012&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Egerod, Kristoffer Lihme</display_name>&nbsp;</td><td>D-9481-2012&nbsp;</td><td>0000-0003-2781-1594&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Galli, Gina</display_name>&nbsp;</td><td>ABC-9596-2020&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Humphries, Martin</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-4331-6967&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Thompson, Jonatan</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-4969-9927&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Logantha, Sunil Jit</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-8994-2296&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Schrolkamp, Maren</display_name>&nbsp;</td><td>R-1265-2016&nbsp;</td><td>0000-0003-3522-6379&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Galli, Gina</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-1023-915X&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Logantha, Sunil</display_name>&nbsp;</td><td>ABE-1403-2020&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Olsen, Jesper</display_name>&nbsp;</td><td>HLG-7809-2023&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Pers, Tune</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-0207-4831&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Zhang, Henggui</display_name>&nbsp;</td><td>AAM-1043-2020&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Schr&ouml;lkamp, Maren</display_name>&nbsp;</td><td>R-1265-2016&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Linscheid, Nora</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-8937-6247&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Olsen, Jesper</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-4747-4938&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>NATURE PORTFOLIO</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>HEIDELBERGER PLATZ 3, BERLIN, 14197, GERMANY</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Multidisciplinary Sciences</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Science &amp; Technology - Other Topics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>IF5ON</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>2041-1723</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>NAT COMMUN</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Nat. Commun.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>19</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Carlsberg Foundation</grant_agency>&nbsp;</td><td>
<div>CF17-0209&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Danish Council for independent Research</grant_agency>&nbsp;</td><td>
<div>DFF-4092-00045&nbsp;</div>
<div>DFF-6110-00166&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Lundbeck Foundation</grant_agency>&nbsp;</td><td>
<div>R19020143904&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>British Heart foundation</grant_agency>&nbsp;</td><td>
<div>RG/18/2/33392&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Wellcome Trust</grant_agency>&nbsp;</td><td>
<div>203128/Z/16/Z&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Novo Nordisk Foundation</grant_agency>&nbsp;</td><td>
<div>NNF14CC0001&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Lundbeck Foundation</grant_agency>&nbsp;</td><td>
<div>R190-2014-3904&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>NNF Center for Basic Metabolic Research</grant_agency>&nbsp;</td><td>
<div>Pers Group&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Novo Nordisk Fonden</grant_agency>&nbsp;</td><td>
<div>NNF16OC0021496&nbsp;</div>
<div>NNF14CC0001&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Novo Nordisk Foundation Center for Protein Research</grant_agency>&nbsp;</td><td>
<div>PI Jesper Velgaard Olsen&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>Supported by grants from the Carlsberg Foundation (CF17-0209) and The Danish Council for independent Research DFF-4092-00045 and DFF-6110-00166 to A.L., the Lundbeck Foundation to T.H.P. (R19020143904) and the British Heart foundation (RG/18/2/33392) to M.B. The Wellcome Trust Centre for Cell-Matrix Research, University of Manchester, is supported by core funding from the Wellcome Trust (203128/Z/16/Z). The Novo Nordisk Foundation Centre for Basic Metabolic Research is partially funded by an unconditional donation from the Novo Nordisk Foundation. The Novo Nordisk Foundation Centre for Protein Research is funded in part by a generous donation from the Novo Nordisk Foundation (NNF14CC0001). We would like to thank the staff of the Wellcome Centre for Cell Matrix Research Electron Microscope facility in the Faculty of Biology Medicine and Health, University of Manchester for their support. In particular, we acknowledge the support of staff members David J. Smith, Aleksandr Mironov and Tobias Starborg, as well as Andrew J. Atkinson, Abimbola Akerele and Tharsika Sritharan. We would furthermore like to thank the PRO-MS Danish National Mass Spectrometry Platform for Functional Proteomics and the CPR Mass Spectrometry Platform for instrument support and assistance. We would further like to thank Alexander Hogrebe for input on figure graphics. Some elements in Figs. 1a and 2b were downloaded and reused from Servier Medical Art [https://smart.servier.com/] by Servier licensed under a Creative Commons Attribution 3.0 Unported License [https://creativecommons.org/licences/by/3.0/].</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>Green Submitted, gold</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 18 of 32</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>&lt;i&gt; In situ&lt;/i&gt; transcriptome characteristics are lost following culture adaptation of adult cardiac stem cells</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Kim, T (Kim, Taeyong); Echeagaray, OH (Echeagaray, Oscar H.); Wang, BYJ (Wang, Bingyan J.); Casillas, A (Casillas, Alexandria); Broughton, KM (Broughton, Kathleen M.); Kim, BH (Kim, Bong-Hyun); Sussman, MA (Sussman, Mark A.)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>SCIENTIFIC REPORTS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>8</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>12060</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1038/s41598-018-30551-1</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2018 AUG 13</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>21</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>32</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
0</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
7</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>46</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Regenerative therapeutic approaches for myocardial diseases often involve delivery of stem cells expanded ex vivo. Prior studies indicate that cell culture conditions affect functional and phenotypic characteristics, but relationship(s) of cultured cells derived from freshly isolated populations and the heterogeneity of the cultured population remain poorly defined. Functional and phenotypic characteristics of ex vivo expanded cells will determine outcomes of interventional treatment for disease, necessitating characterization of the impact that ex vivo expansion has upon isolated stem cell populations. Single-cell RNA-Seq profiling (scRNA-Seq) was performed to determine consequences of culture expansion upon adult cardiac progenitor cells (CPCs) as well as relationships with other cell populations. Bioinformatic analyses demonstrate that identity marker genes expressed in freshly isolated cells become undetectable in cultured CPCs while low level expression emerges for thousands of other genes. Transcriptional profile of CPCs exhibited greater degree of similarity throughout the cultured population relative to freshly isolated cells. Findings were validated by comparative analyses using scRNA-Seq datasets of various cell types generated by multiple scRNA-Seq technology. Increased transcriptome diversity and decreased population heterogeneity in the cultured cell population may help account for reported outcomes associated with experimental and clinical use of CPCs for treatment of myocardial injury.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000441444100007</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>30104715</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
GENE-EXPRESSION; PROGENITOR CELLS; RNA-SEQ; EXPANSION; MOUSE; SERUM; THERAPY</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Kim, Taeyong; Echeagaray, Oscar H.; Wang, Bingyan J.; Casillas, Alexandria; Broughton, Kathleen M.; Sussman, Mark A.] San Diego State Univ, San Diego Heart Res Inst, San Diego, CA 92182 USA. <br>
[Kim, Bong-Hyun] Frederick Natl Lab Canc Sci, Washington, DC USA. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Sussman, MA (corresponding author), San Diego State Univ, San Diego Heart Res Inst, San Diego, CA 92182 USA.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
heartman4ever@icloud.com</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
California State University System; San Diego State University</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Wang, Bingyan</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-3522-2848&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Sussman, Mark</display_name>&nbsp;</td><td>H-2284-2014&nbsp;</td><td>0000-0002-0104-4799&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Kim, Taeyong</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-4147-3635&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Kim, Bong-Hyun</display_name>&nbsp;</td><td>A-1041-2009&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Wang, Bingyan</display_name>&nbsp;</td><td>AAB-3883-2019&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Sussman, Mark</display_name>&nbsp;</td><td>IYJ-9672-2023&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>NATURE PORTFOLIO</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>HEIDELBERGER PLATZ 3, BERLIN, 14197, GERMANY</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Multidisciplinary Sciences</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Science &amp; Technology - Other Topics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>GQ1ZT</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2045-2322</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>SCI REP-UK</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Sci Rep</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>9</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Basic Science Research Program through the National Research Foundation of Korea (NRF) - Ministry of Education</grant_agency>&nbsp;</td><td>
<div>2015R1A6A3A03019855&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>NIH</grant_agency>&nbsp;</td><td>
<div>R01HL067245&nbsp;</div>
<div>R37HL091102&nbsp;</div>
<div>R01HL105759&nbsp;</div>
<div>R01HL113647&nbsp;</div>
<div>R01HL117163&nbsp;</div>
<div>P01HL085577&nbsp;</div>
<div>R01HL122525&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Foundation Leducq</grant_agency>&nbsp;</td><td>
<div>600010319&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>The authors wish to thank all members of the Sussman lab for helpful input and valuable discussions. T.K. is supported by the Basic Science Research Program through the National Research Foundation of Korea (NRF) funded by the Ministry of Education (2015R1A6A3A03019855). M.A.S. is supported by NIH Grants: R01HL067245, R37HL091102, R01HL105759, R01HL113647, R01HL117163, P01HL085577, and R01HL122525, as well as an award from the Foundation Leducq (600010319).</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>gold, Green Submitted, Green Published</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 19 of 32</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Effects of time-restricted exercise on activity rhythms and exercise-induced adaptations in the heart</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Dial, MB (Dial, Michael B.); Malek, EM (Malek, Elias M.); Neblina, GA (Neblina, Greco A.); Cooper, AR (Cooper, Austin R.); Vaslieva, NI (Vaslieva, Nikoleta I.); Frommer, R (Frommer, Rebecca); Girgis, M (Girgis, Magdy); Dawn, B (Dawn, Buddhadeb); Mcginnis, GR (Mcginnis, Graham R.)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>SCIENTIFIC REPORTS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>14</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>1</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>146</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1038/s41598-023-50113-4</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2024 JAN 2</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>5</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>5</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
2</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
7</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>49</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Circadian rhythms play a crucial role in the regulation of various physiological processes, including cardiovascular function and metabolism. Exercise provokes numerous beneficial adaptations in heart, including physiological hypertrophy, and serves to shift circadian rhythms. This study investigated the impact of time-restricted exercise training on exercise-induced adaptations in the heart and locomotor activity rhythms. Male mice (n = 45) were allocated to perform voluntary, time-restricted exercise in the early active phase (EAP), late active phase (LAP), or remain sedentary (SED) for 6 weeks. Subsequently, mice were allowed 24-h ad libitum access to the running wheel to assess diurnal rhythms in locomotor activity. Heart weight and cross-sectional area were measured at sacrifice, and cardiac protein and gene expression levels were assessed for markers of mitochondrial abundance and circadian clock gene expression. Mice rapidly adapted to wheel running, with EAP mice exhibiting a significantly greater running distance compared to LAP mice. Time-restricted exercise induced a shift in voluntary wheel activity during the 24-h free access period, with the acrophase in activity being significantly earlier in EAP mice compared to LAP mice. Gene expression analysis revealed a higher expression of Per1 in LAP mice. EAP exercise elicited greater cardiac hypertrophy compared to LAP exercise. These findings suggest that the timing of exercise affects myocardial adaptations, with exercise in the early active phase inducing hypertrophy in the heart. Understanding the time-of-day dependent response to exercise in the heart may have implications for optimizing exercise interventions for cardiovascular health.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001142781100733</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>38168503</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
CIRCADIAN CLOCK; CARDIAC ADAPTATION; ENDURANCE EXERCISE; GENE-EXPRESSION; MICE; CONTRACTILE; METABOLISM; PATHWAYS; MOUSE</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Dial, Michael B.; Malek, Elias M.; Neblina, Greco A.; Cooper, Austin R.; Vaslieva, Nikoleta I.; Mcginnis, Graham R.] Univ Nevada, Dept Kinesiol &amp; Nutr Sci, 4505 S Maryland Pkwy,Bigelow Hlth Sci BHS Bldg 323, Las Vegas, NV 89154 USA. <br>
[Frommer, Rebecca; Girgis, Magdy; Dawn, Buddhadeb] Univ Nevada, Kirk Kerkorian Sch Med, Dept Internal Med, Las Vegas, NV USA. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Mcginnis, GR (corresponding author), Univ Nevada, Dept Kinesiol &amp; Nutr Sci, 4505 S Maryland Pkwy,Bigelow Hlth Sci BHS Bldg 323, Las Vegas, NV 89154 USA.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
graham.mcginnis@unlv.edu</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Nevada System of Higher Education (NSHE); University of Nevada Las Vegas; Nevada System of Higher Education (NSHE); University of Nevada Las Vegas</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Dial, Michael</display_name>&nbsp;</td><td>AAL-3089-2021&nbsp;</td><td>0000-0003-0675-4463&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Malek, Elias</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-3914-5357&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>NATURE PORTFOLIO</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>HEIDELBERGER PLATZ 3, BERLIN, 14197, GERMANY</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Multidisciplinary Sciences</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Science &amp; Technology - Other Topics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>EZ5U3</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2045-2322</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>SCI REP-UK</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Sci Rep</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>10</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Institute of General Medical Sciences</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>No Statement Available</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>Green Published, gold</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 20 of 32</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Unsupervised deep network for image texture transformation: Improving the quality of cross-correlation analysis and mechanical vortex visualisation during cardiac fibrillation</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Mangileva, D (Mangileva, Daria); Kursanov, A (Kursanov, Alexander); Katsnelson, L (Katsnelson, Leonid); Solovyova, O (Solovyova, Olga)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>HELIYON</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>9</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>11</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>e22207</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1016/j.heliyon.2023.e22207</value>&nbsp;&nbsp;<b>Early Access Date:</b> 
<value>NOV 2023</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2023 NOV 13</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>1</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>1</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
0</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
1</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>51</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Visualisation of cardiac fibrillation plays a very considerable role in cardiophysiological study and clinical applications. One of the ways to obtain the image of these phenomena is the registration of mechanical displacement fields reflecting the track from electrical activity. In this work, we read these fields using cross-correlation analysis from the video of open pig's epicardium at the start of fibrillation recorded with electrocardiogram. However, the quality of obtained displacement fields remains low due to the weak pixels heterogeneity of the frames. It disables to see more clearly such interesting phenomena as mechanical vortexes that underline the mechanical dysfunction of fibrillation. The applying of chemical or mechanical markers to solve this problem can affect the course of natural processes and falsify the results. Therefore, we developed a novel scheme of an unsupervised deep neural network that is based on the state-of-art positional coding technology for a multilayer perceptron. This network enables to generate a couple of frames with a more heterogeneous pixel texture, that is more suitable for cross-correlation analysis methods, from two consecutive frames. The novel network scheme was tested on synthetic pairs of images with different texture heterogeneity and frequency of displacement fields and also it was compared with different filters on our cardiac tissue image dataset. The testing showed that the displacement fields obtained with our method are closer to the ground truth than those which were computed only with the cross-correlation analysis in low contrast images case where filtering is impossible. Moreover, our model showed the best results comparing with the one of the popular filter CLAHE on our dataset. As a result, using our approach, it was possible to register more clearly a mechanical vortex on the epicardium at the start of fibrillation continuously for several milliseconds for the first time.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001121582500001</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>38053873</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
Unsupervised deep learning; Image texture transformation; Image cross-correlation analysis; Biomechanical displacement field</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
ENHANCEMENT; FIELDS</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Mangileva, Daria; Kursanov, Alexander; Katsnelson, Leonid; Solovyova, Olga] Ural Fed Univ, Dept Computat Math &amp; Comp Sci, Ekaterinburg 620000, Russia. <br>
[Kursanov, Alexander; Katsnelson, Leonid; Solovyova, Olga] Russian Acad Sci, Ural Branch, Inst Immunol &amp; Physiol, Ekaterinburg 620049, Russia. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Mangileva, D (corresponding author), Ural Fed Univ, Dept Computat Math &amp; Comp Sci, Ekaterinburg 620000, Russia.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
daria.mangileva@urfu.ru</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Ural Federal University; Russian Academy of Sciences; Institute of Immunology &amp; Physiology UB RAS</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Mangileva, Daria</display_name>&nbsp;</td><td>AAC-7980-2022&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Kursanov, Alexander</display_name>&nbsp;</td><td>A-4003-2015&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Solovyova, Olga</display_name>&nbsp;</td><td>T-8232-2017&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>CELL PRESS</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>50 HAMPSHIRE ST, FLOOR 5, CAMBRIDGE, MA 02139 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Multidisciplinary Sciences</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Science &amp; Technology - Other Topics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>AW8Q6</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>2405-8440</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>HELIYON</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Heliyon</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>12</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Ministry of Science and Higher Education of the Russian Federation</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>The research funding from the Ministry of Science and Higher Education of the Russian Federation (Ural Federal University project within the Priority-2030 Program) is gratefully acknowledged.</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>Green Submitted, gold</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 21 of 32</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Metabolic adaptations during extreme anoxia in the turtle heart and their implications for ischemia-reperfusion injury</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Bundgaard, A (Bundgaard, Amanda); James, AM (James, Andrew M.); Gruszczyk, AV (Gruszczyk, Anja V.); Martin, J (Martin, Jack); Murphy, MP (Murphy, Michael P.); Fago, A (Fago, Angela)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>SCIENTIFIC REPORTS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>9</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>2850</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1038/s41598-019-39836-5</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2019 FEB 26</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>65</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>72</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
0</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
12</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>50</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
ATP depletion and succinate accumulation during ischemia lead to oxidative damage to mammalian organs upon reperfusion. In contrast, freshwater turtles survive weeks of anoxia at low temperatures without suffering from oxidative damage upon reoxygenation, but the mechanisms are unclear. To determine how turtles survive prolonged anoxia, we measured similar to 80 metabolites in hearts from cold-acclimated (5 degrees C) turtles exposed to 9 days anoxia and compared the results with those for normoxic turtles (25 degrees C) and mouse hearts exposed to 30 min of ischemia. In turtles, ATP and ADP decreased to new steady-state levels during fasting and cold-acclimation and further with anoxia, but disappeared within 30 min of ischemia in mouse hearts. High NADH/NAD(+) ratios were associated with succinate accumulation in both anoxic turtles and ischemic mouse hearts. However, succinate concentrations and succinate/fumarate ratios were lower in turtle than in mouse heart, limiting the driving force for production of reactive oxygen species (ROS) upon reoxygenation in turtles. Furthermore, we show production of ROS from succinate is prevented by re-synthesis of ATP from ADP. Thus, maintenance of an ATP/ADP pool and low succinate accumulation likely protects turtle hearts from anoxia/reoxygenation injury and suggests metabolic interventions as a therapeutic approach to limit ischemia/reperfusion injury in mammals.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000459702900019</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>30808950</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
TRACHEMYS-SCRIPTA; CHRYSEMYS-PICTA; TOLERANT VERTEBRATE; HYPOXIA TOLERANCE; END-PRODUCTS; OXYGEN; WATER; SUBMERGENCE; 3-DEGREES-C; MECHANISMS</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Bundgaard, Amanda; Fago, Angela] Aarhus Univ, Dept Biosci, Aarhus, Denmark. <br>
[Bundgaard, Amanda; James, Andrew M.; Gruszczyk, Anja V.; Martin, Jack; Murphy, Michael P.] Univ Cambridge, MRC Mitochondrial Biol Unit, Cambridge, England. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Fago, A (corresponding author), Aarhus Univ, Dept Biosci, Aarhus, Denmark.<br>Murphy, MP (corresponding author), Univ Cambridge, MRC Mitochondrial Biol Unit, Cambridge, England.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
mpm@mrc-mbu.cam.ac.uk; angela.fago@bios.au.dk</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Aarhus University; University of Cambridge</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Fago, Angela</display_name>&nbsp;</td><td>J-5946-2013&nbsp;</td><td>0000-0001-7315-2628&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Murphy, Michael</display_name>&nbsp;</td><td>C-2120-2009&nbsp;</td><td>0000-0003-1115-9618&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>James, Andrew</display_name>&nbsp;</td><td>A-2639-2015&nbsp;</td><td>0000-0002-0515-9649&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>NATURE PORTFOLIO</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>HEIDELBERGER PLATZ 3, BERLIN, 14197, GERMANY</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Multidisciplinary Sciences</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Science &amp; Technology - Other Topics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>HM8BH</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2045-2322</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>SCI REP-UK</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Sci Rep</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>10</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Independent Research Fund Denmark, Natural Sciences</grant_agency>&nbsp;</td><td>
<div>4181-00094&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Medical Research Council UK</grant_agency>&nbsp;</td><td>
<div>MC_U105663142&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Wellcome Trust Investigator award</grant_agency>&nbsp;</td><td>
<div>110159/Z/15/Z&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Graduate School of Science and Technology, Aarhus University</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Niels Bohr Foundation</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>MRC</grant_agency>&nbsp;</td><td>
<div>MR/L017520/1&nbsp;</div>
<div>MC_UU_00015/3&nbsp;</div>
<div>MC_U105663142&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Medical Research Council</grant_agency>&nbsp;</td><td>
<div>MC_UU_00015/3&nbsp;</div>
<div>MC_U105663142&nbsp;</div>
<div>MR/L017520/1&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Wellcome Trust</grant_agency>&nbsp;</td><td>
<div>110158/Z/15/Z&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>The authors would like to acknowledge Ana S. H. Costa and Christian Frezza from the Medical Research Council Mitochondrial Biology Unit, Cambridge, UK for the mass spectrometry sample analysis and for valuable comments on the work. This work was funded by the Independent Research Fund Denmark, Natural Sciences (grant 4181-00094 to A.F.), the Medical Research Council UK (grant MC_U105663142 to M.P.M.), the Wellcome Trust Investigator award (grant 110159/Z/15/Z to M.P.M.), and by grants from the Graduate School of Science and Technology, Aarhus University, and the Niels Bohr Foundation (to A.B.).</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>gold, Green Submitted</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 22 of 32</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Adenoviral VEGF-B186R127S gene transfer induces angiogenesis and improves perfusion in ischemic heart</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Korpela, H (Korpela, Henna); H&auml;tinen, OP (Hatinen, Olli-Pekka); Nieminen, T (Nieminen, Tiina); Mallick, R (Mallick, Rahul); Toivanen, P (Toivanen, Pyry); Airaksinen, J (Airaksinen, Jonna); Valli, K (Valli, Kaisa); Hakulinen, M (Hakulinen, Mikko); Poutiainen, P (Poutiainen, Pekka); Nurro, J (Nurro, Jussi); Yl&auml;-Herttuala, S (Yla-Herttuala, Seppo)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>ISCIENCE</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>24</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>12</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>103533</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1016/j.isci.2021.103533</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2021 DEC 17</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>9</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>10</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
1</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
3</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>33</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Vascular endothelial growth factor B (VEGF-B) is an interesting therapeutic candidate for coronary artery disease. However, it can also cause ventricular arrhythmias, potentially preventing its use in clinics. We cloned VEGF-B isoforms with different receptor binding profiles to clarify the roles of VEGFR-1 and Nrp-1 in angiogenesis and to see if angiogenic properties can be maintained while avoiding side effects. VEGF-B constructs were studied in vivo using adenovirus (Ad)-mediated intramyocardial gene transfers into the normoxic and ischemic porcine heart (n = 51). It was found that the unprocessed isoform VEGF-B186R127S is as efficient angiogenic growth factor as the native VEGF-B186 in normoxic and ischemic heart. In addition, AdVEGF-B186R127S increased myocardial perfusion reserve by 22% in ischemic heart without any side effects. AdVEGF-B127 (VEGFR-1 and Nrp-1 ligand) and AdVEGF-B109 (VEGFR-1 ligand) did not induce angiogenesis. Thus, VEGF-B186 is angiogenic only before its proteolytic processing to VEGF-B127. Only the VEGF-B186 C-terminal fragment was associated with arrhythmias.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000740243000005</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>34917905</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
GROWTH-FACTOR-B; VEGF-B; NEUROPILIN-1; ACTIVATION; THERAPY; BINDING; LACKING; SPLICE; MOUSE</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Korpela, Henna; Hatinen, Olli-Pekka; Nieminen, Tiina; Mallick, Rahul; Toivanen, Pyry; Airaksinen, Jonna; Valli, Kaisa; Nurro, Jussi; Yla-Herttuala, Seppo] Univ Eastern Finland, AI Virtanen Inst Mol Sci, Kuopio, Finland. <br>
[Nieminen, Tiina] Kuopio Ctr Gene &amp; Cell Therapy, Kuopio, Finland. <br>
[Hakulinen, Mikko; Poutiainen, Pekka] Kuopio Univ Hosp, Kuopio, Finland. <br>
[Yla-Herttuala, Seppo] Kuopio Univ Hosp, Heart Ctr, Kuopio, Finland. <br>
[Yla-Herttuala, Seppo] Kuopio Univ Hosp, Gene Therapy Unit, Kuopio, Finland. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Yl&auml;-Herttuala, S (corresponding author), Univ Eastern Finland, AI Virtanen Inst Mol Sci, Kuopio, Finland.<br>Yl&auml;-Herttuala, S (corresponding author), Kuopio Univ Hosp, Heart Ctr, Kuopio, Finland.<br>Yl&auml;-Herttuala, S (corresponding author), Kuopio Univ Hosp, Gene Therapy Unit, Kuopio, Finland.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
seppo.ylaherttuala@uef.fi</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
University of Eastern Finland; Kuopio University Hospital; University of Eastern Finland; University of Eastern Finland Hospital; Kuopio University Hospital; University of Eastern Finland; University of Eastern Finland Hospital; Kuopio University Hospital; University of Eastern Finland; University of Eastern Finland Hospital</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Hakulinen, Mikko</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-8408-8710&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Komi, Henna</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-5343-496X&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Mallick, Rahul</display_name>&nbsp;</td><td>AAL-6843-2020&nbsp;</td><td>0000-0002-4065-3474&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>CELL PRESS</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>50 HAMPSHIRE ST, FLOOR 5, CAMBRIDGE, MA 02139 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Multidisciplinary Sciences</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Science &amp; Technology - Other Topics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>YD2YV</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>2589-0042</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>ISCIENCE</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>iScience</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>15</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Finnish Academy</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>ERC Advanced Grant</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Finnish Medical Foundation</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Finnish Cardiac Society</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>CardioReGenix EU Horizon 2020 grant</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This study was supported by grants from Finnish Academy, ERC Advanced Grant, The Finnish Medical Foundation, Finnish Cardiac Society, and CardioReGenix EU Horizon 2020 grant.</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>gold, Green Submitted</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 23 of 32</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Sinus venosus adaptation models prolonged cardiovascular disease and reveals insights into evolutionary transitions of the vertebrate heart</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Gafranek, JT (Gafranek, Jacob T.); D'Aniello, E (D'Aniello, Enrico); Ravisankar, P (Ravisankar, Padmapriyadarshini); Thakkar, K (Thakkar, Kairavee); Vagnozzi, RJ (Vagnozzi, Ronald J.); Lim, HW (Lim, Hee-Woong); Salomonis, N (Salomonis, Nathan); Waxman, JS (Waxman, Joshua S.)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>NATURE COMMUNICATIONS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>14</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>1</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>5509</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1038/s41467-023-41184-y</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2023 SEP 7</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>3</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>3</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
2</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
18</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>149</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
How two-chambered hearts in basal vertebrates have evolved from single-chamber hearts found in ancestral chordates remains unclear. Here, we show that the teleost sinus venosus (SV) is a chamber-like vessel comprised of an outer layer of smooth muscle cells. We find that in adult zebrafish nr2f1a mutants, which lack atria, the SV comes to physically resemble the thicker bulbus arteriosus (BA) at the arterial pole of the heart through an adaptive, hypertensive response involving smooth muscle proliferation due to aberrant hemodynamic flow. Single cell transcriptomics show that smooth muscle and endothelial cell populations within the adapting SV also take on arterial signatures. Bulk transcriptomics of the blood sinuses flanking the tunicate heart reinforce a model of greater equivalency in ancestral chordate BA and SV precursors. Our data simultaneously reveal that secondary complications from congenital heart defects can develop in adult zebrafish similar to those in humans and that the foundation of equivalency between flanking auxiliary vessels may remain latent within basal vertebrate hearts.
<br>Nr2fs are conserved transcription factors that regulate atrial chamber and venous development. Here, the authors use adult zebrafish nr2f1a mutants to investigate compensatory remodeling of the inflow tract and hypotheses of cardiac evolution.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001065300300025</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>37679366</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
PLURIPOTENT STEM-CELLS; SMOOTH-MUSCLE-CELLS; CIONA-INTESTINALIS; ARTERIAL POLE; NEURAL CREST; PULMONARY-HYPERTENSION; MORPHOLOGICAL ANALYSIS; TRANSCRIPTION FACTOR; CARDIAC DEVELOPMENT; CHROMAFFIN TISSUE</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Gafranek, Jacob T.] Univ Cincinnati, Coll Med, Mol &amp; Dev Biol Grad Program, Cincinnati, OH 45267 USA. <br>
[Gafranek, Jacob T.; Ravisankar, Padmapriyadarshini; Waxman, Joshua S.] Cincinnati Childrens Hosp Med Ctr, Div Mol Cardiovasc Biol, Cincinnati, OH 45229 USA. <br>
[Gafranek, Jacob T.; Ravisankar, Padmapriyadarshini; Waxman, Joshua S.] Cincinnati Childrens Hosp Med Ctr, Heart Inst, Cincinnati, OH 45229 USA. <br>
[D'Aniello, Enrico] Stn Zool Anton Dohrn, Dept Biol &amp; Evolut Marine Organisms, I-80121 Naples, Italy. <br>
[Thakkar, Kairavee; Lim, Hee-Woong; Salomonis, Nathan] Cincinnati Childrens Hosp Med Ctr, Div Biomed Informat, Cincinnati, OH 45229 USA. <br>
[Thakkar, Kairavee] Univ Cincinnati, Coll Med, Dept Pharmacol &amp; Syst Physiol, Cincinnati, OH 45267 USA. <br>
[Vagnozzi, Ronald J.] Univ Colorado Anschutz Med Campus, Gates Ctr Regenerat Med, Div Cardiol, Consortium Fibrosis Res &amp; Translat CFReT, Aurora, CO 80045 USA. <br>
[Lim, Hee-Woong; Salomonis, Nathan; Waxman, Joshua S.] Univ Cincinnati, Coll Med, Dept Pediat, Cincinnati, OH 45267 USA. <br>
[Waxman, Joshua S.] Cincinnati Childrens Hosp Med Ctr, Div Dev Biol, Cincinnati, OH 45229 USA. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Waxman, JS (corresponding author), Cincinnati Childrens Hosp Med Ctr, Div Mol Cardiovasc Biol, Cincinnati, OH 45229 USA.<br>Waxman, JS (corresponding author), Cincinnati Childrens Hosp Med Ctr, Heart Inst, Cincinnati, OH 45229 USA.<br>Waxman, JS (corresponding author), Univ Cincinnati, Coll Med, Dept Pediat, Cincinnati, OH 45267 USA.<br>Waxman, JS (corresponding author), Cincinnati Childrens Hosp Med Ctr, Div Dev Biol, Cincinnati, OH 45229 USA.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
joshua.waxman@cchmc.org</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
University System of Ohio; University of Cincinnati; Cincinnati Children's Hospital Medical Center; Cincinnati Children's Hospital Medical Center; Stazione Zoologica Anton Dohrn; Cincinnati Children's Hospital Medical Center; University System of Ohio; University of Cincinnati; University of Colorado System; University of Colorado Anschutz Medical Campus; University System of Ohio; University of Cincinnati; Cincinnati Children's Hospital Medical Center</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Waxman, Joshua</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-8132-487X&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>D'Aniello, Enrico</display_name>&nbsp;</td><td>HRD-6860-2023&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Thakkar, Kairavee</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-1397-486X&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Vagnozzi, Ronald</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-8027-1809&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Lim, Hee-Woong</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-8599-1808&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Salomonis, Nathan</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-9689-2469&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Vagnozzi, Ronald</display_name>&nbsp;</td><td>AAR-6891-2020&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Lim, Hee-Woong</display_name>&nbsp;</td><td>AAX-6835-2021&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>NATURE PORTFOLIO</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>HEIDELBERGER PLATZ 3, BERLIN, 14197, GERMANY</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Multidisciplinary Sciences</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Science &amp; Technology - Other Topics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>R6FU0</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>2041-1723</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>NAT COMMUN</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Nat. Commun.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>19</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Institutes of Health</grant_agency>&nbsp;</td><td>
<div>T32 HL125204&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Ruth L. Kirschstein Predoctoral Individual National Research Service Award</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>&nbsp;</td><td>
<div>R01 HL137766&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This manuscript was funded by the National Institutes of Health grants R01 HL137766, R01 HL141186, and R01 HL154522 to J.S.W., and Ruth L. Kirschstein Predoctoral Individual National Research Service Award F31 HL147399 and Training Grant T32 HL125204 to J.T.G.</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>gold</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 24 of 32</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Polygenic adaptation and convergent evolution on growth and cardiac genetic pathways in African and Asian rainforest hunter-gatherers</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Bergey, CM (Bergey, Christina M.); Lopez, M (Lopez, Marie); Harrison, GF (Harrison, Genelle F.); Patin, E (Patin, Etienne); Cohen, JA (Cohen, Jacob A.); Quintana-Murci, L (Quintana-Murci, Lluis); Barreiro, LB (Barreiro, Luis B.); Perry, GH (Perry, George H.)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>115</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>48</value>&nbsp;&nbsp;<b>Pages:</b> 
<value>E11256-E11263</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1073/pnas.1812135115</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2018 NOV 27</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>30</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>31</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
0</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
14</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>71</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Different human populations facing similar environmental challenges have sometimes evolved convergent biological adaptations, for example, hypoxia resistance at high altitudes and depigmented skin in northern latitudes on separate continents. The "pygmy" phenotype (small adult body size), characteristic of hunter-gatherer populations inhabiting both African and Asian tropical rainforests, is often highlighted as another case of convergent adaptation in humans. However, the degree to which phenotypic convergence in this polygenic trait is due to convergent versus population-specific genetic changes is unknown. To address this question, we analyzed high-coverage sequence data from the protein-coding portion of the genomes of two pairs of populations: Batwa rainforest hunter-gatherers and neighboring Bakiga agriculturalists from Uganda and Andamanese rainforest hunter-gatherers and Brahmin agriculturalists from India. We observed signatures of convergent positive selection between the rainforest hunter-gatherers across the set of genes with "growth factor binding" functions (P &lt; 0 : 001). Unexpectedly, for the rainforest groups, we also observed convergent and populationspecific signatures of positive selection in pathways related to cardiac development (e.g., "cardiac muscle tissue development"; P = 0.001). We hypothesize that the growth hormone subresponsiveness likely underlying the adult small body-size phenotype may have led to compensatory changes in cardiac pathways, in which this hormone also plays an essential role. Importantly, in the agriculturalist populations, we did not observe similar patterns of positive selection on sets of genes associated with growth or cardiac development, indicating our results most likely reflect a history of convergent adaptation to the similar ecology of rainforests rather than a more general evolutionary pattern.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000451351000012</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>30413626</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Author Keywords:</b>
rainforest hunter-gatherers; convergent evolution; population genomics; stature</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
BLOOD-PRESSURE; POSITIVE SELECTION; HORMONE DEFICIENCY; PYGMY PHENOTYPE; SHORT STATURE; REVEALS; ASSOCIATION; SIGNATURES; HEIGHT; INSIGHTS</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Bergey, Christina M.; Perry, George H.] Penn State Univ, Dept Anthropol, University Pk, PA 16802 USA. <br>
[Bergey, Christina M.; Cohen, Jacob A.; Perry, George H.] Penn State Univ, Dept Biol, University Pk, PA 16802 USA. <br>
[Lopez, Marie; Patin, Etienne; Quintana-Murci, Lluis] Inst Pasteur, Unit Human Evolutionary Genet, F-75015 Paris, France. <br>
[Lopez, Marie; Patin, Etienne; Quintana-Murci, Lluis] CNRS, Genom Evolut Modelisat &amp; Sante, UMR 2000, F-75015 Paris, France. <br>
[Lopez, Marie; Patin, Etienne; Quintana-Murci, Lluis] Inst Pasteur, Ctr Bioinformat Biostat &amp; Integrat Biol, F-75015 Paris, France. <br>
[Harrison, Genelle F.; Barreiro, Luis B.] Univ Montreal, Ctr Hosp Univ CHU St Justine, Ctr Rech, Montreal, PQ H3T 1C5, Canada. <br>
[Perry, George H.] Penn State Univ, Huck Inst Life Sci, University Pk, PA 16802 USA. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Bergey, CM; Perry, GH (corresponding author), Penn State Univ, Dept Anthropol, University Pk, PA 16802 USA.<br>Bergey, CM; Perry, GH (corresponding author), Penn State Univ, Dept Biol, University Pk, PA 16802 USA.<br>Perry, GH (corresponding author), Penn State Univ, Huck Inst Life Sci, University Pk, PA 16802 USA.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
cxb585@psu.edu; ghp3@psu.edu</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Pennsylvania Commonwealth System of Higher Education (PCSHE); Pennsylvania State University; Pennsylvania State University - University Park; Pennsylvania Commonwealth System of Higher Education (PCSHE); Pennsylvania State University; Pennsylvania State University - University Park; Pasteur Network; Universite Paris Cite; Institut Pasteur Paris; Centre National de la Recherche Scientifique (CNRS); Pasteur Network; Universite Paris Cite; Institut Pasteur Paris; Universite de Montreal; Centre Hospitalier Universitaire Sainte-Justine; Pennsylvania Commonwealth System of Higher Education (PCSHE); Pennsylvania State University; Pennsylvania State University - University Park</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Barreiro, Luis</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-9456-367X&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Patin, Etienne</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-9911-4459&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Patin, Etienne</display_name>&nbsp;</td><td>G-6768-2012&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Quintana-Murci, Lluis</display_name>&nbsp;</td><td>AAB-7468-2022&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Bergey, Christina</display_name>&nbsp;</td><td>ADV-1840-2022&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Barreiro, Luis</display_name>&nbsp;</td><td>I-7772-2015&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Cohen, Jacob</display_name>&nbsp;</td><td>&nbsp;</td><td>0009-0007-9264-0409&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Bergey, Christina</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-8336-8078&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Harrison, Genelle</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-3386-8159&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>NATL ACAD SCIENCES</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>2101 CONSTITUTION AVE NW, WASHINGTON, DC 20418 USA</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Multidisciplinary Sciences</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Science &amp; Technology - Other Topics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>HB8OX</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>0027-8424</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>1091-6490</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>P NATL ACAD SCI USA</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Proc. Natl. Acad. Sci. U. S. A.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>8</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>NIH</grant_agency>&nbsp;</td><td>
<div>R01-GM115656&nbsp;</div>
<div>1 F32 GM125228-01A1&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Agence Nationale de la Recherche AGRHUM</grant_agency>&nbsp;</td><td>
<div>ANR-14-CE02-0003-01&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Fondation pour la Recherche Medicale</grant_agency>&nbsp;</td><td>
<div>FDT20170436932&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Agence Nationale de la Recherche (ANR)</grant_agency>&nbsp;</td><td>
<div>ANR-14-CE02-0003&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>We thank the Batwa and Bakiga communities and all individuals who participated in this study and J. A. Hodgson and E. C. Reeves for helpful discussions. This work was supported by NIH Grant R01-GM115656 (to G.H.P. and L.B.B.), NIH Grant 1 F32 GM125228-01A1 (to C.M.B), and Agence Nationale de la Recherche AGRHUM ANR-14-CE02-0003-01 (to L.Q.-M.). M.L. was supported by the Fondation pour la Recherche Medicale (FDT20170436932). This research was conducted with Advanced CyberInfrastructure computational resources provided by The Institute for CyberScience at The Pennsylvania State University.</p>
</span></td>
</tr>
<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 25 of 32</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Heart rate variability adaptation due to head-down tilt bed rest: insights from pooled data of six campaigns</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Solbiati, S (Solbiati, Sarah); Caiani, EG (Caiani, Enrico Gianluca)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>NPJ MICROGRAVITY</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>11</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>1</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>48</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1038/s41526-025-00509-5</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2025 JUL 31</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>0</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>0</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
2</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
2</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>42</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Microgravity challenges cardiovascular regulation, necessitating adaptive responses to maintain homeostasis. This retrospective study examines the changes in heart rate variability (HRV) induced by head-down tilt (HDT) bed rest, a ground-based analog for microgravity. Standardized 24-h Holter electrocardiograms (ECG) were collected from 66 male volunteers across six ESA HDT campaigns of different durations (5, 21, and 60 days). HRV indices, including time, frequency, and nonlinear metrics, were computed to evaluate autonomic modulation during HDT and following re-exposure to orthostatic stress. Results showed decreased high- and low-frequency power during HDT, indicating impaired autonomic modulation. Additional alterations were observed at HDT discontinuation, likely reflecting cardiovascular deconditioning and gravity readaptation. The use of harmonized protocols and a large pooled dataset provided valuable insights into cardiovascular adaptation to HDT. These findings emphasize the importance of standardized protocols and large pooled datasets in future studies investigating cardiovascular responses to simulated microgravity.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001540937200001</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>40744934</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
TERM SPACE-FLIGHT; CIRCADIAN-RHYTHMS; RATE DYNAMICS; SLEEP; MICROGRAVITY; PRESSURE; PATTERNS</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Solbiati, Sarah; Caiani, Enrico Gianluca] Politecn Milan, Dept Elect Informat &amp; Bioengn, Milan, Italy. <br>
[Caiani, Enrico Gianluca] IRCCS Ist Auxol Italiano, Milan, Italy. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Caiani, EG (corresponding author), Politecn Milan, Dept Elect Informat &amp; Bioengn, Milan, Italy.<br>Caiani, EG (corresponding author), IRCCS Ist Auxol Italiano, Milan, Italy.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
enrico.caiani@polimi.it</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Polytechnic University of Milan; IRCCS Istituto Auxologico Italiano</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>CAIANI, Enrico</display_name>&nbsp;</td><td>I-1903-2019&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>NATURE PORTFOLIO</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>HEIDELBERGER PLATZ 3, BERLIN, 14197, GERMANY</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Multidisciplinary Sciences</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Science &amp; Technology - Other Topics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>5PB1W</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>2373-8065</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>NPJ MICROGRAVITY</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>NPJ Microgravity</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>14</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Agenzia Spaziale Italiana</grant_agency>&nbsp;</td><td>
<div>2013-033-R.0&nbsp;</div>
<div>2018-7-U.0 (QT-BED)&nbsp;</div>
<div>2022-9-U.0&nbsp;</div>
<div>2022-10-U.0&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Italian Space Agency</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>We would like to thank all the involved ESA, MEDES, DLR personnel for their invaluable competence in organizing such experiments, as well as all the participating volunteers. This work was supported by the Italian Space Agency (Contracts 2013-033-R.0, 2018-7-U.0 (QT-BED), 2022-9-U.0 (CMR-PP), 2022-10-U.0 (CMR-SDM), PI and recipient EC).</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>gold, Green Submitted</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 26 of 32</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Metabolic and Cardiac Adaptation to Chronic Pharmacologic Blockade of Facilitative Glucose Transport in Murine Dilated Cardiomyopathy and Myocardial Ischemia</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Heitmeier, MR (Heitmeier, Monique R.); Payne, MA (Payne, Maria A.); Weinheimer, C (Weinheimer, Carla); Kovacs, A (Kovacs, Attila); Hresko, RC (Hresko, Richard C.); Jay, PY (Jay, Patrick Y.); Hruz, PW (Hruz, Paul W.)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>SCIENTIFIC REPORTS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>8</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>6475</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1038/s41598-018-24867-1</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2018 APR 24</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>9</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>11</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
0</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
6</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>48</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
GLUT transgenic and knockout mice have provided valuable insight into the role of facilitative glucose transporters (GLUTs) in cardiovascular and metabolic disease, but compensatory physiological changes can hinder interpretation of these models. To determine whether adaptations occur in response to GLUT inhibition in the failing adult heart, we chronically treated TG9 mice, a transgenic model of dilated cardiomyopathy and heart failure, with the GLUT inhibitor ritonavir. Glucose tolerance was significantly improved with chronic treatment and correlated with decreased adipose tissue retinol binding protein 4 (RBP4) and resistin. A modest improvement in lifespan was associated with decreased cardiomyocyte brain natriuretic peptide (BNP) expression, a marker of heart failure severity. GLUT1 and -12 protein expression was significantly increased in left ventricular (LV) myocardium in ritonavir-treated animals. Supporting a switch from fatty acid to glucose utilization in these tissues, fatty acid transporter CD36 and fatty acid transcriptional regulator peroxisome proliferator-activated receptor alpha (PPAR alpha) mRNA were also decreased in LV and soleus muscle. Chronic ritonavir also increased cardiac output and dV/dt-d in C57B1/6 mice following ischemia-reperfusion injury. Taken together, these data demonstrate compensatory metabolic adaptation in response to chronic GLUT blockade as a means to evade deleterious changes in the failing heart.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000430662500004</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>29691457</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
HIV-PROTEASE INHIBITORS; INSULIN-RESISTANCE; HEART-FAILURE; ANTIRETROVIRAL THERAPY; CARDIOVASCULAR-DISEASE; MOLECULAR-MECHANISMS; ADIPOSE-TISSUE; OVEREXPRESSION; RITONAVIR; DISPOSAL</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Heitmeier, Monique R.; Payne, Maria A.; Hresko, Richard C.; Jay, Patrick Y.; Hruz, Paul W.] Washington Univ, Sch Med, Dept Pediat, St Louis, MO 63110 USA. <br>
[Weinheimer, Carla; Kovacs, Attila] Washington Univ, Sch Med, Dept Internal Med, St Louis, MO 63110 USA. <br>
[Jay, Patrick Y.] Washington Univ, Sch Med, Dept Genet, St Louis, MO 63110 USA. <br>
[Hruz, Paul W.] Washington Univ, Sch Med, Dept Cell Biol &amp; Physiol, St Louis, MO 63110 USA. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Hruz, PW (corresponding author), Washington Univ, Sch Med, Dept Pediat, St Louis, MO 63110 USA.<br>Hruz, PW (corresponding author), Washington Univ, Sch Med, Dept Cell Biol &amp; Physiol, St Louis, MO 63110 USA.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
hruz_p@kids.wustl.edu</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Washington University (WUSTL); Washington University (WUSTL); Washington University (WUSTL); Washington University (WUSTL)</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Hruz, Paul W</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-1478-3355&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Hruz, Paul</display_name>&nbsp;</td><td>LDE-7189-2024&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>NATURE PORTFOLIO</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>HEIDELBERGER PLATZ 3, BERLIN, 14197, GERMANY</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Multidisciplinary Sciences</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Science &amp; Technology - Other Topics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>GD7BB</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2045-2322</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>SCI REP-UK</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Sci Rep</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>10</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Institutes of Health</grant_agency>&nbsp;</td><td>
<div>HL092798&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Children's Discovery Institute of Washington University</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>St. Louis Children's Hospital</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This study was supported in part by grants from the National Institutes of Health (HL092798), the Children's Discovery Institute of Washington University and St. Louis Children's Hospital. Echocardiography was performed in the Mouse Phenotyping Core Facility at Washington University.</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>gold, Green Submitted</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 27 of 32</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>AAV delivery strategy with mechanical support for safe and efficacious cardiac gene transfer in swine</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Mazurek, R (Mazurek, Renata); Tharakan, S (Tharakan, Serena); Mavropoulos, SA (Mavropoulos, Spyros A.); Singleton, DT (Singleton, Deanndria T.); Bikou, O (Bikou, Olympia); Sakata, T (Sakata, Tomoki); Kariya, T (Kariya, Taro); Yamada, K (Yamada, Kelly); Kohlbrenner, E (Kohlbrenner, Erik); Liang, LF (Liang, Lifan); Ravichandran, AJ (Ravichandran, Anjali J.); Watanabe, S (Watanabe, Shin); Hajjar, RJ (Hajjar, Roger J.); Ishikawa, K (Ishikawa, Kiyotake)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>NATURE COMMUNICATIONS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>15</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>1</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>10450</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1038/s41467-024-54635-x</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2024 DEC 1</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>4</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>6</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
5</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
10</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>39</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Adeno-associated virus-based gene therapy is a promising avenue in heart failure treatment, but has shown limited cardiac virus uptake in humans, requiring new approaches for clinical translation. Using a Yorkshire swine ischemic heart failure model, we demonstrate significant improvement in gene uptake with temporary coronary occlusions assisted by mechanical circulatory support. We first show that mechanical support during coronary artery occlusions prevents hemodynamic deterioration (n = 5 female). Subsequent experiments show that coronary artery occlusions during gene delivery improve gene transduction, while adding coronary sinus occlusion (Stop-flow) further improves gene expression up to &gt;1 million-fold relative to conventional intracoronary infusion. Complete survival during and after delivery (n = 10 female, n = 10 male) further indicates safety of the approach. Improved cardiac gene expression correlates with virus uptake without an increase in extra-cardiac expression. Stop-flow delivery of virus-sized gold nanoparticles exhibits enhanced endothelial adherence and uptake, suggesting a mechanism independent of virus biology. Together, utilizing mechanical support for cardiac gene delivery offers a clinically-applicable strategy for heart failure-targeted therapies.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001377329000006</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>39617804</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
PERCUTANEOUS CORONARY INTERVENTION; PRESSURE-REGULATED RETROINFUSION; HEART; THERAPY; EXPRESSION; OUTCOMES; MODEL</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Mazurek, Renata; Tharakan, Serena; Mavropoulos, Spyros A.; Singleton, Deanndria T.; Bikou, Olympia; Sakata, Tomoki; Kariya, Taro; Yamada, Kelly; Kohlbrenner, Erik; Liang, Lifan; Ravichandran, Anjali J.; Watanabe, Shin; Hajjar, Roger J.; Ishikawa, Kiyotake] Icahn Sch Med Mt Sinai, Cardiovasc Res Inst, New York, NY 10029 USA. <br>
[Hajjar, Roger J.] Massachusetts Gen Brigham Gene &amp; Cell Therapy Inst, Cambridge, MA USA. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Ishikawa, K (corresponding author), Icahn Sch Med Mt Sinai, Cardiovasc Res Inst, New York, NY 10029 USA.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
kiyotake.ishikawa@mssm.edu</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Icahn School of Medicine at Mount Sinai</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Mavropoulos, Spyros</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-1328-1708&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Bikou, Olympia</display_name>&nbsp;</td><td>GZA-4320-2022&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Kariya, Taro</display_name>&nbsp;</td><td>AAF-4551-2021&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>NATURE PORTFOLIO</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>HEIDELBERGER PLATZ 3, BERLIN, 14197, GERMANY</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Multidisciplinary Sciences</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Science &amp; Technology - Other Topics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>P3Z4F</value>
</td>
</tr>

<tr>
<td>
<b>eISSN:</b>

<value>2041-1723</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>NAT COMMUN</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Nat. Commun.</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>12</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Heart and Lung Institute (NHLI)</grant_agency>&nbsp;</td><td>
<div>R01 HL139963&nbsp;</div>
<div>R01 HL173593&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>NIH</grant_agency>&nbsp;</td><td>
<div>T32HL007824-23&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Institutes of Health</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Gene Therapy Resource Program (GTRP) of the National Heart, Lung, and Blood Institute, National Institutes of Health</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Neuropathology Brain Bank and Research CoRE</grant_agency>&nbsp;</td><td>
<div>S10OD034330&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>NIH on the animal catheterization lab C-arm</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported by NIH R01 HL139963, R01 HL173593 (K.I.), and an Abiomed-sponsored research grant to the institution. S.A.M. was supported by the National Institutes of Health T32HL007824-23. We would like to acknowledge the Gene Therapy Resource Program (GTRP) of the National Heart, Lung, and Blood Institute, National Institutes of Health, for providing the gene vectors used in this study. For electron microscopy at the Icahn School of Medicine at Mount Sinai, we would like to acknowledge Allison Sowa and William Janssen of the Microscopy and Advanced Bioimaging CoRE for processing and image analysis and Dr. Ronald Gordon at the Department of Pathology for contributing expert opinion. For RNAscope at the Icahn School of Medicine at Mount Sinai, we would additionally like to acknowledge Dr. Claudia De Sanctis and the Neuropathology Brain Bank and Research CoRE. We would like to acknowledge the equipment funding from NIH on the animal catheterization lab C-arm (S10OD034330).</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>Green Submitted, gold</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 28 of 32</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>The Clock Mechanism Influences Neurobiology and Adaptations to Heart Failure in &lt;i&gt;Clock&lt;/i&gt;&lt;SUP&gt;Δ&lt;i&gt;19&lt;/i&gt;/Δ&lt;i&gt;19&lt;/i&gt;&lt;/SUP&gt; Mice With Implications for Circadian Medicine</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Duong, ATH (Duong, Austin T. H.); Reitz, CJ (Reitz, Cristine J.); Louth, EL (Louth, Emma L.); Creighton, SD (Creighton, Samantha D.); Rasouli, M (Rasouli, Mina); Zwaiman, A (Zwaiman, Ashley); Kroetsch, JT (Kroetsch, Jeffrey T.); Bolz, SS (Bolz, Steffen-Sebastian); Winters, BD (Winters, Boyer D.); Bailey, CDC (Bailey, Craig D. C.); Martino, TA (Martino, Tami A.)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>SCIENTIFIC REPORTS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>9</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>4994</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1038/s41598-019-41469-7</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2019 MAR 21</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>17</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>17</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
0</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
6</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>72</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
In this study we investigated the role of the circadian mechanism on cognition-relevant brain regions and neurobiological impairments associated with heart failure (HF), using murine models. We found that the circadian mechanism is an important regulator of healthy cognitive system neurobiology. Normal Clock(Delta 19/Delta 19) mice had neurons with smaller apical dendrite trees in the medial prefrontal cortex (mPFC), and hippocampus, showed impaired visual-spatial memory, and exhibited lower cerebrovascular myogenic tone, versus wild types (WT). We then used the left anterior descending coronary artery ligation model to investigate adaptations in response to HF. Intriguingly, adaptations to neuron morphology, memory, and cerebrovascular tone occurred in differing magnitude and direction between Clock(Delta 19/Delta 19) and WT mice, ultimately converging in HF. To investigate this dichotomous response, we performed microarrays and found genes crucial for growth and stress pathways that were altered in Clock(Delta 19/Delta 19) mPFC and hippocampus. Thus these data demonstrate for the first time that (i) the circadian mechanism plays a role in neuron morphology and function; (ii) there are changes in neuron morphology and function in HF; (iii) CLOCK influences neurobiological gene adaptations to HF at a cellular level. These findings have clinical relevance as patients with HF often present with concurrent neurocognitive impairments. There is no cure for HF, and new understanding is needed to reduce morbidity and improve the quality of life for HF patients.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000461886100003</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>30899044</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
GENE-EXPRESSION; CARDIOVASCULAR-DISEASE; MYOGENIC RESPONSIVENESS; CONTRACTILE FUNCTION; PERIRHINAL CORTEX; MURINE HEART; RHYTHMS; DISRUPTION; SLEEP; DEPRESSION</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Duong, Austin T. H.; Reitz, Cristine J.; Louth, Emma L.; Rasouli, Mina; Zwaiman, Ashley; Bailey, Craig D. C.; Martino, Tami A.] Univ Guelph, Ctr Cardiovasc Invest, Biomed Sci, Guelph, ON, Canada. <br>
[Creighton, Samantha D.; Winters, Boyer D.] Univ Guelph, Dept Psychol, Guelph, ON, Canada. <br>
[Kroetsch, Jeffrey T.; Bolz, Steffen-Sebastian] Univ Toronto, Dept Physiol, Toronto, ON, Canada. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Bailey, CDC; Martino, TA (corresponding author), Univ Guelph, Ctr Cardiovasc Invest, Biomed Sci, Guelph, ON, Canada.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
baileyc@uoguelph.ca; tmartino@uoguelph.ca</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
University of Guelph; University of Guelph; University of Toronto</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Louth, Emma</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-5357-1315&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Louth, Emma</display_name>&nbsp;</td><td>AAN-7517-2020&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Bailey, Craig</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-4320-6872&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>NATURE PORTFOLIO</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>HEIDELBERGER PLATZ 3, BERLIN, 14197, GERMANY</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Multidisciplinary Sciences</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Science &amp; Technology - Other Topics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>HP7RJ</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2045-2322</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>SCI REP-UK</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Sci Rep</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>18</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Canadian Institute of Health Research (CIHR)</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Heart and Stroke Foundation of Canada (HSFC)</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Natural Sciences and Engineering Research Council of Canada (NSERC)</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Canada Foundation for Innovation (CFI)</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported by funding to T.A.M. from the Canadian Institute of Health Research (CIHR), and Heart and Stroke Foundation of Canada (HSFC). T.A.M. is a Mid-Career Investigator of the Heart and Stroke Foundation of Ontario. Funding to C.D.C.B. was from the Natural Sciences and Engineering Research Council of Canada (NSERC) and the Canada Foundation for Innovation (CFI).</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>Green Published, gold</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 29 of 32</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Coronary heart disease prediction method fusing domain-adaptive transfer learning with graph convolutional networks (GCN)</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Lin, HZ (Lin, Huizhong); Chen, KZ (Chen, Kaizhi); Xue, YT (Xue, Yutao); Zhong, SP (Zhong, Shangping); Chen, LL (Chen, Lianglong); Ye, MF (Ye, Mingfang)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>SCIENTIFIC REPORTS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>13</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>1</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>14276</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1038/s41598-023-33124-z</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2023 AUG 31</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>6</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>6</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
3</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
22</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>40</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Graph convolutional networks (GCNs) have achieved impressive results in many medical scenarios involving graph node classification tasks. However, there are difficulties in transfer learning for graph representation learning and graph network models. Most GNNs work only in a single domain and cannot transfer the learned knowledge to other domains. Coronary Heart Disease (CHD) is a high-mortality disease, and there are non-public and significant differences in CHD datasets for current research, which makes it difficult to perform unified transfer learning. Therefore, in this paper, we propose a novel adversarial domain-adaptive multichannel graph convolutional network (DAMGCN) that can perform graph transfer learning on cross-domain tasks to achieve cross-domain medical knowledge transfer on different CHD datasets. First, we use a two-channel GCN model for feature aggregation using local consistency and global consistency. Then, a uniform node representation is generated for different graphs using an attention mechanism. Finally, we provide a domain adversarial module to decrease the discrepancies between the source and target domain classifiers and optimize the three loss functions in order to accomplish source and target domain knowledge transfer. The experimental findings demonstrate that our model performs best on three CHD datasets, and its performance is greatly enhanced by graph transfer learning.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001059852100028</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>37652917</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Lin, Huizhong; Chen, Lianglong; Ye, Mingfang] Fujian Med Univ, Fujian Inst Coronary Heart Dis, Fujian Heart Med Ctr, Dept Cardiol,Union Hosp, Fuzhou 350001, Peoples R China. <br>
[Chen, Kaizhi; Xue, Yutao; Zhong, Shangping] Fuzhou Univ, Coll Comp &amp; Data Sci, Fuzhou 350108, Fujian, Peoples R China. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Ye, MF (corresponding author), Fujian Med Univ, Fujian Inst Coronary Heart Dis, Fujian Heart Med Ctr, Dept Cardiol,Union Hosp, Fuzhou 350001, Peoples R China.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
xieheyemingfang@163.com</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Fujian Medical University; Fuzhou University</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>ye, mingfang</display_name>&nbsp;</td><td>HKW-8384-2023&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>NATURE PORTFOLIO</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>HEIDELBERGER PLATZ 3, BERLIN, 14197, GERMANY</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Multidisciplinary Sciences</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Science &amp; Technology - Other Topics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>Q8GM9</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2045-2322</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>SCI REP-UK</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Sci Rep</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>14</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>National Natural Science Foundation of China (NSFC)</grant_agency>&nbsp;</td><td>
<div>61972187&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Natural Science Foundation of Fujian Province, China</grant_agency>&nbsp;</td><td>
<div>2022J01119&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Fujian Province Young and Middle-aged Teacher Education Research Project</grant_agency>&nbsp;</td><td>
<div>JAT200004&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Fujian provincial health technology project</grant_agency>&nbsp;</td><td>
<div>2021TG008&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Joint Funds for the Innovation of Science and Technology, Fujian province</grant_agency>&nbsp;</td><td>
<div>2020Y9069&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work is supported by the National Natural Science Foundation of China (NSFC) under Grant 61972187, Natural Science Foundation of Fujian Province, China (Grant No.2022J01119), Fujian Province Young and Middle-aged Teacher Education Research Project under Grant JAT200004. Data collection and preliminary analysis were sponsored by the Fujian provincial health technology project (No. 2021TG008) and the Joint Funds for the Innovation of Science and Technology, Fujian province (No.2020Y9069).</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>gold, Green Published</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 30 of 32</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Evaluation of transfer learning in deep convolutional neural network models for cardiac short axis slice classification</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Ho, N (Ho, Namgyu); Kim, YC (Kim, Yoon-Chul)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>SCIENTIFIC REPORTS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>11</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>1</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>1839</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1038/s41598-021-81525-9</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2021 JAN 19</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>28</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>33</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
1</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
6</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>44</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
In computer-aided analysis of cardiac MRI data, segmentations of the left ventricle (LV) and myocardium are performed to quantify LV ejection fraction and LV mass, and they are performed after the identification of a short axis slice coverage, where automatic classification of the slice range of interest is preferable. Standard cardiac image post-processing guidelines indicate the importance of the correct identification of a short axis slice range for accurate quantification. We investigated the feasibility of applying transfer learning of deep convolutional neural networks (CNNs) as a means to automatically classify the short axis slice range, as transfer learning is well suited to medical image data where labeled data is scarce and expensive to obtain. The short axis slice images were classified into out-of-apical, apical-to-basal, and out-of-basal, on the basis of short axis slice location in the LV. We developed a custom user interface to conveniently label image slices into one of the three categories for the generation of training data and evaluated the performance of transfer learning in nine popular deep CNNs. Evaluation with unseen test data indicated that among the CNNs the fine-tuned VGG16 produced the highest values in all evaluation categories considered and appeared to be the most appropriate choice for the cardiac slice range classification.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000676336800085</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>33469077</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
LEFT-VENTRICLE; ARCHITECTURES</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Ho, Namgyu] Sogang Univ, Dept Comp Sci &amp; Engn, Seoul, South Korea. <br>
[Kim, Yoon-Chul] Sungkyunkwan Univ, Samsung Med Ctr, Clin Res Inst, Sch Med, 81 Irwon Ro, Seoul 06351, South Korea. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Kim, YC (corresponding author), Sungkyunkwan Univ, Samsung Med Ctr, Clin Res Inst, Sch Med, 81 Irwon Ro, Seoul 06351, South Korea.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
yoonckim1@gmail.com</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
Sogang University; Sungkyunkwan University (SKKU); Samsung Medical Center</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Kim, Yoon-Chul</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-5012-0268&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Kim, Eung Yeop</display_name>&nbsp;</td><td>A-9807-2013&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>NATURE PORTFOLIO</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>HEIDELBERGER PLATZ 3, BERLIN, 14197, GERMANY</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Multidisciplinary Sciences</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Science &amp; Technology - Other Topics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>TN6IT</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2045-2322</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>SCI REP-UK</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Sci Rep</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>11</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Basic Science Research Program through the National Research Foundation of Korea</grant_agency>&nbsp;</td><td>
<div>NRF-2018 R1D1A1B07042692&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported by the Basic Science Research Program through the National Research Foundation of Korea (NRF-2018 R1D1A1B07042692).</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>Green Published, gold</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 31 of 32</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Low dose Adenoviral Vammin gene transfer induces myocardial angiogenesis and increases left ventricular ejection fraction in ischemic porcine heart</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
J&auml;rvel&auml;inen, N (Jarvelainen, Niko); Halonen, PJ (Halonen, Paavo J.); Nurro, J (Nurro, Jussi); Kuivanen, A (Kuivanen, Antti); Pajula, J (Pajula, Juho); Tarkia, M (Tarkia, Miikka); Gr&ouml;nman, M (Gronman, Maria); Saraste, A (Saraste, Antti); Laakkonen, J (Laakkonen, Johanna); Toivanen, P (Toivanen, Pyry); Nieminen, T (Nieminen, Tiina); Rissanen, TT (Rissanen, Tuomas T.); Knuuti, J (Knuuti, Juhani); Yl&auml;-Herttuala, S (Yla-Herttuala, Seppo)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>SCIENTIFIC REPORTS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>14</value>&nbsp;&nbsp;<b>Issue:</b> 
<value>1</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>30003</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1038/s41598-024-81773-5</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2024 DEC 3</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>0</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>0</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
0</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
1</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>37</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
This preliminary study investigated if VEGFR-2 selective adenoviral Vammin (AdVammin) gene therapy could induce angiogenesis and increase perfusion in the healthy porcine myocardium. Also, we determined using a clinically relevant large animal model if AdVammin gene therapy could improve the function of a chronically ischemic heart. Low doses of AdVammin (dose range 2 x 109-2 x 1010 vp) gene transfers were performed into the porcine myocardium using an endovascular injection catheter. AdCMV was used as a control. The porcine model of chronic myocardial ischemia was used in the ischemic studies. The AdVammin enlarged the mean capillary area and stimulated pericyte coverage in the target area 6 days after the gene transfers. Using positron emission tomography 15O-radiowater imaging, we demonstrated that AdVammin gene therapy increased perfusion in healthy myocardium at rest. AdVammin treatment also increased ejection fraction at stress in the ischemic heart, as detected using left ventricular cine angiography. In addition, we demonstrated successful in vivo imaging of enhanced angiogenesis using [68Ga]NODAGA-RGD peptide. However, AdVammin also increased tissue permeability and was associated with significant pericardial fluid accumulation, limiting AdVammin's therapeutic potential and emphasizing the importance of correct dosage.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:001376826900006</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>39623213</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
CELL BEHAVIOR; PERMEABILITY; THERAPY; PERFUSION; DISEASES; MUSCLE; VENOM</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Jarvelainen, Niko; Halonen, Paavo J.; Nurro, Jussi; Kuivanen, Antti; Pajula, Juho; Laakkonen, Johanna; Toivanen, Pyry; Nieminen, Tiina; Rissanen, Tuomas T.; Yla-Herttuala, Seppo] Univ Eastern Finland, AI Virtanen Inst Mol Sci, POB 1627, Kuopio 70211, Finland. <br>
[Tarkia, Miikka; Gronman, Maria; Saraste, Antti; Knuuti, Juhani] Univ Turku, Turku PET Ctr, Turku, Finland. <br>
[Tarkia, Miikka] Univ Helsinki, Helsinki, Finland. <br>
[Rissanen, Tuomas T.] North Karel Cent Hosp, Heart Ctr, Joensuu, Finland. <br>
[Yla-Herttuala, Seppo] Kuopio Univ Hosp, Heart Ctr, Kuopio, Finland. <br>
[Yla-Herttuala, Seppo] Kuopio Univ Hosp, Gene Therapy Unit, Kuopio, Finland. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Yl&auml;-Herttuala, S (corresponding author), Univ Eastern Finland, AI Virtanen Inst Mol Sci, POB 1627, Kuopio 70211, Finland.<br>Yl&auml;-Herttuala, S (corresponding author), Kuopio Univ Hosp, Heart Ctr, Kuopio, Finland.<br>Yl&auml;-Herttuala, S (corresponding author), Kuopio Univ Hosp, Gene Therapy Unit, Kuopio, Finland.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
seppo.ylaherttuala@uef.fi</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
University of Eastern Finland; University of Turku; University of Helsinki; University of Eastern Finland; University of Eastern Finland Hospital; Kuopio University Hospital; Kuopio University Hospital; University of Eastern Finland; University of Eastern Finland Hospital</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Laakkonen, Johanna</display_name>&nbsp;</td><td>IUN-4631-2023&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Rissanen, Tuomas</display_name>&nbsp;</td><td>KII-1122-2024&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Jarvelainen, Niko</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0002-5250-816X&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Knuuti, Juhani</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-3156-9593&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>NATURE PORTFOLIO</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>HEIDELBERGER PLATZ 3, BERLIN, 14197, GERMANY</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Multidisciplinary Sciences</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Science &amp; Technology - Other Topics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>P3G1K</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2045-2322</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>SCI REP-UK</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Sci Rep</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>12</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Academy of Finland</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Sigrid Juselius Foundation</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>European Research Council (ERC)</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Finnish Foundation for Cardiovascular Research</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Ida Montin Foundation</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Aarne Koskelo Foundation</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Maud Kuistila Memory Foundation</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Finnish Medical Foundation</grant_agency>&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>This work was supported by the Academy of Finland, Sigrid Juselius Foundation, European Research Council (ERC) Advanced Grant, and Finnish Foundation for Cardiovascular Research. PJH received grants from the Finnish Cultural Foundation, Ida Montin Foundation, Aarne Koskelo Foundation, Maud Kuistila Memory Foundation and Finnish Medical Foundation. ESFRI - EATRIS infrastructure, Biocenter Kuopio and National Virus Vector Laboratories (University of Eastern Finland, Kuopio, Finland) were used for adenovirus vector production.</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>Green Submitted, gold</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr><table>
<tr>
<td><b>Record 32 of 32</b></td>
</tr>
<tr>
<td>
<b>Title:</b>

<value>Non-invasive detection of adeno-associated viral gene transfer using a genetically encoded CEST-MRI reporter gene in the murine heart</value>
</td>
</tr>

<tr>
<td>
<b>Author(s):</b>
Meier, S (Meier, Shelby); Gilad, AA (Gilad, Assaf A.); Brandon, JA (Brandon, J. Anthony); Qian, CH (Qian, Chenghao); Gao, E (Gao, Erhe); Abisambra, JF (Abisambra, Jose F.); Vandsburger, M (Vandsburger, Moriel)</td>
</tr>

<tr>
<td><b>Source:</b> 
<value>SCIENTIFIC REPORTS</value>&nbsp;&nbsp;<b>Volume:</b> 
<value>8</value>&nbsp;&nbsp;<b>Article Number:</b> 
<value>4638</value>&nbsp;&nbsp;<b>DOI:</b> 
<value>10.1038/s41598-018-22993-4</value>&nbsp;&nbsp;<b>Published Date:</b> 
<value>2018 MAR 15</value>&nbsp;&nbsp;</td>
</tr>
<tr>
<td>
<b>Times Cited in Web of Science Core Collection:</b>

<value>20</value>
</td>
</tr>

<tr>
<td>
<b>Total Times Cited:</b>

<value>25</value>
</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Last 180 days):</b>
0</td>
</tr>

<tr>
<td>
<b>Usage Count
    (Since 2013):</b>
17</td>
</tr>

<tr>
<td>
<b>Cited Reference Count:</b>

<value>37</value>
</td>
</tr>

<tr>
<td>
<b>Abstract:</b>
Research into gene therapy for heart failure has gained renewed interest as a result of improved safety and availability of adeno-associated viral vectors (AAV). While magnetic resonance imaging (MRI) is standard for functional assessment of gene therapy outcomes, quantitation of gene transfer/expression relies upon tissue biopsy, fluorescence or nuclear imaging. Imaging of gene expression through the use of genetically encoded chemical exchange saturation transfer (CEST)-MRI reporter genes could be combined with clinical cardiac MRI methods to comprehensively probe therapeutic gene expression and subsequent outcomes. The CEST-MRI reporter gene Lysine Rich Protein (LRP) was cloned into an AAV9 vector and either administered systemically via tail vein injection or directly injected into the left ventricular free wall of mice. Longitudinal in vivo CEST-MRI performed at days 15 and 45 after direct injection or at 1, 60 and 90 days after systemic injection revealed robust CEST contrast in myocardium that was later confirmed to express LRP by immunostaining. Ventricular structure and function were not impacted by expression of LRP in either study arm. The ability to quantify and link therapeutic gene expression to functional outcomes can provide rich data for further development of gene therapy for heart failure.</td>
</tr>

<tr>
<td>
<b>Accession Number:</b>

<value>WOS:000427465800018</value>
</td>
</tr>

<tr>
<td>
<b>PubMed ID:</b>

<value>29545551</value>
</td>
</tr>

<tr>
<td>
<b>Language:</b>
English</td>
</tr>

<tr>
<td>
<b>Document Type:</b>
Article</td>
</tr>

<tr>
<td>
<b>KeyWords Plus:</b>
CELL-SURVIVAL; EXPRESSION; INJECTION; TRACKING; THERAPY; TRIAL; PROBE; MICE</td>
</tr>

<tr>
<td>
<b>Addresses:</b>
[Meier, Shelby; Brandon, J. Anthony; Qian, Chenghao; Abisambra, Jose F.] Univ Kentucky, Dept Physiol, Lexington, KY USA. <br>
[Meier, Shelby; Abisambra, Jose F.] Univ Kentucky, Sanders Brown Ctr Aging, Lexington, KY 40536 USA. <br>
[Gilad, Assaf A.] Johns Hopkins Univ, Dept Radiol, Baltimore, MD USA. <br>
[Gilad, Assaf A.] Johns Hopkins Univ, Inst Cell Engn, Baltimore, MD USA. <br>
[Gilad, Assaf A.] Michigan State Univ, Dept Biomed Engn, E Lansing, MI 48824 USA. <br>
[Gilad, Assaf A.] Michigan State Univ, Inst Quantitat Hlth Sci &amp; Engn, E Lansing, MI 48824 USA. <br>
[Gilad, Assaf A.] Michigan State Univ, Dept Radiol, E Lansing, MI 48824 USA. <br>
[Gao, Erhe] Temple Univ, Ctr Translat Med, Philadelphia, PA 19122 USA. <br>
[Vandsburger, Moriel] Univ Calif Berkeley, Dept Bioengn, Berkeley, CA 94720 USA. </td>
</tr>

<tr>
<td>
<b>Corresponding Address:</b>
Vandsburger, M (corresponding author), Univ Calif Berkeley, Dept Bioengn, Berkeley, CA 94720 USA.</td>
</tr>

<tr>
<td>
<b>E-mail Addresses:</b>
moriel@berkeley.edu</td>
</tr>

<tr>
<td>
<b>Affiliations:</b>
University of Kentucky; University of Kentucky; Johns Hopkins University; Johns Hopkins University; Michigan State University; Michigan State University; Michigan State University; Pennsylvania Commonwealth System of Higher Education (PCSHE); Temple University; University of California System; University of California Berkeley</td>
</tr>

<tr>
<td><span class="FR_label"><b>Author Identifiers:</b></span></td>
</tr>
<tr>
<td><span style="display" name="show_resc_blurb" id="show_resc_blurb">
<table class="FR_table_borders" border="1" cellpadding="0" cellspacing="0">
<tr>
<th><span class="FR_label">Author</span></th><th><span class="FR_label">Web of Science ResearcherID</span></th><th><span class="FR_label">ORCID Number</span></th>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Meier, Shelby</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0003-1946-9004&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Brandon, J</display_name>&nbsp;</td><td>LKK-3754-2024&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Abisambra, Jose</display_name>&nbsp;</td><td>IAM-0948-2023&nbsp;</td><td>&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Abisambra, Jose</display_name>&nbsp;</td><td>&nbsp;</td><td>0000-0001-6341-679X&nbsp;</td>
</tr>
<tr class="fr_data_row">
<td>
<display_name>Qian, Chenghao</display_name>&nbsp;</td><td>LFT-4537-2024&nbsp;</td><td>&nbsp;</td>
</tr>
</table>
</span></td>
</tr>
<tr>
<td>
<b>Publisher:</b>

<value>NATURE PORTFOLIO</value>
</td>
</tr>

<tr>
<td>
<b>Publisher Address:</b>

<value>HEIDELBERGER PLATZ 3, BERLIN, 14197, GERMANY</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Index:</b>

<value>Science Citation Index Expanded (SCI-EXPANDED)</value>
</td>
</tr>

<tr>
<td>
<b>Web of Science Categories:</b>
Multidisciplinary Sciences</td>
</tr>

<tr>
<td>
<b>Research Areas:</b>
Science &amp; Technology - Other Topics</td>
</tr>

<tr>
<td>
<b>IDS Number:</b>

<value>FZ3DF</value>
</td>
</tr>

<tr>
<td>
<b>ISSN:</b>

<value>2045-2322</value>
</td>
</tr>

<tr>
<td>
<b>29-char Source Abbrev.:</b>

<value>SCI REP-UK</value>
</td>
</tr>

<tr>
<td>
<b>ISO Source Abbrev.:</b>

<value>Sci Rep</value>
</td>
</tr>

<tr>
<td>
<b>Source Item Page Count:</b>

<value>10</value>
</td>
</tr>

<tr>
<td class="fr_data_row"><span class="FR_label"><b>Funding:</b></span></td>
</tr>
<tr>
<td>
<table cellpadding="3" cellspacing="0" border="1">
<tr>
<td class="fr_dataTable_head" nowrap="true" valign="top"><b>Funding Agency</b></td><td class="fr_dataTable_head" nowrap="true" valign="top"><b>Grant Number</b></td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>NIH</grant_agency>&nbsp;</td><td>
<div>1R01HL28592-01&nbsp;</div>
<div>P20GM103527-07sub5039&nbsp;</div>
<div>1R01NS091329-01&nbsp;</div>
</td>
</tr>
<tr class="fr_data_row">
<td>
<grant_agency>Maryland Stem Cell Research Foundation</grant_agency>&nbsp;</td><td>
<div>MSCRFII-0042&nbsp;</div>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td class="fr_data_row"><span name="show_fund_blurb" id="show_fund_blurb">
<p>MHV is supported by NIH 1R01HL28592-01 and NIH P20GM103527-07sub5039, AAG is supported by Maryland Stem Cell Research Foundation (MSCRFII-0042). JFA is supported by NIH 1R01NS091329-01.</p>
</span></td>
</tr>
<tr>
<td>
<b>Open Access:</b>

<value>Green Submitted, gold</value>
</td>
</tr>

<tr>
<td>
<b>Output Date:</b>

<value>2025-10-01</value>
</td>
</tr>

</table><hr>
<table xmlns:bean="http://ts.thomson.com/ua/bean">
<tr>
<td>End of File</td>
</tr>
</table><div style="background-color: #000000; padding: 10px; color: #FFFFFF; font-family: arial, sans-serif;">
<div style="height: 25px; width: 100px;">
<mat-icon data-mat-icon-name="clarivate" data-mat-icon-type="svg" aria-hidden="true" class="mat-icon notranslate clarivate-logo mat-icon-no-color" svgicon="clarivate" role="img">
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" focusable="false" preserveAspectRatio="xMidYMid meet" version="1.1" viewBox="0 0 92 18" height="100%" width="100%">
<title>Clarivate</title>
<g fill-rule="evenodd" fill="none" stroke-width="1" stroke="none">
<g fill-rule="nonzero" fill="#FFFFFF" transform="translate(-19.000000, -8.000000)">
<g transform="translate(1.000000, 1.000000)">
<g transform="translate(18.000000, 7.000000)">
<path d="M26.6812194,13.2997252 C28.2949641,13.3845068 29.7591419,12.3596036 30.2318671,10.8143235 L33.0910469,10.8143235 C32.3628779,13.742285 29.6973379,15.7705498 26.6812194,15.6916944 C24.8651813,15.7485215 23.1057028,15.0559504 21.8156554,13.7764967 C20.5256081,12.4970431 19.8185526,10.7433351 19.8604035,8.9268901 C19.8185367,7.11043445 20.5255848,5.35671042 21.8156337,4.07724261 C23.1056826,2.7977748 24.8651702,2.10519376 26.6812194,2.16201887 C29.6973451,2.0831702 32.3628861,4.11144761 33.0910469,7.03941847 L30.2318671,7.03941847 C29.7591419,5.49413828 28.2949641,4.46923515 26.6812194,4.55401674 C25.5407067,4.54133389 24.4470406,5.00703241 23.6657805,5.83803342 C22.8845203,6.66903442 22.487126,7.78932829 22.5700912,8.9268901 C22.4871313,10.064447 22.8845287,11.184734 23.6657888,12.0157277 C24.447049,12.8467215 25.540712,13.312413 26.6812194,13.2997252 L26.6812194,13.2997252 Z"/>
<polygon points="36.8477342 15.4674945 34.4371264 15.4674945 34.4371264 2.38628571 36.8477342 2.38628571"/>
<path d="M43.5380316,9.76781183 C43.5380316,9.15114738 43.0522397,8.75870193 42.1178663,8.75870193 C41.2070757,8.82048057 40.3236107,9.0957548 39.5389268,9.5622602 L38.5858286,7.73092465 C39.6930919,7.06514423 40.9567617,6.70409618 42.2486241,6.68441302 C44.1921648,6.68441302 45.9674886,7.58137515 45.9674886,9.84255787 L45.9674886,15.4674945 L43.5380316,15.4674945 L43.5380316,14.6639362 C42.9494098,15.3673061 42.0613957,15.7488608 41.146005,15.6917231 C40.3900371,15.7421039 39.645831,15.484584 39.0826798,14.9777443 C38.5195286,14.4709046 38.1853154,13.7578403 38.1560675,13.0007601 C38.1560675,11.3749188 39.3519468,10.3658089 41.2581433,10.3658089 L42.9400537,10.3658089 C43.3511665,10.3658089 43.5379934,10.1789247 43.5379934,9.86128266 L43.5380316,9.76781183 Z M43.5380316,12.4027631 L43.5380316,11.786089 C43.321275,11.9402969 43.0543042,12.0070182 42.790485,11.9729159 L41.8748173,11.9729159 C41.1086607,11.9729159 40.6602036,12.3092922 40.6602036,12.9072893 C40.6602036,13.486619 41.1272707,13.9164566 41.8748173,13.9164566 C42.303867,13.9699481 42.7348632,13.8332333 43.0546312,13.5422121 C43.3743992,13.2511908 43.5509896,12.8349402 43.5380316,12.4027631 L43.5380316,12.4027631 Z"/>
<path d="M52.6579104,9.3006299 L51.6301236,9.3006299 C50.3966703,9.3006299 49.8361037,9.8799596 49.8361037,11.3749188 L49.8361037,15.4674945 L47.4066179,15.4674945 L47.4066179,6.90863203 L49.7612715,6.90863203 L49.7612715,8.27284306 C50.1001881,7.34759017 51.0007831,6.74968637 51.9850429,6.79648425 C52.2115562,6.79362484 52.4375557,6.81874947 52.6579104,6.87128771 L52.6579104,9.3006299 Z"/>
<path d="M57.4232866,15.4674945 L54.9939445,15.4674945 L54.9939445,8.96425355 L53.7979312,8.96425355 L53.7979312,6.90863203 L57.4232866,6.90863203 L57.4232866,15.4674945 Z M56.0964104,2.38628347 C56.5129556,2.38558064 56.9126431,2.55074046 57.2071855,2.84528284 C57.5017278,3.13982521 57.6668877,3.53951271 57.6661848,3.95605794 C57.6609778,4.814152 56.9638874,5.50702694 56.1057775,5.50702694 C55.2476677,5.50702694 54.5505773,4.814152 54.5453585,3.95605794 C54.5417726,3.09491157 55.2352828,2.39302469 56.0964104,2.38628347 Z"/>
<polygon points="62.8802456 12.8512776 64.8236715 6.90863203 67.3838767 6.90863203 64.33787 15.4674945 61.3292077 15.4674945 58.4139588 6.90863203 61.0488526 6.90863203"/>
<path d="M72.8783522,9.76781183 C72.8783522,9.15114738 72.3924359,8.75870193 71.4580625,8.75870193 C70.5472784,8.82049943 69.6638208,9.09577238 68.8791325,9.5622602 L67.9261491,7.73092465 C69.0333627,7.06513461 70.2969953,6.70408494 71.5888202,6.68441302 C73.5323609,6.68441302 75.3076848,7.58137515 75.3076848,9.84255787 L75.3076848,15.4674945 L72.8783522,15.4674945 L72.8783522,14.6639362 C72.2896971,15.3672586 71.4017035,15.7487939 70.4863256,15.6916944 C69.7303436,15.7420941 68.986117,15.4845835 68.4229464,14.9777426 C67.8597758,14.4709017 67.5255471,13.7578265 67.4962924,13.0007314 C67.4962924,11.3748901 68.6923056,10.3657802 70.5984925,10.3657802 L72.2802785,10.3657802 C72.6913914,10.3657802 72.8783426,10.1788959 72.8783426,9.86125395 L72.8783522,9.76781183 Z M72.8783522,12.4027631 L72.8783522,11.786089 C72.6615874,11.9402794 72.3946236,12.006999 72.1308056,11.9729159 L71.2151378,11.9729159 C70.4488664,11.9729159 70.0004093,12.3092922 70.0004093,12.9072893 C70.0004093,13.486619 70.4676008,13.9164566 71.2151378,13.9164566 C71.6441833,13.9699279 72.0751679,13.8332061 72.394931,13.5421893 C72.7146941,13.2511724 72.8912904,12.8349341 72.8783522,12.4027631 L72.8783522,12.4027631 Z"/>
<path d="M79.9985637,3.65701635 L79.9985637,6.90863203 L81.8485762,6.90863203 L81.8485762,8.96425355 L79.9985637,8.96425355 L79.9985637,12.8325433 C79.9985637,13.3745286 80.4282865,13.4492747 80.8020646,13.4492747 C81.418729,13.4492747 81.9232553,13.3932056 81.9232553,13.3932056 L81.9232553,15.4674945 C81.9232553,15.4674945 80.7459859,15.5796423 80.0357932,15.5796423 C78.8211795,15.5796423 77.5691354,15.3740811 77.5691354,13.6548263 L77.5691354,8.96425355 L76.22363,8.96425355 L76.22363,6.90863203 L77.5691163,6.90863203 L77.5691163,3.65701635 L79.9985637,3.65701635 Z"/>
<path d="M91.5475359,11.8981794 L85.1751771,11.8981794 C85.3031552,12.7204351 85.9194738,13.3815958 86.7307322,13.5669136 C87.5419905,13.7522314 88.3842592,13.4242593 88.856573,12.7391298 L91.3793191,13.0007601 C90.4933306,14.9536735 88.3725856,16.0321989 86.2725384,15.5978628 C84.1724912,15.1635268 82.6535603,13.3322361 82.6149337,11.188092 C82.567807,9.98960603 83.0193435,8.82515843 83.8621298,7.97175307 C84.7049161,7.1183477 85.8636189,6.65226701 87.0626009,6.68438432 C89.4171397,6.68438432 91.5475359,8.21674525 91.5475359,11.8047373 L91.5475359,11.8981794 Z M85.1189837,10.2723955 L88.9499864,10.2723955 C88.8059978,9.34848922 87.9973156,8.67588427 87.0626392,8.70263282 C86.1147409,8.67034807 85.2869107,9.33893202 85.1189837,10.2723955 Z"/>
<path d="M12.6820274,9.85132226 C10.6885049,11.8556057 8.11575776,13.1830474 5.32707189,13.6462054 C5.77780891,14.8758828 6.40559556,16.0331959 7.19058498,17.0815575 C10.3903106,16.2989026 13.3004311,14.619851 15.5794509,12.2414352 C15.0651988,11.0374085 14.3787129,9.91449973 13.5415112,8.90790698 C13.271174,9.23328 12.9846793,9.54772625 12.6820274,9.85124571"/>
<path d="M5.32722498,13.6460332 C4.21304566,10.5852611 4.22627578,7.22758116 5.3645406,4.17568425 C4.07298567,3.95802358 2.75701252,3.92542943 1.45626578,4.07888372 C0.536506305,7.22972054 0.525090565,10.5762272 1.42333236,13.7332656 C2.72217338,13.8896343 4.03666286,13.8602685 5.32722498,13.6460524"/>
<path d="M7.25919787,0.751096346 C6.46424104,1.79423541 5.8259601,2.94792723 5.36457887,4.17561728 C8.13918123,4.64457901 10.6977043,5.96938232 12.6820178,7.96458738 C12.9850078,8.2673988 13.2715025,8.58184824 13.5415017,8.90793568 C14.3788161,7.90142183 15.0653154,6.77849641 15.5794509,5.57440744 C13.3171586,3.21319806 10.4324457,1.54091896 7.25919787,0.751096346"/>
</g>
</g>
</g>
</g>
</svg>
</mat-icon>
</div>
<path d="M5.32722498,13.6460332 C4.21304566,10.5852611 4.22627578,7.22758116 5.3645406,4.17568425 C4.07298567,3.95802358 2.75701252,3.92542943 1.45626578,4.07888372 C0.536506305,7.22972054 0.525090565,10.5762272 1.42333236,13.7332656 C2.72217338,13.8896343 4.03666286,13.8602685 5.32722498,13.6460524"></path>
<path d="M7.25919787,0.751096346 C6.46424104,1.79423541 5.8259601,2.94792723 5.36457887,4.17561728 C8.13918123,4.64457901 10.6977043,5.96938232 12.6820178,7.96458738 C12.9850078,8.2673988 13.2715025,8.58184824 13.5415017,8.90793568 C14.3788161,7.90142183 15.0653154,6.77849641 15.5794509,5.57440744 C13.3171586,3.21319806 10.4324457,1.54091896 7.25919787,0.751096346"></path>
</div>
